{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gensim.models.keyedvectors import KeyedVectors\n",
    "\n",
    "\n",
    "model = KeyedVectors.load_word2vec_format('covid-19.embeddings')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pickle\n",
    "with open(\"tokenizer.pickle\", \"rb\") as f:\n",
    "    tokenizer = pickle.load(f)\n",
    "    \n",
    "with open('document_mappping.pkl', \"rb\") as f:\n",
    "    documents = pickle.load(f)\n",
    "    \n",
    "    \n",
    "word_idx = tokenizer.word_index\n",
    "inv_map = {v: k for k, v in word_idx.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def similarity_finder(inp_id):\n",
    "    results = []\n",
    "    similarity = model.most_similar(str(inp_id), topn=1000)\n",
    "    for similar in similarity:\n",
    "        if not similar[0].startswith(\"d\"):\n",
    "            word = inv_map[int(similar[0])]\n",
    "            results.append((word, similar[1]))\n",
    "        else:\n",
    "            results.append((documents[similar[0]], similar[1]))\n",
    "    return results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('The virus specific intracellular RNA specie two murine coronaviruses MHV A59 MHV JHM Seven virus specific polyadenylated RNA specie identify mouse cell infect murine coronaviruses MHV A59 A59V MHV JHM JHMV MHV infect 17CL·1 cell label 32P orthophosphate presence actinomycin cytoplasmic RNA extract analyze agarose gel electrophoresis These RNA specie range size dalton The A59V JHMV specific RNAs identical molecular weight comigrate agarose gel The large intracellular RNA specie identical RNA isolate purified virion determine agarose gel electrophoresis oligonucleotide fingerprint study ribonuclease digest Oligonucleotide fingerprint six subgenomic RNAS show sequence contain present virion RNA confirm virus specific nature The fingerprinting study also demonstrate six subgenomic RNA specie make nested set The sequence present RNA specie also present large RNA specie These large RNAs also contain additional sequence consistent great size The subgenomic RNAs fulfull many criterion mRNAs Possible mechanism generate RNAs discuss',\n",
       "  0.860636293888092),\n",
       " ('Temporal regulation bovine coronavirus RNA synthesis The structure synthesis bovine coronavirus BCV specific intracellular RNA study genome size RNA seven subgenomic RNAs molecular weight approximately detected Comparisons BCV intracellular RNAs mouse hepatitis virus MHV demonstrate presence additional RNA BCV species dalton BCV RNAs contain nest set structure similar coronaviruses This nested set structure would suggest new RNA capacity encode protein approximately amino acid Kinetic study demonstrate synthesis subgenomic mRNAs genomic RNA differentially regulate post infection subgenomic RNAs synthesize maximal rate represent great total viral RNA synthesize whereas genome size RNA account Later infection genome size RNA much abundant account total RNA synthesize Immunoprecipitations 35S methionine pulse label viral protein demonstrate viral protein synthesis occur early infection concurrent peak viral subgenomic RNA synthesis Western blot analysis suggest proteins stable since protein present high level late The kinetics production virus particle coincides synthesis genomic RNA These study thus indicate differential temporal regulation synthesis genomic RNA subgenomic mRNAs synthesis genomic RNA rate limit step regulate production virus particle',\n",
       "  0.8026864528656006),\n",
       " ('lph', 0.7994921207427979),\n",
       " ('Nascent Synthesis Leader Sequence Containing Subgenomic mRNAs Coronavirus Genome Length Replicative Intermediate RNA Infection coronavirus result accumulation genomic size mRNA six eight subgenomic mRNAs make coterminal nest set structure Genome length negative strand RNA subgenomic length negative strand RNAs correspond subgenomic mRNAs also accumulate infected cell The present study examine whether genome length negative strand RNA serve template subgenomic mRNA synthesis Genome length replicative intermediate RNA purify two dimensional gel electrophoresis intracellular RNAs cell infect mouse hepatitis virus RNase treatment purify genome length result production genome length replicative form RNA indicate genome length include genome length template RNA RNase protection assay use purified genome length two probe correspond region mRNA region mRNA show presence nascent leader sequence contain subgenomic mRNAs genome length These data demonstrate genome length negative strand RNA serve template subgenomic mRNA synthesis',\n",
       "  0.7921939492225647),\n",
       " ('Physical mapping human cytomegalovirus gene Identification DNA sequence cod virion phosphoprotein kDa viral kDa polypeptide Polyadenylated RNA isolate fibroblast culture infect human cytomegalovirus HCMV strain AD169 late phase viral replication The RNA select hybridization series cosmid clone contain entire viral genome partially overlap segment Translation RNA reticulocyte cell free system allow map virus specific polypeptide Nine polypeptide synthesize vitro comigrated major virion structural proteins vitro translate protein kDa precipitate monoclonal antibody direct phosphorylated internal envelope protein kDa The map coordinate viral DNA cod phosphoprotein localize hybrid selection subcloned DNA fragment direction transcription determine hybrid selection single strand DNA clone bacteriophage vector M13mp9 vitro translation size fractionate RNA combine immunoprecipitation Northern blot analysis indicate mRNA encode kDa phosphoprotein mRNA size map coordinate orientation translate abundant kDa polypeptide size major structural phosphoprotein HCMV',\n",
       "  0.7904478907585144),\n",
       " ('MHV nucleocapsid synthesis presence cycloheximide accumulation negative strand MHV RNA find genomic RNA synthesis inhibit cycloheximide cell infect mouse hepatitis virus strain A59 MHV A59 agreement previously publish result Sawicki Sawicki Virol 328–334 present study fate residual genomic RNA synthesize presence cycloheximide determine Nearly genomic RNA synthesize presence drug incorporate nucleocapsid structure suggest even absence protein synthesis genomic RNA synthesis encapsidation couple MHV infect cell Sufficient free nucleocapsid protein available purpose since pool soluble protein determine decay half life approximately one hour Negative strand RNA template synthesis genomic subgenomic positive strand RNA would predict accumulate primarily early phase lytic cycle agreement prediction negative strand RNA accumulate first 5–6 infection little additional accumulation occur next marked contrast positive strand RNA increase 5–6 fold period These result take conjunction publish data suggest negative strand RNA synthesize early period infectious cycle stable infected cell also suggest treatment cycloheximide late time inhibit positive strand RNA synthesis indirectly block formation negative strand template',\n",
       "  0.7782983779907227),\n",
       " ('reducing', 0.7762653827667236),\n",
       " ('RNA polypeptide homology among murine coronaviruses Using 32P complementary DNA cDNA prepare A59 nucleocapsid protein messenger RNA investigate extent homology A59 four strain murine hepatitis virus MHV Analysis hybridization kinetics anneal A59 32P cDNA infect cell RNA four MHV strain demonstrate 70–80 homology gel transfer analysis A59 32P cDNA able detect subgenomic size virus specific RNAs cell infect five MHV strain similar pattern seven viral RNAs detect cell infect A59 MHV MHV JHM contrast cell infect MHV contain seven virus specific RNAs two small specie comigrated RNAs four strain The result suggest previously show A59 Cheley Anderson Cupples Lee Chan Morris Virology 596–604 MHV strain test encode nest set subgenomic RNAs Analysis 35S methionine label viral protein SDS polyacrylamide gel electrophoresis reveal strain MHV specify four major viral polypeptide apparent molecular weight similar previously report PEI polypeptides A59 The strong degree interstrain homology among five MHV strain investigate confirmed comparative chymotryptic peptide map viral proteins majority chymotryptic peptide 35Sknethionine label protein find coelute high performance liquid chromotography Moreover technique peptide map indicate particularly strong relatedness MHV MHV among MHV JHM A59',\n",
       "  0.7698373794555664),\n",
       " ('Equine arteritis virus infect cell contain six polyadenylated virus specific RNAs The kinetics equine arteritis virus growth virus specific RNA synthesis 40° determine BHK cell Maximum titer infectious virus ∼107 PFU observe incorporation uridine virus specific RNA become detectable increase reach maximum rate This RNA label isolate infected cell About bound oligo cellulose material denature use glyoxal dimethyl sulfoxide analyze electrophoresis agarose urea gel Six virus specific RNA specie find follow molecular weight RNA1 RNA2 RNA3 RNA4 RNA5 RNA6 RNA1 comigrated viral genome Artifacts cause defective interfering particle breakdown RNA exclude Subsequently target size template synthesis genome size RNA five subgenomic RNAs determine transcription mapping Infected cell irradiate The effect reasing dos RNA synthesis determine quantitation individual RNAs separation agarose gel electrophoresis The target size calculate template RNAs 2–5 close physical size RNA1 The target size template RNA6 small dalton although much great physical size The data consistent model individual RNAs derive large precursor RNA molecule The consequence finding taxonomy Togaviridae discuss',\n",
       "  0.7694998383522034),\n",
       " ('Characterization leader relate small RNAs coronavirus infect cell Further evidence leader prim mechanism transcription Mouse hepatitis virus MHV murine coronavirus replicate cytoplasm synthesize viral mRNAs contain identical stretch leader RNA sequence end RNA The leader cod sequence end genomic RNA least nucleotide length join viral mRNAs unique mechanism Utilizing leader specific cDNA probe detect several free leader RNA species range nucleotide length The predominant leader RNA approximately nucleotide addition large distinct leader contain RNAs also detect range nucleotide length The 70–82 nucleotide leader relate RNAs present cytosol membrane fraction infect cell They also detect small RNA fraction associate replicative intermediate RNA These data suggest leader RNAs associate membrane bound transcription complex least part dissociate RNA template also identify temperature sensitive mutant synthesize leader RNA mRNAs nonpermissive temperature indicate leader RNA synthesis distinct transcription mRNAs These data support leader prim mechanism coronavirus transcription suggest one free leader RNAs use primer mRNA synthesis',\n",
       "  0.7635678052902222),\n",
       " ('limpet', 0.7592262625694275),\n",
       " ('reversed', 0.7563430070877075),\n",
       " ('Importance coronavirus negative strand genomic RNA synthesis prior subgenomic RNA transcription The strand viral RNAs result infection cell coronaviruses possess RNA genome message polarity genomic size subgenomic size Each strand subgenomic RNAs corresponds size subgenomic mRNA specie make infected cell test whether strand subgenomic RNAs might initially synthesize input single strand strand genomic RNA prior production subgenomic mRNAs use mouse hepatitis virus MHV defective interfering RNA subgenomic RNA produce RNA replicate cell RNA functional MHV intergenic region insert interior MHV sample contain particle irradiate light superinfect cell infect MHV prior superinfection Northern blot analysis intracellular RNAs extract superinfection show genomic RNA subgenomic RNA similar target size indicate strand genomic RNA synthesis input genomic RNA probably occur prior subgenomic size RNA synthesis discuss course coronavirus transcription strand genomic length coronavirus RNA synthesis might occur subgenomic size RNAs either polarity make',\n",
       "  0.7473985552787781),\n",
       " ('Recombinant Genomic RNA Coronavirus MHV A59 Coreplication RNA Containing MHV Spike Gene strategy target RNA recombination spike gene genomic RNA MHV A59 synthetic RNA contain MHV spike gene describe The MHV spike gene contain several nucleotide difference MHV A59 spike gene could use genetic marker include stretch additional nucleotide start nucleotide The MHV gene cDNA nucleotide termination codon insert frame pMIDI full length cDNA clone MHV A59 yield pDPRIS Using vaccinia vTF7 system RNA transcribe pDPRIS upon transfection MHV A59 infect cell DPRIS RNA show replicate passaged efficiently MHV A59 DPRIS particle copassaged several time Using highly specific sensitive PCR recombinant genomic RNA detect intracellular RNA total lysates pDPRIS transfected MHV A59 infect cell among genomic RNA agarose gel purify lysates More significantly specific PCR product find virion RNA progeny virus PCR product absent control mix intracellular RNA MHV A59 infect cell andin vitro transcribe DPRIS RNA PCR product intracellular RNA virion RNA clone independent clone sequence Crossovers A59 RNA find upstream nucleotide occur nucleotide border nucleotide border sequence homologous A59 gene conclude homologous RNA recombination take place genomic RNA template synthetic RNA template different location generate series MHV recombinant genome chimeric gene',\n",
       "  0.7423228025436401),\n",
       " ('Identification Specific Interaction Coronavirus Mouse Hepatitis Virus A59 Nucleocapsid Protein Packaging Signal The coronavirus mouse hepatitis virus MHV envelop positive strand RNA virus infected cell MHV produce coterminal nest set subgenomic messenger RNAs Only genomic RNA however encapsidated nucleocapsid protein incorporate infectious MHV virion believe RNA package signal present genomic RNA responsible selectivity Earlier study map signal stem–loop structure position end ORF1b The selective encapsidation mechanism probably initiate specific interaction packaging signal nucleocapsid protein study demonstrate thein vitrointeraction MHV A59 nucleocapsid protein package signal MHV use gel retardation cross link assay This interaction observe nucleocapsid protein infect cell also purify virion cell express recombinant nucleocapsid protein The specificity interaction demonstrate competition experiment nonlabeled contain RNAs tRNA total cytoplasmic RNA The result indicate virus specific modification protein presence viral protein require thisin vitrointeraction The assay describe report provide powerful tool study encapsidation initiation detail',\n",
       "  0.7318222522735596),\n",
       " ('Analysis genomic intracellular viral RNAs small plaque mutant mouse hepatitis virus JHM strain The genomic RNA intracellular RNA mouse hepatitis virus strain JHM MHV JHM two plaque mutant isolate persistently infected culture JHM analyze resistant oligonucleotide finger print The genomic RNA virus population JHM virus release different passage level persistent infection also analyze The analysis show location within genomic intracellular RNAs resistant oligonucleotides confirm early study Leibowitz Wilhelmsen Bond Virology 39–51 show six subgenomic RNAs MHV JHM form coterminal nest set extend different length direction The analysis also identify subgenomic RNA large oligonucleotides derive noncontiguous region genome mRNA synthesis Two important conclusion reach analysis mutant virus First virus population release persistent infection represent fairly constant mixture viruses fluctuate emergence variant predominant specie culture occur Second data indicate particular intracellular RNAs mutant virus sequence rearrangement occur subgenomic mRNA synthesis different correspond intracellular RNA wild type virus The result may indicate potential flexibility leader body fusion process previously recognize',\n",
       "  0.7273333668708801),\n",
       " ('Identification Nucleocapsid Binding Sites within Coronavirus Defective Genomes The coronavirus nucleocapsid protein major structural component virion associate genomic RNA form helical nucleocapsid appear multifunctional protein since data also suggest protein may involve viral RNA replication translation All function presumably involve interaction viral RNAs step toward understand interact viral RNAs map high efficiency binding site within BCV MHV defective genome Both vivo vitro assay use study bind BCV MHV proteins viral nonviral RNAs N–viral RNA complex detect bovine coronavirus BCV infect cell cell transiently express protein Filter binding use map binding site within Drep BCV defective genome replicate package presence helper virus One high efficiency bind site identify nucleotides end ORF within Drep For comparative purpose bind site also map mouse hepatitis coronavirus MHV defective interfering RNA MIDI Binding efficiency similar Drep measure RNA transcript region encompass MHV packaging signal 3949–4524 well region end MHV ORF nts 4837–5197 within MIDI Binding full length MIDI transcript ∼5500 transcript gene region 935–1986 MIDI exclude package signal significantly high measure small transcript This first identification binding sequence BCV also first report demonstrate interact vitro sequence package signal leader within MHV genome The data clearly demonstrate bind coronavirus RNAs efficiently nonviral RNAs The result implication regard multifunctional role',\n",
       "  0.7267230749130249),\n",
       " ('irn', 0.7262865304946899),\n",
       " ('sjl', 0.7235878109931946),\n",
       " ('Detection negative strand subgenomic RNAs free leader LDV infect macrophages The mechanism synthesis seven subgenomic mRNAs lactate dehydrogenase elevate virus LDV explore One propose mechanism leader prim transcription predicts formation free leader infect cell prime reinitiation transcription specific complementary site antigenomic template free LDV leader nucleotide detect LDV infect macrophage Another mechanism independent replication subgenomic mRNAs predict presence negative complement subgenomic mRNAs infect cell might generate subgenomic mRNAs virion Full length antigenomic RNA detect LDV infect macrophage Northern hybridization level genomic RNA negative polarity subgenomic RNAs Negative complement subgenomic mRNAs however detect reverse transcription total RNA infect macrophage use primer oligonucleotide complementary antileader follow polymerase chain reaction amplification use sense primer combination various oligonucleotide primer complementary segment downstream junction leader body subgenomic RNA unclear whether minute amount negative subgenomic RNAs function replication subgenomic mRNAs They could also product RNA replication process Finally subgenomic mRNAs detect LDV virion',\n",
       "  0.720780074596405),\n",
       " ('97–100', 0.7205976843833923),\n",
       " ('16c', 0.7171961069107056),\n",
       " ('electropherotypes', 0.7098439335823059),\n",
       " ('polycistronic mRNA specify coronavirus infectious bronchitis virus The third large nested set subgenomic mRNAs mRNA3 coronavirus infectious bronchitis virus IBV contain three separate open reading frame present next small mRNAs suggest mRNA may functionally polycistronic However although protein product identify open reading frame date cod function establish present nucleotide sequence data suggest three open read frame conserve variety different IBV strain show preparation monospecific antiserum bacterial fusion protein IBV infect cell contain small amount product ORFs vitro translation study use synthetic mRNAs contain open read frames suggest strongly three protein translate single molecular specie expression study carry intact cell support conclusion Thus mRNA3 IBV appear functionally tricistronic',\n",
       "  0.7075761556625366),\n",
       " ('urogenital', 0.7050830721855164),\n",
       " ('Sequence membrane protein gene avian coronavirus IBV cDNA clone prepare genomic RNA coronavirus IBV sequence The nucleotide sequence complete region mRNA present mRNAs determine sequence base present contains long open reading frame predict polypeptide molecular weight This agreement molecular weight report unglycosylated form membrane polypeptide',\n",
       "  0.7031735181808472),\n",
       " ('cycloheximide', 0.7026609182357788),\n",
       " ('colds', 0.7024624347686768),\n",
       " ('Nucleotide sequence bovine rotavirus gene expression gene product Baculovirus The nucleotide sequence gene encode structural viral protein VP1 bovine rotavirus strain determine The sequence data indicate segment contains rich The positive strand segment contains single open reading frame extend codon possess terminal untranslated region respectively The first AUG conforms Kozak consensus sequence utilize would yield protein calculated molecular weight 124–847 close apparent molecular weight VP1 The deduce amino acid sequence present significant similarity RNA dependent RNA polymerase several RNA virus VP1 also synthesize baculovirus use two transfer vecors pAC461 pVL941 Following infection Sf9 cell recombinant baculovirus full length nonfusion protein synthesise share property authentic VP1 make monkey kidney cell The level VP1 synthesis fold high baculovirus recombinant derive pVL941 transfer vector case VP1 express yield approximately equivalent cellular protein The recombinant protein immunoprecipitated hyperimmune serum raise purified rotavirus also immunogenic hyperimmune serum make guinea pig react VP1 use immunoprecipitation Western blot This serum possess neutralization activity',\n",
       "  0.7024540901184082),\n",
       " ('trigonal', 0.7017835378646851),\n",
       " ('Enhanced Accumulation Coronavirus Defective Interfering RNA Expressed Negative Strand Transcripts Coexpressed Positive Strand RNA Transcripts Expression negative strand murine coronavirus mouse hepatitis virus MHV defective interfering RNA transcript MHV infect cell result accumulation positive strand RNAs Joo Virol 5769–5776 However express negative strand RNA transcript poor template positive strand RNA synthesis The present study demonstrate RNA accumulation express negative strand RNA transcript MHV infect cell enhance coexpression complementary RNA transcript correspond region positive strand RNA The positive strand RNA transcripts correspond end 7–2 RNA similar enhancement effect The coexpressed positive strand RNA transcript lack leader sequence contain leader sequence fail demonstrate enhancement effect demonstrate presence leader sequence coexpressed positive strand RNA transcript necessary sufficient enhancement RNA accumulation coexpressed negative strand RNA transcript Negative strand RNA transcript coexpressed partial length positive strand RNA transcripts stable express alone suggest high stability express negative strand RNA transcripts unlikely reason high RNA accumulation cell coexpressing two complementary RNA transcript Sequence analysis unexpectedly demonstrate leader sequence majority accumulate RNAs switch helper virus derive leader sequence suggest enhancement RNA accumulation mediate efficient utilization helper virus derive leader sequence RNA synthesis Furthermore data suggest leader switch type homologous RNA–RNA recombination occur positive strand RNA synthesis MHV positive strand RNA synthesis mechanism may preference toward recognize double strand RNA structure single strand negative strand RNA produce positive strand RNAs',\n",
       "  0.7004261612892151),\n",
       " ('edged', 0.700181245803833),\n",
       " ('zymosan', 0.6994974613189697),\n",
       " ('Subgenomic messenger RNAs Mastering regulation strand RNA virus life cycle Many strand RNA virus use subgenomic RNAs messenger protein expression regulate viral life cycle Three different mechanism describe synthesis RNAs The first mechanism involve internal initiation strand RNA template require internal SGP promoter The second mechanism make prematurely terminate strand RNA use template make RNA The third mechanism use discontinuous RNA synthesis make strand RNA templates Most RNAs translate structural protein protein related pathogenesis however RNAs regulate transition translation replication function riboregulators replication translation support RNA–RNA recombination review discus function RNAs influence viral replication translation recombination',\n",
       "  0.6940789222717285),\n",
       " ('Species independent detection RNA virus representational difference analysis use non ribosomal hexanucleotides reverse transcription method isolation genomic fragment RNA virus base cDNA representational difference analysis cDNA RDA develop cDNA RDA apply subtraction poly RNAs poly RNAs RNA virus genome owe vast quantity ribosomal RNAs construct primer inefficient reverse transcription ribosomal sequence base distribution analysis hexanucleotide pattern ribosomal RNA The analysis reveal distribution hexanucleotide pattern ribosomal RNA virus genome different construct hexanucleotides non ribosomal hexanucleotides use mixed primer reverse transcription cDNA RDA synchronous analysis hexanucleotide pattern know viral sequence show know genomic size viral sequence include non ribosomal hexanucleotides model experiment non ribosomal hexanucleotides use primer vitro transcribe plasmid RNA efficiently reverse transcribed compare ribosomal RNA rat cell Using non ribosomal primer cDNA fragment severe acute respiratory syndrome coronavirus bovine parainfluenza virus efficiently amplify subtract cDNA amplicons derive uninfected cell derive virus infected cell The result suggest cDNA RDA non ribosomal primer use specie independent detection virus include new virus',\n",
       "  0.6934475302696228),\n",
       " ('flopppy', 0.6913344860076904),\n",
       " ('glu47', 0.686272144317627),\n",
       " ('Conserved Motif End Mouse Hepatitis Virus Genomic RNA Required Host Protein Binding Viral RNA Replication conserve nucleotide sequence UGAAUGAAGUU end genomic RNA coronavirus mouse hepatitis virus require host protein bind viral RNA synthesis RNA probe contain nucleotide sequence bound four cellular protein highly label protein kDa three minor specie size kDa Mutation nucleotide motif abolish cellular protein bind The RNA–protein complex observe cytoplasmic extract MHV JHM infect cell RNase protection gel mobility shift cross link assay indistinguishable observed extract uninfected cell Both negative strand synthesis positive strand replication viral defective interfere RNAs presence helper virus affect mutation disrupt RNA–protein complex formation even though mutate nucleotide convert wild type sequence presumably recombination helper virus Kinetic analysis indicate recombination RNA helper virus occur relatively early MHV replicative cycle postinfection time viral RNA synthesis replication positive strand RNA barely detectable level RNA mutation upstream protein bind element replicate efficiently wild type without undergo recombination Thus nucleotide conserve host protein bind motif appear play important role viral RNA replication',\n",
       "  0.6851989030838013),\n",
       " ('Analysis protein expression mammalian cell line stably express lactate dehydrogenase elevate virus ORF ORF protein Lactate dehydrogenase elevate virus LDV strict specie specificity Because subset mouse primary macrophage identify support LDV replication vitro precise molecular mechanism viral entry replication remain unclear analyze LDV envelope proteins probably mediate viral attachment host cell develop mammalian system stable expression LDV open reading frame ORF ORF encode protein ORF ORF protein correspond envelope respectively compare expressed protein native one Western blot analysis combine glycanase digestion reveal ORF ORF protein similar size native ORF protein glycosylated like native Immunofluorescence microscopy reveal ORF ORF protein distribute throughout cytoplasm colocalized cell Moreover ORF protein localize perinuclear region Golgi complex transport cell surface This mammalian expression system exogenously express protein closely resemble native protein provide experimental basis study interaction LDV envelope proteins host cell',\n",
       "  0.6794024705886841),\n",
       " ('Coronavirus mRNA synthesis Identification novel transcription initiation signal differentially regulate different leader sequence The mRNA synthesis mouse hepatitis virus MHV propose result interaction leader RNA intergenic site Previously identify transcription initiation site mRNA efficiently transcribed virus contain two copy UCUAA sequence leader RNA three copy study identify several site regulate opposite way namely efficiently transcribed leader RNA three UCUAA copy two copy These site characterize primer extension amplification polymerase chain reaction One sit gene region recombinant virus A59 JHM strain MHV Another gene region MHV strain Both site sequence similar different consensus transcription initiation signal UCUAAUCUAUC UUUAAUCUU oppose UCUAAAC These two novel intergenic sequence present genome JHM strain consistent absence mRNAs JHM infect cell The discovery type transcription initiation site provide additional evidence importance leader RNA transcription initiation MHV mRNAs',\n",
       "  0.6782512068748474),\n",
       " ('Defective interfering particle murine coronavirus Mechanism synthesis defective viral RNAs The mechanism synthesis defective viral RNAs cell infect defective interfering particles mouse hepatitis virus study Two specific RNA specie DIssA genomic size DIssE subgenomic size detect infect cell Purified particles however find contain predominantly DIssA trace amount DIssE RNA Despite negligible amount DIssE RNA virion appear serve template synthesis DIssE RNA infect cell This conclusion support two study First target size DIssE RNA synthesis significantly small DIssA Second purify DIssE RNA transfected cell infect helper virus DIssE RNA could replicate become predominant RNA specie infect cell Thus DIssE RNA synthesize genomic RNA particle study relationship virus dilution amount intracellular viral RNA synthesis show DIssE RNA synthesis require helper function utilize leader sequence helper virus contrast DIssA synthesis appear helper independent replicate Thus DIssA cod functional RNA polymerase',\n",
       "  0.6758540272712708),\n",
       " ('punctae', 0.6755210161209106),\n",
       " ('trophism', 0.6742651462554932),\n",
       " ('transactivating', 0.6732211709022522),\n",
       " ('Rapid isolation morbillivirus nucleocapsid genomic RNA cDNA clone production specific core protein antisera procedure describe rapid isolation canine distemper virus nucleocapsid free contaminate viral non core host cellular protein Nucleocapsid isolate manner amenable ultrastructural evaluation protein isolation production monospecific hyperimmune serum genomic RNA isolation cDNA clone Nucleocapsid defective variant isolate Vero cell infect Ond CDV readily visualize cesium gradient The calculated density Ultrastructurally appear long uninterrupted strand length diameter well define capsid subunit truncate uniform length diameter total protein obtain every cell infect mass represent kDa protein kDa protein 160–200 kDa protein agreement constituency paramyxoviruses Viral proteins purify SDS PAGE use production hyperimmune serum Specificity demonstrate Western blot analysis Both antiserum capable detect viral antigen persistently lytically CDV infect cell indirect immunofluorescence single high molecular weight specie nucleic acid isolate purified nucleocapsids compatible morbillivirus genome Although efficiency RNA extraction purify low sufficient RNA obtain gel analysis establishment genomic RNA cDNA clone',\n",
       "  0.6732089519500732),\n",
       " ('longquan', 0.6726611256599426),\n",
       " ('Detection bovine viral diarrhea virus serum cattle dot blot hybridization assay dot blot hybridization assay develop use rapid screen test detect bovine viral diarrhea virus BVDV serum infect cattle kilobase cDNA prepare BVDV genome molecularly clone use study Insert cDNA remove pUC9 plasmid vector Pst restriction endonuclease digestion purify plasmid DNA agarose electrophoresis electroelution The hybridization probe prepare nick translation presence gamma dCT32P label specific activity cpm DNA Specificity determine dot blot hybridization infect cell culture supernate nine different BVDV strain The probe hybridize equally strain BVDV test include four cytopathic five noncytopathic strain BVDV Serum collect veal calf respiratory tract disease unthriftiness anorexia poor condition Serum sample treat nonidet P40 detergent denature formaldehyde heat prior application micron nylon membrane filter use vacuum dot blot apparatus Hybridization relatively stringent condition formamide 42°C total serum sample different calf test sample positive dot blot hybridization BVDV RNA Eight calve test week later remain positive BVDV RNA',\n",
       "  0.6724940538406372),\n",
       " ('Coronaviruses share extensive nucleotide homology murine coronavirus MHV A59 share human murine coronaviruses cDNA probe represent genome mouse hepatitis virus MHV strain A59 MHV A59 use measure nucleotide sequence homology among murine human coronaviruses coronaviruses isolate Burks Since isolate inoculation multiple sclerosis central nervous system CNS tissue mouse culture mouse cell important determine relationship murine human coronavirus isolate Our result indicate share almost complete nucleotide homology approximately MHV A59 generate subgenomic RNAs size MHV A59 The human coronavirus HCV strain test show less homology MHV A59 The immunologically unrelated HCV show nucleotide homology MHV A59 The immunologically crossreactive HCV OC43 show nucleotide homology MHV A59 blot hybridization hybridize solution assay nuclease digestion',\n",
       "  0.6721839308738708),\n",
       " ('Synthesis Putative Red Clover Necrotic Mosaic Virus RNA Polymerase Ribosomal Frameshifting Vitro The red clover necrotic mosaic virus RCNMV genome split two single strand RNA specie term RNA RNA RNA directs synthesis kDa p88 kDa p57 kDa p37 kDa p27 polypeptide RNA kDa p35 polypeptide vitro The cod order RNA product determine p27 p57 p37 Antibodies synthetic peptide represent carboxyl terminal portion p27 p57 immunoprecipitated respective polypeptide addition p88 suggest p88 fusion protein frameshift heptanucleotide sequence element identify RCNMV RNA addition stable stem loop secondary structure adjacent heptanucleotide sequence predict Together sequence element suggest ribosomal frameshifting event occur allow translational readthrough p27 open reading frame p57 open reading frame generate observed p88 product RNA expression construct fuse p57 open reading frame engineer investigate ribosomal frameshifting event antibody immunoprecipitated fusion protein predict size contain carboxyl portion Site direct mutagenesis frameshift element indicate vitro p88 also express alternatively suppression amber termination codon Based data propose putative RCNMV RNA polymerase kDa polypeptide express ribosomal frameshifting mechanism similar utilized retrovirus',\n",
       "  0.6720253229141235),\n",
       " ('histocompatible', 0.6717103123664856),\n",
       " ('Molecular Characterization Transmissible Gastroenteritis Coronavirus Defective Interfering Genomes Packaging Heterogeneity Three transmissible gastroenteritis virus TGEV defective RNAs select serial undiluted passage PUR46 strain cell These RNAs respectively detect passage remain stable upon passage cell culture significantly interfere helper mRNA synthesis RNA analysis purify virion show three defective RNAs efficiently package Virions different density contain either full length defective RNAs sort sucrose gradient indicate defective full length genome independently encapsidated RNAs amplify reverse transcription polymerase chain reaction clone sequenced genome form three four discontinuous region wild type genome respectively contain nucleotide end genome two fragment mostly gene end genome RNAs include sequence pseudoknot motif encode polymerase metal ion bind helicase motif RNA structure closely related RNA two main difference maintains entire ORF show heterogeneity size end deletion This heterogeneity map begin gene natural TGEV recombination event observe suggest either process template switching occur high frequency point derive genome selective advantage',\n",
       "  0.671464741230011),\n",
       " ('sl5', 0.671183168888092),\n",
       " ('Hepatitis virus HEV Molecular clone sequence full length viral genome recently describe clone portion hepatitis virus HEV confirm etiologic association enterically transmit waterborne epidemic non non hepatitis The virus consist single strand positive sense RNA genome approximately polyadenylated end report clone nucleotide sequence overlap contiguous set cDNA clone represent entire genome HEV Burma strain HEV The large open reading frame extend approximately Fend contain RNA direct RNA polymerase nucleoside triphosphate binding motifs The second major open reading frame ORF2 begin downstream first extend approximately termination codon present terminal stretch poly residues ORF2 contain consensus signal peptide sequence amino terminus capsid like region high content basic amino acid similar see virus capsid proteins third open reading frame partially overlap first second encompass addition full length genomic transcript two subgenomic polyadenylated message approximately detect infected liver use probe third genome The genomic organization virus consistent Fend encode nonstructural end encode viral structural gene The expression strategy virus involve use three different open reading frame least three different transcript HEV previously determine nonenveloped particle diameter 27–34 These finding genetic organization expression strategy HEV suggest prototype human pathogen new class RNA virus perhaps separate genus within Caliciviridae family',\n",
       "  0.6709362864494324),\n",
       " ('Characterization defective interfering RNAs rubella virusgenerated serial undiluted passage During serial undiluted passage rubella virus RUB Vero cell two specie defective interfere RNAs approximately nucleotide length generate Frey Hemphill Virology 22–29 study RNAs characterize molecular clone hybridization probe define sequence primer extension The RNA specie find authentic RNAs contain single deletion structural protein open reading frame ORF region genome The RNAs find subgenomic RNAs synthesize large RNA templates Analysis extent deletion use reverse transcription PCR protocol reveal end deletion extend beyond terminal genome The end deletion extend nonstructural protein ORF however RNAs subgenomic start site delete present Following serial undiluted passage seven independent stock RUB pattern RNAs generate RNAs length majority RNA species persistently infect line Vero cell show type RUB RNAs generate select However supernatant persistently infect cell passaged RNAs present two passage length indicate specie selective advantage type RNAs serial passage',\n",
       "  0.6701720952987671),\n",
       " ('Primary structure translation defective interfering rna murine coronavirus intracellular defective interfering RNA DIssE mouse hepatitis virus MHV obtain serial high multiplicity passage virus clone sequenced DIssE RNA compose three noncontiguous genomic region represent first nucleotide Fend internal nucleotide polymerase gene nucleotide end parental MHV genome The DIssE sequence contain one large continuous open reading frame Two protein product open reading frame identify vitro translation infect cell Sequence comparison DIssE corresponding part parental virus genome reveal DIssE three base substitution within leader sequence also deletion nine nucleotide locate junction leader remain genomic sequence The end DIssE RNA heterogeneous respect number UCUAA repeat within leader sequence The parental MHV genomic RNA appear extensive stable secondary structure region RNA rearrangement occur These data suggest MHV RNA may generate result discontinuous nonprocessive manner MHV RNA synthesis',\n",
       "  0.6695149540901184),\n",
       " ('Vaccinia virus produce late mRNAs discontinuous synthesis describe unusual structure vaccinia virus late mRNA molecule protein cod sequence major late structural polypeptide precede long leader RNAs case thousand nucleotide long These sequence map different region viral genome one instance separate late gene DNA Moreover leader sequence map either upstream downstream late gene transcribe either DNA strand fuse late gene cod sequence via poly stretch This demonstrate vaccinia virus produce late mRNAs tag protein cod sequence onto end RNAs',\n",
       "  0.6656054854393005),\n",
       " ('80nm', 0.6651242971420288),\n",
       " ('avidin', 0.6643003821372986),\n",
       " ('Lactate dehydrogenase elevate virus LDV subgenomic mRNAs mRNA leader comparison terminal sequence two LDV isolate The terminal nucleotides genome one isolate lactate dehydrogenase elevate virus LDV derive sequence analysis cDNAs several genomic library compare another LDV isolate LDV Godeny Virol 768–771 The non cod segment nucleotide two LDV genome identical whereas mark vary nucleotide amino acid divergence apparent three upstream overlap open reading frame ORF The third ORF end exhibit nucleotide amino acid identity whereas terminal ORF encode nucleocapsid protein exhibit approximately amino acid identity The second terminal ORF encode kDa protein lack glycosylation sit possess potential transmembrane helix terminal half molecule similar membrane organization observe correspond protein equine arteritis virus protein mouse hepatitis virus The sequence analysis combine Northern hybridization analyse RNA LDV infect macrophage spleen LDV infect mouse indicate three ORFs encode terminal end LDV genome express via three small mRNAs mRNAs 6–8 seven subgenomic mRNAs LDV mRNAs 2–8 range size All mRNAs show carry poly tract common leader sequence The seven mRNAs produce infected macrophage culture concomitantly genomic LDV RNA Maximum LDV RNA synthesis observe post infection The seven subgenomic mRNAs detect macrophage infect three different isolates LDV different relative amount mRNAs produce The relative proportion molecule mRNAs 1–8 present LDV infect macrophage total respectively',\n",
       "  0.66380774974823),\n",
       " ('Strand Specific Quantitative Reverse Transcription Polymerase Chain Reaction Assay Measurement Arenavirus Genomic Antigenomic RNAs Arenaviruses segment single strand RNA virus cause significant human disease The manner regulate replication genome well understood This partly due absence highly sensitive assay measure individual specie arenavirus replicative RNAs overcome obstacle design quantitative reverse transcription PCR assay selective quantitation lymphocytic choriomeningitis virus LCMV genomic antigenomic RNAs During course assay design identify nonspecific prim phenomenon whereby absence primer cDNAs complementary LCMV replicative RNA specie generate successfully circumvent nonspecific prim event use biotinylated primer reaction permit affinity purification primer specific cDNAs use streptavidin coat magnetic bead proof principle use assay map dynamic LCMV replication acute persistent time point determine quantity genomic antigenomic RNAs incorporate LCMV particle This assay adapt measure total segment derive viral RNAs therefore represent highly sensitive diagnostic platform screen LCMV infection rodent human tissue sample also use quantify virus cell attachment',\n",
       "  0.6627099514007568),\n",
       " ('Defective interfering particle mouse hepatitis virus After six eight serial undiluted passage mouse hepatitis virus JHM strain DBT cell culture decrease yield infectious virus occur passage fluctuate yield infectious virus observe The serially passaged virus interfere multiplication standard JHM virus vesicular stomatitis virus After sucrose equilibrium centrifugation high passage virus single peak contain infectious virus interfere activity This virus population resemble original JHM virus structural protein contain increased proportion protein molecular weight Genomic RNA standard JHM virus contain single specie RNA molecular weight After five undiluted passage however virion population contain two RNA specie molecular weight RNase resistant oligonucleotide finger print RNAs show low molecular weight RNA lose several oligonucleotide spot present genomic RNA standard JHM virus After several serial dilute passage passage virus single virus population obtain standard virus RNA molecular weight lack interfering activity These result indicate defective interfering particle generate serial undiluted passage JHM virus',\n",
       "  0.6620462536811829),\n",
       " ('The virion protein infectious bronchitis virus phosphorylated protein infect cell lysates Because phosphorylation infectious bronchitis virus IBV nucleocapsid protein may regulate multiple role viral replication dynamic phosphorylation examine 32P orthophosphate labeling Western blot analysis confirm viral protein phosphorylated Pulse label 32P orthophosphate indicate IBV protein phosphorylated virion well time infection either chicken embryo kidney cell Vero cell Pulse chase analyse follow immunoprecipitation IBV proteins use rabbit anti IBV polyclonal antibody demonstrate phosphate protein stable least Simultaneous label 32P orthophosphate leucine identify fold increase 32P count per minute cpm ratio virion compare 32P cpm ratio cell lysates chicken embryo kidney cell whereas Vero cell 32P cpm ratio virion fold great 32P cpm ratio cell lysates These study consistent phosphorylation IBV play role assembly maturation viral particle',\n",
       "  0.6608834266662598),\n",
       " ('80°c', 0.6598453521728516),\n",
       " ('contradictory', 0.6584374904632568),\n",
       " ('requisite', 0.6550523638725281),\n",
       " ('debliecki', 0.6548053026199341),\n",
       " ('Novel splice open reading frame reveal long read direct RNA sequence adenovirus transcript Adenovirus common human pathogen relies host cell process production process viral RNA Although adenoviral promoter splice junction cleavage polyadenylation site characterize use low throughput biochemical technique short read cDNA base sequence technology fully capture complexity adenoviral transcriptome combine Illumina short read nanopore long read direct RNA sequencing approach map transcription start sit cleavage polyadenylation site across adenovirus genome The canonical viral early late RNA cassette confirm analysis splice junction within long RNA read reveal additional novel viral transcript These RNAs include seven new splice junction lead expression canonical open reading frame ORF well transcript encode message potentially alter protein function truncation fusion canonical ORFs addition also detect RNAs bypass canonical cleavage site generate potential chimeric protein link separate gene transcription unit Our work highlight long read sequence technology reveal complexity within viral transcriptomes',\n",
       "  0.6546576023101807),\n",
       " ('cys1225', 0.6546492576599121),\n",
       " ('The Production Recombinant Infectious Particles Murine Coronavirus Absence Helper Virus study production release infectious particle vaccinia polymerase recombinant virus infect cell transfected five different plasmid express synthetic RNA MIDI four structural protein murine coronavirus MHV A59 The cDNA contain hepatitis delta ribozyme sequence generate transfected cell define end study transfected cell virus like particle VLP observe vesicle Release particles medium study immunoprecipitations protein release culture supernatant Particle release independent require Coexpression sufficient particle release Coexpression structural protein MIDI RNA result production release infectious particle Infectivity particle determine add helper virus MHV A59 medium contain VLPs use mixture infect new cell Intracellular RNA several subsequent undiluted passage isolate detect MIDI RNA Passage MIDI RNA dependent expression structural protein MHV A59 transfected cell absence either MIDI RNA could passaged fresh cell thus develop system produce coronavirus like particle assay test infectivity',\n",
       "  0.6539414525032043),\n",
       " ('altavista', 0.6536913514137268),\n",
       " ('recombinants', 0.6515949964523315),\n",
       " ('tryptic', 0.6515925526618958),\n",
       " ('Sequence analysis bovine enteritic coronavirus F15 genome —Sequence gene cod nucleocapsid protein analysis predict protein Summary Sequences encode protein bovine enteritic coronavirus F15 strain BECV F15 clone PBR322 plasmid use cDNA produce prim oligo purify viral genomic RNA Some insert contain clone study Hybridization insert poly RNA extract infected cell lead conclusion locate end genome After subcloning M13 phage DNA clone sequence Sanger technique nucleotide sequence correspond gene cod viral protein establish show overlap open reading frame ORF The non cod end gene nucleotide sequence common homologous genome area MHV TGE IBV viruses This sequence may represent polymerase RNA bind site upstream sequence surround first AUG small ORF corresponds potentially functional initiation codon The sequence primary translation product deduce DNA sequence predicts polypeptide amino acid high leucine content possess hydrophobic terminal end The large ORF cod capacity amino acid correspond protein molecular weight The deduced protein possess serine residue total amino acid content may phosporylated involved protein RNA bind protein also region high basic amino acid content One also serine rich strong homology site MHV TGE IBV virus first part terminal amino acid sequence PRWYFYYLGTGP highly conserve BECV F15 JHM TGE IBV viruses BCV Mebus strain BECV F15 minor difference protein sequence',\n",
       "  0.6512633562088013),\n",
       " ('ssol', 0.6503183841705322),\n",
       " ('The Terminal Nucleotides Bovine Coronavirus Defective Interfering RNA Harbor Cis Acting Elements Required Both Negative Positive Strand RNA Synthesis The synthesis negative strand strand complement ∼30 kilobase positive strand strand coronaviral genome necessary early step genome replication The identification act element require strand RNA synthesis coronaviruses however hamper due insufficiency technique use detect strand RNA specie Here employ method head tail ligation real time quantitative reverse transcription polymerase chain reaction qRT PCR detect quantitate synthesis bovine coronavirus BCoV defective interfering RNA strand Furthermore use aforementioned technique along Northern blot assay specifically define act RNA element within terminal nucleotide function synthesis strand BCoV RNA The major finding follow within terminal cis act element responsible strand BCoV RNA synthesis −34 within terminal cis acting element require strand BCoV RNA synthesis iii nucleotide specie position important critical strand BCoV RNA synthesis These result demonstrate terminal BCoV RNA harbor act RNA element require strand RNA synthesis extend knowledge mechanism coronavirus replication The method head tail ligation qRT PCR employ study may also apply identify cis act element require strand RNA synthesis coronaviruses',\n",
       "  0.650198221206665),\n",
       " ('Generation cytopathogenic subgenomic RNA classical swine fever virus persistently infect porcine cell line Two biological clone classical swine fever virus strain Alfort recombinant virus vA187 derive cDNA clone Alfort use establish persistently infected culture oft swine kidney cell line find cell passaged culture positive viral antigen throughout course experiment Additionally supernatant collect upon passaging cell continuously contain high titer infectious virus six separate culture persistently infect either biological clone recombinant virus cytopathic effect occur spontaneously passage The cytopathogenic agent supernatant culture could passaged repeatedly suggest generation mutant virus Analysis RNA culture reveal presence subgenomic viral RNA approximately kilobases six case RNA identical internal deletion nucleotides include region cod structural proteins The subgenomic RNA replicate package presence wild type virus Cells infect cytopathogenic virus contain increased amount viral protein NS3 think involve pestivirus cytopathogenicity',\n",
       "  0.6501400470733643),\n",
       " ('wpre', 0.6496987342834473),\n",
       " ('8–10', 0.6489431858062744),\n",
       " ('The kDa hydrophobic protein encode end porcine transmissible gastroenteritis coronavirus genome membrane associate The open reading frame potentially encode amino acid hydrophobic protein mapping end porcine transmissible gastroenteritis coronavirus TGEV genome show expressed virus replication The clone gene place plasmid control RNA polymerase promoter vitro translation transcript generate vitro yield kDa protein immunoprecipitable porcine hyperimmune anti TGEV serum Antiserum raise rabbit amino acid synthetic polypeptide represent central hydrophilic region specifically immunoprecipitated TGEV infect cell show become associate microsomal membrane synthesis vitro find closely associate endoplasmic reticulum cell surface membrane infect cell The intracellular location suggest may play role membrane association replication complex virion assembly',\n",
       "  0.648154616355896),\n",
       " ('aminoterminal', 0.6470438241958618),\n",
       " ('endodomains', 0.6457205414772034),\n",
       " ('xxx', 0.6452006101608276),\n",
       " ('Sequence analysis nucleocapsid gene leader RNA human coronavirus OC43 The nucleotide sequence end genomic RNA human coronavirus OC43 HCV OC43 determine cDNA clone intracellular virus specific mRNAs The nucleotide sequence predict amino acid sequence main open reading frame ORF represent nucleocapsid protein highly homologous bovine coronavirus BCV Mebus strain This ORF predict protein amino acid Additional small ORFs also present different reading frame also determine leader sequence present end HCV OC43 mRNAs primer extension study This sequence highly homologous mouse hepatitis virus particularly end leader sequence postulate involved unique mechanism leader prim transcription These data suggest HCV OC43 BCV might diverge fairly recently end leader sequence significant functional role',\n",
       "  0.6449956893920898),\n",
       " ('Equine Arteritis Virus Subgenomic RNA Transcription Inactivation Translation Inhibition Studies The expression genetic information equine arteritis virus EAV arterivirus involve synthesis six subgenomic mRNAs These coterminal since composed leader body sequence identical end genome respectively Previously suggest thatcis splice genome length precursor RNA involve synthesis This reevaluate comparative analysis RNA synthesis EAV coronavirus mouse hepatitis virus MHV alphavirus Sindbis virus transcription mapping show majority EAV RNAs make later stage infection derive genome length precursor However complete independence RNA synthesis genomic RNA never observe course infection The possibility result irradiation induce effect synthesis viral replicase investigate inhibit translation use cycloheximide For EAV ongoing protein synthesis find important synthesis RNA genomic RNA general MHV transcription extremely sensitive translation inhibition whereas EAV genomic RNA synthesis become independent ofde novoprotein synthesis late infection',\n",
       "  0.6448941826820374),\n",
       " ('Replication Synthetic Defective Interfering RNAs Derived Coronavirus Mouse Hepatitis Virus A59 analyze replication deletion mutant defective interfere RNAs derive coronavirus mouse hepatitis virus MHV A59 presence MHV A59 Using two parental RNAs MIDI MIDIΔH twin set deletion mutant generate progressively short stretch sequence colinear genomic RNA All deletion mutant contain frame ORFs show transfected cell one passage RNAs detectable accumulation positively correlate length sequence contain However accumulation two twin mutant sequence nucleotide position fused position undetectable passage cell transfected passage cell recombination genomic RNA lead appearance parental RNAs The accumulation parental RNAs inversely correlate length sequence deletion mutant high sample sharp contrast data report MHV JHM derive RNAs show MHV A59 derive mutant RNAs require internal sequence domain replication The data suggest coronavirus replication involve RNA superstructure end genome one comprising end genomic RNA also conclude recombination data frame mutant impaired replication signal fit frame mutant intact replication signal',\n",
       "  0.6445801854133606),\n",
       " ('Synthesis virus specific RNA permeabilized murine coronavirus infected cell develop permeabilized cell system assay mouse hepatitis virus specific RNA polymerase activity This activity characterize requirement mono divalent cation requirement exogenous energy source optimum This system faithfully reflect MHV specific RNA synthesis intact cell regard time appearance course infection product synthesize The system efficient RNA product identical observe intact MHV infect cell judge agarose gel electrophoresis hybridization Permeabilized cell appear ideal system study coronavirus RNA synthesis since closely mimic vivo condition allow much experimental flexibility truly cell free system',\n",
       "  0.6420023441314697),\n",
       " ('Transition mRNA sequence downstream initiation codon single strand conformation strongly promote bind initiator tRNA Using RNA footprinting technique accessible site mRNA initiation region bound ribosome determine Chemical probe experiment presence absence initiator tRNA dimethyl sulfate kethoxal carbodiimide reagent probe mRNA mini mRNA contain initiation region bacteriophage gene cro use This region characterize long single strand Shine Dalgarno sequence follow two hairpin structure first one comprise loop initiation codon compare free mRNA nucleotide additionally protect binary mRNA ribosome complex belong sequence initiation codon The protection nucleotide change Addition initiator RNA result modification nucleotides stem downstream hairpin structure initiation region This reflect transition single strand conformation promote tRNA possible implication finding decode process discuss',\n",
       "  0.64136803150177),\n",
       " ('rhmpvδm2', 0.6411805152893066),\n",
       " ('rn3d', 0.6403952240943909),\n",
       " ('togaviridae', 0.6387391090393066),\n",
       " ('RNA bind protein coronavirus MHV Detection monomeric multimeric protein RNA overlay protein blot assay RNA binding protein coronavirus MHV A59 identify use RNA overlay protein blot assay ROPBA The major viral RNA binding protein virion infect cell phosphorylated nucleocapsid protein 50K new 140K virus structural protein identify minor RNA binding protein virion infect cell The 140K protein antigenically relate upon reduction yield 50K Thus 140K protein probably trimer subunit link intermolecular disulfide bond Several cellular RNA binding protein also detect RNA bind nucleotide sequence specific Single strand RNA MHV VSV cellular origin DNA probe MHV leader sequence double strand bovine rotavirus RNA could bind Binding MHV RNA optimal RNA could elute NaCl The ROPBA useful method initial identification RNA binding protein 140K protein murine coronavirus',\n",
       "  0.637864351272583),\n",
       " ('microsequencing', 0.637842059135437),\n",
       " ('general method induction screen antiserum cDNA encode polypeptide antibody specific coronavirus putative polymerase encode gene prokaryotic vector pGE374 contain recA lacZ gene frame use expression cDNA derive putative polymerase encode gene coronavirus mouse hepatitis virus strain A59 MHV A59 The pGE374 viral recombinant vector generate tripartite bacterial viral protein compose segment RecA protein terminus coronaviral sequence middle enzymatically active galactosidase terminus Rabbits immunize recombinant protein generate antibody MHV A59 portion tripartite protein Because MHV A59 polymerase proteins difficult identify infection use novel method demonstrate viral specificity antiserum The viral cDNA excise expression vector transfer pGem vector downstream frame portion cat gene This construct contain bacteriophage RNA polymerase promoter enable cell free synthesis fusion protein use verify antibody generate expressed viral DNA This strategy show successfully result specific generation antibody encode information viral cDNA Furthermore method general applicability generation characterization antibody direct protein encode cDNAs',\n",
       "  0.637419581413269),\n",
       " ('ontogeny', 0.6368986368179321),\n",
       " ('permease', 0.636838436126709),\n",
       " ('stems', 0.63649582862854),\n",
       " ('subviral', 0.6360560059547424),\n",
       " ('Enteric coronavirus TGEV partial sequence genomic RNA organization expression The sequence nucleotide genome RNA Purdue strain transmissible gastroenteritis virus TGEV porcine coronavirus determine cDNA clone The available sequence corresponds part genome total length express subgenomic mRNAs The subgenomic genomic RNA specie detect TGEV infect cell form coterminal nest structure unique feature Coronaviridae The transcription initiation site TGEV subgenomic RNAs appear involve hexameric sequence 5′CTAAAC present upstream cod region addition previously identify gene encode three structural protein two region correspond non overlap portion mRNAs may code far unidentified non structural polypeptides The predicted polypeptide kDa highly hydrophobic The sequence region allow translation two non ovelapping product X2a kDa x2b kDa RNA species liable express extreme open reading frame find',\n",
       "  0.6358838081359863),\n",
       " ('Emergence coronavirus infectious bronchitis virus mutant truncate gene functional characterization protein pathogenesis replication The subgenomic RNA IBV show tricistronic mRNA encode three product IBV infect cell explore least expressed ORF ORF encodes nonstructural protein evolutionarily conserve functionally indispensable viral propagation culture cell Beaudette strain IBV propagate chicken embryonated egg three passage adapt monkey kidney cell line Vero The gene passage embryonated egg passages Vero cell amplify reverse transcription–polymerase chain reaction sequence The result show viral RNA extract passage contain single insertion stretch gene nucleotide whereas early passage carry normal gene This insertion result frameshift event therefore express terminally truncate protein show frameshifting product clone plasmid express vitro cell transfected mutant construct The normal product gene amino acid long whereas frameshifting product amino acid long homogeneous amino acid residue terminal half Immunofluorescent study reveal normal protein localize nucleus truncate product show free distribution pattern indicate terminal portion responsible nuclear localization Comparison complete genome sequence isolates p20c22 p36c12 passage respectively reveal p36c12 contains three amino acid substitution two kDa protein encode gene one protein addition frameshifting product Further characterization two isolates demonstrate p36c12 show growth advantage p20c22 Vero cell chicken embryos virulent chicken embryo p20c22 These result suggest gene product essential replication IBV',\n",
       "  0.6354689002037048),\n",
       " ('Transcriptome Wide Cleavage Site Mapping Cellular mRNAs Reveals Features Underlying Sequence Specific Cleavage Viral Ribonuclease SOX Many viruses express factor reduce host gene expression widespread degradation cellular mRNA example class proteins mRNA target endoribonuclease SOX gamma herpesvirus Kaposi sarcoma associate herpesvirus KSHV Previous study indicate cleavage messenger RNAs mRNA SOX occur specific location define sequence target RNA odds regulation large portion cellular transcript study address paradox use high throughput sequence cleavage intermediate combine custom bioinformatics base analysis pipeline identify SOX cleavage sit across mRNA transcriptome These data couple targeted mutagenesis reveal cleavage sit specific reproducible defined degenerate sequence motif contain small number conserve residue rather strong consensus sequence This degenerate element well represent human KSHV mRNA presence correlate RNA destabilization SOX This represent new endonuclease target strategy use degenerate target element enables RNA cleavage specific location without restrict range target Furthermore show strong target selectivity achieve without high degree sequence specificity',\n",
       "  0.6336644291877747),\n",
       " ('Characterizations Coronaviruscis Acting RNA Elements Transcription Step Affecting Its Transcription Efficiency Seven eight specie viral subgenomic mRNAs produce coronavirus infected cell These mRNAs produce different quantity molar ratio remain constant viral replication study RNA element affect coronavirus transcription efficiency characterize series clone coronavirus mouse hepatitis virus MHV defective interfering RNAs contain insert intergenic sequence subgenomic RNA transcribe MHV infected cell Certain combination upstream downstream flanking sequence intergenic sequence suppress subgenomic RNA transcription yet change one flanking sequence different sequence eliminate transcription suppression The suppressive effect certain combination flank sequence combination could counteract alter intergenic sequence Thus combination intergenic sequence flank sequence affect transcription efficiency also characterize another set RNAs design clarify transcription step determine relative molar ratio coronavirus mRNAs Our study indicate subgenomic mRNAs exclusively synthesize negative strand genomic RNA relative molar ratio coronavirus mRNAs likely determine synthesis genomic size template RNA negative strand subgenomic RNAs templates subgenomic mRNAs relative molar ratio coronavirus mRNAs probably determine synthesis genomic size template RNA use subgenomic sized RNA transcription prior completion synthesis subgenomic size RNAs contain leader sequence The relative molar ratio coronavirus mRNAs therefore seem establish prior putative replicon type amplification subgenomic mRNAs',\n",
       "  0.6323330998420715),\n",
       " ('nondefective', 0.6305668354034424),\n",
       " ('Identification Noncanonical Signal Transcription Novel Subgenomic mRNA Mouse Hepatitis Virus Implication Mechanism Coronavirus RNA Transcription Subgenomic RNA transcription coronaviruses involve interaction leader antileader intergenic sequence However clear two sequence interact report previously unrecognized minor specie subgenomic mRNA term mRNA5–1 identify cell infect mouse hepatitis virus MHV strain JHM2c JHM JHM A59 MHV Sequence analysis reveal leader body fusion site mRNA locate approximately nucleotide downstream consensus sequence mRNA sequence homology know consensus sequence determine whether sequence function independently promoter clone sequence ≈70 upstream ≈70 downstream fusion site viral genomic RNA place front reporter gene defective interfere RNA chloramphenicol acetyltransferase CAT reporter vector Transfection reporter RNA MHV infect cell result synthesis CAT specific subgenomic mRNA detect reverse transcription polymerase chain reaction PCR The strength promoter similar IG7 mRNA measured CAT activity Deletion analysis show sequence sufficient initiate mRNA transcription mutation within abolish mRNA transcription vitro translation study confirm envelope protein translate mRNA5–1 encodes open reading frame ORF end indicate mRNA5–1 functional message Furthermore ORF5b replace CAT gene place context viral mini genome CAT express first ORF mRNA5–1 also second third ORF mRNA5 genomic RNA respectively suggest one mechanism involve regulation ORF5b expression Our finding thus support notion base pair leader antileader sole mechanism subgenomic RNA transcription',\n",
       "  0.6295608282089233),\n",
       " ('rap1', 0.6295099258422852),\n",
       " ('cysts', 0.6290991306304932),\n",
       " ('Altering SARS Coronavirus Frameshift Efficiency Affects Genomic Subgenomic RNA Production previous study difference amount genomic subgenomic RNA produce coronaviruses mutation program ribosomal frameshift signal ORF1a observe clear difference due change genomic sequence protein sequence frequency frameshifting Here viruses synonymous codon change show produce different ratio genomic subgenomic RNA These finding demonstrate protein sequence primary cause alter genomic subgenomic RNA production The synonymous codon change affect structure frameshift signal frameshifting efficiency Small difference frameshifting efficiency result dramatic difference genomic RNA production TCID suggest frameshifting frequency must stay certain threshold optimal virus production The data suggest either RNA sequence ratio viral protein result different level frameshifting affect viral replication',\n",
       "  0.627938985824585),\n",
       " ('Tandem Placement Coronavirus Promoter Results Enhanced mRNA Synthesis Downstream Most Initiation Site Insertion nucleotide promoter region bovine coronavirus gene part nucleotide cassette open reading frame clone synthetic defective interfering RNA result synthesis subDI RNA transcript replicate RNA genome Duplicating triplicating promoter sequence tandem cause progressive increase efficiency subgenomic mRNA synthesis despite concurrent decrease rate RNA accumulation specific promoter sequence add Although initiation transcription leader fusion occur three promoter site tandem construct almost transcript find product downstream genome promoter These result show enhancement subgenomic mRNA synthesis property reside within sequence situate near promoter possible role plus strand downstream promoter choice suggest',\n",
       "  0.6279337406158447),\n",
       " ('Subgenomic promoter recognition norovirus RNA dependent RNA polymerases The replication enzyme RNA viruses must preferentially recognize RNAs environment contains abundance cellular RNAs The factor responsible specific RNA recognition well understood part viral RNA synthesis take place within enzyme complex associate modify cellular membrane compartment Recombinant RNA dependent RNA polymerases RdRps human norovirus murine norovirus MNV find preferentially recognize RNA segment contain promoter short template sequence subgenomic RNA synthesis Both promoter template sequence contribute stable RdRp bind accurate initiation subgenomic RNAs efficient RNA synthesis Using method combine RNA crosslinking mass spectrometry residue near template channel MNV RdRp find contact hairpin RNA motif Mutations hairpin contact site MNV RdRp reduce MNV replication virus production cell This work demonstrate specific recognition norovirus subgenomic promoter bind viral RdRp',\n",
       "  0.626542329788208),\n",
       " ('Biotinylated radioactive cDNA probe detection hybridization bovine enteric coronavirus cDNA synthesize bovine coronavirus BCV genomic RNA template could use detect small quantity viral RNA hybridization either radioisotopic label biotinylated recombinant plasmid Virus optimally attach nitrocellulose membrane spot SSC whereas SSC superior viral RNA Denaturation RNA fixation RNA still encapsidated virus particle isolate genomic RNA achieve bake blot vacuum Virus detection supernatant infect HRT cell feasible improve significantly proteinase treatment homology observe virus cDNA either plasmid DNA nucleic acid isolate non infect HRT cell Hybridization radioisotopic label probe higher formamide concentration increase detection signal possibly reduce reassociation probe Significant detection amplification 30–50 time achieved case biotinylated probe stimulation hyperpolymer formation already hybridize target sequence additional hybridization biotinylated pUC detection amplification also obtain hybridization two probe pBC pBC contain non overlap viral sequence Although detectability surpass biotinylated probe sensitivity superior radioisotopic virus detection',\n",
       "  0.6256880760192871),\n",
       " ('a23187', 0.6254607439041138),\n",
       " ('key role carboxy terminal tail murine coronavirus nucleocapsid protein coordination genome package The prototype coronavirus mouse hepatitis virus MHV exhibit highly selective packaging genomic positive strand RNA assemble virion despite presence infect cell large excess subgenomic viral mRNAs One component selectivity MHV packaging signal RNA structure find genomic RNA subgenomic RNAs previously show major determinant recognition second two RNA binding domains viral nucleocapsid protein find recognition additionally depend upon segment carboxy terminal tail domain protein Since domain also region protein interact membrane protein find suggests mechanism selective genome packaging accomplish couple genome encapsidation virion assembly',\n",
       "  0.6249333024024963),\n",
       " ('High Resolution Analysis Coronavirus Gene Expression RNA Sequencing Ribosome Profiling Members family Coronaviridae large genome RNA viruses typically region kilobases Several coronaviruses Severe acute respiratory syndrome relate coronavirus SARS CoV Middle East respiratory syndrome relate coronavirus MERS CoV medical importance high mortality rate case SARS CoV significant pandemic potential Other coronaviruses Porcine epidemic diarrhea virus Avian coronavirus important livestock pathogen Ribosome profiling technique exploit capacity translate ribosome protect around nucleotide mRNA ribonuclease digestion Ribosome protect mRNA fragment purify subjected deep sequencing map back transcriptome give global snap shot translation Parallel RNA sequence allows normalization transcript abundance Here apply ribosome profile cell infect Murine coronavirus mouse hepatitis virus strain A59 MHV A59 model coronavirus genus SARS CoV MERS CoV The data obtain allow study kinetics virus transcription translation exquisite precision study timecourse positive negative sense genomic subgenomic viral RNA production relative translation efficiency different virus ORFs Virus mRNAs find translate efficiently host mRNAs rather virus translation dominate host translation later time point due high level virus transcript Triplet phase profile data allow precise determination translate read frames reveal several translate short open read frame upstream embed within know virus protein cod region Ribosome pause site identify virus replicase polyprotein pp1a ORF investigate experimentally Contrary expectation ribosome find pause ribosomal frameshift site knowledge first application ribosome profile RNA virus',\n",
       "  0.6246774792671204),\n",
       " ('Sequencing coronavirus IBV genomic RNA base open reading frame encode mRNA DNA sequence genomic cDNA clone avian infectious bronchitis virus IBV carry base determine include genomic sequence span termini two small mRNAs coterminal nest set mRNA mRNA This region contain complete cod sequence mRNA additional present mRNA Two open read frame present predict protein',\n",
       "  0.6243017911911011),\n",
       " ('Evidence Trans splice trypanosomes The end trypanosome mRNAs consist identical sequence nucleotide This mini exon sequence derive end nucleotide RNA medRNA The remainder mRNA derive protein cod exon link mini exon propose medRNA splice trans novo initiate transcript protein cod gene This trans splice model predicts downstream portion medRNA part branch structure release free product minRNA demonstrate significant level minRNA exist trypanosome RNA Furthermore minRNA release high molecular weight RNA HeLa cell S100 debranching extract conclude trnas splice physiological process mature mRNA molecule synthesize trypanosomes',\n",
       "  0.6242864727973938),\n",
       " ('dolichol', 0.6234878301620483),\n",
       " ('hemograms', 0.6232312917709351),\n",
       " ('rmd', 0.6227223873138428),\n",
       " ('proapo', 0.6224188804626465),\n",
       " ('sidechains', 0.6221149563789368),\n",
       " ('n3d', 0.6215488910675049),\n",
       " ('novo generation defective interfere RNAs tomato bushy stunt virus high multiplicity passage Defective interfering RNAs generate novo independent isolates tomato bushy stunt virus TBSV upon serial passage high multiplicity infection plant additional isolates serial passage low The RNAs detect RNA isolate virus particle LiCl soluble RNA fraction isolate inoculated leaf Symptom attenuation lead persistent infection closely correlated passage DIs first develop Comparisons nucleotide sequence cDNA clone RNA population previously characterize TBSV RNA reveal four region sequence TBSV genome strictly conserve RNAs virus leader sequence base region approximately 200–250 base viral polymerase gene approximately base terminus viral pl9 p22 gene approximately base 3′terminal noncoding region Conservation sequence motif present DIs suggest might common mechanism formation well selection pressure maintain sequence essential replication encapsidation',\n",
       "  0.6202486753463745),\n",
       " ('cloudy', 0.6201624870300293),\n",
       " ('Synthesis subcellular localization murine coronavirus nucleocapsid protein The synthesis processing nucleocapsid protein pp60 JHM strain murine coronaviruses examine Pulse chase experiment show pp60 synthesize initially protein approximately molecular weight p57 Immunoprecipitation use mouse anti JHMV antiserum indicate p57 virus specific Immunoprecipitation monoclonal antibody specific pp60 show p57 antigenically relate pp60 phosphorylated intracellular protein comigrated virion nucleocapsid protein pp60 phosphorylated The p57 find exclusively cytosol majority pp60 associate membrane fraction pp60 integral membrane protein',\n",
       "  0.6193658113479614),\n",
       " ('biorobot', 0.6191238164901733),\n",
       " ('Characterisation Structural Proteins Chronic Bee Paralysis Virus CBPV Using Mass Spectrometry Chronic bee paralysis virus CBPV etiological agent chronic paralysis infectious contagious disease adult honeybees CBPV positive single strand RNA virus contain two major viral RNA fragment RNA RNA encode three four putative open reading frame ORFs respectively RNA think encode viral RNA dependent RNA polymerase RdRp since amino acid sequence derive ORF share similarity RdRP family Nodaviridae Tombusviridae The genomic organization CBPV silico analysis suggest RNA encodes non structural protein RNA encodes structural protein probably encode ORFs study purify CBPV particle use characterize virion protein mass spectrometry Several polypeptide correspond protein encode ORF RNA detect Their role formation viral capsid discuss',\n",
       "  0.619022011756897),\n",
       " ('2000–2004', 0.6189815998077393),\n",
       " ('Rapid diagnosis avian infectious bronchitis virus polymerase chain reaction simple sensitive specific polymerase chain reaction PCR procedure develop order detect infectious bronchitis virus IBV directly tissue sample Viral RNA extract allantoic fluid cell culture infect experimentally different strain IBV tissue naturally infect bird Viral RNA amplify identify nested PCR assay use two set primer flank well conserve region nucleocapsid gene The select IBV nucleocapsid sequence detect successfully simple direct electrophoresis amplify material',\n",
       "  0.618820071220398),\n",
       " ('niesters', 0.6185073852539062),\n",
       " ('Detection murine coronavirus nonstructural protein encode downstream open reading frame Mouse hepatitis virus MHV gene contains two open read frame express second open reading frame gene gene ORF Escherichia coli expression system This system utilized plasmid contain promoter first codon recA gene fuse frame MHV gene ORF fuse turn galactosidase gene The protein product gene fusion use raise antibody gene ORF The specificity antibody verified immunoprecipitation vitro transcribe translated protein product gene ORF The second reading frame MHV gene show expressed course infection immunocytochemistry radioimmunoprecipitation use antibody raise coli fusion protein two dimensional gel electrophoresis',\n",
       "  0.6182450652122498),\n",
       " ('Three Stemmed mRNA Pseudoknot SARS Coronavirus Frameshift Signal wide range RNA viruses use program ribosomal frameshifting production viral fusion protein Inspection overlap region ORF1a ORF1b SARS CoV genome reveal similar coronaviruses program ribosomal frameshift could use virus produce fusion protein Computational analysis frameshift signal predict presence mRNA pseudoknot contain three double strand RNA stem structure rather two Phylogenetic analysis show conservation potential three stem pseudoknots frameshift signal coronaviruses GenBank database Though presence three stem structure support nuclease mapping two dimensional nuclear magnetic resonance study find suggest interaction stem structure may result local distortion form RNA These distortion particularly evident vicinity predict bulge stem vitro vivo frameshifting assay show SARS CoV frameshift signal functionally similar viral frameshift signal promote efficient frameshifting standard assay system sensitive drug genetic mutation know affect frameshifting efficiency yeast virus Mutagenesis study reveal specific sequence structure stem important efficient frameshifting identify new RNA structural motif capable promote efficient program ribosomal frameshifting The high degree conservation three stem mRNA pseudoknot structure among coronaviruses suggest present novel target antiviral therapeutic',\n",
       "  0.6178997755050659),\n",
       " ('assumed', 0.6178649663925171),\n",
       " ('Helper virus independent trans replication hepatitis virus derive minigenome previously describe synthetic driven cDNA minigenome contain antisense sequence luciferase gene internal ribosome entry site encephalomyocarditis virus flank end sequence hepatitis virus HCV contain act replication element Synthesis minus strand RNA artificial minigenome determine use Huh cell harbor autonomously replicate HCV subgenome helper provision functional replication component confirm extend study investigate whether minigenome replication system could reconstitute transfection naïve Huh cell plasmid express nonstructural proteins Reporter assay Northern blot analysis reveal trans expression proteins result high level luciferase activity synthesize minus strand RNA The analogous result also obtain minigenome derive HCV HCV derive protein able support chimeric minigenomes whose end replace respective region heterologous virus These result provide basis establish reverse genetic system helpful study trans act factor involve HCV RNA replication',\n",
       "  0.6169440746307373),\n",
       " ('thermostability', 0.61656653881073),\n",
       " ('a33', 0.6164553761482239),\n",
       " ('Characterisation RNA bind property coronavirus infectious bronchitis virus nucleocapsid protein amino terminal region The coronavirus nucleocapsid protein bind viral RNA form ribonucleocapsid regulate RNA synthesis The interaction protein viral RNA investigate use circular dichroism surface plasmon resonance protein underwent conformational change upon bind viral RNA data indicate electrostatic interaction involve bind protein RNA Kinetic analysis suggest amino terminal region facilitates long range non specific interaction protein viral RNA thus bring RNA close proximity protein allow specific contact form via lure lock mechanism',\n",
       "  0.6161714792251587),\n",
       " ('Global analysis polysome associate mRNA vesicular stomatitis virus infect cell Infection mammalian cell vesicular stomatitis virus VSV result inhibition cellular translation viral translation proceeds efficiently VSV RNA synthesis occur entirely within cytoplasm transcription viral polymerase produce mRNAs structurally indistinct cellular mRNAs respect cap structure polyadenylate tail Using global approach massively parallel sequence total cytoplasmic monosome polysome associate mRNA interrogate impact VSV infection HeLa cell translation Analysis sequence read different fraction show total cytoplasmic polysome associate read map viral gene hour post infection time point robust host cell translational shut observe Consistent overwhelm abundance viral mRNA polysome fraction read map cellular gene reduce The cellular mRNAs remain polysome associate follow infection longer half life typically large rich feature share viral mRNAs Several mRNAs encode protein know positively affect viral replication use chemical inhibition siRNA depletion confirm host chaperone heat shock protein hsp90 eukaryotic translation initiation factor eIF3A —encoded mRNAs—support viral replication Correspondingly regulate development DNA damage Redd1 encode host mRNA reduce polysome association inhibit viral infection These data underscore importance viral mRNA abundance shut host translation VSV infect cell link differential translatability cellular mRNAs pro antiviral function',\n",
       "  0.6160367131233215),\n",
       " ('Oligonucleotide probe infectious bronchitis virus diagnosis strain identification Genomic RNA fingerprint infectious bronchitis virus IBV strain M41 Conn46 prepare identify RNase resistant oligonucleotides unique two IBV strain Such oligonucleotides subsequently elute gel nucleotide sequence determine When oligonucleotide probe sequence synthesize used dot blot hybridization assay probe lack IBV strain specificity react RNAs homologous well heterologous IBV strain Based result method use study need apply large number oligonucleotide probes find one might suitable IBV strain serotype specific oligonucleotide probe',\n",
       "  0.6158584356307983),\n",
       " ('so3h', 0.6155364513397217),\n",
       " ('Processing Coronavirus MHV JHM Polymerase Polyprotein Identification Precursors Proteolytic Products Spanning Kilodaltons ORF1a The replicase mouse hepatitis virus strain JHM MHV JHM encode two overlap open reading frame ORF1a ORF1b translate produce kDa precursor polyprotein The polyprotein propose processed viral proteinase generate functional replicase complex date MHV JHM amino terminal protein p28 p72 process p65 identify elucidate biogenesis MHV JHM replicase clone expressed five region ORF1a bacteria prepare rabbit antiserum region Using immune serum immunoprecipitate radiolabeled protein MHV JHM infect cell determine MHV JHM ORF1a initially process generate p28 p72 p250 p150 Pulse–chase analysis reveal intermediate process generate p65 p210 p40 p27 MHV like proteinase p15 putative replicase complex consist p250 p210 p40 p150 large protein kDa coprecipitate infect cell disrupt indicate protein closely associate even initial proteolytic processing Immunofluorescence study reveal punctate labeling ORF1a proteins perinuclear region infect cell consistent membrane association replicase complex Furthermore vitrotranscription translation study MHV JHM 3Cpro flank hydrophobic domain confirm protease activity significantly enhance presence canine microsomal membrane Overall result demonstrate MHV JHM ORF1a polyprotein process protein intermediate product require membrane efficient biogenesis detect discrete membranous region cytoplasm infected cell',\n",
       "  0.6152782440185547),\n",
       " ('t3c31', 0.6149970293045044),\n",
       " ('glucosylated', 0.6149359941482544),\n",
       " ('reasing', 0.6146414279937744),\n",
       " ('lamv', 0.614200234413147),\n",
       " ('desiccation', 0.6139342784881592),\n",
       " ('Mouse hepatitis virus ORF express cytosol infect mouse fibroblasts fragment cDNA within gene ORF MHV A59 use construct vector encode bacterial viral fusion protein Antiserum raise fusion protein specifically immunoprecipitates 30K protein infect 17CI mouse fibroblasts The protein localize primarily cytosol membrane This consistent predict sequence potential role RNA binding protein',\n",
       "  0.6135151982307434),\n",
       " ('h43', 0.6124790906906128),\n",
       " ('rodrigues', 0.6111606359481812),\n",
       " ('awa', 0.6105817556381226),\n",
       " ('lrd', 0.6100525856018066),\n",
       " ('ammo', 0.6096628904342651),\n",
       " ('kdain', 0.6095107793807983),\n",
       " ('stuffer', 0.6082983016967773),\n",
       " ('microthrix', 0.6079409718513489),\n",
       " ('hinterland', 0.6078099608421326),\n",
       " ('Detection feline infectious peritonitis virus infection cell culture peripheral blood mononuclear leukocytes experimentally infected cat use biotinylated cDNA probe dot blot hybridization assay use biotinylated cDNA probe able detect feline infectious peritonitis virus FIPV RNA Felis catus whole fetus fcwf cell infect FIPV isolate DF2 UCDI UCD2 The probe cross hybridize dot blot assay nucleic acid closely related feline coronavirus feline enteric coronavirus FECV construct probe kilobase cDNA prepare FIPV DF2 genomic RNA molecularly clone The recombinant cDNA clone digest restriction endonuclease Rsa basepair Rsa fragment isolated vector DNA agarose electrophoresis glassmilk purification This fragment complementary three fourth nucleocapsid gene The hybridization probe prepared random prim label presence biotin dUTP Using avidin alkaline phosphatase conjugate chemiluminescent substrate detection system virus could detect infected cell vivo study probe use detect FIPV RNA peripheral blood mononuclear leukocytes PBML isolate various post infection day PID cat experimentally infect FIP produce coronavirus isolate FIPV FIPV DF2 Viral RNA could detect PBML isolate cat PID PBML PID There consistent pattern however hybridization result prognosis severity disease time sample Despite cross hybridization FECV RNA probe useful diagnosis FIP cat infect FECV likely become viremic',\n",
       "  0.6077902317047119),\n",
       " ('psittacines', 0.6077478528022766),\n",
       " ('fluoropropyl', 0.607275128364563),\n",
       " ('aureusviruses', 0.6068894863128662),\n",
       " ('cloaked', 0.6063188314437866),\n",
       " ('Molecular characterization genome segment encode RNA dependent RNA polymerase Antheraea mylitta cytoplasmic polyhedrosis virus Genome segment Antheraea mylitta cypovirus AmCPV convert cDNA clone sequenced consist nucleotide long ORF encoding amino acid long protein 123kDa BLAST phylogenetic analysis show sequence identity close relatedness AmCPV RNA dependent RNA polymerase RdRp insect cypoviruses suggest common origin insect cypoviruses The ORF express 123kDa soluble His tagged fusion protein insect cell via baculovirus recombinant exhibit RdRp activity vitro RNA polymerase assay without intrinsic terminal transferase activity Maximum activity observe 37°C presence 3mM MgCl2 Site direct mutagenesis confirm importance conserve GDD motif This first report functional characterization cypoviral RdRp may lead development anti viral agent',\n",
       "  0.6058671474456787),\n",
       " ('Localization extensive deletion structural gene two neurotropic variant murine coronavirus JHM The intracellular RNA two neurotropic variant JHM strain mouse hepatitis virus MHV independently isolate brain spinal cord infect Wistar Furth rat compare parental virus The mRNAs correspond gene encode peplomer hemagglutinin esterase proteins variant virus find small size The possible sequence change study oligonucleotide fingerprint direct RNA sequence Both variant large deletion amino acid carboxy terminal end protein However truncate protein detect infected cell suggest either translational regulation rapid degradation truncate protein cell The variant virus isolate spinal cord second deletion amino acid amino terminal half protein This deletion site correspond hypervariable region deletion frequently note among MHV variant different biological property These finding suggest change pathogenic property two neural isolates associate drastic alteration viral structural glycoprotein',\n",
       "  0.6055676937103271),\n",
       " ('cdkn', 0.6054332852363586),\n",
       " ('Coronavirus Transcription Mediated Sequences Flanking Transcription Consensus Sequence study murine coronavirus transcription continue use defective interfering RNAs mouse hepatitis virus MHV insert transcription consensus sequence order mimic subgenomic RNA synthesis nondefective genome Using subgenomic system study effect sequence flank MHV transcription consensus sequence subgenomic RNA transcription obtain follow result Insertion nucleotide long sequence include UCUAAAC transcription consensus sequence different location RNA result different efficiency subgenomic RNA synthesis Differences amount subgenomic RNA define sequence flank nucleotide long sequence affect location nucleotide long sequence RNA iii Naturally occur flank sequence intergenic sequence gene 6–7 gene 1–2 2–3 contain transcription suppressive element Each three naturally occur flank sequence MHV genomic cryptic transcription consensus sequence MHV gene also contain transcription suppressive element These data show sequence flank transcription consensus sequence affect MHV transcription',\n",
       "  0.6053258180618286),\n",
       " ('19–29', 0.6052963733673096),\n",
       " ('switched', 0.6052099466323853),\n",
       " ('Complete sequence end cod region structural protein gene turkey coronavirus Overlapping fragment genomic RNA spanning nucleotides end spike protein gene end nucleocapsid protein gene turkey coronavirus TCoV amplify reverse transcription polymerase chain reaction PCR The primer derive corresponding sequence infectious bronchitis virus IBV The PCR product clone sequenced nucleic acid structure similarity publish sequence coronaviruses analyze Sequencing subsequent analysis reveal open read frame ORFs represent entire protein gene tricistronic gene membrane protein gene bicistronic gene protein gene order 5′–3′ The overall nucleic acid structure encode region TCoV similar homologous region IBV The consensus transcription regulate sequence TRS IBV AACAA highly conserve TCoV genome level nucleotide sequence location regard initiation codon individual gene Pair wise comparison gene gene gene gene sequence counterpart IBV reveal high level 1–92 similarity Phylogenetic analysis base deduce amino acid sequence protein demonstrate TCoV cluster within genomic lineage IBV strain mammalian coronaviruses group separate cluster correspond antigenic group There substantial difference protein sequence TCoV IBV 8–33 similarity',\n",
       "  0.6041929125785828),\n",
       " ('vitro assemble recombinant infectious bronchitis virus demonstrate open reading frame essential replication Molecular clone infectious bronchitis virus IBV derive Vero cell adapt Beaudette strain construct use vitro assembly method vitro transcribe RNA cDNA template construct seven cDNA fragment encompass entire genome IBV electroporated BHK cell The cell overlay onto susceptible Vero cell viable virus recover molecular clone The molecularly clone IBV MIBV demonstrate growth kinetics plaque size morphology resemble parental Beaudette strain IBV The recombinant virus manipulate express enhance green fluorescent protein EGFP replace open reading frame ORF group specific gene ORF EGFP ORF The rescue recombinant virus express EGFP GIBV replicate low viral titer form small plaque compare parental virus MIBV After six passage GIBV minority plaque observe reverted large plaque size virus plaque longer express EGFP Direct sequence PCR product derive cell infect plaque purify virus lose expression EGFP confirm loss EGFP ORF The loss EGFP expression Δ5a IBV also accompany reversion growth kinetics resemble standard virus intact recombinant virus This study demonstrate ORF essential viral multiplication Vero cell',\n",
       "  0.6039001941680908),\n",
       " ('Effective Amplification DNA Reverse Transcription PCR Polymerase chain reaction apply amplification long DNA fragment variety source include genomic mitochondrial viral DNAs However polymerase chain reaction amplification cDNA template produce reverse transcription generally restrict product less kilobases paper report system effectively amplify fragment kilobases human coronavirus 229E genomic RNA demonstrate integrity RNA template prevention false prim event reverse transcription critical parameter achieve synthesis long cDNAs The optimization polymerase chain reaction condition enable improve specificity yield product definitive Finally show reverse transcription polymerase chain reaction technology use amplification extend region dystrophin mRNA cellular RNA relatively low abundance',\n",
       "  0.6037269830703735),\n",
       " ('ubc4', 0.6037100553512573),\n",
       " ('Preferential requirement frameshift product p88 replication Red clover necrotic mosaic virus RNA1 The genome Red clover necrotic mosaic virus RCNMV consist RNA1 RNA2 RNA1 encode terminally overlap replication protein p27 p88 translate cap independent manner The untranslated region RNA1 contain RNA element essential cap independent translation negative strand RNA synthesis study investigate p27 p88 engage replication RCNMV genomic RNAs use DNA vector vitro transcribe RNAs protoplast cell free extract evacuolated protoplast Our result show cis preferential requirement p88 p27 replication RNA1 This mechanism might help facilitate switch role RNA1 mRNA replication template',\n",
       "  0.6031848192214966),\n",
       " ('monodispersity', 0.603087842464447),\n",
       " ('peudorabies', 0.6029495000839233),\n",
       " ('mediana', 0.602774441242218),\n",
       " ('troa', 0.602719783782959),\n",
       " ('iir', 0.6025745868682861),\n",
       " ('intérêt', 0.6023483872413635),\n",
       " ('The Nucleocapsid Protein Coronavirus Mouse Hepatitis Virus Interacts Cellular Heterogeneous Nuclear Ribonucleoprotein Vitro Vivo The nucleocapsid protein mouse hepatitis virus MHV cellular heterogeneous nuclear ribonucleoprotein hnRNP RNA bind protein bind leader RNA intergenic sequence MHV negative strand template RNAs respectively Previous study suggest role hnRNP proteins MHV RNA synthesis However know whether two protein interact Here employ series method determine interaction vitro vivo Both hnRNP gene clone express Escherichia coli glutathione transferase GST fusion protein interaction determine GST bind assay Results show protein directly specifically interact hnRNP vitro dissect protein bind domain protein deletion construct make PCR express GST fusion proteins Two hnRNP binding site identify protein site locate amino acid site amino acid addition find protein interact self interact domain coincide site site Using fluorescence double staining technique show protein colocalized hnRNP cytoplasm particularly perinuclear region MHV infect cell viral RNA replication transcription occur The protein hnRNP coimmunoprecipitated lysates MHV infect cell either hnRNP specific monoclonal antibody indicate physical interaction hnRNP proteins Furthermore use yeast two hybrid system show protein interact hnRNP vivo These result thus establish MHV protein interact hnRNP vitro vivo',\n",
       "  0.6020388603210449),\n",
       " ('baculovirus—categorized', 0.6017839908599854),\n",
       " ('brown', 0.6013741493225098),\n",
       " ('schrödinger', 0.6012895107269287),\n",
       " ('Immunopurification apply study virus protein composition encapsidation protocol obtain small amount highly pure virus preparation start reduce volume 5ml infected tissue culture supernatant describe This procedure adapt study transmissible gastroenteritis coronavirus TGEV protein composition RNA encapsidation This protocol rely virion capture monoclonal antibody specific virion membrane protein These antibody bind protein coat ELISA well arm rabbit anti mouse IgG antibodies example purification 35S label TGEV virion perform The quality virus preparation assess quantify common contaminate RNAs real time PCR The critical factor influence purity degree analyse discuss',\n",
       "  0.60054612159729),\n",
       " ('kümel', 0.6003333330154419),\n",
       " ('imprison', 0.5996694564819336),\n",
       " ('demystify', 0.5996685028076172),\n",
       " ('Detection bovine enteric coronavirus clinical specimen hybridization cDNA probe Molecular hybridization previously optimize purified bovine coronavirus BCV adapt detection virus clinical specimen For purpose accuracy exist test improve suitable mean removal extraneous molecule establish Six radioactive probe use obtain adequate detection signal These probe contain complete gene sequence sequence hybridize total length viral RNA Genomic RNA could detect direct spotting sample prior Freo nextraction centrifugation specimens cushion sucrose improve considerably positive identification virus contain sample RNA detection positive clinical specimen significantly good hybridization immunological detection BCV ELISA although difference slight two passage virus HRT cell monolayers Consequently reliability positive negative test result hybridization test Freon extract specimen well ELISA However result extraction organic solvent inferior The accuracy ELISA surpass hybridization assay Background signal due vector homology find negligible sample analyze',\n",
       "  0.5995780229568481),\n",
       " ('kakegawa', 0.5981764793395996),\n",
       " ('Identification small rna contain trypanosome splice leader donor share sequence trypanosomatid mRNAs The nucleotide spliced leader sequence find end numerous trypanosome mRNAs yet tandemly organize reiteration unit encode leader detectably link structural gene Here report presence class discrete small RNA molecules derive genomic reiteration unit Trypanosoma brucei Trypanosoma cruzi Leptomonas collosoma These small RNAs nucleotide respectively contain terminal like sequence nuclease analyse demonstrate small RNAs transcribe continuous sequence within respective reiteration unit With exception sequence concensus donor splice site immediately follow small RNAs well conserve suggest small RNAs may function donor sequence intermolecular process place terminus many trypanosomatid mRNAs',\n",
       "  0.5971258282661438),\n",
       " ('novel method capture base purification whole viral native RNA genome Current technology target characterization manipulation viral RNA primarily involve amplification ultracentrifugation isopycnic gradient viral particle decrease host RNA background The former strategy non compatible characterizing property innate RNA strand secondary structure RNA–RNA interaction also nanopore direct RNA sequence involve sequence native RNA strand The latter strategy ultracentrifugation cause loss genomic information due inability retrieve unassembled viral RNA address develop novel application current nucleic acid hybridization technology direct characterization RNA particular modify current enrichment protocol capture whole viral native RNA genome downstream RNA assay circumvent abovementioned problem This technique involve hybridization biotinylated bait nucleotide interval stringent wash release free native RNA strand use DNase treatment turnaround time min qPCR use primary proof concept capture base purification indeed remove host background Subsequently capture base purification apply direct RNA sequence proof concept capture base purification couple downstream RNA assay report protocol able successfully purify viral RNA fold also report application protocol direct RNA sequencing yield reduction human host RNA background fold recovery viral genome least 15× coverage mean coverage across genome 120× This report best knowledge first description capture base purification method assay involve direct manipulation characterisation native RNA This report also describe successful application capture base purification direct RNA sequence strategy address certain limitation current strategy sequence RNA viral genome ELECTRONIC SUPPLEMENTARY MATERIAL The online version article s13568 contains supplementary material available authorized user',\n",
       "  0.5968791246414185),\n",
       " ('Characterization protein self association coronavirus ribonucleoprotein complex Mouse hepatitis virus MHV nucleocapsid protein bind large single strand positive sense viral genomic RNA form helical nucleocapsid structure mature virion addition protein bind intracellular form genomic RNA MHV subgenomic mRNAs express non MHV RNA transcript form ribonucleoprotein RNP complex infect cell Among intracellular viral RNP complex genomic RNP complex package virus particles The present study demonstrate protein MHV virion nucleocapsid intracellular genome length RNP complex bound viral envelope protein tightly self associate association retain even extensive RNase treatment RNP complexes The RNase resistant tight protein association virion nucleocapsid mediate intermolecular disulfide bridge proteins contrast protein association majority intracellular RNP complex susceptible RNase treatment Because RNP complex specifically interact protein selectively package MHV particle present data suggest distinct difference protein association viral genomic RNP complex undergo packaging virus particle subgenomic RNP complex package MHV particle',\n",
       "  0.5966171026229858),\n",
       " ('Detection relate positive strand RNA virus genome reverse transcription polymerase chain reaction use degenerate primer common replicase sequence set degenerate sense antisense primer design basis short segment identical amino acid predict ORF replicase protein lactate dehydrogenase elevate virus LDV equine arteritis virus EAV porcine reproductive respiratory syndrome virus strain Lelystad virus PRRSV member new group positive strand RNA viruses Reverse transcription polymerase chain reaction amplification use set degenerate primer yield product expect size genome three virus also yield product appropriate size genome another strain PRRSV VR2332 ORF sequence unknown end genome differs PRRSV genome product generate genome simian hemorhagic fever virus SHFV another member virus group However appropriate product generate second set degenerate primer design ORF segment LDV EAV PRRSV first set basis human codon preference Sequence analysis show amplified SHFV ORF segment exhibit nucleotide identity correspond segment ORF LDV EAV PRRSV The result show degenerate primer set might useful search relate virus mammalian specie',\n",
       "  0.5956677794456482),\n",
       " ('The terminus VSV particle RNAs sufficient signal RNA encapsidation replication bud generate infectious particle Infectious defective interfering particles negative strand RNA virus vesicular stomatitis virus VSV havebeen recover negative sense transcript plasmid contains full length cDNA derive particle genome order determine act sequence necessary RNA replication encapsidation bud approximate minimal size RNA package infectious particle construct series internal deletion cDNA generate plasmid could transcribe yield RNAs range size nucleotides nucleotide All deletion plasmid retain least nucleotide terminus nucleotide terminus genome cell express five VSV protein delete RNAs examine ability encapsidated replicate bud produce infectious particles RNA small nucleotide contain nucleotide end nucleotide end genome encapsidated replicate bud least efficiently full length wild type RNA contrast nucleotide RNA contain nucleotide end RNA perfect nucleotide complement end replicate poorly fail bud infectious particle However RNA insertion nucleotide stuffer sequence non VSV origin two nucleotide complementary terminus replicate also bud infectious particle These data show signal necessary RNA encapsidation replication package infectious particle contain within terminal nucleotide terminal nucleotide RNA genome Furthermore result show heterologous sequence replicate package infectious particle flank RNA terminus',\n",
       "  0.5955802202224731),\n",
       " ('Quantitative sense specific determination murine coronavirus RNA reverse transcription polymerase chain reaction many application useful know sense amount viral RNAs present sample theory sense specific measurement viral RNAs may achieve reverse transcription polymerase chain reaction PCR assay utilize primer define polarity step However practice show assay prone artifacts non specific prim drastically diminish reliability Using murine coronavirus MHV model describe validate several modification PCR procedure eliminate artifact Key PCR parameter optimize include design tag primer DNase treatment vitro transcribe RNA standard specification temperature difference PCR anneal step use competitive RNA template quantitative assay The assay describe may use determine sense abundance viral host RNA interest complex biological specimen',\n",
       "  0.5954920053482056),\n",
       " ('rnas', 0.5951570272445679),\n",
       " ('Host cell proteins interact end TGEV coronavirus genome influence virus replication Coronavirus RNA synthesis perform multienzymatic replicase complex together cellular factor This process require specific recognition RNA act signal locate end viral genome identify cellular protein involve coronavirus RNA synthesis transmissible gastroenteritis coronavirus TGEV genome end harbor essential act signal replication use bait RNA affinity protein purification Ten proteins preferentially pull either end genome identified proteomic analysis Nine include member heterogeneous ribonucleoprotein family protein hnRNPs poly bind protein PABP p100 transcriptional activator protein two aminoacyl tRNA synthetases show preferential binding end genome whereas polypyrimidine tract bind protein PTB preferentially pull end genome The potential function end interacting protein virus replication study analyze effect silence use TGEV derive replicon infectious virus Gene silence PABP hnRNP glutamyl prolyl tRNA synthetase EPRS cause significant fold reduction viral RNA synthesis Interestingly silence glyceraldehyde phosphate dehydrogenase GAPDH initially use control gene cause fold increase viral RNA synthesis system These data suggest PABP hnRNP EPRS play positive role virus infection could mediate interaction viral end GAPDH negative effect viral infection',\n",
       "  0.5951486229896545),\n",
       " ('Identification RNase Dependent End Modified Viral Small RNAs Sindbis Virus Infected Mammalian Cells Small RNAs play critical role host pathogen interaction Indeed small RNA mediate silence RNA interference RNAi one early form antiviral immunity Although represent main defense system viruses many organism antiviral role RNAi clearly proven high vertebrate However well establish response viral infection relies recognition viral RNAs host pattern recognition receptor PRRs trigger activation interferon pathway present work report existence novel small noncoding RNA population produce mammalian cell upon RNA virus infection Using Sindbis virus SINV prototypic arbovirus model profile small RNA population infect cell human African green monkey cell line Here provide evidence presence discrete small RNAs viral origin associate RNA induce silence complex RISC highly express detected Northern blot analysis accumulate nucleotide specie infection report cellular antiviral endoribonuclease RNase cleave viral genome produce turn small RNAs Surprisingly uncover presence modification end nucleotide SINV derive viral small RNAs SvsRNAs might origin stability Altogether find show stable modify small viral RNAs could represent novel way modulate host virus interaction upon SINV infection',\n",
       "  0.5950810313224792),\n",
       " ('yyyn', 0.5949442386627197),\n",
       " ('appraisal different method detection walnut strain cherry leafroll virus Three method evaluate detection cherry leafroll virus ELISA dot blot reverse transcriptional polymerase chain reaction PCR Dot blot PCR carry crude plant extract without RNA purification Dot blot hybridization use 32P labelled DNA probe sensitive previously report ELISA result cherry leafroll virus detection The sensitive method PCR amplify specific fragment untranslated region viral genomic RNAs PCR use detect cherry leafroll virus infect walnut bud twig',\n",
       "  0.5945864915847778),\n",
       " ('wozniak', 0.5943486094474792),\n",
       " ('cydia', 0.5940064191818237),\n",
       " ('permeable', 0.5939664840698242),\n",
       " ('Biosynthesis processing ribophorins endoplasmic reticulum Ribophorins two transmembrane glycoprotein characteristic rough endoplasmic reticulum think involve bind ribosome Their biosynthesis study vivo use line culture rat hepatocytes clone pituitary cell cell free synthesis experiment vitro translation mRNA extract free bound polysomes clone cell demonstrate ribophorins make exclusively bound polysomes The primary translation product ribophorin messenger obtain cultured hepatocytes regenerate liver migrate respective mature proteins slightly high apparent molecular weight unglycosylated form immunoprecipitated cell treat tunicamycin This indicate ribophorins contrast endoplasmic reticulum membrane proteins previously study contain transient amino terminal insertion signal remove translationally Kinetic pulse chase experiment 35S methionine mannose demonstrated ribophorins subject electrophoretically detectable posttranslational modification proteolytic cleavage trim terminal glycosylation oligosaccharide side chain Direct analysis oligosaccharides ribophorin show contain terminal sugar characteristic complex oligosaccharide range composition Man8GlcNAc Man5GlcNAc These finding well observation mature proteins sensitive endoglycosidase insensitive endoglycosidase consistent notion biosynthetic pathway ribophorins require stage passage Golgi apparatus',\n",
       "  0.5938949584960938),\n",
       " ('oligo', 0.5935506820678711),\n",
       " ('intranares', 0.5932024121284485),\n",
       " ('subgenomic', 0.5929040312767029),\n",
       " ('Experimental confirmation recombination upstream hypervariable region infectious bronchitis virus Chimeric infectious bronchitis virus IBV genomes cross site gene generate infection two distinct IBV strain Recombinant virus collect chicken embryo embryonic culture cell chicken infect Ark99 Mass41 strain purify differential centrifugation The recombinant gene identify reverse transcription polymerase chain reaction RTPCR use heterologous primer confirm nucleotide sequence The recombinant Ark99 Mass41 sequence identify follow vitro ovo vivo infection Mixed RNA extract Ark99 Mass41 produce RTPCR product primer PCR condition use Cross site within amplified Mass41 Ark99 base gene could detect nucleotide While region lie upstream hypervariable region correspond site commonly identify naturally occur isolates recombination site identify study could detect within HVR genomes chimeric virus',\n",
       "  0.5929038524627686),\n",
       " ('Identification kDa Polypeptide Processed Coronavirus Infectious Bronchitis Virus Polyprotein like Proteinase Determination Its Cleavage Sites report identification kDa polypeptide IBV infect Vero cell immunoprecipitation region specific antiserum raise rabbit IBV sequence encode nucleotide Coexpression deletion mutagenesis study demonstrate protein encode ORF nucleotide release polyprotein like proteinase mediate proteolysis previously predict Q3462S3463 dipeptide bond encode IBV sequence nucleotide identify terminal cleavage site Q3672N3673 dipeptide bond encode IBV sequence nucleotide identify terminal cleavage site kDa polypeptide',\n",
       "  0.5928101539611816),\n",
       " ('rs1801275gg', 0.5919702053070068),\n",
       " ('rabid', 0.5918769240379333),\n",
       " ('Structure black beetle virus genome functional implication The black beetle virus BBV isometric insect virus whose genome consist two messenger active RNA molecule encapsidated single virion The nucleotide sequence BBV RNA1 base determine together sequence BBV RNA2 base provide complete primary structure BBV genome The RNA1 sequence encompass non cod region nucleotide cod region protein predict molecular weight protein implicated viral RNA synthesis proximal region encode RNA3 base subgenomic messenger RNA make infected cell encapsidated virion The RNA3 sequence start base inside cod region protein contains two overlap open reading frame proteins molecular weight one believe protein make BBV infected cell limited homology exists sequence RNA1 RNA2 Sequence region identify provide energetically favorable bond RNA2 RNA1 possibly facilitate common encapsidation RNA2 negative strand RNA1 possibly regulate production RNA3',\n",
       "  0.5916129350662231),\n",
       " ('Regulation Coronaviral Poly Tail Length Infection The positive strand coronavirus genome kilobase length subgenomic mRNAs short lengths terminal virtue common cap leader common polyadenylated untranslated region Here ligate head tail viral RNAs bovine coronavirus infect cell sequence across ligate junction learn time peak viral RNA synthesis hour postinfection hpi poly tail genomic sgmRNAs nucleotide length Surprisingly length find vary throughout infection immediately virus entry hpi later hpi hpi hpi With method poly sequence length simultaneously find ligated viral negative strand RNAs Functional analyse poly tail length specific viral RNA specie furthermore reveal translation vivo RNAs longer poly tail enhance short poly Although mechanisms tail length vary unknown experimental result together suggest length poly poly tail regulate One potential function regulate poly tail length might coronavirus genome long poly favor translation The regulation coronavirus translation poly tail length resemble embryonal development suggesting may mechanistic parallel',\n",
       "  0.5915724039077759),\n",
       " ('bmmy', 0.5912365913391113),\n",
       " ('xestospongia', 0.5911242961883545),\n",
       " ('The protein minor structural component PRRSV Porcine reproductive respiratory syndrome virus PRRSV ORF2 contain internal ORF code small non glycosylated protein know Previous work identify presence 10kDa protein virus infect cell induction anti response PRRSV infect pig well possible association virion Virology 183–191 study utilized two experimental approach include use peptide specific monoclonal antibody demonstrate PRRSV protein integral component PRRSV virion This study suggest PRRSV similar protein EAV form minor structural component virion',\n",
       "  0.5909963250160217),\n",
       " ('Analysis intermolecular RNA–RNA recombination rubella virus investigate whether rubella virus RUB undergo intermolecular RNA–RNA recombination cell cotransfected pair vitro transcript genomic cDNA plasmid vector engineer contain nonoverlapping deletion replicative transcript maintain proximal nonstructural ORF contain replicase make RNA replication competent deletion proximal structural protein ORF maintain end genome include putative act element CSE nonreplicative transcript consist half genome include ORF CSE Cotransfection yield plaque form virus synthesize standard genomic subgenomic RNAs thus generate RNA–RNA recombination Using transcript tag terminal deletion find recombinant contain end derive replicative strand indicate cis preference initiation negative strand synthesis cotransfections replicative transcript lack CSE recombination occur albeit low efficiency indicate initiation trans ORF occur The CSE sufficient nonreplicative transcript show serve promoter negative strand RNA synthesis While deletion mutagenesis show presence junction untranslated region UTR ORFs appear necessary transcript recombination region genome analysis transcript tag restriction site show UTR hot spot recombination compare neighboring region ORFs Sequence analysis recombinant reveal precise homologous imprecise recombination aberrant homologous result duplication occur however imprecise recombination involve UTR end ORF UTR maintain ORF indicate selection pressure duplication region genome',\n",
       "  0.5909338593482971),\n",
       " ('interperiod', 0.5905661582946777),\n",
       " ('Preparation Evaluation Ribonuclease Resistant Viral HIV RNA Standards Based Armored RNA Technology BACKGROUND The human immunodeficiency virus type HIV infectious viral agent gradually extinguish immune system result acquire immune deficiency syndrome AIDS The aim study construct RNA positive control base armored RNA technology use HIV RNA model METHODS The MS2 maturase coat protein gene position HIV pol gene clone pET 32a plasmid The prepared plasmid transform Escherichia coli strain BL2 DE3 expression construct induce isopropyl thio galactopyranoside IPTG obtain fabricated RNA The RNA precipitate purified use polyethylene glycol Sephacryl chromatography RESULTS The stability RNA evaluate treatment DNase RNase confirmed transmission electron microscopy gel agarose electrophoresis Tenfold serial dilution RNA prepared Real time PCR assay range detection addition value slope standard curve CONCLUSION Prepared RNA positive control could use basis launch house HIV virus assay infectious agent readily available laboratory HIV research center The RNA non infectious highly resistant ribonuclease enzyme reduce risk infection clinical laboratory',\n",
       "  0.5903645157814026),\n",
       " ('antimouse', 0.5898983478546143),\n",
       " ('The Functional Roles Cis act Elements Bamboo mosaic virus RNA Genome Bamboo mosaic virus BaMV belong genus Potexvirus family Alphaflexiviridae single strand positive sense RNA genome approximately nucleotide length Positive sense RNA viruses use genomic RNA template translation replication enter suitable host cell Furthermore viral RNA recognize capsid protein package viral movement protein movement protein complex cell cell systemic movement Hence viral RNA must contain signal different function complete viral infection cycle review examine various act element genome BaMV The highly structured untranslated region UTR BaMV genomic RNA play multiple role BaMV infection cycle include target chloroplast RNA replication provide initiation site synthesis minus strand RNA signal polyadenylation direct viral long distance movement The extreme end structure terminus minus strand RNA involve initiation plus strand genomic RNA synthesis Both region map report interact viral encode RNA dependent RNA polymerase Moreover sequence upstream open reading frame ORFs involved regulate subgenomic RNA synthesis The act element identify BaMV RNA discuss compare potexviruses',\n",
       "  0.5895464420318604),\n",
       " ('beclin1', 0.5888361930847168),\n",
       " ('Proteolytic Mapping Coronavirus Infectious Bronchitis Virus Polyprotein Evidence Presence Four Cleavage Sites like Proteinase Identification Two Novel Cleavage Products previously report like proteinase coronavirus infectious bronchitis virus IBV responsible process polyproteins three mature product kDa Liuet Liu The terminal cleavage site kDa protein define Q891 S892 dipeptide bond encode nucleotide Liu Brown report cleavage sit like proteinase polyprotein encode ORF region map coexpression deletion site direct mutagenesis study Using two ORF specific antiserum V58 V17 three dipeptide bond encode nucleotides respectively demonstrated cleavage sit like proteinase Cleavage four position would result release four mature product molecular mass approximately kDa Among kDa protein specifically identify IBV infected cell Taken together kDa protein previously identify result suggest ORF region IBV mRNA1 may able encode five mature product',\n",
       "  0.588314414024353),\n",
       " ('Further Identification Characterization Novel Intermediate Mature Cleavage Products Released ORF Region Avian Coronavirus Infectious Bronchitis Virus Polyprotein The coronavirus like proteinase one viral proteinases responsible process polyproteins multiple mature product cell infected avian coronavirus infectious bronchitis virus IBV three proteins kDa respectively previously identify mature cleavage product release region polyprotein like proteinase report show identification two cleavage product kDa release region polyprotein addition two stable intermediate cleavage product molecular mass kDa respectively identify IBV infected cell The kDa protein show intermediate cleavage product cover kDa protein kDa protein intermediate cleavage product cover kDa protein Immunofluorescent stain IBV infected cell cell express individual cleavage product show kDa protein associate membrane endoplasmic reticulum kDa protein display diffuse distribution pattern',\n",
       "  0.5881900191307068),\n",
       " ('Polymerase chain reaction probe synthesis direct amplification detection bovine coronavirus The polymerase chain reaction PCR use synthesize probe detection bovine coronavirus BCV use recombinant plasmid template molecules The probe detect minimum viral genome exposure detection limit similar nick translate probes exposure film More exposure blot probe probe result complete darkening film The probe synthesize asymmetric PCR linearize plasmid permit detection genome equal capacity random prim probe Prolonged exposure increase background case PCR probe blot Probes synthesize asymmetric PCR random prim label similar specific activity well term sensitivity detectability oppose nick translate probe However specificity detection probe random prim probe increase About viral genome fragment specific amplification PCR detect agarose gel analysis PCR probe synthesis simple highly reproducible allow synthesis probe specific fragment',\n",
       "  0.5881300568580627),\n",
       " ('Recombination Coronavirus Defective Interfering RNAs Naturally occur defective interfering RNAs find coronavirus specie They range size approximately parent virus genome The large RNAs case appear require helper virus intracellular replication postulate may function agent disease Coronavirus RNAs appear arise internal deletion nonhomologous recombination event virus genome RNAs large size polymerase strand switch copy choice mechanism addition use study virus RNA replication virus assembly coronavirus RNAs use major way study mechanism high frequency site specific RNA recombination event lead leader acquisition virus replication leader fusion event occur synthesis subgenomic mRNAs leader switch event occur RNA replication distinguish feature coronaviruses arteriviruses Coronavirus RNAs also engineer vehicle generation target recombinant parent virus genome',\n",
       "  0.5879695415496826),\n",
       " ('Bovine coronavirus protein synthesis follow ribosomal scan bicistronic mRNA The mRNA encode kDa nucleocapsid protein bovine coronavirus bicistronic kDa protein term protein internal open reading frame ORF also synthesize reading frame begin downstream start codon Sequences flank start codon suggest ORF might access scan ribosome pass start codon Here test idea demonstrate translation study vitro vivo protein synthesized accord leaky scan model initiation translation subgenomic mRNA molecule',\n",
       "  0.5878092050552368),\n",
       " ('griffonia', 0.5873204469680786),\n",
       " ('Polyprotein Processing Determinant Vitro Activity Semliki Forest Virus Replicase Semliki Forest virus SFV arthropod borne alphavirus induces membrane invagination spherule host cell These harbor viral replication complex synthesize viral RNA Alphaviruses four replicase nonstructural protein nsPs nsP1–4 express polyprotein P1234 early synthesize minus strand RNA form polyprotein P123 polymerase nsP4 Further proteolytic cleavage result late consist nsP1–4 synthesize plus strand Here show late RCs highly active RNA synthesis vitro Furthermore demonstrate active RCs isolate virus infected cell cell transfected wild type replicase combination plasmid express template RNA When uncleavable polyprotein P123 polymerase nsP4 express together template high level minus strand RNA produce cell RCs isolate cell hardly active vitro Furthermore observe uncleavable polyprotein P123 polymerase nsP4 previously show form spherule even absence template replicate exogenous template Consequently hypothesize replicase protein sequester spherule longer able recruit template',\n",
       "  0.5868781208992004),\n",
       " ('Sequence analysis BECV F15 matrix protein Clones bovine enteric coronavirus F15 cDNA library clone pBR322 sequence method Sanger Coulson This lead identification sequence base contain single open reading frame base yield protein property matrix protein comprise amino acid molecular weight hydrophobic net charge neutral Analysis secondary structure could establish simple transmembrane arrangement amino acid Comparison nucleotide sequence BECV Mebus strain show two base change result homology two amino acid sequence Furthermore conserve structure appear comparison Dayoff optimal alignment MHV A59 MHV JHM TGEV IBV Beaudette IBV 82M amino acid sequence two strain BECV F15 Mebus present antigenic difference lead reconsider role viral antigen specificity hypothesis seem possess necessary information transmembrane region ideal candidate viral budding process',\n",
       "  0.5868079662322998),\n",
       " ('mg1655', 0.5866612195968628),\n",
       " ('Characterization coronavirus RNA polymerase gene product',\n",
       "  0.5866014957427979),\n",
       " ('Genomic characterization equine coronavirus The complete genome sequence first equine coronavirus ECoV isolate NC99 strain accomplish directly sequence overlap fragment RT–PCR amplify viral RNA The ECoV genome nucleotide length exclude polyA tail Analysis sequence identify open read frame encode two replicase polyproteins five structural protein hemagglutinin esterase spike envelope membrane nucleocapsid four accessory protein NS2 p12 The two replicase polyproteins predict proteolytically process three virus encode protease non structural protein nsp1–16 The ECoV nsp3 protein considerable amino acid deletion insertion compare nsp3 protein bovine coronavirus human coronavirus OC43 porcine hemagglutinate encephalomyelitis virus three group coronaviruses phylogenetically closely related ECoV The structure subgenomic mRNAs analyze Northern blot analysis sequence leader–body junction mRNA',\n",
       "  0.5864881277084351),\n",
       " ('Integration membrane protein endoplasmic reticulum require GTP examine requirement ribonucleotides ribonucleotide triphosphate hydrolysis early event membrane integration two membrane protein protein vesicular stomatitis virus hemagglutinin neuraminidase glycoprotein Newcastle disease virus Both proteins contain single transmembrane span segment integrate membrane opposite orientation The protein amino terminal signal sequence stop transfer sequence locate near carboxy terminus The glycoprotein single sequence near amino terminus function signal sequence transmembrane span segment Membrane insertion explore use cell free system direct transcribed mRNAs encode amino terminal segment two protein Ribosome bound nascent polypeptide assemble ribonucleotides remove gel filtration chromatography ribosome incubate microsomal membrane condition define ribonucleotide content Nascent chain insertion membrane require presence signal recognition particle functional signal recognition particle receptor absence ribonucleotides insertion nascent membrane protein detect GTP nonhydrolyzable GTP analogue promote efficient insertion ATP comparatively ineffective Surprisingly majority nascent chain remain ribosome associate puromycin treatment Ribosome associate nascent chain remain competent membrane insertion free chain competent conclude GTP bind protein performs essential function ribosome dependent insertion membrane protein endoplasmic reticulum unrelated protein synthesis',\n",
       "  0.5864133238792419),\n",
       " ('travepinet', 0.5859013795852661),\n",
       " ('sequentially', 0.5858926177024841),\n",
       " ('calorimetric', 0.58576500415802),\n",
       " ('orf29a', 0.5853558778762817),\n",
       " ('ofde', 0.585243821144104),\n",
       " ('sp6', 0.5852317214012146),\n",
       " ('28a', 0.5850341320037842),\n",
       " ('Identification sequence determination capsid protein gene human astrovirus serotype present sequence open reading frame ORF end human astrovirus serotype Primer extension experiment show RNA express gene short complete ORF could form protein The protein express recombinant baculovirus recognize anti virion serum indicate structural role Sequence comparison indicate astrovirus serotypes differ markedly terminal half protein well conserve towards terminus',\n",
       "  0.584902286529541),\n",
       " ('p450s', 0.5847070813179016),\n",
       " ('The small envelope protein porcine reproductive respiratory syndrome virus possess ion channel protein like property The small envelope protein porcine reproductive respiratory syndrome virus PRRSV hydrophobic amino acid protein encode internal open reading frame ORF bicistronic mRNA2 first step towards understand biological role protein PRRSV replication gene expression block full length infectious clone mutate ATG translational initiation GTG full length mutant genomic clone unable synthesize protein DNA transfection PRRSV susceptible cell gene knock genomic clone show absence virus infectivity P129 transfected cell however produce virion particle culture supernatant particle contain viral genomic RNA demonstrate protein essential PRRSV infection dispensable virion assembly Electron microscopy suggest P129 virion assemble absence similar appearance wild type particle Strand specific PCR demonstrate protein negative non infectious P129 virus particle able enter cell step replication interrupt The entry PRRSV suggest via receptor mediate endocytosis lysomotropic basic compound know ion channel block agent inhibit PRRSV replication effectively uncoating process The expression protein Escherichia coli mediate cell growth arrest increase membrane permeability Cross link experiment cell infect PRRSV transfected gene show protein able form homo oligomers Taken together data suggest PRRSV protein likely ion channel protein embed viral envelope facilitate uncoating virus release genome cytoplasm',\n",
       "  0.5845426917076111),\n",
       " ('Two RNA bind site plant fibrillarin provide interaction various RNA substrates Fibrillarin one major protein nucleolus play several essential role ribosome biogenesis include pre rRNA process ribose methylation rRNA snRNAs Recently show fibrillarin play role virus infection associate viral RNPs Here demonstrate ability recombinant fibrillarin Arabidopsis thaliana AtFib2 interact RNAs different length type include rRNA snoRNA snRNA siRNA viral RNAs vitro Our data also indicate AtFib2 possess two RNA bind site central 138–179 amino acid terminal 225–281 amino acid part protein respectively The conserved GCVYAVEF octamer bind RNA directly suggest early may assist proper fold central RNA binding site',\n",
       "  0.5843693017959595),\n",
       " ('Switch Cap Factorless IRES Dependent Frame Translation Cellular Stress Dicistrovirus Infection Internal ribosome entry sit IRES utilized subset cellular viral mRNAs initiate translation cellular stress virus infection canonical cap dependent translation compromise The intergenic region IGR IRES Dicistroviridae use streamline mechanism directly recruit ribosome absence initiation factor initiate translation use non AUG codon subset IGR IRESs include honey bee virus also direct translation overlap frame gene study systematically examine cellular condition lead IGR IRES mediate frame translation Drosophila cell Towards novel bicistronic reporter exploit stop peptide develop allow detection IRES mediate translation vivo Both frame translation IGR IRES stimulate number cellular stress cell infect cricket paralysis virus demonstrate switch cap dependent IRES dependent translation The regulation IGR IRES mechanism ensure frame viral structural protein frame ORFx protein optimally express virus infection',\n",
       "  0.5842685103416443),\n",
       " ('Construction tobacco mosaic virus subgenomic replicons replicate spread systemically tobacco plant Two tobacco mosaic virus TMV derive replicons create deletion kDa open reading frame ORF replicate systemically invaded tobacco plant support wild type TMV One RNA replicon contain internal direct repeat nucleotide end kDa ORF Although RNA replicate progeny heterogeneous size small original transcript second TMV derive RNA replicon without internally repeat sequence contain deletion portion kDa ORF well kDa ORF create coinoculated wild type TMV helper This RNA also replicate efficiently systemically invaded tobacco plant examination sequence cDNA clone obtain PCR amplification progeny population RNA replicon demonstrate observed size heterogeneity due deletion insertion adjacent artificially create deletion junction These data demonstrate TMV infection capable support artificially create RNA replicon similar defective interfering RNAs satellite However dependent RNAs replicate without noticeably interfere wild type TMV symptom replication',\n",
       "  0.5841783881187439),\n",
       " ('Sequence mouse hepatitis virus A59 mRNA Indications RNA recombination coronaviruses influenza virus The nucleotide sequence unique region coronavirus MHV A59 mRNA determine Two open read frame ORF predict ORF1 potentially encode protein amino acid amino acid sequence contains element indicate nucleotide bind property ORF2 predict amino acid protein lack translation initiation codon therefore probably pseudogene The amino acid sequence ORF2 share homology HA1 hemagglutinin sequence influenza virus short stretch nucleotide immediately upstream ORF2 share homology MHC class nucleotide sequence discuss possibilitythat similarity result recombination present model acquisition subsequent inactivation ORF2 model applies also MHV A59 related coronaviruses expect ORF2 still functional',\n",
       "  0.5837279558181763),\n",
       " ('Trypanosome mRNAs share common spliced leader sequence terminal leader sequence nucleotide find present multiple trypanosome RNAs Based representation cDNA library estimate many trypanosome mRNAs contain leader This leader originally identify mRNAs encode molecule responsible antigenic variation variant surface glycoprotein Studies select cDNAs contain leader sequence reveal leader contain transcript stage specific stage regulate constitutive They abundant rare transcribe single multigene family linkage genomic leader sequence structural gene exon observe Possible mechanisms leader sequence add trypanosome mRNAs discuss',\n",
       "  0.5832042694091797),\n",
       " ('ferment', 0.5831647515296936),\n",
       " ('Sequence vitro expression phosphoprotein gene avian pneumovirus The phosphoprotein gene two subgroup strain avian pneumovirus comprise nucleotide contain one substantial open reading frame encode protein amino acid predict vitro translation mRNA wheat germ system result synthesis two polypeptide Comparison deduce protein sequence know mammalian pneumoviruses reveal overall amino acid identity range suggest distant relationship However much high identity 2–68 region residue include heptad repeat sequence',\n",
       "  0.5829808115959167),\n",
       " ('Downstream Ribosomal Entry Translation Coronavirus TGEV Gene Gene ORF porcine transmissible gastroenteritis coronavirus TGEV encode putative nonstructural polypeptide kDa unknown function translation vitro capable become glycosylated integral membrane protein kDa virulent Miller strain TGEV ORF terminal mRNA 3–1 presumably translate follow cap dependent ribosomal entry For three strain TGEV virulent British FS772 Taiwanese TFI avirulent Purdue mRNA specie 3–1 make ORF present non overlap second ORF mRNA ORF begin base mRNA Purdue strain vitro expression ORF Purdue mRNA like transcript fully conform predict leaky scan pattern suggest ribosome might also enter internally With mRNA like transcript modify carry large ORFs upstream ORF demonstrate ribosome reach ORF enter distant downstream site manner resemble ribosomal shunt Deletion analysis fail identify postulate internal ribosomal entry structure IRES within ORF The result indicate internal entry mechanism possibly conjunction leaky scan use expression ORF TGEV mRNA One possible consequence feature ORF might also express mRNAs',\n",
       "  0.5829718112945557),\n",
       " ('été', 0.5827370882034302),\n",
       " ('18–21', 0.5827303528785706),\n",
       " ('ekv', 0.5823134779930115),\n",
       " ('azodicarbonamide', 0.5819913148880005),\n",
       " ('chonburi', 0.5819188356399536),\n",
       " ('Binding untranslated region coronavirus RNA zinc finger CCHC type RNA binding motif enhances viral replication transcription Coronaviruses RNA synthesis occur cytoplasm regulate host cell protein screen base yeast three hybrid system use untranslated region UTR SARS coronavirus SARS CoV RNA bait human cDNA library derive HeLa cell find positive candidate cellular protein zinc finger CCHC type RNA binding motif MADP1 able interact region SARS CoV genome This interaction subsequently confirm coronavirus infectious bronchitis virus IBV The specificity interaction MADP1 UTR IBV investigate confirm use RNA pull assay The RNA binding domain map terminal region MADP1 protein bind sequence stem–loop IBV UTR MADP1 find translocate cytoplasm partially localized viral replicase transcriptase complex RTCs IBV infect cell deviate usual nuclear localization normal cell use indirect immunofluorescence Using small interfere RNA siRNA MADP1 defective viral RNA synthesis observe knockdown cell therefore indicate importance protein coronaviral RNA synthesis',\n",
       "  0.5818734169006348),\n",
       " ('cruciate', 0.5817379355430603),\n",
       " ('auxquelles', 0.5814184546470642),\n",
       " ('a59', 0.5813066959381104),\n",
       " ('Structure glycoprotein gene sonchus yellow net virus plant rhabdovirus The nucleotide sequence glycoprotein gene sonchus yellow net virus SYNV plant rhabdovirus determine viral genomic mRNA cDNA clone The cistron nucleotide long protein mRNA open reading frame ORF determine cDNA sequence contains encodes protein amino acid Immunoblots antibody elicit purified glycoprotein virus particle react fusion protein produce Escherichia coli indicate clone ORF encodes protein The end protein mRNA correspond relative end viral minus sense genome determine primer extension mRNA isolate infected plant extend position genomic RNA untranslated leader sequence untranslated end flank ORF mRNA The gene junction either side gene genomic RNA identical previously describe SYNV gene similar sequence separate gene animal rhabdoviruses The predicted molecular weight protein value less estimate glycosylated protein virus particles The deduced amino acid sequence SYNV protein share little direct relatedness proteins rhabdovirus appear contain similar signal sequence transmembrane anchor domain glycosylation signal addition SYNV protein contain putative nuclear target site near carboxy terminus may involve transit nuclear membrane prior morphogenesis',\n",
       "  0.5810091495513916),\n",
       " ('Expression frameshifting extremely inefficient proteolytic processing HIV gag pol gene product stably transfected rodent cell line Expression ribosomal frameshifting proteolytic processing HIV GAG POL proteins investigate heterologous mammalian cell order elucidate influence cellular background event DNA fragment encode gag pol region express two rodent cell line LTK BHK Both stably transfected cell line continuously produce recombinant protein react HIV specific antisera The GAG precursor kDa proteolytic fragment thereof major recombinant protein detect Expression gag pol region lead production GAG POL precursor Ribosomal frameshifting HIV shifty sequence typical extent could positively demonstrate enzyme assay Despite presence viral protease within GAG POL precursor proteolytic processing HIV derive polyproteins extremely inefficient The efficiency could enhance overexpression HIV protease encode region',\n",
       "  0.581008791923523),\n",
       " ('The interaction SARS coronavirus non structural protein cellular oxido reductase system cause extensive cytopathic effect The pathological mechanism SARS CoV infection investigate The gene SARS CoV non structural protein locate open reading frame pp1a pp1ab gene synthesize use screen specific cellular gene cod protein interact nsp10 protein human embryo lung cDNA library use yeast trap method The result indicate apart two subunit cellular RNA polymerase complex BTF3 ATF5 nsp10 protein also able interact specifically NADH subunit cytochrome oxidase Further study reveal activity NADH cytochrome alter inner mitochondrial membrane depolarize transfected human embryo lung fibroblast nsp10 protein gene The cytopathic effect Coronavirus 229E strain appear extensive cell control cell',\n",
       "  0.5809097290039062),\n",
       " ('endectocides', 0.580866813659668),\n",
       " ('délétions', 0.5801928043365479),\n",
       " ('The conserved stem loop structure untranslated region Japanese encephalitis virus genome require formation subgenomic flaviviral RNA Flaviviruses accumulate abundant subgenomic RNA sfRNA infect cell report sfRNA result stall host exoribonuclease XRN1 highly structure RNA untranslated region UTR Although XRN1 digestion terminal RNA could stall position generate sfRNA vitro find knock XRN1 effect accumulation sfRNA Japanese encephalitis virus JEV infect cell Mutagenesis study reveal stemloop SLII UTR require accumulation sfRNA According result vitro RNA dependent RNA polymerase RdRp assay RNA fragment contain putative promoter antigenome sfRNA transcription bind RdRp protein exhibit strong promoter activity Taken together result indicate JEV sfRNA could transcribe initially trim XRN1 unidentified exoribonucleases',\n",
       "  0.5801470875740051),\n",
       " ('tetramerize', 0.5800571441650391),\n",
       " ('Genosensor gold film enzymatic electrochemical detection SARS virus sequence hybridisation base genosensor design 100nm sputtered gold film This material work immobilisation transduction surface mer sequence encode short lysine rich region unique SARS severe acute respiratory syndrome virus choose target complementary strand probe label thiol group end immobilise film After block surface hybridisation biotin conjugate SARS strand end take place Interaction alkaline phosphatase label streptavidin permit amplify indirect electrochemical detection The analytical signal constitute electrochemical process indigo carmine soluble product enzymatic hydrolysis indoxyl phosphate The use sensitive electrochemical technique square wave voltammetry allow detection limit 6pM obtain DNA sequence low find bibliography The parameter affect methodology study special attention place selectivity Specificity clearly enhance interaction time stringency form formamide percentage increase With strand interaction employ formamide hybridisation buffer base mismatch strand perfectly distinguish complementary',\n",
       "  0.5799664258956909),\n",
       " ('design principle single strand RNA genome replicate less double strand formation Single strand RNA ssRNA simple form genetic molecule constitute genome virus presumably primitive life form However innate unsolved problem regard ssRNA genome formation inactive double strand RNA dsRNA replication Here address problem focus secondary structure systematically design RNAs various structure observe dsRNA formation replication use RNA replicase replicase From result extract simple rule regard ssRNA genome replication less dsRNA formation less number loop design artificial RNA encodes domain galactosidase gene base rule also obtain evidence rule governs natural genome bacterial fungal virus presently know This study reveal one structural design principle ssRNA genome replicates continuously less dsRNA formation',\n",
       "  0.5795686841011047),\n",
       " ('ar001', 0.5794011950492859),\n",
       " ('bmcpv', 0.5790218114852905),\n",
       " ('transformations', 0.5789462327957153),\n",
       " ('sight', 0.5789201259613037),\n",
       " ('Development robust vitro RNA dependent RNA polymerase assay possible platform antiviral drug test dengue NS5 large conserve protein among four dengue virus DENV serotypes target interest antiviral drug development due major role replication NS5 consist two domain terminal methyltransferase domain terminal catalytic RNA dependent RNA polymerase RdRp domain unstable protein prone inactivation upon prolonged incubation room temperature thus affect inhibitor screen assay current study express purified DENV RdRp alone Esherichia coli coli cell The terminally His tagged construct DENV RdRp transform coli expression strain DE3 pLysS cell Protein expression induce isopropyl thiogalactopyranoside IPTG final concentration 4mM The induced culture grow 20h 18°C cell harvest centrifugation 6000xg 15min 4°C The recombinant protein purify use HisTrap affinity column NTA sample subject size exclusion chromatography successfully remove degradation product obtain previous purification step The vitro polymerase activity RdRp successfully demonstrate use homopolymeric polycytidylic acid poly RNA template This study describe high level production enzymatically active DENV RdRp protein use develop assay test large number compound high throughput manner RdRp novo initiation activity vitro polymerase assay protein provide platform highly robust efficient antiviral compound screen system',\n",
       "  0.5787850022315979),\n",
       " ('transcribe', 0.5787404775619507),\n",
       " ('quartz', 0.578729510307312),\n",
       " ('g–a', 0.5782930254936218),\n",
       " ('lambdaa', 0.5778573751449585),\n",
       " ('impossibility', 0.5774076581001282),\n",
       " ('hidden gene astroviruses encode cell permeabilizing protein involve virus release ABSTRACTHuman astroviruses small nonenveloped virus positive sense single strand RNA genome contain three main open reading frame ORF1a ORF1b ORF2 Astroviruses cause acute gastroenteritis child worldwide associate encephalitis meningitis immunocompromised individual Through comparative genomic analysis astrovirus sequence identify conserved ORFX overlap capsid encode ORF2 genogroup III astroviruses ORFX appear subject purify selection consistent encode functional protein product term Using ribosome profile cell infect human astrovirus confirm initiation ORFX AUG knockout astroviruses strongly attenuate passaging partly restore viral titer via pseudo reversion thus demonstrate play important role virus growth investigate develop astrovirus replicon system demonstrate minor effect RNA replication structural protein production Instead associate plasma membrane extracellular terminus topology promote efficient virus release Using two different assay show expression human relate astrovirus XPs lead cell permeabilization suggest viroporin like activity The discovery advance knowledge important human virus open new direction research astrovirus replication pathogenesis',\n",
       "  0.5771912932395935),\n",
       " ('ipft', 0.5771743059158325),\n",
       " ('delineating', 0.576635479927063),\n",
       " ('Functional Genetic Analysis Coronavirus Replicase Transcriptase Proteins The coronavirus replicase transcriptase complex assembly viral cellular protein mediate synthesis genome subgenome size mRNAs virus infect cell Here report genetic functional analysis temperature sensitive mutant Murine hepatitis virus MHV A59 unable synthesize viral RNA infection initiate maintain non permissive temperature Both classical biochemical complementation analysis lead predict majority MHV A59 ORF1a replicase gene product non structural protein nsp1–nsp11 form single complementation group cistron1 replicase gene product encode ORF1b non structural protein nsp12–nsp16 able function trans comprise least three possibly five complementation group cistron II–VI Also identify mutation non structural protein nsp nsp5 nsp10 nsp12 nsp14 nsp16 responsible phenotype eight MHV A59 mutant allow conclude protein essential assembly functional replicase transcriptase complex Finally analysis viral RNA synthesis mutant virus infect cell allow discriminate three phenotype regard inability specific mutant synthesize viral RNA non permissive temperature Mutant ts6 appear defective continue negative strand synthesis mutant Alb ts16 appear form negative strand utilized positive strand RNA synthesis mutant Alb ts22 defective elongation positive negative strand RNA basis result propose model describe pathway viral RNA synthesis MHV A59 infect cell Further biochemical analysis mutant allow identify intermediate pathway elucidate precise function viral replicase protein involve',\n",
       "  0.5766147971153259),\n",
       " ('self‐collected', 0.5756831765174866),\n",
       " ('Site specific target binding control RNA cleavage efficiency Kaposi sarcoma associate herpesvirus endonuclease SOX During lytic replication Kaposi sarcoma associate herpesvirus KSHV gene expression landscape cell remodel evade immune response create environment favorable viral replication major driver gene expression change virally encode messenger RNA mRNA specific endonuclease term SOX SOX cleave majority cytoplasmic mRNAs specific internal site loosely defined degenerate sequence motif RNA sequence directs SOX target remain unknown address question use recombinant highly purified SOX endonuclease series biochemical assay reconstitute cleavage reaction vitro gain significant insight biochemical mechanism SOX target recognition endonucleolytic cleavage Using system determine cut site specificity preserve purified SOX validate target RNA thus require additional cellular cofactor Furthermore show SOX display robust sequence specific RNA binding residue proximal cleavage site must present particular structural context The strength SOX binding dictate cleavage efficiency provide explanation breadth target RNA susceptibility observe cell Significance StatementKaposi sarcoma associate herpesvirus KSHV oncogenic human virus cause Kaposi sarcoma primary effusion lymphoma multicentric Castleman disease During viral replication KSHV express enzyme call SOX cut inactivate majority cellular messenger RNAs prevent translation proteins Some mRNAs efficiently cleave SOX others poorly cleave mechanistic basis underlie selectivity remain largely unknown Here reveal efficiency RNA cleavage heavily impact RNA sequence proximal cleavage site serve SOX bind platform This help explain range RNA cleavage efficiency observe SOX express cell well sequence specificity underlie SOX target',\n",
       "  0.5756539106369019),\n",
       " ('Sequence analysis nucleocapsid protein gene human coronavirus 229E Human coronaviruses important human pathogen also implicate multiple sclerosis understand molecular biology human coronavirus 229E HCV 229E molecular clone sequence analysis viral RNA initiate Following establish protocol terminal nucleotides genome sequence large open reading frame encode amino acid protein presumably nucleocapsid protein The predicted protein similar size chemical property amino acid sequence nucleocapsid proteins coronaviruses This especially evident sequence compare antigenically related porcine transmissible gastroenteritis virus TGEV region amino acid sequence homology find Hydropathy profile reveal existence several conserve domain could functional significance intergenic consensus sequence precede end propose nucleocapsid protein gene The consensus sequence present coronaviruses propose site bind leader sequence mRNA transcriptional start This region also examine primer extension analysis mRNAs identify nucleotide leader sequence The noncoding region genome contain nucleotide sequence relatively conserve throughout Coronavirus family lend support theory region important replication negative strand RNA',\n",
       "  0.5756426453590393),\n",
       " ('719g', 0.5756009817123413),\n",
       " ('oiv', 0.5752530097961426),\n",
       " ('act RNA sequence control gag pol translation readthrough murine leukemia virus The pol gene Moloney murine leukemia virus MuLV express Gag Pol fusion protein frame suppression UAG termination codon locate two gene The role nucleotide context suppression investigate rabbit reticulocyte lysate translation system use site direct mutagenesis The result indicate translational readthrough mediate least base long RNA sequence locate gag UAG termination codon Within sequence short purine rich sequence adjacent amber codon highly conserve among different retrovirus appear essential MuLV suppression Two alternative putative stem loop like RNA structure draw gag pol junction one abutting gag UAG codon second downstream None structure appear important suppression process',\n",
       "  0.5749461054801941),\n",
       " ('cxj', 0.5748820900917053),\n",
       " ('chan', 0.5748443603515625),\n",
       " ('iiels', 0.5748380422592163),\n",
       " ('orf1', 0.5745756030082703),\n",
       " ('sulfhydryl', 0.5744571089744568),\n",
       " ('superstructure', 0.5740557909011841),\n",
       " ('Rice dwarf phytoreovirus segment S12 transcript tricistronic Vitro Sequence analysis reveal rice dwarf phytoreovirus segment S12 nucleotide long small phase overlap open reading frame ORF well major ORF The large ORF position encodes amino acid small one base encodes amino acid additional frame AUG codon position 337–339 nucleotide downstream first one Transcripts full length cDNA direct vitro synthesis three polypeptide consider translate long ORF kDa Alteration two ATG codon small ORF demonstrate involvement generation kDa polypeptide Although still unknown whether protein express vivo small ORF show conserve S9s two member genus Phytoreovirus rice gall dwarf virus wound tumor virus suggest common important function',\n",
       "  0.5739597082138062),\n",
       " ('Identification formation mechanism novel noncoding RNA produce avian infectious bronchitis virus Viral noncoding RNAs show play important role viral life cycle Many virus employ different mechanism produce ncRNAs Here report coronavirus infectious bronchitis virus IBV produce novel ncRNA virus infect cell This ncRNA consist nucleotide exclude poly tail mainly derive untranslated region IBV genome contain long terminal leader sequence derive end viral genome Using mutagenesis reverse genetics reveal ncRNA subgenomic RNA generate discontinuous transcription mechanism',\n",
       "  0.5734549164772034),\n",
       " ('artds', 0.5729238986968994),\n",
       " ('spytag', 0.5728269815444946),\n",
       " ('strand', 0.5724995136260986),\n",
       " ('Sequence analysis end feline coronavirus FIPV genome Comparison genome porcine coronavirus TGEV reveal large insertion The genetic information carry mRNA feline infectious peritonitis virus FIPV strain 79–1146 determine sequence analysis cDNA clone derive end FIPV genome Two ORFs find encode polypeptide ORF 22K ORF The FIPV sequence compare end sequence transmissible gastroenteritis virus TGEV ORF homologous counterpart ORF TGEV genome ORFs locate position relative nucleocapsid gene However result frame insertion deletion ORF nucleotide large ORF similar event occur immediately downstream ORF1 nucleotide segment contain complete ORF absent TGEV sequence Most sequence similarity find noncoding sequence ORF ORF precede sequence AAC TAAAC assume transcription initiation signal FIPV TGEV Kapke Brian Virology 41–49 nuclease analysis end FIPV RNA map immediately upstream sequence nucleotide TGEV specific RNA find cross hybridization FIPV end probe suggest TGEV ORF also carry separate mRNA The difference end FIPV TGEV genomes maybe result RNA recombination event',\n",
       "  0.5722939372062683),\n",
       " ('Exploring IRES Region Accessibility Interference Foot Mouth Disease Virus Infectivity Translation initiation picornavirus RNA drive internal ribosome entry site IRES element locate upstream initiator codon RNA structure organization well RNA protein interaction play fundamental role internal initiation IRES activity mainly analyze context reporter gene lack region viral genome potentially affect translation efficiency With aim understand vulnerability IRES translation start region small molecule context viral genome design set customized RNase resistant 2′O methyl antisense oligoribonucleotides 2′OMe AONs base RNA structure data These AONs use monitor capacity interfere viral RNA translation thus inhibit virus yield Foot mouth disease virus FMDV RNA translation initiate two frame AUG codons show 2′OMe AON complementary AUG2 inhibit viral multiplication efficiently one target AUG1 Furthermore response viral RNA AONs target IRES region denote important difference tissue culture cell cell free system reinforce need analyze viral RNA response live cell Importantly identify four specific motif within IRES element target viral inhibitor tissue culture cell cell free system The identified target define accessible region small molecule disturb either RNA structural organization RNA protein interaction need initiate translation FMDV RNA',\n",
       "  0.5721309185028076),\n",
       " ('cuarto', 0.5721200108528137),\n",
       " ('tmpv3', 0.571640133857727),\n",
       " ('Molecular characterization proteins two enterotropic murine coronavirus strain Enterotropic strain murine coronaviruses MHV MHV differ extensively pathogenesis prototypic respiratory strain murine coronaviruses effort determine viral protein might determinant enterotropism immunoblots MHV MHV virion use anti protein specific antiserum perform The uncleaved MHV MHV proteins migrate slightly faster MHV A59 protein The MHV protein inefficiently cleave The MHV MHV MHV A59 proteins show minor difference migration The gene MHV MHV clone sequence found encode amino acid long proteins respectively The presence several amino acid change upstream predict cleavage site MHV protein may contribute inefficient cleavage high degree homology find MHV MHV protein whereas homology MHV protein proteins MHV strain much low These result indicate enterotropism MHV MHV may determine different amino acid change protein change viral protein',\n",
       "  0.5715757608413696),\n",
       " ('scheller', 0.571332573890686),\n",
       " ('70g', 0.5713092088699341),\n",
       " ('cdna', 0.5709322690963745),\n",
       " ('84cu', 0.5708988904953003),\n",
       " ('Molecular characterization Lelystad virus Lelystad virus prototype porcine reproductive respiratory syndrome virus small envelop virus contain positive strand RNA genome tentatively classify family Arteriviridae consist lactate dehydrogenase elevate virus LDV equine arteritis virus EAV simian hemorrhagic fever virus SHFV These virus similar genome organization replication strategy coronaviruses size genome much small 12–15 different morphological physicochemical property The genome contain eight open read frame ORFs encode replicase gene ORFs envelope protein ORFs nucleocapsid protein ORF7 Genomic comparison European North American isolate show structural protein encode ORFs vary widely The amino acid sequence ORFs North American strain share identical amino acid European strain Using polyvalent porcine anti serum gene specific anti peptide sera monoclonal antibody identify six structural protein correspond gene These kDa unglycosylated nucleocapsid protein encode ORF7 kDa unglycosylated integral membrane protein encode ORF6 kDa glycosylated protein encode ORF5 31–35 kDa glycosylated protein encode ORF4 45–50 kDa glycosylated protein encode ORF3 29–30 kDa glycosylated protein encode ORF2 nomenclature structural protein propose',\n",
       "  0.5707393884658813),\n",
       " ('mc57', 0.570513904094696),\n",
       " ('Regulation hepatitis virus replication core protein interaction viral RNA polymerase The hepatitis virus HCV core protein structural component nucleocapsid show modulate cellular signal pathway interaction various cellular protein present study investigate role HCV core protein viral RNA replication Immunoprecipitation experiment demonstrate core protein bind amino terminal region RNA dependent RNA polymerase RdRp encompass finger palm domain Direct interaction HCV RdRp core protein lead inhibition RdRp RNA synthesis activity vitro Furthermore expression core protein derivative lack RdRp interact domain suppress HCV replication hepatoma cell line harbor HCV subgenomic replicon RNA Collectively result suggest core protein bind RdRp inhibit RNA synthesis activity viral regulator HCV RNA replication',\n",
       "  0.5703311562538147),\n",
       " ('nvdc', 0.5702957510948181),\n",
       " ('taco™', 0.5700699687004089),\n",
       " ('rbm', 0.5699746608734131),\n",
       " ('The architecture SARS CoV transcriptome SARS CoV betacoronavirus responsible COVID pandemic The genome SARS CoV report recently transcriptomic architecture unknown Utilizing two complementary sequence technique present high resolution map SARS CoV transcriptome epitranscriptome DNA nanoball sequencing show transcriptome highly complex owe numerous recombination event canonical noncanonical addition genomic RNA subgenomic RNAs common coronaviruses SARS CoV produce large number transcript encode unknown ORFs fusion deletion frameshift Using nanopore direct RNA sequence find least RNA modification sit viral transcript frequent motif AAGAA Modified RNAs shorter poly tail unmodified RNAs suggest link internal modification tail Functional investigation unknown ORFs RNA modification discover study open new direction understanding life cycle pathogenicity SARS CoV HighlightsWe provide high resolution map SARS CoV transcriptome epitranscriptome use nanopore direct RNA sequence DNA nanoball sequence The transcriptome highly complex owe numerous recombination event canonical noncanonical addition genomic subgenomic RNAs common coronaviruses SARS CoV produce transcript encode unknown ORFs discover least potential RNA modification sit AAGAA motif',\n",
       "  0.5699077844619751),\n",
       " ('formulae', 0.5695309042930603),\n",
       " ('The amino carboxyl domains infectious bronchitis virus nucleocapsid protein interact genomic RNA Previous study indicate nucleocapsid protein infectious bronchitis virus IBV interact specific sequence non cod region IBV RNA order identify domains protein bind RNA whole protein amino acid six overlap fragment express fusion polypeptide six histidine tag Using gel shift assay intact protein amino polypeptide residues residue carboxyl polypeptide extend residue residue find interact positive strand IBV RNA represent end genome The two 32P label probe interact amino carboxyl fragment RNA consist IBV gene adjacent non cod terminus RNA consist nucleotide sequence end untranslated region contrast polypeptide fragment middle region residue 101–283 interact IBV RNAs The binding site amino region either present partially present first residue interaction RNA observe polypeptide incorporate residue Cache Valley virus express histidine tag bovine serum albumin basic lysozyme protein shift IBV RNA The low molarity carboxyl fragment compare residue 1–274 fragment need equivalent shift suggest bind avidity RNA carboxyl domain great amino domain',\n",
       "  0.569324791431427),\n",
       " ('unsure', 0.5690063238143921),\n",
       " ('kuri', 0.5690010786056519),\n",
       " ('Translation Elongation Factor Facilitates Assembly Tombusvirus Replicase Stimulates Minus Strand Synthesis Replication plus strand RNA virus depend host factor recruit viral replicase complex Previous study show eukaryotic translation elongation factor eEF1A one resident host proteins highly purified tombusvirus replicase complex Using random library eEF1A mutant identify one mutant decrease three mutant increase Tomato bushy stunt virus TBSV replication yeast model host Additional vitro assay whole cell extract prepare yeast strain express eEF1A mutant demonstrate several function eEF1A TBSV replication facilitate recruitment viral RNA template replicase complex assembly viral replicase complex enhancement minus strand synthesis promote initiation step These role eEF1A separate canonical role host viral protein translation emphasize critical function abundant cellular protein TBSV replication',\n",
       "  0.5689671039581299),\n",
       " ('The base asparaginyl tRNA dispensable efficient ribosomal frameshifting eukaryotes Edited Karn The frameshift signal avian coronavirus infectious bronchitis virus IBV contain two act signal essential efficient frameshifting heptameric slippery sequence UUUAAAC RNA pseudoknot structure locate downstream The frameshift take place slippery sequence two ribosome bound tRNAs slip back simultaneously one nucleotide zero phase UUA AAC phase UUU AAA Asparaginyl tRNA decode site codon AAC modify base wobble position anticodon QUU speculate may require frameshifting test measure frameshifting cell passaged growth medium contain calf serum horse serum Growth horse serum contain free queuine eliminate cellular tRNA population upon repeat passage Over ten cell passage however find significant difference frameshift efficiency cell type argue role frameshifting confirm cell cultured horse serum devoid purify tRNAs assess content tRNA transglycosylase assay couple HPLC mass spectroscopy Supplementation growth medium cell grow either horse serum calf serum free queuine effect frameshifting either These finding recapitulate vitro system use rabbit reticulocyte lysates largely deplete endogenous tRNAs resupplemented free contain tRNA population Thus base require frameshifting IBV signal explanation require account slipperiness eukaryotic asparaginyl tRNA',\n",
       "  0.5687918663024902),\n",
       " ('Another triple span envelope protein among intracellularly bud RNA viruses The torovirus protein The nucleotide sequence Berne virus envelope protein gene determine translation product identify vitro transcription translation Computer analysis protein sequence reveal characteristic class III membrane protein lack cleave signal sequence contain three successive transmembrane helices terminal half much coronavirus membrane protein The disposition protein membrane study vitro translation presence microsome subsequent proteinase digestion Only small portion either end polypeptide find expose opposite side vesicle membrane Experiments hybrid protein contain terminal tail coronavirus protein anti peptide serum available show terminus present cytoplasmic side membrane another similarity coronavirus protein Immunofluorescence experiment indicate protein express recombinant vaccinia virus accumulate intracellular membrane predominantly endoplasmic reticulum The common feature torovirus coronavirus protein support hypothesis evolutionary relationship exist group intracellularly bud virus',\n",
       "  0.5687639713287354),\n",
       " ('Heterologous recombination segment dsRNA genome bacteriophage The genome bacteriophage compose three unique segment double strand RNA package within procapsid One segment recombine another region share little sequence similarity Although recombination therefore heterologous crossover point usually consist two six identical nucleotide The frequency recombinant enhance condition prevent hinder minus strand synthesis single plus strand segment Recombination serve repair system well mean change genetic structure virus The reaction study vitro packaging replication system involve purified procapsids ssRNA Although striking difference mechanisms recombination RNA viruses also strong similarity All seem use copy choice template switch action recombination The system useful model recombination segment double strand RNA virus Reoviridae',\n",
       "  0.5686821341514587),\n",
       " ('The RNA pseudoknots foot mouth disease virus dispensable genome replication essential production infectious virus The positive strand RNA genome picornaviruses comprise single large open reading frame flank untranslated region UTRs Foot mouth disease virus FMDV unusually large UTR contain five structural domain These include internal ribosome entry site IRES facilitate initiation translation act replication element cre Less well characterise structure terminal nucleotide stem loop variable length poly tract approximately nucleotide series two four tandemly repeat pseudoknots PKs investigate structure PKs selective hydroxyl acetylation analyse primer extension SHAPE analysis determine contribution genome replication mutation deletion experiment SHAPE mutation experiment confirm importance previously predict structure function Deletion experiment show although PKs essential replication provide genome competitive advantage However although replicons full length genome lack PKs replication competent infectious virus rescue genome contain less one copy This consistent early report describe presence putative packaging signal region',\n",
       "  0.5685104131698608),\n",
       " ('phenoxy', 0.5684489011764526),\n",
       " ('Cellular Proteins Associated Interior Exterior Vesicular Stomatitis Virus Virions Virus particles virion often contain virus encode also host encode proteins Some host protein enclose within virion structure others case envelop virus embed host derive membrane While many host protein incorporation likely accidental may play role virus infectivity replication immunoreactivity next host Host protein incorporation may especially important therapeutic application large number virus particle administer Vesicular stomatitis virus VSV prototypic rhabdovirus candidate vaccine gene therapy oncolytic vector Using mass spectrometry previously examine cell type dependent host protein content VSV virion use intact whole virion purify three cell line originate different specie Here aim determine localization host protein within VSV virion analyze whole VSV virion whole VSV virion treat Proteinase remove protein outside viral envelope total protein identify identified whole virion identified Proteinase treat virion Most protein previously show associate VSV Functional enrichment analysis indicate overrepresented category protein associated vesicle vesicle mediate transport protein localization Using western blot presence several host protein include previously show association VSV Yes1 Prl1 Ddx3y confirm relative quantity various virion fraction determine Our study provide valuable inventory virion associate host protein investigation role replication cycle pathogenesis immunoreactivity VSV',\n",
       "  0.568336546421051),\n",
       " ('gaultheria', 0.5682120323181152),\n",
       " ('gorgonian', 0.5679317712783813),\n",
       " ('Detection avian infectious bronchitis viral infection use situ hybridization recombinant DNA recombinant DNA probe specificity end genomic RNA Ark strain infectious bronchitis virus IBV find hybridize extract RNA three strain Ark serotype well Mass41 Holl52 Gray JMK Conn Fla SE17 strain IBV Viral infection detect cytoplasm chicken embryo kidney cell inoculate Mass41 Ark99 SE17 two recent field isolate IBV use situ cytohybridization biotinylated probe vivo infection detect individual cell tracheas lung day inoculation chick Mass41 Ark99 situ hybridization Ark99 infect tissue section use 32P dATP label probe indicated viral replication present trachea day either day whereas viral RNA find lung day day inoculation',\n",
       "  0.5679298639297485),\n",
       " ('Common RNA replication signal exist among group coronaviruses evidence vivo recombination animal human coronavius molecule UTR sequence coronavirus genome know carry act element RNA replication presumably also virus genome replication UTR adjacent coding sequence also think harbor act element Here determine UTR adjacent sequence UTR sequence six group coronaviruses compare three previously report group member Extensive region highly similar UTR sequence find small region divergence also find indicate group coronaviruses could subdivide bovine coronavirus BCoV like BCoV human respiratory coronavirus OC43 human enteric coronavirus porcine hemagglutinate encephalomyelitis virus equine coronavirus murine hepatitis virus MHV like A59 JHM strain MHV puffinosis virus rat sialodacryoadenitis virus The UTRs BCoV MHV previously show interchangeable Here reporter contain BCoV RNA show replicate five BCoV like helper virus MHV human cell adapt MHV strain representative MHV like subgroup demonstrate group common replication signal element BCoV RNA furthermore acquire leader HCoV OC43 leader switch demonstrate first time vivo recombination animal human coronavirus molecule These result indicate common replication signal element exist among group coronaviruses despite two cluster pattern within group imply could exist high potential recombination among group member',\n",
       "  0.5676862001419067),\n",
       " ('imunostimulant', 0.5676038861274719),\n",
       " ('acetone', 0.5675206780433655),\n",
       " ('rna2', 0.5674629211425781),\n",
       " ('c457a', 0.5672776699066162),\n",
       " ('Functional Analysis Genomic Sequence Bovine Norovirus BACKGROUND Jena Virus bovine Norovirus cause enteric disease cattle represent potential model study enteric norovirus infection pathogenesis The positive sense RNA genome organise ORF1 non structural protein ORF2 major capsid protein ORF3 minor capsid protein The lack cell culture system study replication mean work date rely upon vitro system study non structural protein synthesis processing PRINCIPAL FINDINGS Only two three major ORF1 protein identify p110 follow vitro translation RNA term protein detect The term encode genomic sequence 5′GS test IRES like function cistronic system display evidence IRES like activity The site translation initiation determine predict nucleotide Following insertion epitope within 5′GS term protein identify vitro within RNA transfected cell CONCLUSIONS The vitro transcription translation system currently best system analyse protein synthesis processing Unlike similarly study human noroviruses initially appear express terminal protein present possibility encode RNA sequence regulatory function likely involved translation initiation IRES like manner This case follow determination site translation initiation term protein detect use epitope tag vitro vivo Although slightly large predict term protein detect processed form vivo thus demonstrate initial translation ORF1 polyprotein also activity viral protease These finding indicate block noroviral replication culture cell lie elsewhere',\n",
       "  0.5672421455383301),\n",
       " ('universal BMV base RNA recombination system—how search general rule RNA recombination present doubt RNA recombination one major factor responsible generation new RNA viruses retroviruses Numerous experimental system create investigate complex phenomenon Consequently specific RNA structural motif mediate recombination identify several virus Unfortunately till unified model genetic RNA recombination formulate mainly due difficulty direct comparison data obtain different RNA base virus solve problem attempt construct universal system recombination activity various RNA sequence could test end use brome mosaic virus model RNA virus plant structural requirement RNA recombination well define The effectiveness new homomolecular system proven experiment involve two RNA sequence derive hepatitis virus genome addition comparison data obtain homomolecular system generate earlier use heteromolecular one provide new evidence mechanism homologous non homologous recombination different depend virus mode replication',\n",
       "  0.5672042369842529),\n",
       " ('rpas', 0.566802978515625),\n",
       " ('miscarriage', 0.5667670965194702),\n",
       " ('Monoclonal antibody murine hepatitis virus strain JHM define viral glycoprotein responsible attachment cell cell fusion Hybridoma cell line produce monoclonal antibody JHM strain mouse hepatitis virus MHV establish indirect immunofluorescence immune precipitation monoclonal antibody three viral polypeptide specificity characterize Monoclonal antibody nucleocapsid react cytoplasm infected cell precipitate 000d nucleocapsid polypeptide MHV Other monoclonal antibody react cytoplasm surface infect cell find precipitate 000d viral glycoprotein third set monoclonal antibody react cytoplasm surface infect cell precipitate 000d viral glycoprotein precursor 000d AntiGP alone direct neutralizing activity MHV virus presence complement anti anti neutralize virus Only anti ability inhibit spread infection due fusion L241 cell Thus viral glycoprotein likely contains attachment fusion activity MHV',\n",
       "  0.5665346384048462),\n",
       " ('fbovs', 0.5664920210838318),\n",
       " ('ceacam1a', 0.5663862228393555),\n",
       " ('Characterization protein encode ORFs Lelystad virus The genome Lelystad virus positive strand RNA virus length contain open read frame ORFs encode putative viral protein ORFs cloned plasmid downstream Sp6 RNA polymerase promoter translation transcript generate vitro yield protein could immunoprecipitated porcine anti serum Synthetic polypeptides amino acid select amino acid sequence ORFs antipeptide serum raise rabbit Antisera immunoprecipitated vitro translation product ORFs obtain Sera contain antibody direct peptide ORFs react positively infect alveolar lung macrophage immunoperoxidase monolayer assay Using antipeptide serum porcine anti serum identify three structural protein assign correspond gene Virions find contain nucleocapsid protein kDa unglycosylated membrane protein kDa glycosylated membrane protein kDa The protein encode ORF7 protein encode ORF6 protein encode ORF5 The protein virus particle contain one two glycans resistant endo acetyl glucosaminidase This finding indicate high mannose glycans process complex glycans Golgi compartment The protein composition virion confirm evolutionarily relate equine arteritis virus simian hemorrhagic fever virus lactate dehydrogenase elevate virus',\n",
       "  0.5659354329109192),\n",
       " ('Bovine torovirus Sequencing structural gene expression nucleocapsid protein Breda virus Breda virus BRV member genus torovirus establish etiological agent diarrhea cattle find two separate serotypes BRV BRV study fragment BRV genome bracket gene structural protein BRV amplify long PCR amplicon purify sequence directly Sequence analysis reveal presence four open read frame ORF correspond peplomer envelope nucleocapsid gene ORF novel gene locate gene This new gene identical nucleotide sequence hemagglutinin esterase gene BRV With exception new ORF BRV manifest nucleotide sequence identity torovirus prototype Berne virus BEV region end genome contains gene structural proteins base segment contain ORF BRV gene amplify PCR clone Escherichia coli expression system The result protein purify SDS PAGE use immunize guinea pig Hyperimmune serum reactive bovine torovirus BTV human torovirus HTV antigen immunoelectron microscopy show aggregate break intact torovirus particle BTV positive fecal specimen immunoblot hyperimmune serum react specifically proteins BTV HTV well express protein BRV BRV immune serum gnotobiotic calf human convalescent serum HTV infect patient react expressed protein immunoblot These finding apply design dot blot assay could specifically detect BTV HTV fecal specimen',\n",
       "  0.5657108426094055),\n",
       " ('0‐5', 0.5656555891036987),\n",
       " ('polypoides', 0.5654314160346985),\n",
       " ('Intracellular processing terminal ORF protein coronavirus MHV A59 require multiple proteolytic event Several polypeptide product MHV A59 ORF characterize MHV A59 infect DBT cell use antiserum direct fusionn roteins encode first ORF1a These include previously identify terminal ORF product p28 well kDa polypeptide P28 always detect discrete band without large precursor suggest rapid cleavage p28 immediately synthesis Once p28 cleave little degradation protein period The intracellular cleavage p28 inhibit protease inhibitor leupeptin contrast result obtain vitro translation genome RNA Denison Perlman These data suggest different protease activity may responsible cleavage p28 vitro vivo The kDa protein intermediate cleavage product derive precursor great kDa The kDa product subsequently cleave secondary product kDa The intracellular cleavage kDa polypeptide inhibit leupeptin concentration inhibit early cleavage p28 cleavage kDa product large polyprotein precursor presence zinc chloride product kDa detect appear incorporate p28 amino terminus This suggest least two protease activity may necessary processing ORF1a proteins one cleave p28 sensitive zinc chloride resistant leupeptin cleave kDa precursor sensitive inhibitor Both kDa protein contain sequence predict encode two papain like protease activity',\n",
       "  0.5652544498443604),\n",
       " ('novel superfamily nucleoside triphosphate binding motif contain protein probably involve duplex unwinding DNA RNA replication recombination statistically significant similarity demonstrate amino acid sequence Escherichia coli helicases helicase subunit family non structural protein eukaryotic positive strand RNA viruses herpesvirus protein contain NTP bind sequence motif Based sequence analysis secondary structure prediction generalize structural model ATP bind core propose suggest protein constitute superfamily helicases helicase subunit involve NTP dependent duplex unwind DNA RNA replication recombination',\n",
       "  0.5650633573532104),\n",
       " ('Rearrangement viral sequence cytopathogenic pestiviruses Two cytopathogenic isolates bovine viral diarrhea virus cpBVDV analyze For virus two region genomic RNAs find duplicate rearrange The viral genome contain small duplicated element derive genomic end far downstream original context This sequence follow large duplication encompass region cod protein p80 molecular marker cpBVDV The element cod viral protease p20 case virus analyze aminoterminus p80 generate autoproteolytic removal precede encode protease For one cpBVDV isolate specific fusion protein p28 could identify compose p20 part p10 another viral protein Molecular characterization respective noncytopathogenic counterpart reveal duplication rearrangement sequence well expression p28 p80 specific cytopathogenic virus',\n",
       "  0.5648561716079712),\n",
       " ('funiculus', 0.5648514032363892),\n",
       " ('anyone', 0.5643625259399414),\n",
       " ('whitefly', 0.5640311241149902),\n",
       " ('loracarbef', 0.5638141632080078),\n",
       " ('brassica', 0.5635617971420288),\n",
       " ('xce2−', 0.5630761981010437),\n",
       " ('19–69', 0.5630545616149902),\n",
       " ('integrally', 0.5628995895385742),\n",
       " ('Mutagenesis murine hepatitis virus nsp1 cod region identifies residues important protein process viral RNA synthesis viral replication Despite ongoing research investigate mechanism coronavirus replication function many viral nonstructural protein nsps remain unknown current study reverse genetic approach use define role kDa amino terminal product nsp1 gene polyprotein replication coronavirus murine hepatitis virus MHV cell culture determine whether nsp1 require MHV replication identify residues critical protein function mutant virus contain deletion point mutation within nsp1 cod region generate assayed defect viral replication viral protein expression protein localization RNA synthesis The result demonstrate carboxy terminal half nsp1 residue K124 L241 dispensable virus replication culture require efficient proteolytic cleavage nsp1 gene polyprotein optimal viral replication Furthermore whereas deletion nsp1 residue amino terminal K124 fail produce infectious virus point mutagenesis nsp1 amino terminus allow recovery several mutant alter replication RNA synthesis This study identify nsp1 residue important protein process viral RNA synthesis viral replication',\n",
       "  0.5628184080123901),\n",
       " ('124–847', 0.5626296401023865),\n",
       " ('Rapid Recovery Classical Swine Fever Virus Directly Cloned cDNA The reverse genetics classical swine fever virus CSFV currently base transfection vitro transcribe RNA viral genomic cDNA clone inefficient time consume This study aim develop improve method rapid recovery CSFV directly clone cDNA Full length genomic cDNA CSFV Shimen strain flank promoter hepatitis delta virus ribozyme terminator sequence clone low copy vector pOK12 produce pOKShimen RzTΦ Direct transfection pOKShimen RzTΦ cell derive cell line stably express bacteriophage RNA polymerase allow CSFV rescue rapidly efficiently least fast fold great viral titer compare vitro transcription base rescue system Furthermore progeny virus rescue cell indistinguishable vitro vivo parent virus virus rescue classical reverse genetics The reverse genetics base intracellular transcription efficient convenient cost effective The cell line use rescue CSFV directly clone cDNA also use intracellular transcription expression system study structure function viral gene',\n",
       "  0.562627911567688),\n",
       " ('dissa', 0.5626248717308044),\n",
       " ('blunt', 0.5625561475753784),\n",
       " ('sosui', 0.5623789429664612),\n",
       " ('Molecular Characterization Porcine Reproductive Respiratory Syndrome Virus Member Arterivirus Group Porcine reproductive respiratory syndrome virus PRRSV specific cDNA clone span terminal genomic RNA isolate sequence used probe identification PRRSV specific RNAs The PRRSV genome positive strand polyadenylated RNA infected cell coterminal nest set six major subgenomic mRNAs could demonstrate Within terminal PRRSV genome six overlap read frame ORFs identify likely express one subgenomic mRNAs Amino acid sequence comparison reveal terminal ORF ORF7 encode PRRSV nucleocapsid protein calculate molecular weight kDa display amino acid identity capsid protein lactate dehydrogenase elevate virus LDV equine arteritis virus EAV The product ORF6 second terminal ORF represent putative membrane protein exhibit amino acid identity correspond LDV EAV polypeptides respectively Similar EAV ORFs might encode glycosylated viral proteins The polypeptide deduce ORF ORF1b contain two conserved domain common EAV coronavirus polymerases Genome organization strategy gene expression sequence deduce protein show PRRSV belongs Arterivirus group virus',\n",
       "  0.5621675252914429),\n",
       " ('Detection enteroviruses use cDNA synthetic oligonucleotide probes This study compare detection enterovirus RNA cDNA probe prepare end genome coxsackíe A21 use synthetic oligonucleotides prepare short highly conserved sequence end non cod region picornavirus genome The cDNA probe detect enterovirus variable level sensitivity presumably depend degree genomic homology detect probe Generally probe coxsackievirus A21 detect enterovirus similar probe coxsackievirus Probes end genome virus sensitive end probe contrast synthetic oligonucleotides detect enterovirus efficiently suggest probe could useful universal probe detect enterovirus This paper discuss application probe diagnosis differentiation enterovirus',\n",
       "  0.5618414878845215),\n",
       " ('haemolymphatic', 0.5617673397064209),\n",
       " ('macrodomains', 0.5617106556892395),\n",
       " ('The clone sequence virion protein gene British isolate porcine respiratory coronavirus comparison transmissible gastroenteritis virus gene Previous analysis porcine respiratory coronavirus PRCV mRNA specie show mRNAs small corresponding transmissible gastroenteritis virus TGEV mRNA specie Page Gen Virol 579–587 Sequence analysis show mRNA small due presence new putative RNA leader bind site upstream PRCV ORF gene However observation explain deletion observe PRCV mRNA Polymerase chain reaction PCR use generate cDNA cod region putative polymerase gene poly tail PRCV comparison equivalent region TGEV The PRCV protein find consist amino acid similarity TGEV protein However PRCV gene contain nucleotide deletion correspond amino acid residues TGEV protein nucleotide downstream gene initiation codon The PRCV genome ORF gene poly tail sequence comparison TGEV order identify potential difference two virus Four ORFs identify show similarity TGEV ORF ORF gene deletion PRCV specific sequence identify',\n",
       "  0.5614916086196899),\n",
       " ('Suppression subtractive hybridization method identification new strain murine hepatitis virus xenografted SCID mouse During attempt clone retroviral determinant associate mouse model Langerhans cell histiocytosis LCH suppression subtractive hybridization SSH use identify unique virus liver severe combine immunodeficiency SCID mouse transplant LCH tissue partial genomic sequence murine coronavirus identify whole genome coronavirus subsequently sequence use PCR clone technique Nucleotide sequence comparison reveal genome sequence new virus identical know murine hepatitis viruses MHVs The predicted open reading frame nucleotide sequence encode known protein MHVs Analysis protein level show virus closely relate highly virulent MHV JHM strain The virus strain name MHV type retroviruses find Degenerate PCR target type retrovirus RACE target type retrovirus confirm absence retroviral association LCH xenografted SCID mouse',\n",
       "  0.5614145994186401),\n",
       " ('tollip', 0.5613724589347839),\n",
       " ('unassociated', 0.5610383749008179),\n",
       " ('arachidonate', 0.5609764456748962),\n",
       " ('hydrothermal', 0.5608166456222534),\n",
       " ('determinated', 0.5604656338691711),\n",
       " ('Viral enzyme Viral genome show unequalled diversity range single strand DNA double strand RNA Moreover viruses quickly adapt host immune response drug treatment Although tend make optimal use host cell reservoir proteins virus need carry enzymatic function may available accessible infect cell Recently progress make structural understanding viral enzyme involve stage viral life cycle include entry hijack replication exit stage',\n",
       "  0.560181736946106),\n",
       " ('mimicries', 0.5600007772445679),\n",
       " ('76–90', 0.5599629878997803),\n",
       " ('diagnostic oligonucleotide microarray simultaneous detection grapevine virus least virus report infect grapevine cause economic damage globally Conventional detection strategy base serological assay biological index PCR target one virus assay widely use Grapevines prone contain mixed infection several virus make use technique time consume mer oligonucleotide microarray able detect simultaneously broad spectrum know virus well new virus cross hybridization highly conserve probe report present study The array contain unique probe design highly conserved specie specific region plant viral genome addition probe design plant housekeep gene also include use random prim PCR amplification strategy grapevine double strand RNA enrich sample viral agent detect single mixed infection The microarray accuracy detect grapevine virus compare PCR yield consistent result For purpose grapevine sample contain single mixed infection Grapevine leafroll associate virus Grapevine fanleaf virus Grapevine rupestris stem pit associate virus Grapevine virus Grapevine virus use Genomic library contain complete viral genome also use part validation process The specific probe hybridization pattern obtain virus make approach powerful tool high throughput plant certification purpose also virus discovery new viral genomic sequence partial similarity microarray probe Three Closteroviridae member Grapevine leafroll associate virus detect first time Chilean grapevine use microarray',\n",
       "  0.5599220395088196),\n",
       " ('Antisense induce ribosomal frameshifting Programmed ribosomal frameshifting provide mechanism decode information locate two overlap read frame divert proportion translate ribosome second open reading frame ORF The result production two protein product standard translation ORF1 ORF1–ORF2 fusion protein Such programmed frameshifting commonly utilized gene expression mechanism virus infect eukaryotic cell subset cellular gene RNA secondary structure consist pseudoknots stem–loops locate downstream shift site often act cis stimulators frameshifting Here demonstrate first time antisense oligonucleotides functionally mimic RNA structure induce ribosomal frameshifting annealed downstream frameshift site UCC UGA Antisense induce shift ribosome reading frame highly efficient rabbit reticulocyte lysate translation reaction culture mammalian cell The efficiency antisense induce frameshifting site responsive sequence context shift site polyamine level',\n",
       "  0.5599166750907898),\n",
       " ('monoclonally', 0.5598289966583252),\n",
       " ('s437–459', 0.5597494840621948),\n",
       " ('filancemeter', 0.5594213008880615),\n",
       " ('eif4e', 0.5591831207275391),\n",
       " ('marshall', 0.5591747760772705),\n",
       " ('2016‐2018', 0.5589522123336792),\n",
       " ('The severe acute respiratory syndrome coronavirus novel structural protein The severe acute respiratory syndrome coronavirus SARS CoV protein one opening reading frames viral genome homologue know coronaviruses Expression protein demonstrate vitro vivo infection Here present biochemical data show novel coronavirus structural protein detected virion purify SARS CoV infect Vero cell although two truncated product present predominantly instead full length protein Vero cell transiently transfected cDNA construct express similar cleavage observe Furthermore expression membrane envelope proteins use baculovirus system show full length truncate assembled virus like particle This first report demonstrate incorporation virion show SARS CoV encode novel coronavirus structural protein',\n",
       "  0.5588721036911011),\n",
       " ('Identification Interactions Sindbis Virus Capsid Protein Cytoplasmic vRNA Novel Virulence Determinants Alphaviruses arthropod borne virus represent significant threat public health global level While formation alphaviral nucleocapsid core consist cargo nucleic acid viral capsid protein essential molecular process infection precise interaction two partner ill define CLIP seq approach use screen candidate site interaction viral Capsid protein genomic RNA Sindbis virus SINV model alphavirus The data present report indicate SINV capsid protein bind specific viral RNA sequence cytoplasm infect cell interaction genomic RNA mature extracellular viral particle largely non specific term nucleotide sequence Mutational analyse cytoplasmic viral RNA capsid interaction sit reveal functional role capsid bind early infection Interaction site mutant exhibit decreased viral growth kinetics however defect function decrease particle production Rather mutation cytoplasmic capsid RNA interaction sit negatively affected functional capacity incoming viral genomic RNAs lead decreased infectivity Furthermore cytoplasmic capsid interaction site mutant attenuate murine model neurotropic alphavirus infection Collectively finding study indicate identify cytoplasmic interaction viral capsid protein genomic RNA essential particle formation necessary genomic RNA function early infection This previously unappreciated role capsid protein alphaviral replication cycle also constitute novel virulence determinant',\n",
       "  0.558131217956543),\n",
       " ('syncerus', 0.5580718517303467),\n",
       " ('Next Generation Sequencing Reveals HIV Mediated Suppression Cell Activation RNA Processing Regulation Noncoding RNA Expression CD4 Cell Line Next generation sequence NGS enable highly sensitive measurement whole transcriptomes report first application knowledge technology analysis RNA CD4 cell line infect intact HIV sequence total mRNA infect cell detect difference expression host viral mRNA Viral read represent large portion total map sequence read approximately postinfection hpi hpi also detect small significant suppression cell activation relate gene hpi This suppression persist expand hpi provide new possible marker virus induced cell cytopathology hpi expression detectable host locus also alter indicate widespread alteration host process include RNA processing splice transport extent previously report addition next generation sequence provide insight alternative viral RNA splice event expression noncoding RNAs include microRNA host gene',\n",
       "  0.5574513673782349),\n",
       " ('backcalculation', 0.5574139356613159),\n",
       " ('snrnas', 0.5571204423904419),\n",
       " ('mycoplasmes', 0.5567806363105774),\n",
       " ('gc−', 0.5566868185997009),\n",
       " ('vpu', 0.5564558506011963),\n",
       " ('chimeric', 0.5563148856163025),\n",
       " ('immunoregulator', 0.5559899806976318),\n",
       " ('nucleotidylating', 0.5559273958206177),\n",
       " ('Enhancement vaccinia virus phage RNA polymerase expression system use encephalomyocarditis virus untranslated region sequence recombinant vaccinia virus produce bacteriophage RNA polymerase use express foreign gene eukaryotic cell Translation efficiency expression system enhance significantly employ encephalomyocarditis virus EMCV untranslated region UTR confers cap independent translation direct internal initiation translation The enhancement accomplish fuse open reading frame ORFs terminus EMCV polyprotein cod region thus utilize highly efficient translation initiation site Expression vector construct allow clone three read frame reporter gene use lacZ gene number gene encode coronavirus structural protein among others gene encode glycoprotein terminal signal sequence The signal sequence glycoprotein locate internally primary translation product demonstrate interfere translocation glycosylation yield biologically active proteins The usefulness sequence direct internal initiation extend use EMCV UTR express two three ORFs polycistronic mRNAs',\n",
       "  0.5555186867713928),\n",
       " ('otolaryngologic', 0.5554912090301514),\n",
       " ('epistasis', 0.5553431510925293),\n",
       " ('antagomir', 0.5553053617477417),\n",
       " ('gp124', 0.5552651286125183),\n",
       " ('randomly', 0.5551438331604004),\n",
       " ('Generation porcine reproductive respiratory syndrome virus vitro assembly viral genomic cDNA fragment Porcine reproductive respiratory syndrome virus PRRSV causative agent swine disease affect pig industry worldwide Infection PRRSV lead reproductive complication respiratory illness weak immunity secondary infection good control PRRSV infection novel approach generate control measure critically need Here vitro Gibson assembly viral genomic cDNA fragment test use quick simple method recover infectious PRRSV cell culture involve activity exonuclease Phusion polymerase Taq ligase join overlap cDNA fragment isothermal condition Four overlap cDNA fragment cover entire PRRSV genome one vector fragment use create plasmid capable express PRRSV genome The assembled product use transfect culture 293T MARC cell Supernatants transfected cell passaged onto MARC cell rescue infectious virus particle Verification characterization recover virus confirm protocol generate infectious PRRSV similar characteristic parental virus This approach test generation chimeric virus replace one four genomic fragment another virus strain chimeric virus successfully recover via conclusion study describe first time use simple yet powerful tool generate infectious PRRSV need study PRRSV biology develop novel vaccine',\n",
       "  0.554962158203125),\n",
       " ('qualifies', 0.5548852682113647),\n",
       " ('63–449', 0.5548704862594604),\n",
       " ('likeliness', 0.5547856688499451),\n",
       " ('The terminal heptad repeat region reovirus cell attachment protein responsible oligomer stability posse intrinsic oligomerization function The oligomerization domain reovirus cell attachment protein probe use type reovirus synthesize vitro Trypsin cleave protein 49K molecular weight approximately middle yield 26K terminal fragment 23K terminal fragment Under condition allow identification intact oligomeric form ∼200K sodium dodecyl sulfate polyacrylamide gel electrophoresis terminal 26K fragment find exist stable trimer 80K less extent dimer 54K whereas terminal fragment remain monomeric form polypeptide amino acid contain terminal heptad repeat region synthesize vitro capable form stable dimer trimer Using various criterion demonstrate stability intact oligomer confer mainly terminal heptad repeat region Our result summarize model individual heptad repeat hold together three strand helical coil coil structure via hydrophobic electrostatic interaction',\n",
       "  0.554703414440155),\n",
       " ('ifnγr', 0.5544492602348328),\n",
       " ('Sequestration IFIT1 Impairs Translation 2′O unmethylated Capped RNA Viruses generate cap RNA lack 2′O methylation first ribose severely affected antiviral activity Type interferons use proteome wide affinity purification couple mass spectrometry identify human mouse proteins specifically bind cap RNA different methylation state This analysis complement functional validation experiment reveal IFIT1 sole interferon induce protein display high affinity unmethylated methylated capped RNA IFIT1 tether specie specific protein complex consist IFITs RNA Pulsed stable isotope label amino acid cell culture couple mass spectrometry well vitro competition assay indicate IFIT1 sequesters 2′O unmethylated cap RNA thereby impair bind eukaryotic translation initiation factor 2′O unmethylated RNA template result inhibition translation The specificity IFIT1 2′O unmethylated RNA serve potent antiviral mechanism virus lack 2′O methyltransferase activity time allow unperturbed progression antiviral program infect cell',\n",
       "  0.5544108152389526),\n",
       " ('seronegativity', 0.5539184212684631),\n",
       " ('blank', 0.5538679361343384),\n",
       " ('Secondary structure mutational analysis ribosomal frameshift signal rous sarcoma virus Edited Draper Expression Gag Pol polyprotein Rous sarcoma virus RSV require ribosomal frameshifting event overlap region gag pol open reading frame The signal frameshifting compose two essential mRNA element slippery sequence AAAUUUA ribosome change read frame stimulatory RNA structure locate immediately downstream This RNA predict complex stem loop may also form RNA pseudoknot investigate structure RSV frameshift signal combination enzymatic chemical structure probe site direct mutagenesis The stimulatory RNA indeed complex stem loop long stable stem two additional stem loop contain substructure within main loop region The substructure however require frameshifting Evidence additional interaction stretch nucleotide main loop region downstream generate RNA pseudoknot obtain analysis frameshifting property RSV mutant translate rabbit reticulocyte lysate vitro translation system Mutations disrupt predict pseudoknot form sequence reduce frameshifting mutation combine form pseudoknot frameshifting restored level approach wild type construct also observe predict pseudoknot form region reduce sensitivity cleavage single strand probe imidazole Overall however structure probe data indicate pseudoknot interaction weak may form transiently comparison characterise RNA structure present viral frameshift signal RSV stimulator fall novel group can not consider simple hairpin loop yet distinct well characterise frameshift induce RNA pseudoknots overall contribution RSV pseudoknot frameshifting less dramatic',\n",
       "  0.5537735819816589),\n",
       " ('card®', 0.5536729097366333),\n",
       " ('immunostimulator', 0.5536184310913086),\n",
       " ('cdnas', 0.5535868406295776),\n",
       " ('subglottis', 0.5535790920257568),\n",
       " ('coliphage', 0.5535241365432739),\n",
       " ('Apoptosis induce cytopathic hepatitis virus dependent caspase activation follow ribosomal RNA degradation occur absence 2′–5′ oligoadenylate synthetase present previously evidence cytopathogenic 18f strain hepatitis virus HAV induce degradation ribosomal RNA rRNA infect cell Arch Virol 1275–1300 contrast non cytopathogenic parent virus HM175 clone effect rRNA integrity present data show rRNA degradation follow apoptosis accompany characteristic DNA ladder cytoplasm 18f infected cell The DNA laddering coincide detection caspase PARP cleavage dependent upon activation caspase pathway since treatment VAD FMK pan caspase inhibitor inhibit event RNase mRNA present virus infect uninfected cell Messenger RNA interferon inducible enzyme 2′–5′ oligoadenylate synthetase 2′–5′ OAS polymerizes ATP 2′–5′ oligo adenylate 2–5A activator RNase presence double strand RNA detect follow virus infection 2′–5′ OAS mRNA induce treatment cell interferon IFN IFN mRNA marginally induce follow infection However phosphorylated STAT key regulator interferon stimulate gene transcription detect virus infected cell STAT phosphorylation response IFN treatment low virus infect cell compare uninfected cell treat interferon suggest 18f virus infection interfere interferon signal The result suggest 18f infection cause induction 2–5A independent RNase like activity',\n",
       "  0.5534874200820923),\n",
       " ('recyclability', 0.5534630417823792),\n",
       " ('ordinary', 0.5534276962280273),\n",
       " ('2–31', 0.5526043772697449),\n",
       " ('Coronavirus nucleocapsid protein RNA chaperone RNA chaperone nonspecific nucleic acid bind protein long disorder region help RNA molecule adopt functional conformation Coronavirus nucleoproteins nonspecific RNA binding protein long disorder region Therefore investigate whether transmissible gastroenteritis coronavirus TGEV protein RNA chaperone Purified protein enhance hammerhead ribozyme self cleavage nucleic acid anneal property define RNA chaperone contrast another RNA binding protein PTB show activity protein chaperone activity block specific monoclonal antibody Therefore conclude TGEV protein RNA chaperone addition show purify severe acute respiratory syndrome SARS CoV protein also RNA chaperone activity silico prediction disorder domain show similar pattern coronavirus proteins evaluate Altogether data lead suggest coronavirus protein might RNA chaperone',\n",
       "  0.5525063276290894),\n",
       " ('Target Dependent Enrichment Virions Determines Reduction High Throughput Sequencing Virus Discovery Viral infection cause many different disease stem well characterize viral pathogen also emerge virus search novel virus continue great importance High throughput sequence important technology purpose However viral nucleic acid often constitute minute proportion total genetic material sample infected tissue Techniques enrich viral target high throughput sequence report sensitivity method well establish This study compare different library preparation technique target DNA RNA without virion enrichment optimize selection intact virus particle physical enzymatic approach assess effectiveness specific enrichment viral sequence compare non enriched sample preparation selectively look count read sequence obtain shotgun sequence Using shotgun sequence total DNA RNA viral target detect concentration correspond predict level provide foundation estimate effectiveness virion enrichment Virion enrichment typically produced fold increase proportion DNA virus sequence For RNA virion gain less pronounced maximum fold increase This enrichment varied different sample concentration clear trend Despite less sequencing require identify target sequence evident data low detection level achieve virion enrichment compare shotgun sequence',\n",
       "  0.5525034666061401),\n",
       " ('methacrylate', 0.5524803400039673),\n",
       " ('Non plaque form virion Modified Vaccinia virus Ankara express viral gene cell culture infection vaccinia virus number count virus particle substantially high number plaque obtain titration find standard vaccine preparation recombinant Modified Vaccinia virus Ankara produce 20–30 plaque form virion fully permissive cell culture evaluate biological activity non plaque form particle generate recombinant virus express fluorescent reporter proteins transcriptional control specific viral early late promoter Live cell imaging automate count fluorescent microscopy indicate virtually virus particle enter cell switch viral gene expression Although non plaque form infection arrest level viral early gene expression detect activation late viral transcription 10–20 single infected cell Thus non plaque form particle biologically active likely contribute immunogenicity vaccinia virus vaccine',\n",
       "  0.5524072647094727),\n",
       " ('back‐up', 0.5522703528404236),\n",
       " ('ddb1', 0.5522258281707764),\n",
       " ('Analysis multimerization SARS coronavirus nucleocapsid protein Severe Acute Respiratory Syndrome SARS emerge disease characterize atypical pneumonia recently attribute novel coronavirus The genome SARS Coronavirus SARS CoV recently sequence number gene identify include nucleocapsid protein note however protein SARS CoV SARS CoV share little homology nucleocapsid proteins member coronavirus family Science Science proteins coronavirus report involved forming viral core also package transcription viral RNA data generate viral system coronaviruses suggest viral N–N self interaction may necessary subsequent formation nucleocapsid assembly viral particle decide investigate SARS CoV N–N interaction use mammalian two hybrid system sucrose gradient fractionation homotypic interaction detect two hybrid analysis The mammalian two hybrid assay reveal approximately fold increase SEAP activity measurement protein–protein interaction N–N interaction compare observe either M–M mock transfection Furthermore mutational analysis characterize serine arginine rich motif SSRSSSRSRGNSR amino acid crucial protein oligomerization since deletion region completely abolish protein self multimerization Finally full length nucleocapsid protein express purify baculovirus system find form different level high order structure detect Western blot analysis fractionate protein Collectively result may aid elucidate mechanism pertain formation viral nucleocapsid core design molecular approach intervene SARS CoV replication',\n",
       "  0.5521417856216431),\n",
       " ('nonmuscle', 0.5521241426467896),\n",
       " ('1983–1989', 0.5520997643470764),\n",
       " ('greyhound', 0.5520178079605103),\n",
       " ('vascularis', 0.5519742369651794),\n",
       " ('Sequence analysis human coronavirus 229E mRNAs evidence polymorphism homology myelin basic protein Human coronaviruses HCV important pathogen responsible respiratory gastrointestinal possibly neurological disorder good understand molecular biology prototype HCV 229E strain nucleotide sequence unique region mRNAs determine clone cDNAs Sequence analysis cDNAs synthesize mRNA reveal major difference previously publish result However polymerase chain reaction amplification region show sequenced cDNAs produce minor RNA specie indication possible genetic polymorphism region viral genome The mutate messenger RNA contains two ORFs ORF4a consist nucleotide potentially encode amino acid polypeptide cod sequence precede consensus transcriptional initiation sequence CUAAACU similar one find upstream gene ORF4b nucleotide potentially encode amino acid basic leucine rich polypeptide hand mRNA contain one single ORF nucleotide could encode amino acid basic leucine rich polypeptide This putative protein present significant degree amino acid homology counterpart find transmissible gastroenteritis coronavirus TGEV The protein two different virus exhibit similar molecular weight extremely hydrophobic Interestingly sequence homology five amino acid find protein encode ORF4b HCV 229E immunologically important region human myelin basic protein',\n",
       "  0.5519595742225647),\n",
       " ('1×109', 0.551720380783081),\n",
       " ('cammarata', 0.5517147183418274),\n",
       " ('Protein direct ribosomal frameshifting temporally regulate gene expression Programmed ribosomal frameshifting mechanism gene expression whereby specific signal within messenger RNAs direct proportion translate ribosome shift continue translate new read frame Such frameshifting normally occur set ratio utilized expression many viral gene number cellular gene open question whether protein might function trans act switch turn frameshifting response cellular condition Here show frameshifting model RNA virus encephalomyocarditis virus trans activate viral protein result frameshifting efficiency increase one high know mammalian system course infection temporally regulate expression level viral structural enzymatic protein',\n",
       "  0.5516895055770874),\n",
       " ('namdinh', 0.5516279935836792),\n",
       " ('midi', 0.5515639781951904),\n",
       " ('Multifaceted Regulation Translational Readthrough RNA Replication Elements Tombusvirus Translational readthrough stop codon ribosomes recode event use variety virus include plus strand RNA tombusviruses Translation viral RNA dependent RNA polymerase RdRp tombusviruses mediate use strategy investigate process use variety vitro vivo approach Our result indicate readthrough generate RdRp require novel long range RNA RNA interaction span distance occur large RNA stem loop locate proximal stop codon RNA replication structure term RIV end viral genome Interestingly long distance RNA RNA interaction modulate mutually exclusive RNA structure RIV represent type RNA switch Moreover different long range RNA RNA interaction previously show necessary viral RNA replicase assembly also require efficient readthrough production RdRp Accordingly multiple replication associate RNA element involve modulate readthrough event tombusviruses propose integrate mechanistic model describe regulatory network could advantageous provide quality control system cull truncated viral genome early stage replication process mediate cis preferential replication viral genome iii coordinate translational readthrough RdRp viral genome replication Based comparative sequence analysis experimental data basic element regulatory model extend member Tombusviridae well viruses outside family',\n",
       "  0.5513479709625244),\n",
       " ('xenobiotics', 0.5513435006141663),\n",
       " ('immunoproteomics', 0.5513279438018799),\n",
       " ('Sequence analysis membrane protein gene human coronavirus 229E Human coronaviruses HCV ubiquitous pathogen cause respiratory gastrointestinal possibly neurological disorder good understand molecular biology prototype HCV 229E strain complete nucleotide sequence membrane protein gene determine clone cDNA The open reading frame precede consensus transcriptional initiation sequence UCUAAACU identical one find upstream gene The gene encode amino acid polypeptide molecular weight slightly high apparent 000–22 observe unprocessed protein obtain vitro translation immunoprecipitation The amino acid sequence present significant degree homology counterpart transmissible gastroenteritis coronavirus TGEV The protein HCV 229E highly hydrophobic hydropathicity profile show transmembranous region compose three major hydrophobic domains characteristic typical coronavirus protein About amino acid HCV 229E protein constitute hydrophilic probably external portion One glycosylation three potential glycosylation sit find expose domain',\n",
       "  0.5510022044181824),\n",
       " ('assam', 0.5509063005447388),\n",
       " ('phytopthora', 0.550879955291748),\n",
       " ('immunocytological', 0.5507679581642151),\n",
       " ('Oligonucleotide base microarray detection plant virus employ sequence independent amplification target The potential DNA microarrays detection plant virus hamper underutilization sequence independent amplification method target nucleic acid enrichment microarray system describe unbiased detection plant virus use short 30nt long 70nt oligonucleotide probe The assay involve amplification target nucleic acid use random primer follow vitro transcription whose cRNA product label chemically fragment use target hybridization Initial optimization test Turnip vein clear virus Cauliflower mosaic virus show increased hybridization efficiency short cDNA target 100bp long probe 70nt The system validated pure mixed sample detection three Tymovirus specie Asclepias asymptomatic virus Kennedya yellow mosaic virus Turnip yellow mosaic virus The method could detect sequence variant 70–75 high sequence identity indicate possible utility approach virus discovery Array performance comparison long probes demonstrate competence mers provide satisfactory balance detection sensitivity specificity The work describe significant step towards method assess one assay presence large diversity relative know virus plant',\n",
       "  0.5505067110061646),\n",
       " ('flavivriuses', 0.5504052639007568),\n",
       " ('realaccurate', 0.5503264665603638),\n",
       " ('Sequence analysis turkey coronavirus nucleocapsid protein gene untranslated region identifies virus close relative infectious bronchitis virus The end turkey coronavirus TCV genome base include nucleocapsid gene untranslated region UTR sequence compare publish sequence avian mammalian coronaviruses The deduced sequence TCV protein determine amino acid molecular mass approximately kDa The TCV protein identical size great amino acid identity publish protein sequence infectious bronchitis virus IBV less identity observe protein bovine coronavirus transmissible gastroenteritis virus The UTR show variation among three TCV strain examine two TCV strain Minnesota Indiana contain base segment present NC95 strain Nucleotide sequence identity UTRs TCV IBV great Similarities size sequence TCV IBV proteins UTRs provide additional evidence avian coronaviruses closely related',\n",
       "  0.5503004789352417),\n",
       " ('generalisation', 0.550075352191925),\n",
       " ('undecynoic', 0.5500732660293579),\n",
       " ('celĺs', 0.5500372648239136),\n",
       " ('Construction infectious cDNA clone Enterovirus Insights factor ensure experimental success Enterovirus causative agent mild Hand Foot Mouth Disease capable cause severe complication CNS young child Reverse genetics technology currently widely use study pathogenesis virus The aim work determine evaluate factor contribute infectivity RNA transcript vitro Two strategy overlap PCR long distance PCR employ obtain full length genome cDNA clone virus The length poly tail presence non viral terminal sequence study regard effect infectivity vitro RNA transcript cell culture The data reveal cDNA clone obtain long distance PCR infectious difference observe virus titre transfection vitro RNA harbour poly tail adenine length irrespective non viral sequence terminus',\n",
       "  0.5498793125152588),\n",
       " ('78kb', 0.5498397946357727),\n",
       " ('lsu', 0.5496538877487183),\n",
       " ('Leaderless Genome Identified Persistent Bovine Coronavirus Infection Associated Attenuation Gene Expression The establishment persistent viral infection often associate selection one mutant viruses For example find intraleader open reading frame ORF genomic subgenomic mRNA sgmRNA molecule select bovine coronavirus BCoV persistence lead translation attenuation downstream ORF Here report unexpected identification leaderless genome addition leader contain genome cell culture persistently infect BCoV The discovery make use head tail ligation method examines genomic terminal sequence different time postinfection Functional analysis leaderless genomic RNA BCoV defective interfering RNA reveal leaderless genome able serve template synthesis negative strand genome although can not perform replicative positive strand genomic RNA synthesis leaderless genome retain function translation transcription although efficiency process impair Therefore previously unidentified leaderless genome associate attenuation genome expression Whether leaderless genome contributes establishment persistent infection remains determine',\n",
       "  0.5496420860290527),\n",
       " ('mosaik', 0.5496161580085754),\n",
       " ('Role protease mouse hepatitis virus induce cell fusion Studies cold sensitive mutant isolate persistent infection plaque mutant isolate Kirsten mouse sarcoma virus transform BALB cell persistently infect mouse hepatitis virus strain MHV While wild type produce large plaque consist fused cell fusion type 32° mutant produce small fusion type plaques 39° 32° long incubation plaque consist round dead cell non fusion type obtain The mutant grow equally well temperatures Thus mutant cold sensitive inducing cell fusion replication ultimate cell kill The cold sensitivity overcome addition trypsin culture medium treat virion trypsin SDS polyacrylamide gel electrophoresis virion protein fail detect difference wild type mutant intracellular viral protein immunoprecipitated anti MHV rabbit serum protein dalton 68K protein present wild type infect cell absent mutant infect cell trypsin treatment incubation 39° mutant infect cell fail induce 68K protein After six seven undiluted passage mutant segregate variety mutant partially totally reverted wild type phenotype also whose cell fusion induction absent even 39° All mutant fail induce 68K protein infect cell Thus linkage presence 68K protein fusion induction Trypsin treatment infect cell enable MHV form fusion plaque otherwise resistant cell MHV producer non fusion type plaque form fusion type plaque Protease appear play important role MHV induce cell fusion general',\n",
       "  0.5495234727859497),\n",
       " ('qs21', 0.549439549446106),\n",
       " ('Rapid approach identify unrecognized viral agent For epidemic control rapid identification characterization responsible unknown agent crucial address critical question method develop virus discovery base flexible nested PCR subtraction hybridization positive control use hepatitis virus hypothetical unrecognized virus discover sample Using template switch universal long PCR produce large quantity cDNA nest PCR base subtractive hybridization couple single strand deletion technology remove common cDNA Following subtraction hybridization cDNA library construct displayed differential reverse dot blot hybridization This new genomic subtraction hybridization method ideally suit identify rapidly previously unrecognized viral agent',\n",
       "  0.5491650700569153),\n",
       " ('peroxisomes', 0.5490905046463013),\n",
       " ('Single Cell Many Different Transcripts Journey World Long Non Coding RNAs late evidence human genome transcribe small percentage transcript translate This observation support importance non cod RNAs confirm several organisms The abundant non translate transcript long non cod RNAs lncRNAs contrast protein cod RNAs show cell specific expression understand function lncRNAs fundamental investigate cell preferentially express detect subcellular localization Recent improvement technique localize single RNA molecule tissue like single cell RNA sequence fluorescence amplification method give considerable boost knowledge lncRNA function recent year single cell transcription variability associate non cod RNA expression reveal class RNAs important transcript cell lineage specification The purpose review collect update information lncRNA classification new finding function derive single cell analysis also retain useful researcher describe method available single cell analysis databases collect single cell lncRNA data Tables include schematize describe compare expose concept',\n",
       "  0.5490652918815613),\n",
       " ('sb1a', 0.5490394830703735),\n",
       " ('husband', 0.5488783717155457),\n",
       " ('reagents', 0.5487748980522156),\n",
       " ('Evolution end genomic rna murine coronaviruses passages vitro The end genomic RNA subgenomic mRNAs murine coronavirus MHV stretch approximately nucleotide leader sequence The region leader sequence contain several repeat pentanucleotide UCUAA whose number vary among different MHV strain demonstrate UCUAA repeat play crucial role discontinuous transcription MHV mRNAs present study demonstrate number UCUAA repeat leader sequence MHV genome rapidly decrease serial passage virus susceptible cell The downward evolution number UCUAA repeat due high growth rate virus few repeat seem due homologous interference virus different number UCUAA repeat The ease variant virus arise suggests high frequency occurrence deletion RNA replication This finding agreement propose discontinuous nonprocessive mode coronavirus RNA synthesis Analysis intracellular subgenomic mRNA specie viruses different number UCUAA repeat MHV recombinant virus suggest number pentanucleotide repeat end leader sequence may regulate synthesis certain mRNA specie agreement leader prim transcription mechanism',\n",
       "  0.5485864877700806),\n",
       " ('shfv', 0.5485121011734009),\n",
       " ('Proteomics Analysis Nucleolus Adenovirus infect Cells Adenoviruses replicate primarily host cell nucleus well establish adenovirus infection affect structure function host cell nucleoli addition cod number nucleolar target viral protein Here use unbiased proteomics method include high throughput mass spectrometry couple stable isotope label amino acid cell culture SILAC traditional two dimensional gel electrophoresis identify quantitative change protein composition nucleolus adenovirus infection Two dimensional gel analysis reveal change six protein contrast SILAC base approach identify protein protein show least fold change infection four previously report aberrant localization functional relevance adenovirus infection total protein identify change amount proteomics method examine infected cell use confocal microscopy Eleven proteins show altered pattern localization adenovirus infected cell Comparing data effect actinomycin nucleolar proteome reveal adenovirus infection apparently specifically target relatively small subset nucleolar antigen time point examine',\n",
       "  0.5484319925308228),\n",
       " ('Direct RNA nanopore sequence full length coronavirus genome provide novel insight structural variant enable modification analysis Sequence analyse RNA virus genome remain challenging due exceptional genetic plasticity virus Because high mutation recombination rate genome replication viral RNA dependent RNA polymerases lead population closely related virus call quasispecies Standard short read sequence technology ill suit reconstruct large number full length haplotype RNA virus genome subgenome length RNAs comprise noncontiguous genome region Here use full length direct RNA sequence DRS approach base nanopores characterize viral RNAs produce cell infect human coronavirus Using DRS able map long contiguous read viral reference genome combine Illumina nanopore sequence reconstruct highly accurate consensus sequence human coronavirus HCoV 229E genome Furthermore use long read require assembly step able identify infect cell diverse novel HCoV 229E RNAs remain characterized Also DRS approach circumvents reverse transcription amplification RNA allow detect methylation sit viral RNAs Our work pave way haplotype base analyse viral quasispecies demonstrate feasibility intra sample haplotype separation Even though several technical challenge remain addressed exploit potential nanopore technology fully work illustrates direct RNA sequencing may significantly advance genomic study complex virus population include prediction long range interaction individual full length viral RNA haplotype',\n",
       "  0.5484178066253662),\n",
       " ('principally', 0.5482101440429688),\n",
       " ('diethylaminoethyl', 0.548119068145752),\n",
       " ('Sequence analysis bovine coronavirus nucleocapsid matrix protein gene The end genome Mebus strain bovine enteric coronavirus BCV copy cDNA clone PstI site pUC9 vector Four clone end genome sequence either completely part determine sequence first base Within sequence identify order noncoding region base gene amino acid nucleocapsid protein molecular weight gene amino acid matrix protein molecular weight third large open reading frame contain entirely within gene sequence position different reading frame potentially encode polypeptide amino acid molecular weight high degree amino acid sequence homology find protein BCV MHV protein For proteins BCV MHV notable difference find amino terminus probable site glycosylation sequence Met Ser Ser Val Thr Thr BCV Met Ser Ser Thr Thr MHV BCV apparently use two six potential glycosylation site',\n",
       "  0.5481040477752686),\n",
       " ('Characterization Rotavirus RNAs That Activate Innate Immune Signaling RIG Like Receptors mammalian cell first line defense viral pathogen innate immune response characterize induction type interferons IFN pro inflammatory cytokine establish antiviral milieu infect cell neighbor uninfected cell Rotavirus double strand RNA virus Reoviridae family primary etiological agent severe diarrhea young child worldwide Previous study demonstrate rotavirus replication induces MAVS dependent type IFN response involve RIG MDA5 two cytoplasmic viral RNA sensor This study report isolation characterization rotavirus RNAs activate IFN signal Using vitro approach purify rotavirus double layer particle nascent single strand RNA ssRNA transcript term vitro ssRNA find potent IFN inducer addition large RNAs isolate rotavirus infected cell six hour post infection term vivo large RNAs also activate IFN signal whereas comparable large RNA fraction isolate cell infected one hour lack stimulatory activity Experiments use knockout murine embryonic fibroblast show RIG require MDA5 partly contribute innate immune signal vitro ssRNA vivo large RNAs Enzymatic study demonstrate vitro ssRNA vivo large RNA sample contain uncapped RNAs expose phosphate group RNAs lack methylated cap structure also detect vivo large RNA sample Taken together data provide strong evidence rotavirus VP3 enzyme encodes guanylyltransferase methyltransferase activity completely efficient either cap methylation cap structure viral transcript way produce RNA pattern activate innate immune signal RIG like receptor',\n",
       "  0.5480014681816101),\n",
       " ('complementary', 0.5478503704071045),\n",
       " ('r563a', 0.5477135181427002),\n",
       " ('toluenesulfonate', 0.5476009249687195),\n",
       " ('iodinated', 0.5472234487533569),\n",
       " ('bra', 0.5470734238624573),\n",
       " ('Complete nucleotide sequence polyprotein gene genome organization turkey coronavirus The complete nucleotide sequence polyprotein gene assemble full length genome sequence present turkey coronavirus TCoV isolate ATCC The TCoV polyprotein gene encode two open read frame ORFs translate two product pp1a pp1ab latter produce via frameshift translation TCoV polyprotein pp1a pp1ab predict process non structure proteins nsp2–nsp16 nsp1 miss ClustalW analysis reveal identity similarity pp1ab TCoV avian infectious bronchitis virus IBV amino acid level The whole genome consist 749nucleotides 816nucleotides ATCC exclude poly tail total ORFs predict TCoV Five subgenomic RNAs detect ATCC infect turkey small intestine Northern blot The whole genome sequence identity TCoV IBV support TCoV group coronavirus',\n",
       "  0.5470384359359741),\n",
       " ('cyp4f3', 0.5470155477523804),\n",
       " ('d279n', 0.5468029379844666),\n",
       " ('Stimulation ribosomal frameshifting antisense LNA Programmed ribosomal frameshifting translational recoding mechanism commonly use RNA virus express two protein single mRNA fix ratio essential element process presence RNA secondary structure pseudoknot hairpin locate downstream slippery sequence Here test efficiency RNA oligonucleotides anneal downstream slippery sequence induce frameshifting vitro Maximal frameshifting observe oligonucleotides 12–18 Antisense oligonucleotides bearing lock nucleid acid LNA modification also prove efficient frameshift stimulators contrast DNA oligonucleotides The number sequence location LNA base otherwise DNA oligonucleotide carefully manipulate obtain optimal level frameshifting Our data favor model RNA stability entrance ribosomal tunnel major determinant stimulate slippage rather specific three dimensional structure stimulate RNA element',\n",
       "  0.5467519760131836),\n",
       " ('Inhibition SARS CoV replication cycle small interference RNAs silence specific SARS proteins The severe acute respiratory syndrome coronavirus SARS CoV genome potential open reading frame ORFs The first ORF translate full length genomic mRNA remain ORFs translate eight subgeomic RNAs sgRNAs study design small interference RNAs siRNAs target sgRNA tested efficiency specificity silence protein translate targeted sgRNA Our result demonstrate siRNA could inhibit sgRNA show nucleotide match sgRNA show 19nt matching inhibit full length genomic mRNA show 19nt match Overall siRNAs inhibit target sgRNA without affect full length genomic mRNA sgRNAs show mismatch two Thus siRNA could design knockdown expression viral protein target sgRNA viral infection thereby allow contribution individual viral protein viral infection delineate When Vero cell express siRNA infected SARS CoV significant reduction yield progeny virus observe Indirect immunofluorescence assay show infected cell express siRNAs aspecific silence respectively expression nucleocapsid protein affect Thus data suggest accessory protein could play important role replication cycle SARS CoV',\n",
       "  0.5466998219490051),\n",
       " ('Identification Protein Interaction Partners Mammalian Cells Using SILAC immunoprecipitation Quantitative Proteomics Quantitative proteomics combine immuno affinity purification SILAC immunoprecipitation represent powerful mean discovery novel protein protein interaction allow accurate relative quantification protein abundance control test sample true interaction may easily distinguish experimental contaminant Low affinity interaction preserve use less stringent buffer condition remain readily identifiable This protocol discuss label tissue culture cell stable isotope label amino acid transfection immunoprecipitation affinity tag protein interest follow preparation submission mass spectrometry facility This protocol discuss analyze interpret data return mass spectrometer order identify cellular partner interact protein interest example technique apply identify protein bind eukaryotic translation initiation factor eIF4AI eIF4AII',\n",
       "  0.5465684533119202),\n",
       " ('aa4', 0.5462989807128906),\n",
       " ('SARS Coronavirus Replication Transcription Complexes Are Membrane Protected Need Host Factor Activity Vitro SARS coronavirus SARS CoV replication transcription mediate replication transcription complex RTC virus encode non structural protein nsps primary constituent The SARS CoV nsps produce autoprocessing two large precursor polyproteins The RTC believe associate characteristic virus induce double membrane structure cytoplasm SARS CoV infect cell investigate link structure viral RNA synthesis dissect RTC organization function isolate active RTCs infect cell use develop first robust assay vitro activity The synthesis genomic RNA eight subgenomic mRNAs faithfully reproduce RTC vitro system Mainly positive strand RNAs synthesize protein synthesis require RTC activity vitro All RTC activity enzymatic putative membrane span nsps viral RNA cosedimented heavy membrane structure Furthermore pelleted RTC require addition cytoplasmic host factor reconstitution vitro activity Newly synthesize subgenomic RNA appear release genomic RNA remain predominantly associate RTC contain fraction RTC activity destroy detergent treatment suggest important role membranes The RTC appear protected membrane newly synthesize viral RNA several replicase transcriptase subunits protease nuclease resistant become susceptible degradation upon addition non ionic detergent Our data establish vital functional dependence SARS CoV RNA synthesis virus induce membrane structure',\n",
       "  0.5458016395568848),\n",
       " ('torpor', 0.5457044839859009),\n",
       " ('mulv', 0.5456658601760864),\n",
       " ('Sequence analysis gene gene gene avian infectious bronchitis virus strain previously report nucleotide sequence gene spike protein gene gene nucleocapsid protein gene end untranslated region novel avian infectious bronchitis virus IBV strain Jia Arch Virol 259–271 present report describe sequence remain gene strain gene exception gene RNA polymerase gene Gene contain three open read frame ORFs nucleotide respectively Gene membrane protein gene consist single ORF Gene contain two ORFs respectively Thus total end protein gene end genome The overall sequence homology gene strain 1–90 8–88 4–96 respectively The predict amino acid sequence homology reveal gene conserve Each individual gene strain exception RNA polymerase gene different level homology homologous gene strain suggest evolution IBV strain general complex yet poorly understood process Elsevier Science All right reserve',\n",
       "  0.5456026792526245),\n",
       " ('1±3', 0.5454562902450562),\n",
       " ('mvmp', 0.54533451795578),\n",
       " ('novel synthesis detection method cap associate adenosine modification mouse mRNA method describe detection certain nucleotide modification adjacent methyl guanosine cap mRNAs individual gene The method quantitatively measure relative abundance methyl dimethyladenosine two common modification order identify quantitatify amount dimethyladenosine novel method synthesis modify adenosine phosphoramidites develop This method one step synthesis product directly use production dimethyladenosine contain RNA oligonucleotides The nature cap adjacent nucleotide show characteristic mRNAs individual gene transcribe liver testis',\n",
       "  0.5452682971954346),\n",
       " ('glutaraldehyde', 0.5452225208282471),\n",
       " ('Ribosomal RNA depletion exclusion negligible effect detection viruses pan viral microarray Pan viral DNA microarrays detect know novel multiple viral infection major laboratory asset contribute control infectious diseases The large quantity ribosomal RNA rRNA find tissue sample think major factor contribute comparatively low sensitivity detect RNA virus sequence independent PCR use amplify unknown sample microarray analysis This study aim determine whether depletion exclusion rRNA improve microarray detection simplify analysis Therefore two different rRNA depletion exclusion protocol RiboMinus™ technology non rRNA bind hexanucleotides apply microarray sample process outcome compare sequence independent amplification protocol This study conclude two procedure describe deplete exclude rRNA negligible effect microarrays detection analysis might combination technique result significant enhancement sensitivity Currently exist protocol random amplification background adjustment pertinent purpose sample processing microarray analysis',\n",
       "  0.5451672077178955),\n",
       " ('biomark', 0.5450776815414429),\n",
       " ('Genic amplification entire coding region HEF RNA segment influenza virus order provide easy powerful analysis influenza viral HEF RNA segment recent strain combination reverse transcription polymerase chain reaction use amplify entire coding region HEF gene laboratory strain virus call Johannesburg widely use bind esterase activity study well strain isolated Paris patient suffer severe flu syndrome The sequence amplify kilobases long work show forward universal primer Unil use influenza viruses cDNA synthesis also use influenza virus The PCR primer design contain restriction site make PCR product ready used purpose restriction analysis PCR product combine analysis human influenza virus HEF gene sequence publish far permitted design set oligonucleotides prime PCR cDNA unknown influenza virus clone',\n",
       "  0.5450097322463989),\n",
       " ('Characterization kDa Polypeptide Encoded Gene Human Coronavirus HCV 229E Gene human coronavirus HCV 229E encompasses approximately contains two overlap open reading frame ORF ORF The downstream ORF express mechanism involve ribosomal frameshifting Translation mRNA think equivalent viral genomic RNA result synthesis two large polyproteins pp1a pp1ab These polyproteins contain motifs characteristic papain like like proteinase RNA dependent RNA polymerases helicases metal bind protein study produce pp1ab specific monoclonal antibody use detect intracellular kDa viral polypeptide contain putative RNA polymerase domain Furthermore usingtranscleavage assay bacterially express HCV 229E like proteinase demonstrate kDa polypeptide release pp1ab cleavage dipeptide bond Gln Ser Gln Ala These data contribute characterization coronavirus like proteinase mediate process pp1ab provide first identification HCV 229E ORF 1ab encode polypeptide virus infect cell',\n",
       "  0.5449457168579102),\n",
       " ('glycoproteins–raft', 0.5449326038360596),\n",
       " ('pan‐coronavirus', 0.5449296832084656),\n",
       " ('sarawak', 0.5446335077285767),\n",
       " ('Development specific nucleic acid probe differentiation porcine rotavirus serotypes dot blot hybridization assay describe detection differentiation porcine rotavirus serotypes Recombinant complementary DNA cDNA represent gene gene encode neutralization antigen VP7 glycoprotein OSU porcine rotavirus serotype Gottfried porcine rotavirus serotype strain use determine optimal hybridization condition allow specific detection group porcine rotaviruses Probes prepared excision insert recombinant plasmid radiolabeling cDNA 32P random primer extension method Probes hybridize various stringency viral RNA different rotavirus serotypes bound nylon membrane Hybridization low stringency base pair mismatch stable hybrid formation high sensitivity low specificity Hybridization high stringency base pair mismatch stable hybrid formation produce high specificity decrease sensitivity observe low stringency Probes specific rotavirus stringency hybridize nucleic acid porcine virus Subgenomic gene fragment test provide specific probe fragment OSU gene nucleotide fragment Gottfried gene nucleotide find specific hybridization probes These study demonstrate feasibility dot blot hybridization assay use subgenomic fragment gene detect differentiate serotypes porcine rotavirus Additional study warrant evaluate sensitivity capability probe detect porcine field isolates serotype',\n",
       "  0.5445846319198608),\n",
       " ('Recombinant nucleocapsid protein potentially inexpensive effective serodiagnostic reagent infectious bronchitis virus The nucleocapsid protein Gray strain infectious bronchitis virus IBV highly immunogenic cross reactive among various distinct serotypes Recombinant nucleocapsid polypeptide express bacteria histidine tag amino terminus use antigen develop assay detect IBV specific antibody This fusion protein produce readily bacteria easily purify nickel column bound histidine tag Conditions optimize use preparation IBV specific ELISA Although difference optical density could detect pre immune positive serum Ark Mass Gray strain antigen concentration per well great difference could detect protein per well Maximum difference optical density pre immune positive serum obtain use per well protein serum dilute addition little dot recombinant nucleocapsid consistently detect IBV specific serum immunoblot assay convenient field application',\n",
       "  0.5444353222846985),\n",
       " ('Site specific target binding control RNA cleavage efficiency Kaposi sarcoma associate herpesvirus endonuclease SOX number virus remodel cellular gene expression landscape globally accelerate messenger RNA mRNA degradation Unlike mammalian basal mRNA decay enzymes largely target mRNA end virus instead use endonuclease cleave target internally This hypothesize rapidly inactivate mRNA maintain selective power potentially though use target motif Yet mRNA endonuclease specificity achieve mammalian cell remain largely unresolved Here reveal key feature underlie biochemical mechanism target recognition cleavage SOX endonuclease encode Kaposi sarcoma associate herpesvirus KSHV Using purify KSHV SOX protein reconstitute cleavage reaction vitro reveal SOX display robust sequence specific RNA binding residue proximal cleavage site must present particular structural context The strength SOX binding dictate cleavage efficiency provide explanation breadth mRNA susceptibility observe cell Importantly establish cleavage site specificity require additional cellular cofactor previously propose Thus viral endonuclease may use combination RNA sequence structure capture broad set mRNA target still preserve selectivity',\n",
       "  0.5443977117538452),\n",
       " ('pristane', 0.5443332195281982),\n",
       " ('Phi29 polymerase base random amplification viral RNA alternative random PCR BACKGROUND Phi29 polymerase base amplification method provide amplified DNA minimal change sequence relative abundance many biomedical application RNA virus detection use microarrays however present challenge phi29 DNA polymerase can not amplify RNA small cDNA fragment base obtain reverse transcription certain viral RNA genome Therefore ligation cDNA fragment necessary prior phi29 polymerase base amplification adapt QuantiTect Whole Transcriptome Kit Qiagen purpose designate method Whole Transcriptome Amplification WTA RESULTS WTA successfully amplify cDNA panel RNA viruses represent diversity ribovirus genome size amplify range genome copy number use WTA yield quantity amplify DNA high target copy The amplification factor vary also demonstrate amplification occur viral RNA mixed bacterial DNA CONCLUSION This first report scientific literature show modified WGA WTA approach successfully applied viral genomic RNA size Amplifying viral RNA WTA provide considerably well sensitivity accuracy detection compare random PCR',\n",
       "  0.5442103147506714),\n",
       " ('Formation activity covalent conjugate poliovirus ligand bind cell surface structure Disulfide link conjugate poliovirus streptavidin concanavalin form bind conjugate mouse cell lack natural poliovirus receptor study The conjugate streptavidin specifically bound biotinylated cell unmodified cell The conjugate conA bound cell absence presence methyl mannoside Incubation cell bound conjugate produce virus although conjugate highly infectious HeLa cell contain natural poliovirus receptor This suggest artificially bound virus unable penetrate cell start replication The possibility bind virus natural receptor require efficient infection discuss',\n",
       "  0.5440507531166077),\n",
       " ('Identification bovine viral diarrhea virus receptor different cell type Anti idiotypic antibody anti use successfully study bovine viral diarrhea virus BVDV receptor laboratory The anti specifically bound culture cell identify kDa cellular membrane protein think specific receptor BVDV study flow cytometric analysis demonstrate anti also specifically bound different cell type namely MDBK EBK PK15 MA104 Vero Experiments virus attachment replication show BVDV adsorb cell replicate except monkey kidney cell MA104 Vero non permissive Results plaque reduction assay indicate cellular membrane protein cell line competitively inhibit BVDV attachment culture MDBK cell suggest presence BVDV receptor cell Immunoblotting cell membrane proteins anti reveal kDa protein permissive nonpermissive cell Subcloned synchronized MDBK cell demonstrate significant difference bind anti compare normal cultured cell',\n",
       "  0.5438085794448853),\n",
       " ('biotinylated', 0.5437339544296265),\n",
       " ('intergroup', 0.5434909462928772),\n",
       " ('soya', 0.5432146787643433),\n",
       " ('sialate', 0.5431294441223145),\n",
       " ('SARS Coronavirus nsp1 Protein Induces Template Dependent Endonucleolytic Cleavage mRNAs Viral mRNAs Are Resistant nsp1 Induced RNA Cleavage SARS coronavirus SCoV nonstructural protein nsp potent inhibitor host gene expression possess unique mode action bind 40S ribosome inactivate translation function induce host mRNA degradation Our previous study demonstrate nsp1 induces RNA modification near end reporter mRNA short untranslated region RNA cleavage encephalomyocarditis virus internal ribosome entry site IRES region dicistronic RNA template IRES elements hepatitis cricket paralysis virus use primarily cell free vitro translation system present study reveal nsp1 induce endonucleolytic RNA cleavage mainly near untranslated region cap mRNA template Experiments use dicistronic mRNAs carry different IRESes show nsp1 induce endonucleolytic RNA cleavage within ribosome loading region type type picornavirus IRES elements classical swine fever virus IRES characterize hepatitis virus like IRES The nsp1 induce RNA cleavage template mRNAs exhibit apparent preference specific nucleotide sequence RNA cleavage sit Remarkably SCoV mRNAs cap structure poly tail like typical host mRNAs susceptible nsp1 mediate RNA cleavage importantly presence end leader sequence protect SCoV mRNAs nsp1 induce endonucleolytic RNA cleavage The escape viral mRNAs nsp1 induce RNA cleavage may important strategy virus circumvents action nsp1 lead efficient accumulation viral mRNAs viral protein infection',\n",
       "  0.5431209802627563),\n",
       " ('amass', 0.5428407192230225),\n",
       " ('farglitazar', 0.5427688360214233),\n",
       " ('alb18', 0.5427619218826294),\n",
       " ('Expression purification turkey coronavirus nucleocapsid protein Escherichia coli Purification turkey coronavirus TCoV nucleocapsid protein express prokaryotic expression system histidine tag fusion protein demonstrate present study Turkey coronavirus partially purify infected intestine turkey embryo sucrose gradient ultracentrifugation RNA extract The protein gene amplify extract RNA reverse transcription polymerase chain reaction clone The recombinant expression construct pTri identify polymerase chain reaction sequence analysis Expression histidine tag fusion protein molecular mass 57kd determine Western blot analysis chromatography nickel agarose column express protein purify near homogeneity judge sodium dodecyl sulfate polyacrylamide gel electrophoresis analysis The protein recovery could 5mg 100ml bacterial culture The purified protein recognize antibody TCoV Western blot assay The capability recombinant protein differentiate positive serum turkey infect TCoV normal turkey serum evident enzyme link immunosorbent assay ELISA These result indicate expressed protein superior source TCoV antigen development antibody capture ELISA detection antibody TCoV',\n",
       "  0.5425020456314087),\n",
       " ('ser38', 0.5424845218658447),\n",
       " ('basc', 0.5424214601516724),\n",
       " ('Detection infectious canine parvovirus type mRNA real time PCR TaqMan real time PCR assay develop detection RNA transcript produce replicate CPV pair primer TaqMan probe target splice NS2 mRNA design synthetic DNA fragment construct mimic spliced NS2 mRNA PCR base gene assembly use generation standard RNAs The detection limit assay 1×102 RNA copy standard curve display linear range 1×102 1×109 copy good reproducibility The assay apply determine mRNA load tissue dog naturally infect CPV mRNA detect variety tissue include central nervous system',\n",
       "  0.5421856641769409),\n",
       " ('e00801', 0.5420128107070923),\n",
       " ('sufu', 0.5418546795845032),\n",
       " ('transwithout', 0.541846513748169),\n",
       " ('farrowing', 0.5412842035293579),\n",
       " ('The replication murine coronaviruses enucleate cell Coronaviruses JHMV A59V show replicate produce viral specific antigen cytopathic effect CPE enucleate 17CL cell',\n",
       "  0.5412019491195679),\n",
       " ('eif', 0.5409790277481079),\n",
       " ('FLDS Comprehensive dsRNA Sequencing Method Intracellular RNA Virus Surveillance Knowledge distribution diversity RNA virus still limit spite possible environmental epidemiological impact RNA virus specific metagenomic method yet develop herein constructed effective metagenomic method RNA viruses target long double stranded RNA cellular organism hallmark infection replication dsRNA single strand RNA virus except retrovirus This novel dsRNA target metagenomic method characterize extremely high recovery rate viral RNA sequence retrieval terminal sequence uniform read coverage previously report metagenomic method target RNA viruses This method reveal previously unidentified viral RNA diversity complete RNA viral genome include dsRNA ssRNA virus associate environmental diatom colony Our approach powerful tool catalog RNA virus associate organisms interest',\n",
       "  0.5409544706344604),\n",
       " ('sulphide', 0.5408613681793213),\n",
       " ('4–6', 0.5407394766807556),\n",
       " ('Sequence analysis porcine transmissible gastroenteritis coronavirus nucleocapsid protein gene The end genome Purdue strain porcine transmissible gastroenteritis coronavirus TGEV copy eDNA prim oligo double strand product clone PstI site pUC9 vector One clone contain part poly tail sequence entirety use chemical method Maxam Gilbert Another clone also contain part poly tail sequence part confirm primary structure end genome Two potential nonoverlapping gene identify within terminal base sequence examination open reading frame The first gene encodes amino acid protein mol apparent nucleocapsid protein basis size chemical property amino acid sequence homology coronavirus nucleocapsid proteins flank side least part matrix protein gene The second encodes hypothetical amino acid protein mol hydrophobic end proximal noncoding sequence base also determine conserve stretch nucleotide near poly tail find common among TGEV mouse hepatitis coronavirus avian infectious bronchitis coronavirus',\n",
       "  0.5407050848007202),\n",
       " ('sulfonate', 0.5406991243362427),\n",
       " ('Intracellular andin Vitro Translated kDa Proteins Contain like Proteinase Activity Coronavirus MHV A59 The coronavirus mouse hepatitis virus A59 MHV A59 encode serine like proteinase like proteinase 3CLpro ORF gene nucleotide previously demonstrate protein expressedin vitrofrom cDNA clone 3CLpro region posse proteinase activity proteinase able cleave substratein trans seek determine kDain vitrocleavage product p27 active form 3CLpro whether consistent 3CLpro express virus infect cell Antibodies direct 3CLpro domain detect kDa MHV proteinsin vitroand MHV A59 infect cell The kDa protein able cleave substratein transwithout protein cofactor supplemental membrane p27 proteinase activity retain purification immunoprecipitation gel electrophoresis When p27 expressedin vitrowith portion amino carboxy terminal flanking domains MP1 MP2 p27 liberate byciscleavage The proteolytic activity kDa protein inhibit variety cysteine serine proteinase inhibitor eliminate cysteine proteinase inhibitor E64d These result indicate kDa protein mature proteinase MHV A59 infect cell appropriate processing molecule occursin vitro',\n",
       "  0.5403999090194702),\n",
       " ('Parallel ClickSeq Nanopore sequence elucidates rapid evolution defective interfering RNAs Flock House virus Defective Interfering RNAs RNAs long know play important role virus replication transmission RNAs emerge virus passaging cell culture host result non homologous RNA recombination However principles RNA emergence subsequent evolution remain elusive Using combination long short read Next Generation Sequencing characterize formation RNAs serial passaging Flock House virus FHV cell culture period day order elucidate pathway potential mechanism RNA emergence evolution For short read RNAseq employ ClickSeq due ability sensitively confidently detect RNA recombination event nucleotide resolution parallel use Oxford Nanopore Technologies ONT MinION resolve full length defective wild type viral genome Together accurately resolve rare common RNA recombination event determine correlation recombination event quantifies relative abundance different RNAs throughout passaging observe formation diverse pool defective RNAs stage viral passaging However many intermediate specie present early stage passaging accumulate After approximately day passaging observe rapid accumulation RNAs correlate reduction specific infectivity Nanopore data find RNAs characterize multiple RNA recombination event This suggest intermediate RNA specie competitive multiple recombination event interact epistatically confer mature RNAs selective advantage allow rapid accumulation Alternatively possible mature RNA specie generate single event involve multiple RNA rearrangement These insight important consequence understand mechanisms determinant limitation emergence evolution RNAs',\n",
       "  0.5403732061386108),\n",
       " ('electrophoresed', 0.5403677225112915),\n",
       " ('Complete genomic sequence turkey coronavirus Turkey coronavirus TCoV one least characterized know coronaviruses isolate outbreak acute enteritis young turkey Ontario Canada full length genomic sequence determine The full length genome nucleotide plus poly tail Two open read frame ORFs reside first two third genome nine additional downstream ORFs identify gene hemagglutinin esterase absent TCoV The region membrane nucleocapsid protein gene contain three potential small ORFs ORF previously uncharacterized ORF associate putative TRS within gene apparently share among group III coronaviruses previously describe ORFs The TCoV genome organize follow UTR replicase ORFs spike protein ORF3 ORFs small envelop protein membrane protein ORF5 ORFs nucleocapsid protein −3′ UTR poly TCoV genome structure sequence similar distinct avian infectious bronchitis virus IBV This first complete genome sequence TCoV confirm TCoV belongs group III coronaviruses',\n",
       "  0.5403529405593872),\n",
       " ('t279a', 0.540124773979187),\n",
       " ('hybridization', 0.5401104688644409),\n",
       " ('irf3–5d', 0.5399099588394165),\n",
       " ('toassess', 0.5398872494697571),\n",
       " ('41±1', 0.5397509336471558),\n",
       " ('209–2', 0.5396766066551208),\n",
       " ('Microarray Based Approach Identification Common Foodborne Viruses oligonucleotide array microarray incorporate element represent select strain hepatitis virus HAV human coxsackieviruses CVA CVB genogroups Norovirus human rotavirus gene segment design base principle tile Each oligonucleotide base long start every 5th base every sequence result overlap base two consecutive oligonucleotides The applicability array virus identification examine use PCR amplified product multiple HAV strain PCR product label biotin hybridized array biotin detect use brief reaction Cy3 label streptavidin array subject laser scan hybridization data plot fluorescence intensity oligonucleotide array The combined signal intensity probe represent particular strain virus calculate plotted virus strain identify linear representation array The profile total signal intensity identify strain likely represent amplified cDNA target The result obtain HAV indicate hybridization profile thus generate used identify closely related viral strain This represent significant improvement current method virus identification use PCR amplification amplicon sequence',\n",
       "  0.5395610332489014),\n",
       " ('Yeast base assays high throughput screen inhibitor coronavirus RNA cap guanine methyltransferase The cap structure distinct feature eukaryotic mRNAs important RNA stability protein translation provide molecular signature distinction self non self mRNA Eukaryotic virus generally modify end RNAs mimic cellular mRNA structure thereby facilitate viral replication host cell However molecular organization biochemical mechanism viral cap apparatus typically differ cellular counterpart make viral cap enzymes attractive target drug discovery Our previous work show SARS coronavirus SARS CoV non structural protein represent structurally novel unique guanine methyltransferase MTase able functionally complement yeast cellular MTase present study develop yeast base system identify screen inhibitor coronavirus MTase use well well microtiter plat The MTase inhibitor previously identify vitro biochemical assay test sinefungin effectively suppress MTase yeast system However compound ATA AdoHcy exert inhibitory effect within cellular context These result validate yeast assay system inhibitor screen yet also demonstrate difference cell base vitro biochemical assay The yeast system apply screen natural product extract three observe potently inhibit activity coronavirus human MTase',\n",
       "  0.5394782423973083),\n",
       " ('Mutational analysis slippery sequence component coronavirus ribosomal frameshifting signal The ribosomal frameshift signal genomic RNA coronavirus IBV compose two element heptanucleotide slippery sequence downstream RNA pseudoknot investigate kind slippery sequence function IBV frameshift site analyse frameshifting property series slippery sequence mutant firstly confirm site frameshifting IBV heptanucleotide stretch UUUAAAC use knowledge pseudoknot structure suitable reporter gene prepare expression construct allow magnitude direction ribosomal frameshifting determine candidate slippery sequence Our result show almost sequence test frameshifting strictly reading frame Monotonous run nucleotide however give detectable level frameshift product stretch particular give significant level Preliminary evidence suggest RNA pseudoknot may play role influence frameshift direction The spectrum slip sequence test analysis include know suspect utilized vivo Our result indicate triplets functional decode ribosomal site follow slippage XXX YYYN although triplet least effective site XXY YYY triplet non functional The identity nucleotide position slippery sequence XXXYYY find critical determinant frameshift efficiency show hierarchy frameshifting exist site codon These observation lead suggest ribosomal frameshifting particular site determine least part strength interaction normal cellular tRNAs site codon necessarily involve specialized shifty tRNAs',\n",
       "  0.5392657518386841),\n",
       " ('inactivated West Nile Virus vaccine derive chemically synthesize cDNA system cDNA comprise complete genome West Nile Virus WNV generate chemical synthesis use publish sequence data independent preformed viral component The synthetic WNV produce transfection vitro transcribe RNA cell culture exhibit undistinguishable biological property compare correspond animal derive wild type virus difference find concern viral growth mammalian insect cell line concern expression viral proteins cell There also significant difference virulence mouse follow intranasal challenge After immunization mice experimental vaccine derive synthetic wild type virus protection lethal challenge achieve similar amount antigen Both vaccine preparation also induce comparable level neutralize antibody mouse addition synthetic approach turn accurate since rescue WNV genome contain undesired mutation Thus first flavivirus base chemical gene synthesis indistinguishable parent virus This demonstrate virus isolates animal source dispensable derive seed virus vaccine production research',\n",
       "  0.5392501950263977),\n",
       " ('immunomediated', 0.5388278365135193),\n",
       " ('Dissection SARS Coronavirus Spike Protein Discrete Folded Fragments The spike protein severe acute respiratory syndrome coronavirus SARS CoV mediate cell fusion bind target cell surface receptor This paper report simple method dissect viral protein search foldable fragment random systematic manner The method involve digestion DNase generate pool short DNA segment follow additional step reassembly segment produce library DNA fragment random end controllable length rapidly screen discrete fold polypeptide fragment reassemble gene fragment clone vector terminal fusion fold reporter gene variant green fluorescent protein Two foldable fragment identify SARS CoV spike protein coincide various anti SARS peptides derive hepated repeat region spike protein The method applicable viral protein isolate antigen vaccine candidate thus provide alternative full length proteins subunit linear short peptide',\n",
       "  0.5387350916862488),\n",
       " ('tsg31', 0.53871089220047),\n",
       " ('Rapid differentiation avian infectious bronchitis virus isolate sample residual ratio quantitation use real time reverse transcriptase polymerase chain reaction rapid diagnostic assay differentiate avian infectious bronchitis virus IBV isolate develop The basis assay cleavage target RNA RNase mediate sequence specific chimeric oligonucleotides follow sample residual ratio quantitation SRRQ use RRT PCR Four serotype specific chimeric oligonucleotides design one Massachusetts Connecticut Arkansas Delaware Georgia serotypes test ability mediate specific cleavage target RNA know homologous heterologous strain IBV Specific cleavage target RNAs chimeric oligonucleotide verify use agarose gel analysis RRT PCR There non specific cleavage product Eight different IBV strain represent seven serotypes test chimeric oligonucleotide mediate cleavage target RNA strain within serotype chimeric design The SRRQ assay evaluate sample without prior knowledge group correctly identify serotype sample The assay rapid six sample test approximately addition primer set amplifies IBV RNAs test date provide build control detect IBV whether typeable',\n",
       "  0.5384649038314819),\n",
       " ('ecto', 0.5382071733474731),\n",
       " ('Current Research Non Coding Ribonucleic Acid RNA Non cod ribonucleic acid RNA without doubt capture interest biomedical researcher The ability screen entire human genome high throughput sequence technology greatly enhance identification annotation prediction functionality non cod RNAs review discus current landscape non cod RNA research quantitative analysis Non cod RNA categorize two major group size long non cod RNAs small RNAs long non cod RNA discus regular long non cod RNA pseudogenes circular RNA small RNA discus miRNA transfer RNA piwi interact RNA small nucleolar RNA small nuclear RNA RNA single recognition particle RNA 7SK RNA elaborate origin detection method potential association disease putative functional mechanism public resource non cod RNAs aim provide reader complete overview non cod RNAs incite additional interest non cod RNA research',\n",
       "  0.5381349325180054),\n",
       " ('dissident', 0.538087010383606),\n",
       " ('kunjin', 0.5379644632339478),\n",
       " ('hypoglycemic', 0.5377392768859863),\n",
       " ('fixatives', 0.5375769734382629),\n",
       " ('genealogy', 0.5375219583511353),\n",
       " ('benyvirus', 0.5371502041816711),\n",
       " ('t279', 0.5370298027992249),\n",
       " ('decapsidation', 0.5369778275489807),\n",
       " ('Identification polymerase polyprotein product p72 p65 murine coronavirus MHV JHM The RNA polymerase gene murine coronavirus MHV JHM encode polyprotein great kDa This polyprotein propose process two papain like cysteine proteinase PCP PCP poliovirus like proteinase domain pro generate protein product The amino terminal product MHV polymerase polyprotein p28 generate cleavage polyprotein PCP identify viral product downstream p28 generate fusion protein specific antiserum direct region adjacent p28 use antiserum detect virus specific proteins MHV JHM infect cell When antiserum use immunoprecipitate radiolabeled protein MHV JHM infect cell lysates virus specific proteins kDa detect Furthermore pulse chase experiment demonstrate p72 likely precursor mature protein product p65 investigate viral proteinase may responsible generate p72 p65 express region MHV JHM RNA polymerase gene include two papain like cysteine proteinase domains vitro transcription translation system analyze translation product proteolytic processing also clone express kDa region immediately downstream p28 test ability vitro translate PCP PCP cleave p72 p65 trans Our result indicate neither viral proteinase domain PCP PCP capable cleavage p72 produce p65 vitro The role MHV proteinases process p72 p65 discuss',\n",
       "  0.5368776321411133),\n",
       " ('anzueto', 0.536798357963562),\n",
       " ('case CUG initiate cod sequence overlap torovirus ORF1a encode novel kDa product The genus Torovirus order Nidovirales include number specie infect livestock These virus linear positive sense ssRNA genome encode large polyprotein express genomic RNA several additional protein express nested set coterminal subgenomic RNAs brief report describe bioinformatic discovery new apparently cod ORF overlap end polyprotein cod sequence ORF1a reading frame The new ORF strong coding signature fact conserve amino acid level overlap region ORF1a propose new ORF utilize non AUG initiation codon namely conserve CUG codon strong Kozak context upstream ORF1a AUG initiation codon result novel amino acid protein dub 30K',\n",
       "  0.5367566347122192),\n",
       " ('Alternative Polyadenylation Human Bocavirus Its End Regulated Multiple Elements Affects Capsid Expression Alternative processing human bocavirus HBoV promoter transcribe RNA critical generate structural nonstructural protein encode mRNA transcripts The regulatory mechanism HBoV RNA transcript polyadenylated proximal distal polyadenylation site still unclear construct recombinant HBoV infectious clone study alternative polyadenylation regulation HBoV Surprisingly addition report distal polyadenylation site novel distal polyadenylation site locate right end hairpin REH identify infectious clone transfection recombinant virus infection contain typical hexanucleotide polyadenylation signal upstream element USE downstream element DSE accord sequence analysis Further study show HBoV nonstructural protein NS1 REH element necessary sufficient efficient polyadenylation The distance sequence also play key role regulation polyadenylation Finally demonstrate efficient polyadenylation result increased HBoV capsid mRNA transcript protein translation Thus study reveal bocaviruses distal poly signal right end palindromic terminus alternative polyadenylation HBoV end regulate capsid expression IMPORTANCE The distal polyadenylation site HBoV locate nucleotide right end palindromic terminus different bovine parvovirus BPV canine minute virus MVC genus whose distal polyadenylation locate right end stem loop structure novel polyadenylation site identify right end hairpin HBoV infectious clone transfection recombinant virus infection Sequence analysis show contain typical polyadenylation signal last form stem loop structure almost identical MVC Further study show NS1 REH element require efficient polyadenylation Polyadenylation enhance capsid expression Our study demonstrate alternative polyadenylation end HBoV suggest additional mechanism capsid expression regulate',\n",
       "  0.5367410182952881),\n",
       " ('Cytoplasmic Translocation Polypyrimidine Tract Binding Protein Its Binding Viral RNA Japanese Encephalitis Virus Infection Inhibits Virus Replication Japanese encephalitis virus JEV single strand positive sense RNA genome contain single open reading frame flank non cod region NCRs The virus genome replicate via negative sense RNA intermediate The NCRs complementary sequence negative sense RNA sit assembly RNA replicase complex thereby regulate RNA synthesis virus replication study show kDa polypyrimidine tract bind protein PTB interact vitro NCR positive sense genomic RNA 5NCR complementary sequence negative sense replication intermediate RNA 3NCR The interaction viral RNA PTB validate infected cell JEV RNA immunoprecipitation JEV RNA PTB colocalization experiment Interestingly observe phosphorylation couple translocation nuclear PTB cytoplasmic focus localize JEV RNA early JEV infection Our study employ PTB silence expression culture cell establish inhibitory role PTB JEV replication Using RNA protein bind assay show PTB competitively inhibit association JEV 3NCR RNA viral RNA dependent RNA polymerase NS5 protein event require synthesis plus sense genomic RNA cAMP know promote Protein kinase PKA mediate PTB phosphorylation show cell treat cAMP analogue enhance level phosphorylated PTB cytoplasm significantly suppress JEV replication Data present show novel cAMP induce PTB mediate innate host response could effectively suppress JEV replication mammalian cell',\n",
       "  0.536447286605835),\n",
       " ('croos', 0.5364406704902649),\n",
       " ('rdc', 0.536386251449585),\n",
       " ('digitize', 0.5363659262657166),\n",
       " ('Identification novel ligand RNA pseudoknot regulate ribosomal frameshifting many virus ribosomal frameshifting −1RF regulate synthesis protein crucial virus production RNA pseudoknot one essential component viral −1RF system Thermodynamic kinetic control pseudoknot fold may important regulate efficiency −1RF Thus small molecule interact viral RNA pseudoknots may disrupt −1RF system show antiviral activity study conduct virtual screen chemical database target ray crystal structure RNA pseudoknot complexed biotin identify ligand may regulate −1RF system contain biotin aptamer RNA pseudoknot component After dock screen compound high rank hit select activity examine vitro cell base frameshifting assay Six compound increase efficiency frameshifting novel small molecule compound regulate −1RF',\n",
       "  0.5359717011451721),\n",
       " ('surveys', 0.5358597040176392),\n",
       " ('Interactions Between Virus Proteins Host Cell Membranes During Viral Life Cycle The structure function cell critically dependent membrane separate interior cell environment also define internal compartment therefore surprising major step life cycle viruses animal plant also depend cellular membrane Indeed interaction viral protein host cell membrane important virus enter host cell replicate genome produce progeny particle replicate genome virus first need cross plasma membrane Some virus also modify intracellular membrane host cell create compartment genome replication take place Finally viruses acquire envelope derive either plasma membrane internal membrane host cell This paper review recent finding interaction viral protein host cell membrane viral life cycle',\n",
       "  0.5357348918914795),\n",
       " ('froth', 0.535691499710083),\n",
       " ('37–2', 0.5355095863342285),\n",
       " ('pernasal', 0.5353373885154724),\n",
       " ('nm4', 0.5353180170059204),\n",
       " ('Post translational processing glycoprotein lymphocytic choriomeningitis virus Intracellular event synthesis glycosylation transport lymphocytic choriomeningitis virus LCMV glycoprotein examine show glycanase digestion LCMV strain Arm bear five oligosaccharide two pulse chase label experiment presence drug inhibit link oligosaccharide addition processing demonstrate addition high mannose precursor oligosaccharide necessary transport cleavage viral glycoprotein Moreover presence tunicamycin inhibits bloc addition mannose rich side chain virus bud substantially decrease infectious virion reduce fold supernatant medium Incubation presence castantospermine permit addition oligomannosyl rich chain block process restore transport cleavage maturation virion Finally temperature block experiment determine maturation oligosaccharides endoglycosidase resistant form precedes cleavage The latter process likely occur Golgi post Golgi compartment',\n",
       "  0.5352861285209656),\n",
       " ('Identification Characterization Novel Non Structural Protein Bluetongue Virus Bluetongue virus BTV causative agent major disease livestock bluetongue For two decade widely accept segment dsRNA genome BTV encode structural non structural proteins The non structural protein NS1 NS2 NS3 NS3a play different key role viral replication cycle study show BTV express fourth non structural protein designate NS4 encode open reading frame segment overlap open reading frame encode VP6 NS4 77–79 amino acid residue length highly conserve among several BTV serotypes strain NS4 express early post infection localize nucleoli BTV infect cell reverse genetics show NS4 dispensable BTV replication vitro mammalian insect cell affect viral virulence murine model bluetongue infection Interestingly NS4 confer replication advantage BTV BTV cell interferon IFN induce antiviral state However BTV NS4 confer replication advantage BTV reassortant contain entire segment BTV BTV mutant NS4 identical homologous BTV protein Collectively study suggest NS4 play important role virus host interaction one mechanism play least BTV counteract antiviral response host addition distinct nucleolar localization NS4 express virus replicates exclusively cytoplasm offer new avenue investigate multiple role play nucleolus biology cell',\n",
       "  0.5352535247802734),\n",
       " ('ikeda', 0.5352356433868408),\n",
       " ('Discontinuous RNA Platform Mediates RNA Virus Replication Building Integrated Model RNA–based Regulation Viral Processes Plus strand RNA viruses contain RNA element within genome mediate variety fundamental viral process The traditional view element local RNA structure This perspective however change due increase discovery functional viral RNA element form long range RNA–RNA interaction often span thousand nucleotides The plus strand RNA genomes tombusviruses exemplify concept possess different long range RNA–RNA interaction regulate viral translation transcription Here report third fundamental tombusvirus process viral genome replication require long range RNA–based interaction span ∼3000 vivo vitro analyse suggest discontinuous RNA platform form interaction facilitates efficient assembly viral RNA replicase This finding allow build integrated model role global RNA structure regulate reproduction eukaryotic RNA virus insight gain extended understanding multifunctional nature viral RNA genome',\n",
       "  0.5349795818328857),\n",
       " ('giscience', 0.5349767804145813),\n",
       " ('townships', 0.5347847938537598),\n",
       " ('tombusvirids', 0.534637451171875),\n",
       " ('appendicitis', 0.5346282720565796),\n",
       " ('12–798', 0.5343900322914124),\n",
       " ('peg10', 0.5343567728996277),\n",
       " ('long distance RNA–RNA interaction play important role program ribosomal frameshifting translation p88 replicase protein Red clover necrotic mosaic virus Programmed ribosomal frameshifting PRF one viral translation strategy express overlap gene positive strand RNA viruses Red clover necrotic mosaic virus RCNMV use strategy express replicase component protein p88 study use cell free translation system map act RNA element require PRF Our result show small stem loop structure adjacent cap independent translation element untranslated region UTR RCNMV RNA1 require PRF Site direct mutagenesis experiment suggest stem loop regulate PRF via base pair complementary sequence bulge stem loop adjacent shifty site The existence RNA elements responsible PRF cap independent translation replicase proteins UTR RNA1 might important switch translation replication regulate ratio p88 p27',\n",
       "  0.5342249870300293),\n",
       " ('Cooperation transmissible gastroenteritis coronavirus TGEV structural protein vitro induction virus specific antibody Following infection haplotype define NIH miniswine virulent transmissible gastroenteritis coronavirus TGEV isolate mesenteric lymph node CD4 cell mount specific proliferative response infectious inactivate purified virus secondary vitro stimulation specific dose dependent response three major recombinant viral protein spike membrane nucleoprotein purify affinity chromatography characterize Induction vitro antibody synthesis analyze The purified recombinant viral protein induce vitro synthesis neutralize TGEV specific antibody porcine TGEV immune cell stimulate combination make two major structural protein minor extent individual protein protein dissociate purified virus use detergent micellar protein oligomers rosettes form remove detergent These occur preferentially association protein trimmer These rosettes collaboration either proteins elicit TGEV specific antibody titer respectively induced whole virus protein could partially substitute peptide represent helper epitope previously identify protein Antón These result indicate induction high level TGEV specific antibody require stimulation least two viral protein optimum response induce combination rosettes nucleoprotein',\n",
       "  0.5341098308563232),\n",
       " ('potentiative', 0.5340100526809692),\n",
       " ('microcarriers', 0.5339493751525879),\n",
       " ('mcw', 0.533910870552063),\n",
       " ('mp4', 0.5337525606155396),\n",
       " ('priproet', 0.5337194204330444),\n",
       " ('affix', 0.5334440469741821),\n",
       " ('csd', 0.5333556532859802),\n",
       " ('prominent antigenic surface polypeptide involved biogenesis function vaccinia virus envelope Polypeptides vaccinia virus envelope expose surface identify mean sulfo hydroxysuccinimidobiotin surface tag Among surface express polypeptide kDa antigen previously designate Ag35 Both monoclonal mAb monospecific affinity pure antibody direct Ag35 neutralize vaccinia infectiousness indicate prominent surface antigen function early virus host cell interactions The bind several monoclonal antibody various region Ag35 test react CNBr fragment derive polypeptide employ Western blot All mAbs test reacted region Ag35 Estimation molecular weight base migration CNBr peptides sodium dodecyl sulfate polyacrylamide gel electrophoresis reveal partial digestion product contain proline rich amino acid limit digest fragment present position approximately kDa large predict DNA sequence contrast partial limit digest product lack proline rich fragment migrate position expect length DNA sequence This observation demonstrate departure predict kDa anomalous Ag35 confer proline rich peptide The surface location Ag35 confirm immune electron microscopy competition test binding specificity mAb affinity purify antibody surface virion could demonstrate Evidence association Ag35 virus envelope various stage biogenesis vaccinia obtain immune electron microscopy whole mount thin section Presence Ag35 early component immature mature virion probably reside bilayer membrane structure detect distinction therefore make Ag35 several vaccinia envelope polypeptide synthesize late function add late stage envelope assembly',\n",
       "  0.532738447189331),\n",
       " ('Enhancement target DNA hybridization efficiency pre hybridization sequence orientate micro arrayed probes The enhancement hybridization efficiency deoxyribonucleic acid DNA target use oligonucleotide pre hybridization study two sequence inversed micro arrayed probes The sequence pre hybridize oligo target DNA design fully complementary share DNA probe coaxial stack configuration hybridize immediately alongside along continuous complement probe strand The pre hybridize oligo target DNA differentiate label two distinct fluorescent dye cooperative effect hybridization efficiency investigate comparison stack individual hybridization configuration base detection signal label dye The result show pre hybridization DNA oligo enhances subsequent hybridization efficiency target DNA couple onto probe The efficiency enhance hybridization position occur site close substrate surface',\n",
       "  0.5326945781707764),\n",
       " ('Native Tertiary Structure Nucleoside Modifications Suppress tRNA Intrinsic Ability Activate Innate Immune Sensor PKR Interferon inducible protein kinase PKR essential component innate immunity activate long stretch dsRNA provide first line host defense pathogens inhibit translation initiation infect cell Many cellular viral transcript contain nucleoside modification tertiary structure could affect PKR activation previously demonstrate end triphosphate–a signature certain viral bacterial transcripts–confers ability relatively unstructured model RNA transcript activate PKR inhibit translation activation abrogate certain modification present cellular RNAs order understand biological implication native RNA tertiary structure nucleoside modification PKR activation study heavily modify cellular tRNAs unmodified lightly modify mitochondrial tRNAs tRNA find transcript yeast tRNA Phe natively extract total bovine liver tRNA activate PKR vitro whereas native coli bovine liver yeast wheat tRNA Phe variety base sugar modify transcript These result support activation PKR natively fold transcript tRNA Phe vivo support importance tRNA modification suppress PKR activation cell also examine PKR activation transcript A14G pathogenic mutant tRNA Leu know dimerize find misfolded dimeric form activate PKR vitro monomeric form Overall vitro vivo finding herein indicate tRNAs intrinsic ability activate PKR nucleoside modification native RNA tertiary folding may function least part suppress activation thus serve distinguish self non self tRNA innate immunity',\n",
       "  0.5326472520828247),\n",
       " ('inventories', 0.5326291918754578),\n",
       " ('subfragment', 0.5326237678527832),\n",
       " ('The Major Product Porcine Transmissible Gastroenteritis Coronavirus Gene Integral Membrane Glycoprotein kDa The open reading frame potentially encode polypeptide kDa locate second three ORFs gene gene genome Purdue strain porcine transmissible gastroenteritis coronavirus TGEV clone expressedin vitroto examine property protein Gene postulate role pathogenesis truncate form laboratory passaged strain TGEV lead suggestion essential virus replication During synthesisin vitroin presence microsomes kDa polypeptide become integral membrane protein retain postulated hydrophobic terminal signal sequence underwent glycosylation apparently two asparagine linkage site attain final molecular mass kDa kDa terminally truncate nonglycosylated nonanchored form protein also make via unknown mechanism The existence transmembrane soluble form gene product cell suggest immunofluorescence pattern show punctated perinuclear diffuse intracytoplasmic distribution gene product find gradient purified Purdue TGEV Western blotting procedure would detect molecule virion indicate protein probably structural component virion',\n",
       "  0.5325280427932739),\n",
       " ('ts16', 0.5322585701942444),\n",
       " ('ranpb5', 0.5321136116981506),\n",
       " ('417–422', 0.5320239067077637),\n",
       " ('Glucose trim mannose trim affect different phase maturation Sindbis virus infect BHK cell The role glucose mannose trim maturation Sindbis virus BHK cell investigate use inhibitor glycoprotein oligosaccharide processing presence glucosidase inhibitor methyl deoxynojirimycin viral glycoprotein equip oligosaccharide composition GIc3Man8 GIcNAc yield virus extracellular medium reduce result block proteolytic cleavage precursor pE2 viral envelope glycoprotein The mannosidase inhibitor deoxymannojirimycin dMM also inhibit appearance virus medium oligosaccharides viral glycoprotein composition Man9 GIcNAc However pE2 cleave condition find yield virus cell medium together consider difference untreated dMM treat culture suggest presence intracellular virus particle dMM treat culture When examine electron microscopy dMM treat culture find contain intracellular virus particle addition nucleocapsids find line intracellular membrane These observation take together plaque test data intimate Sindbis virus preferentially bud internal membrane BHK cell treat dMM The result confirm essential role glucose trim Sindbis virus BHK cell system suggest initial stage mannose removal maybe important',\n",
       "  0.5319917798042297),\n",
       " ('Molecular clone expression purification SARS CoV nsp13 The SARS nsp13 protein identified mRNA cap1 methyltransferase study nsp13 gene clone SARS CoV PUMC02 strain viral RNA PCR insert expression plasmid pET30a The recombinant plasmid pET30a nsp13 confirm restriction enzyme sequence analysis transform Escherichia coli BL21 DE3 The His tag fuse protein express induction 5mM IPTG purify single Ni2 affinity chromatography The protein validate western blot analysis large quantity nsp13 protein obtain method may useful study structure function',\n",
       "  0.531916618347168),\n",
       " ('hypo', 0.5315855741500854),\n",
       " ('toxicodynamics', 0.5312401056289673),\n",
       " ('rtlamp', 0.5312366485595703),\n",
       " ('bellomo', 0.5312106609344482),\n",
       " ('≥100g', 0.5309536457061768),\n",
       " ('p301', 0.5307286977767944),\n",
       " ('astrocytosis', 0.5306907892227173),\n",
       " ('SHAPE analysis RNA secondary structure Mouse Hepatitis Virus untranslated region terminal nsp1 cod sequence SHAPE technology use analyze RNA secondary structure MHV A59 genome encompass minimal act region require defective interfering RNA replication The structure generate agreement previous characterization SL1 SL4 two recently predict secondary structure element SL5A SHAPE provide biochemical support four additional stem–loops previously functionally investigated MHV Secondary structure prediction region MHV A59 BCoV SARS CoV similar despite high sequence divergence The pattern SHAPE reactivity virio genomic RNA virio genomic RNA vitro synthesize RNA similar suggest binding protein proteins virion RNA fail protect RNA reaction lipid permeable SHAPE reagent Reverse genetic experiment suggest SL5C SL6 within nsp1 cod sequence require viral replication',\n",
       "  0.530632734298706),\n",
       " ('6–100', 0.5305802822113037),\n",
       " ('Expression alternate reading frame protein hepatitis virus Escherichia coli detection antibody Indian patient Apart core 21kD novel hepatitis virus HCV frame shift protein synthesize initiation codon polyprotein sequence follow ribosomal frame shift reading frame date information available protein Indian isolate hence detection antibody protein Indian patient assume great relevance Specific primer design amplify sequence cod 120aa truncated tF1 The amplified tF1 clone bacterial expression vector pET21b expression Escherichia coli Partially purify express protein subjected western blot analysis use patient sera Three HCV positive serum employ western analysis show positive signal tF1 serum uninfected individual fail give signal Further result western blot carry patient serum titrate purify core protein confirm presence antibody specific The positive signal observe western analysis HCV infect serum suggest protein synthesize natural course HCV infection Indian patient well Presence antibody protein subtype demonstrate first time Indian patient',\n",
       "  0.5305255651473999),\n",
       " ('Inhibits Coronavirus Arterivirus RNA Polymerase Activity Vitro Zinc Ionophores Block Replication These Viruses Cell Culture Increasing intracellular concentration zinc ionophores like pyrithione efficiently impair replication variety RNA viruses include poliovirus influenza virus For virus effect attribute interference viral polyprotein processing study demonstrate combination low concentration inhibit replication SARS coronavirus SARS CoV equine arteritis virus EAV cell culture The RNA synthesis two distantly related nidoviruses catalyze RNA dependent RNA polymerase RdRp core enzyme multiprotein replication transcription complex RTC Using activity assay RTCs isolate cell infected SARS CoV EAV—thus eliminate need transport across plasma membrane—we show efficiently inhibits RNA synthesize activity RTCs viruses Enzymatic study use recombinant RdRps SARS CoV nsp12 EAV nsp9 purify coli subsequently reveal directly inhibit vitro activity nidovirus polymerases More specifically find block initiation step EAV RNA synthesis whereas case SARS CoV RdRp elongation inhibit template binding reduce chelate MgEDTA inhibitory effect divalent cation could reverse provide novel experimental tool vitro study molecular detail nidovirus replication transcription',\n",
       "  0.5303453803062439),\n",
       " ('amcase', 0.5301969051361084),\n",
       " ('Coronavirus nucleocapsid protein assemble constitutively high molecular oligomers Coronaviruses CoV envelop virus rely nucleocapsid protein incorporate positive strand genomic RNA virion CoV proteins form oligomers mechanism relevance underlying multimerization remain fully understood Using vitro pull experiment density glycerol gradient find least region distribute entire length mediate self interaction mouse hepatitis virus MHV severe acute respiratory syndrome coronavirus SARS CoV protein The fact region bind reciprocally provide possible molecular basis protein oligomerization Interestingly cytoplasmic molecule MHV infect cell constitutively assemble oligomers process require bind genomic RNA Based data propose model constitutive protein oligomerization allow optimal load genomic viral RNA ribonucleoprotein complex via presentation multiple viral RNA binding motif',\n",
       "  0.5301036834716797),\n",
       " ('Identification Hepatotropic Viruses Plasma Using Deep Sequencing Next Generation Diagnostic Tool conduct unbiased metagenomics survey use plasma patient chronic hepatitis chronic hepatitis autoimmune hepatitis AIH non alcoholic steatohepatitis NASH patient without liver disease control RNA DNA library sequence plasma filtrate enrich viral particle catalog virus population Hepatitis virus readily detect high coverage patient chronic viral hepatitis limit number sequence resemble virus find The exception library patient diagnose hepatitis virus HCV infection contain multiple sequence match virus GBV Abundant GBV read also find plasma patient AIH whereas Torque teno virus TTV find high frequency sample patient AIH NASH After taxonomic classification sequence BLASTn substantial fraction library range remain unclassified These unknown sequence assemble scaffold along virus phage endogenous retrovirus sequence analyze BLASTx non redundant protein database Nearly full genome heretofore unknown circovirus assemble many scaffold encode protein similarity plant insect mammalian viruses The presence novel circovirus confirm PCR BLASTx also identify many polypeptide resemble nucleo cytoplasmic large DNA virus NCLDV proteins evaluate alignment profile hidden Markov method HHblits observe inconsistency target protein report different algorithm This suggest sequence alignment insufficient identify NCLDV protein especially alignment small portion target protein Nevertheless establish reliable protocol identification virus plasma also adapt patient sample urine bile saliva body fluid',\n",
       "  0.5300989747047424),\n",
       " ('fluorophores', 0.5300142765045166),\n",
       " ('Differential Antigen Recognition Cells Spleen Central Nervous System Coronavirus Infected Mice CD8 cytotoxic lymphocytes CTLs isolated central nervous system CNS C57Bl mouse acutely infect mouse hepatitis virus strain JHM MHV JHM analyze directex vivocytotoxicity assay recognize two epitope 2Db 2Kb restrict encompass amino acid 510–518 598–605 respectively within surface glycoprotein contrast CD8 cell isolate spleens mouse inoculate intraperitoneally MHV JHM restimulatedin vitroonly respond 2Db restrict epitope report preferential recognition 2Db restrict epitope confirm use splenocytes stimulatedin vitrowith either MHV JHM infect MC57 cell cell line express protein analyze secondary CTL assay determine whether result represent difference epitope recognition spleen CNS secondary CTL assay perform use spleen cell coat peptide encompass CTL epitopes stimulators Under condition epitopes sensitize cell lysis spleen derive CTLs suggest epitope recognize splenic CD8 cell infectionin vivo Furthermore limit dilution analysis indicate precursor frequency splenic CD8 cell specific 2Kb 2Db restrict epitope significantly different Thus result suggest thatin vitrostimulation splenocytes specific 2Kb restrict epitope inefficient endogenous processing inefficiency correct peptide provide exogenously sufficiently high concentration consequence result also show cell responsive previously identify CNS derive CD8 cell epitopes present infect spleen nearly frequency',\n",
       "  0.5298395156860352),\n",
       " ('Further Characterization Coronavirus Infectious Bronchitis Virus like Proteinase Determination New Cleavage Site Coronavirus infectious bronchitis virus IBV encode trypsin like proteinase like proteinase ORF demonstrate play pivotal role proteolytic processing gene encode polyproteins previous study proteinase identify kDa protein IBV infect cell catalytic center show consist H2820 C2922 residues release polyproteins autoprocessing two Q–S dipeptide bond Q2779–S2780 Q3086–S3087 report characterization two cleavage site demonstrate terminal Q2779–S2780 site tolerant mutation position Deletion terminal region proteinase show significant amount enzymatic activity maintain upon deletion amino acid suggest extreme terminal region may dispensable proteolytic activity proteinase Analysis autoprocessing kinetics vitro reveals proteolysis Q2779–S2780 site first cleavage event mediate proteinase This follow cleavage Q3086–S3087 site The occurrence cleavage event intact cell potentially rapid efficient intermediate cleavage product cover proteinase detect either IBV infect transfected cell Immunofluorescence microscopy subcellular fractionation study show differential subcellular localization proteinase IBV infect cell cell express like proteinase alone indicate additional role viral replication might play protein Finally Q–A Q3379–A3380 dipeptide bond encode nucleotide demonstrated cleavage site proteinase',\n",
       "  0.5298306941986084),\n",
       " ('The Spike Not Hemagglutinin Esterase Protein Bovine Coronavirus Necessary Sufficient Viral Infection The spike hemagglutinin esterase bovine coronavirus BCV two envelope protein recognize receptor determinant acetylneuraminic acid host cell However precise relative role two protein BCV infectivity remain elusive unequivocally determine role viral cytopathogenicity develop system phenotypically chimeric virus generate infect closely relate mouse hepatitis virus MHV cell stably express individual BCV protein The chimeric virus use infect human rectal tumor HRT cell permissive BCV nonsusceptible MHV Using approach find chimeric virus contain BCV protein virion surface enter replicated HRT cell specifically block prior treatment virus neutralize antibody specific BCV protein indicate BCV protein responsible initiate chimeric virus infection contrast chimeric virus contain biologically active functional BCV protein surface fail enter HRT cell indicate BCV protein alone sufficient BCV infection Taken together result demonstrate protein protein BCV necessary sufficient infection chimeric virus HRT cell suggest BCV likely protein primary vehicle infect permissive cell',\n",
       "  0.5294029116630554),\n",
       " ('rsps', 0.5292179584503174),\n",
       " ('Activity purify His tagged like proteinase coronavirus infectious bronchitis virus Previous study vitro process clone polyprotein fragment coronavirus infectious bronchitis virus IBV large open reading frame ORF1 confirm activity predict like proteinase 3CLP domain suggest proteinase release autocatalytically polyprotein form kDa protein 3CLpro capable cleavage trans order identify cleavage within ORF1 polyprotein mediate 3CLpro proteinase express bacteria purify used trans cleavage assay polyprotein fragment lack 3CLP domain target The proteinase express polyprotein fragment able process expression bacterial cell release mature 3CLpro histidine His6 tag introduce close terminus proteinase aid purification Processing demonstrate tagged proteinase indistinguishable wild type enzyme indicate site choose tag permissive From study able demonstrate trans cleavage consistent use previously predict identify site within open reading frame gene This tentatively complete process map ORF1 region respect 3CLpro',\n",
       "  0.5292150378227234),\n",
       " ('hjabbari', 0.5292090773582458),\n",
       " ('virus–a59', 0.5291567444801331),\n",
       " ('siRNAs target terminal sequence SARS associate coronavirus membrane gene inhibit protein expression degradation mRNA SARS associate coronavirus SCoV protein play key role viral assembly bud Recent study reveal protein could interact protein Golgi complex study show SCoV protein localize Golgi apparatus Golgi vector marker study protein function three candidate small interfere RNAs siRNAs correspond gene sequence design transcribed vitro test ability silence protein expression The plasmid pEGFP encode SCoV protein fusion protein EGFP use silence reporter gene detection HEK 293T cell transfected siRNA construct The result show mean green fluorescence intensity RNA transcripts significantly reduce expression glycoprotein strongly inhibit cell transfected specific siRNAs These finding demonstrate three specific siRNAs able specifically effectively inhibit glycoprotein expression culture cell block accumulation mRNA provide approach study function protein development novel prophylactic therapeutic agent SCoV infection',\n",
       "  0.5291098952293396),\n",
       " ('sylvaticus', 0.5291059017181396),\n",
       " ('emt6', 0.5288126468658447),\n",
       " ('Hepatitis Virus Core Protein Promotes miR Destabilization Inhibiting GLD The liver specific microRNA miR essential role liver development metabolism key proviral factor hepatitis virus HCV Despite crucial role liver HCV life cycle little known molecular mechanism miR expression regulation HCV infection Here show HCV core protein downregulates abundance miR promote destabilization via inhibition GLD non canonical cytoplasmic poly polymerase The decrease miR expression result dysregulation know function miR include proviral activity HCV high throughput sequence small RNAs human liver biopsy find nucleotide prototype miR modify end terminal non templated templated nucleotide addition Remarkably proportion miR isomer bear single nucleotide tail ribonucleotide decrease liver specimens patient HCV find single nucleotide tail miR isomer display increase miRNA activity stability prototype miR terminal extension catalyze unique terminal nucleotidyl transferase activity GLD capable add single ribonucleotide without preference adenylate miR end The HCV core protein specifically inhibit GLD interaction GLD cytoplasm find responsible miR downregulation Collectively result provide new insight regulatory role HCV core protein control viral RNA abundance miR function miR stability modulation',\n",
       "  0.5288033485412598),\n",
       " ('subdomains', 0.5285819172859192),\n",
       " ('vitrocleavage', 0.528495192527771),\n",
       " ('The novel hemagglutinin esterase gene human torovirus Breda virus Human torovirus HTV Breda virus BRV member genus torovirus family Coronaviridae establish infectious agent human cattle respectively The hemagglutinin esterase gene Breda virus serotype BRV identify nucleotide sequence BRV serotype BRV genome contains open read frames viral structural protein report reveal presence gene whose nucleotide sequence identical BRV gene study amplify 2kb gene HTV genome use long PCR sequence amplicon directly nucleotide level HTV gene manifest sequence identity gene BRV BRV identity pseudogene sequence BEV The amplicons contain gene BRV HTV clone express baculovirus system proteins purified sodium dodecyl sulphate polyacrylamide gel electrophoresis Hyperimmune serum prepare guinea pig proteins reactive bovine torovirus BTV human torovirus HTV antigen immunoblot react specifically kDa protein correspond size torovirus protein Furthermore hyperimmune serum preimmune serum react series BTV positive HTV positive fecal specimen immunoblot dot blot analysis immunoelectron microscopy IEM torovirus particle BTV positive specimen calf diarrhea HTV positive specimen patient aggregate hyperimmune serum Human convalescent serum gnotobiotic calf post infection serum react immunoblot express kDa protein The express protein HTV important diagnostic potential',\n",
       "  0.5283827781677246),\n",
       " ('Bacterial Artificial Chromosomes Functional Genomics Tool Study Positive strand RNA Viruses Reverse genetics approach rescue infectious virus entirely clone cDNA revolutionized field positive strand RNA viruses whose genome polarity cellular mRNA The cDNA base reverse genetics system seminal method enables direct manipulation viral genomic RNA thereby generate recombinant virus molecular genetic study viral RNA element gene product viral replication pathogenesis also provide valuable platform allow development genetically define vaccine viral vector delivery foreign gene For many positive strand RNA viruses Japanese encephalitis virus JEV however clone cDNAs unstable pose major obstacle construction propagation functional cDNA Here present report describe strategic consideration create amplify genetically stable full length infectious JEV cDNA bacterial artificial chromosome BAC use follow general experimental procedure viral RNA isolation cDNA synthesis cDNA subcloning modification assembly full length cDNA cDNA linearization vitro RNA synthesis virus recovery This protocol provide general methodology applicable clone full length cDNA range positive strand RNA viruses particularly genome length BAC vector infectious RNAs transcribe vitro bacteriophage RNA polymerase',\n",
       "  0.5283650159835815),\n",
       " ('221–330', 0.5283218622207642),\n",
       " ('Sensitive Assay Virus Discovery Respiratory Clinical Samples 5–40 respiratory infection child diagnostics remain negative suggest patient might infect yet unknown pathogen Virus discovery cDNA AFLP VIDISCA virus discovery method base recognition restriction enzyme cleavage sit ligation adaptor subsequent amplification PCR However direct discovery unknown pathogen nasopharyngeal swab difficult due high concentration ribosomal RNA rRNA act competitor current study optimize VIDISCA adjust reverse transcription enzyme decrease rRNA amplification reverse transcription use hexamer oligonucleotides anneal rRNA Residual cDNA synthesis rRNA template reduce oligonucleotides anneal rRNA extend due dideoxy modification With modification reduction rRNA amplification establish Further improvement VIDISCA sensitivity obtain high throughput sequence VIDISCA Eighteen nasopharyngeal swab analyse contain know respiratory virus could identify proper virus majority sample test The median load VIDISCA positive sample viral genome copy range E3–7 Our result show optimization VIDISCA subsequent high throughput sequence enhances sensitivity drastically provide opportunity perform virus discovery directly patient material',\n",
       "  0.5282831192016602),\n",
       " ('p459', 0.5282214879989624),\n",
       " ('Identification analysis gag pol ribosomal frameshift site feline immunodeficiency virus The pol gene retrovirus translate gag pol fusion proteins ribosomal frameshifting within gag pol overlap region During ribosomal frameshift event gag open reading frame shift allow phase read pol open reading frame consensus frameshift signal sequence GGGAAAC within gag pol overlap region feline immunodeficiency virus FIV identify follow sequence potential pseudoknot tertiary structure Using recombinant baculoviruses frameshift occur efficiently consensus sequence show site frameshift event mutation create termination codon downstream putative frameshift signal sequence upstream potential pseudoknot structure make short gag product affect efficiency frameshifting mutation create termination codon upstream putative frameshift signal make short product essentially abrogate frameshifting Mutations first stem second stem potential pseudoknot structure severely reduce frameshifting efficiency Mutations alter length frameshift signal pseudoknot structure call spacer region also reduce frameshift efficiency The insertion palindromic sequence could form hairpin structure upstream frameshift signal sequence also affect frameshifting These result support view ribosomal frameshift event FIV gag pol region involve identify signal sequence appear require precisely position downstream sequence indicate pseudoknot structure efficient frameshifting',\n",
       "  0.5281776189804077),\n",
       " ('hku7', 0.5281100273132324),\n",
       " ('Tale Two RNAs Viral Infection How Viruses Antagonize mRNAs Small Non Coding RNAs The Host Cell Viral infection initiate array change host gene expression Many viruses dampen host protein expression attempt evade host anti viral defense machinery Host gene expression suppress several stage host messenger RNA mRNA formation include selective degradation translationally competent messenger RNAs Besides mRNAs host cell also express variety noncoding RNAs include small RNAs may also subject inhibition upon viral infection review focus different way virus antagonize cod noncoding RNAs host cell advantage',\n",
       "  0.5279824733734131),\n",
       " ('146–179', 0.5279647707939148),\n",
       " ('h298', 0.5279140472412109),\n",
       " ('Frameshift mutation infectious cDNA clone Citrus tristeza virus strategy minimize toxicity viral sequence Escherichia coli The advent reverse genetics revolutionize study positive strand RNA virus amenable clone cDNAs high copy number plasmid Escherichia coli However virus inherently refractory clone high copy number plasmid due toxicity viral sequence coli report strategy compromise infectivity RNA transcripts toxicity coli effect introduce frameshift mutation slippery sequence near viral toxicity sequence viral cDNA Citrus tristeza virus CTV cDNA sequence toxic coli The original full length infectious cDNA CTV derivative replicon CTV ΔCla clone pUC119 result unusually limited coli growth However upon sequence cDNAs additional uridinylate find stretch result change reading frame stop codon Yet vitro produce RNA transcript clone infect protoplast result progeny virus repair Correction frameshift mutation CTV cDNA construct result increased infectivity vitro produce RNA transcript also cause substantial increase toxicity coli require day develop visible colony Frameshift mutation create sequence suspect facilitate reading frame shift silent mutation introduce oligo region result complete loss infectivity suggest oligo region facilitate repair frameshift mutation Additional frameshift mutation introduce oligo region also result transcript reduce infectivity similarly original clone insertion However frameshift mutation introduce oligo region near toxicity region improve growth stability coli These data demonstrate host sufficiently susceptible infection transcript reduce specific infectivity introduction frameshift mutation slippery sequence near toxic region viral cDNAs use additional strategy clone recalcitrant viral sequence high copy number plasmid reverse genetics',\n",
       "  0.527848482131958),\n",
       " ('w86', 0.5278072357177734),\n",
       " ('96±0', 0.5277520418167114),\n",
       " ('microplus', 0.5276356935501099),\n",
       " ('The Nsp12 cod region type PRRSV require viral subgenomic mRNA synthesis one many nonstructural protein porcine reproductive respiratory syndrome virus PRRSV nonstructural protein Nsp12 receive relatively little attention role virus replication essentially unknown application reverse genetic manipulation infectious PRRSV clone current study first demonstrate Nsp12 key component PRRSV replication addition biochemical property Nsp12 evaluate reveal Nsp12 form dimer expose oxidative condition Furthermore systemically analyze function Nsp12 PRRSV RNA synthesis use strand specific PCR method surprise Nsp12 find involve minus strand genomic RNA gRNA synthesis importantly result indicate Nsp12 involve synthesis plus minus strand subgenomic mRNAs sgmRNA sgmRNA Finally find combination cysteine cysteine Nsp12 require sgmRNA synthesis knowledge first report biological role Nsp12 PRRSV lifecycle conclude Nsp12 involve synthesis sgRNA sgRNA',\n",
       "  0.5275965929031372),\n",
       " ('alphatetraviruses', 0.527292788028717),\n",
       " ('Phosphatidic Acid Produced Phospholipase Promotes RNA Replication Plant RNA Virus Eukaryotic positive strand RNA RNA viruses intracellular obligate parasite replicate use membrane bound replicase complex contain multiple viral host component replicate RNA virus exploit host resource modify host metabolism membrane organization Phospholipase PLD phosphatidylcholine phosphatidylethanolamine hydrolyze enzyme catalyzes production phosphatidic acid lipid second messenger modulates diverse intracellular signal various organism normally present small amount less total phospholipid rapidly transiently accumulate lipid bilayers response different environmental cue biotic abiotic stress plant However precise function PLD remain unknown Here report role PLD genomic RNA replication plant RNA virus Red clover necrotic mosaic virus RCNMV find RCNMV RNA replication complex form Nicotiana benthamiana contain PLDα PLDβ Gene silence pharmacological inhibition approach show PLDs PLDs derive require viral RNA replication Consistent exogenous application enhance viral RNA replication plant cell plant derive cell free extract also find viral auxiliary replication protein bound vitro amount increase RCNMV infect plant leave Together finding suggest RCNMV hijack host produce enzyme replicate',\n",
       "  0.5272903442382812),\n",
       " ('s1–s2', 0.5272515416145325),\n",
       " ('uridine', 0.5272179245948792),\n",
       " ('Isolation Characterization Novel Gammaherpesvirus Microbat Cell Line While employ deep sequencing novo assembly characterize mRNA transcript profile cell line derive microbat Myotis velifer incautus serendipitously identify mRNAs encode protein high level identity herpesviruses majority closely related protein equine herpesvirus EHV horse gammaherpesvirus demonstrate electron microscopy presence herpesvirus like particle microbat cell Passage supernatant microbat cell Vero cell result syncytium formation expression viral gene amplification viral DNA demonstrate quantitative PCR Susceptibility human cell line productive infection also demonstrate Next generation sequence novo assembly viral genome supernatant Vero cell yield single contig approximately least open read frame ORFs predict microRNAs miRNAs gammaherpesvirus genomic organization Phylogenic analysis envelope glycoprotein DNA polymerase POLD1 reveal similarity multiple gammaherpesviruses include yet uncultured virus Rhadinovirus genus obtain deep sequencing bat tissue Moreover assemble genome reveal ORFs share little homology know ORFs EHV similar accessory protein gammaherpesviruses Some also striking homology predict Myotis bat proteins Cumulatively study provide first isolation characterization replication competent bat gammaherpesvirus IMPORTANCE Bats significant interest reservoir zoonotic viral pathogen however tool dissect bat virus interaction limited availability This study serendipitously identify establish bat cell line fully replication competent gammaherpesvirus determine complete genome sequence virus generate viral transcript map This virus replicate select human nonhuman primate cell line However analyse viral sequence support bat origin virus therefore refer virus bat gammaherpesvirus BGHV8 The viral genome contains unique open reading frame likely encode modulators bat innate adaptive immune signal pathway express viral miRNAs The virus gene product provide unique tool dissect bat gammaherpesvirus biology',\n",
       "  0.5271943211555481),\n",
       " ('sec7', 0.5270373225212097),\n",
       " ('hphi', 0.5270214080810547),\n",
       " ('hbaa', 0.5269320011138916),\n",
       " ('inclination', 0.5269190669059753),\n",
       " ('neodebliecki', 0.5266479253768921),\n",
       " ('mcnemar', 0.5266226530075073),\n",
       " ('Alphavirus Nucleocapsid Packaging Assembly Alphavirus nucleocapsids assemble cytoplasm infected cell copy capsid protein approximately positive strand genomic RNA However challenge capsid specifically select RNA package assemble around remain elusive one solve review summarize know alphavirus capsid protein package signal role mechanism packaging assembly review discovery packaging signal much evidence well requirement specify packaging genomic RNA Finally compare model viral system include particular reference relatively new idea RNA packaging base presence multiple minimal packaging signal throughout genome know two stage mechanism This review provide basis investigate fundamental way RNA viruses able select cargo relative chaos cytoplasm',\n",
       "  0.5264426469802856),\n",
       " ('prosomatostatin', 0.5263974666595459),\n",
       " ('aca', 0.5263134241104126),\n",
       " ('fkbp52', 0.5260968208312988),\n",
       " ('499–600', 0.5259544849395752),\n",
       " ('gyrus', 0.5258982181549072),\n",
       " ('pyridoxine', 0.5258898138999939),\n",
       " ('RNA Binding Domain Nucleocapsid Protein Gill Associated Nidovirus Penaeid Shrimp Gill associate virus GAV infect Penaeus monodon shrimp type specie okavirus Roniviridae invertebrate nidoviruses know currently Electrophoretic mobility shift assay EMSAs use His tag full length truncate protein employ examine nucleic acid bind property GAV nucleocapsid protein vitro The EMSAs show full length protein bind synthetic single strand RNAs test independent sequence The ssRNAs include sense region GAV genome well sense region RNA segment Mourilyan virus crustacean bunya like virus GAV protein also bind double stranded RNAs prepare GAV ORF1b gene region bacteriophage M13 genomic ssDNA EMSAs use five protein construct variable length terminal terminal truncation localize RNA bind domain amino acid terminal sequence span Met Arg Similarly RNA bind protein first portion sequence proline arginine rich examine domain detail peptide PVRRPLPPQPPRNARLI encompass sequence synthesize find bind nucleic acid similarly full length protein EMSAs The data indicate fundamental role GAV protein proline arginine rich domain nucleate genomic ssRNA form nucleocapsids Moreover synthetic peptide form high order complex presence RNA domain might also play role protein protein interaction stabilize helical structure GAV nucleocapsids',\n",
       "  0.5258087515830994),\n",
       " ('mhvr', 0.5257823467254639),\n",
       " ('Transcriptional Translational Landscape Equine Torovirus The genus Torovirus subfamily Torovirinae family Coronaviridae order Nidovirales encompass range specie infect domestic ungulate include cattle sheep goat pig horse cause acute self limit gastroenteritis Using prototype specie equine torovirus EToV perform parallel RNA sequence RNA seq ribosome profile Ribo seq analyze relative expression level know torovirus proteins transcript chimeric sequence produce via discontinuous RNA synthesis characteristic nidovirus replication cycle change host transcription translation result EToV infection RNA sequencing confirm EToV utilizes unique combination discontinuous nondiscontinuous RNA synthesis produce subgenomic RNAs sgRNAs indeed identify transcript arise mechanism would result sgRNAs encode nucleocapsid Our ribosome profiling analysis reveal ribosome efficiently translate two novel CUG initiate open reading frame ORFs locate within call untranslated region term result protein Comparative genomic analysis confirm ORFs conserve across available torovirus sequence infer amino acid sequence subject purify selection indicate functionally relevant This study provide first high resolution analysis transcription translation neglect group livestock pathogens IMPORTANCE Toroviruses infect cattle goat pig horse worldwide cause gastrointestinal disease There treatment vaccine ability spill human assess These virus relate important human pathogen include severe acute respiratory syndrome SARS coronavirus share common feature however mechanism use produce sgRNA molecule differs Here perform deep sequencing determine equine torovirus produce sgRNAs also identify two previously unknown open reading frame hidden within genome Together result highlight similarity difference domestic animal virus relate pathogen human livestock',\n",
       "  0.5257335901260376),\n",
       " ('Expression antiviral protein Lonomia obliqua hemolymph baculovirus insect cell system The control viral infection mainly cause influenza virus great interest Public Health Several study show presence active property hemolymph arthropod interest development new pharmacological drug Recently demonstrate existence potent antiviral property hemolymph Lonomia obliqua caterpillars The aim study produce antiviral protein baculovirus Sf9 cell system The result bacmid contains sequence cod antiviral protein previously describe group Total RNA obliqua caterpillars extracted Trizol use reverse transcription assay oligo primer follow polymerase chain reaction PCR specific primer cDNA cod antiviral protein base sequence deposit GenBank database Restriction sit insert cDNA ligation donor plasmid pFastBac1™ The recombinant plasmid select Escherichia coli DH5α subsequently use transformation coli DH10Bac construction recombinant bacmid This bacmid use expression antiviral protein baculovirus Sf9 cell system After identify protein western blot activity test perform show purified recombinant protein able significantly reduce viral replication logs Studies optimization expression system production antiviral protein insect cell progress',\n",
       "  0.5256863236427307),\n",
       " ('Rapid Genome Sequencing RNA Viruses develop system rapid determination viral RNA sequence whereby genomic sequence obtain culture virus isolates without subcloning plasmid vector This method afford new opportunity address challenge unknown untypeable emerging virus',\n",
       "  0.5255842208862305),\n",
       " ('Composition bias genome polarity RNA viruses observe relationship content cod sequence RNA viruses genome polarity Positive strand RNA virus significantly high content negative strand RNA virus Coding sequence negative strand RNA virus bias toward high coding strand high genome two distinct pattern observe among positive strand RNA genome This finding suggest RNA virus different genome polarity different mutational pressure may consequence difference strategy viral genome expression replication The content directly affect viral codon adaptation index use highly express human gene reference set may theoretically predict efficiency viral gene expression human cell',\n",
       "  0.5255557298660278),\n",
       " ('Identification linear epitope capsid protein classical swine fever virus The capsid protein Classical swine fever virus CSFV propose play essential role replication translation viral RNA study monoclonal antibody mAb direct protein generate recombinant protein express Escherichia coli immunogen IFA IPMA analysis show native protein CSFV virion reactive mAb truncate protein identify linear epitope recognize mAb correspond amino acid 61TQDGLYHNKN70 CSFV protein well conserve among pestiviruses Laser confocal analysis show protein mainly locate cellular nucleoplasm nucleolus cell The result implication study CSFV replication',\n",
       "  0.5254415273666382),\n",
       " ('absentes', 0.525312066078186),\n",
       " ('10–30', 0.5252143144607544),\n",
       " ('thr', 0.5251972675323486),\n",
       " ('58–72', 0.5251944065093994),\n",
       " ('sub', 0.5251724123954773),\n",
       " ('cd2v', 0.5251258611679077),\n",
       " ('Isolated protein hemagglutinate encephalomyelitis virus bovine coronavirus receptor destroy receptor binding activity Bovine coronavirus BCV hemagglutinate encephalomyelitis virus HEV swine find grow high titer MDCK cell subline Madin Darby canine kidney cell Virus grown cell use isolate purify protein This protein show recently acetylesterase activity function receptor destroy enzyme BCV Here show HEV contain enzyme The glycoprotein solubilized treatment virion octylglucoside Following centrifugation sucrose gradient surface protein hemagglutinin esterase obtain purified form After removal detergent dialysis form rosettes show electron microscopy The purify protein retain acetylesterase activity able function receptor destroy enzyme render red blood cell resistant agglutination coronaviruses protein release viral membrane fail agglutinate red blood cell However find recognize glycoconjugates contain acetyl acetylneuraminic acid indicate bind assay rat serum protein blot nitrocellulose ability inhibit hemagglutinate activity BCV HEV influenza virus The purified enzyme provide useful tool analyze cellular receptor coronaviruses',\n",
       "  0.525054931640625),\n",
       " ('Programmed ribosomal frameshifting decode SARS CoV genome Programmed ribosomal frameshifting essential mechanism use expression orf1b coronaviruses Comparative analysis frameshift region reveals universal shift site UUA AAC follow predict downstream RNA structure form either pseudoknot kiss stem loop Frameshifting SARS CoV characterize cultured mammalian cell use dual luciferase reporter system mass spectrometry Mutagenic analysis SARS CoV shift site mass spectrometry affinity tag frameshift product confirm tandem tRNA slippage sequence UUA AAC Analysis downstream pseudoknot stimulator frameshifting SARS CoV show propose RNA secondary structure loop two unpaired nucleotide stem I–stem junction SARS CoV important frameshift stimulation These result demonstrate key sequence require efficient frameshifting utility mass spectrometry study ribosomal frameshifting',\n",
       "  0.5249651670455933),\n",
       " ('Inhibition Hepatitis Virus Spread Natural Compound Silvestrol Every year Mio hepatitis virus HEV infection death associate HEV worldwide present exists specific therapy HEV The natural compound silvestrol potent antiviral effect strand RNA virus Ebola virus also strand RNA virus Corona Picorna Zika virus The inhibitory effect virus spread due inhibition DEAD box RNA helicase eIF4A require unwind structured untranslated region UTRs This lead impaired translation viral RNA The HEV strand RNA genome contain cap short UTR This study aim analyze impact silvestrol HEV life cycle Persistently infect A549 cell instrumental This study identify silvestrol potent inhibitor release HEV infectious viral particle This along strongly reduce HEV capsid protein translation retention viral RNA inside cytoplasm without major cytotoxic effect Interestingly parallel silvestrol affect activity antiviral major vault protein MVP translocation cytoplasm perinuclear membrane These data characterize complex antiviral activity silvestrol show silvestrol broad spectrum function since HEV virus without complex secondary structure genome still affect',\n",
       "  0.5248839855194092),\n",
       " ('flatter', 0.5247159004211426),\n",
       " ('Arterivirus Nsp1 Modulates Accumulation Minus Strand Templates Control Relative Abundance Viral mRNAs The gene expression plus strand RNA viruses polycistronic genome depend translation replication genomic mRNA well synthesis subgenomic mRNAs Arteriviruses coronaviruses distantly related member nidovirus order employ unique mechanism discontinuous minus strand RNA synthesis generate subgenome length templates synthesis nest set mRNAs Non structural protein nsp1 arterivirus equine arteritis virus EAV multifunctional regulator viral RNA synthesis virion biogenesis previously implicate control balance genome replication mRNA synthesis Here employ reverse forward genetics gain insight multiple regulatory role nsp1 Our analysis reveal relative abundance viral mRNAs tightly control intricate network interaction involve nsp1 subdomains Distinct nsp1 mutation affect quantitative balance among viral mRNA specie data implicate nsp1 control accumulation full length subgenome length minus strand templates viral mRNA synthesis The moderate differential change viral mRNA abundance nsp1 mutant result similarly altered viral protein level progeny virus yield greatly reduce Pseudorevertant analysis provide compelling genetic evidence balance EAV mRNA accumulation critical efficient virus production This first report protein mediate mRNA specific control nidovirus RNA synthesis reveals existence integral control mechanism fine tune replication mRNA synthesis virus production establishes major role nsp1 coordinate arterivirus replicative cycle',\n",
       "  0.524560809135437),\n",
       " ('Identification Hepta Octo Uridine stretch sole signal program ribosomal frameshifting translation SARS CoV ORF variant Programmed frameshifting one translational recode mechanism read genetic code alternative way This process generally program signal define location specific mRNA study report identification hepta octo uridine stretch sole signal program ribosomal frameshifting translation severe acute respiratory syndrome coronavirus SARS CoV ORF variant SARS CoV ORF encodes minor structural protein amino acid Over course clone expression gene mixed population clone six seven eight nine stretch locate downstream initiation codon find vitro vivo expression clone six seven eight respectively show detection full length protein Mutagenesis study lead identification hepta octo uridine stretch slippery sequence efficient frameshifting Interestingly stimulatory element find sequence upstream downstream slippage site When hepta octo uridine stretch use replace original slippery sequence SARS CoV ORF efficient frameshift event observe Furthermore efficiency frameshifting mediate hepta octo uridine stretch affect mutation introduce downstream stem–loop structure totally abolish frameshift event mediate original slippery sequence ORF Taken together study identifies hepta octo uridine stretch function sole element efficient ribosomal frameshift event',\n",
       "  0.5244678854942322),\n",
       " ('titres', 0.524310827255249),\n",
       " ('c13', 0.524169921875),\n",
       " ('beushausen', 0.5240719318389893),\n",
       " ('pca236', 0.5240517854690552),\n",
       " ('phones', 0.5239238739013672),\n",
       " ('pelletron', 0.5239186882972717),\n",
       " ('machamer', 0.5238873958587646),\n",
       " ('omentum', 0.5237641334533691),\n",
       " ('hepated', 0.5237078070640564),\n",
       " ('tax1bp1', 0.523363471031189),\n",
       " ('75μg', 0.5233572125434875),\n",
       " ('obligatorily', 0.5232598781585693),\n",
       " ('saccharopolyspora', 0.5230861306190491),\n",
       " ('Identification Cis Acting Elements Positive Strand Subgenomic mRNA Required Synthesis Negative Strand Counterpart Bovine Coronavirus demonstrate addition genomic RNA sgmRNA able serve template synthesis negative strand strand complement However cis act element positive strand strand sgmRNA require strand sgmRNA synthesis yet systematically identify study employ real time quantitative reverse transcription polymerase chain reaction analyze act element bovine coronavirus BCoV sgmRNA required synthesis strand counterpart deletion mutagenesis The major finding follow Deletion terminal leader sequence sgmRNA decreased synthesis strand sgmRNA complement Deletions untranslated region UTR bulge stem loop show effect strand sgmRNA synthesis however deletion UTR pseudoknot decrease yield strand sgmRNA Nucleotides position −15 −34 sgmRNA terminal region require efficient strand sgmRNA synthesis Nucleotide specie position sgmRNA correlated efficiency strand sgmRNA synthesis These result together suggest principle terminal sequence sgmRNA harbor act element critical efficient strand sgmRNA synthesis BCoV',\n",
       "  0.5229920148849487),\n",
       " ('gastroenterologist', 0.5229912996292114),\n",
       " ('harger', 0.5229461193084717),\n",
       " ('rs3747811at', 0.5228817462921143),\n",
       " ('site1', 0.522588312625885),\n",
       " ('dieckol', 0.522527813911438),\n",
       " ('Viral Discovery Sequence Recovery Using DNA Microarrays Because constant threat pose emerge infectious disease limitation exist approach use identify new pathogen great demand new technological method viral discovery describe herein DNA microarray base platform novel virus identification characterization Central approach DNA microarray design detect wide range know virus well novel member exist viral family microarray contain highly conserved 70mer sequence every fully sequence reference viral genome GenBank During outbreak severe acute respiratory syndrome SARS March hybridization microarray reveal presence previously uncharacterized coronavirus viral isolate cultivate SARS patient characterize new virus approximately unknown virus genome clone physically recover viral sequence hybridize individual array element Sequencing fragment confirm virus indeed new member coronavirus family This combination array hybridization follow direct viral sequence recovery prove general strategy rapid identification characterization novel virus emerge infectious disease',\n",
       "  0.5224478244781494),\n",
       " ('Amino acid substitution insertion spike glycoprotein extend host range murine coronavirus MHV A59 The murine coronavirus murine hepatitis virus MHV limit infection susceptible mouse murine cell line specificity spike glycoprotein receptor murine carcinoembryonic antigen cell adhesion molecule mCEACAM1a recently show substitution insert terminal region associate extend host range virus variant derive murine cell persistently infect A59 strain MHV MHV A59 use target RNA recombination TRR generate isogenic virus differ MHV A59 substitution insert Only viruses substitution insert infect hamster feline monkey cell These virus also infect murine cell presence block anti mCEACAM1a antibody Thus relatively change terminal region sufficient permit MHV A59 interact alternative receptor murine non murine cell',\n",
       "  0.5224406123161316),\n",
       " ('littermates', 0.5223180055618286),\n",
       " ('PTB PAB proteins bind untranslated region Norwalk virus genomic RNA Noroviruses human enteric caliciviruses cell culture available Consequently mechanisms factor involve replication difficult study attempt analyze trans act factor could role replication untranslated region genome study Use Zuker mfold software predict 3′UTR contains stem loop structure Proteins HeLa cell extract PTB form stable complexes region The addition poly tail 3′UTR permit specific bind poly binding protein PABP present HeLa cell extract well recombinant PABP Since PTB PABP important trans act factor require viral translation replication RNA–protein interaction may play role replication translation',\n",
       "  0.5222699642181396),\n",
       " ('Rapid preparation plasma membrane avian lymphoid cell fibroblasts virus bind study simple rapid protocol preparation plasma membrane chicken embryo fibroblast chicken lymphoid cell develop Characterization preparation morphological biochemical serological method indicate specific enrichment plasma membrane well cell surface proteins Binding infectious bursal disease virus IBDV particle demonstrate immobilization plasma membrane cell type specific difference observe Although result study reflect interaction IBDV isolate cell partially advantage plasma membrane preparation specific enrichment cell surface proteins constant quality possibility store aliquots several month make useful tool virus bind study avian cell',\n",
       "  0.5222088098526001),\n",
       " ('The transcriptional translational landscape equine torovirus The genus Torovirus subfamily Torovirinae family Coronaviridae order Nidovirales encompass range specie infect domestic ungulate include cattle sheep goat pig horse cause acute self limit gastroenteritis Using prototype specie equine torovirus EToV perform parallel RNA sequence RNA seq ribosome profile Ribo seq analyse relative expression level know torovirus proteins transcript chimaeric sequence produce via discontinuous RNA synthesis characteristic nidovirus replication cycle change host transcription translation result EToV infection RNA sequencing confirm EToV utilises unique combination discontinuous non discontinuous RNA synthesis produce subgenomic RNAs indeed identify transcript arise mechanism would result sgRNAs encode nucleocapsid Our ribosome profiling analysis reveal ribosome efficiently translate two novel CUG initiate ORFs locate within call UTR term result protein Comparative genomic analysis confirm ORFs conserve across available torovirus sequence infer amino acid sequence subject purify selection indicate functionally relevant This study provide first high resolution analysis transcription translation neglect group livestock pathogens ImportanceToroviruses infect cattle goat pig horse worldwide cause gastrointestinal disease There treatment vaccine ability spill human assess These virus relate important human pathogen include severe acute respiratory syndrome SARS coronavirus share common feature however mechanism use produce subgenomic RNA molecule differs Here perform deep sequencing determine equine torovirus produce subgenomic RNAs also identify two previously unknown open reading frame hidden within genome Together result highlight similarity difference domestic animal virus relate pathogen human livestock',\n",
       "  0.5222021341323853),\n",
       " ('Novel system detect SARS coronavirus nucleocapsid protein use ssDNA aptamer The outbreak severe acute respiratory syndrome SARS affect thousand people efficient diagnostic system need accurate detection SARS coronavirus SARS CoV prevent limit future outbreak several SARS CoV structural protein nucleocapsid protein show good diagnostic marker study ssDNA aptamer specifically bind SARS CoV nucleocapsid protein isolated DNA library contain nuceotide random sequence middle 88mer single strand DNA After twelve cycle systematic evolution ligands exponential enrichment SELEX procedure ssDNA aptamers identify Enzyme link immunosorbent assay ELISA analysis use identify aptamer high bind affinity SARS CoV nucleocapsid protein Using approach ssDNA aptamer bind nucleocapsid protein 93±0 30nM identified Western blot analysis demonstrate ssDNA aptamer could use efficiently detect SARS CoV nucleocapsid protein compare nucleocapsid antibody Therefore believe select ssDNA aptamer may good alternative detection probe rapid sensitive detection SARS',\n",
       "  0.5221682786941528),\n",
       " ('Application RNA interference inhibit replication feline immunodeficiency virus chronically infect cell line RNA interference RNAi process double strand RNA induces post transcriptional sequence specific degradation homologous messenger RNA The present study carry apply RNAi technology inhibit replication feline immunodeficiency virus FIV Four small interfere RNAs siRNAs homologous FIV gag gene synthesize transfected feline fibroblastic cell line chronically infect FIV CRFK FIV These synthetic siRNAs efficiently inhibit replication FIV Next examine effect retroviral vector mediate transfer FIV specific short hairpin RNA shRNA replication FIV feline cell line chronically infect FIV FL4 The retroviral vector mediate transfer FIV specific shRNA show markedly inhibit replication FIV FL4 cell These result provide useful information development RNAi base gene therapy strategy control FIV infection',\n",
       "  0.5221477746963501),\n",
       " ('ahorrar', 0.5220967531204224),\n",
       " ('Correcting error synthetic DNA consensus shuffle Although efficient method exist assemble synthetic oligonucleotides gene genome suffer presence 1–3 random error DNA Here introduce new method term consensus shuffle demonstrate use significantly reduce random error synthetic DNA method error reveal mismatch hybridization population The DNA fragment mismatched fragment remove upon bind immobilized mismatch bind protein MutS PCR assembly remain fragment yield new population full length sequence enrich consensus sequence input population show two iteration consensus shuffle improve population synthetic green fluorescent protein GFPuv clone ∼60 fluorescent decrease error fold final value error per addition two iteration consensus shuffle correct population GFPuv clone member non functional population clone fluorescent Consensus shuffle facilitate rapid accurate synthesis long DNA sequence',\n",
       "  0.5220891237258911),\n",
       " ('Non canonical Translation Plant RNA Viruses Viral protein synthesis completely dependent upon host cell translational machinery Canonical translation host mRNAs depend structural element cap structure poly tail mRNAs Although many viral mRNAs devoid one structure still translate efficiently use non canonical mechanism Here review tool utilized positive sense single strand RNA plant virus initiate non canonical translation focus act sequence present viral mRNAs highlight element may interact host translation factor speculate contribution achieve translational control also describe translation strategy use plant virus optimize usage cod capacity compact genome include leaky scan initiation ribosomal frameshifting stop codon readthrough Finally future research perspectives unusual translational strategy ssRNA virus discuss include parallelisms viral host mRNAs mechanisms translation particularly host mRNAs translate stress condition',\n",
       "  0.5218108892440796),\n",
       " ('schildgen', 0.5217787623405457),\n",
       " ('Reverse Genetics System Cypovirus Based Bacmid Expressing RNA Polymerase Dendrolimus punctatus cypovirus DpCPV belong genus Cypovirus within family Reoviridae consider destructive pest pine forest worldwide DpCPV genome consist linear double strand RNA segment establish reverse genetics system clone cDNAs encode genomic segment DpCPV three reverse genetics vector segment transcribe control RNA polymerase promoter terminator tag hepatitis delta virus ribozyme sequence also construct vp80 knockout Autographa californica multiple nucleopolyhedrovirus bacmid express RNA polymerase codon optimize Sf9 cell Following transfection Sf9 cell three vector bacmid occlusion body OBs typical morphology cypovirus polyhedron observe optical microscopy The rescue system verified incorporation HindIII restriction enzyme site null mutant 9th genomic segment Furthermore transfected Sf9 cell reverse genetics vector bacmid additional vector bear egfp gene flank untranslated region 10th genomic segment aggregate green fluorescence localize OBs observe The rescue OBs able infect Spodopetra exigua larva although infectivity significantly low wild type DpCPV This reverse genetics system DpCPV could use explore viral replication pathogenesis facilitate development novel bio insecticides expression system exogenous protein',\n",
       "  0.5216481685638428),\n",
       " ('Comparison nucleotide deduce amino acid sequence gene specify virulent avirulent strain bovine coronaviruses The entire nucleotide sequence spike glycoprotein gene highly virulent bovine coronavirus BCV strain BCV LY138 avirulent BCV relate Norden Vaccine BCV Vaccine strain determine use polymerase chain reaction PCR amplify cDNAs obtain reverse transcription viral RNA produce single strand cDNAs DNA sequence The gene sequence viral strain compare recently publish strain BCV Mebus BCV Quebec BCV F15 open reading frame nucleotides encode protein amino acid residue find six strain Frameshifts insertion deletion observe except BCV The gene sequence conserve overall inspite different origin six virus There difference virulent avirulent group difference among four avirulent strain Comparison deduce amino acid sequence indicate proteins typical property membrane glycoprotein Nineteen Winked glycosylation site predict five strain conserve avirulent strain BCV The sequence KRRSRR predict proteolytic cleavage site identify five strain sequence KRRSVR find BCV Substitutions amino acid putative fusogenic domain two proline antigenic domain may cause alter immunogenic functional property protein specify virulent avirulent BCV strain Nine amino acid substitution virulent avirulent group may correlate BCV virulence',\n",
       "  0.5215969681739807),\n",
       " ('sgk', 0.5214074850082397),\n",
       " ('glucocorticosteroid', 0.5213127136230469),\n",
       " ('rna5', 0.5212665796279907),\n",
       " ('hybridisation', 0.5212544798851013),\n",
       " ('performing', 0.5212149620056152),\n",
       " ('readaptation', 0.5211586952209473),\n",
       " ('naproxen', 0.5211552977561951),\n",
       " ('Common unique feature viral RNA dependent polymerases Eukaryotes bacteria infect wide variety RNA viruses average pathogen share little sequence similarity use different replication transcription strategy Nevertheless member nearly RNA virus family depend activity virally encode RNA dependent polymerase condensation nucleotide triphosphates This review provide overview current understanding viral RNA dependent polymerase structure biochemistry biophysics involve replicate transcribe genetic material RNA virus',\n",
       "  0.5211078524589539),\n",
       " ('summertime', 0.5210899114608765),\n",
       " ('k‐252a', 0.5210345387458801),\n",
       " ('ultrastructurally', 0.5205854177474976),\n",
       " ('24–3', 0.5205637216567993),\n",
       " ('Design validation consensus degenerate hybrid oligonucleotide primer broad sensitive detection corona toroviruses The ssRNA family Coronaviridae include two subfamily prototyped coronaviruses toroviruses cause respiratory enteric infection facilitate identification new distantly related member family Coronaviridae develop molecular assay broad specificity The consensus degenerate hybrid oligonucleotide primer CODEHOP strategy modify design primer target conserve motif RNA dependent RNA polymerase locus They evaluate initially RNA templates virus infected cell use two step PCR protocol advance one step assay The sensitivity assay range RNA copy number individual corona torovirus template test respectively without excess RNA human cell This primer set compare design accord original CODEHOP rule show 10–103 fold great sensitivity evaluate corona torovirus templates detect respiratory specimen positive human coronaviruses well stool specimens positive bovine torovirus The high sensitivity broad virus range assay make suitable screen biological specimen search new virus family Coronaviridae',\n",
       "  0.5205132961273193),\n",
       " ('Immunogenic characterization epitope map transmissible gastroenteritis virus RNA dependent RNA polymerase Coronavirus RNA synthesis sophisticate process perform viral multienzymatic replicase complex together cellular factor key enzyme replication complex RNA dependent RNA polymerase RdRp study replication coronavirus genome six monoclonal antibody mAbs specific transmissible gastroenteritis virus TGEV RdRp generate characterize His tagged RdRp express baculovirus purified use immunogen produce mAbs The TGEV RdRp recognize mAbs context virus infection immunofluorescence analysis Western blot Epitope mapping Pepscan indicate RdRp mAbs recognize four non overlap linear epitope locate amino acid region terminal domain suggest region may constitute immunodominant domain The availability TGEV RdRp mAbs instrumental study coronavirus replication analyze function RdRp pathogenesis',\n",
       "  0.5204183459281921),\n",
       " ('Optimization vitro growth condition enterotropic murine coronavirus strain Enterotropic mouse hepatitis virus MHV strain difficult grow cell culture attempt develop efficient vitro cultivation system enterotropic MHV strain MHV MHV murine cell line inoculate MHV MHV infect infant mouse intestinal homogenate screen production cytopathic effect MHV MHV consistently produce cytopathic effect J774A cell Both strain produce titer TCID50 subsequent passage J774 cell MHV strain A59 JHM DVIM also produce high titer viral stock J774A cell Therefore J774A cell first cell find support replication enterotropic respiratory MHV strain After passage J774A cell MHV MHV could infect previously non susceptible cell line 17C1 CMT N18 NCTC though cytopathic effect often negligible MHV MHV MHV grow Percy cell Using Percy cell agarose overlay Giemsa stain MHV could quantify plaque assay Infant mouse bioassay plaque assay cell culture infection compare sensitivity detect MHV infect intestinal homogenate cell supernatant',\n",
       "  0.520074725151062),\n",
       " ('Detection feline immunodeficiency proviral DNA peripheral blood lymphocyte polymerase chain reaction Feline immunodeficiency virus FIV proviral DNA detect polymerase chain reaction method PCR PCR product detect gel electrophoresis ethidium bromide stain The region gag gene FIV choose target sequence amplification three primer pair prepare The PCR product subject amplification primer pair find possess sites digestion restriction enzyme hypothesize They react feline leukemia virus FeLV infect feline syncytium form virus FeSFV infect cell derive DNA specifically amplify FIV infect cell derive DNA FIV proviral DNA detect PCR method either primer pair one step amplification single PCR DNA derive peripheral blood lymphocyte PBL FIV antibody positive cat When PCR product cat subject second amplification use primer pair two step amplification double PCR FIV proviral DNA detect cat When PBL sample collect three cat negative three positive single PCR culture week presence interleukin FIV proviral DNA detect six cat single PCR method The result suggest either use culture PBL sample performance double PCR method enable simple specific detection FIV proviral DNA PBL',\n",
       "  0.5200482606887817),\n",
       " ('ahvaz', 0.5199860334396362),\n",
       " ('The nonstructural protein nsp8 SARS coronavirus interacts ORF6 accessory protein Severe acute respiratory syndrome SARS coronavirus SARS CoV cause severe outbreak several region world The SARS CoV genome predict contain functional open reading frame ORFs The first ORF encodes large polyprotein cleave nonstructural protein nsp The ORFs encode four structural protein spike membrane nucleocapsid envelope well eight SARS CoV specific accessory proteins report clone predict nsp8 gene ORF6 gene SARS CoV study ability interact express two protein fusion protein yeast two hybrid system demonstrate protein–protein interaction test use yeast genetic cross Further strength interaction measure challenging growth positive interaction clone increase gradient amino trizole The interaction verify express protein separately vitro couple transcription translation system coimmunoprecipitation mammalian cell Finally colocalization experiment perform SARS CoV infect Vero mammalian cell confirm nsp8–ORF6 interaction best knowledge first report interaction SARS CoV accessory protein nsp8 finding suggest ORF6 protein may play role virus replication',\n",
       "  0.5198898315429688),\n",
       " ('vp1s', 0.5197327733039856),\n",
       " ('New insight regulation Nidovirus subgenomic mRNA synthesis The member Order Nidovirales share similar genome organization two overlap nonstructural polyproteins encode two third structural protein encode third They also express region protein nest set terminal subgenomic messenger RNAs mRNAs Some Nidovirus mRNAs also common leader sequence acquire discontinuous RNA synthesis mechanism regulate multiple body transcription regulate sequence TRSs leader TRS Initial study detect single major body TRS mRNA alternative functional TRSs report The recent application advance technique next generation sequence ribosomal profile study arteriviruses coronaviruses reveal expanded mRNA transcriptome coding capacity',\n",
       "  0.5196470022201538),\n",
       " ('tricine', 0.5193930268287659),\n",
       " ('dismal', 0.5193420648574829),\n",
       " ('rps18', 0.5192577838897705),\n",
       " ('breath', 0.519235372543335),\n",
       " ('Virucidal effect murine duodenal extract study lactate dehydrogenase elevate virus Mucosal resistance infection lactate dehydrogenase elevate virus LDV previously demonstrate LDV system present important murine model study mucosal barrier viral infection present study duodenal molecule isolate normal mice potent virucidal activity test LDV well canine herpes canine hepatitis Semliki forest visna viruses The virucidal activity demonstrate non immune nature present apparently non enzymatic protein molecule molecular mass 10–100 kDa membrane filtration 10–17 kDa gel filtration The anti LDV activity molecule suppress anti duodenum antibody vitro vivo study suggest possible protective role anti viral molecule conclude normal mouse duodenum contain potent virucidal molecule interest study biological molecular mechanism viral resistance',\n",
       "  0.5191243290901184),\n",
       " ('infectious RNA hepta adenosine stretch responsible programmed ribosomal frameshift derive full length cDNA clone Hibiscus latent Singapore virus Hibiscus latent Singapore virus HLSV member Tobamovirus full length cDNA clone construct The vitro transcript two HLSV full length cDNA clone contain hepta adenosine stretch pHLSV octo adenosine stretch pHLSV infectious The replication level HLSV Nicotiana benthamiana protoplasts fold low compare HLSV The replicase proteins HLSV produce program ribosomal frameshift PRF stretch slippery sequence PRF Mutations downstream pseudoknot stretch show pseudoknot require frameshift vitro The stretch find extend subsequent replication cycle vivo envisage HLSV employ monotonous run PRF convert reach high replication survival plant',\n",
       "  0.5191238522529602),\n",
       " ('Fragmentation Golgi apparatus provide replication membrane human rhinovirus All virus positive strand RNA genome replicate genomic RNA association membrane host cell Here demonstrate novel organelle source replication membrane human rhinovirus HRV HRV infection induces fragmentation Golgi apparatus Golgi membrane rearrange vesicle approximately 250–500nm diameter The newly distribute Golgi membrane localize viral RNA replication templates strongly suggest observe vesicle sit viral RNA replication Expression HRV protein induces alteration Golgi stain pattern similar see viral infection express localizes Golgi derive membrane Taken together data show HRV infection protein play role fragment Golgi complex generate vesicle use site viral RNA replication',\n",
       "  0.5189275145530701),\n",
       " ('Human Coronavirus NL63 Open Reading Frame encodes virion incorporate glycosylated membrane protein BACKGROUND Human pathogenic coronavirus NL63 hCoV NL63 group alpha coronavirus commonly associate respiratory tract infection addition know non structural structural protein coronaviruses one accessory protein whose function mostly unknown Our study focus hCoV NL63 open reading frame ORF highly conserved accessory protein among coronaviruses RESULTS silico analysis amino acid sequence hCoV NL63 ORF predict triple membrane span protein Expression infect CaCo LLC MK2 cell confirm immunofluorescence Western blot analysis The protein detect within endoplasmatic reticulum Golgi intermediate compartment ERGIC coronavirus assembly bud take place Subcellular localization study use recombinant ORF protein transfected Huh cell reveal occurrence ERGIC Golgi lysosomal compartment fluorescence microscopy differently tag envelope membrane nucleocapsid proteins show ORF protein colocalizes extensively within ERGIC Using terminally FLAG tag ORF protein antiserum specific terminus verify propose topology extracellular terminus cytosolic terminus vitro translation analysis subsequent endoglycosidase digestion show ORF protein glycosylated terminus Analysis purify viral particle reveal ORF protein incorporate virion therefore additional structural protein CONCLUSIONS This study first extensive expression analysis group hCoV ORF protein give evidence ORF protein structural glycosylated virion incorporate protein',\n",
       "  0.5188258290290833),\n",
       " ('amsacta', 0.5187910795211792),\n",
       " ('c483y', 0.5183864831924438),\n",
       " ('asimismo', 0.5183737874031067),\n",
       " ('singular', 0.5182754993438721),\n",
       " ('polymerizes', 0.5182065963745117),\n",
       " ('Subcellular localization membrane association SARS CoV protein SARS CoV protein unique protein SARS coronavirus SARS CoV identify SARS CoV infect cell SARS patient specimen Recent study reveal could interact specifically many SARS CoV structural protein protein Expressed protein report localize Golgi complex SARS CoV infect cell study show protein mainly locate Golgi apparatus different tag terminus The localization pattern similar different transfected cell With assay truncate protein show might contain three transmembrane region second third region properly responsible Golgi localization ultra centrifugation experiment different extraction buffer confirm integral membrane protein embed phospholipid bilayer Immunofluorescence assay indicate localize protein Golgi complex transfected cell These result provide new insight study protein pathogenesis SARS CoV',\n",
       "  0.5180914998054504),\n",
       " ('1–137', 0.518074095249176),\n",
       " ('Arterivirus RNA dependent RNA polymerase Vital enzymatic activity remain elusive All RNA viruses encode RNA dependent RNA polymerase RdRp arteriviruses express terminal domain nonstructural protein nsp9 Previously potent primer dependent RdRp activity demonstrate homologous polymerase subunit nsp12 distantly relate coronaviruses The previous study focus vitro activity nsp9 arterivirus equine arteritis virus EAV report weak novo polymerase activity homopolymeric RNA templates However activity retain Mn2 ion omit assay biologically relevant template supply prompt revisit biochemical property polymerase Based property active site mutant conclude RNA synthesize activity observe novo primer dependent polymerase terminal transferase assay can not attribute recombinant EAV nsp9 RdRp Our result illustrate potential pitfall characterize polymerase use highly sensitive biochemical assay',\n",
       "  0.5179932117462158),\n",
       " ('chemoimmunotherapy', 0.5179581046104431),\n",
       " ('5–98', 0.5179562568664551),\n",
       " ('homogeneous', 0.5179449319839478),\n",
       " ('27r', 0.5178146362304688),\n",
       " ('ucuaaac', 0.5176844596862793),\n",
       " ('gp82', 0.5175815224647522),\n",
       " ('dma', 0.5175501704216003),\n",
       " ('The pattern proteins synthesize liver profoundly modify upon infection susceptible mouse mouse hepatitis virus Summary Susceptible BALB mouse experimental infection mouse hepatitis virus MHV3 reveal virus titre liver increase gradually peak PFU tissue day infection mouse die acute hepatitis BALB mouse infect MHV3 subsequently label vivo 35S methionine liver preparation infect non infect animal subject two dimensional gel electrophoresis Comparisons pattern computer image analysis reveal gene product increase gene product decrease upon virus infection two dimensional gel spot intensity conclude MHV3 infection susceptible strain mouse major modification protein synthesis occur The pattern alteration relate virus gene product mostly endogenous mouse proteins Whether proteins result defence attempt animal dictate virus order prevent protective response happen remains show',\n",
       "  0.5174875855445862),\n",
       " ('More one component Newcastle disease virus particle capable interferon induction The interferon IFN induce capacity intact NDV virion propiolactone inactivate particle several structural component compare use human PBML IFN produce cell Intact inactivate virion well nucleocapsid fraction differ significantly IFN induce capacity contrast genomic RNA well protein fraction envelope induce IFN titre level achieved virion NDV induce IFN production could block specifically incubation polychonal anti NDV monoclonal antibody mAbs two three anti mAbs anti NDV mAbs direct protein addition IFN induction fix MDBK cell express NDV surface proteins infection NDV Ulster inhibit one two anti mAbs The result suggest induction IFN synthesis human PBML complex process involve protein also uncleaved protein precursor component protein fraction enter cell genomic RNA',\n",
       "  0.517463743686676),\n",
       " ('experimentation', 0.5174633264541626),\n",
       " ('gilpin', 0.5174089670181274),\n",
       " ('castantospermine', 0.5172094106674194),\n",
       " ('illiterate', 0.5171517133712769),\n",
       " ('Identification novel RNA secondary structure within hepatitis virus genome reveals cooperative involvement genome package The specific packaging hepatitis virus HCV genome hypothesise drive Core RNA interaction identify region viral genome involved process use SELEX systematic evolution ligands exponential enrichment identify RNA aptamers bind specifically Core vitro Comparison aptamers multiple HCV genome reveal presence conserve terminal loop motif within short RNA stem loop structure postulate interaction motifs well sub motif present HCV genome statistically significant level Core protein may drive virion assembly mutate predict motif within HCV infectious molecular clone JFH thereby produce range mutant virus predict posse alter RNA secondary structure RNA replication viral titre unaltered virus possess one mutated structure However infectivity titre decrease virus possess high number mutate region This work thus identify multiple novel RNA motif appear contribute genome packaging suggest structure act cooperative packaging signal drive specific RNA encapsidation HCV assembly',\n",
       "  0.5166488885879517),\n",
       " ('nick', 0.5165214538574219),\n",
       " ('mayo', 0.5164768099784851),\n",
       " ('Small Interfering RNA Effectively Inhibits Expression SARS Coronavirus Membrane Gene Two Novel Targeting Sites Small interfere RNA siRNA class duplex RNA molecules nucleotide length function post transcriptionally downregulate target gene expression The membrane protein severe acute respiratory syndrome associate coronavirus SARS CoV highly abundant viral infection critical element viral assembly Nucleotide substitution viral genome occur frequently SARS CoV infection current study analyze gene sequence derive SARS CoV isolates uncover six nucleotide substitution among isolates Interestingly nucleotide substitution locate half gene Based information previous report create two novel siRNAs target two unexplored well conserved region gene The effect two siRNAs test semi quantitative PCR EGFP fusion gene expression The result demonstrate siRNAs effectively specifically block target gene expression Real time quantitative PCR qRT PCR reveal siRNA target half gene induce potent inhibition target half Both significantly downregulated gene mediate upregulation interferon expression Thus result indicate SARS CoV gene specific siRNA might function sequence dependent manner',\n",
       "  0.5164260268211365),\n",
       " ('Translation viral mRNAs require eIF4E block inhibitor 4EGI High throughput screen render new inhibitor eukaryotic protein synthesis One molecule 4EGI report selectively block initiation factor eIF4E investigate action inhibitor translation direct several viral mRNAs principle utilize eIF4E find 4EGI inhibits translation direct poliovirus IRES rabbit reticulocyte lysates similar extent cap mRNA Moreover 4EGI inhibits translation drive poliovirus IRES vitro culture cell despite cleavage eIF4G picornavirus proteases Finally translation vesicular stomatitis virus mRNAs Sindbis virus subgenomic mRNA block 4EGI infected cell similar extent cellular mRNAs These finding cast doubt selective action inhibitor suggest molecule may affect step protein synthesis unrelated cap recognition eIF4E',\n",
       "  0.5164029598236084),\n",
       " ('act1', 0.516227126121521),\n",
       " ('bromopalmitate', 0.5162060260772705),\n",
       " ('Induciton MHC class antigens glial cell dependent persistent mouse hepatitis virus infection class antigens class antigens detected surface glial cell persistently infect mouse hepatitis virus strain A59 MHV A59 late day post infection Uninfected glial cell remain negative class class surface antigens previously show conditioned medium infect glial cell culture supernatant contain factor unrelated infectious virus capable induce class antigens uninfected glial cell The synthesis factor appear dependent production infectious virus since inducing activity could detect day follow addition neutralize antibody culture This suggest induce activity contain unstable component synthesis dependent continual virus production Persistent MHV infection class antigen expression may play role MHV induce demyelination',\n",
       "  0.5160356760025024),\n",
       " ('determinism', 0.5154547691345215),\n",
       " ('delphacodes', 0.5153766870498657),\n",
       " ('mvsk', 0.5152477025985718),\n",
       " ('Efficient Autoproteolytic Processing MHV A59 like Proteinase Flanking Hydrophobic Domains Requires Membranes The replicase gene coronavirus MHV A59 encode serine like proteinase similar proteinase picornaviruses This proteinase domain flank side hydrophobic potentially membrane span region Cell free expression plasmid encode like proteinase 3CLpro result synthesis kDa protein specifically recognize antibody direct carboxy terminal region proteinase protein identical mobility detect MHV A59 infect cell lysates vitroexpression plasmid encode 3CLpro portion two flank hydrophobic region result inefficient processing kDa protein However efficiency process event enhance addition canine pancreatic microsome translation reaction removal one flanking hydrophobic domains Proteolysis inhibit presence ofN ethylmaleimide NEM mutagenesis catalytic cysteine residue proteinase indicate 3CLpro responsible autoproteolytic cleavage flanking domains Microsomal membrane unable enhance thetransprocessing precursor contain inactive proteinase domain hydrophobic region recombinant 3CLpro express fromEscherichia coli Membrane association assay demonstrate kDa 3CLpro present soluble fraction reticulocyte lysates polypeptide contain hydrophobic domain associate membrane pellet With help viral epitope tag identify kDa membrane associate polypeptide proteolytic product contain amino terminal hydrophobic domain',\n",
       "  0.5150966048240662),\n",
       " ('Characterization polyadenylation activity replicase complex Bamboo mosaic virus infect Nicotiana benthamiana plant Bamboo mosaic virus BaMV positive sense single strand RNA genome cap poly tail characterize polyadenylation activity BaMV replicase complex perform vitro polyadenylation various BaMV templates conduct polyadenylation activity assay BaMV RNA use partially purify BaMV replicase complex The result show approximately adenylates end RNA generate endogenous RNA templates Specific fraction derive uninfected Nicotiana benthamiana plant enhanced polyadenylation activity imply host factor involve polyadenylation Furthermore polyadenylation detect newly synthesize plus strand RNA vitro use exogenous BaMV minus strand minigenome For polyadenylation exogenous plus strand minigenome end require least reach polyadenylation activity The result indicate BaMV replicase complex recognize end BaMV polyadenylation',\n",
       "  0.515074610710144),\n",
       " ('14–21', 0.5148413777351379),\n",
       " ('Structural Functional Basis Fidelity Nucleotide Selection Flavivirus RNA Dependent RNA Polymerases Viral RNA dependent RNA polymerase RdRps play central role viral replication also genetic evolution viral RNAs After bind RNA template select triphosphate ribonucleosides viral RdRps synthesize RNA copy accord Watson Crick base pairing rule The copy process sometimes deviate base pairing rule specify template natural ribose selectivity thus process error prone due intrinsic fidelity viral RdRps These enzyme share number conserve amino acid sequence string call motif A–G define structural functional point view relation gradually emerge mutation motifs viral genome evolution observe mutation rate Here review current knowledge motif role structural mechanistic basis fidelity nucleotide selection RNA synthesis Flavivirus RdRps',\n",
       "  0.5148224830627441),\n",
       " ('rica', 0.514763593673706),\n",
       " ('18–39', 0.5147024393081665),\n",
       " ('collagenous', 0.5146476030349731),\n",
       " ('nucleocapsids', 0.5145421028137207),\n",
       " ('3xtg', 0.5145387053489685),\n",
       " ('somamers', 0.5144486427307129),\n",
       " ('Expression cellular localisation porcine transmissible gastroenteritis virus proteins recombinant vaccinia virus Porcine transmissible gastroenteritis virus TGEV nucleoprotein integral membrane protein gene clone vaccinia virus insertion vector pGS20 correct orientation expression control vaccinia promoter Recombinant vaccinia virus generate vivo homologous recombination insertion vector strain vaccinia virus Nucleoprotein express recombinant vaccinia virus TGEV relative molecular mass susceptible degradation terminus yield discrete breakdown product The integral membrane protein express recombinant vaccinia virus TGEV sensitive endogly cosidase reduce mature polypeptide specie Expression recombinant vaccinia virus inhibit early infection due cryptic vaccinia virus transcriptional termination signal within TGEV cod sequence Indirect immunofluorescence show localise cell cytoplasm either TGEV recombinant vaccinia virus infect cell Antisera mouse infect recombinant virus immunoprecipitated specific TGEV antigens lysates TGEV infect cell little significant TGEV neutralising activity vitro',\n",
       "  0.5143985152244568),\n",
       " ('nhap', 0.5143508911132812),\n",
       " ('kda', 0.5142502784729004),\n",
       " ('Expression Hemagglutinin Esterase Mouse Hepatitis Virus Coronavirus Defective–Interfering RNA Alters Viral Pathogenesis defective interfering RNA mouse hepatitis virus MHV develop vector express MHV hemagglutinin esterase protein The virus contain express protein A59 generate infect cell MHV A59 express transfecting thein vitro transcribe RNA contain gene similar virus A59 CAT express chloramphenicol acetyltransferase CAT use control These virus inoculate intracerebrally mice role protein viral pathogenesis evaluate Results show mice infect parental A59 A59 CAT succumb infection day postinfection demonstrate inclusion alter pathogenesis contrast mouse infect A59 survive infection CAT specific subgenomic mRNAs detect brain day later indicate gene vector express early stage viral infection significant difference virus titer viral antigen expression brain observed A59 A59 CAT infect mouse suggest virus replication brain affect expression However day slight increase extent inflammatory cell infiltration brain A59 infect mice Surprisingly virus titer liver A59 infect mice log10lower A59 CAT infected mouse day Also substantially less necrosis viral antigen detect liver A59 infect mice This may account reduced mortality mice The possible contribution host immune system difference pathogenesis analyze compare expression four cytokine Results show tumor necrosis factor interleukin mRNAs increase brain A59 infect mice day whereas interferon interleukin mRNAs similar A59 A59 CAT infect mice These data suggest transient expression protein enhance early innate immune response possibly contribute eventual clearance virus liver This study indicate feasibility expression system study role viral protein MHV infection',\n",
       "  0.5141196250915527),\n",
       " ('Simultaneous rapid sequence multiple RNA virus genome Comparing sequence archive virus collect many year present allows study viral evolution contributes design new vaccine However difficulty time expense generate full length sequence individually archive sample hampered study Next generation sequence technology utilized analysis clinical environmental sample identify viral pathogen may present This led discovery many new uncharacterized virus number viral family Use sequence technology would advantageous examine viral evolution study sequencing procedure use sequence simultaneously rapidly multiple archive sample use single standard protocol This procedure utilized primer compose base know sequence random base end also serve identify barcode allow differentiation viral library follow pool sequence This conferred sequence independence random prim first second strand cDNA synthesis Viral stock treat nuclease cocktail reduce presence host nucleic acid Viral RNA extract follow single tube random prim double strand cDNA synthesis The resultant cDNAs amplify primer specific PCR pool size fractionate sequenced Ion Torrent PGM platform The individual virus genome readily assemble novo template assist assembly methods This procedure consistently result near full length full length genomic sequence use sequence multiple bovine pestivirus coronavirus isolate simultaneously',\n",
       "  0.5140406489372253),\n",
       " ('Chemo enzymatic synthesis site specific isotopically labeled nucleotide use NMR resonance assignment dynamic structural characterization Stable isotope label central NMR study nucleic acid Development method incorporate label specific atomic position within nucleotide promise expand size range RNAs study NMR Using recombinantly express enzyme chemically synthesize ribose nucleobase develop inexpensive rapid chemo enzymatic method label ATP GTP site specifically high yield incorporate nucleotide RNAs size range nucleotide use vitro transcription Site iron responsive element fluoride riboswitch Bacillus anthracis frame shift element human corona virus Finally showcase improvement spectral quality arise reduce crowd narrowed linewidths accurate analysis NMR relaxation dispersion CPMG TROSY base CEST experiment measure time scale motion improve NOESY strategy resonance assignment Applications selective labeling technology promise reduce difficulty associate chemical shift overlap rapid signal decay make challenge study structure dynamic large RNAs beyond median size find PDB',\n",
       "  0.5140270590782166),\n",
       " ('shiverer', 0.5139074921607971),\n",
       " ('uridines', 0.5138925313949585),\n",
       " ('The nucleocapsid protein SARS coronavirus high binding affinity human cellular heterogeneous nuclear ribonucleoprotein The nucleocapsid protein SARS coronavirus SARS CoV major structural component virion appear multifunctional protein involve viral RNA replication translation Heterogeneous nuclear ribonucleoprotein hnRNP relate pre mRNA splice nucleus translation regulation cytoplasm report base relevant biophysical biochemical assay nucleocapsid protein SARS CoV SARS discover exhibit high bind affinity human hnRNP GST pull result clearly demonstrate SARS protein could directly specifically bind human hnRNP vitro Yeast two hybrid assay indicate vivo binding relate fragment 161–210 SARS Gly rich domain 203–320 hnRNP Moreover kinetic analysis surface plasmon resonance SPR technology reveal SARS protein specific bind affinity human hnRNP 35±0 02μM 83±0 42×103 M−1 s−1 06±0 12×10−3 s−1 suggest SARS hnRNP proteins possibly within SARS CoV replication transcription complex SARS human hnRNP interaction might function regulation SARS CoV RNA synthesis addition determine result show SARS protein one binding domain interact human hnRNP different mouse hepatitis virus MHV binding case nucleocapsid protein MHV MHV find two binding domain involve MHV hnRNP interaction thereby suggest SARS protein might carry different bind mode bind human hnRNP function performance comparison MHV',\n",
       "  0.5138762593269348),\n",
       " ('thy', 0.5138387084007263),\n",
       " ('ns2h126r', 0.5138235092163086),\n",
       " ('batcovs', 0.5138084888458252),\n",
       " ('ria', 0.5137373208999634),\n",
       " ('Spatiotemporal dynamic HSV genome nuclear entry compaction state transition use bioorthogonal chemistry super resolution microscopy investigate spatiotemporal dynamic HSV genome transport initiation infection use virus contain bioorthogonal traceable precursor incorporate genome HSV EdC vitro assay reveal structural alteration capsid induce upon HSV EdC bind solid support allow couple external capture agent demonstrate vast majority individual virion contain bioorthogonally tag genome Using HSV EdC vivo reveal novel aspect kinetics localisation mechanistic entry requirement morphological transition infect genome Uncoating nuclear import observe within min genome define compaction state fold volume increase capsid Free cytosolic uncoated genome infrequent 7–10 total uncoated genome likely consequence subpopulation cell receive high particle number Uncoated nuclear genome underwent temporal transition condensation state ICP4 efficiently associate condensed foci initial infect genome relationship switch away residual long live condensed focus increasingly decondensed genome infection progress Inhibition transcription effect nuclear entry absence transcription genome persist tightly condense focus Ongoing transcription absence protein synthesis reveal distinct spatial cluster genome term genome congregation see non transcribe genome Genomes expand decondensed form absence DNA replication indicate additional transitional step During full progression infection genome decondensed diffuse low intensity signal dissipate within replication compartment frequently tight focus remain peripherally represent unreplicated genome condense parental strand replicate DNA Uncoating nuclear entry independent proteasome function resistant inhibitor nuclear export Together additional data result reveal new insight spatiotemporal dynamic HSV genome uncoating transport organisation',\n",
       "  0.5136244893074036),\n",
       " ('rna', 0.5135228633880615),\n",
       " ('Proteomic analysis purify coronavirus infectious bronchitis virus particle BACKGROUND Infectious bronchitis virus IBV coronavirus domestic chicken cause major economic loss poultry industry Because complexity IBV life cycle small number viral structural protein important virus host relationship likely remain discovered Toward goal perform two dimensional gel electrophoresis fractionation couple mass spectrometry identification approach perform comprehensive proteomic analysis purify IBV particle RESULTS Apart virus encode structural protein detect host protein purify virion group several functional category include intracellular trafficking proteins molecular chaperone macromolcular biosynthesis proteins cytoskeletal protein signal transport proteins protein degradation chromosome associate proteins ribosomal protein function protein Interestingly total host protein report present virion virus family major vault protein TENP protein ovalbumin scavenger receptor protein Following identification host protein proteomic method presence protein purify IBV preparation verify western blot immunogold label detection CONCLUSIONS The result present first standard proteomic profile IBV may facilitate understand pathogenic mechanism',\n",
       "  0.5134994983673096),\n",
       " ('wc1−', 0.5134060382843018),\n",
       " ('Recombination North American strain porcine reproductive respiratory syndrome virus Porcine reproductive respiratory syndrome virus PRRSV recently discover arterivirus swine pathogen show undergo homologous recombination infection cell two culture adapt North American PRRSV strain result recombinant viral particle contain chimeric ORF ORF protein Nucleotide sequence analysis clone recombinant PCR product encompass base viral genome reveal six independent recombination event Recombinant product persist culture least three passage indicate continuous formation recombinant viruses growth recombinant viruses competition parental viruses The frequency recombination estimate fragment analyze similar recombination frequency observe coronaviruses apparent example natural ORF recombination naturally occur wild type virus also find indicate recombination likely important genetic mechanism contribute PRRSV evolution',\n",
       "  0.5133576393127441),\n",
       " ('vsars', 0.5131468772888184),\n",
       " ('Characterization Ribosomal Frameshifting Theiler Murine Encephalomyelitis Virus Theiler murine encephalomyelitis virus TMEV member genus Cardiovirus Picornaviridae family positive sense single strand RNA viruses Previously demonstrate related cardiovirus Encephalomyocarditis virus program ribosomal frameshift PRF occur conserve GUU UUU sequence within encode region polyprotein open reading frame ORF Here show PRF occur similar site translation TMEV genome addition demonstrate predict RNA stem loop structure noncanonical space downstream shift site require efficient frameshifting TMEV frameshifting also require virus infection Mutating GUU UUU shift site inhibit frameshifting result attenuate virus reduce growth kinetics small plaque phenotype Frameshifting virus context find extremely efficient knowledge high frameshifting efficiency record date virus propose highly efficient PRF TMEV provide mechanism escape confines equimolar expression normally inherent single polyprotein expression strategy picornaviruses IMPORTANCE Many virus utilize programmed ribosomal frameshifting PRF produce different protein product define ratio translate overlap ORFs increase cod capacity With exception PRF occur specific slippery heptanucleotide sequence stimulate RNA structure begin nucleotide downstream slippery site Here describe unusual case PRF Theiler murine encephalomyelitis virus TMEV extraordinarily efficient ribosome shift alternative reading frame stark contrast example PRF dependent upon stem loop structure begin downstream slippery site Furthermore TMEV base reporter construct transfected cell efficient frameshifting critically dependent upon virus infection suggest TMEV evolve frameshifting novel mechanism remove ribosome message ribosome sink downregulate synthesis encode replication protein',\n",
       "  0.5130965709686279),\n",
       " ('proteinsin', 0.513020932674408),\n",
       " ('epimerase', 0.5130162239074707),\n",
       " ('Site addition acetyl galactosamine glycoprotein mouse hepatitis virus A59 pulse chase label 35S methionine long term label sugar glycoprotein coronavirus MHV A59 show acquire link oligosaccharides two step process About min synthesis protein acetyl galactosamine add This follow approximately min later addition galactose sialic acid give mature oligosaccharides This sequence addition confirm analyze labeled oligosaccharide bound form use gel filtration columns The intracellular location first step determine exploit temperature sensitivity virus release The virus normally bud first smooth membrane compartment lie rough endoplasmic reticulum cis side Golgi stack Tooze degree virus assemble appear enter Golgi stack The addition acetyl galactosamine unaffected although addition galactose sialic acid inhibit These result strongly suggest addition acetyl galactosamine occur bud compartment morphology similar transitional element vesicle',\n",
       "  0.5129828453063965),\n",
       " ('intersequence', 0.5129528045654297),\n",
       " ('neutralizing', 0.5129203200340271),\n",
       " ('tn93', 0.5129177570343018),\n",
       " ('encapsidation', 0.5128762125968933),\n",
       " ('homogenize', 0.5128618478775024),\n",
       " ('Mouse Hepatitis Virus Replicase Protein Complexes Are Translocated Sites Protein Accumulation ERGIC Late Times Infection The coronavirus mouse hepatitis virus MHV direct synthesis viral RNA discrete membranous complex distribute throughout cell cytoplasm These putative replication complex compose intimately associate biochemically distinct membrane population contain protein process replicase gene polyprotein Specifically one membrane population contain gene protein p65 p1a contains gene protein p28 helicase well structural nucleocapsid protein newly synthesize viral RNA study immunofluorescence confocal microscopy use define relationship membrane population comprise putative replication complex different time infection MHV A59 infect delayed brain tumor cell postinfection membrane contain helicase colocalized membrane contain p1a p65 focus distinct site accumulation however membrane contain helicase predominantly perinuclear distribution colocalized contrast p1a p65 contain membrane retain peripheral punctate distribution time infection colocalize late time infection helicase also colocalized ERGIC p53 specific marker endoplasmic reticulum–Golgi–intermediate compartment These data demonstrate putative replication complex separate component membrane relocalized course infection These result suggest membrane population within MHV replication complex serve distinct function RNA synthesis delivery replication product sit virus assembly',\n",
       "  0.5128561854362488),\n",
       " ('minimal approach design flavivirus infectious DNA The infectious DNA approach base vivo transcription RNA virus genome cDNA cassettes eukaryotic promoter transfected cell become popular alternative classical scheme infectious clone methodology Its use however often limited instability plasmid due transcriptional activity eukaryotic promoter Escherichia coli result synthesis product toxic bacterial host Using highly unstable representative infectious clone Japanese encephalitis flavivirus test new approach design problematic infectious DNA construct base minimize unwanted transcription bacterial host plasmid contain full genome size cDNA control minimal cytomegalovirus CMV promoter propagate coli growth stability characteristic similar construct control promoter Transfection plasmid susceptible cell lead establishment productive infectious cycle Reinsertion CMV enhancer end cassette substantially increase specific infectivity without affect stability growth characteristic construct This approach useful stabilization infectious clone modification viral cDNA cassette feasible suitable alternative',\n",
       "  0.5128459334373474),\n",
       " ('nephritis', 0.5128090381622314),\n",
       " ('Coronavirus derive expression system Both helper dependent expression system base two component single genome construct targeted recombination use infectious cDNA clone develop The sequence regulate transcription characterize mainly use helper dependent expression system possible validate use single genome The genome coronaviruses engineer modification infectious cDNA lead efficient ml−1 stable passage expression foreign gene The possibility engineering tissue specie tropism target expression different organ animal specie include human increase potential coronaviruses vector Thus coronaviruses promise virus vector vaccine development possibly gene therapy',\n",
       "  0.51279217004776),\n",
       " ('ectopeptidases', 0.5127427577972412),\n",
       " ('sf9', 0.5126587748527527),\n",
       " ('pöhlmann', 0.5126478672027588),\n",
       " ('lengthy', 0.5123386979103088),\n",
       " ('constitution', 0.5122398138046265),\n",
       " ('env', 0.5121021270751953),\n",
       " ('ayant', 0.5120821595191956),\n",
       " ('Selective Degradation Host RNA Polymerase Transcripts Influenza Virus Host Shutoff Protein Influenza viruses IAVs inhibit host gene expression process know host shutoff Host shutoff limit host innate immune response may also redirect translation apparatus production viral proteins Multiple IAV protein regulate host shutoff include ribonuclease remain incompletely characterize report selectively target host RNA polymerase Pol transcribe mRNAs sparing product Pol Pol III Interestingly show also target Pol transcribe RNAs nucleus include non cod RNAs destine translate reporter transcript RNA hairpin structure block ribosome load Transcript degradation likely occur nucleus enrich nucleus nuclear localization correlate reduction target RNA level Complete degradation host mRNAs follow mediate endonucleolytic cleavage dependent host exonuclease Xrn1 IAV mRNAs structurally similar host mRNAs synthesize modify end action viral RNA dependent RNA polymerase complex Infection cell wild type IAV recombinant deficient virus reveal IAV mRNAs resist mediate degradation infection time loss result change synthesis select viral mRNAs decrease viral protein accumulation Collectively result significantly advance understand IAV host shutoff suggest cause selective degradation host mRNAs discriminate aspect Pol dependent RNA biogenesis nucleus',\n",
       "  0.5120222568511963),\n",
       " ('1–16', 0.5119742155075073),\n",
       " ('truth', 0.5119687914848328),\n",
       " ('full length inter genotypic chimeric genome hepatitis virus infectious vitro The protein hepatitis virus HCV function ion channel planar lipid bilayers function vital virus life cycle study replace either entire partial genotype strain JFH1 HCV strain replicates produce virus progeny vitro correspond region protein genotype Australian isolate HCV Compared wild type chimeric virus reach peak infectivity delay produce comparable titer wild type virus progeny virus able infect naive permissive cell Amantadine treatment wild type chimeric virus reduce virus titer respectively Therefore study first time demonstrate genotype JFH1 strain genome encode full length genotype gene produce infectious particle vitro These chimeric virus valuable instrument comparative study protein',\n",
       "  0.5119437575340271),\n",
       " ('Variable polyadenylation Wheat yellow mosaic virus novel effect translation replication BACKGROUND Polyadenylation influence many aspect mRNA well viral RNA variable polyadenylation end report RNA virus interest identify characteristic potential role polyadenylation Wheat yellow mosaic virus WYMV report contain two genomic RNAs poly tail cause severe disease wheat East Asia region METHODS RACE use identify sequence end WYMV RNAs naturally infect wheat WYMV vitro translation assay perform analyze effect UTRs WYMV without polyadenylation translation vitro replication mediate WYMV NIb protein perform evaluate effect variable polyadenylation replication RESULTS Variable polyadenylation WYMV RNAs identify via RACE WYMV RNAs naturally infect wheat China simultaneously present region long short adenylation end The effect variable polyadenylation translation replication WYMV RNAs evaluate UTR UTR WYMV RNA1 RNA2 synergistically enhance translation firefly luciferase Fluc gene vitro WGE system whereas additional adenylates oppositive effect enhancement translation mediate UTRs WYMV Additional adenylates remarkably inhibit synthesis complementary strand viral genome RNA vitro replication mediate WYMV NIb protein CONCLUSIONS end WYMV RNAs present variable polyadenylation even polyadenylation polyadenylation opposite effect translation mediate UTRs WYMV RNA1 RNA2 polyadenylation negative effect minus strand synthesis WYMV RNA vitro Variable polyadenylation WYMV RNAs may provide sufficient selection template translation replication ELECTRONIC SUPPLEMENTARY MATERIAL The online version article s12985 contain supplementary material available authorized user',\n",
       "  0.5117692947387695),\n",
       " ('RNA Aptamer That Specifically Binds Glycosylated Hemagglutinin Avian Influenza Virus Suppresses Viral Infection Cells The influenza virus surface glycoprotein hemagglutinin responsible viral attachment sialic acid contain host cell receptor facilitate initial stage viral infection present study isolate RNA aptamer specific glycosylated receptor bind domain protein gHA1 cycle systematic evolution ligands exponential enrichment procedure SELEX investigate select aptamer suppress viral infection host cell Nitrocellulose filter bind enzyme link immunosorbent assay ELISA experiment reveal RNA aptamer HA12 bound specifically gHA1 protein Cell viability assay show HA12 RNA aptamer suppress viral infection host cell enhance cell viability Immunofluorescence microscopic analysis demonstrate HA12 RNA aptamer suppress viral attachment host cell neutralize receptor bind site influenza virus These result indicate isolated RNA aptamer develop antiviral reagent influenza appropriate therapeutic formulation',\n",
       "  0.5117629170417786),\n",
       " ('Primary structure glycoprotein coronavirus MHV A59 identification trypsin cleavage site The nucleotide sequence peplomer gene MHV A59 determine set overlap cDNA clone The gene encodes protein amino acid include hydrophobic signal peptide second large hydrophobic domain find near COOH terminus probably represent membrane anchor Twenty glycosylation site predict Cleavage protein result two different 90K specie 90A 90B Sturman Ricard Holmes Virol 904–911 activates cell fusion Protein sequence trypsin generate terminus reveal position cleavage site 90A 90B could identify terminal terminal part respectively Amino acid sequence comparison A59 1HM protein show extensive homology reveal stretch amino acid 90B region A59 protein absent JHM',\n",
       "  0.5115609169006348),\n",
       " ('peplomers', 0.5115512609481812),\n",
       " ('cart', 0.5113778710365295),\n",
       " ('colombaroli', 0.5113678574562073),\n",
       " ('Annexin Binds RNA Reduces Frameshifting Efficiency Infectious Bronchitis Virus Annexin ANXA2 protein implicate diverse cellular function include exocytosis DNA synthesis cell proliferation recently propose involved RNA metabolism show associate cellular mRNA Here identify ANXA2 RNA bind protein RBP bind IBV Infectious Bronchitis Virus pseudoknot RNA first confirm bind ANXA2 IBV pseudoknot RNA ultraviolet crosslinking show bind RNA pseudoknot ANXA2 protein vitro cell Since RNA pseudoknot locate frameshifting region IBV use bait cellular RBPs test whether ANXA2 could regulate frameshfting IBV pseudoknot RNA dual luciferase assay Overexpression ANXA2 significantly reduce frameshifting efficiency IBV pseudoknot RNA knockdown protein strikingly increased frameshifting efficiency The result suggest ANXA2 cellular RBP modulate frameshifting efficiency viral RNA enable act anti viral cellular protein hint role RNA metabolism cellular mRNAs',\n",
       "  0.511307954788208),\n",
       " ('caveolar', 0.5111496448516846),\n",
       " ('32p', 0.5110688805580139),\n",
       " ('permissiveness', 0.5110557079315186),\n",
       " ('53x', 0.5110110640525818),\n",
       " ('High Affinity Recognition HIV Frameshift Stimulating RNA Alters Frameshifting Vitro Interferes HIV Infectivity Image see text The life cycle human immunodeficiency virus type HIV absolute requirement ribosomal frameshifting protein translation order produce polyprotein precursor viral enzyme While RNA stem loop structure HIV Frameshift Stimulating Signal HIV FSS control frameshift efficiency hypothesize attractive therapeutic target develop compound selectively bind RNA interfere HIV replication proven challenge Building prior discovery hit molecule able bind stem loop report development compound display high affinity HIV FSS These compound able enhance frameshifting dual luciferase assay human embryonic kidney cell strongly inhibit infectivity pseudotyped HIV virion',\n",
       "  0.5110002160072327),\n",
       " ('Silencing SARS CoV spike gene small interfere RNA HEK 293T cell Two candidate small interfering RNAs siRNAs correspond severe acute respiratory syndrome associate coronavirus SARS CoV spike gene design vitro transcribe explore possibility silence SARS CoV gene The plasmid pEGFP optS contain codon optimized SARS CoV gene express spike EGFP fusion protein EGFP silence target express reporter transfected siRNAs HEK 293T cell various time point posttransfection level EGFP expression amount spike mRNA transcript detect fluorescence microscopy flow cytometry Western blot real time quantitative PCR respectively The result show cell transfected pEGFP optS express EGFP fusion protein high level compare transfected pEGFP contain wildtype SARS CoV spike gene sequence The green fluorescence mean fluorescence intensity SARS CoV RNA transcript find significantly reduce expression SARS CoV glycoprotein strongly inhibit cell transfected either EGFP specific siRNAs Our finding demonstrate specific siRNAs use study able specifically effectively inhibit SARS CoV glycoprotein expression culture cell block accumulation mRNA may provide approach study function SARS CoV gene development novel prophylactic therapeutic agent SARS CoV',\n",
       "  0.5109622478485107),\n",
       " ('Protein composition tomato spot wilt virus Analysis protein composition tomato spot wilt virus TSWV purify improved procedure polacrylamide gel electrophoresis reveal three major structural protein 000d minor one 000d The three major protein constitute total viral protein three show glycoprotein One major protein 000d minor protein show associated subviral particle isolate treatment virus nonionic detergent Nonidet Only trace two protein detect subviral particle Synthesis virus induce protein TSWV infect tobacco leaf study label infect healthy tissue 14C valine respectively The labeled tissue fractionate crude subcellular fraction protein pattern healthy infected tissue compare coelectrophoresis polyacrylamide gel Only one virus specific protein 000d detect virus enrich fraction correspond viral structural protein 000d',\n",
       "  0.5108555555343628),\n",
       " ('colony', 0.5107781887054443),\n",
       " ('trypanosome', 0.5106927752494812),\n",
       " ('cys1227', 0.5106241703033447),\n",
       " ('recompute', 0.5105735063552856),\n",
       " ('cd95', 0.5105723142623901),\n",
       " ('intralesionally', 0.5104803442955017),\n",
       " ('nucifera', 0.5104195475578308),\n",
       " ('tablets', 0.5103815793991089),\n",
       " ('regularity', 0.5101137757301331),\n",
       " ('indeed', 0.5100634694099426),\n",
       " ('Structural protein human respiratory coronavirus OC43 The human respiratory coronavirus OC43 grown human rectal tumor cell line isotopically label amino acid glucosamine orthophosphate analyze virion structural proteins Four major protein specie resolve electrophoresis many property deduce digestion study use proteolytic enzyme The four protein kDa protein presume peplomeric protein glycosylated proteolytically cleavable trypsin subunit kDa The subunit represent different amino acid sequence basis peptide map kDa protein glycosylated behaved disulfide link dimer kDa molecule apparent virion hemagglutinin basis digestion study trypsin bromelain pronase kDa nucleocapsid protein phosphorylated kDa matrix protein glycosylated The kDa specie therefore designate respectively They exist molar ratio calculated present rate molecule per virion respectively',\n",
       "  0.5099974870681763),\n",
       " ('aspirat', 0.509882390499115),\n",
       " ('ORF protein present significant level virion arterivirus lactate dehydrogenase elevate virus MRNA2 arteriviruses lactate dehydrogenase elevate virus LDV equine arteritis virus EAV encode two protein read different frame kDa minor envelope glycoprotein kDa protein lack glycosylation sit signal peptide possess central hydrophobic segment Recent study show protein EAV translate mRNA EAV infect BHK cell kDa protein membrane associate small amount recover purified virion Snijder van Tol Pederson Raamsman Vries Identification novel structural protein arteriviruses Virol 6335–6345 The author conclude kDa protein another arterivirus envelope protein designate protein However detect significant level kDa protein LDV virion thus conclude structural virion component',\n",
       "  0.5098797082901001),\n",
       " ('minimise', 0.5098420977592468),\n",
       " ('aquabirnavirus', 0.5096511840820312),\n",
       " ('hepatoencephalitis', 0.5096105337142944),\n",
       " ('The viral RNA cap machinery target antiviral drug Most virus modify genomic mRNA end addition RNA cap allow efficient mRNA translation limit degradation cellular 5′–3′ exonuclease avoid recognition foreign RNA host cell Viral RNA cap synthesize acquire use cap machinery exhibit significant diversity organization structure mechanism relative cellular host Therefore viral RNA cap emerge interesting field antiviral drug design Here review different pathway mechanism use produce viral mRNA cap present current structure mechanisms inhibitor know act viral RNA capping',\n",
       "  0.5095700025558472),\n",
       " ('phlebovirus', 0.5094994306564331),\n",
       " ('rudolph', 0.5094113945960999),\n",
       " ('culicoides', 0.5093517303466797),\n",
       " ('papbpc1', 0.5093023777008057),\n",
       " ('TGEV corona virus ORF4 encode membrane protein incorporate virion The cod potential open reading frame ORF4 amino acid transmissible gastroenteritis virus TGEV confirm expression use baculovirus vector Five monoclonal antibody MAbs raise 10K recombinant product immunoprecipitated polypeptide similar size TGEV infect cell Immunofluorescence assay perform insect mammalian cell reveal ORF4 membrane associate protein find consistent prediction membrane span segment ORF4 sequence Two epitope localize within last terminal residue sequence peptide scan analysis reactivity truncate ORF4 recombinant protein Since relevant MAbs find induce cell surface fluorescence data suggest ORF4 may integral membrane protein Cexo Nendo orientation Anti ORF4 MAbs also use show ORF4 polypeptide may detect TGEV virion preparation estimate number molecule incorporate per particle Comparison amino acid sequence data provide strong evidence coronaviruses encode polypeptide homologous TGEV ORF4 Our result lead propose ORF4 represent novel minor structural polypeptide tentatively designate small membrane protein',\n",
       "  0.5091968774795532),\n",
       " ('rbd–hdpp4', 0.5090909004211426),\n",
       " ('Fixed cell immunoperoxidase technique study surface antigen induce coronavirus transmissible gastroenteritis TGEV immunoperoxidase technique perform TGEV infect cell develop detection virus induce antigens The presence antigen TGEV surface infect cell demonstrate technique This finding contrast protein murine hepatitis coronavirus migrate Golgi apparatus transport plasma membrane The time course appearance antigens surface TGEV infect cell study technique',\n",
       "  0.5090516805648804),\n",
       " ('Contribution molecular biology study porcine interferon system perform molecular study pig interferon IFN system analyse role play endogenous IFN neonatal viral enteritis transmissible gastroenteritis possibly obtain via recombinant DNA technology new anti infectious immunomodulatory agent specie characterize structure biological function IFN like antiviral activity produce porcine embryo time implantation uterus probe porcine genomic library human porcine IFN probe isolate relate gene show porcine IFN multigene family include like several mammalian specie two subfamily relate distinct gene Class subfamily contain least locus locate chromosome among nine clone two potentially functional sequenced Class subfamily specifically express embryo ruminant implantation contain least seven locus among six clone One sequenced class loci PoIFN encode amino acid preprotein After removal sequence encode putative signal peptide terminal gene insert Escherichia coli bicistronic expression vector allow intracellular synthesis mature porcine IFN methionyl IFN Expression recombinant protein optimize insertion seven base pair long random synthetic sequence intercistronic region follow clone coli immunodetection clone express high amount recombinant protein The coli strain obtain produce high level protein exhibit vitro overall biological property leucocyte derive porcine IFN LeuIFN However strong antiviral effect porcine cell LeuIFN After immunoaffinity purification specific activity 5−10 International Units protein pharmacokinetic pharmacological study realize determine vivo half life rIFN pig These experiment reveal major toxic effect newborn give 106IU adult 106IU pig significant pyrogenic effect 5°C note adult',\n",
       "  0.5089828372001648),\n",
       " ('Mouse Hepatitis Coronavirus RNA Replication Depends GBF1 Mediated ARF1 Activation Coronaviruses induce infect cell formation double membrane vesicle sit RNA replication Not much know formation vesicle although recent observation indicate important role endoplasmic reticulum formation mouse hepatitis coronavirus MHV replication complex RCs show MHV replication sensitive brefeldin BFA Consistently expression dominant negative mutant ARF1 know mimic action drug inhibit MHV infection profoundly Immunofluorescence analysis quantitative electron microscopy demonstrate BFA block formation RCs per rather reduce number MHV RNA replication sensitive BFA MDCK cell know express BFA resistant guanine nucleotide exchange factor GBF1 Accordingly individual knockdown Golgi resident target BFA transfection small interfere RNAs siRNAs show GBF1 BIG1 BIG2 critically involve MHV RNA replication ARF1 cellular effector GBF1 also appear involved MHV replication siRNAs target small GTPase inhibit MHV infection significantly Collectively result demonstrate GBF1 mediate ARF1 activation require efficient MHV RNA replication reveal early secretory pathway MHV replication complex formation closely connect',\n",
       "  0.5087308883666992),\n",
       " ('trans splice leader sequence actin mRNA elegans While determine end elegans actin mRNAs discover nucleotide spliced leader sequence The leader sequence find mRNA three four nematode actin gene The leader also appear present nonactin mRNAs The actin mRNA leader sequence identical first nucleotide novel nucleotide RNA transcribe adjacent opposite orientation ribosomal gene The evidence suggest actin mRNA leader sequence acquire novel nucleotide transcript intermolecular trans splice mechanism',\n",
       "  0.5086767673492432),\n",
       " ('columbian', 0.50838702917099),\n",
       " ('novel two pronged strategy suppress host protein synthesis SARS coronavirus Nsp1 protein Severe acute respiratory syndrome coronavirus nsp1 protein suppress host gene expression include type interferon production promote host mRNA degradation inhibit host translation infect cell present evidence nsp1 novel two pronged strategy inhibit host translation gene expression Nsp1 bind 40S ribosomal subunit inactivate translational activity 40S subunit Furthermore nsp1 40S ribosome complex induce modification region cap mRNA template render template RNA translationally incompetent Nsp1 also induce RNA cleavage template carry internal ribosome entry site IRES encephalomyocarditis virus carry IRESs hepatitis cricket paralysis virus demonstrate nsp1 induce RNA modification template dependent speculate mRNAs underwent nsp1 mediate modification mark rapid turnover host RNA degradation machinery',\n",
       "  0.5083626508712769),\n",
       " ('methodsthirty', 0.5082152485847473),\n",
       " ('antananarivo', 0.5082026720046997),\n",
       " ('icv', 0.5081416368484497),\n",
       " ('9×107', 0.508056640625),\n",
       " ('vp1–vp3', 0.5080314874649048),\n",
       " ('ul23', 0.5077188611030579),\n",
       " ('meskauskas', 0.5076167583465576),\n",
       " ('column', 0.507609486579895),\n",
       " ('26–34', 0.5074694156646729),\n",
       " ('marama', 0.5073904991149902),\n",
       " ('Acidic induces fusion cell infect baculovirus form syncytia The enveloped baculovirus⧹insect cell system use extensively expression recombinant protein include viral fusion proteins test wild type baculovirus endogenous fusion protein activity Syncytia formation dye transfer capacitance change observe incubate infect Spodoptera frugiperda cell acidic medium consistent fusion protein activity Only short acidic pulse need trigger syncytium formation Identical result obtain recombinant baculovirus This new system convenient study activate cell cell fusion However use envelop virus study mechanism recombinant fusion proteins require caution',\n",
       "  0.5072099566459656),\n",
       " ('tcap', 0.5071632862091064),\n",
       " ('incontinent', 0.507097601890564),\n",
       " ('aggregative', 0.5070560574531555),\n",
       " ('Efficient Sensing Infected Cells Absence Virus Particles Blasmacytoid Dendritic Cells Blocked Viral Ribonuclease rns Plasmacytoid dendritic cell pDC show efficiently sense HCV HIV infect cell use virion free pathway Here demonstrate classical swine fever virus member Flaviviridae process much efficient term interferon alpha induction compare direct stimulation virus particle employment virus replicon particles infectious RNA replicate form novo virion exclude transfer virus donor cell pDC pDC activation infect cell mediate contact dependent RNA transfer pDC sensitive TLR7 inhibitor This inhibited drug affect cytoskeleton membrane cholesterol demonstrate unique viral protein ribonuclease activity viral protein pestiviruses efficiently prevent process This require intact ribonuclease function intracellular compartment propose pathway activation could particular importance virus tend mostly cell associate cause persistent infection non cytopathogenic',\n",
       "  0.5070381164550781),\n",
       " ('trifluoromethyloxirane', 0.5070027112960815),\n",
       " ('Envelope glycoprotein interaction coronavirus assembly Coronaviruses assemble bud smooth membrane intermediate Golgi compartment study association viral membrane glycoprotein formation virion envelope Using coimmunoprecipitation analysis demonstrate proteins mouse hepatitis virus MHV interact specifically form heteromultimeric complex infect cell These could detect detergent use solubilization cell virion carefully choose combination nonionic ionic deoxycholic acid detergent prove optimal Pulse chase experiment reveal newly make proteins engage complex formation different kinetics Whereas protein appear complex immediately synthesis newly synthesize protein lag phase min Newly make incorporated virus particle faster suggest associate preexist molecule Using vaccinia virus drive coexpression also demonstrate formation complex absence coronaviral protein Pulse chase labelings coimmunoprecipitation analyse reveal associate pre Golgi membrane unglycosylated form appear complex rapidly Since association detect protein export block brefeldin stable complex likely arise Golgi intermediate compartment Sucrose velocity gradient analysis show complexes heterogeneous high order suggest maintain homo heterotypic interaction complex colocalized alpha mannosidase resident Golgi protein They acquire Golgi specific oligosaccharide modification detect cell surface Thus protein transport plasma membrane retain Golgi complex association protein Because coronaviruses bud pre Golgi membrane result implies envelope glycoprotein complex determine site budding Yet self association MHV envelope glycoproteins high order complex indicative role sort viral membrane protein drive formation viral lipoprotein coat virus assembly',\n",
       "  0.5068943500518799),\n",
       " ('whole‐body', 0.5067866444587708),\n",
       " ('bioprocessing', 0.5067248344421387),\n",
       " ('stressors', 0.5066642761230469),\n",
       " ('refuge', 0.5066593289375305),\n",
       " ('Adenovirus mediate expression terminal domain SARS CoV spike protein sufficient induce apoptosis Vero cell The pro apoptotic property severe acute respiratory syndrome coronavirus SARS CoV structural protein study vitro monitor apoptosis indicator include chromatin condensation cellular DNA fragmentation cell membrane asymmetry demonstrate adenovirus mediate expression SARS CoV spike protein terminal domain induce apoptosis Vero cell time dosage dependent manner whereas expression terminal domain structural protein include envelope membrane nucleocapsid protein These finding suggest possible role protein SARS CoV induce apoptosis molecular pathogenesis SARS',\n",
       "  0.5064636468887329),\n",
       " ('bola', 0.5063821077346802),\n",
       " ('translation', 0.506355881690979),\n",
       " ('Molecular characterisation virulent canine coronavirus strain This paper characterise virulent strain canine coronavirus CCoV isolate internal organ pup die systemic disease without evidence common canine pathogen High viral RNA titre detect internal organ real time PCR assay specific CCoV type Sequence analysis end 7kb genomic RNA strain reveal conserve structural well non structural protein exception truncate form non structural protein The exceptional form due nucleotide deletion frame shift ORF3b introduce early stop codon phylogenetic analysis structural protein spike protein find cluster feline coronavirus type strain whereas envelope membrane nucleocapsid proteins segregate together reference strain Purdue transmissible gastroenteritis virus swine',\n",
       "  0.5061543583869934),\n",
       " ('Nuclease escape element protect messenger RNA cleavage multiple viral endonuclease During lytic Kaposi sarcoma associate herpesvirus KSHV infection viral endonu clease SOX promote widespread degradation cytoplasmic messenger RNA mRNA However select mRNAs include transcript encode interleukin escape SOX induce cleavage escape mediate UTR RNA regulatory element override SOX target mechanism Here reveal protective RNA element function broadly restrict cleavage range homologous non homologous viral endonuclease However impede cleavage cellular endonucleases The protective sequence may representative large class nuclease escape element identify similar protective element GADD45B mRNA The GADD45B derive element display similarity sequence putative structure several associate RNA bind protein However overall composition ribonucleoprotein complex appear distinct lead difference breadth nuclease restrict These finding highlight RNA element selectively control transcript abundance background widespread virus induce mRNA degradation',\n",
       "  0.506125807762146),\n",
       " ('entitle', 0.506089985370636),\n",
       " ('immunoabsorbent', 0.5059534311294556),\n",
       " ('rs12979860', 0.5057452321052551),\n",
       " ('forl', 0.5056908130645752),\n",
       " ('begomovirus', 0.5056328773498535),\n",
       " ('suck', 0.5055162906646729),\n",
       " ('versleijen', 0.5054658651351929),\n",
       " ('femv', 0.5053866505622864),\n",
       " ('σns', 0.5052716732025146),\n",
       " ('session', 0.5052205324172974),\n",
       " ('cgmmv', 0.5051553845405579),\n",
       " ('cosedimented', 0.5051066279411316),\n",
       " ('microdispensing', 0.5050602555274963),\n",
       " ('masculino', 0.5050538182258606),\n",
       " ('biot', 0.505010724067688),\n",
       " ('1ml', 0.5050009489059448),\n",
       " ('chloromethyl', 0.5049322247505188),\n",
       " ('22p', 0.5049266219139099),\n",
       " ('Construction characterization improve DNA launch infectious clone duck hepatitis virus type BACKGROUND DNA launch infectious system useful tool high rescue efficiency allow introduction mutation specific position investigate function individual viral element Rescued virus particle could harvest directly transfecting DNA launch recombinant plasmid host cell reduce labor experimental cost skip vitro transcription assay METHODS total four overlap fragment cover entire viral genome amplify assembled transformation vector base pIRES2 EGFP establish DNA launch infectious system duck hepatitis virus type DHAV name pIR DHAV Reverse transcription polymerase chain reaction PCR detection quantitative real time polymerase chain reaction qRT PCR western blot assay indirect immunofluorescence IFA conduct rescue virus identification total recombinant plasmid pIR DHAV vitro transcribe product RNA launch infectious clone name DHAV transfected BHK cell analyze rescue efficiency Following tissue tropism rescue virus rDHAV parental virus pDHAV assayed virulence test day old duckling RESULTS Rescued virus particle carry design genetic marker could harvest directly transfecting pIR DHAV BHK cell The qRT PCR western blot result indicate rDHAV share similar growth characteristic pDHAV Furthermore DNA launch infectious system possess much high rescue efficiency assay compare RNA launch infectious system The mutation position impact viral replication tissue tropism From post infection hpi hpi viral RNA copy rDHAV liver high among six test tissue exception thymus hpi viral RNA copy number heart kidney alternately low CONCLUSION construct genetically stable highly pathogenic DNA launch infectious clone rescue virus could harvest direct transfection recombinant plasmid rDHAV share similar growth characteristic tissue tropism pDHAV The DNA launch infectious system DHAV possess high rescue efficiency compare traditional RNA launch infectious system',\n",
       "  0.5048807859420776),\n",
       " ('saplovirus', 0.5048441886901855),\n",
       " ('Characterization efficient coronavirus ribosomal frameshifting signal Requirement RNA pseudoknot The genomic RNA coronavirus IBV contain efficient ribosomal frameshifting signal junction two overlap open read frame define deletion analysis nucleotide sequence encompass overlap region sufficient allow frameshifting heterologous context The upstream boundary signal consist sequence UUUAAAC likely site ribosomal slippage show creation complementary nucleotide change RNA downstream slippery sequence fold tertiary structure term pseudoknot formation essential efficient frameshifting',\n",
       "  0.5048345327377319),\n",
       " ('boid', 0.504684567451477),\n",
       " ('Improved microarray gene expression profile virus infect cell removal viral RNA BACKGROUND Sensitivity accuracy key point use microarrays detect alteration gene expression different condition Critical acquisition reliable result preparation RNA field virology analyze host cell reaction infection often high representation viral RNA vRNA within total RNA preparation infect cell likely interfere microarray analysis Yet effect investigate despite many report describe gene expression profile virus infect cell use microarrays RESULTS study use coronaviruses model show vRNA indeed interfere microarray analysis decrease sensitivity accuracy also demonstrate removal vRNA total RNA sample mean virus specific oligonucleotide capture significantly reduce number false positive hit increase sensitivity method test different array platform CONCLUSION therefore recommend specific removal vRNA abundant contaminate RNAs total RNA sample improve quality reliability microarray analysis',\n",
       "  0.5045499801635742),\n",
       " ('syphilis—has', 0.5045483708381653),\n",
       " ('sphingolipid', 0.5045068264007568),\n",
       " ('Cultivation partial characterization bovine astrovirus Bovine astrovirus serotype US2 adapt primary neonatal kidney cell NBK culture addition ml−1 trypsin medium Infectious virus release cell within day post infection early passage within day later passage absence trypsin neither passage infect cell release infectious virus occur The virus show similar fecal astrovirus neutralization test ultrastructural study infect cell Primary embryo bovine kidney EBK NBK cell culture support infection fecal tissue culture adapt TCA astrovirus The time relate development infection study immunofluorescence similar fecal TCA astrovirus cell culture type The first indication viral infection expression viral antigen occur post infection characterize appearance diffuse faint immunofluorescence cytoplasm Soon two three brilliant granules observe nucleus appeared involve nucleoli Subsequently dense granular see perinuclear region cytoplasm later extend involve cytoplasmic area EBK NBK culture infect either fecal tissue culture adapt astrovirus minority cell become infected even multiplicity infection exceed one Occasionally 10–20 cell infected culture proportion exceed NBK culture calf infected cell observe The virus infect bovine cell line Infectivity virus remove treatment chloroform iododeoxyuridine actinomycin add medium block replication Masses virion observe electron microscopy discrete area cytoplasm similar distribution viral antigen focus see The mean diameter virion conclusion bovine astrovirus lack essential lipid envelope probably RNA genome may nuclear phase replication involve nucleoli block DNA inhibitor selective cell tropism',\n",
       "  0.5044898986816406),\n",
       " ('The methyltransferase domain dengue virus protein NS5 ensure efficient RNA synthesis initiation elongation polymerase domain Viral RNA dependent RNA polymerases RdRps responsible replication single strand RNA virus genome exert function context complex replication machinery Within replication complex polymerase activity often highly regulate RNA element proteins domain multi domain polymerase Here present data influence methyltransferase domain NS5 MTase dengue virus DENV protein NS5 RdRp activity polymerase domain NS5 Pol The steady state polymerase activity DENV recombinant NS5 NS5 Pol compare use different biochemical assay allow dissection novo initiation transition elongation step RNA synthesis show NS5 MTase ensure efficient RdRp activity stimulate novo initiation elongation phase This stimulation relate high affinity NS5 toward single strand RNA template indicate NS5 MTase either complete high affinity RNA bind site promotes correct formation template tunnel Furthermore NS5 MTase increase affinity prim nucleotide ATP upon novo initiation cause high catalytic efficiency polymerase upon elongation The complex stimulation pattern discuss perspective NS5 adopt several conformation RNA synthesis',\n",
       "  0.504409670829773),\n",
       " ('abstracthost', 0.5043556690216064),\n",
       " ('apamin', 0.5043114423751831),\n",
       " ('bonaventure', 0.5041577219963074),\n",
       " ('aya', 0.5041385293006897),\n",
       " ('spielburger', 0.5040774345397949),\n",
       " ('The RNA polymerase activity SARS coronavirus nsp12 primer dependent RNA dependent RNA polymerase RdRp central catalytic subunit RNA synthesize machinery positive strand RNA viruses Usually RdRp domains readily identifiable comparative sequence analysis biochemical confirmation characterization hamper intrinsic protein property technical complication presume replication transcription ∼30 severe acute respiratory syndrome SARS coronavirus SARS CoV RNA genome catalyze RdRp domain terminal part nonstructural protein nsp12 one replicase subunit However thus far full length nsp12 proven refractory expression bacterial system hinder biochemical characterization coronavirus RNA synthesis RdRp target antiviral drug design Here describe combine strategy involve bacterial expression nsp12 fusion protein vivo cleavage generate purify stable SARS CoV nsp12 kDa natural terminus terminal hexahistidine tag This recombinant protein possess robust vitro RdRp activity well significant DNA dependent activity may facilitate future inhibitor study The SARS CoV nsp12 primer dependent homo heteropolymeric template support likeliness close enzymatic collaboration intrigue RNA primase activity recently propose coronavirus nsp8',\n",
       "  0.5036673545837402),\n",
       " ('3–1', 0.50362628698349),\n",
       " ('y494', 0.5035080909729004),\n",
       " ('vasodilatation', 0.5034515857696533),\n",
       " ('recurrences', 0.5034185647964478),\n",
       " ('Synergistic Roles Eukaryotic Translation Elongation Factors 1Bγ Stimulation Tombusvirus Minus Strand Synthesis Host factor recruit viral replicase complex aid replication plus strand RNA viruses paper show deletion eukaryotic translation elongation factor 1Bgamma eEF1Bγ reduces Tomato bushy stunt virus TBSV replication yeast host Also knock eEF1Bγ level plant host decrease TBSV accumulation eEF1Bγ bind viral RNA one resident host protein tombusvirus replicase complex Additional vitro assay whole cell extract prepare yeast strain lack eEF1Bγ demonstrate role minus strand synthesis opening structure end viral RNA reduce possibility utilization strand template repeat strand synthesis within replicase also show eEF1Bγ play synergistic role eukaryotic translation elongation factor tombusvirus replication possibly via stimulation proper position viral RNA dependent RNA polymerase promoter region viral RNA template These role translation factor TBSV replication separate canonical role host viral protein translation',\n",
       "  0.5033377408981323),\n",
       " ('tiar', 0.5033326745033264),\n",
       " ('serpulina', 0.503214955329895),\n",
       " ('Characterization murine hepatitis virus JHM RNA rat experimental encephalomyelitis When Wistar Furth rat inoculate intracerebrally murine hepatitis virus JHM often develop demyelinate disease result hind leg paralysis Using RNA transfer procedure hybridization kinetic analysis virus specific RNA rat characterize The pattern JHM specific RNA vary individual infection Wistar Furth rat However two specie JHM specific RNA nucleocapsid 1–2 RNA specie generally present general decrease JHM specific RNA brain spinal cord sample take later day postinoculation observe however JHM specific RNA persist spinal cord longer brain rat',\n",
       "  0.503034770488739),\n",
       " ('s216g', 0.5029876232147217),\n",
       " ('gp90', 0.502976655960083),\n",
       " ('hached', 0.502872884273529),\n",
       " ('Inhibition hepatitis virus replication ligand mediate activation RNase RNase cellular endoribonuclease activate link oligoadenylates 2–5A unique specific ligand synthesize family interferon inducible dsRNA activate enzymes name oligoadenylate synthetases typical antiviral pathway activate RNase degrade viral cellular RNAs thus limit viral replication spread Although antiviral activity RNase demonstrate several RNA virus little evidence regard role DNA virus present study potential antiviral activity RNase hepatitis virus HBV explore utilize recently report infection protocol base human hepatoma HepG2 cell stably complement virus entry factor NTCP Viral replication expression cell type markedly inhibit poly 2–5A mediated activation RNase however inhibition significantly reverse RNase knockdown Further analysis HBV1 transfected Huh hepatoma cell indicate antiviral activity RNase depend ribonuclease function also provide evidence specific role OAS family member process These result suggest HBV replication regulate interferon mediate RNA decay pathway activation host antiviral factor may represent novel therapeutic strategy HBV infection',\n",
       "  0.5028693079948425),\n",
       " ('wolves', 0.5026946067810059),\n",
       " ('accessibility', 0.5024998188018799),\n",
       " ('Assembly vaccinia virus role intermediate compartment endoplasmic reticulum Golgi stack Vaccinia virus prototype Poxviridae large DNA virus replicate cytoplasm host cell The assembly pathway vaccinia virus display several unique feature production two structurally distinct infectious form One term intracellular naked virus INV remain cell associate term extracellular envelop virus EEV release cell addition long believe INVs acquire lipid envelope unique example novo membrane biogenesis examine structure assembly vaccinia virus use immunoelectron microscopy use antibody proteins different subcellular compartment well phospholipid analysis purify INV EEV Our data consistent novo model viral membrane synthesis rather argue vaccinia virus DNA become enwrap membrane cisterna derive intermediate compartment Golgi stack thus acquire two membrane one step Phospholipid analysis purify INV support derivation early biosynthetic compartment This unique assembly process repeat INV becomes enwrap additional membrane cisterna agreement early report The available data suggest fusion outer envelope plasma membrane mature EEV release cell',\n",
       "  0.5024096369743347),\n",
       " ('4–94', 0.5021429061889648),\n",
       " ('Interaction Coronavirus Infectious Bronchitis Virus Membrane Protein Actin Its Implication Virion Assembly Budding Coronavirus protein essential component virion play pivotal role virion assembly bud maturation The protein integrate viral envelope three transmembrane domain flank short amino terminal ectodomain large carboxy terminal endodomain study show purification protein coronavirus infectious bronchitis virus IBV actin understand cellular factor may involve virion assembly bud maturation process IBV use bait yeast two hybrid screen result identification actin potentially interacting partner This interaction subsequently confirm coimmunoprecipitation immunofluorescence microscopy mammalian cell mutation amino acid A159 K160 protein abolish interaction Introduction A159 K160 mutation infectious IBV clone system block infectivity clone although viral RNA replication subgenomic mRNA transcription actively detect Disruption actin filament cell permeable agent cytochalasin early stage infection cycle lead detection viral protein synthesis infect cell release virus particle culture medium However treatment late stage infection cycle affect release virus particle medium suggest disruption actin filament might block virion assembly bud release virus particles This study reveal essential function actin replication cycle coronavirus',\n",
       "  0.5017061233520508),\n",
       " ('Efficient amplification clone near full length hepatitis virus genome clinical sample Long PCR LRP amplification RNA templates sometimes difficult compare long PCR DNA templates Among RNA templates hepatitis virus HCV represent excellent example challenge potential LRP technology due extensive secondary structure difficulty readily culture vitro The source viral genome amplification clinical sample HCV usually present low titer create comprehensive optimization protocol allow robust amplification 1kb fragment HCV follow efficient clone novel vector Detailed analyse indicate lack potential LRP mediate recombination preservation viral diversity Thus LRP protocol could apply amplification difficult RNA template may facilitate RNA virus research link viral mutation reverse genetics',\n",
       "  0.5016682147979736),\n",
       " ('pertinently', 0.5015527009963989),\n",
       " ('mauvais', 0.5013127326965332),\n",
       " ('jhm', 0.5012450218200684),\n",
       " ('négatifs', 0.5011844635009766),\n",
       " ('inflamação', 0.5011757612228394),\n",
       " ('countries—anthrax', 0.5011584758758545),\n",
       " ('rpf', 0.5011277198791504),\n",
       " ('spermatid', 0.5010801553726196),\n",
       " ('chain—can', 0.5009899139404297),\n",
       " ('gilbert', 0.5009502172470093),\n",
       " ('o153', 0.5008154511451721),\n",
       " ('Identification link glycosylation site spike protein functional impact replication infectivity coronavirus infectious bronchitis virus cell culture Spike glycoprotein viral envelope main determinant infectivity The protein coronavirus infectious bronchitis virus IBV contain putative asparagine link glycosylation site These post translational modification may assist protein fold play important role functionality protein study use bioinformatics tool predict link glycosylation site analyze distribution IBV strain variant Among site site confirm protein extract partially purify virus particle proteomics approach substitution predict site introduce infectious clone system The impact protein mediate cell cell fusion viral recovery infectivity assess leading identification sit essential function IBV protein Further characterization uncharacterized site may reveal novel aspect link glycosylation coronavirus replication pathogenesis',\n",
       "  0.500800371170044),\n",
       " ('zanzibar', 0.5007656216621399),\n",
       " ('deae', 0.5007539391517639),\n",
       " ('Detection rabbit haemorrhagic disease virus RHDV situ hybridisation digoxigenin label RNA probe situ hybridisation ISH technique detection rabbit haemorrhagic disease virus RHDV develop Thirteen seronegative adult rabbit infect oro nasally use BS89 RHDV strain Liver spleen sample collect post infection repeat every till Each sample test immunohistochemically sandwich ELISA ISH RNA probe match genomic fragment cod VP60 structural protein RHDV arrange Two RNA probe sense antisense transcribe vitro UTP digoxigenin label The antisense probe clearly detect positivity cytoplasm hepatocytes Labelled hepatocytes scatter throughout section follow diffuse perilobular positive reaction much weak signal similar distribution detect use sense RNA probe All spleen sample test negative probe Liver sample positive use ELISA immunoperoxidase test Spleen sample positive use ELISA This study show RHDV replication occur almost immediately inoculation liver appear main site replication',\n",
       "  0.5007479190826416),\n",
       " ('Hepatitis Virus Genotype form core like particle unique structural property SUMMARY determine structure core capsid unusual variant hepatitis virus genotype HBV resolution use cryo electron microscopy The structure reveal surface feature present prototype HBV genotype HBV novel unique insertion downstream core gene start codon This result twelve amino acid insertion terminal end core protein two stop codon precore region prevent expression HBeAg HBV replication patient associate infection another genotype HBV suggest HBV may reduce replication efficiency vivo localize terminal insertion HBV show form two additional mass core surface adjacent dimer spike model structure additional residue within density show position insertion would interfere translocation nucleic acid pore core interior compartment However insertion may partially obscure several residue core surface know play role envelopment secretion virion could affect structural rearrangement may trigger envelopment DNA second strand synthesis',\n",
       "  0.5006687641143799),\n",
       " ('Global Organization Positive strand RNA Virus Genome The genome plus strand RNA viruses contain many regulatory sequence structure direct different viral process The traditional view RNA elements local structure present non cod region However view change due discovery regulatory element cod region functional long range intra genomic base pairing interaction The long RNA genome tombusvirus tomato bushy stunt virus TBSV contain type structural feature include six different functional long distance interaction hypothesize achieve multiple interaction viral genome must utilize large scale organizational strategy accordingly seek ass global conformation entire TBSV genome Atomic force micrographs genome indicate mostly condensed structure compose interconnected protrusion extend central hub This configuration consistent genomic secondary structure model generate use high throughput selective hydroxyl acylation analyse primer extension SHAPE predict different size RNA domains originate central region Known RNA element identify domain inter domain region novel structural feature predict functionally confirm Interestingly two six long range interaction know form present structural model However interaction form complementary partner sequence position relatively close structure suggest secondary structure level viral genome structure could provide basic scaffold formation different long range interaction The high order structural model TBSV RNA genome provide snapshot complex framework allow multiple functional component operate concert within confine context',\n",
       "  0.500162661075592),\n",
       " ('richness', 0.5001430511474609),\n",
       " ('ngna', 0.5001112222671509),\n",
       " ('aminoethylglycine', 0.5000292658805847),\n",
       " ('crinivirus', 0.49996644258499146),\n",
       " ('Mouse Hepatitis Virus Strain JHM Infects Human Hepatocellular Carcinoma Cell Line Mouse hepatitis virus MHV strain JHM coronavirus cause encephalitis demyelination susceptible rodents The known receptor MHV member carcinoembryonic antigen family Although human form MHV receptor function MHV receptor assay human cell line identify support wild type MHV infection Here describe infection human hepatocellular carcinoma cell line HuH MHV HuH cell susceptible strain JHM JHM yield virus titer nearly identical see mouse DBT cell contrast HuH cell marginally susceptible completely resistant infection MHV strain include A59 JHM produce strong cytopathic effect HuH cell formation round plaques Studies various recombinant virus JHM A59 strain suggest ability JHM infect HuH cell determine multiple viral genetic element Blocking viral spike protein neutralize antibody soluble form MHV receptor inhibit infection HuH cell suggest infection mediate protein Transfection prototype mouse receptor biliary glycoprotein render HuH cell susceptible infection MHV strain well suggest JHM use receptor distinct classical MHV receptor infect HuH cell Possible implication human disease discuss',\n",
       "  0.499919056892395),\n",
       " ('impliqué', 0.4999023377895355),\n",
       " ('flaviviridae', 0.4996633529663086),\n",
       " ('parameterisation', 0.49963873624801636),\n",
       " ('unenveloped', 0.49963128566741943),\n",
       " ('synchronous', 0.49952322244644165),\n",
       " ('90a', 0.4995025396347046),\n",
       " ('O030 Evasion OAS RNase pathway murine coronavirus ns2 protein require viral replication hepatitis Introduction The oligoadenylate synthetase ribonuclease OAS RNase system potent IFN induce antiviral pathway Following infection IFNs induce group OAS gene whose product activate viral double strand RNA OAS use ATP generate link oligoadenylates bind activate ubiquitous cellular endoribonuclease RNase cause cleavage single strand region viral cellular RNA thus inhibit viral replication addition detection newly generate short RNAs cellular pattern recognition receptor MDA5 RIG enhance IFN production ensue antiviral activity The intracellular concentration believe primary factor control RNase activation The liver contain abundant innate immune cell provide first line defense pathogen However factor determine whether virus bypass defense access infect liver parenchyma well understood The murine coronavirus mouse hepatitis virus MHV strain A59 infection mouse provide model virus induce hepatitis The MHV accessory protein ns2 antagonize type IFN response macrophage promote induction hepatitis Here describe ns2 protein facilitate development viral hepatitis block OAS RNase pathway Methods Bone marrow macrophage BMM wild type RNase mouse infect A59 ns2 mutant MHV Viral titer determine plaque assay RNase activity monitor rRNA integrity RNA chip Intracellular level measure use RNase activation assay Effects ns2 level cell determine transfecting ns2 mutant ns2 cDNAs HEK 293T cell Recombinant ns2 mutant ns2H126R protein purify incubated vitro breakdown product measure HPLC Hepatitis determine histology follow inoculation A59 ns2 mutant MHV RNase mouse Results find evidence new molecular mechanism subversion RNase pathway macrophage regulate acute hepatitis MHV infection Coronavirus ns2 belongs LigT like protein family within phosphoesterase superfamily possess cyclophosphodiesterase activity CPD find ns2 CPD instead phosphodiesterase PDE cleave thus eliminate activator RNase observe ns2 block IFN inducible OAS RNase pathway facilitate hepatitis development Ns2 prevents activation RNase consequently limit viral RNA degradation ns2 mutant virus unable replicate liver induce hepatitis mouse highly pathogenic RNase mouse Thus RNase critical cellular factor protection viral infection liver result hepatitis Conclusion MHV accessory protein ns2 PDE degrades limit RNase activation thus facilitate virus induce hepatitis mouse',\n",
       "  0.4994009733200073),\n",
       " ('200–280', 0.4993974566459656),\n",
       " ('g1595', 0.49937504529953003),\n",
       " ('ribosomal', 0.49934497475624084),\n",
       " ('mnpv', 0.49931854009628296),\n",
       " ('One tube fluorogenic reverse transcription polymerase chain reaction quantitation feline coronaviruses one tube reverse transcription polymerase chain reaction PCR absolute feline coronavirus FCoV quantitation develop The assay base nuclease activity Thermus flavus Tfl polymerase fluorogenic probe generates fluorescence cleave The fluorogenic probe also call TaqMan™ probe Perkin Elmer Foster City USA oligonucleotide design bind two PCR primer target cDNA label reporter quencher dye intact probe quencher dye suppresses fluorescence reporter dye Förster type energy transfer During polymerase extension step Tfl exonuclease activity cleaves hybridise probe result generation fluorescent emission reporter dye The threshold cycle value indicate increase reporter fluorescence directly relate initial amount target cDNA RNA respectively Fluorescence monitor real time cycle Perkin Elmer ABI Prism® Sequence Detector After completion amplification value sample calculate back standard curve generate amplification dilute standard molecules The one tube PCR describe allows precise quantitation highly sensitive rapid separate reverse transcription step post amplification step easy handle allows high sample throughput show good reproducibility execute low risk contamination The design primers–probe combination enable detection know FCoV strain also useful detection canine coronavirus transmissible gastroenteritis virus porcine respiratory coronavirus',\n",
       "  0.4992724657058716),\n",
       " ('psks', 0.4991765320301056),\n",
       " ('va70', 0.4989665448665619),\n",
       " ('sofia®', 0.49896240234375),\n",
       " ('bedded', 0.49895986914634705),\n",
       " ('Detection Fourth Orbivirus Non Structural Protein The genus Orbivirus include insect tick borne viruses The orbivirus genome compose segment dsRNA encodes structural protein VP1–VP7 non structural protein NS1–NS3 open reading frame ORF span almost entire length genome segment Seg encodes VP6 viral helicase However bioinformatic analysis recently identify overlap ORF ORFX Seg show ORFX encode new non structural protein identify NS4 Western blot confocal fluorescence microscopy use antibody raise recombinant NS4 Bluetongue virus BTV insect borne Great Island virus GIV tick borne demonstrate protein synthesise BTV GIV infect mammalian cell respectively BTV NS4 also express Culicoides insect cell NS4 form aggregate throughout cytoplasm well nucleus consistent identification nuclear localisation signal within NS4 sequence Bioinformatic analyse indicate NS4 contains coil coil related protein bind nucleic acid associate membrane show similarity nucleolar protein UTP20 processome subunit Recombinant NS4 GIV protect dsRNA degradation endoribonucleases RNAse III family indicate interacts dsRNA However BTV NS4 half putative size GIV NS4 protect dsRNA RNAse III cleavage NS4 GIV BTV protect DNA degradation DNAse NS4 find associate lipid droplet cell infect BTV GIV transfected plasmid expressing NS4',\n",
       "  0.4988965690135956),\n",
       " ('Baculovirus expression HCoV OC43 nucleocapsid protein development Western blot assay detection human antibody HCoV OC43 The nucleocapsid gene human coronavirus strain OC43 HCoV OC43 amplify reverse transcriptase polymerase chain reaction clone pENTR™ TOPO® plasmid This plasmid contain gene recombine BaculoDirect™ baculovirus DNA design order express protein fusion terminal polyhistidine tag contain epitope Sf21 cell transfected recombinant baculovirus DNA Recombinant protein extract infected cell analyse SDS PAGE Western blot purify Ni2 affinity procedure Sera healthcare worker five 2–3 year old child test Western blot assay use purify recombinant protein All serum adult two serum child positive result',\n",
       "  0.4988032579421997),\n",
       " ('Detection genetically modify organisms GMOs use isothermal amplification target DNA sequence BACKGROUND The common method GMO detection base upon amplification GMO specific DNA amplicons use polymerase chain reaction PCR Here apply loop mediate isothermal amplification LAMP method amplify GMO relate DNA sequence internal commonly use motif control transgene expression event specific plant transgene junction RESULTS test specificity sensitivity technique use GMO study Results show detection GMO equivalent background DNA possible dilution template suggest detection single copy template may possible use LAMP CONCLUSION This work show GMO detection carry use LAMP routine screen well specific event detection Moreover sensitivity ability amplify target even high background DNA demonstrate highlight advantage isothermal amplification apply GMO detection',\n",
       "  0.4988026022911072),\n",
       " ('The coronavirus avian infectious bronchitis virus require cell nucleus host transcriptional factor Replication avian infectious bronchitis virus permissive BHK cell block cell enucleate irradiated ultraviolet light prior infection cell treat amanitin virus growth cycle This coronavirus like influenza virus replicate normally presence amanitin Chinese hamster ovary cell possess drug resistant RNA polymerase These finding indicate avian infectious bronchitis virus require intact cell nucleus one host transcriptional function productive infection Preliminary data suggest cellular function involve aspect virus direct RNA synthesis',\n",
       "  0.4987488389015198),\n",
       " ('b54gfp', 0.4986720681190491),\n",
       " ('Multiple sit recombination within RNA genome foot mouth disease virus Recombinant foot mouth disease virus isolate cell infected mixture temperature sensitive mutant belong different subtype strain order select recombination event many different region genome cross perform various pair mutant mutation different region genome progeny analyse electrofocusing virus induce protein RNase fingerprint RNA All independent isolates nine cross prove recombinant RNA genome Maps genome base knowledge location unique oligonucleotides construct Most could interpret product single genetic cross over although three recombinant form two cross over Cross over least twelve distinct region genome identify This evidence large number recombination sit suggest RNA recombination picornaviruses general oppose site specific phenomenon',\n",
       "  0.4986101984977722),\n",
       " ('tripeptidase', 0.49850818514823914),\n",
       " ('Tandem leader protease Grapevine leafroll associate virus Host specific function infection cycle Several virus genus Closterovirus include Grapevine leafroll associate virus GLRaV encode tandem papain like leader protease whose functional profile remain largely uncharacterized generate series full length reporter tag clone GLRaV demonstrate systemically infectious upon agroinfection experimental host plant Nicotiana benthamiana These clone correspond minireplicon derivative use address function GLRaV infection cycle find deletion genome region encode entire L1–L2 tandem result fold reduction minireplicon RNA accumulation Five fold reduction RNA level observe upon deletion cod region contrast deletion cod region affect RNA accumulation also find autocatalytic cleavage essential genome replication Analysis correspond mutant context benthamiana infection launch full length GLRaV clone reveal cod region essential virus ability establish infection play accessory role viral systemic transport Strikingly tag minireplicon variant use leaf agroinfiltration GLRaV natural host Vitis vinifera deletion either result dramatic reduction minireplicon ability establish infection attest host specific requirement tandem proteases virus infection cycle',\n",
       "  0.4984322786331177),\n",
       " ('summarycoronaviruses', 0.4983147978782654),\n",
       " ('titrate', 0.4983143210411072),\n",
       " ('spv', 0.498159259557724),\n",
       " ('superfamily', 0.49796178936958313),\n",
       " ('cyclopentylcytosine', 0.49786633253097534),\n",
       " ('Isolation sequence AGO bound RNAs reveal characteristic mammalian stem loop processing vivo ABSTRACTMicroRNA precursor pre miRNAs short hairpin RNAs rapidly process mature microRNAs miRNAs cytoplasm Due low abundance cell sequence base study pre miRNAs limit successfully enrich deep sequence pre miRNAs human cell capture RNAs interaction Argonaute AGO proteins Using approach detect pre miRNAs human cell pre miRNAs reanalysis similar study mouse cell uncover widespread trimming non templated addition end pre mature miRNAs Additionally create index microRNA precursor processing efficiency This analysis reveal subset pre miRNAs produce low level mature miRNAs despite abundant precursor include annotate miRNA UTR DiGeorge syndrome critical region mRNA transcript This lead search AGO associate stem loop originate mRNA specie identify hundred putative pre miRNAs derive human mouse mRNAs summary provide wealth information mammalian pre miRNAs identify novel microRNA microRNA like element localize mRNAs',\n",
       "  0.4977489113807678)]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "similarity_finder(\"d100\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Word closeness"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_words =tokenizer.word_counts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_words_freq = sorted(list(all_words.items()), key=lambda x: x[1])[-2000:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_words_freq = [x[0] for x in all_words_freq]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "        <script type=\"text/javascript\">\n",
       "        window.PlotlyConfig = {MathJaxConfig: 'local'};\n",
       "        if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}\n",
       "        if (typeof require !== 'undefined') {\n",
       "        require.undef(\"plotly\");\n",
       "        requirejs.config({\n",
       "            paths: {\n",
       "                'plotly': ['https://cdn.plot.ly/plotly-latest.min']\n",
       "            }\n",
       "        });\n",
       "        require(['plotly'], function(Plotly) {\n",
       "            window._Plotly = Plotly;\n",
       "        });\n",
       "        }\n",
       "        </script>\n",
       "        "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import gensim\n",
    "import numpy as np\n",
    "from sklearn.manifold import TSNE\n",
    "from nltk.corpus import stopwords\n",
    "import plotly.offline as plt\n",
    "import plotly.graph_objs as go\n",
    "plt.init_notebook_mode(connected=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.cluster import KMeans\n",
    "import numpy as np\n",
    "\n",
    "\n",
    "def get_coordinates(model, words):\n",
    "    X = []\n",
    "    arr = np.empty((0,64), dtype='f')\n",
    "    labels = []\n",
    "    for wrd_score in words:\n",
    "        try:\n",
    "            X.append(model[str(word_idx[wrd_score])])\n",
    "            arr = np.append(arr, np.array([model[str(word_idx[wrd_score])]]), axis=0)\n",
    "            labels.append(wrd_score)\n",
    "        except Exception as e:\n",
    "            print(e)\n",
    "            pass\n",
    "  \n",
    "    tsne = TSNE(n_components=2, random_state=0)\n",
    "    np.set_printoptions(suppress=True)\n",
    "    Y = tsne.fit_transform(arr)\n",
    "    x_coords = Y[:, 0]\n",
    "    y_coords = Y[:, 1]\n",
    "    return x_coords, y_coords, labels, X"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "x, y, labels, X = get_coordinates(model, all_words_freq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "kmeans = KMeans(n_clusters=20, random_state=0).fit(X)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'kmeans' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-12-6306cd9c2eb9>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      5\u001b[0m                     \u001b[0mtextposition\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'bottom center'\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m                     \u001b[0mhoverinfo\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m'text'\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m                     marker=dict(size=5,opacity=0.8, color=kmeans.labels_))]\n\u001b[0m\u001b[1;32m      8\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      9\u001b[0m \u001b[0mlayout\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mgo\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mLayout\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtitle\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'prod cat projection'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'kmeans' is not defined"
     ]
    }
   ],
   "source": [
    "plot = [go.Scatter(x = x,\n",
    "                    y = y,\n",
    "                    mode = 'markers+text',\n",
    "                    text = labels,\n",
    "                    textposition='bottom center',\n",
    "                    hoverinfo = 'text',\n",
    "                    marker=dict(size=5,opacity=0.8, color=kmeans.labels_))]\n",
    "\n",
    "layout = go.Layout(title='prod cat projection')\n",
    "# fig = go.Figure(data=plot, layout=layout)\n",
    "# plt.iplot(fig)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Document similarity"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.cluster import KMeans\n",
    "import numpy as np\n",
    "\n",
    "\n",
    "def get_coordinates(model, words):\n",
    "    X = []\n",
    "    arr = np.empty((0,64), dtype='f')\n",
    "    labels = []\n",
    "    for wrd_score in words:\n",
    "        try:\n",
    "            X.append(model[wrd_score])\n",
    "            arr = np.append(arr, np.array([model[wrd_score]]), axis=0)\n",
    "            labels.append(wrd_score)\n",
    "        except Exception as e:\n",
    "            print(e)\n",
    "            pass\n",
    "  \n",
    "    tsne = TSNE(n_components=3, random_state=0)\n",
    "    np.set_printoptions(suppress=True)\n",
    "    Y = tsne.fit_transform(arr)\n",
    "    x_coords = Y[:, 0]\n",
    "    y_coords = Y[:, 1]\n",
    "    z_coords = Y[:, 2]\n",
    "    return x_coords, y_coords,z_coords, labels, X"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "words = [\"d\"+str(i) for i in range(18500)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "x, y,z_coords, labels, X = get_coordinates(model, words)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [],
   "source": [
    "kmeans = KMeans(n_clusters=10, random_state=0).fit(X)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [],
   "source": [
    "km_labels = list(kmeans.labels_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "18500"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# indices = [i for i, x in enumerate(kmeans.labels_) if x == 2]\n",
    "len(km_labels)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1349\n",
      "1717\n",
      "1658\n",
      "2455\n",
      "2309\n",
      "1361\n",
      "2508\n",
      "1195\n",
      "1476\n",
      "2472\n"
     ]
    }
   ],
   "source": [
    "import random\n",
    "X_new, y_new,z_new, labels_new = [], [], [],[]\n",
    "\n",
    "kmeans_labels = []\n",
    "\n",
    "for i in range(10):\n",
    "    indices =[]\n",
    "    for idx in range(len(km_labels)):\n",
    "        if km_labels[idx] == i:\n",
    "            indices.append(idx)\n",
    "    print(len(indices))\n",
    "    if len(indices) > 20:\n",
    "        idx = random.sample(indices, 20)\n",
    "        for index in idx:\n",
    "            X_new.append(x[index])\n",
    "            y_new.append(y[index])\n",
    "            z_new.append(z_coords[index])\n",
    "            labels_new.append(labels[index])\n",
    "            kmeans_labels.append(i)\n",
    "    else:\n",
    "        for index in indices:\n",
    "            X_new.append(x[index])\n",
    "            y_new.append(y[index])\n",
    "            z_new.append(z_coords[index])\n",
    "            labels_new.append(labels[index])\n",
    "            kmeans_labels.append(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [],
   "source": [
    "hover = [documents[i] for i in labels_new]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [],
   "source": [
    "def wrap_by_word(s, n):\n",
    "    '''returns a string where \\\\n is inserted between every n words'''\n",
    "    a = s.split()\n",
    "    ret = ''\n",
    "    for i in range(0, len(a), n):\n",
    "        ret += ' '.join(a[i:i+n]) + '\\n'\n",
    "\n",
    "    return ret\n",
    "\n",
    "\n",
    "hover = [wrap_by_word(x, 8) for x in hover]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [],
   "source": [
    "hover = [x.replace(\"\\n\", \"<br>\") for x in hover]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [],
   "source": [
    "plot = [go.Scatter3d(x = X_new,\n",
    "                    y = y_new,\n",
    "                    z=z_new,\n",
    "                    mode = 'markers+text',\n",
    "                    text = labels_new,\n",
    "                    hovertext=hover,\n",
    "                    textposition='bottom center',\n",
    "                    marker=dict(size=5,opacity=0.8, color=kmeans_labels))]\n",
    "\n",
    "layout = go.Layout(title='prod cat projection')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "config": {
        "linkText": "Export to plot.ly",
        "plotlyServerURL": "https://plot.ly",
        "showLink": false
       },
       "data": [
        {
         "hovertext": [
          "Risk Factors Avian Influenza Infection Chickens Live Bird<br>Retail Stalls Lahore District Pakistan 2009–2010 This study<br>conduct identify risk factor associate AIV infection live<br>bird retail stall LBRS Lahore District Pakistan cross<br>sectional survey LBRS conduct December February use two<br>stage cluster sample base probability proportional size total<br>oropharyngeal swab sample pool collect bird cluster test<br>qRT PCR matrix gene type influenza virus gene<br>subtypes Thirty four sample positive gene also positive<br>sample find positive Data potential risk factor collect<br>questionnaire analyze survey weight logistic regression prevalence odds<br>ratio associate risk factor calculate final multivariable model<br>identify three risk factor infection LRBS namely obtain<br>bird mixed source 4–3 keep bird outside cage<br>4–7 keep chicken breed broiler 7–23 Sourcing bird<br>dealer wholesaler keep bird inside cage avoid mix<br>different breed cage could reduce risk infection LRBS<br>",
          "Pulsed electromagnetic field stimulation prevents steroid induced osteonecrosis<br>rat BACKGROUND Pulsed electromagnetic field PEMF stimulation use<br>successfully treat nonunion fracture femoral head osteonecrosis relatively<br>little known effect prevent steroid induce osteonecrosis The<br>purpose study investigate effect PEMF stimulation prevention steroid<br>induce osteonecrosis rat explore underlying mechanism METHODS Seventy<br>two male adult Wistar rat divide three group<br>treat follow PEMF stimulation group PEMF group intravenously<br>inject lipopolysaccharide LPS day intramuscularly inject methylprednisolone acetate<br>MPSL day subject PEMF stimulation per day week<br>Methylprednisolone treat group MPSL group inject dose LPS<br>MPSL PEMF group without exposure PEMF Control group<br>group inject saline mode time point The incidence<br>osteonecrosis serum lipid level mRNA protein expression transform<br>growth factor TGF proximal femur measure week last<br>MPSL saline injection RESULTS The incidence osteonecrosis PEMF<br>group significantly low observed MPSL group osteonecrosis observe<br>group The serum lipid level significantly low PEMF<br>group MPSL group Compared MPSL group mRNA expression<br>TGF increase reach peak week PEMF treatment remain<br>high week decline week whereas protein expression TGF<br>increase reach peak week PEMF treatment remain high<br>week CONCLUSIONS PEMF stimulation prevent steroid induce osteonecrosis<br>rat underlie mechanism involve decrease serum lipid level<br>increase expression TGF<br>",
          "Different inflammatory response associate Ureaplasma parvum induce UTI<br>urolith formation BACKGROUND Epidemiologic study show strong association<br>Ureaplasmas urogenital tract disease human Since healthy human<br>colonize Ureaplasmas role pathogen remain controversial order begin<br>define role host disease develop rodent model urinary<br>tract infection UTI use Fischer F344 rat Animals<br>inoculate sterile broth log CFU rat adapt strain<br>Ureaplasma parvum RESULTS Infected animal exhibit two distinct<br>profile asymptomatic UTI UTI complicate struvite urolithiasis Inoculum<br>dose parvum affect incidence UTI animal inoculate CFU<br>parvum remain infected However inoculum dose influence immune<br>response parvum Asymptomatic UTI characterize minimal immune response<br>predominantly monocytic lymphocytic limited lesion elevate urinary level<br>IFN MCP UTI complicate struvite formation characterize exaggerated<br>immune response mostly neutrophilic lesion show extensive uroepithelial<br>hyperplasia predominance GRO urine Animals asymptomatic UTI also<br>significantly high rate kidney infection CONCLUSION Complications associate<br>parvum infection primarily dependent upon host specific factor<br>rather Ureaplasma microbial load The immune response F344<br>rat similar occurs human ureaplasmal associate disease Therefore<br>model infection useful tool elucidate parvum host interaction<br>confer UTI disease<br>",
          "longitudinal study bovine coronavirus enteric respiratory infection dairy<br>calf two herd Ohio This prospective longitudinal study<br>examine epidemiology disease syndrome associate bovine coronavirus BCV<br>infection cohort conventional calf day age two dairy<br>herd Ohio The period respiratory shed BCV determine<br>direct immunofluorescent DIF stain nasal epithelial cell ELISA<br>nasal swab supernatant fluids The period fecal shed<br>BCV determine ELISA immunoelectron microscopy IEM The isotype<br>specific antibody titer BCV serum select interval day<br>age post suckle birth total immunoglobulin level examine<br>ELISA zinc sulfate turbidity test respectively calf study<br>evidence BCV respiratory DIF ELISA enteric infection IEM<br>ELISA association diarrhea rhinitis even though calf show<br>increase one serum antibody isotypes BCV calf show<br>BCV respiratory enteric antigen shed ELISA Serological antibody<br>titer increase occur calf day age calf day<br>age two latter calf second rise serum antibody<br>titer BCV initial rise serological antibody titer increase<br>observe one calf This suggest BCV infection may<br>common close herd may occur old calf although<br>many may subclinical may recurrent There statistically significant<br>correlation total serum immunoglobulin level BCV antibody isotype<br>titer serum 24–48 birth clinical disease infection BCV<br>however calve low level IgA BCV antibodies serum<br>24–48 birth significantly great average number day diarrhea<br>calf high level IgA BCV specific antibody serum<br>",
          "Characterization enterotoxigenic coli ETEC Shiga toxin produce coli<br>STEC necrotoxigenic coli NTEC isolate diarrhoeic Mediterranean water<br>buffalo calve Bubalus bubalis Two hundred twenty Escherichia<br>coli isolate Mediterranean water buffalo calve less 4weeks<br>old affect severe diarrhoea lethal outcome characterize presence<br>virulence factor Stx1 Stx2 haemolysins intimin CNF1 CNF2<br>CDT CDT CDT III CDT F17 relate fimbriae<br>F17a F17b F17c F17d The prevalence ETEC STEC<br>NTEC respectively The ETEC isolate positive negative The<br>STEC isolate Stx intimin positive Stx1 frequent Stx2<br>The NTEC isolate CNF Hly positive CNF2 frequent<br>CNF1 Susceptibility assay antimicrobial display high rate resistance<br>antimicrobial test These data show prevalent strain diarrhoeic<br>water buffalo calf NTEC mostly CNF2 HlyA positive<br>strong association CNF2 CDT III CNF2 F17c<br>",
          "Chapter Infectious Diseases Domestic Rabbits<br>",
          "Galactose phosphate uridyltransferase GalT vivo induce antigen Actinobacillus<br>pleuropneumoniae serovar strain L20 provide immunoprotection serovar strain<br>MS71 GALT important antigen Actinobacillus pleuropneumoniae APP show<br>provide partial protection APP infection previous study lab<br>The main purpose present study investigate GALT induce<br>cross protection different APP serotypes elucidate key mechanism<br>immune response GALT antigenic stimulation Bioinformatic analysis demonstrate<br>galT highly conserve gene APP widely distribute across<br>multiple pathogenic strain Homologies two strain range regard<br>galT locus Indirect enzyme link immunosorbent assay ELISA<br>confirm GALT specific antibody could induce inactivated APP<br>L20 MS71 whole cell bacterin preparations recombinant fusion<br>GALT protein derive APP L20 however proven effective<br>cross protective antigen APP sevorar MS71 APP sevorar<br>L20 Histopathological examination confirm recombinant GALT vaccinate animal<br>show less severe pathological sign lung tissue negative<br>control APP challenge Immunohistochemical IHC analysis indicate infiltration<br>neutrophil negative group significantly increase compare normal control<br>surviving animal decrease compare negative group Anti GALT<br>antibodies show mediate phagocytosis neutrophil After interaction anti<br>GALT antibodies survival rate APP challenge vaccinated animal<br>significantly reduce This study demonstrate GALT effective cross<br>protective antigen could use potential vaccine candidate multiple<br>APP serotypes<br>",
          "Multicausal etiology enteric syndrome rabbit Mexico Enteropathies rabbit<br>difficult diagnose etiology involve pathogen act synergistically causing<br>damage intestine The aim present study isolate enteric<br>pathogens rabbit Mexico Using parasitological bacteriological molecular analysis<br>screen sample intestinal content rabbit clinical history enteric<br>disease southeastern part State Mexico Out sample analyze<br>total identification make Eimeria spp find follow Aeromonas<br>spp Escherichia coli identify enteropathogenic coli EPEC presence<br>follow agent also confirm Salmonella spp Klebsiella spp<br>Streptococcus spp Staphylococcus aureus Enterococcus spp Mannheimia spp<br>Rotavirus The concurrent presence Eimeria spp Aeromonas frequent<br>statistical significance presence association clinical profile Eimeria spp<br>Mannheimia spp Salmonella spp Klebsiella spp Streptococcus spp<br>Enterococcus spp<br>",
          "Enteric coronavirus infection adult horse new enteric virus<br>adult horse equine coronavirus ECoV recently recognize associate<br>fever lethargy anorexia less frequently colic diarrhea This<br>enteric virus transmit via feco oral route horse<br>become infected ingest fecally contaminate feed water Various<br>outbreaks report since Japan Europe USA While clinical<br>sign fairly non specific lymphopenia neutropenia often see<br>Specific diagnosis make detection ECoV feces either quantitative<br>real time PCR electron microscopy antigen capture ELISA<br>Supportive treatment usually require infection self limit However<br>rare complication endotoxemia septicemia hyperammonemia associate encephalopathy report<br>related loss barrier function intestinal mucosa This review<br>article focus late information pertain virus epidemiology clinical<br>sign diagnosis pathology treatment prevention ECoV infection adult<br>horse<br>",
          "super spreading ewe infect hundred fever farmer market<br>Germany BACKGROUND May Soest County Health Department inform<br>unusually large number patient hospitalized atypical pneumonia METHODS<br>exploratory interview patient mention visited farmer market sheep<br>lamb Serologic test confirm diagnosis fever ask local<br>health department Germany identiy notify fever patient visit<br>farmer market investigate risk factor infection conduct case<br>control study case fever patient control randomly select<br>Soest citizen cohort study among vendor market The<br>sheep exhibit market herd originate well sheep herd<br>hold vicinity Soest test Coxiella burnetii burnetii RESULTS<br>total report fever case link outbreak The mean<br>incubation period day interquartile range 16–24 day The<br>case control study identify close proximity stop least<br>second sheep pen significant risk factor Vendors within<br>approximately meter sheep pen increase risk disease compare<br>locate far away Wind play significant role The<br>clinical attack rate adult child estimate respectively case<br>hospitalize The ewe lamb well herd test positive<br>burnetii antibody CONCLUSION Due size point source nature<br>outbreak permit assessment fundamental seldom study epidemiological parameter<br>consequence outbreak recommend pregnant sheep display public trimester<br>test animal pet zoos regularly burnetii<br>",
          "Longitudinal study morbidity mortality white veal calf Belgium<br>BACKGROUND Mortality morbidity hardly document white veal industry<br>despite high level antimicrobial drug use resistance The<br>objective present study determine cause epidemiology morbidity mortality<br>dairy beef crossbreed white veal production total calf<br>house production cohort follow one production cycle Causes<br>mortality determine necropsy Morbidity daily record producer RESULTS<br>The total mortality risk significantly high beef veal<br>production compare dairy crossbreds The main cause mortality<br>pneumonia calf risk ruminal disorder idiopathic peritonitis enterotoxaemia<br>enteritis Belgian Blue beef calf likely die pneumonia<br>enterotoxaemia arthritis Detection bovine viral diarrhea virus necropsy<br>associate chronic pneumonia pleuritis calf treat individually morbidity<br>rate case per calf day risk The incidence<br>rate respiratory disease diarrhea arthritis otitis case per<br>calf day risk respectively Morbidity peak first three<br>week arrival gradually decline towards end production cycle<br>CONCLUSIONS The present study provide insight cause epidemiology<br>morbidity mortality white veal calf Belgium house frequent<br>housing system Europe The necropsy finding identify risk<br>period difference production system guide veterinarians producer towards<br>profitable ethical preventive therapeutic protocol<br>",
          "vivo growth porcine reproductive respiratory syndrome virus engineer<br>nsp2 deletion mutant Prior study PRRSV strain non<br>structural protein nsp2 show much amino acid could<br>remove hypervariable region without lose virus viability vitro<br>utilize select nsp2 deletion mutant examine vivo growth<br>Young swine pig group control swine inoculate intramuscularly<br>one nsp2 deletion mutant rΔ727–813 rΔ543–726 rΔ324–523 rΔ324–726<br>full length recombinant virus rVR Serum sample collect<br>various day post inoculation analyze HerdChek ELISA PRRSV<br>real time PCR gamma interferon IFN ELISA nucleotide<br>sequence analysis entire nsp2 cod region Tracheobronchial lymph<br>node weight compare body weight record animal use<br>clinical measurement viral pathogenesis Results show deletion mutant<br>grow less robustly full length recombinant virus yet<br>large deletion virus rΔ324–726 recover parental viral RNA<br>level study end Swine receive rΔ727–813 mutant significant<br>decrease lymph node enlargement compare rVR While swine<br>infection rVR cause rapid rise serum IFN level<br>IFN protein produce infection deletion mutant viruses significantly<br>reduce perhaps due difference viral growth kinetics The<br>rΔ543–726 nsp2 mutant virus although growth impair mimic<br>rVR induce host serum IFN response exhibit week<br>delay Targeted sequence showed deletion stable region cod<br>nsp2 one swine passage The data suggest select<br>nsp2 deletion mutant growth attenuate swine alter induction<br>serum IFN innate cytokine unknown function PRRSV clearance<br>point domain may influence tracheobronchial lymph node size<br>",
          "Differential expression miR collapse steroid induce osteonecrosis femoral<br>head BACKGROUND Femoral head collapse key reference point<br>determine treatment regimen femoral head osteonecrosis However effective<br>preventive measure efficacy hip preserving surgery unsatisfactory due<br>unclear mechanism collapse This study aim identify validate<br>miRNAs differentially express collapse non collapse area osteonecrotic<br>femoral head predict target gene pathway miRNAs RESULTS<br>Nine sample pass quality control test total differentially<br>express miRNAs detect among miRNAs show differential expression<br>group compare group miRNAs show differential expression group<br>compare group miRNAs show differential expression group compare<br>group After combine data three patient miRNAs show<br>differential expression collapse area compare non collapse area<br>Compared normal area miR show significant downregulation Expression<br>result PCR reveal expression hsa miR collapse area<br>significantly low non collapse area normal area gene<br>perdicted target gene hsa miR MATERIALS AND METHODS<br>Femoral head three patient male female treat total<br>hip arthroplasty surgery steroid induce femoral head osteonecrosis<br>select base inclusion exclusion criterion Bone tissue sample<br>obtain collapse area non collapse area normal area<br>accord anatomical structure osteonecrotic femoral head Total RNA<br>extract sample microarray chip scan miRNAs show differential<br>expression fold select validated PCR TargetScan mirBase miRanda<br>bioinformatics software use predict target gene identify possible<br>pathway involve gene CONCLUSIONS miR show significant difference<br>collapse area osteonecrotic femoral head suggest collapse may<br>relate downregulation miR<br>",
          "Cryptosporidium infection veal calf cohort France molecular characterization<br>specie longitudinal study Feces animal collect farm region<br>Brittany France Each sample directly collect rectum animal<br>identify ear tag number Animals sample three time<br>week age After DNA extraction stool sample nest<br>PCR perform amplify partial 18S rDNA kDa glycoprotein<br>gene Cryptosporidium The parasite detect farm One hundred<br>calf find parasitized Cryptosporidium Amplified fragment sequence Cryptosporidium<br>specie identification reveal presence parvum ryanae bovis One<br>animal infect Cryptosporidium ubiquitum The prevalence specie relate<br>age animal parvum cause Cryptosporidium infection week old<br>calve week old animal The analysis result show<br>animal could infect successively parvum ryanae bovis study<br>period parvum gp60 genotyping identifies IIa subtypes represent<br>IIaA15G2R1 This work confirm previous study country show<br>zoonotic parvum dominant specie see young calf<br>",
          "Pyogranulomatous Pleuropneumonia Mediastinitis Ferrets Mustela putorius furo associate<br>Pseudomonas luteola Infection Summary Between three pet ferret<br>different source present acute episode dyspnoea Cytological examination<br>pleural exudate reveal severe purulent inflammation abundant cluster<br>rod shaped microorganism clear surround halo Treatment ineffective<br>ferret die 2–5 day later Two ferret subject<br>necropsy examination reveal pyothorax mediastinal lymphadenopathy multiple white<br>nodule 1–2mm lung Microscopical examination show multifocal necrotizing<br>pyogranulomatous pleuropneumonia lymphadenitis aggregate encapsulate microorganisms positively stain<br>periodic acid–Schiff alcian blue situ hybridization Pneumocystis spp<br>Ziehl–Neelsen stain immunohistochemistry distemper coronavirus influenza antigen negative<br>case Electron microscopically bacteria 2–3μm long thick electron<br>lucent capsule Microbiology one ferret yield pure culture<br>gram negative bacteria identify phenotypically Pseudomonas luteola This<br>speciation later confirm 16S RNA gene amplification<br>",
          "Respiratory Pathogens Québec Dairy Calves Their Relationship Clinical<br>Status Lung Consolidation Average Daily Gain BACKGROUND Bovine<br>respiratory disease BRD important cause morbidity mortality dairy<br>calve Surprisingly field data scant concern prevalence respiratory<br>pathogen involve BRD preweaned dairy calf especially small<br>herd OBJECTIVES identify main respiratory pathogen isolate calf<br>Québec dairy herd high incidence BRD determine association<br>presence pathogens clinical sign pneumonia lung consolidation average<br>daily gain ANIMALS Cross‐sectional study use convenience sample<br>preweaned dairy calf dairy herd METHODS enrollment calf<br>weigh clinically examine swab nasal nasopharyngeal lung ultrasonography<br>perform One month later calf reweighed RESULTS Twenty‐two<br>calve clinical BRD ultrasonographic evidence lung consolidation Pasteurella<br>multocida Mannheimia haemolytica Histophilus somni isolate calf respectively<br>Mycoplasma bovis identify PCR test culture calf calf<br>find positive Mycoplasma spp Bovine coronavirus detect calf<br>bovine respiratory syncytial virus Only presence bovis associate<br>high odds clinical sign lung consolidation low average<br>daily gain CONCLUSIONS AND CLINICAL IMPORTANCE Results suggest<br>nasopharyngeal carriage bovis detrimental health growth dairy calf<br>small herd high incidence BRD<br>",
          "Investigation Pathology Pathogens Associated Porcine Respiratory Disease Complex<br>Denmark Summary Respiratory infection among important disease grow<br>pig order elucidate multifactorial aetiology porcine respiratory disease<br>complex PRDC Denmark lungs finishing pig cranioventral bronchopneumonia<br>case group pig without lung lesion control group<br>collect abattoirs The pathogen involve PRDC interaction identify<br>link histopathological diagnosis The lung sample culture bacteria<br>test multiplex polymerase chain reaction presence swine influenza<br>virus type porcine reproductive respiratory syndrome virus European<br>type porcine circovirus type PCV2 porcine respiratory coronavirus<br>porcine cytomegalovirus Mycoplasma hyopneumoniae Mycoplasma hyorhinis All case<br>cranioventral lobular bronchopneumonia consistent PRDC There broad range<br>microscopical lesion case characterize acute subacute chronic bronchopneumonia<br>Five bacterial specie five virus two Mycoplasma spp<br>detect different combination PCV2 hyopneumoniae hyorhinis Pasteurella multocida<br>detect frequently among PRDC affect swine diversity number<br>pathogens high animal compare control clear cut association<br>detect pathogen histological lesion histopathological diagnosis PRDC occur<br>frequently enzootic pneumonia among Danish finishing pig complex<br>vary histopathology<br>",
          "Multidose intramuscular allogeneic adipose stem cell decrease severity<br>canine atopic dermatitis pilot study AIM The aim<br>pilot study evaluate therapeutic safety performance intramuscular treatment<br>protocol multidose allogeneic adipose stem cell ASCs isolate<br>characterize expanded vivo healthy canine donor MATERIALS AND<br>METHODS Twelve dog diagnose canine atopic dermatitis CAD<br>intramuscularly treat 5×10 cryopreserved ASCs healthy immunize young<br>canine Ehrlichia canis free donor weekly week Treatment<br>efficacy evaluate pruritus index CAD Lesion Index CADLI<br>test Safety adverse effect determine injection site reaction<br>weight blood chemistry liver function whole blood count<br>RESULTS Canine ASCs obtain donor meet minimum quality<br>require type cell show viability thaw The efficacy<br>CADLI score pruritus index dog atopic dermatitis statistically<br>significant efficacy adverse reaction observe intramuscular application site<br>relation animal weight blood cell population liver renal<br>function CONCLUSION These result suggest intramuscular administration cryopreserved<br>ASCs dog atopic dermatitis promise cellular therapeutic product<br>relief symptom disease however duration effect obtain dose<br>dos evaluate well possible immune reaction far know<br>first report use multiple intramuscular dos cryopreserved ASCs<br>treat atopic dermatitis<br>",
          "Strategies control PRRS summary field research experience Various<br>method control PRRS virus publish The technology nursery<br>depopulation appear effectively control spread virus member endemically<br>infected population consist strategic adjustment pigflow base presence<br>specific serologic pattern detect indirect fluorescent antibody test<br>This pattern indicate low seroprevalence antibody detect breed<br>herd recently wean piglet contrast high seroprevalence week<br>old piglet carry swine farm result indicate improvement<br>nursery piglet growth rate mortality level Three example<br>provide follow text Recently modify live virus vaccine<br>RespPRRS NOBL Laboratories Boerhinger Ingleheim become commercially available<br>currently approve use piglets week age however potential<br>use adult animal currently investigation<br>",
          "Bovine respiratory syncytial virus outbreak reduce bull weight<br>gain feed conversion eight month Norwegian beef herd<br>BACKGROUND Cost benefit evaluation measure respiratory disease cattle<br>require account associate production loss Investigations naturally occur<br>respiratory infection herd set opportunity accurate estimate consequence<br>This article present estimate base individual monitoring weight<br>concentrate intake several hundred bull previous respiratory infection<br>outbreak bovine respiratory syncytial virus BRSV main pathogen<br>The aim study analyse association exposure BRSV weight<br>gain feed conversion rate quantify change parameter estimate<br>duration change production RESULTS comparison growth curve bull<br>present outbreak reveal bull severe clinical sign clear<br>consistent trend poor growth rate milder sign The<br>weight age ratio 04–0 lower severely affected bull<br>evident throughout study period month comparison growth rate<br>apparently healthy bull present outbreak comparable group bull<br>exactly year later show reduced growth rate day<br>first group The difference amount extra day need<br>reach reference weight Feed conversion also reduce weight<br>gain kilogram concentrate consume outbreak year CONCLUSION This<br>study indicate significant negative effect performance animal develop<br>severe clinical sign acute stage growth production negatively<br>affect many month apparent recovery addition performance apparently<br>healthy animal expose outbreak severely negatively affect The<br>duration decrease production animal recovery animal show disease<br>previously document These loss easily underestimate contribute significantly<br>cost producer The finding emphasize importance BRSV infection<br>profitability animal welfare cattle husbandry The study also<br>illustrate utilising intra herd comparison health production parameter<br>productive approach estimate consequence outbreak<br>",
          "Complete Genome Sequence Bordetella bronchiseptica Strain KM22 Bordetella<br>bronchiseptica isolate KM22 use experimental infection swine model<br>clinical bronchiseptica infection study host host transmission The<br>draft genome sequence KM22 report Here report complete<br>genome sequence KM22<br>",
          "Expression Dynamics Innate Immunity Influenza Virus Infected Swine<br>The current circulating swine influenza virus subtypes Europe<br>H1N1 H1N2 H3N2 associate clinical outbreak disease However<br>show pig could susceptible strain able cross specie<br>barrier work extend investigation whether different strain able<br>cross specie barrier might give rise different innate<br>immune response could associate pathological lesion For purpose<br>use sample collect previous study healthy pig infect<br>H3N2 Swine four different H3N8 strain circulate different<br>animal specie Pigs clinically inspect four subject group<br>sacrifice day post infection present study group mock<br>exhibit antibody response nucleoprotein protein Pulmonary lesion high<br>titered viral replication observe pig infect swine adapt<br>virus Interestingly pig infect avian seal H3N8 strain<br>also show moderate lesion viral replication whereas equine<br>canine IVs cause overt pathological sign replication barely<br>detectable Swine infection induce interferon IFN alpha interleukin<br>response bronchoalveolar fluid BALF day post infection oppose<br>non swine adapt virus strain However IFN alpha<br>response swine adapt virus associate increase local constitutive<br>expression IFN alpha gene Remarkably Equine strain give<br>rise Serum Amyloid response BALF despite little replication<br>Each virus strain could associate expression cytokine gene<br>proteins infection These response observe well beyond period<br>virus replication suggest prolonged homeostatic imbalance innate immune<br>system<br>",
          "Health zoonotic Infections snow leopard Panthera unica South<br>Gobi desert Mongolia Background Snow leopards Panthera uncia<br>threaten apex predator scatter across mountain Central South<br>Asia Disease threats wild snow leopard investigate Methods<br>Results Between twenty snow leopard South Gobi desert<br>Mongolia capture immobilised health screen radio collar Blood<br>sample external parasite collect pathogen analysis use enzyme<br>link immunosorbent assay ELISA microscopic agglutination test MAT<br>next generation sequence NGS techniques The animal show<br>clinical sign disease however serum antibody significant zoonotic<br>pathogen detect These pathogen include Coxiella burnetii prevalence<br>Leptospira spp Toxoplasma gondii Ticks collect snow leopard<br>contain potentially zoonotic bacteria genus Bacillus Bacteroides Campylobacter<br>Coxiella Rickettsia Staphylococcus Streptococcus Conclusions The zoonotic pathogen<br>identify study short term appear cause illness snow<br>leopard cause illness wild felid Therefore surveillance pathogen<br>implement monitor potential long term disease impact snow<br>leopard population<br>",
          "Identification Novel Hypovirulence Inducing Hypovirus From Alternaria alternata<br>Mycoviruses wide spread throughout almost group fungi small<br>number mycoviruses attenuate growth virulence fungal host Alternaria<br>alternata ascomycete fungus cause leaf spot diseases various<br>crop plant study identify novel ssRNA mycovirus infect<br>alternata mali strain isolate apple orchard China Sequence<br>analyse reveal virus related hypoviruses particular Wuhan insect<br>virus unclassified hypovirus identify insect meta transcriptomics well<br>hypoviruses belong genus Hypovirus therefore virus design Alternaria<br>alternata hypovirus AaHV1 The genome AaHV1 contain single<br>large open reading frame encode putative polyprotein ∼479<br>kDa cysteine proteinase like replication associate domain Curing<br>AaHV1 fungal host strain indicate virus responsible slow<br>growth reduce virulence host AaHV1 defective RNA RNA<br>internal deletion emerge fungal subculture presence RNA affect<br>AaHV1 accumulation pathogenicities Moreover AaHV1 could replicate confer<br>hypovirulence Botryosphaeria dothidea fungal pathogen apple white rot<br>disease This finding could facilitate good understand alternata<br>pathogenicity relevant development biocontrol method fungal disease<br>",
          "new generic real time reverse transcription polymerase chain<br>reaction assay vesiviruses vesiviruses detect human sample Different<br>virus belong genus Vesivirus infect broad range animal<br>cause gastroenteritis vesicular skin lesion hemorrhagic disease respiratory<br>disease condition recent report Vesivirus viremia detect PCR<br>sample patient hepatitis unknown etiology USA suggest zoonotic<br>potential vesiviruses These result confirm another laboratory order<br>generic PCR assay RNA polymerase region develop validate<br>RNA different Vesivirus specie Except SMSV serotype specie<br>test detect include one suggest involved zoonotic transmission<br>USA SMSV serotype The generic Vesivirus assay use<br>RNA extract serum sample patient hepatitis stool sample<br>patient gastroenteritis throat swab specimens patient rash illness<br>throat swab nose swabs patient acute respiratory disease<br>cell culture cytopathologic effect enterovirus surveillance pathogen find<br>None find positive study generic Vesivirus assay develop<br>concluded vesiviruses unlikely cause common illness human Netherlands<br>",
          "Association Mannose bind Lectin Polymorphisms Tuberculosis Susceptibility among<br>Chinese Tuberculosis cause infection Mycobacterium tuberculosis Host genetic<br>variability important determinant risk develop human Although association<br>MBL2 polymorphism study various population result controversial study<br>four functional single nucleotide polymorphism SNPs across MBL2<br>gene genotyped direct DNA sequence PCR product case<br>control population Chinese Han origin consist patient pulmonary<br>control find individual carry variant allele namely genotype<br>associate increased susceptibility odds ratio confidence interval 30–1<br>Additionally LYPB haplotype show significant association increase risk<br>27–1 global haplotype association Furthermore individual bear low<br>medium MBL expression haplotype pair increase risk 29–1<br>Thus reduce expression functional MBL secondary MBL2 variant<br>may partially mediate increased susceptibility risk<br>",
          "Isolation characterization novel microsatellite locus West African leaf<br>nose bat Hipposideros aff ruber BACKGROUND Noack leaf<br>nosed bat Hipposideros ruber cryptic specie within Hipposideros<br>caffer specie complex Despite widespread distribution Africa host<br>potentially zoonotic virus genetic structure ecology ruber poorly<br>know Here describe development novel polymorphic microsatellite locus<br>facilitate investigation genetic structure FINDINGS select microsatellite sequence<br>identify high throughput sequence read PCR amplify individual<br>yield consistently amplify scorable locus The number alleles<br>per locus range two observe heterozygosity evidence linkage<br>disequilibrium observe nine marker show departure Hardy Weinberg<br>equilibrium demonstrate successful amplification two closely related specie<br>two divergent lineage caffer specie complex CONCLUSIONS These<br>new marker provide valuable tool investigate genetic structure<br>poorly understood Hipposideros caffer specie complex<br>",
          "Complete Genome Sequence Nephropathogenic Infectious Bronchitis Virus Strain<br>Isolated China Infectious bronchitis virus IBV cause tremendous<br>economic loss poultry industry Here report complete genome<br>analysis result new natural recombination nephropathogenic IBV strain<br>name SAIBK isolate Sichuan province China<br>",
          "binning tool reconstruct viral haplotype assemble contigs MotivationInfections<br>RNA viruses Influenza HIV still pose serious threat<br>human health despite extensive research viral diseases One<br>challenge produce effective prevention treatment strategy high intra<br>specie genetic diversity different strain may different biological<br>property characterize genetic diversity thus important vaccine drug<br>design Next generation sequence technology enables comprehensive characterization<br>know novel strain widely adopt sequence viral population<br>However genome scale reconstruction haplotype still challenge problem<br>particular haplotype assembly program often produce contigs rather<br>full genome mutation one gene mask phenotypic effect<br>mutation another locus cluster contigs genome scale haplotype<br>still need ResultsWe develop contig bin tool VirBin<br>cluster contigs different group group represent haplotype Commonly<br>use feature base sequence composition contig coverage can<br>not effectively distinguish viral haplotype high sequence similarity<br>heterogeneous sequence coverage RNA viruses VirBin apply prototype<br>base cluster cluster region likely contain mutation specific<br>haplotype The tool test multiple simulated sequence data<br>different haplotype abundance distribution contig size also mock<br>quasispecies sequence data The benchmark result contig binning<br>tool demonstrate superior sensitivity precision VirBin contig bin<br>viral haplotype reconstruction Availabilityhttps github com chjiao VirBinContactyannisun<br>cityu edu<br>",
          "Bat origin human coronaviruses Bats recognize natural reservoir<br>large variety virus Special attention pay bat coronaviruses<br>two emerge coronaviruses cause unexpected human disease outbreaks<br>21st century Severe Acute Respiratory Syndrome Coronavirus SARS<br>CoV Middle East Respiratory Syndrome Coronavirus MERS CoV<br>suggest originated bat Various specie horseshoe bat China<br>find harbor genetically diverse SARS like coronaviruses Some<br>strain highly similar SARS CoV even spike protein<br>able use receptor SARS CoV cell entry hand<br>diverse coronaviruses phylogenetically related MERS CoV discover worldwide<br>wide range bat specie classify coronavirus specie MERS<br>CoV Coronaviruses genetically related human coronavirus 229E NL63<br>detect bat well Moreover intermediate host believe play<br>important role transmission emergence coronaviruses bat human Understanding<br>bat origin human coronaviruses helpful prediction prevention another<br>pandemic emergence future<br>",
          "Altered pathogenicity immunogenicity tissue tropism 7kb region sequence<br>avian infectious bronchitis coronavirus strain serial passage embryo<br>study attenuate Chinese LX4 type nephropathogenic infectious bronchitis<br>virus IBV strain LHLJ 04V serial passage embryonated<br>chicken egg Based sequence analysis 7kb region LHLJ<br>04V virus population contain subpopulation mixture genetic mutant<br>The titer virus increase gradually serial passage replication<br>capacity decrease chickens The virus partially attenuate passage<br>P40 P70 fully attenuate P110 lose immunogenicity kidney<br>tropism P110 P70 respectively Amino acid substitution find<br>7kb region primarily spike protein Substitutions subunit occur<br>P40 subpopulation virus passage show substitution Other substitution<br>occur P70 P110 however find subpopulation virus passages<br>deletion UTR observe subpopulation P70 P110 might relate<br>virus replication transcription pathogenicity The change describe 7kb<br>region virus possibly responsible virus attenuation immunogenicity decrease<br>tissue tropism change however can not exclude possibility<br>part genome may also involve change<br>",
          "Alphacoronaviruses New World Bats Prevalence Persistence Phylogeny Potential<br>Interaction Humans Bats reservoirs many different coronaviruses CoVs<br>well many important zoonotic virus sample feces anal<br>swab insectivorous bat family specie different location within<br>Colorado detect alphacoronavirus RNA bat specie big brown<br>bat Eptesicus fuscus prevalence long legged bat Myotis<br>volans prevalence little brown bat Myotis lucifugus prevalence<br>western long eared bat Myotis evotis prevalence Overall<br>juvenile bat twice likely positive CoV RNA adult<br>bat two rural sample site CoV RNAs detect<br>big brown long legged bat three sequential summer<br>study CoV RNA detect big brown bat five<br>urban maternity roost sample throughout period test Individually<br>tag big brown bat positive CoV RNA later<br>sample become CoV RNA negative Nucleotide sequence RdRp<br>gene fell main cluster distinct Old World bat<br>Similar nucleotide sequence find amplicons gene spike gene<br>big brown long legged bat indicate CoV may<br>capable infect bat different genus These data suggest<br>ongoing evolution CoVs bat create possibility continue threat<br>emergence host specie Alphacoronavirus RNA detect high prevalence<br>big brown bat roost close proximity human habitation<br>know direct contact people suggest significant potential opportunity<br>exist cross specie transmission viruses Further CoV surveillance<br>study bat throughout Americas warrant<br>",
          "PEDV PDCoV Pathogenesis The Interplay Between Host Innate<br>Immune Responses Porcine Enteric Coronaviruses Enteropathogenic porcine epidemic<br>diarrhea virus PEDV porcine deltacoronavirus PDCoV member coronavirus<br>family account majority lethal watery diarrhea neonatal pig<br>past decade These two virus pose significant economic<br>public health burden even continue emerge reemerge worldwide<br>The ability evade circumvent subvert host first line<br>defense namely innate immune system key determinant pathogen<br>virulence survival establishment successful infection Unfortunately start unravel<br>underlying viral mechanism use manipulate host innate immune<br>response review gather current knowledge concern interplay viruses<br>component host innate immunity focus type interferon induction<br>signal particular mechanism virus encode gene product antagonize<br>subvert host innate immune response Finally provide perspective<br>advantage gain well understand host pathogen interaction This<br>include implication future development PEDV PDCoV vaccine knowledge<br>molecular mechanism underlie virus pathogenesis virulence host coevolution<br>",
          "Novel Picobirnaviruses Respiratory Alimentary Tracts Cattle Monkeys Large<br>Intra Inter Host Diversity Picobirnaviruses PBVs mostly find<br>animal alimentary sample study among respiratory specimen mammal<br>chicken genogroup PBVs detect three cattle three monkey<br>genogroup PBV positive sample collect one cattle specimen<br>More one PBV sequence type observe two one<br>genogroup PBV positive sample cattle monkey respectively Twenty<br>four complete near complete segment nine respiratory alimentary<br>sample cattle monkey genogroup PBVs one complete segment<br>cattle genogroup PBV sequence Similar study cattle PBVs<br>also show high diversity contrast monkey PBVs observe<br>study cluster three distinct clade Within clade sequence<br>show amino acid identities This unique phenomenon probably<br>due fact monkey locality reside separate troop minimal<br>inter troop contact<br>",
          "Host viral trait predict zoonotic spillover mammal The<br>majority human emerge infectious disease EIDs zoonotic virus<br>originate wild mammal particular concern HIV Ebola SARS<br>1–3 Understanding pattern viral diversity wildlife determinant successful<br>cross specie transmission spillover therefore key goal pandemic<br>surveillance program However analytical tool exist identify host<br>specie likely harbor next human virus virus cross<br>specie boundary 5–7 Here conduct comprehensive analysis yet<br>mammalian host virus relationship show total number virus<br>infect give specie proportion likely zoonotic predictable After<br>control research effort proportion zoonotic virus per specie<br>predict phylogenetic relatedness human host taxonomy human population<br>within specie range may reflect human wildlife contact<br>demonstrate first time bat harbor significantly high proportion<br>zoonotic virus mammalian order identify taxa geographic region<br>large estimate number miss virus miss zoonosis therefore<br>high value future surveillance show phylogenetic host breadth<br>viral trait significant predictor zoonotic potential providing novel<br>framework ass newly discover mammalian virus could infect<br>people<br>",
          "The Role Viral Population Diversity Adaptation Bovine Coronavirus<br>New Host Environments The high mutation rate RNA<br>viruses enables diverse genetic population viral genotype exist<br>within single infected host host genetic diversity could<br>better position virus population respond adapt diverse array<br>selective pressure host switch event Multiple new coronaviruses<br>include SARS identify human sample within last ten<br>year demonstrate potential coronaviruses emergent human pathogen Deep<br>sequencing use characterize genomic change coronavirus quasispecies simulate<br>host switching Three bovine nasal sample infect bovine<br>coronavirus use infect human bovine macrophage lung cell<br>line The virus reproduce relatively well macrophages lung<br>cell line infect efficiently enough allow passage non<br>lab adapt sample Approximately genome amplified passage sequence<br>average coverage nearly 950× sequence 000× Illumina The<br>consensus sequence many passaged sample nucleotide insert consensus<br>sequence spike gene multiple point mutation associate presence<br>insert Deep sequencing reveal insert present rare unpassaged<br>sample could quickly shift dominate population place different<br>environment The insert cod three arginine residue occur<br>region associate fusion entry host cell may allow<br>infection new cell type via heparin sulfate binding<br>Analysis deep sequencing data indicate two distinct genotype<br>circulate different frequency level sample support hypothesis mutation<br>present passaged strain select pre exist pool rather<br>novo mutation subsequent population fixation<br>",
          "Evolution Genome Size Complexity Rhabdoviridae RNA viruses exhibit<br>substantial structural ecological genomic diversity However genome size<br>RNA viruses likely limited high mutation rate result<br>evolution various mechanisms increase complexity minimise genome expansion<br>Here conduct large scale analysis genome sequence animal<br>rhabdovirus include genome determine novo identify pattern genome<br>expansion evolution genome complexity All seven rhabdovirus cluster<br>well support monophyletic group eight correspond establish genus<br>seven assign new genus two taxonomically ambiguous show<br>acquisition loss new gene appear central theme rhabdovirus<br>evolution associate appearance alternative overlap consecutive ORFs within<br>major structural protein gene insertion loss additional ORFs<br>gene junction clade specific manner Changes length gene<br>junction account much variation genome size small large<br>genome frequency new ORFs observe increase direction along<br>genome also identify several new family accessory gene<br>encode region show non canonical expression strategy involve<br>TURBS like termination reinitiation ribosomal frame shift leaky<br>ribosomal scanning appear common conclude rhabdovirus unusual capacity<br>genomic plasticity may link discontinuous transcription strategy negative<br>sense single strand RNA genome propose model account<br>regular occurrence genome expansion contraction throughout evolution Rhabdoviridae<br>",
          "Increasing diversity swine parvoviruses epidemiology African pig Detection<br>infectious viral agent increase globally advent usage sensitive<br>selective novel molecular technique epidemiological study viral disease<br>economic importance swine industry The observation different pig<br>infect member subfamily Parvovirinae family Parvoviridae application novel<br>molecular method like metagenomics bring detection many novel<br>member group Surprisingly list keep increase day day<br>possess zoonotic potential last one decade less ten<br>novel swine infecting virus add subfamily ceaseless effort<br>top gear determine occurrence prevalence old new swine<br>parvovirus herd pig produce country worldwide The story<br>however contrary African continent presently dearth information surveillance<br>initiatives virus among swine herd pig produce country<br>region Timely detection characterization viral pathogen highly imperative<br>implementation effective control prevention spread This review therefore<br>present concise overview epidemiology novel porcine parvovirus globally<br>also provide date highlight report case viral agent<br>African sub region<br>",
          "Development combine canine distemper virus specific PCR protocol<br>differentiation infect vaccinated animal DIVA genetic characterization hemagglutinin<br>gene seven Chinese strain demonstrate dog combined reverse<br>transcription polymerase chain reaction PCR method develop detection<br>differentiation wild type vaccine strain canine distemper virus<br>CDV pair primer use detect CDV wild type<br>strain vaccine Another pair use detect CDV wild<br>type strain 335bp fragment amplify genomic RNA vaccine<br>wild type strain 555bp fragment amplify specifically genomic<br>RNA wild type strain amplification achieve uninfected cell<br>cell infected canine parvovirus canine coronavirus canine adenovirus<br>The combined PCR method detect effectively differentiate CDV<br>wild type vaccine strain two separate PCRs The<br>method use clinical detection epidemiological surveillance The phylogenetic<br>analysis hemagglutinin gene local wild type CDV strain<br>reveal seven local isolates belong Asia lineage cluster<br>closely one another location These result suggest CDV<br>genotype Asia circulate currently domestic dog China<br>",
          "Detection novel circovirus PCV3 pig cardiac multi systemic<br>inflammation BACKGROUND Porcine circovirus cause different clinical syndrome<br>result significant economic loss pork industry Three pig<br>unexplained cardiac multi organ inflammation test negative PCV2<br>know porcine pathogen analyze METHODS Histology use identify<br>microscopic lesion multiple tissue Metagenomics use detect viral<br>sequence tissue homogenates situ hybridization use detect viral<br>RNA expression cardiac tissue RESULTS three case characterize<br>genome new circovirus call PCV3 replicase capsid proteins<br>show identity genetically closest protein bat feces associate<br>circovirus even distant porcine circovirus Common microscopic lesion<br>include non suppurative myocarditis cardiac arteriolitis Viral mRNA<br>detect intralesionally cardiac cell Deep sequence tissue also<br>reveal presence porcine astrovirus three animal well rotavirus<br>porcine cytomegalovirus porcine hemagglutinate encephalomyelitis virus individual case<br>CONCLUSION The pathogenicity molecular epidemiology new circovirus alone<br>context infection warrant investigation<br>",
          "Utilizing poxviral vectored vaccine antibody induction—Progress prospect Over<br>last decade poxviral vector emerge mainstay approach induction<br>cell mediate immunity vaccination suitability human use lead<br>widespread clinical test candidate vector infectious intracellular pathogen<br>cancer contrast poxvirus widely perceive vaccine field poor<br>choice vector induction humoral immunity However grow body<br>data animal model recent clinical trial suggest vector<br>successfully utilized prime boost cell effective antibody responses<br>Significant progress make context heterologous prime–boost immunization regimes<br>whereby poxvirus able boost response prim vector lead<br>induction high titre antigen specific antibody response case<br>poxviral vector show stimulate humoral immunity encode transgene<br>particular viral surface protein influenza haemagglutinin veterinary field<br>recombinant poxviral vector make significant impact numerous vector<br>license use variety animal virus study continue explore<br>potential poxviral vector modulate qualitative aspect humoral response<br>well amenability adjuvantation seek improve quantitative antibody immunogenicity<br>Nevertheless underlie mechanism cell induction recombinant poxviruses remain<br>poorly define work necessary help guide rational optimization<br>future poxviral vaccine candidate aim induce antibody<br>",
          "Long term human coronavirus myelin cross reactive cell<br>clone derive multiple sclerosis patient Autoimmune reaction associate<br>involve genetic environmental factor Because murine coronaviruses induce<br>like disease human coronaviruses HCoV attractive candidate environmental<br>factor involve demyelinate pathology previously report isolation HCoV<br>229E myelin basic protein MBP cross reactive cell<br>line TCL patient investigate antigenic cross reactivity molecular<br>level long term cell clone TCC derive patient<br>vitro selection MBP proteolipid protein PLP HCoV strain<br>229E OC43 Overall TCC virus specific specific myelin<br>HCoV myelin cross reactive Twenty eight virus specific<br>TCC myelin specific TCC obtain six healthy donor<br>RACE PCR amplification chain five ten cross reactive<br>TCC confirm clonality sequence identify CDR3 region associate<br>cross reactivity Our finding promise implication investigation role<br>molecular mimicry coronaviruses myelin mechanism related disease initiation<br>relapse<br>",
          "Protection Newborn Animals Maternal Immunization Summary Providing protective<br>immunity neonatal animal early life associate numerous challenge<br>regard vaccine safety efficacy much simpler approach maternal<br>vaccination either pregnancy provide neonate passively transfer immunity<br>human medical societal legal risk immunize pregnant woman<br>important consideration undertake approach contrast maternal vaccination successfully<br>employ animal health industry decades These veterinary vaccine<br>proven safe efficient Although passively transfer antibody extensively<br>study immunological mechanism may equally important provide maternally<br>derive immunity<br>",
          "Overlapping synthetic peptide vaccine Several vaccine strategy aim<br>generate cell mediate immunity CMI microorganisms tumor While<br>epitope base vaccine offer advantage knowledge specific epitope<br>frequency major histocompatibility complex MHC allele require Here<br>show use promiscuous overlap synthetic peptide OSP immunogen<br>generate peptide specific CMI vaccinate outbred mouse different<br>strain inbred mice CMI response also recognize viral<br>protein OSP immunogen also induce CMI vivo dendritic<br>cell cell cocultures involve cell individual different HLA<br>haplotype Thus broad CMI induce OSP different experimental<br>setting use different immunogen without identify either epitope<br>MHC background vaccinee<br>",
          "Situación actual desarrollo una vacuna preventiva frente VIH<br>avance epidemia sida convertido obtención una vacuna eficaz<br>frente virus inmunodeficiencia humana VIH como objetivo científico<br>prioritario momento actual disponemos una vacuna preventiva frente<br>infección por VIH ningún modelo animal conseguido protección<br>frente infección esta revisión analizan dificultades existentes desarrollo<br>una vacuna contra sida especial los mecanismos escape<br>viral respuesta inmunitaria describen los prototipos vacunas preventivas<br>terapéuticas desarrollo los resultados obtenidos Por otra parte<br>sitúa esta investigación contexto sanitario económico social pandemia<br>sida analizan las polémicas actualmente planteadas desarrollo ensayos<br>clínicos con los diferentes tipos vacunas Due uncontrolled<br>progression aid epidemic development HIV vaccine become major<br>objective scientific research effective preventive vaccine human immunodeficiency<br>virus HIV available sterilise immunity yet achieve animal<br>model review major challenge develop aid vaccine particular<br>mechanism involve viral escape immune response analyse result<br>obtain different prototype therapeutic preventive vaccine summarize Finally<br>social economic health aspect relate research HIV vaccines<br>present controversy regard development clinical trial discuss<br>",
          "Kunjin replicon base simian immunodeficiency virus gag vaccine<br>RNA base non cytopathic replicon vector system base<br>flavivirus Kunjin show considerable promise new vaccine delivery<br>system Here describe test mice four different SIVmac239<br>gag vaccine deliver Kunjin replicon virus like particles<br>The four vaccine encode wild type gag gene<br>RNA optimise gag gene codon optimise gag gene<br>modify gag pol gene construct The vaccine behave<br>quite differently induction effector memory central memory response<br>mediation protection respect insert stability SIV gag pol<br>vaccine provide optimal performance These result illustrate RNA<br>base vector RNA sequence antigen profound unforeseen consequence<br>vaccine behaviour<br>",
          "Generation protective efficacy cold adapt attenuate genotype porcine<br>epidemic diarrhea virus The recent emergence emergence porcine<br>epidemic diarrhea virus PEDV underscore urgent need development<br>novel safe effective vaccine prevail strain study generate<br>cold adapt live attenuate vaccine candidate Aram P29<br>short term passage virulent PEDV isolate successively low<br>temperature Vero cell Whole genome sequence identify amino<br>acid change cold adapt strain insertion deletion throughout<br>genome Animal inoculation experiment confirm attenuated phenotype Aram<br>P29 virus natural host Pregnant sow orally administer<br>P29 live vaccine two dos week interval prior<br>parturition newborn piglet challenge parental virus The oral<br>homologous prime boost vaccination P29 significantly improve survival<br>rate piglet notably mitigate severity diarrhea PEDV fecal<br>shed challenge Furthermore strong antibody response PEDV detect<br>serum colostrum immunize sow sera offspring These result<br>demonstrate cold adapt attenuated virus use live vaccine<br>maternal vaccination strategy provide durable lactogenic immunity confer<br>passive protection litter PEDV<br>",
          "Characterization application monoclonal antibody protein SARS coronavirus Severe<br>acute respiratory syndrome coronavirus SARS CoV cause infectious<br>disease respiratory route Diagnosing disease effectively accurately early<br>stage essential prevent disease transmission perform antiviral treatment<br>study raise monoclonal antibody mAbs nucleocapsid protein SARS<br>CoV map epitope use different truncate protein fragment<br>The mapping epitopes valuable construct pair Abs use<br>serological diagnosis The result show six raise mAbs<br>divide two group recognize region amino acids 249–317<br>group 317–395 group This region span amino acid<br>249–395 contains predominant cell epitope locate terminus protein<br>One pair Abs consist protein specific rabbit polyclonal<br>antibody SARS CoV protein specific mAb select construct<br>sandwich ELISA kit The kit able specifically detect<br>SARS CoV proteins serum sample<br>",
          "Efficacy intranasal spray delivery adjuvanted live vaccine infectious<br>bronchitis virus experimentally infected poultry Live vaccine widely<br>use avian industry Such vaccine either inject deliver<br>animal mucosa usually adjuvanted study show live vaccine<br>efficacy improve formulation adjuvant model mucosal delivery live<br>infectious bronchitis vaccine chicken Three adjuvant technology test<br>use intranasal spray delivery method poultry Those technology<br>water oil water emulsion nanoparticles polymer adjuvant Intranasal<br>delivery polymer nanoparticles adjuvanted live vaccine improve significantly<br>antibody titer protection challenge observe compare commercial non<br>adjuvanted reference Moreover spray delivery polymer adjuvanted vaccine<br>show significantly high protection compare non adjuvanted reference<br>Our data demonstrates use MontanideTM adjuvant formulation live<br>poultry vaccine mucosal delivery confer vaccinate animal significantly<br>improve protection pathogen<br>",
          "Evaluation Mucosal Systemic Immune Responses Elicited GPI Adjuvanted<br>Influenza Vaccine Delivered Different Immunization Strategies Vaccines protection<br>respiratory infection optimally induce mucosal immune response respiratory<br>tract addition systemic immune response However current parenteral<br>immunization modality generally fail induce mucosal immunity mucosal<br>vaccine delivery often result poor systemic immunity order<br>find immunization strategy satisfies need induction mucosal systemic<br>immunity compare local systemic immune response elicit two<br>mucosal immunization give either intranasal intrapulmonary IPL route<br>response elicit mucosal prime follow systemic boost immunization<br>The study conduct BALB mouse vaccine formulation influenza<br>subunit vaccine supplement GPI saponin derive adjuvant While<br>optimal mucosal antibody titer obtain two intrapulmonary vaccination<br>optimal systemic antibody response achieve intranasal prime follow<br>intramuscular boost The latter strategy also result best<br>cell response yet ineffective inducing nose lung IgA<br>Successful induction secretory IgA IgG cell response achieve<br>prime boost strategy involve intrapulmonary immunization optimal immunization<br>give via intrapulmonary route Our result underline immunization<br>via lung particularly effective prim well boost local<br>systemic immune response<br>",
          "The attenuation vaccinia Tian Tan strain removal viral<br>M1L K2L gene generate safe vaccinia Tian Tan<br>VTT base vaccine vector necessary develop method attenuate<br>virus modify VTT MVTT2 GFP construct replace viral<br>M1L K2L gene GFP gene comparison parental VTT<br>MVTT2 GFP lose replication capacity rabbit RK13 human<br>HeLa cell line The life cycle viral replication<br>block different stage two cell line determine electron<br>microscope examination MVTT2 GFP less virulent VTT fold<br>measuring mouse body weight loss intranasal viral inoculation<br>fold determine intracranial LD50 value mice The foreign<br>GFP gene stable genetically round passage Vero cell<br>Importantly MVTT2 GFP elicit humoral cell mediate immune<br>response GFP gene mouse With two intramuscular inoculation<br>PFU virus anti GFP antibody reciprocal endpoint titer<br>reach determined ELISA The number IFN secrete cell<br>reach 350SFU per million splenocytes CD8 cell specific<br>epitope GFP Collectively removal M1L K2L gene useful<br>method generate attenuate vaccinia Tian Tan vaccine vector<br>",
          "Comparison efficiency antibody selection semi synthetic scFv non<br>immune Fab phage display libraries protein target rapid<br>development diagnostic immunoassay Rapid development diagnostic immunoassay novel<br>emerge genetically modify pathogen emergency situation dependent timely<br>isolation specific antibody Non immune antibody phage display<br>libraries efficient vitro method select monoclonal antibody hence<br>ideal circumstance Such library construct variety source cell<br>cDNA synthetically generate use variety antibody format typically<br>scFv Fab However antibody source format impact quality<br>antibody generate hence effectiveness methodology timely production antibody<br>carry comparative screen two antibody library semi synthetic<br>scFv library human derive Fab library protective antigen<br>toxin component Bacillus anthracis epsilon toxin Clostridium botulinum<br>show synthetic library produce diverse collection specific scFv<br>phage contain high frequency unnatural amber stop glycosylation<br>site limit conversion IgG also high number lose<br>specificity express IgG contrast limitation overcome use natural<br>human library Antibodies library could use develop sandwich<br>ELISA assay similar sensitivity However ease speed full<br>length IgG could generate human derive Fab library<br>make screen type library preferable method rapid antibody<br>generation diagnostic assay development<br>",
          "Peptide Mimicrying Between SARS Coronavirus Spike Protein Human<br>Proteins Reacts SARS Patient Serum Molecular mimicry define<br>similar structure share molecule dissimilar gene proteins general<br>strategy use pathogen infect host cell Severe acute<br>respiratory syndrome SARS new human respiratory infectious disease<br>cause SARS coronavirus SARS CoV The spike protein<br>SARS CoV play important role virus entry cell<br>study eleven synthetic peptide protein select base sequence<br>homology human proteins Two peptide D07 residues 927–937<br>D08 residue 942–951 recognize serum SARS patient Murine<br>hyperimmune serum peptides bound protein human lung epithelial<br>cell A549 Another peptide D10 residues 490–502 stimulated<br>A549 proliferate secrete The present result suggest select<br>protein region share sequence homology human protein may<br>play important role SARS CoV infection<br>",
          "Human monoclonal antibody prophylaxis SARS coronavirus infection ferret<br>Summary SARS coronavirus continue cause sporadic case severe<br>acute respiratory syndrome SARS China active passive immunoprophylaxis<br>disease induce SARS coronavirus available investigate prophylaxis SARS<br>coronavirus infection neutralise human monoclonal antibody ferret readily<br>infect virus Prophylactic administration monoclonal antibody reduce replication<br>SARS coronavirus lung infect ferret 3·3 log 2·6–4·0<br>log 0·001 completely prevent development SARS coronavirus induce<br>macroscopic lung pathology 0·013 abolish shed virus pharyngeal<br>secretions The data generate animal model show administration<br>human monoclonal antibody might offer feasible effective prophylaxis<br>control human SARS coronavirus infection<br>",
          "Identification Neutralizing Monoclonal Antibodies Targeting Novel Conformational Epitopes<br>Porcine Epidemic Diarrhoea Virus Spike Protein Since newly<br>identify variant porcine epidemic diarrhoea virus PEDV cause<br>high mortality neonatal piglet devastate swine industry The<br>spike glycoprotein PEDV contain multiple neutralize epitope major<br>target PEDV neutralization vaccine development understand antigenicity new<br>PEDV variant characterize neutralize epitope new genotype PEDV<br>isolate Taiwan PEDV Pintung PEDV generation neutralize monoclonal<br>antibody NmAbs Two NmAbs P4B E10E 1–10 recognize<br>ectodomain full length recombinant PEDV protein exhibit neutralizing<br>ability PEDV virus select Recombinant truncate protein use<br>identify target sequence NmAbs P4B show recognize terminus<br>26K equivalent epitope COE amino acid 575–639 PEDV<br>Interestingly E10E 1–10 could recognize novel neutralize epitope<br>435–485 within domain PEDV protein whose importance function<br>yet determine Moreover NmAbs could bind linearized proteins<br>indicate conformational epitope recognize This data could improve<br>understand antigenic structure PEDV protein facilitate future development<br>novel epitope base vaccine<br>",
          "Unique Epitopes Recognized Antibodies Induced Chikungunya Virus Infected<br>Non Human Primates Implications Study Immunopathology Vaccine Development<br>Chikungunya virus CHIKV Alphavirus cause chronic incapacitate arthralgia<br>human Although patient cohort study show production CHIKV<br>specific antibody fine specificity antibody response CHIKV completely<br>define The macaque model CHIKV infection establish due<br>limitation clinical specimen More importantly close relation human<br>allow study chronic infection identify important CHIKV target<br>study serum sample CHIKV infect macaque collect different<br>time point post infection use characterize antibody production<br>pattern kinetics Results reveal anti CHIKV antibody neutralize<br>glycoprotein Capsid nsP1 nsP3 nsP4 protein target anti<br>CHIKV antibody response macaque Furthermore linear cell epitope<br>recognize anti CHIKV antibody identify mapped structural localization<br>This characterize specificity anti CHIKV antibody response macaque<br>demonstrate importance different region CHIKV encode protein adaptive<br>immune response Information study provide critical knowledge aid<br>understand CHIKV infection immunity vaccine design pre clinical<br>efficacy study<br>",
          "Host protective ASP base vaccine parasitic nematode Ostertagia<br>ostertagi trigger cell activation mixed IgG1 IgG2 response<br>The mucus dwell parasite Ostertagia ostertagi one important<br>gastrointestinal nematode cattle Our group previously demonstrate protective<br>capacity vaccine parasite base native activation associate secrete<br>protein ASP1 nASP combination saponin adjuvant QuilA The<br>aim current study analyse effect antigen adjuvant cellular<br>humoral vaccine induce immune response compare native ASP<br>recombinant version express Pichia pastoris pASP replace QuilA<br>Immunization cattle protective nASP QuilA vaccine associate antigen<br>induce proliferation natural killer cell combine IFN secretion<br>induction mixed IgG1 IgG2 antibody response ASP specific<br>activation proliferation cell also observe mouse follow vaccination<br>regime Replacing QuilA nASP pASP significantly decrease capacity<br>vaccine trigger cell activation antibody response fail induce<br>protection challenge infection Reduction structurally anchor disulphide bond<br>nASP completely abolish ability induce cell activation antibody<br>responses highlight importance protein conformation immunostimulatory activity<br>",
          "Dengue Virus Reporter Replicon Valuable Tool Antiviral Drug<br>Discovery Analysis Virus Replication Mechanisms Dengue prevalent arthropod<br>borne viral disease cause dengue virus DENV member<br>Flaviviridae family considerable public health threat country billion<br>people live high risk area However specific antiviral<br>drug license vaccine currently target DENV infection The<br>replicon system factor need viral replication cell Since<br>development replicon system transient stable reporter replicons well<br>reporter virus use study various virological aspect DENV<br>identification DENV inhibitor review summarize DENV reporter replicon<br>system application high throughput screen HTS identification anti<br>DENV inhibitor also describe use system elucidation mechanism<br>virus replication viral dynamic vivo vitro<br>",
          "Protection chickens infectious bronchitis virus multivalent DNA vaccine<br>boost inactivated vaccine The protective efficacy DNA plasmid<br>encode avian infectious bronchitis virus IBV protein investigate<br>chicken Chickens inoculate monovalently plasmid pVAX1 16S1 pVAX1<br>16M pVAX1 16N alone multivalently combination three different<br>plasmid pVAX1 16S1 prime boost immunization protocol IBV<br>develop Chickens immunized multivalent DNA vaccine twice boost<br>inactivated vaccine Antibody titer chicken immunize pVAX1 16S1<br>much high monovalent group protective rate observe pVAX1<br>16S1 group The serum antibody titer prime boost<br>bird significantly high multivalent DNA vaccine group significantly<br>different compare inactivated vaccine group day age Additionally<br>prime boost group also show high level IBV<br>specific cellular proliferation compare monovalent group significant difference<br>find compare multivalent DNA vaccine group prime boost<br>group completely protect followed viral challenge<br>",
          "Biological molecular characterization ArkGA novel Arkansas serotype vaccine<br>highly attenuate efficacious protective homologous challenge Almost commercial<br>poultry vaccinate avian coronavirus infectious bronchitis virus IBV<br>use live attenuate vaccine mass administer spray day<br>hatch Although many different type IBV vaccine use<br>successfully ArkDPI serotype vaccine apply spray infect replicate<br>sufficiently provide protection homologous challenge study examine different<br>Ark vaccine strain Ark99 longer use commercially due<br>reactivity one day old chick determine could attenuate<br>passage embryonated egg still provide adequate protection Further<br>attenuation Ark99 vaccine achieve passage embryonated egg ArkGA<br>P20 P40 designate ArkGA still reactive suitable vaccine<br>candidate However ArkGA P60 give spray little vaccine<br>reaction one day old broiler chick induce protection<br>clinical sign ciliostasis follow homologous challenge addition vaccinate<br>challenged bird significantly less challenge virus important measure<br>protection compare non vaccinate challenged control The full<br>length genome viruses egg passage sequence use Illumina<br>platform data show single nucleotide polymorphisms SNPs accumulate<br>region genome associate viral replication pathogenicity cell tropism<br>ArkGA P60 accumulate SNPs key gene associate pathogenicity<br>polyprotein gene 1ab cell tropism spike gene compare<br>previous passage likely result attenuated phenotype These result<br>indicate ArkGA P60 vaccine safe spray vaccination broiler<br>chick induces suitable protection challenge pathogenic Ark type<br>virus<br>",
          "Biochemical biophysical characterization reovirus cell attachment protein Evidence<br>homotrimer The oligomerization state reovirus cell attachment protein<br>49K monomeric molecular weight determine biochemical biophysical mean<br>Full length protein product designate terminal truncate protein<br>product designate serotype reovirus mRNA transcript synthesizedin vitro<br>cotranslated rabbit reticulocyte lysate product analyze sodium dodecyl<br>sulfate polyacrylamide gel electrophoresis SDS PAGE condition allow<br>identification oligomeric form total four oligomeric protein band<br>correspond A2B1 A1B2 respectively consistently observe suggest protein<br>make three monomeric subunit Biophysical characterization purify use<br>column filtration sucrose gradient sedimentation analysis confirm highly<br>asymmetric shape allow determine molecular weight native protein<br>∼132K trimer Similar biophysical analysis two tryptic fragment<br>terminal fibrous tail 26K monomeric molecular weight terminal<br>globular head 23K monomeric molecular weight yield molecular<br>weight 77K 64K respectively correspond trimer therefore conclude<br>protein homotrimer provide supportive experimental evidence rationale anomalous<br>behavior oligomeric protein SDS polyacrylamide gel couple chemical<br>cross link study part lead previous suggestion might<br>high order oligomer<br>",
          "Functional analysis stem loop structure coronavirus 3′UTR design<br>series mutation separately destabilize two helical stem designate<br>predict covariation base model coronavirus 3′UTR Zust Mouse<br>hepatitis virus genome contain three four nucleotide mutation<br>destabilize either viable whereas genome carry mutation viable<br>genome carrying mutation plus compensatory mutation restore base<br>pairing yield virus wild type phenotype Larger mutation<br>completely disrupt generate various phenotype Mutations open lethal<br>Disruptions generate viable lethal mutant Genomes carry original<br>mutation plus compensatory mutation restore base pair viable<br>robust growth phenotypes These result support Zust model<br>coronavirus 3′UTR suggest stem require virus viability<br>",
          "The Molecular Biology Frog Virus Iridoviruses Infecting Cold<br>Blooded Vertebrates Frog virus FV3 best characterized member<br>family Iridoviridae FV3 study provide insight replication family<br>member serve model viral transcription genome replication virus<br>mediate host shutoff Although broad outline FV3 replication<br>elucidate precise role viral protein remain unknown Current<br>study use knock mediate antisense morpholino oligonucleotides asMO<br>small interfering RNAs siRNA knock follow replacement target<br>gene selectable marker homologous recombination ectopic viral gene<br>expression recombinant viral protein enable researcher systematically ascertain<br>replicative virulence relate gene function addition application molecular<br>tool ecological study provide novel way field biologists<br>identify potential pathogen quantify infection trace evolution ecologically<br>important viral specie review summarize current study use<br>FV3 also iridoviruses infect ectotherm described general principle<br>ascertain use FV3 serve model family study utilize<br>ranaviruses megalocytiviruses confirm extended understanding iridovirus replication Collectively<br>future effort elucidate molecular event viral replication intrinsic<br>extrinsic factor contribute disease outbreaks role host immune<br>system protection disease<br>",
          "The tetraspanin CD9 facilitate MERS coronavirus entry scaffold<br>host cell receptor proteases Infection envelop coronaviruses CoVs<br>initiate viral spike proteins bind cellular receptor follow<br>proteolytic cleavage receptor bound proteins prompt protein mediate<br>virus cell membrane fusion Infection therefore require close<br>proximity receptor proteases consider whether tetraspanins scaffold protein<br>know facilitate CoV infection hold receptor proteases together<br>cell membrane Using knockout cell line find tetraspanin<br>CD9 tetraspanin CD81 form cell surface complex dipeptidyl<br>peptidase DPP4 MERS CoV receptor type transmembrane serine<br>protease TTSP member TMPRSS2 CoV activate protease This<br>CD9 facilitate condensation receptor proteases allowed MERS CoV<br>pseudoviruses enter cell rapidly efficiently Without CD9 MERS<br>CoV virus activate TTSPs traffic endosomes cleave much<br>later less efficiently cathepsins Thus identify DPP4 CD9<br>TTSP protein complex necessary early efficient MERS CoV<br>entry evaluate importance complex vivo CoV infection model<br>use recombinant Adenovirus rAd5 vector express human DPP4<br>mouse lung thereby sensitize animal MERS CoV infection<br>When rAd5 hDPP4 vector express small RNAs silence<br>Cd9 Tmprss2 animal significantly less susceptible indicate CD9<br>TMPRSS2 facilitate robust vivo MERS CoV infection mouse<br>lung Furthermore proteins virulent mouse adapt MERS CoVs<br>acquire CD9 dependent cell entry character suggest CD9<br>selective agent evolution CoV virulence<br>",
          "The Lepidopteran endoribonuclease domain protein P102 display dramatically<br>reduce enzymatic activity form functional amyloid Hemocytes Heliothis<br>virescens Lepidoptera Noctuidae larva produce protein P102 putative<br>endoribonuclease domain previous work show P102 involve Lepidopteran<br>immune response form amyloid fibril catalyze localize melanin<br>deposition around non self intruder encapsulation prevent harmful<br>systemic spreading Here demonstrate P102 belong new class<br>protein least Lepidoptera diminish endoribonuclease activity probably due<br>lack two five catalytically essential residue show P102<br>homolog Trichoplusia Lepidoptera Noctuidae display catalytic site residue<br>identical P102 residual endoribonuclease activity ability form functional<br>amyloid basis result well sequence structural analysis hypothesize<br>Lepidoptera endoribonuclease orthologs catalytic site residue identical P102<br>form subfamily similar function<br>",
          "Evidence RNA pseudoknot loop helix interaction essential efficient<br>ribosomal frameshifting Edited Draper RNA pseudoknots structural element<br>participate variety biological process ribosomal frameshifting site several<br>type pseudoknot identify differ organisation functionality The pseudoknot<br>find infectious bronchitis virus IBV typical posse long<br>stem long loop second group pseudoknots distinguishable contain<br>stem shorter loop The NMR structure one pseudoknot<br>mouse mammary tumor virus MMTV reveal kink stem<br>stem junction kink conformation essential efficient frameshifting recently<br>investigate effect frameshifting modulate stem length stability IBV<br>base pseudoknots find stem least need efficient frameshifting<br>Here describe sequence manipulation necessary bypass requirement stem<br>convert short non functional IBV derive pseudoknot highly<br>efficient kink frameshifter pseudoknot Simple insertion adenine residue<br>stem stem junction essential feature kink pseudoknot sufficient<br>create functional pseudoknot additional change need efficient frameshifting<br>recover last nucleotide loop change The requirement end<br>loop consistent loop helix contact similar describe RNA<br>tertiary structure mutational analysis partner propose interaction loop<br>terminal adenine residue two G·C pair near top<br>stem reveal interaction essential efficient frameshifting The specific<br>requirement terminal residue lose loop increase suggest loop<br>helix contact may require pseudoknots short loop<br>",
          "The requirement environmental acidification Ibaraki virus infection host<br>cell The effect environmental acidification Ibaraki virus IBAV<br>infection test use endosomal inhibitory chemical low treatment<br>Treatment target cell endosomal inhibitor significantly decrease progeny<br>virus production IBAV outer capsid proteins VP5 VP2<br>remove virion purify IBAV expose low environment Further<br>experiment show exposure low buffer facilitate IBAV infection<br>cellular endosomal pathway impair bafilomycin Results obtain study<br>suggest acidic environment essential initiate IBAV infection<br>",
          "quadruplexes viruses function potential therapeutic application rich nucleic<br>acid form non canonical quadruplex structure G4s four<br>guanine fold planar arrangement Hoogsteen hydrogen bond Although<br>many biochemical structural study focus DNA sequence contain<br>successive adjacent guanine spontaneously fold G4s evidence vivo<br>relevance recently begin accumulate Complete sequence human genome<br>highlight presence ∼300 sequence potentially form G4s Likewise<br>presence putative sequence report various virus genome Human<br>immunodeficiency virus HIV Epstein–Barr virus EBV papillomavirus HPV<br>Many study focus telomeric G4s dynamic regulate enable<br>telomere synthesis Moreover role G4s propose cellular viral<br>replication recombination gene expression control parallel DNA aptamers<br>form G4s describe inhibitor diagnostic tool detect virus<br>hepatitis virus HAV EBV cauliflower mosaic virus CaMV<br>severe acute respiratory syndrome virus SARS simian virus<br>SV40 Here special emphasis give possible role structure<br>virus life cycle well use form oligonucleotides potential<br>antiviral agent innovative tool<br>",
          "Design Characterization Viral Polypeptide Inhibitors Targeting Newcastle Disease<br>Virus Fusion Paramyxovirus infection detect worldwide emerge zoonotic<br>virus currently specific therapeutic treatment vaccine available many<br>disease Recent study demonstrate peptide derive two heptad<br>repeat region HR1 HR2 paramyxovirus fusion protein could<br>use inhibitor virus fusion The mechanism underlying activity<br>accordance class virus fusion protein human immunodeficiency virus<br>HIV influenza virus fusion proteins member For class<br>virus fusion protein HR1 fragment bind HR2 form<br>six helix bundle three HR1 fragment form central<br>coil bundle surround three coil HR2 fragment post<br>fusion conformational state fusion core hypothesize introduce exogenous<br>HR1 HR2 compete endogenous counterpart result fusion inhibition<br>Using Newcastle disease virus NDV model design several<br>protein inhibitor denote HR212 well asHR121 Helix could<br>bind HR1 HR2 region fusion protein respectively All<br>protein express purified use GST fusion expression system<br>Escherichia coli The HR212 GST HR212 protein bind<br>HR1 peptide vitro display inhibitory activity NDV mediate<br>cell fusion HR121 Helix protein bind HR2 peptide<br>vitro inhibit virus fusion avirulent NDV strain add<br>cleavage fusion protein These result show design HR212<br>HR121 Helix protein could serve specific antiviral agent<br>These data provide additional insight difference virulent avirulent<br>strain NDV<br>",
          "Expression biochemical characterization nsP2 cysteine protease Chikungunya virus<br>Chikungunya virus CHIKV mosquito borne alphavirus cause epidemic<br>fever rash polyarthralgia Africa Asia Although know since<br>1950s new epidemiological clinical feature report recent outbreak<br>Indian Ocean regard emergence new disease Numerous severe<br>form infection describe put emphasis lack efficient antiviral<br>therapy Among virus encode enzymes nsP2 constitutes attractive<br>target development antiviral drug multifunctional protein approximately 90kDa<br>helicase motif terminal portion protein papain like protease<br>activity reside terminal portion The nsP2 proteinase essential<br>enzyme whose proteolytic activity critical virus replication work<br>recombinant CHIKV nsP2pro terminally truncate variant express Escherichia<br>coli purify metal–chelate chromatography The enzymatic property proteinase<br>determine use specific synthetic fluorogenic substrates This study<br>constitute first characterization recombinant CHIKV nsP2 cysteine protease<br>may useful future drug screening<br>",
          "Structural mechanistic basis preferential deadenylation snRNA Usb1 Post<br>transcriptional modification snRNA central spliceosome function Usb1 exoribonuclease<br>shorten oligo uridine tail snRNA result terminal cyclic<br>phosphate group eukaryotes include human Loss function mutation<br>human Usb1 cause rare disorder poikiloderma neutropenia result<br>snRNAs elongate end aberrantly adenylated Here show human<br>Usb1 remove adenosine fold great efficiency uridines explain<br>presence adenylated snRNAs cell lack Usb1 determine three<br>high resolution crystal structure Usb1 wild type Usb1<br>bound substrate analog adenosine monophosphate inactive mutant bound<br>RNAs terminal adenosine uridine These structure along simulation<br>catalytic mechanism illuminate molecular basis preferential deadenylation snRNA<br>The extent Usb1 processing influence secondary structure snRNA<br>",
          "The dengue virus non structural protein NS1 secrete<br>mosquito cell association intracellular cholesterol transporter chaperone caveolin<br>complex ABSTRACTDengue virus DENV mosquito borne virus family<br>Flaviviridae The RNA viral genome encodes polyprotein translationally<br>process three structural protein seven non structural proteins<br>The non structural protein NS1 multifunctional viral protein<br>actively secrete vertebrate mosquito cell DENV infection mosquito<br>cell NS1 secrete caveolin CAV dependent manner unconventional<br>pathway The caveolin chaperone complex CCC cytoplasmic complex<br>form caveolin chaperone FKBP52 Cy40 CyA responsible cholesterol<br>traffic inside cell work demonstrate infect mosquito cell<br>DENV NS1 secrete early unconventional route bypass Golgi<br>apparatus close association CCC Treatment mosquito cell classic<br>secretion inhibitor brefeldin golgicide Fli show effect NS1<br>secretion significant reduction recombinant luciferase secretion virion release<br>Silencing expression CAV1 FKBP52 siRNAs inhibition CyA cyclosporine<br>result significant decrease NS1 secretion without affect virion<br>release Using localization inmunoprecipitation proximity ligation assay NS1<br>find localize interact protein component CCC mosquito infect<br>cell addition CAV FKBP52 expression find augment DENV<br>infected cell Finally treatment ZIKV infect mosquito cell<br>brefeldin golgicide show effect NS1 secretion affect virion<br>release ZIKV NS1 also find localize CAV infect<br>mosquito cell These result suggest mosquito cell ZIKV<br>NS1 follow secretory pathway observed DENV NS1 The<br>association NS1 cholesterol transporter CCC agree lipoprotein nature<br>secrete hexameric NS1 AUTHOR SUMMARYDengue protein NS1 secrete<br>infected mosquito vertebrate cell human secrete NS1 associated<br>pathogenesis mosquito cell NS1 follow unconventional secretion pathway<br>dependent Caveolin This work show mosquito cell NS1<br>secretion associate chaperone caveolin complex complex form caveolin<br>several chaperone charge cholesterol transport within cell Reduction<br>expression activity chaperone caveolin complex mosquito infect cell<br>diminish secretion NS1 without affect virion release Direct<br>interaction NS1 chaperone caveolin complex protein demonstrate several<br>assay Moreover increase expression caveolin chaperone FKBP52 dengue<br>infection find presumably response high requirement proteins dengue<br>virus infection Results obtain ZIKV infect mosquito cell<br>suggest also ZIKV NS1 release follow unconventional secretory<br>route association chaperone caveolin complex The function secrete<br>NS1 within mosquito unclear However give importance soluble<br>NS1 vertebrate host manipulation NS1 secretory route may<br>prove valuable strategy dengue mosquito control patient treatment<br>",
          "High Resolution Analysis Coronavirus Gene Expression RNA Sequencing<br>Ribosome Profiling Members family Coronaviridae large genome RNA<br>viruses typically region kilobases Several coronaviruses Severe acute<br>respiratory syndrome relate coronavirus SARS CoV Middle East<br>respiratory syndrome relate coronavirus MERS CoV medical importance<br>high mortality rate case SARS CoV significant pandemic<br>potential Other coronaviruses Porcine epidemic diarrhea virus Avian<br>coronavirus important livestock pathogen Ribosome profiling technique exploit<br>capacity translate ribosome protect around nucleotide mRNA ribonuclease<br>digestion Ribosome protect mRNA fragment purify subjected deep<br>sequencing map back transcriptome give global snap shot<br>translation Parallel RNA sequence allows normalization transcript abundance<br>Here apply ribosome profile cell infect Murine coronavirus<br>mouse hepatitis virus strain A59 MHV A59 model<br>coronavirus genus SARS CoV MERS CoV The data<br>obtain allow study kinetics virus transcription translation exquisite<br>precision study timecourse positive negative sense genomic subgenomic<br>viral RNA production relative translation efficiency different virus<br>ORFs Virus mRNAs find translate efficiently host mRNAs<br>rather virus translation dominate host translation later time<br>point due high level virus transcript Triplet phase<br>profile data allow precise determination translate read frames<br>reveal several translate short open read frame upstream<br>embed within know virus protein cod region Ribosome<br>pause site identify virus replicase polyprotein pp1a ORF<br>investigate experimentally Contrary expectation ribosome find pause ribosomal<br>frameshift site knowledge first application ribosome profile RNA<br>virus<br>",
          "Venezuelan equine encephalitis virus non structural protein nsP3<br>interacts RNA helicases DDX1 DDX3 infect cell The<br>mosquito bear New World alphavirus Venezuelan equine encephalitis<br>virus VEEV Category select agent approve vaccine therapy<br>treat infected human Therefore imperative identify novel target<br>target effective therapeutic intervention aim identify validate interaction<br>VEEV nonstructural protein nsP3 host protein determine consequence<br>interaction viral multiplication use tag nsP3 infectious clone<br>rTC nsP3 identify validate two RNA helicases DDX1<br>DDX3 interact VEEV nsP3 addition DDX1 DDX3 knockdown<br>result decrease infectious viral titer Furthermore propose functional<br>model nsP3 DDX3 complex interacts host translational machinery<br>essential viral life cycle This study lead future<br>investigation understand importance VEEV nsP3 viral multiplication apply<br>information discovery novel host target therapeutic option<br>",
          "Contribution porcine aminopeptidase porcine deltacoronavirus infection Porcine deltacoronavirus<br>PDCoV member genus Deltacoronavirus emerge swine enteropathogenic coronavirus<br>CoV Although outstanding effort lead identification Alphacoronavirus Betacoronavirus<br>receptor receptor Deltacoronavirus unclear Here compare amino acid<br>sequence several representative CoVs Phylogenetic analysis show PDCoV<br>spike protein close cluster contain transmissible gastroenteritis virus<br>TGEV utilize porcine aminopeptidase pAPN functional receptor Ectopic<br>expression pAPN non susceptible BHK cell render susceptible<br>PDCoV These result indicate pAPN may functional receptor<br>PDCoV infection However treatment APN specific antibody inhibitor<br>completely block PDCoV infection IPI porcine intestinal epithelial<br>cell pAPN knockout IPI cell completely block TGEV<br>infection slightly decrease PDCoV infection Homologous model pAPN<br>terminal domain CTD PDCoV TGEV show TGEV CTD<br>adopt turn β1–β2 β3–β4 form tip barrel recognize<br>pAPN However top residue β1–β2 turn PDCoV CTD<br>possibility support interaction pAPN β3–β4 turn fail contact<br>pAPN also discuss evolution variation PDCoV CTD base<br>structure information provide clue explain usage pAPN PDCoV<br>Taken together result present herein reveal pAPN likely<br>critical functional receptor PDCoV although involve PDCoV infection<br>",
          "BUHO MATLAB Script Study Stress Granules Processing Bodies<br>High Throughput Image Analysis The spontaneous reversible formation<br>focus filament contain protein involve different metabolic process<br>common nucleus cytoplasm Stress granules SGs process body<br>PBs belong novel family cellular structure collectively know<br>mRNA silence foci harbour repress mRNAs associate protein<br>SGs PBs highly dynamic form upon stress dissolve<br>thus release repress mRNAs accord change cell physiology<br>addition aggregate contain abnormal protein frequent neurodegenerative disorder<br>spite grow relevance supramolecular aggregate diverse cellular function<br>reliable automated tool systematic analysis lack Here report<br>MATLAB Script term BUHO high throughput image analysis<br>cellular focus use BUHO assess number size distribution<br>distinct object minimal deviation manually obtain parameter BUHO<br>successfully address induction SGs PBs mammalian insect cell<br>expose different stress stimulus also use BUHO ass<br>dynamic specific mRNA silence focus term Smaug focus<br>focus primary neuron upon synaptic stimulation Finally use<br>BUHO analyze role candidate gene formation RNAi base<br>experiment find FAK56D GCN2 PP1 govern formation The<br>role PP1 conserve mammalian cell judge effect PP1<br>inhibitor salubrinal involve dephosphorylation translation factor eIF2α All<br>experiment analyze manually BUHO result differ less average<br>value The automated analysis user friendly method allow<br>high throughput image processing short time provide robust<br>flexible reliable alternative laborious sometimes unfeasible visual scrutiny<br>",
          "Factors affect translation program ribosomal frameshifting site Cocksfoot<br>mottle virus RNA vivo The ratio proteins P27<br>replicase Cocksfoot mottle virus CfMV regulate via program<br>ribosomal frameshift PRF minimal frameshift signal slippery UUA<br>AAC heptamer downstream stem–loop structure insert dual reporter<br>vector direct PRF efficiency yeast bacteria P27 encode<br>CfMV sequence flank minimal frameshift signal cause fold<br>increase PRF efficiencies yeast bacteria addition expect fusion<br>proteins termination product end putatively frameshift site find<br>yeast cell propose amount premature translation termination control<br>mRNAs play role determine calculate −1PRF efficiency expression<br>CfMV P27 dual reporter vector contain minimal frameshift<br>signal reduce production downstream reporter whereas replicase expression<br>pronounce effect This finding allow propose CfMV protein<br>P27 may influence translation frameshift site mechanism need<br>elucidate<br>",
          "Comprehensive structural analysis design incomplete polypeptide chain replicase<br>nonstructural protein severe acute respiratory syndrome coronavirus The<br>cotranslational folding recognize cooperative process occur nearly completion<br>polypeptide sequence domain Here investigate challenge face polypeptide<br>segment non vectorial barrel fold Besides biological interest<br>behind SARS coronavirus non structural protein nsp1 amino<br>acid study model two structural feature motivate use<br>work recombinant production dependent temperature great solubility express<br>low temperature This indication cotranslational guidance native protein<br>conformation Conversely nsp1 six strand mixed parallel antiparallel<br>barrel intricate long range interaction indicate need full<br>length protein fold properly use non denature purification<br>condition allow characterization polypeptide chain different length mimic<br>landscape cotranslational fold barrel avoid major technical hindrance<br>work nascent polypeptide bound ribosome Our result show<br>partially fold state form soon amino acid second<br>strand present amino acid These partially fold state<br>different base length polypeptide chain The native helix<br>amino acid 24–37 identified transient structure 20–30 propensity<br>identify presence regular secondary structure fourth native strand<br>present amino acid parallel collapse non native structure<br>Interestingly polypeptide sequence native strand characteristic helix Our<br>comprehensive analysis support idea incomplete polypeptide chain one<br>nascent protein much earlier end translation adopt abundance<br>specific transient fold instead disorder conformation<br>",
          "RNA pseudoknots downstream frameshift site retroviruses RNA pseudoknot<br>structural motif could implications wide range biological process<br>RNAs study potential RNA pseudoknots downstream know suspected<br>retroviral frameshift site predict Rous sarcoma virus primate<br>immunodeficiency viruses HIV HIV SIV equine infectious anemia<br>virus visna virus bovine leukemia virus human cell<br>leukemia virus type mouse mammary tumor virus Mason<br>Pfizer monkey virus simian SRV type retrovirus Also<br>putative RNA pseudoknots detect gag pol overlap two<br>retrotransposons Drosophila gypsy mouse intracisternal particle For sequence<br>thermodynamic stability statistical significance secondary structure involve predicted<br>tertiary structure assess compare Our result show stem<br>loop structure pseudoknots thermodynamically highly stable statistically significant<br>relative configuration potentially occur gag pool gag pro<br>pro pol junction domains viruses nucleotide upstream downstream<br>possible frameshift site include Moreover structural feature predict<br>pseudoknots follow frameshift site propol overlap HTLV HTLV<br>retrovirus structurally well conserve The occurence eight compensatory<br>base change tertiary interaction two related sequence allow<br>conservation tertiary structure spite sequence divergence The result<br>support possible control mechanism frameshifting propose Brierley Jacks<br>",
          "The rubella virus glycoprotein arrest novel post pre<br>Golgi compartment Evidence accumulate distinct compartment exist secretory<br>pathway interpose rough RER Golgi stack study define<br>novel post RER pre Golgi compartment unassembled subunit<br>rubella virus glycoprotein accumulate When express CHO cell<br>absence glycoprotein transport Golgi complex arrest The compartment<br>accumulate consist tubular network smooth membrane continuity RER<br>distinctive property either RER Golgi previously characterize intermediate<br>compartment lack RER Golgi membrane protein disrupt agent<br>disrupt either RER thapsigargin ionomycin Golgi nocodazole brefeldin<br>However luminal proteins bear KDEL signal access compartment<br>Kinetically site arrest lie distal site palmitylation occur<br>proximal low temperature degree block Taken together find<br>suggest site arrest corresponds locate close exit site<br>This compartment could identify morphologically highly amplified cell<br>overexpressing unassembled subunit may counterpart among transitional element<br>non transfected cell conclude site arrest may represent<br>new compartment differentiate proximal moiety intermediate compartment<br>",
          "Exploring challenge health system preparedness communicable disease Arbaeen<br>mass gather qualitative study Background Infectious diseases common<br>problem mass gathering especially lack health system preparedness<br>Since Iran one important country walk path Arbaeen<br>vital role provide health service pilgrims experience health<br>challenge participant key importance The aim study explore<br>stakeholder experience health system preparedness challenge provide suggestion<br>prevent infectious disease Arbaeen mass gathering Methods qualitative<br>research method use conventional content analysis approach The<br>number participant include executive manager health policymakers enter<br>study among participant Semi structure interview use generate<br>data Interviews analyze mean content analysis face face<br>interview Results Data analysis result extraction four main<br>theme sub theme Health infrastructure defect Iraq three<br>sub theme health abandonment Iraq weaknesses health culture<br>problem relate health system poor control causative factor<br>infectious disease three sub theme underlying factor prevalence<br>contagious disease health system response communicable disease ignore<br>risk Arbaeen ceremony low perception risk pilgrim three<br>sub theme lack awareness pilgrim fatalism pilgrims unhygienic<br>belief pilgrim ineffectiveness health education two sub theme<br>train shortage target group educational content problem show<br>participant experience health system challenge cop infectious disease<br>Arbaeen ceremony Conclusion Pilgrim base training plan control<br>challenge may change threat opportunity improve health participant<br>mass gather Arbaeen region<br>",
          "Multicenter EuroTravNet GeoSentinel Study Travel relate Infectious Diseases<br>Europe analyze prospective data European patient seek treatment<br>GeoSentinel sit Gastrointestinal illness particularly tourists fever visit<br>friend relatives VFRs skin disorder tourist common reason<br>seek medical care Diagnoses vary country origin region<br>visit category traveler VFRs return sub Saharan Africa<br>Indian Ocean islands likely experience falciparum malaria group<br>Multiple correspondence analysis identify Italian French Swiss VFRs<br>expatriate traveler sub Saharan Africa Indian Ocean Islands<br>likely exhibit febrile illnesses German tourist Southeast south<br>central Asia likely seek treatment acute diarrhea Non<br>European travelers patient industrialized country less likely acquire<br>certain travel associate infectious disease These result consider<br>practice travel medicine development health recommendation European traveler<br>",
          "Infectiologie itinérante Résumé Objectifs Une prise conscience impose<br>égard développement actuel des épidémies infection sévères développement<br>considérable transport internationaux beaucoup amplifié circulation agent infectieux<br>véhiculés par voyageurs migrant mais aussi par les<br>animaux marchandises Observations distribution infection potentiel épidémique dont<br>certaines malady émergentes tendance devenir planétaire Cette évolution<br>impose renforcement surveillance internationale capacités réponse aux alertes<br>Solutions Les pay industrialisés mieux équipés moyens structure<br>opérationnelles interviennent façon prépondérante dans cette lutte Leur<br>contribution est nécessaire dans les pay développement particulièrement<br>affectés par les endémo épidémies infectieuses cette contribution<br>progresse doit être soutenue Cette revue rapporte travers<br>des observation personnelles pathology les plus menaçantes travers<br>monde Objectives become aware current development international travel<br>contribute amplify circulation sort infectious agent describe current<br>situation Observations Infectious agent carry mean traveller immigrant<br>well animal good merchandise The distribution infection potentially<br>responsible epidemic tend become worldwide emergent disease Solutions<br>Such situation impose enhancement international surveillance development resource<br>control response alert Developed country posse advance equipment<br>interventional structure may intervene predominantly fight epidemic The<br>necessary contribution industrialize country increase develop country infectious<br>epidemic spread intervention must sustain The content article<br>review major threaten infectious epidemic disease contribution personal<br>experience<br>",
          "Infection Prevention Control Tropics Tropical setting present unique<br>challenge practice infection prevention control These multi faceted<br>due difference climate culture social political milieu low<br>middle income country situate tropic well lack resource<br>The emergence communicable disease low vaccination coverage also<br>lead nosocomial augmentation community outbreaks increase economic burden<br>hospital management Addressing challenge require innovative low cost<br>tailor solution suit tropical environment<br>",
          "The Politics SARS Rational Responses Ambiguity Symbols Chaos<br>The main focus article SARS event political process<br>involve political leader administrator health professional How understand<br>reaction SARS main actor institution What aspects preoccupy<br>definition SARS change process selection jurisdiction choose China<br>Hong Kong SAR Canada World Health Organization explore<br>question The start point view reaction can not<br>primarily see instrumental base rational standard operating procedure<br>SOPs technical expertise may better understood garbage perspective<br>From review event publicly report find suggested garbage<br>theory politician administrator response SARS outbreak combination compete<br>rationality overlap agendas Critical decision trigger extraneous factor<br>administrative action shape dramatic switch one set standard<br>operating procedures another event unfold The public health<br>issue constantly vie agendas compelling alignment among occurred<br>professional technical rationale solve problem become dominant<br>",
          "Low medium low risk perception dengue chikungunya Zika<br>outbreaks infectious disease physician France Western Europe BACKGROUND<br>Many tropical country currently experience dengue DEN chikungunya<br>CHIK also recently Zika ZIKA epidemic particularly Latin<br>America Although risk transmission spread infection temperate region<br>remain controversial issue vector borne diseases widely report<br>medium focus preventive strategy national international policy maker<br>public health authority context want determine extent risk<br>perception infectious disease physician current future risk arboviral<br>disease introduction autochthonous case development epidemic scenario France<br>Western Europe METHODS aim developed original standardize questionnaire<br>survey disseminate French Infectious Diseases Society physician member<br>RESULTS find physician perceive risk introduction outbreak development<br>DEN CHIK ZIKA France low medium low Generalized<br>Linear Model identify medical school training extent professional<br>experience awareness French national plan regard arboviral infection<br>significant predictor low risk perception among respondent CONCLUSION<br>Despite fact arboviral disease increasingly import France sometimes<br>result sporadic autochtonous transmission French physicians perceive risk<br>high Better communication education target health professional citizen<br>need enhance effectiveness French national plan prepare arboviral<br>disease ELECTRONIC SUPPLEMENTARY MATERIAL The online version article<br>s12889 contains supplementary material available authorized user<br>",
          "insightful recollection since distort key theory bear year<br>ago During last three decade many effort make<br>study protein cleavage sit disease cause enzyme HIV<br>human immunodeficiency virus protease SARS severe acute respiratory<br>syndrome coronavirus main proteinase become increasingly clear via<br>minireview motivation drive aforementioned study quite wise result<br>acquire study reward particularly develop peptide drug<br>",
          "Dark Side Therapies Immunoglobulin Concentrates The Adverse Events<br>Therapy human immunoglobulin IgG concentrate success story ongoing<br>decade ever increase demand plasma product The success<br>IgG concentrate clinical level document slowly increase number<br>register indication rapid increase label use topic dealt<br>another contribution special issue Frontiers Immunology part success<br>adverse event profile IgG concentrate even life long<br>need therapy excellent Transmission pathogen last decade could<br>entirely control antecedent introduction authority regulatory network instal<br>quality standard plasma fractionation industry The cornerstone regulatory<br>network current good manufacturing practice Non infectious AEs<br>occur rarely mainly mild moderate However recent time<br>increase frequency hemolytic thrombotic AEs raise worry question<br>possible background AEs Below review element non infectious<br>AEs particularly focus hemolysis thrombosis discuss introduction plasma<br>fractionation ion exchange chromatography polish immunoaffinity chromatographic step<br>might alter repertoire specificity influence profile efficacy IgG<br>concentrate<br>",
          "One health ecological evolutionary framework tackle Neglected Zoonotic<br>Diseases Understanding complex population biology transmission ecology multihost<br>parasite declare one major challenge biomedical science 21st<br>century Neglected Zoonotic Diseases NZDs perhaps neglect Neglected<br>Tropical Diseases NTDs Here consider multihost parasite transmission<br>evolutionary dynamic may affect success human animal disease<br>control program particularly neglect disease develop world review<br>different type zoonotic interaction occur ecological evolutionary potential<br>relevance current human control activity make suggestion development<br>empirical evidence base theoretical framework well understand predict<br>outcome interaction particular consider whether preventive chemotherapy current<br>mainstay NTD control successful without One Health approach<br>Transmission within animal reservoir human important ecological evolutionary<br>consequence drive evolution establishment drug resistance well provide<br>selective pressure spill‐over host switching hybridization introgressions animal<br>human parasites Our aim highlight importance elucidate disease<br>ecology include identify key host tailor control effort<br>accordingly understanding parasite evolution precisely infectious agent may<br>respond adapt anthropogenic change Both elements essential alleviate<br>disease risk NZDs human domestic animal wildlife<br>",
          "The forecasting International Expo tourism use quantitative qualitative<br>technique This study predict number visitor international tourism<br>Expo hold Korea unprecedented event host city Forecasting<br>demand mega event receive limited attention literature Unlike<br>study forecast international tourism demand forecast Expo demand<br>involve use quantitative forecasting model qualitative technique data<br>limitation Combining quantitative technique willingness visit WTV survey<br>predict Expo demand million visitor comparison use Delphi<br>method expert predict Expo demand million visitor For<br>study Delphi method provide conservative estimate estimate combine<br>quantitative technique WTV Policy implication present directed toward<br>Expo planner practitioner term demand supply side application<br>result decision make process future challenge surround demand<br>forecasting<br>",
          "farm biosecurity perceive professional visit Swedish farm BACKGROUND<br>farm biosecurity important part disease prevention control apply<br>live animal contact well indirect contact via professional<br>visit farm work The objective study investigate professional<br>visit animal farm Sweden daily work perceive farm<br>condition biosecurity factor influence biosecurity routine describe obstacle<br>biosecurity Suggestions improvement also ask Questionnaires distribute professional<br>visit farm daily work veterinarian livestock haulier artificial<br>insemination technician animal welfare inspector cattle hoof trimmer<br>The sample convenience sample base accessibility register collaboration<br>organisation distribute questionnaire Respondents ask availability certain biosecurity<br>condition related farm visit facility hand wash available<br>important different factor routine open end question describe<br>obstacle suggestion improvement RESULTS After data clean response<br>person There difference proportion visit farm report certain<br>biosecurity measure place relate animal specie present farm<br>general visit pig farm high proportion biosecurity measure<br>place whereas condition poorer sheep goat farm horse<br>farm There also difference visitor category perceive condition<br>biosecurity vary group livestock haulier access hand wash<br>facility often veterinarians group majority respondent perceive obstacle<br>farm biosecurity among veterinarian perceive obstacle Many report<br>obstacle related basic biosecurity access soap water Responsibility<br>identify key issue farmer expect visitor take responsibility<br>keep biosecurity provide adequate farm condition CONCLUSIONS Many<br>respondent report obstacle keep good biosecurity relate farm<br>condition There gap come responsibility need clarify Visitors<br>need take responsibility avoid spread disease farmer need<br>assume responsibility provide adequate condition farm biosecurity<br>",
          "Translational Rodent Models Research Parasitic Protozoa—A Review Confounders<br>Possibilities Rodents particular Mus musculus long invaluable history<br>model human diseases biomedical research although translational value<br>challenge number case provide example rodent suboptimal model<br>human biology discus confounders influence experiment may explain<br>misleading result Infections rodent protozoan parasite exception require<br>close consideration upon model choice focus significant difference<br>inbred outbred wild animal importance factor microbiota gain<br>attention crucial variable infection experiment Frequently mouse rat<br>model choose convenience availability institution rather unbiased evaluation<br>whether provide answer give question Apart general discussion<br>translational success failure provide example infection single celled<br>parasite choose lab rodent give contradictory misleading result<br>possible discuss reason present emerge alternative traditional rodent<br>model humanize mice organoid primary cell culture call<br>recombinant inbred strain Collaborative Cross collection also potential<br>solution certain challenge addition emphasize advantage use wild<br>rodent certain immunological ecological behavioral question The experimental<br>challenge availability specie specific reagent come use non<br>model system also discuss Our intention foster critical<br>judgment traditional newly available translational rodent model research<br>parasitic protozoa complement exist mouse rat model<br>",
          "Nonconventional opponent review malaria leishmaniasis among United States<br>Armed Forces United States military engage different country<br>culture throughout world personnel become exposed new biosphere<br>well There many infectious pathogen endemic two particular<br>importance Plasmodium Leishmania respectively cause malaria leishmaniasis These<br>parasite know cause significant disease burden endemic locale<br>thus pose threat military traveler This review introduce<br>reader basic life cycle disease mechanism Local military<br>epidemiology describe specific action take military prevention treatment<br>purpose Complications measure regard human health also discuss<br>include possible chemical toxicity Additionally poor recognition diseases<br>upon individual return lead complication treatment delay United<br>States examine Information canine leishmaniasis poorly study relative<br>human manifestation importance due utilization dog military endeavor<br>present Future implication American healthcare system regard malaria<br>leishmaniasis also present<br>",
          "SARS hospital priority set qualitative case study evaluation<br>BACKGROUND Priority set one difficult issue face hospital<br>fund restriction change patient need deadly communicable disease<br>outbreak Severe Acute Respiratory Syndrome SARS Toronto amplifies<br>difficulties hospital priority set The purpose study describe<br>evaluate priority set hospital response SARS use ethical<br>framework accountability reasonableness METHODS This study conduct large<br>tertiary hospital Toronto Canada There two data source<br>key document email bulletin interview key informant Analysis<br>use modify thematic technique three phase open cod<br>axial cod evaluation RESULTS Participants describe type priority<br>setting decision decision making process reason use Although<br>hospital leadership make effort meet condition accountability reasonableness<br>acknowledge decision make ideal describe good practice opportunity<br>improvement CONCLUSIONS Accountability reasonableness framework use guide fair<br>priority set health care organization hospital midst crisis<br>SARS guidance incomplete consequence uncertain information constantly change<br>hour hour decision involve life death fairness important<br>rather less<br>",
          "Tackling challenge health equity China Summary term economic<br>development China widely acclaim miracle economy Over period<br>rapid economic growth however China reputation health slip<br>1970s China shin example health development longer Government<br>public concern health equity grow China health equity<br>challenge truly daunt vicious cycle three synergistic factor<br>social determinant health become inequitable imbalance roles market<br>government develop concern among public grow fairness health<br>With economic boom grow government revenue China unlike<br>country challenge health inequity afford necessary reform economic<br>development hand hand improve health equity Reforms improve<br>health equity receive immense popular support governmental commitment<br>interest public health community worldwide<br>",
          "Emerging Reemerging Infectious Diseases Global Overview<br>",
          "Coping flexibility college student depressive symptom BACKGROUND The<br>current study explore prevalence depress mood among Chinese<br>undergraduate student examine cop pattern degree flexibility flexibility<br>pattern associate mood METHODS set questionnaire assess cop<br>pattern cop flexibility depressive symptom administer student men<br>woman RESULTS total participant complete entire set questionnaire<br>report frequency two stressful life event criterion need<br>calculate variance perceive controllability Findings show high level<br>depressive symptom significantly associate high level event frequency<br>event impact low level perceive controllability cop effectiveness<br>ratio strategy situation fit Depressive symptom significantly associate<br>cognitive flexibility variance perceive controllability Gender significant moderator<br>report association CONCLUSIONS Findings indicate Chinese university student<br>depressive symptom report experience great number negative event<br>non depress university student addition undergraduate depressive symptom<br>likely undergraduates utilize maladaptive cop method Such finding<br>highlight potential importance intervention aim help undergraduate student<br>low cop flexibility develop skill cope stressful life<br>event<br>",
          "Ruling trouble Unacceptable behaviour control Hong Kong public<br>housing Resident satisfaction contingent housing design construction also<br>neighbourhood quality However many neighbourhood around world distress<br>neighbourhood nuisance unacceptable behaviour resident While interventionist approach<br>incentive adopt many country curb problem punitive measure<br>use Hong Kong instead The Housing Department launch<br>Marking Scheme Tenancy Enforcement Public Housing Estates immediately<br>SARS epidemic The scheme operate penalty point system<br>sit tenant expel public housing receive sixteen point<br>misdeed commit Yet mark scheme put onto stage<br>without prior public consultation Besides criticised unfair tenure<br>bias enforcement Also whether scheme widely accept highly<br>doubtful Against background study aim explore tenant view<br>regard resident unacceptable behaviour mark scheme Hong Kong<br>public housing The finding research offer valuable insight<br>perceive extent cause neighbourhood problem Moreover research let<br>public administrator know acceptability mark scheme among public<br>housing tenant<br>",
          "Les virus bougent périls planétaires SUMMARY Emerging viral<br>disease nothing new Smallpox probably reach Europe Asia<br>5th century yellow fever emerge Americas 16th century<br>consequence African slave trade Dengue fever arise simultaneously<br>South East Asia Africa North America 18th century<br>1918–1919 call Spanish flu spread like wildfire five<br>continent kill million people The second half 20th<br>century saw emergence HIV AIDS among viral disease<br>Even worry fact emerge emerge viral disease tendency<br>spread quickly widely last decade invade whole country<br>continent witness recent outbreak Nipah virus West Nile<br>Rift Valley fever SARS monkeypox avianflu H5N1 Chikungunya<br>The complex factor underlie new trend briefly discuss<br>",
          "Bibliometric Analysis Publications Pluripotent Stem Cell Research OBJECTIVE<br>Human pluripotent stem cell self renew cell ability<br>differentiate variety cell view great potential field regenerative<br>medicine Research pluripotent stem cell hold great promise<br>patient specific therapy various disease study pluripotent stem<br>cell article publish screen retrieve Science Citation Index<br>Expanded SCI EXPANDED MATERIALS AND METHODS retrospective study<br>publication trend citation trend top article distribution journal<br>Web Science category analyze Five bibliometric indicator include<br>total article independent article collaborative article first author<br>article correspond author article apply compare publication country<br>institution RESULTS The impact top article change year<br>year Top cite article previous publication year recent<br>year Induced pluripotent stem cell embryonic stem cell<br>use author keywords pluripotent stem cell research addition<br>winner Nobel Prize physiology medicine Prof Shinya Yamanaka<br>publish four top ten frequently cite article CONCLUSION<br>The comprehensive analysis highly cite article stem cell<br>field could identify milestone important contributor give historic<br>perspective scientific progress<br>",
          "Repositioning salicylanilide anthelmintic drug treat adenovirus infection The<br>reposition drug already approve regulatory agency indication emerge<br>alternative development new antimicrobial therapy The reposition process<br>involve low risk cost novo development novel antimicrobial<br>drug Currently infection adenovirus show steady increment high<br>clinical impact immunosuppressed immunocompetent patient The lack safe<br>efficacious drug treat infection support search new antiviral<br>drug Here evaluate anti adenovirus activity niclosanide oxyclozanide<br>rafoxanide three salicylanilide anthelmintic drug Also carry cytotoxicity<br>evaluation partial characterization mechanism action drug The salicylanilide<br>anthelmintic drug show significant anti adenovirus activity low<br>micromolar concentration little cytotoxicity Moreover mechanistic assay suggest<br>difference way drug exert anti adenovirus activity Niclosamide<br>rafoxanide target transport HAdV particle endosome nuclear envelope<br>whilst oxyclozanide specifically target adenovirus immediately early gene<br>E1A transcription Data suggest study salicylanilide anthelmintic drug<br>could suitable clinical evaluation development new antiviral drug<br>treat infection adenovirus immunosuppressed patient immunocompetent individual community<br>acquire pneumonia<br>",
          "Toxic proteins plant Plants evolve synthesize variety noxious<br>compound cope unfavorable circumstance among large group toxic<br>proteins play critical role plant defense predator microbes<br>wide range harmful protein discover different plant include<br>lectin ribosome inactivating protein protease inhibitor ureases arcelins<br>antimicrobial peptide pore form toxin fulfill role plant<br>defense proteins exhibit various degree toxicity towards animal<br>insect bacteria fungi Numerous study carry investigate toxic<br>effect mode action plant proteins order explore possible<br>application Indeed biological activity toxic plant protein also<br>consider potentially useful tool crop protection biomedical application<br>cancer treatment Genes encode toxic plant protein introduce<br>crop genome use genetic engineering technology order increase<br>plant resistance pathogen diseases Despite availability ample information<br>toxic plant proteins publication attempt summarize research progress<br>make last decade This review focus diversity toxic<br>plant proteins view toxicity well mode action Furthermore<br>outlook towards biological role proteins potential application discuss<br>",
          "Inhibition SARS CoV 3CL protease flavonoid There severe<br>panic cause Severe Acute Respiratory Syndrome SARS Middle<br>East Respiratory Syndrome Coronavirus Therefore research target virus<br>require Coronaviruses CoVs rise target flavonoids The antiviral<br>activity flavonoid CoVs presume directly cause inhibit like<br>protease 3CLpro Here apply flavonoid library systematically probe<br>inhibitory compound SARS CoV 3CLpro Herbacetin rhoifolin pectolinarin<br>find efficiently block enzymatic activity SARS CoV 3CLpro<br>The interaction three flavonoid confirm use tryptophan base<br>fluorescence method induced fit dock analysis indicate S3′<br>sit involve bind flavonoid The comparison previous study<br>show Triton play critical role object false positive<br>overestimated inhibitory activity flavonoid With systematic analysis three<br>flavonoid suggest template design functionally improve inhibitor<br>",
          "Proteome analysis vaccinia virus IHD infect HEK cell<br>dimensional gel electrophoresis MALDI PSD TOF solid phase<br>support terminally sulfonated peptide BACKGROUND Despite successful eradication<br>smallpox WHO lead vaccination programme pox virus infection<br>remain considerable health threat The possible use smallpox<br>bioterrorism agent well continuous occurrence zoonotic pox virus<br>infection document relevance deepen understand virus host interaction<br>Since permissiveness pox infection independent host surface receptor<br>correlate ability virus infiltrate antiviral host response directly<br>depend host proteome set report proteome HEK293 cell<br>infect Vaccinia Virus strain IHD analyze dimensional gel<br>electrophoresis MALDI PSD TOF bottom approach RESULTS The<br>cellular viral proteome VACV IHD infect HEK293 cell<br>inactivate VACV IHD treat well non infect cell<br>compare Derivatization peptides sulfophenyl isothiocyanate SPITC carry ZipTipμ<br>C18 column enable protein identification via peptide primary<br>sequence provide improved ratio well signal intensity PSD<br>spectra The expression human protein modulate viral infection<br>Effects inactivate infectious virus host proteome concern energy<br>metabolism protein associate gene expression protein biosynthesis quite<br>similar These effect might therefore attribute virus entry<br>virion proteins However modulation protein involve apoptosis clearly<br>correlate infectious virus CONCLUSIONS The proteome analysis infected<br>cell provide insight apoptosis modulation regulation cellular gene<br>expression regulation energy metabolism The confidence protein identification<br>clearly improve peptide derivatization SPITC solid phase support<br>Some identified protein describe context poxvirus infection need<br>characterise identify mean apoptosis modulation pathogenesis<br>",
          "Recent Reports Solid Phase Cyclohexapeptide Synthesis Applications Macrocyclic<br>peptides privileged scaffold drug development constitute significant portion<br>macrocyclic drug market today field span infectious disease<br>oncology Developing orally bioavailable peptide base drug remain<br>challenge task however macrocyclization linear peptide effective strategy<br>improve membrane permeability proteolytic stability oral bioavailability overall<br>drug like characteristic class Significant advance solid phase<br>peptide synthesis SPPS enable efficient construction macrocyclic peptide<br>peptidomimetic library macrolactamization perform resin solution phase The<br>primary goal review summarize solid phase cyclohexapeptide synthesis<br>use resin solution phase macrocyclization methodology publish since<br>also highlight broad application range natural product total<br>synthesis synthetic methodology development medicinal chemistry drug development<br>analyse conformational physiochemical property<br>",
          "Influence supercritical water treatment heavy metal medical waste<br>incinerator fly ash work medical waste incinerator fly<br>ash different type incinerator subject supercritical water SCW<br>SCW H2O2 SCWH treatments Sequential extraction experiment show<br>SCW treatment heavy metal exchangeable carbonate form ash<br>could transfer relatively stable form residual fraction organic<br>matter fraction SCWH treatment could stabilize heavy metal<br>Fe–Mn oxides residual fraction However behavior quite different<br>heavy metal could leach residue fraction SCW SWCH<br>treatments The leached tended absorb onto Fe–Mn oxides<br>organic matter near neutral environment could react Ca2<br>low increase mobility element Therefore necessary neutralize acidic<br>ash near neutral condition subject SCW SCWH treatment<br>effectively stabilize hazardous element ash Consequently believe SCWH<br>treatment effective alternative hazardous element detoxification fly ash<br>",
          "High Throughput Screening Small Molecule Enhancers Interferon Signaling<br>Pathway Drive Next Generation Antiviral Drug Discovery Most<br>current strategy antiviral therapeutic target virus specifically directly<br>alternative approach drug discovery might enhance immune response<br>broad range virus Based clinical observation human successful<br>genetic strategy experimental model reason improved interferon IFN<br>signal system might better protect viral infection Here<br>aim identify small molecular weight compound might mimic<br>beneficial effect improve antiviral defense Accordingly develop cell<br>base high throughput screen HTS assay identify small<br>molecule enhance IFN signal pathway component The assay<br>base phenotypic screen increase IFN stimulate response element<br>ISRE activity fully automate robust format Application assay<br>system library compound include already approve approvable drug<br>lead identification compound significant ISRE activity From choose<br>anthracycline antibiotic idarubicin validation mechanism base activity sub<br>range find idarubicin action increase ISRE activity manifest<br>member drug class independent cytotoxic topoisomerase inhibitory effect<br>well endogenous IFN signal production also observe compound<br>confer consequent increase IFN stimulate gene ISG expression<br>significant antiviral effect use similar dose range cell<br>culture system inoculate encephalomyocarditis virus EMCV The antiviral<br>effect also find compound concentration one observe cytotoxicity<br>Taken together result provide proof concept use activator<br>component IFN signal pathway improve IFN efficacy antiviral<br>immune defense well validated HTS approach identify small<br>molecule might achieve therapeutic benefit<br>",
          "Evaluation polyphenols Broussonetia papyrifera coronavirus protease inhibitors The<br>current study design assess inhibitory activity Broussonetia papyrifera<br>derive polyphenols chymotrypsin like papain like coronavirus cysteine<br>proteases The isolated compound broussochalcone broussochalcone hydroxyisolonchocarpin papyriflavonol<br>methylbut enyl trihydroxyflavane kazinol kazinol broussoflavan kazinol kazinol<br>All polyphenols potent papain like protease pro chymotripsin<br>like protease 3CL pro therefore investigated structural feature<br>responsible selectivity Compound potent inhibitor pro value The<br>active compound display kinetic behavior bind constant interaction<br>pro determine surface plasmon resonance analysis Our result<br>suggest papyrifera constituent promise candidate development potential anti<br>coronaviral agent<br>",
          "Broad spectrum antiviral activity carbodine carbocyclic analogue cytidine<br>Carbocyclic cytidine Cyd broad spectrum antiviral agent active<br>DNA virus pox vaccinia RNA virus toga Sindbis<br>Semliki forest corona RNA viruses orthomyxo influenza paramyxo<br>parainfluenza measles rhabdo vesicular stomatitis RNA viruses reo<br>The target enzyme Cyd suppose CTP synthetase convert<br>UTP CTP keep assumption observation Cyd effect dose<br>dependent inhibition RNA synthesis virus infect uninfected cell<br>exogenous addition either Urd Cyd reverse antiviral cytocidal<br>activity Cyd whereas addition dThd dCyd fail The<br>selectivity Cyd Sindbis vesicular stomatitis reo virus markedly<br>increase Cyd combine Cyd This combination may therefore<br>worth pursue chemotherapeutic modality treatment virus infection<br>",
          "Structure base virtual screening SARS 3CLpro identify novel<br>non peptidic hit Severe acute respiratory syndrome highly<br>infectious upper respiratory tract disease cause SARS CoV<br>previously unidentified human coronavirus SARS 3CLpro viral cysteine<br>protease critical pathogen life cycle hence therapeutic target<br>importance The recently elucidate crystal structure enzyme provide<br>opportunity discovery inhibitor rational drug design current study<br>Gold dock program utilized conduct extensive docking study<br>target crystal structure develop robust predictive docking protocol<br>The validated docking protocol use conduct structure base<br>virtual screening Asinex Platinum collection Biological evaluation screen<br>selection compound carry identify novel inhibitor viral protease<br>",
          "Computational Docking Study Ion Channel HCV Genotype Genotype<br>Its Interaction Natural Compounds BACKGROUND The current standard<br>care therapy hepatitis virus HCV infection consist two<br>regime namely interferon base interferon free treatment The<br>treatment combination ribavirin pegylated interferon expensive mildly effective<br>associate severe side effect two direct act antiviral<br>DAA drug boceprevir telaprevir license show enhanced sustain<br>virologic response SVR phase III clinical trial however<br>interferon free treatment sensitive HCV genotype infection The<br>variable nature HCV limit number inhibitor develop thus<br>aim expand repertoire available drug target result target<br>virus assembly therapeutically AIM conduct study predict structure<br>protein HCV genotypes Approximately amino acid residue encode<br>HCV render channel sensitive inhibitor make promising target<br>novel therapy HCV protein form small membrane know<br>viroporin essential effective self assembly large channel conduct<br>cation assembly discharge infectious virion particle METHOD study<br>screen drug flavonoid know disrupt translation production HCV<br>proteins target active site residue HCV genotype GT3<br>isolatek3a HCV genotype GT4 isolateED43 Furthermore conduct quantitative<br>structure–activity relationship dock interaction study RESULTS The drug<br>DNJ form high number hydrogen bond interaction model<br>protein high interaction energy follow BIT225 flavonoid screen<br>demonstrate Epigallocatechin gallate EGCG nobiletin quercetin bind mode<br>GT3 GT4 Thus predict protein molecule HCV GT3<br>GT4 provide general avenue target structure base antiviral<br>compound CONCLUSIONS hypothesize inhibitor viral identify screen may<br>new class potent agent confirmation vitro vivo essential<br>This structure guide drug design GT3 GT4 lead<br>identification drug like natural compound confirm new target<br>rapidly increase era HCV<br>",
          "efficient method recovery target ssDNA base amino modify<br>silica coat magnetic nanoparticles paper improve recovery method<br>target ssDNA use amino modify silica coat magnetic<br>nanoparticles ASMNPs report This method take advantage amino<br>modify silica coat magnetic nanoparticles prepare use water<br>oil microemulsion technique employ amino modify silica shell<br>iron oxide core magnetic nanoparticles The nanoparticles silica<br>surface amino group conjugate desired bio molecule many<br>exist amino group chemistry research linear DNA probe<br>immobilize onto nanoparticles streptavidin conjugation use covalent bond<br>target ssDNA TMR CGCATAGGGCCTCGTGATAC successfully recovered crude sample<br>magnet field special recognition hybridization design ssDNA fragment<br>severe acute respiratory syndrome SARS virus much low<br>concentration target ssDNA also recover high efficiency good<br>selectivity<br>",
          "Phosphonylated Acyclic Guanosine Analogues Triazole Linker novel series<br>amino chloro purin methyl triazol alkylphosphonates amino oxo<br>dihydro purin methyl triazol alkylphosphonates acyclic analogue guanosine<br>synthesize assessed antiviral activity broad range DNA RNA<br>viruses cytostatic activity toward three cancerous cell line<br>HeLa L1210 CEM They devoid antiviral activity however<br>several phosphonates find slightly cytostatic HeLa cell 80–210<br>range Compounds 17k 17k show high inhibitory effect<br>15–30 proliferation murine leukemia L1210 human lymphocyte CEM<br>cell line<br>",
          "Production anti Trichophyton rubrum egg yolk immunoglobulin therapeutic<br>potential treating dermatophytosis The aim study estimate therapeutic<br>potential specific egg yolk immunoglobulin IgY dermatophytosis cause<br>Trichophyton rubrum The IgY produce immunize hen cell<br>wall protein rubrum extract egg PEG precipitation purify<br>ammonium sulfate precipitation The cross reactivity fungi growth<br>inhibition rubrum vitro therapeutic effect rubrum infection BALB<br>mouse specific IgY evaluate Anti rubrum cell wall<br>proteins IgY anti trCWP IgY present certain degree<br>cross reactivity different fungi vitro vivo activity research<br>Anti trCWP IgY show significant dose dependent growth<br>inhibitory effect rubrum vitro significant dose dependent therapeutic<br>effect rubrum infection BALB mouse<br>",
          "The phenylalanine ammonia lyase gene family Isatis indigotica<br>Fort molecular clone characterization expression analysis Phenolic compound<br>metabolite phenylpropanoid pathway play important role growth environmental<br>adaptation many plant Phenylalanine ammonia lyase PAL first<br>key enzyme phenylpropanoid pathway The present study design<br>investigate whether multi gene family Indigotic characterize property<br>conduct comprehensive survey transcription profile database use tBLASTn<br>analysis Several bioinformatics method employ perform prediction composition<br>physicochemical character The expression level IiPAL gene various<br>tissue indigotica stress treatment examine quantitative real time<br>PCR Protoplast transient transformation use observe location IiPALs<br>IiPALs functionally characterize expression pET 32a vector Escherichia<br>colis strain BL21 DE3 Integration transcripts metabolite accumulation<br>use reveal relation IiPALs target compound new gene<br>IiPAL2 identify IiPALs conserve enzymatic active site Ala<br>Ser Gly classify member dicotyledon IiPAL1 IiPAL2 express<br>root stem leave flower high expression level IiPAL1<br>IiPAL2 observe stem root respectively The two gene<br>respond exogenous elicitor different manner Subcellular localization experiment<br>show IiPALs localize cytosol The recombinant protein show<br>catalyze conversion Phe trans cinnamic acid Correlation analysis<br>indicate IiPAL1 close biosynthesis secondary metabolite IiPAL2 conclusion<br>present study provide basis elucidation role IiPALs gene<br>biosynthesis phenolic compound help metabolic engineering improve accumulation<br>bioactive component indigotica<br>",
          "Biguanide modifiable pharmacophore recruitment endogenous inhibit cysteinyl cathepsins<br>review implication ABSTRACT Excessive activity cysteinyl cathepsins CysCts<br>contribute progress many disease however therapeutic inhibition problematic<br>natural inhibitor proteases CysHis dyads CysHis Xaa triad<br>Biguanide form bidentate metal complex two imino nitrogen<br>Here discuss phenformin phenylethyl biguanide model recruitment endogenous<br>inhibit CysHis CysHis peptidolysis Phenformin interactive anti proteolytic<br>agent bioassay live tissue Benzoyl arginine amide BAA<br>classical substrate papain like protease amide bond scissile<br>review structure BAA phenformin complex compare silico Their<br>chemistry dimension discuss light active site papain like<br>protease The phenyl moiety structure bind substrate bind<br>site typical many protease When phenyl moiety BAA<br>bind scissile amide bond directed position thiolate imidazolium<br>ion pair hydrolyze However phenyl moiety phenformin bind<br>coordinate direct identical position catalysis inhibit Phenformin stabilize<br>sandwich drug protease active site Hundreds biguanide derivative<br>synthesize nitrogen position many conceivable Various substituent moiety<br>register various array substrate bind site align coordinate<br>catalytic partner diverse protease Biguanide identify modifiable pharmacophore<br>synthesis therapeutic CysCt inhibitor wide range potency specificities<br>GRAPHICAL ABSTRACT Phenformin Complex Image see text<br>",
          "Structural characterization immunomodulatory effect polysaccharide HCP Houttuynia cordata<br>Immunomodulation natural polysaccharide hot topic research recent year<br>order explore immunomodulatory effect Houttuynia cordata Thunb water<br>extract study polysaccharide HCP molecular weight 000Da isolated<br>chromatography use DEAE Sepharose Sephacryl columns The structure<br>characterization HCP perform Fourier transform infrared spectroscopy FTIR<br>acidic hydrolysis PMP derivation HPLC analysis nuclear magnetic<br>resonance spectrum NMR HCP elucidate pectic polysaccharide linear<br>chain link galacturonic acid residue part carboxyl group<br>methyl esterify part hydroxyl group acetylate The bioactivity<br>assay show HCP could increase secretion interleukin tumor<br>necrosis factor TNF macrophage inhibitory protein MIP macrophage<br>inhibitory protein MIP RANTES regulate activation normal cell<br>express secreted human peripheral blood mononuclear cell PBMCs<br>play critical role innate immune system shape adaptive<br>immunity Our result imply HCP could immune enhancer<br>",
          "Simple Platform Rapid Development Antimicrobials Recent infectious outbreak<br>highlight need platform technology quickly deploy develop therapeutic<br>need contain outbreak present simple concept rapid development<br>new antimicrobial The goal produce little one week<br>thousand dose intervention new pathogen test feasibility system<br>base antimicrobial synbodies The system involve create array<br>peptide select broad capability bind kill viruses bacteria<br>The peptide pre screen low cell toxicity prior<br>large scale synthesis Any pathogen assay chip find<br>peptides bind kill Peptides combine pairs synbodies screen<br>activity toxicity The lead synbody quickly produce large<br>scale completion entire process one week<br>",
          "Effects Non Conservative Sequence Properties glucuronidase Aspergillus terreus<br>clone glucuronidase gene AtGUS Aspergillus terreus encode amino<br>acid transform glycyrrhizin glycyrrhetinic acid monoglucuronide GAMG glycyrrhetinic<br>acid Based sequence alignment terminal non conservative sequence<br>show low identity specie thus partial sequence AtGUS<br>1–592 amplify determine effect non conservative sequence enzymatic<br>property AtGUS AtGUS express coli BL21 produce AtGUS<br>AtGUS respectively similar optimum temperature 55°C AtGUS AtGUS<br>condition thermal stability AtGUS enhance 65°C metal ion<br>show opposite effect AtGUS AtGUS respectively Furthermore AtGUS<br>nearly one seventh AtGUS whereas catalytic efficiency AtGUS<br>fold high AtGUS reveal truncation non conservative sequence<br>significantly improve catalytic efficiency AtGUS Conformational analysis illustrate<br>significant difference secondary structure AtGUS AtGUS circular dichroism<br>The result show truncation non conservative sequence could<br>preferably alter influence stability catalytic efficiency enzyme<br>",
          "Graphene graphene oxide biofunctionalization application biotechnology Graphene basic<br>building block fullerene carbon nanotube graphite Graphene unique<br>planar structure well novel electronic property attract great<br>interest scientist This review selectively analyzes current advance<br>field graphene bioapplications particular biofunctionalization graphene biological application<br>fluorescence resonance energy transfer base biosensor development use<br>graphene graphene base nanomaterials investigation graphene graphene base<br>nanomaterials live cell study summarize detail Future perspectives<br>possible challenge rapidly develop area also discuss<br>",
          "Lobeglitazone Novel Thiazolidinedione Improves Non Alcoholic Fatty Liver<br>Disease Type Diabetes Its Efficacy Predictive Factors Related<br>Responsiveness Despite rapidly increase prevalence non alcoholic fatty<br>liver disease NAFLD type diabetes T2D treatment modality<br>currently available investigate hepatic effect novel thiazolidinedione TZDs<br>lobeglitazone Duvie T2D patient NAFLD recruit drug naïve<br>metformin treat T2D patient NAFLD conduct multicenter prospective<br>open label exploratory clinical trial Transient liver elastography<br>Fibroscan Echosens Paris France control attenuation parameter CAP<br>use non invasively quantify hepatic fat content Fifty<br>patient CAP value treat daily lobeglitazone week The<br>primary endpoint decline CAP value secondary endpoint include<br>change component glycemic lipid liver profile Lobeglitazone treat<br>patient show significantly decrease CAP value baseline week<br>regardless glycemic control Lobeglitazone improve HbA value well<br>lipid liver profile treat patient Moreover multivariable linear<br>regression analysis show hepatic fat reduction lobeglitazone independently<br>associate baseline value CAP liver stiffness liver enzyme<br>metformin use Lobeglitazone treatment reduce intrahepatic fat content<br>assess transient liver elastography improve glycemic liver lipid<br>profile T2D patient NAFLD Further randomize control trial<br>use liver histology end point necessary evaluate efficacy<br>lobeglitazone NAFLD treatment Clinical trial NCT02285205<br>",
          "Severe Acute Respiratory Syndrome Temporal Stability Geographic Variation<br>Death Rates Doubling Times analyze temporal stability geographic<br>trend cumulative case fatality rate average double time<br>severe acute respiratory syndrome SARS part account correlation<br>case fatality rate double time difference control measure<br>Factors may alter future estimate case fatality rate<br>reason heterogeneity double time among country implication control<br>SARS discuss<br>",
          "Prebiotic probiotic supplementation prevents rhinovirus infection preterm infant<br>randomize placebo control trial Background Simple safe strategy<br>prevention viral respiratory tract infection RTIs need Objective<br>hypothesize early prebiotic probiotic supplementation would reduce risk<br>virus associate RTIs first year life cohort preterm<br>infant Methods randomized double blind placebo control trial<br>ClinicalTrials gov NCT00167700 preterm infant gestational age ≥32<br>≤36 week birth weight treat Turku University Hospital<br>Turku Finland allocate receive oral prebiotics galacto oligosaccharide<br>polydextrose mixture probiotic Lactobacillus rhamnosus ATCC placebo microcrystalline<br>cellulose day life The primary outcome incidence clinically<br>define virus associate RTI episode confirm nasal swab<br>use nucleic acid test Secondary outcome severity duration<br>RTIs Results significantly low incidence RTIs detect infant<br>receive prebiotics rate ratio probiotic compare receiving placebo<br>Also incidence rhinovirus induce episode comprise RTI episode<br>find significantly low prebiotic probiotic group compare placebo<br>group difference emerge among study group rhinovirus RNA<br>load infection duration rhinovirus RNA shed duration severity<br>rhinovirus infection occurrence rhinovirus RNA asymptomatic infant Conclusions<br>Gut microbiota modification specific prebiotics probiotic might offer<br>novel cost effective mean reduce risk rhinovirus infection<br>",
          "Non invasive ventilation acute respiratory failure Summary Non<br>invasive mechanical ventilation increasingly use avoid serve alternative<br>intubation Compared medical therapy instance invasive mechanical ventilation<br>improve survival reduces complication select patient acute respiratory<br>failure The main indication exacerbation chronic obstructive pulmonary<br>disease cardiogenic pulmonary oedema pulmonary infiltrates immunocompromised patient<br>wean previously intubate stable patient chronic obstructive pulmonary<br>disease Furthermore technique use postoperative patient neurological disease<br>palliate symptom terminally ill patient help bronchoscopy however<br>study need situation regard first line treatment Non<br>invasive ventilation implement alternative intubation provide intensive care<br>high dependency unit When use prevent intubation otherwise<br>stable patient safely administer adequately staffed monitor ward<br>",
          "Nontraumatic Osteonecrosis Distal Tibia Case Presentation Review Literature<br>Osteonecrosis although commonly occur hip also affect leg<br>foot foot commonly occur talus The incidence osteonecrosis<br>occur tibia relatively rare report case woman present<br>clinic ankle pain idiopathic nature Subsequent magnetic resonance<br>imaging show finding consistent osteonecrosis bilateral distal tibias<br>several lesion locate shoulder hip calcaneus The present<br>report also serve review etiology treatment osteonecrosis relate<br>low extremity<br>",
          "Pharmacotherapy Lower Respiratory Tract Infections Elderly—Focused Antibiotics Lower<br>respiratory tract infection LRTIs refer inflammation trachea bronchus<br>bronchiole lung tissue Old people increase risk develop<br>LRTIs compare young adult The prevalence LRTIs elderly<br>population relate underlying disease age also variety clinical<br>issue history hospitalization previous antibacterial therapy mechanical ventilation<br>antibiotic resistance These factor mention led increase prevalence<br>mortality LRTIs elderly new medical strategy target LRTIs<br>population urgently need After systematic review current randomize<br>controlled trial relate study recommend novel pharmacotherapies demonstrate<br>advantage management LRTIs people age also briefly review<br>current medication respiratory communicable disease elderly Various source<br>information use ensure relevant study include search Pubmed<br>MEDLINE OvidSP EMBASE OvidSP ClinicalTrials gov Strengths limitation<br>drug evaluate base whether novelty mechanism favorable pharmacokinetic<br>pharmacodynamic profile avoidance interaction intolerance simplicity dose ability<br>cope challenge mainly evaluated primary secondary endpoint The<br>purpose review recommend promise antibiotic treatment LRTIs elderly<br>hospital outpatient set base exist result clinical study<br>novel antibiotic briefly review current medication respiratory communicable<br>disease elderly aim well management LRTIs clinical practice<br>",
          "Influenza vaccination respiratory virus interference among Department Defense<br>personnel 2017–2018 influenza season Purpose Receiving influenza vaccination<br>may increase risk respiratory virus phenomenon know virus<br>interference Test negative study design often utilized calculate<br>influenza vaccine effectiveness The virus interference phenomenon basic<br>assumption test negative vaccine effectiveness study vaccination change<br>risk infection respiratory illness thus potentially bias vaccine<br>effectiveness result positive direction This study aim investigate<br>virus interference compare respiratory virus status among Department<br>Defense personnel base influenza vaccination status Furthermore individual<br>respiratory virus association influenza vaccination examine Results compare<br>vaccination status people non influenza respiratory virus people<br>pan negative result Comparing vaccinated non vaccinated patient<br>adjust odds ratio non flu viruses confidence interval<br>Additionally vaccination status case influenza compare three different<br>control group control non influenza positive control pan<br>negative control The adjust ORs comparison among three<br>control group vary much range 46–0 Conclusions Receipt<br>influenza vaccination associate virus interference among population Examining<br>virus interference specific respiratory virus show mixed result<br>Vaccine derive virus interference significantly associate coronavirus human<br>metapneumovirus however significant protection vaccination associate influenza virus<br>also parainfluenza RSV non influenza virus coinfections<br>",
          "Single detection human bocavirus high viral load severe<br>respiratory tract infection previously healthy child BACKGROUND Human<br>bocavirus newly discover parvovirus Multiple study confirm presence<br>human bocavirus1 HBoV1 respiratory tract sample child The<br>viral load presentation single detection role causative agent<br>severe respiratory tract infection thoroughly elucidate METHODS investigate<br>presence HBoV1 quantitative polymerase chain reaction PCR nasopharyngeal<br>aspirate specimen child hospitalize respiratory tract infections The<br>sample analyze respiratory virus PCR respiratory virus viral<br>culture RESULTS least one virus detect child two<br>virus detect child HBoV1 detect child case HBoV1<br>virus detect Seasonal variation observe high HBoV1 infection<br>rate summer cutoff value copy use distinguish high<br>low HBoV1 viral load nasopharyngeal aspirate High viral<br>load HBoV1 note predominantly absence viral agent whereas<br>primarily detection case low HBoV1 viral load There<br>difference clinical symptom severity HBoV1 single detection detection<br>case HBoV1 single detection high viral load group<br>prevalent among child dyspnea wheeze low viral load<br>group clinical severity significant difference record high viral<br>load low viral load group HBoV1 positive patient<br>associate severe respiratory tract infection patient belong HBoV1<br>high viral load group patient case HBoV1 single<br>detection CONCLUSIONS HBoV1 high viral load frequently find<br>detection respiratory virus single detection high viral load<br>could etiological agent severe respiratory tract infection ELECTRONIC<br>SUPPLEMENTARY MATERIAL The online version article doi contains<br>supplementary material available authorized user<br>",
          "cluster randomize clinical trial comparing fit‐tested non‐fit‐tested N95<br>respirator medical mask prevent respiratory virus infection health<br>care worker Please cite paper MacIntyre cluster randomize<br>clinical trial comparing fit‐tested non‐fit‐tested N95 respirator medical<br>mask prevent respiratory virus infection health care worker<br>Influenza Other Respiratory Viruses DOI 1750‐2659 Background compare<br>efficacy medical mask N95 respirator fit test non<br>fit test health care worker HCWs Methods cluster<br>randomize clinical trial RCT HCWs Beijing hospital perform<br>winter Participants wear masks respirator entire work shift<br>week Outcomes include clinical respiratory illness CRI influenza‐like<br>illness ILI laboratory‐confirmed respiratory virus infection influenza convenience<br>no‐mask respirator group health worker nine hospital compare<br>Findings The rate CRI 3·9 versus 6·7 ILI<br>0·3 versus 0·6 laboratory‐confirmed respiratory virus 1·4 versus<br>2·6 influenza 0·3 versus infection consistently low N95<br>group compare medical mask intention‐to‐treat analysis value adjust<br>cluster non‐fit‐tested N95 respirator significantly protective medical mask<br>CRI outcomes significant The rate outcomes high convenience<br>no‐mask group compare intervention arm There significant difference<br>outcome N95 arm without fit test Rates fit<br>test failure low post hoc analysis adjust potential<br>confounders N95 mask hospital level significant medical mask<br>vaccination handwash high‐risk procedure Interpretation Rates infection medical<br>mask group double N95 group benefit respirator suggest<br>would need confirm large trial study may underpowered<br>The finding fit test specific type respirator use<br>study can not generalize respirator Trial registration Australian<br>New Zealand Clinical Trials Registry ANZCTR ACTRN ACTRN12609000257268<br>http www anzctr org<br>",
          "Multiplex PCR reveal virus frequent bacteria child cystic<br>fibrosis Background Cystic fibrosis degenerative disease characterize progressive<br>epithelial secretory gland dysfunction associate repeat respiratory infection<br>Bacterial infection frequent child cystic fibrosis rapid Methods<br>screen wide variety potentially involved virus unavailable recently<br>frequency viral presence unknown Multiplex PCR enable screen<br>many virus involve respiratory infection Objectives This study<br>aim evaluate frequency virus bacteria respiratory specimen child<br>cystic fibrosis clarify incidence characteristic seasonality age patient<br>different virus detect child cystic fibrosis Study design<br>year prospective study obtain paired nasopharyngeal swab sputum<br>specimens child cystic fibrosis clinical respiratory examination separate<br>least 14days analyze virus nasopharyngeal swab specimen multiplex<br>PCR bacteria sputum standard method Results analyze paired<br>specimen child detect virus bacteria Bacteria commoner spring<br>summer virus commoner autumn winter every season Staphylococcus<br>aureus common bacteria rhinovirus common virus Nearly infection<br>Haemophilus influenzae occur autumn winter Viruses prevalent child<br>year old bacteria prevalent child ≥12 year old<br>Conclusions Multiplex PCR screen respiratory virus feasible child<br>cystic fibrosis clinical implication screen warrant study<br>",
          "High Rate Viral Identification Coinfections Infants Acute Bronchiolitis<br>OBJECTIVES determine virus risk factor associate hospital intensive<br>care unit ICU admission infant acute bronchiolitis INTRODUCTION<br>Bronchiolitis major cause morbidity infant Widespread use molecular<br>base method yield new insight etiology impact viral<br>etiology early outcome still unclear METHODS Seventy seven<br>infant bronchiolitis two year age visit emergency unit<br>include Using molecular base method sample test different<br>respiratory virus Logistic regression model use identify clinical<br>virological variable associate main endpoint hospital admission ICU<br>admission RESULTS identify least one virus patient coinfections<br>find nearly patient RSV common pathogen follow rhinovirus<br>Identification RSV associate increase risk hospital admission univariate<br>model Younger age enterovirus infection associate increase risk<br>hospital admission atopy first degree relative show protective<br>effect Prematurity associate increase risk admission ICU Coinfections<br>associate bad outcome CONCLUSIONS Molecular base method result<br>high rate viral identification change significant role RSV<br>acute bronchiolitis Younger age enterovirus infection risk factor<br>hospital admission prematurity appear significant risk factor admission<br>ICU acute viral bronchiolitis<br>",
          "Aspergillus meningitis rare clinical manifestation central nervous system<br>aspergillosis Case report review case Summary Objectives describe<br>pathogenesis clinical presentation cerebrospinal fluid finding outcome Aspergillus<br>meningitis meningoencephalitis arachnoiditis Methods case Aspergillus meningitis describe<br>comprehensive review English language literature conduct identify report<br>case Aspergillus meningitis describe January December Results Ninety<br>three case include one described herein Aspergillus meningitis<br>identify Fifty two individual without predispose factor know<br>cause immunosuppression Acute chronic meningitis diagnose patient meningoencephalitis<br>remain present spinal arachnoiditis ventriculitis Cerebrospinal fluid culture<br>Aspergillus spp positive case galactomannan antigen test Diagnosis<br>life achieve patient case fatality rate The overall<br>case fatality rate Conclusions Aspergillus meningitis may occur<br>immunocompetent immunocompromised patient run acute chronic course The<br>finding systematic review extend information life threaten infection<br>could assist physician achieve improved outcome<br>",
          "Co‐infections Respiratory Viruses Dogs Bacterial Pneumonia BACKGROUND Bacterial<br>pneumonia inflammation lower airway lung parenchyma secondary bacterial<br>infection The pathogenesis dog complex role canine respiratory<br>virus fully evaluate OBJECTIVES The aim study investigate<br>occurrence viral co‐infections dog ass demographic clinical variable<br>well disease severity associate viral co‐infections ANIMALS Twenty<br>household dog cause opportunistic bacteria dog chronic day<br>tracheobronchitis cause Bordetella bronchiseptica BBTB METHODS Prospective cross‐sectional<br>observational study Diagnosis confirm clinical laboratory finding diagnostic<br>image cytologic microbiologic analysis bronchoalveolar lavage transtracheal wash<br>fluid Canine parainfluenza virus CPIV canine adenovirus canine<br>herpes virus canine influenzavirus canine distemper virus canine<br>respiratory coronavirus CRCoV canine pneumovirus well bronchiseptica Mycoplasma<br>spp analyze respiratory sample use PCR assay RESULTS<br>CPIV detect CRCoV dog Respiratory virus detect dog<br>BBTB There significant difference clinical variable dog without<br>viral co‐infection CONCLUSION AND CLINICAL IMPORTANCE Respiratory viruses<br>find frequently dog may therefore play important role<br>etiology pathogenesis Clinical variable disease severity differ dog<br>without viral co‐infection<br>",
          "Changes pulmonary tuberculosis prevalence evidence population survey populous<br>province China BACKGROUND This paper report finding prevalence<br>survey conduct Shandong China province population million This<br>study aim estimate prevalence province comparison survey compare<br>yield case different case find approach METHODS population<br>base cross sectional survey conduct use multi stage<br>random cluster sample adult participate survey response rate<br>Doctors interview classified participant suspect case present persistent<br>cough abnormal chest ray CXRAY Three sputum specimen<br>suspect case collect sent smear microscopy culture RESULTS<br>Adjusted prevalence rate bacteriologically confirm case per adult<br>Shandong Compared survey prevalence decline bacteriologically confirm case<br>present persistent cough The yield bacteriologically confirm case<br>symptom screen CXRAY Over case among CONCLUSIONS The<br>prevalence rate bacteriologically confirm case significantly reduce compare<br>The survey raise challenge identify case without clear<br>symptom<br>",
          "Diagnostic performance multiplex PCR pulmonary sample versus nasopharyngeal<br>aspirate community acquire severe low respiratory tract infection<br>Background PCR base technique diagnosis community acquire severe<br>low respiratory tract infection become standard care However<br>relative ability identify either atypical bacteria virus cause<br>LRTI clinical sample various source yet determine Objectives<br>study design The aim study compare diagnostic yield<br>nasopharyngeal aspirate pulmonary sample etiological diagnosis severe acute<br>lower respiratory tract infection multiplex PCR Patients adult<br>community acquire pneumonia acute exacerbation chronic obstructive pulmonary<br>disease Results obtain concordant result pair sample discordant<br>result pathogen evidence low respiratory tract sample Conclusions<br>Pulmonary sample similar diagnostic sensitivity virus detection multiplex<br>PCR nasopharyngeal aspirate contrast study diagnostic efficacy pulmonary<br>sample Legionella pneumophila simple aspirate clearly superior<br>",
          "Safety Efficacy Profile Echinacea purpurea Prevent Common Cold<br>Episodes Randomized Double Blind Placebo Controlled Trial Objective<br>investigate safety risk efficacy benefit Echinacea purpurea extract<br>prevention common cold episode large population month period<br>Methods healthy subject allocate receive either alcohol extract<br>freshly harvest purpurea herba root placebo Participants require<br>record adverse event rate cold relate issue diary<br>throughout investigation period Nasal secretion sample acute cold<br>screen virus Results total adverse event occur Echinacea<br>placebo treatment Nine participant experienced adverse event least<br>possibly related study drug adverse drug reaction Thus<br>safety Echinacea noninferior placebo Echinacea reduce total number<br>cold episode cumulate episode day within group pain<br>killer medicate episode Echinacea inhibit virally confirm cold<br>especially prevent enveloped virus infection Echinacea show maximal<br>effect recurrent infection preventive effect increase therapy compliance<br>adherence protocol Conclusions Compliant prophylactic intake purpurea month<br>period appear provide positive risk benefit ratio<br>",
          "Pathological ultrastructural analysis surgical lung biopsy patient swine‐origin<br>influenza type H1N1 acute respiratory failure BACKGROUND Cases<br>H1N1 pulmonary infection evolve acute respiratory failure death<br>co‐infections lung injury predominate immune response thus require<br>early diagnosis improve treatment OBJECTIVE perform detailed histopathological<br>analysis open lung biopsy specimens five patient ARDS<br>confirm H1N1 METHODS Lung specimens underwent microbiologic analysis<br>examination optical electron microscopy Immunophenotyping use characterize macrophage<br>natural killer cell expression cytokine iNOS RESULTS The<br>pathological feature observe necrotizing bronchiolitis diffuse alveolar damage<br>alveolar hemorrhage abnormal immune response Ultrastructural analysis show<br>viral‐like particle case CONCLUSIONS Viral‐like particles successfully demonstrate<br>lung tissue ultrastructural examination without confirmation virus RT‐PCR<br>nasopharyngeal aspirate Bronchioles epithelium rather endothelium probably primary<br>target infection diffuse alveolar damage consequence effect airways<br>obliteration dysfunction innate immunity suggest treatment focus epithelial<br>repair<br>",
          "Prevalence non influenza respiratory virus acute respiratory infection<br>case Mexico BACKGROUND Acute respiratory infection lead cause<br>morbidity mortality worldwide Although viral aetiological agent estimate<br>involved case majority agent never specifically identify Since<br>diagnostic surveillance effort influenza virus apply worldwide However<br>insufficient epidemiological information available many respiratory virus cause<br>Acute respiratory infection METHODS This study evaluate presence<br>non influenza respiratory virus pharyngeal exudate sample use<br>qPCR All sample meet operational definition probable case<br>influenza like illness severe acute respiratory infection previous<br>negative result influenza qPCR RESULTS The presence least<br>one non influenza virus observe sample The frequent<br>virus rhinovirus human respiratory syncytial virus HRSV human<br>metapneumovirus HMPV total case infection cause virus identify<br>Approximately positive case child year age CONCLUSION study<br>identify non influenza respiratory virus could occur season<br>year This study provide evidence prevalence seasonality wide<br>range respiratory virus circulate Mexico constitute risk population<br>Additionally data suggest include test widely diagnostic algorithm<br>influenza may reduce use unnecessary antibiotic reduce hospitalisation<br>time enrich national epidemiological data respect infection cause<br>virus<br>",
          "Respiratory Viruses Respiratory viruses frequent causative agent disease<br>human significant impact morbidity mortality worldwide Common respiratory<br>agent several virus family well adapt efficient person<br>person transmission circulate global scale community base study<br>conduct past five decade confirm virus predominant etiological<br>agent acute respiratory infection ARIs The respiratory viruses<br>commonly circulate continent endemic epidemic agent influenza virus<br>respiratory syncytial virus parainfluenza virus metapneumovirus rhinovirus coronaviruses<br>adenoviruses bocaviruses Although vaccine effective antiviral drug yet<br>available virus much progress make understanding biology fundamental<br>issue host–parasite relationship This article summary current knowledge<br>virus<br>",
          "Asymptomatic Middle East Respiratory Syndrome coronavirus infection use<br>serologic survey Korea OBJECTIVES The rate asymptomatic infection<br>Middle East Respiratory Syndrome MERS coronavirus vary serologic<br>study conduct determine asymptomatic MERS infection rate healthcare<br>worker non healthcare worker exposure status METHODS Study<br>participant select contact MERS patient base priority system<br>region strongly affect MERS outbreak sero epidemiological survey<br>perform contact average duration exposure test month collect<br>serum test use enzyme link immunespecific assay ELISA<br>immunofluorescence assay IFA plaque reduction neutralization antibody test<br>PRNT Among contact ELISA positive case exhibit positive<br>IFA PRNT result RESULTS The asymptomatic infection rate<br>confidence interval The asymptomatic MERS case patient hospitalize<br>patient zero floor hospital time The case quarantine<br>home week discharge underlying disease include hypertension angina<br>degenerative arthritis CONCLUSIONS The asymptomatic infection acquire via<br>healthcare associate transmission Thus necessary extend serologic study<br>include inpatient contact symptom<br>",
          "simple compact smartphone base device quantitative readout colloidal<br>gold lateral flow immunoassay strip Colloidal gold lateral<br>flow immunoassay strip AuNPs LFIS widely apply qualitative<br>diagnostic tool point care test POCT strip reader<br>incorporate use could extend quantitative analysis However cost<br>non portability render commercial strip reader unavailable use<br>either home testing community rural hospital diagnosis This<br>particularly true site test Here smartphone base reader<br>design print quantitatively assess AuNPs LFIS The basic<br>principle devise rely smartphone ambient light sensor SPALS<br>This sensor harness measure transmit light intensity originate<br>line strip transmit light intensity vary concentration AuNP<br>line validate approach newly develop smartphone ambient light<br>sensor base reader SPALS reader use readout AuNPs<br>LFIS three analytical target cadmium ion Cd2 limit<br>detection LOD clenbuterol LOD porcine epidemic diarrhea virus<br>PEDV LOD The result show good consistency result<br>conventional image analysis approach indicate smartphone base device<br>appropriate use AuNPs LFIS readout Compared traditional analysis<br>method develop AuNPs LFIS reader easier operate low<br>cost portable provide site quantitative analysis tool AuNPs<br>LFIS enhance apply range AuNPs LFIS<br>",
          "Detection feline infectious peritonitis virus infection cell culture<br>peripheral blood mononuclear leukocytes experimentally infected cat use<br>biotinylated cDNA probe dot blot hybridization assay use<br>biotinylated cDNA probe able detect feline infectious peritonitis<br>virus FIPV RNA Felis catus whole fetus fcwf<br>cell infect FIPV isolate DF2 UCDI UCD2 The<br>probe cross hybridize dot blot assay nucleic acid<br>closely related feline coronavirus feline enteric coronavirus FECV<br>construct probe kilobase cDNA prepare FIPV DF2 genomic<br>RNA molecularly clone The recombinant cDNA clone digest<br>restriction endonuclease Rsa basepair Rsa fragment isolated vector<br>DNA agarose electrophoresis glassmilk purification This fragment complementary<br>three fourth nucleocapsid gene The hybridization probe prepared<br>random prim label presence biotin dUTP Using avidin<br>alkaline phosphatase conjugate chemiluminescent substrate detection system virus<br>could detect infected cell vivo study probe use<br>detect FIPV RNA peripheral blood mononuclear leukocytes PBML<br>isolate various post infection day PID cat experimentally<br>infect FIP produce coronavirus isolate FIPV FIPV DF2<br>Viral RNA could detect PBML isolate cat PID<br>PBML PID There consistent pattern however hybridization result<br>prognosis severity disease time sample Despite cross hybridization<br>FECV RNA probe useful diagnosis FIP cat infect<br>FECV likely become viremic<br>",
          "Human adenovirus nasopharyngeal blood sample child without respiratory<br>tract infection Background Human adenovirus HAdV double strand<br>DNA virus associate respiratory tract infection RTI child<br>Using polymerase chain reaction PCR test HAdV often<br>detect together virus specie even healthy control Objectives<br>The aim study compare molecular detection HAdV culture<br>examine association various method RTI Study design Nasopharyngeal<br>aspirate NPA collect child admit RTI control The<br>NPAs examine viral bacterial pathogen use inhouse real<br>time PCR assay base TaqMan probes addition bacterial<br>viral culture HAdV concentration evaluate semi quantitatively value<br>quantitatively use ADENOVIRUS gene® Results HAdV DNA detect<br>patient sample control Compared control patient 4–10 mono<br>detection HAdV DNA 0–13 HAdV positive sample grow<br>adenovirus culture HAdV DNA load child RTI consist<br>two cluster one cluster high viral load copy<br>one cluster low viral load whereas among control<br>nearly low viral load 2–27 available plasma sample<br>positive HAdV DNA patient Conclusion The detection HAdV<br>DNA per qualitative PCR useful diagnostic test Detection<br>HAdV use viral culture high viral HAdV DNA<br>load two method strongly associate RTI child<br>",
          "Application Broad Spectrum Sequence Based Pathogen Identification Urban<br>Population broad spectrum detection platform provide sequence level<br>resolution target region would significant impact public health<br>case management mean expand understand etiology diseases previously<br>develop respiratory pathogen microarray RPM demonstrated capability platform<br>purpose This newly develop RPM use analyze well<br>characterize nasal wash specimen patient present febrile respiratory<br>illness Washington metropolitan region For specimen RPM result<br>compare composite reference assay viral bacterial culture appropriate<br>PCR PCR result Across panel RPM assay show<br>≥98 overall agreement organism detect compare reference method<br>Additionally RPM result provide sequence information allow phylogenetic<br>classification circulate influenza virus ∼250 clinical specimen allow<br>monitoring genetic variation well antigenic variability prediction Multiple<br>pathogens 2–4 detect specimen notably increase abundance respiratory<br>colonizer esp pneumoniae viral infection This first ever<br>comparison broad spectrum viral bacterial identification technology type<br>large battery conventional gold standard assay confirms utility<br>approach medical surveillance investigation complex etiology illness cause<br>respiratory infection<br>",
          "Experimental Design Analysis Microarray Data The advent microarray<br>technology significantly change way quantitatively measure observe gene<br>expression mRNA level within give biological sample interest<br>allow monitor ten hundred thousand gene within single<br>experiment The two main array platform spot two<br>colour array one colour situ synthesize array Microarrays<br>use wide range application include gene annotation investigation<br>gene gene interaction elucidation gene regulatory network gene<br>expression profile Saccharomyces cerevisiae fungal organism Academic researcher<br>pharmaceutical agricultural industry enormous interest develop microarrays diagnostic<br>tool use basic research pathogens fungi interact host<br>Microarray experiment generate vast quantity raw gene expression<br>data therefore good experimental design statistical analysis require<br>extraction accurate useful information regard expression gene review<br>firstly provide overview arrival development microarray technology focus<br>issue surround experimental design processing microarray image follow<br>discussion method clean normalize raw gene expression data<br>final discussion importance statistical analysis play identify differentially<br>express gene<br>",
          "Development recombinase polymerase amplification assay lateral flow dipstick<br>rapid detection feline parvovirus Feline panleukopenia cause feline<br>parvovirus FPV single strand DNA virus typically highly<br>contagious often present lethal syndrome The broad spectrum<br>possible host suggest potential transmission animal person close<br>contact pet FPV thus serve example importance new<br>rapid point care field diagnostic tool control prevention<br>transmission especially among rare wild animal pet cat<br>Recombinase polymerase amplification RPA real time isothermal method<br>could affordable alternative PCR combine lateral flow dipstick<br>LFD indicator study report novel FPV lateral flow<br>dipstick RPA LFD RPA instant detection method capable<br>detect range different FPV strain The LFD RPA<br>assay consist specific primer probe nucleic acid strip<br>capable detect copy target nucleic acid per reaction<br>one order magnitude high sensitivity traditional PCR The<br>suitable reaction condition assay min This paper develop<br>efficient visual detection system eliminate need professional staff<br>expensive sophisticated equipment field detection<br>",
          "Visualizing Viral Infection Vivo Multi Photon Intravital Microscopy<br>Viral pathogen adapt host organism exploit cellular machinery<br>virus replication modulate host cell efficient systemic dissemination<br>immune evasion Much knowledge effect virus infection cell<br>originate vitro image study use experimental culture system<br>consist cell line primary cell Recently intravital microscopy<br>use multi photon excitation fluorophores apply observe virus<br>dissemination pathogenesis real time physiological condition live organisms<br>Critical step viral infection pathogenesis could study direct<br>visualization fluorescent virus particle virus infect cell immune<br>response viral infection review summarize late research vivo<br>study viral infection use multi photon intravital microscopy<br>IVM Initially underlie principle multi photon microscopy introduce<br>experimental challenge microsurgical animal preparation fluorescent label strategy<br>intravital image discuss highlight recent study combine IVM<br>optogenetic tool transcriptional analysis powerful approach extend significance<br>vivo image study viral pathogen<br>",
          "Deep Sequencing Identify Causes Viral Encephalitis Deep sequencing<br>allow rapid accurate characterization microbial DNA RNA sequence<br>many type sample Deep sequencing also call next<br>generation sequence NGS develop assist diagnosis wide variety<br>infectious disease study seven frozen brain sample decease<br>subject recent encephalitis investigate RNA sample extract randomly<br>reverse transcribed sequence The sequence analysis perform blinded<br>fashion confirm pathogen specific PCR This analysis successfully<br>identify measles virus sequence two brain sample herpes<br>simplex virus type sequence three brain sample pathogen<br>identify two brain specimen These result concordant pathogen<br>specific PCR partially concordant prior neuropathological examination demonstrate<br>deep sequencing accurately identify viral infection freeze brain<br>tissue<br>",
          "FilmArray® respiratory panel performance respiratory sample neonatal care<br>unit FilmArray Respiratory Panel Idaho Technology Inc Salt<br>Lake City USA performance retrospectively evaluate respiratory sample<br>collect neonate reference neonatology unit Using FilmArray assay<br>sample positive least respiratory virus negative FilmArray result<br>concordant respiratory sample test laboratory develop real time<br>PCR assay sample FilmArray assay detect additional virus<br>human rhinovirus enterovirus bocavirus addition FilmArray result respiratory<br>sample test concordant Seegene Seeplex RV15® detection assay<br>Seegene Inc Seoul South Korea discordant The advantage<br>FilmArray rapid detection respiratory viruses hour wide number<br>pathogen detectable single assay reduce hand time<br>",
          "Comparison two commercial molecular assay simultaneous detection respiratory<br>viruses clinical sample use two automatic electrophoresis detection<br>system Two molecular assay compare real time PCR<br>viral culture simultaneous detection common viruses respiratory sample<br>multiplex ligation dependant probe amplification MLPA dual priming<br>oligonucleotide system DPO addition positive detection MLPA DPO<br>identify use two different automatic electrophoresis system panel<br>culture positive negative sample test molecular assay presence<br>influenza virus respiratory syncytial virus human metapneumovirus rhinovirus<br>coronaviruses parainfluenza virus adenovirus One hundred twenty nine<br>sample positive detect least one method Sixty nine<br>sample positive cell culture exclude human metapneumovirus coronaviruses<br>PCR MLPA DPO The MLPA yield result one<br>attempt sample include sample repeat DPO assay due<br>inconclusive result The MLPA assay perform well combination<br>either electrophoresis system performance DPO assay influence electrophoresis<br>system Both molecular assay comparable real time PCR<br>sensitive viral culture detect dual infection easily Results<br>obtain within day<br>",
          "Comparisons substitution insertion deletion probe resequencing mutational analysis<br>use oligonucleotide microarrays Although oligonucleotide probe complementary single<br>nucleotide substitution commonly use microarray base screen genetic<br>variation little known hybridization property probe complementary small<br>insertion deletion necessary define hybridization property latter probe<br>order improve specificity sensitivity oligonucleotide microarray base mutational<br>analysis disease relate gene Here compare contrast hybridization<br>property oligonucleotide microarrays consist 25mer probe complementary possible<br>single nucleotide substitution insertion one two base deletion<br>cod region ATM ataxia telangiectasia mutate gene Over<br>different dye label single strand nucleic acid target<br>represent ATM cod exon apply microarrays assess hybridization<br>specificity compare relative hybridization signal probe perfectly match<br>ATM sequence contain mismatch Probes complementary two base<br>substitution display high average specificity follow complementary single<br>base substitution single base deletion single base insertion<br>case hybridization specificity strongly influence sequence context possible<br>intra intermolecular probe target structure Furthermore single nucleotide<br>substitution probe display consistent hybridization specificity data follow<br>single base deletion two base deletion single nucleotide<br>insertion Overall study provide valuable empirical data use<br>accurately model hybridization property insertion deletion probe improve<br>design interpretation oligonucleotide microarray base resequencing mutational analysis<br>",
          "Specific detection H5N1 avian influenza virus field specimens<br>one step PCR assay BACKGROUND Continuous outbreaks highly<br>pathogenic H5N1 avian influenza Asia result urgent effort<br>improve current diagnostics aid containment virus lower threat<br>influenza pandemic report development PCR base assay highly<br>specific H5N1 avian influenza virus METHODS one step<br>reverse transcription PCR assay develop detect H5N1 avian<br>influenza virus The specificity assay show test sub<br>type influenza virus viral bacterial pathogen field sample<br>RESULTS Validation field specimen Vietnam Malaysia show assay<br>specific without cross reactivity number infuenza strain well<br>human respiratory relate pathogen Detection allantoic fluid H5N1<br>positive sample suggest reliable sample source accurate detection<br>CONCLUSION The assay develop study indicate primer specific<br>H5N1 influenza virus show field test result assay<br>would highly useful diagnostic tool help identify control<br>influenza epidemic<br>",
          "Development nest PCR assay detection canine coronavirus diagnostic<br>test canine coronavirus CCV infection base nested polymerase<br>chain reaction PCR assay develop test use follow<br>coronavirus strain CCV USDA strain CCV field strain<br>feline infectious peritonitis virus FIPV field strain trasmissible<br>gastroenteritis virus TGEV Purdue strain bovine coronavirus BCV<br>9WBL strain infectious bronchitis virus IBV strain fecal<br>sample dog CCV enteritis segment gene encode transmembrane<br>protein CCV target sequence primer The test describe<br>present study able amplify CCV TGEV strain also<br>give positive result fecal sample CCV infect dog<br>PCR sensitivity high isolation cell culture therefore use<br>diagnosis CCV infection dog<br>",
          "Controlled Microwave Heating Accelerates Rolling Circle Amplification Rolling<br>circle amplification RCA generate single strand DNAs RNA<br>diverse application isothermal technique range sensitive detection nucleic<br>acid analysis single nucleotide polymorphism Microwave chemistry widely<br>apply increase reaction rate well product yield purity<br>The objective present research apply microwave heat RCA<br>indicate factor contribute microwave selective heating effect The<br>microwave reaction temperature strictly control use microwave applicator<br>optimize enzymatic scale reaction Here show microwave assist<br>RCA reaction catalyze either four thermostable DNA polymerase<br>accelerate fold compare conventional RCA Furthermore temperatures individual<br>buffer component specifically influence microwave heating conclude microwave<br>heat accelerate isothermal RCA DNA differential heating mechanism<br>microwave temperature reaction component although overall reaction temperature<br>",
          "Simple rapid detection human enterovirus reverse transcription loop<br>mediate isothermal amplification cryopreservation affect detection ability Human<br>enterovirus EV71 primary pathogen hand foot mouth disease<br>HFMD EV71 infection may lead neurologic damage high<br>incidence fatality compare HFMD pathogens effective drug vaccine<br>EV71 infection currently unavailable desirable determine pathogen HFMD<br>accurately quickly early treatment current study reverse transcription<br>loop mediate isothermal amplification LAMP technology develop detect<br>EV71 The efficacy detect EV71 compare regular nested<br>reverse transcription polymerase chain reaction PCR After detect<br>clinical specimen result show LAMP specifically detect EV71<br>Coxsackie virus A16 exhibit specificity sensitivity high regular<br>PCR The finding indicate LAMP practical method EV71<br>diagnostic application particularly small county institute medical service<br>The detection ability LAMP significantly affect cryopreservation clinical<br>specimen repeatedly subject freeze thawing treatment<br>",
          "multiplex PCR assay detection five influenza virus use<br>dual priming oligonucleotide system BACKGROUND cost effective accurate<br>rapid simultaneous multiplex assay require testing diagnose conventional<br>emerge virus clinical virology laboratories develop optimize dual<br>priming oligonucleotide DPO multiplex PCR assay detect influenza<br>virus include seasonal H1N1 pandemic H1N1 H3N2 influenza<br>H5N1 METHODS The optimized multiplex DPO PCR use<br>detect clinical human sample The result compare obtained<br>qPCR conventional PCR immunochromatographic assay RESULTS Specificity analysis<br>reveal DPO PCR assay amplify target virus without<br>cross amplification Statistical analysis demonstrate multiplex DPO PCR<br>sensitivity high immunochromatographic assay low qPCR significant difference<br>observe compare conventional PCR detect influenza Additional experiment<br>use sample panel indicate significant difference number positive<br>sample detect multiplex DPO PCR qPCR apply value<br>low CONCLUSIONS The five target simultaneous multiplex DPO<br>PCR assay could easily adopt routine practice This<br>approach cost effective short run time low technical<br>requirement detection influenza virus early diagnosis clinical laboratory<br>",
          "Microbial molecular marker epidemiological surveillance era high throughput<br>sequence update IMMEM conference<br>",
          "Comparison performance ITS1 ITS2 barcodes amplicon base sequencing<br>bioaerosols This paper present performance two eukaryotic genomic<br>ribosomal region ITS1 ITS2 describe fungal diversity aerosol<br>sample use amplicon base High Throughput Sequencing HTS<br>Composting sit biomethanization facility dairy farm affect presence<br>fungi visit collect air sample The amplicon base<br>HTS approach target enrichment method relies amplification specific<br>target use particular primer sequence Thus result highly<br>dependent quality amplification For reason author paper use<br>shotgun metagenomic approach compare outcome amplicon base method<br>Indeed shotgun metagenomic rely amplification prior sequence gene<br>sequence without specific target addition culture method add<br>analysis biomethanization dairy farm sample validate contribution fungal<br>diversity aerosols The result obtain unequivocal towards ITS1<br>outperformance ITS2 term richness taxonomic coverage The differential<br>abundance analysis demonstrate taxon exclusively detect ITS2 vice<br>versa ITS1 However shotgun metagenomic approach show taxonomic<br>profile resemble ITS1 ITS2 Based result neither barcodes<br>evaluate perfect term distinguish specie Using barcodes offer<br>broad view fungal aerosol population However actual knowledge<br>author strongly recommend use ITS1 universal fungal barcode<br>quick general analysis diversity limit financial resource available<br>primarily due ability capture taxonomic profile similar obtain<br>use shotgun metagenomic The culture comparison amplicon base<br>sequencing show complementarity approach describe abundant taxon<br>",
          "Nasopharyngeal Protein Biomarkers Acute Respiratory Virus Infection Infection<br>respiratory mucosa viral pathogens trigger complex immunologic event<br>affect host seek characterize response proteomic analysis nasopharyngeal<br>lavage human subject experimentally challenge influenza H3N2 human<br>rhinovirus develop target assay measure peptide involve host<br>response allow classification acute respiratory virus infection Unbiased<br>proteomic discovery analysis identify peptide correspond unique protein<br>reveal infection H3N2 induce significant alteration protein expression<br>These include protein involve acute inflammatory response innate<br>immune response complement cascade These data provide insight<br>nature biological response viral infection upper respiratory tract<br>protein dysregulated viral infection form basis signature accurately<br>classifies infect state Verification signature use targeted mass<br>spectrometry independent cohort subject challenge influenza rhinovirus demonstrate<br>performs high accuracy AUROC TPR TNR With development<br>clinical diagnostic signature may utility rapid screen emerge<br>infection avoidance inappropriate antibacterial therapy rapid implementation appropriate<br>therapeutic public health strategy<br>",
          "Identification characterization unrecognized virus stool sample non polio<br>acute flaccid paralysis child simplify VIDISCA BACKGROUND The<br>use sequence independent method combine next generation sequence<br>identification purpose clinical sample appear promise excite result<br>achieve understand unexplained infection One sequence independent method<br>Virus Discovery base cDNA Amplified Fragment Length Polymorphism<br>VIDISCA capable identifying virus would remain unidentified standard<br>diagnostics cell culture METHODS VIDISCA normally combine next<br>generation sequence however set simplify VIDISCA use case<br>next generation sequence possible Stool sample patient unexplained<br>acute flaccid paralysis show cytopathic effect rhabdomyosarcoma cell<br>mouse cell use test efficiency method characterize virus<br>VIDISCA positive sample amplify sequenced gene specific primer<br>RESULTS Simplified VIDISCA detect seven virus proportion eukaryotic<br>viral sequence sample range Human enterovirus B97 B100<br>echovirus echovirus human parechovirus type human astrovirus probably<br>type recombinant tetnovirus identify Phylogenetic analysis base VP1<br>region demonstrate human enterovirus divergent isolate circulate community<br>CONCLUSION Our data support simplify VIDISCA protocol efficiently<br>identify unrecognized virus grow cell culture low cost<br>limited time without need advance technical expertise Also<br>complex data interpretation avoid thus method use powerful<br>diagnostic tool limit resource Redesigning routine diagnostics might<br>lead additional detection previously undiagnosed virus clinical sample<br>patient ELECTRONIC SUPPLEMENTARY MATERIAL The online version article<br>doi 422X contains supplementary material available authorized user<br>",
          "Space Time Clustering Characteristics Tuberculosis China OBJECTIVES China<br>one tuberculosis high burden country world major public<br>health problem China spatial analysis could apply detect<br>geographic distribution cluster target intervention epidemic METHODS Spatial<br>analysis apply detect cluster county base notification data<br>national notifiable infectious disease case report surveillance system<br>Two indicator epidemic use include new sputum smear<br>positive notification rate total notification rate Global Moran<br>ArcGIS used ass whether cluster trend significant SaTScan<br>software use retrospective space time analysis Possion probability<br>model utilized identify geographic area time period potential<br>cluster notification rate county level RESULTS Two indicator<br>notification present significant spatial autocorrelation globally year Global<br>Moran total notification rate positive trend time The<br>likely cluster two indicator similar spatial distribution size<br>south central region China secondary cluster two region<br>northeastern China western China Besides secondary cluster total<br>notification rate two large cluster center Inner Mongolia<br>Gansu Qinghai provinces several small cluster Shanxi Henan<br>Hebei Jiangsu provinces CONCLUSION The total notification case<br>cluster significantly special area year cluster trend aggregate<br>time The likely secondary cluster overlap among two<br>indicator higher burden risk transmission These focus geographic<br>area control effort prioritize<br>",
          "iGPCR Drug Web Server Predicting Interaction GPCRs Drugs<br>Cellular Networking Involved many disease cancer diabetes neurodegenerative<br>inflammatory respiratory disorder protein couple receptor GPCRs among<br>frequent target therapeutic drug time consume expensive determine<br>whether drug GPCR interact cellular network purely mean<br>experimental technique Although computational method develop regard base<br>knowledge dimensional structure protein unfortunately usage quite limited<br>structure GPCRs still unknown overcome situation sequence base<br>classifier call iGPCR drug develop predict interaction GPCRs<br>drug cellular networking predictor drug compound formulate dimensional<br>fingerprint via 256D vector GPCR PseAAC pseudo amino<br>acid composition generate grey model theory prediction engine<br>operate fuzzy near neighbour algorithm Moreover user friendly<br>web server iGPCR drug establish http www jci<br>bioinfo iGPCR Drug For convenience experimental scientist step<br>step guide provide use web server get desire<br>result without need follow complicate math equation present<br>paper integrity The overall success rate achieve iGPCR<br>drug via jackknife test remarkably high rate exist<br>peer method develop although web server ever establish<br>anticipate iGPCR Drug may become useful high throughput<br>tool basic research drug development approach present also<br>extend study drug target interaction network<br>",
          "Yang Zhong explorer road forever<br>",
          "Cybernetics base Dynamic Infection Model Analyzing SARS COV<br>Infection Stability Predicting Uncontrollable Risks Since December COVID<br>rag Wuhan subsequently China world propose Cybernetics base<br>Dynamic Infection Model CDIM dynamic infection process probability<br>distribute incubation delay feedback principle Reproductive trend stability<br>SARS COV infection city analyze uncontrollable risk forecast<br>really happen The infection mechanism city depict use<br>philosophy cybernetics approach control engineering Distinguished epidemiological model<br>SIR SEIR etc compute theoretical number infect people<br>close population CDIM consider immigration emigration population system<br>input administrative medical resource dynamic control variables The<br>epidemic regulation simulate model support decision make contain<br>outbreak City case study demonstrate verification validation<br>",
          "Determinants Taiwanese tourist hotel industry cycle This paper<br>contribute tourism literature examine determinant Taiwanese tourist hotel<br>industry THI cycle This study use Markov switch<br>model MSM propose Hamilton analyze Taiwanese tourist hotel<br>industry cycle The MSM decomposes tourist hotel industry<br>cycle two distinct state high growth low growth<br>HGS LGS The mean growth rate HGS average<br>growth rate LGS period December February The corresponding<br>standard deviation two regime imply HGS stable LGS<br>Moreover probability stay HGS probability remain LGS The<br>expected duration HGS LGS month respectively Further paper<br>investigate factor keep THI HGS Empirical test result<br>show growth international tourism market industrial production growth<br>rate two key factor keep THI HGS SARS<br>outbreak adverse effect<br>",
          "Pathway Network Approaches Identification Cancer Signature Markers Omics<br>Data The advancement high throughput omic technology past<br>year make possible perform many complex assay much<br>short time traditional approach The rapid accumulation wide<br>availability omic data generate technology offer great opportunity<br>unravel disease mechanism also present significant challenge extract<br>knowledge massive data evaluate finding address challenge number<br>pathway network base approaches introduce This review article<br>evaluate method discuss application cancer biomarker discovery use<br>hepatocellular carcinoma HCC example<br>",
          "Dynamics seasonal airborne propagation Staphylococcus aureus academic dental<br>clinic OBJECTIVE Staphylococcus aureus strain disseminate dental treatment<br>occasionally lead contamination infection patient dentists important public<br>health problem The dynamic airborne propagation genetic diversity<br>aureus isolate academic dental clinic environment investigate use<br>isoenzyme type MATERIAL AND METHODS The isoenzymes previously<br>report isolates obtain fresh culture extract use glass<br>bead Nine isoenzymes investigate use multilocus enzyme electrophoresis<br>MLEE The genetic diversity relationship among strain electrophoretic<br>type determine use statistic previously describe Nei SAHN<br>group method UPGMA algorithm RESULTS Clonal pattern analysis<br>indicate high level genetic polymorphism occur among ETs<br>group five taxon Each taxon present one cluster<br>moderately relate contain two identical highly related isolates<br>reveal seasonal airborne propagation dental clinic environment CONCLUSIONS<br>These data suggest occurrence active microevolutionary process aureus<br>well possibility environmental propagation month time span Such<br>finding important show multiuser academic dental clinic retain<br>certain strain spreadable different niche<br>",
          "The Transmissibility Highly Pathogenic Avian Influenza Commercial Poultry<br>Industrialised Countries BACKGROUND With increased occurrence outbreak H5N1<br>worldwide concern virus could enter commercial poultry farm<br>severe economic consequence METHODOLOGY PRINCIPAL FINDINGS analyse data<br>four recent outbreak highly pathogenic avian influenza HPAI<br>commercial poultry estimate farm farm reproductive number HPAI<br>The reproductive number key measure transmissibility HPAI farm<br>level use evaluate effectiveness control measure outbreak mean<br>farm farm reproductive number prior control range maximum<br>farm base reproductive number range Enhanced bio security<br>movement restriction prompt isolation infect farm four outbreak<br>substantially reduce reproductive number remain close threshold value<br>necessary ensure disease eradicated CONCLUSIONS SIGNIFICANCE Our result<br>show depend particular situation outbreak avian influenza occur<br>current control might enough eradicate disease therefore close<br>monitoring outbreak require The method use estimate reproductive<br>number straightforward implement use real time therefore useful<br>tool inform policy decision<br>",
          "Topology evolve mutagenized viral population quasispecies expansion compression<br>operation negative selection BACKGROUND The molecular event evolutionary<br>force underlie lethal mutagenesis virus virus extinction excess<br>mutation well understood Here apply first time phylogenetic<br>method Partition Analysis Quasispecies PAQ monitor genetic distance<br>intra population structure mutant spectra foot mouth disease<br>virus FMDV quasispecies subject mutagenesis base nucleoside analogues<br>RESULTS Phylogenetic PAQ analyse reveal highly dynamic variation<br>intrapopulation diversity FMDV quasispecies The population diversity first<br>suffers strike expansion presence mutagens compression either presence<br>mutagenic analogue discontinue mutation decrease sensitivity mutagen select<br>The pattern mutation find population agreement behavior correspond<br>nucleotide analogue FMDV vitro Mutations accumulate preferred genomic<br>site ratio indicate operation negative purifying selection population<br>subject mutagenesis evidence unusually elevate genetic distance obtain<br>FMDV population approach extinction CONCLUSION Phylogenetic PAQ analysis<br>provide adequate procedure describe evolution viral sequence subject<br>lethal mutagenesis These method define change intra population<br>structure precisely mutation frequency Shannon entropy PAQ sensitive<br>variation intrapopulation genetic distance Strong negative purifying selection<br>operate FMDV population subject enhanced mutagenesis The quantification<br>provide evidence extinction imply unusual increase intrapopulation complexity<br>support lethal defection model virus extinction<br>",
          "Epidemics panic bioterrorist attack mathematical model Summary bioterrorist<br>attack usually cause epidemic panic targeted population present<br>epidemiologic aspect phenomenon three component model host information<br>attack social network propose mathematical model panic counter<br>measure function time population expose bioterrorist attack The<br>model comprise ordinary differential equation graphically present combination<br>equation parameter Clinically present model sequence psychic condition<br>disorder initiate act bioterrorism This model might helpful<br>attacked community timely properly apply counter measure minimize<br>human mental suffer bioterrorist attack<br>",
          "Distribution Risk Factors Pandemic Influenza H1N1 Mainland China<br>Data report case pandemic influenza H1N1 obtain China<br>Information System Disease Control Prevention The spatiotemporal distribution<br>pattern case characterize spatial analysis The impact travel<br>relate risk factor invasion disease analyze use survival<br>analysis climatic factor related local transmission identify use<br>multilevel Poisson regression county level The result show<br>epidemic span large geographic area affect area western<br>China Significant difference incidence find among age group<br>incidence peak school age child Overall epidemic spread<br>southeast northwest Proximity airport intersect national highway freeways<br>railway variable associate presence disease county Lower temperature<br>lower relative humidity climatic factor facilitate local transmission<br>correction effect school summer vacation public holiday well<br>population density density medical facility These finding indicate<br>intervention focus domestic travel population density climatic factor<br>could play role mitigate public health impact future<br>influenza pandemic<br>",
          "Preliminary estimation basic reproduction number novel coronavirus nCoV<br>China data driven analysis early phase outbreak BackgroundsAn<br>ongoing outbreak novel coronavirus nCoV pneumonia hit major<br>city China Wuhan December subsequently reach province region<br>China country present estimate basic reproduction number nCoV<br>early phase outbreak MethodsAccounting impact variation disease report<br>rate model epidemic curve nCoV case time series<br>mainland China January January exponential growth With estimate<br>intrinsic growth rate estimate use serial interval two<br>well know coronavirus diseases MERS SARS approximation true<br>unknown FindingsThe early outbreak data largely follow exponential<br>growth estimate mean range associate fold fold increase<br>report rate demonstrate change report rate substantially affect<br>estimate ConclusionThe mean estimate nCoV range significantly large<br>Our finding indicate potential nCoV cause outbreak<br>",
          "Documentary Research Human Respiratory Droplet Characteristics Respiratory droplet<br>characteristic key determine droplet borne pathogen transmission provide<br>scientific basis formulate disease prevention droplet transmission control<br>measure Through study data information exist document paper<br>give respiratory droplet characteristic like size concentration velocity<br>etc Meanwhile droplet evaporation droplet borne pathogen activity<br>transmission discuss The droplet size significant difference human<br>health level gender age The size droplet produce<br>health people 10μm produce patient 10μm patient droplet<br>concentration high The coughed droplet concentration change size<br>peak rule The velocity cough droplet big range<br>25m transmission distance<br>",
          "Application corona discharge generate air ion filtration aerosolize<br>virus inactivation filter virus The effect corona discharge<br>generate air ion filtration aerosolize bacteriophage MS2 study<br>carbon fiber ionizer instal upstream medium efficiency air<br>filter generate air ion use charge virus aerosols<br>increase filtration efficiency After virus aerosol capture filter<br>certain time interval expose newly incoming air ion<br>flow Captured virus particle detach filter sonication antiviral<br>efficiency due air ion calculate count plaque form<br>unit The antiviral efficiency increase ion exposure time<br>ion concentration When concentration positive air ion ion<br>cm3 antiviral efficiency exposure time 45min respectively When<br>ionizer operate bipolar mode number concentration positive negative<br>ion 6×106 4×106 ion cm3 respectively antiviral efficiency<br>exposure time 45min respectively quantitative parameter performance evaluation<br>air ion susceptibility constant bacteriophage MS2 positive negative<br>bipolar air ion calculate 5×10−3 4×10−3 5×10−3 respectively<br>These susceptibility constant show bipolar ion treatment effective<br>time unipolar ion treatment<br>",
          "Theoretical condition coexistence viral strain difference phenotypic trait<br>bifurcation analysis investigate dynamic wild type viral strain<br>generates mutant strain differ phenotypic property infectivity virulence<br>mutation rate study mean mathematical model bifurcation analysis<br>condition wild type mutant virus compete host cell<br>coexist The coexistence condition formulate term basic reproductive<br>number strain maximum value mutation rate virulence pathogens<br>The analysis reveals parameter space divide five region<br>distinct dynamic organize around degenerate Bogdanov–Takens zero Hopf<br>bifurcation latter give rise curve transcritical bifurcation periodic<br>orbit These result provide new insight condition viral<br>population may contain multiple coexist strain stable manner<br>",
          "Association energy use poor visibility Hong Kong SAR<br>China city reliance energy increase develop Moreover combustion<br>fossil fuel inevitably generate air pollutant include carbon<br>dioxide nitrogen oxide sulfur dioxide particulate matter others<br>Combining anthropogenic air pollutant visibility many Asian city<br>include Hong Kong deteriorate rapidly past decade This<br>paper explore relationship energy use meteorological factor change<br>visibility Hong Kong use long term time series<br>data The total use primary energy increase number<br>hour reduced visibility increase period time Bivariate correlation<br>show poor visibility significantly associate energy use annual<br>mean air temperature Multiple regression analysis indicate burn<br>aviation gasoline significantly adversely affect visibility Results illustrate<br>number clear day Hong Kong decrease particular due<br>increase air traffic<br>",
          "Pandemic H1N1 Influenza Community Transmission Was Established One<br>Australian State When Virus Was First Identified North<br>America BACKGROUND mid June State Victoria Australia appear<br>high notification rate pandemic H1N1 influenza world hypothesise<br>community transmission pandemic influenza already well establish Victoria<br>time test novel virus commence contrast true pandemic<br>part Australia include Western Australia METHODS use data<br>detailed case follow patient confirm infection Victoria demonstrate<br>difference pandemic curve two Australian state opposite side<br>continent model pandemic state use susceptible infect remove<br>model Bayesian inference accounting import case RESULTS Epidemic<br>transmission occur earlier Victoria later Only first Victorian<br>case locally acquire three brother one family contrast<br>first case associate importation Victoria Using plausible model<br>input data estimation effective reproductive number Victorian epidemic<br>require invoke earlier date commencement transmission explain observe<br>data This required modelling epidemic CONCLUSION Strong circumstantial<br>evidence support modelling suggests community transmission pandemic influenza<br>well establish Victoria time test novel virus commence<br>Australia The virus likely enter Victoria already become<br>establish around time first identify Mexico<br>",
          "Strategies enhance analytical performance nanomaterial base sensor provide<br>state art review main strategy enhancement analytical performance<br>sensor use nanomaterials particularly nanowires carbon base material<br>emphasize way overcome problem device device variation discuss<br>study influence nanomaterial characteristic sensor dimension operational condition<br>sense performance application appropriate calibration model<br>",
          "Realized generation time contraction impact infectious period reproduction<br>number population size One key characteristic transmission dynamic<br>infectious disease generation time refers time interval infection<br>secondary case infection infector The generation time distribution<br>together reproduction number determines rate infection spread population<br>When define generation time distribution calendar time two<br>definition plausible accord whether regard infection time infector<br>infection time infectee The result measurement respectively call<br>forward generation time backward generation time observe mean<br>forward generation time contract around peak epidemic This<br>contraction effect previously attribute either competition among potential<br>infectors depletion susceptibles population The first explanation require<br>many infectives contraction occur whereas latter explanation suggest<br>contraction occur even infectives With simulation study show<br>competition depletion cause mean forward generation time contract<br>Our result also reveal distribution infectious period reproduction<br>number strong effect size timing contraction well mean<br>value generation time forward backward scheme Author summaryInfectious<br>disease remain one great threat human health commerce<br>analysis epidemic data one important application statistic public<br>health Thus reliable estimate fundamental infectious disease parameter<br>critical public health decision maker order take appropriate<br>action global prevention management outbreaks health emergency key<br>example give prediction model reproduction number rely generation<br>time distribution usually estimate contact trace data collect<br>precise calendar time The forward scheme use prediction<br>model knowledge evolution time crucial correctly estimate parameter<br>interest therefore important characterize cause lead contraction mean<br>forward generation time course outbreak paper firstly identify<br>impact epidemiological quantity reproduction number infectious period population<br>size mean forward backward generation time Moreover analyze<br>phenomena competition among infectives depletion susceptible individual highlight<br>effect contraction mean forward generation time The upshot<br>investigation variance infectious period distribution reproduction number strong<br>impact generation time affect mean value evolution time<br>Furthermore competition depletion cause contraction even small value<br>reproduction number suggest epidemic model generation time consider<br>time inhomogeneous estimator account depletion compete risk prefer<br>inference generation interval distribution<br>",
          "Report Federal Panel Formaldehyde The Federal Panel Formaldehyde<br>conclude definitive experiment exist demonstrate mutagenicity carcinogenicity formaldehyde<br>laboratory condition Formaldehyde induces gene mutation chromosomal aberration<br>variety test system Inhalation formaldehyde cause cancer nose<br>rat The concentration formaldehyde inhaled air cause nasal<br>cancer Fisher rat within order magnitude human may<br>expose The data presently available permit direct assessment<br>carcinogenicity formaldehyde man Epidemiologic study expose human population<br>progress may clarify situation Other experimental human study<br>toxic effect teratogenicity reproductive disorder yet inadequate health<br>risk assessment The CIIT month study animal carcinogenicity<br>yet completely evaluated Additional data expect effect prolonged<br>exposure lower dos formaldehyde possible carcinogenicity formaldehyde mouse<br>The panel recommend comprehensive health risk assessment experiment<br>conduct effect mode exposure ingestion skin penetration effect<br>human pharmacokinetics formaldehyde man animal possible role formaldehyde<br>reproductive chronic respiratory disorder conclusion panel formaldehyde presume<br>pose carcinogenic risk human<br>",
          "Identification characterization novel type lectin MjLTL2 kuruma shrimp<br>Marsupenaeus japonicus type lectins LTLs belong lectin family<br>characterize conserved structural motif carbohydrate recognition domain LTLs<br>homologous leguminous lectin study identify functionally characterize LTL<br>kuruma shrimp Marsupenaeus japonicus designate LTL MjLTL2 MjLTL2<br>contain signal peptide Lectin leg domain coil coil<br>transmembrane domain MjLTL2 distribute hemocytes heart hepatopancreas gill<br>stomach intestine high expression level see hemocytes hepatopancreas<br>tissue MjLTL2 upregulated follow challenge shrimp Vibrio anguillarum<br>white spot syndrome virus WSSV MjLTL2 agglutinate several<br>bacteria without Ca2 addition MjLTL2 could bind several<br>Gram positive negative bacteria bind lipopolysaccharide peptidoglycan However<br>MjLTL2 could enhance clearance anguillarum vivo presence WSSV<br>infection MjLTL2 knockdown RNA interference result day lower<br>cumulative mortality japonicus Moreover less VP19 VP24 VP26<br>VP28 mRNAs extract hemocytes MjLTL2 knockdown shrimp control<br>These result suggest MjLTL2 involve immune response shrimp<br>",
          "Impact genotype porcine reproductive respiratory syndrome virus interferon<br>response plasmacytoid dendritic cell Porcine reproductive respiratory syndrome<br>PRRS virus PRRSV infection characterize prolonged viremia viral<br>shed consistent incomplete immunity Type interferons IFN essential<br>mount efficient antiviral innate adaptive immune response recent<br>study North American PRRSV genotype isolates induce even<br>strongly inhibit IFN plasmacytoid dendritic cell pDC represent<br>professional IFN produce cell Since inhibition IFN expression<br>might initiate PRRSV pathogenesis characterize PRRSV effect host<br>modify factor IFN response pDC Surprisingly variety type<br>type PRRSV directly stimulate IFN secretion pDC The<br>effect require live virus mediate TLR7 pathway Furthermore<br>IFN significantly enhance pDC production IFN response PRRSV<br>exposure PRRSV inhibition IFN responses enrich pDC stimulate<br>CpG oligodeoxynucleotides weak absent prototype genotype PRRSV suppress<br>response high level suppression induced Chinese highly pathogenic<br>PRRSV isolate Taken together find demonstrate pDC respond<br>PRRSV suggest suppressive activity pDC moderate strain dependent<br>Thus pDC may source systemic IFN response report<br>PRRSV infected animal contribute puzzle immunopathogenesis PRRS<br>",
          "Novel PDE4 Inhibitors Derived Chinese Medicine Forsythia Cyclic<br>adenosine monophosphate cAMP crucial intracellular second messenger molecule<br>convert extracellular molecule intracellular signal transduction pathway generate<br>cell stimulus specific effect Importantly specific phosphodiesterase PDE<br>subtypes control amplitude duration cAMP induce physiological process<br>therefore prominent pharmacological target currently use variety field<br>Here test extract traditional Chinese medicine Forsythia suspense<br>seed use year relieve respiratory symptom Using structural<br>functional analysis find major lignin Forsynthin act immunosuppressant<br>inhibiting PDE4 inflammatory immune cell Moreover several novel<br>selective small molecule derivative Forsythin test vitro murine<br>model viral bacterial pneumonia sepsis cytokine drive systemic<br>inflammation Thus pharmacological target PDE4 may promise strategy<br>immune related disorder characterize amplified host inflammatory response<br>",
          "Evaluation Anti Toxoplasma gondii Effect Ursolic Acid Novel<br>Toxoplasmosis Inhibitor This study carry evaluate anti parasitic<br>effect ursolic acid Toxoplasma gondii gondii induces toxoplasmosis<br>particularly humans The anti parasitic effect ursolic acid<br>gondii infect cell gondii evaluate different specific assay<br>include immunofluorescence stain animal test Ursolic acid effectively<br>inhibit proliferation gondii compare sulfadiazine consistently induce anti<br>gondii activity effect particular formation parasitophorous vacuole membrane<br>PVM host cell markedly decrease treat ursolic acid<br>effectively suppress Moreover survival rate gondii strongly inhibit<br>gondii group treat ursolic acid inhibitory concentration gondii<br>measure The gondii infect mice treat ursolic acid<br>indicate survival rate activity normal group These result<br>demonstrate ursolic acid cause anti gondii action effect<br>strongly block proliferation gondii direct selective gondii inhibitory<br>ability well increase survival gondii infect mouse This<br>study show ursolic acid potential use promising anti<br>gondii candidate substance develop effective anti parasitic drug<br>",
          "Neuronal CCL2 expression drive inflammatory monocyte infiltration brain<br>acute virus infection BackgroundViral encephalitis dangerous compromise need<br>robustly clear pathogen brain need protect neuron bystander<br>injury Theiler murine encephalomyelitis virus TMEV infection C57Bl<br>mouse model viral encephalitis compromise result hippocampal damage<br>permanent neurological sequela previously identify brain infiltrate inflammatory<br>monocyte primary driver hippocampal pathology mechanism involve recruit<br>cell brain unclear MethodsChemokine expression level hippocampus assess<br>microarray ELISA PCR immunofluorescence Monocyte infiltration acute TMEV<br>infection measure flow cytometry CCL2 level manipulate immunodepletion<br>specific removal neuron mice generate cross line express<br>Cre recombinase behind synapsin promoter animal floxed CCL2<br>ResultsInoculation brain TMEV induce hippocampal production proinflammatory chemokine<br>CCL2 peak hour postinfection whereas inoculation inactivate TMEV<br>elicit response Immunofluorescence reveal hippocampal neuron express high<br>level CCL2 timepoint Genetic deletion CCR2 systemic immunodepletion<br>CCL2 abrogate blunted infiltration inflammatory monocyte brain acute<br>infection Specific genetic deletion CCL2 neuron reduce serum<br>hippocampal CCL2 level inhibit inflammatory monocyte infiltration brain<br>ConclusionsWe conclude intracranial inoculation infectious TMEV rapidly induce<br>expression CCL2 neuron cellular source necessary CCR2 dependent<br>infiltration inflammatory monocyte brain acute stage encephalitis These<br>finding highlight unique role neuronal production chemokines initiation<br>leukocytic infiltration infected central nervous system<br>",
          "Combination treatment quercetin resveratrol attenuate high fat diet<br>induce obesity associate inflammation rat via AMPKα1 SIRT1<br>signal pathway Diet induce obesity associate systemic inflammation<br>consider originate predominantly adipose tissue Quercetin resveratrol two<br>dietary polyphenols exhibit anti inflammatory property anti insulin<br>resistance administer isolation combination CQR remain unknown whether<br>CQR reduces high fat diet HFD induce obesity<br>inflammation rat current study male Wistar rat divide<br>two group one feed normal diet fat one<br>fed HFD fat week Following removal obesity resistant<br>rat HFD group remain rat divide two sub<br>group HFD group HFD CQR group administer day<br>resveratrol day quercetin The result reveal HFD CQR<br>group significantly low body weight week compare HFD<br>group significantly reduce visceral adipose tissue weight adipocyte<br>size Serum lipid profile also significantly ameliorate HFD<br>CQR group CQR attenuate expression systemic proinflammatory adipokines<br>include leptin tumor necrosis factor monocyte chemoattractant protein<br>interleukin also reduce recruitment mast cell epididyotic adipose<br>tissue EAT Furthermore CQR reverse HFD induce suppression<br>adenosine monophosphate activate protein kinase AMPKα1 phosphorylation sirtuin<br>SIRT1 expression EAT conclusion CQR may suppress obesity<br>associate inflammation via AMPKα1 SIRT1 signal pathway rat<br>feed HFD<br>",
          "Cytokine nitric oxide pattern dog immunized LBSap vaccine<br>experimental challenge Leishmania chagasi plus saliva Lutzomyia longipalpis<br>study present dog vaccinate Leishmania Leishmania chagasi challenge<br>infection use preparation Leishmania braziliensis promastigote proteins saponin<br>adjuvant LBSap Vaccination LBSap induce prominent type immune<br>response characterize increased level interleukin interferon gamma IFN<br>production peripheral blood mononuclear cell PBMC upon stimulation<br>soluble vaccine antigen Importantly result show type responsiveness<br>sustain challenge infection day chagasi challenge infection PBMCs<br>LBSap vaccinate dog produce IFN concomitant nitric oxide<br>stimulate Leishmania antigen compare PBMCs respective control group<br>saponin treat non treat control dog Moreover transform<br>growth factor TGF decrease supernatant SLcA stimulate PBMCs<br>LBSap group day Bone marrow parasitological analysis reveal<br>decreased frequency parasitism presence vaccine antigen conclude vaccination<br>dog LBSap vaccine induce long lasting type immune<br>response chagasi challenge infection<br>",
          "Methamphetamine induces autophagy pro survival response apoptotic endothelial<br>cell death Kappa opioid receptor Methamphetamine METH psychostimulant<br>high abuse potential severe neurotoxicity Recent study animal<br>model indicate METH impair blood–brain barrier BBB suggest<br>neurotoxic effect result METH abuse could due barrier<br>disruption report chronic exposure METH disrupt barrier function<br>primary human brain microvascular endothelial cell HBMECs human<br>umbilical vein endothelial cell HUVECs early pro survival<br>response observe follow acute exposure induction autophagic mechanism<br>Acute METH exposure induces early increase Beclin1 LC3<br>recruitment This mediate inactivation protein kinase Akt mammalian<br>target rapamycin mTOR p70S6K pathway upregulation ERK1 Blockade<br>Kappa opioid receptor KOR treatment autophagic inhibitor accelerate<br>METH induce apoptosis suggest early autophagic response survival<br>mechanism endothelial cell mediate kappa opioid receptor Our<br>study indicate kappa opioid receptor therapeutically exploit attenuate<br>METH induce BBB dysfunction<br>",
          "Studies intrathecal humoral immune response canine distemper encephalitis<br>Albumin IgG quantitated paired cerebrospinal fluid CSF serum<br>sample dog demyelinate canine distemper virus CDV infection<br>mean rocket immunoelectrophoresis The IgG index indicator intrathecal<br>immunoglobulin synthesis normal animal non inflammatory demyelinate lesion<br>elevate dog inflammatory myelin lesion Specific antibody CDV<br>myelin quantitated CSF serum dog elevate IgG index<br>Eight dog significant amount antimyelin antibody dog neutralize<br>anti CDV antibody CSF Whereas pathogenetic significance antimyelin<br>antibody remain uncertain intrathecal antiviral immune response provide<br>plausible explanation immunopathologic destruction myelin distemper<br>",
          "Autoimmune Cholangitis SJL Mouse Antigen Non specific Primary<br>biliary cirrhosis PBC autoimmune disease characterize intrahepatic bile<br>duct destruction production anti mitochondrial antibody AMA The<br>absence animal model strike impedance define molecular basis<br>disease Previous work suggest SJL mouse immunize pyruvate<br>dehydrogenase complex PDC major mitochondrial autoantigen PBC lead<br>development lymphoid cell infiltration portal tract model system<br>coin autoimmune cholangitis hypothesize pathology would augment immunization<br>occur presence IFN injection Accordingly SJL mouse immunize<br>PDC purpose control casein Subgroups mouse also treat<br>exogenous IFN expect mouse immunize PDC without IFN<br>develop high titer AMAs contrast mouse immunize casein<br>develop antinuclear antibody More importantly livers mouse immunize<br>PDC immunize casein display lymphoid cell infiltration portal<br>tract irrespective bile duct size Indeed significant difference<br>experimental control group histologic analysis Thus autoimmune cholangitis<br>mouse antigen non specific<br>",
          "Bystander CD4 cell mediate demyelination mouse infect neurotropic<br>coronavirus Demyelination follow infection mouse neurotropic coronavirus mouse<br>hepatitis virus strain JHM MHV immune mediate demonstrate<br>MHV specific CD4 CD8 cell capable cause demyelination<br>independent cell subset Recent work also demonstrate activated<br>bystander CD8 cell mediate significant demyelination The ability<br>bystander CD4 cell mediate demyelination investigate use CD4<br>cell transgenic mouse The result indicate bystander CD4<br>cell unable cause demyelination MHV infect mice despite<br>recruited central nervous system CNS irrespective activation status<br>These result suggest CD4 cell must recognize antigen<br>CNS order cause demyelination<br>",
          "Transcriptional Regulation Antiviral Interferon Stimulated Genes Interferon stimulate<br>gene ISGs group gene product coordinately combat pathogen<br>invasion particular viral infection Transcription ISGs occur rapidly<br>upon pathogen invasion classically provoke via activation Janus<br>kinase signal transducer activator transcription JAK–STAT pathway mainly<br>interferon IFNs However plethora recent study report variety<br>non canonical mechanism regulate ISG transcription These new<br>study extremely important understand quantitative temporal difference ISG<br>transcription specific circumstance Because canonical non canonical regulatory<br>mechanism essential define nature host defense associate detrimental<br>proinflammatory effect comprehensively review state rapidly evolve field<br>clinical implication recently acquire knowledge respect<br>",
          "Mechanisms HIV non progression robust sustain CD4 cell<br>proliferative response p24 antigen correlate control viraemia lack<br>disease progression long term transfusion acquire HIV infection<br>BACKGROUND Elite non progressors plasma viral load copy<br>antiretroviral naive constitute tiny fraction HIV infect individual<br>After year follow cohort long term non progressors<br>LTNP identify individual transfusion acquire HIV infection remain<br>LTNP year infection retain elite LTNP status examine<br>mechanism differentiate delayed progressors LTNP cohort RESULTS survival<br>advantage confer subject least one host genetic factor<br>HLA chemokine receptor TLR polymorphisms viral attenuate factor<br>defective nef associate slow progression However antiviral immune<br>response differentiate course disease beyond second decade infection<br>stable p24 specific proliferative response associate control viraemia<br>retention non progressor status p24 response absent decline<br>viraemic subject Strong Gag dominant cytotoxic lymphocyte CTL<br>responses identify LTNP Pol dominant CTL nef defective<br>HIV infection CTL associate control viraemia combine p24<br>proliferative response However CTL prevent late disease progression<br>Individuals sustain viral suppression CTL recognise numerous Gag<br>epitopes strong restricted response one two immunodominant epitope<br>effective time fail contain viraemia course study Viral<br>escape mutant HLA B27 restrict Gag p24 epitope<br>detect individual whereas decline negative p24 proliferative response<br>occur concurrent increase viraemia CONCLUSION Detectable viraemia study<br>entry predictive loss LTNP status disease progression differentiate<br>slow progressors elite LTNP retain potent virological control<br>Sustained immunological suppression viraemia independently associate preserve p24<br>proliferative response regardless strength breadth CTL response decline<br>protective p24 response precede correlated loss non progressor<br>status sign disease progression<br>",
          "Visceral target organ systemic Louis encephalitis virus infection<br>hamster Ultrastructural aspect Louis encephalitis virus infection major<br>extraneutral organ tissue suckle hamster examine pancreas exocrine<br>endocrine portion equally affect virus feature apparently unique<br>flavivirus accumulation virus particle type secretory granule organ<br>Virus particle see within myocardial fiber within smooth<br>muscle cell endothelial cell small blood vessel heart<br>intestine lamina propria severely infect virus particle accumulate<br>cell type<br>",
          "Multifunctional viral protein γ34 manipulates nucleolar protein NOP53<br>optimal viral replication HSV ensure efficient virus replication<br>herpes simplex virus type HSV encodes several viral<br>protein counter host defense response upon infection Among<br>protein multifunctional viral protein γ34 crucially interfere disrupts<br>several antiviral pathway multiple level The current study<br>show γ34 utilizes nucleolar protein NOP53 facilitate dephosphorylation<br>eukaryotic initiation factor eIF2α efficient viral translation Our<br>study show ectopic expression NOP53 greatly increase intracellular<br>extracellular viral yield HSV wild strain type interferon<br>deficient Vero cell subtly promote viral replication γ34<br>deletion mutant virus HSV Δγ34 NOP53 migrate nuclei<br>HSV infect cell redistribute incompletely infection either HSV<br>Δγ34 ICP4 deletion mutant virus HSV d120 replication<br>inadequate Ectopic expression γ34 consequently induces cytoplasmic translocation<br>NOP53 response HSV Δγ34 infection Increase NOP53 two<br>form transient transfection vitro expression attenuate phosphorylation level<br>eIF2α HSV infect cell fails affect eIF2α phosphorylation<br>induce HSV Δγ34 infection Knockdown NOP53 impair specific<br>interaction γ34 protein phosphatase PP1α disrupt ability γ34<br>maintain HSV virulence NOP53 knockdown also significantly reduce<br>tissue damage decrease viral yield liver HSV infected<br>mice Our finding expand understand underlie mechanism viral<br>protein γ34 induces NOP53 redistribution cytoplasmic NOP53 facilitate<br>γ34 recruitment PP1α dephosphorylate eIF2α optimal viral replication<br>This paper also demonstrate block specific interaction γ34<br>PP1α would useful approach development antiviral agent<br>",
          "Intestinal microbiota chronic constipation Chronic constipation prevalent burdensome<br>gastrointestinal disorder whose aetiology pathophysiology remain poorly understood<br>likely multifactorial Differences composition intestinal microbiota demonstrate constipated<br>patient healthy control compare Growing evidence indicate alteration<br>intestinal microbiota may contribute constipation constipation relate symptoms<br>The intestinal microbiota collection microorganism live within gastrointestinal<br>tract perform many important health promote function The<br>intestinal microbiota aid breakdown food product absorbable nutrient<br>stimulate host immune system prevent growth pathogenic bacteria<br>produce great variety biologically important compound review summarize<br>current evidence support role intestinal microbiota pathogenesis management<br>chronic constipation The discussion shed light novel mechanism<br>intestinal microbiota gut function interaction invaluable ultimately develop<br>new therapeutic tool treatment chronic constipation<br>",
          "Porcine arterivirus activate pathway IκB degradation Nuclear factor<br>kappaB critical regulator innate adaptive immune function well<br>cell proliferation survival The present study demonstrate first<br>time virus belong Arteriviridae family activate MARC cell<br>alveolar macrophage porcine reproductive respiratory syndrome virus PRRSV<br>infect cell activation characterize translocation cytoplasm nucleus increase<br>DNA bind activity regulate gene expression activation increase<br>PRRSV infection progress viral dose dependent manner inactivation<br>PRRSV significantly reduce level activation Degradation IκB protein<br>detect late PRRSV infection overexpression dominant negative form<br>IκBα IκBαDN significantly suppress activation induce PRRSV However<br>IκBαDN affect viral replication viral cytopathic effect PRRSV<br>infection induce oxidative stress cell generate reactive oxygen<br>specie ROS antioxidant inhibit DNA bind activity PRRSV<br>infect cell suggest ROS mechanism activate PRRSV infection<br>Moreover dependent expression matrix metalloproteinase MMP MMP observe<br>PRRSV infect cell observation implies activation biologically significant<br>aspect PRRSV pathogenesis The result present provide basis<br>understand molecular pathway pathology immune evasion associate disease<br>cause PRRSV<br>",
          "Recapitulation SARS CoV Infection Cholangiocyte Damage Human Liver<br>Organoids The newly emerge pandemic coronavirus SARS CoV<br>pose significant public health threat worldwide However mode<br>virus transmission tissue tropism well establish yet Recent<br>finding substantial liver damage patient ACE2 cholangiocytes healthy<br>liver tissue prompt hypothesize human liver ductal organoids<br>could serve model determine susceptibility mechanism underline liver<br>damage upon SARS CoV infection single cell RNA<br>sequence find long term liver ductal organoid culture<br>preserve human specific ACE2 population cholangiocytes Moreover human<br>liver ductal organoids permissive SARS CoV infection support<br>robust replication Notably virus infection impair barrier bile<br>acid transport function cholangiocytes dysregulation gene involve tight<br>junction formation bile acid transportation could explain bile<br>acid accumulation consequent liver damage patient These result<br>indicate control liver damage cause directly viral infection<br>value treat COVID patient Our finding also provide<br>application human organoids investigate tropism pathogenesis SARS CoV<br>would facilitate novel drug discovery<br>",
          "Endorphin protect mice neurological disease induce murine coronavirus<br>MHV JHM The neurotropic murine coronavirus MHV JHM<br>JHMV cause encephalitis paralytic demyelinate disease susceptible strain<br>mice rat serve model human demyelinate disease multiple<br>sclerosis communication report single intracerebral administration naturally occur<br>neuropeptide endorphin reduce incidence JHMV induce paralytic demyelinating<br>disease 40–50 C57Bl mouse Protection disease accompany significantly<br>reduce virus replication brain early day post infection<br>occur irradiate immunoincompetent mouse The data suggest endorphin<br>engage immune mechanism host resistance JHMV infection protect<br>mouse disease<br>",
          "Atg5 Supports Rickettsia australis Infection Macrophages Vitro Vivo<br>Rickettsiae cause life threaten infection human Macrophages one<br>initial target rickettsiae inoculation tick However remain poorly<br>understood rickettsia remain free macrophage prior establish infection<br>microvascular endothelial cell Here demonstrate concentration Rickettsia australis<br>significantly great infected tissue Atg5 flox flox mouse<br>counterpart Atg5 flox flox Lyz Cre mouse association<br>reduce level interleukin serum The great concentration australis<br>Atg5 flox flox bone marrow derive macrophage BMMs<br>Atg5 flox flox Lyz Cre BMMs vitro abolish<br>exogenous treatment recombinant Rickettsia australis induce significantly increase<br>level light chain LC3 form LC3 LC3 puncta<br>Atg5 competent BMMs Atg5 deficient BMMs p62 turnover<br>observe Further analysis find colocalization LC3 small portion<br>australis Rickettsia contain double membrane bound vacuole BMMs<br>mouse Moreover treatment rapamycin significantly increase concentration australis<br>BMMs compare untreated BMM control Taken together result<br>demonstrate Atg5 favor australis infection mouse macrophages association<br>suppress level production active autophagy flux These data<br>highlight contribution Atg5 macrophages pathogenesis rickettsial disease<br>"
         ],
         "marker": {
          "color": [
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           0,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           1,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           2,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           3,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           4,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           5,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           6,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           7,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           8,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9,
           9
          ],
          "opacity": 0.8,
          "size": 5
         },
         "mode": "markers+text",
         "text": [
          "d14089",
          "d8648",
          "d7744",
          "d562",
          "d1339",
          "d867",
          "d14250",
          "d4470",
          "d6063",
          "d7320",
          "d9110",
          "d3203",
          "d13305",
          "d8791",
          "d17820",
          "d12105",
          "d18115",
          "d16261",
          "d4781",
          "d11823",
          "d16369",
          "d13060",
          "d15572",
          "d15498",
          "d2584",
          "d12575",
          "d10682",
          "d9981",
          "d17141",
          "d11731",
          "d4022",
          "d8430",
          "d15169",
          "d15696",
          "d13397",
          "d9490",
          "d10968",
          "d18185",
          "d2645",
          "d12604",
          "d4074",
          "d17761",
          "d18445",
          "d3960",
          "d3773",
          "d3995",
          "d15766",
          "d18130",
          "d6907",
          "d9846",
          "d2562",
          "d17309",
          "d7535",
          "d2021",
          "d15150",
          "d10389",
          "d12279",
          "d12164",
          "d9640",
          "d4132",
          "d162",
          "d17486",
          "d8680",
          "d13367",
          "d2149",
          "d1182",
          "d11869",
          "d10799",
          "d17748",
          "d3167",
          "d14843",
          "d17015",
          "d11906",
          "d2897",
          "d14105",
          "d9469",
          "d7176",
          "d13324",
          "d680",
          "d7566",
          "d14773",
          "d8063",
          "d6337",
          "d838",
          "d6508",
          "d16190",
          "d5222",
          "d10947",
          "d11812",
          "d3889",
          "d10463",
          "d13184",
          "d15061",
          "d7147",
          "d2036",
          "d5279",
          "d8163",
          "d3741",
          "d927",
          "d11095",
          "d15013",
          "d1253",
          "d16173",
          "d8578",
          "d14484",
          "d4327",
          "d9059",
          "d14290",
          "d5",
          "d5678",
          "d11209",
          "d1397",
          "d15023",
          "d5179",
          "d6811",
          "d15739",
          "d2132",
          "d13740",
          "d8820",
          "d3065",
          "d12675",
          "d8301",
          "d1939",
          "d2040",
          "d5924",
          "d16108",
          "d4145",
          "d10620",
          "d12277",
          "d17612",
          "d8264",
          "d2290",
          "d12185",
          "d10166",
          "d6481",
          "d9294",
          "d8300",
          "d13095",
          "d720",
          "d14225",
          "d5387",
          "d589",
          "d18098",
          "d7377",
          "d6796",
          "d2750",
          "d14320",
          "d10344",
          "d4988",
          "d2626",
          "d7160",
          "d7253",
          "d2765",
          "d11435",
          "d4977",
          "d10993",
          "d5339",
          "d16468",
          "d12961",
          "d10850",
          "d10124",
          "d9909",
          "d13938",
          "d16922",
          "d3901",
          "d10886",
          "d14136",
          "d7361",
          "d7623",
          "d4447",
          "d9045",
          "d16591",
          "d6916",
          "d17345",
          "d15120",
          "d4562",
          "d8131",
          "d2484",
          "d17087",
          "d7310",
          "d5892",
          "d9752",
          "d10889",
          "d14331",
          "d16944",
          "d13760",
          "d4373",
          "d10336",
          "d307",
          "d7552",
          "d2454",
          "d5662",
          "d7724",
          "d20",
          "d13974",
          "d12299",
          "d1527",
          "d16940",
          "d351",
          "d14941"
         ],
         "textposition": "bottom center",
         "type": "scatter3d",
         "x": [
          -14.758460998535156,
          -1.5593979358673096,
          6.747421741485596,
          6.883765697479248,
          -2.411346673965454,
          -0.8346594572067261,
          16.403703689575195,
          -2.6025631427764893,
          0.37998080253601074,
          3.4557249546051025,
          4.183548927307129,
          10.026732444763184,
          3.781789779663086,
          -1.4950026273727417,
          7.55897331237793,
          4.557369232177734,
          5.658880710601807,
          0.3425261378288269,
          4.998603820800781,
          5.413995265960693,
          -7.62047004699707,
          6.476214408874512,
          -10.560863494873047,
          1.0201822519302368,
          -6.028838634490967,
          7.65266752243042,
          -7.2347092628479,
          5.497418403625488,
          -13.37186336517334,
          -4.304635524749756,
          10.058675765991211,
          -6.817416191101074,
          12.596207618713379,
          -5.139933109283447,
          -7.448812007904053,
          -14.152554512023926,
          1.1511213779449463,
          -4.533917427062988,
          -6.416491508483887,
          2.294384241104126,
          0.6566562056541443,
          16.404918670654297,
          1.7772338390350342,
          16.748964309692383,
          -5.955498695373535,
          10.919309616088867,
          10.806684494018555,
          4.968904495239258,
          10.049087524414062,
          10.026217460632324,
          9.19050121307373,
          2.537949562072754,
          8.316756248474121,
          5.790224075317383,
          7.640994548797607,
          5.345418453216553,
          15.553848266601562,
          8.579740524291992,
          8.986109733581543,
          9.633508682250977,
          11.697203636169434,
          0.23885835707187653,
          16.09632110595703,
          12.810994148254395,
          7.243946075439453,
          3.932512044906616,
          24.179935455322266,
          2.746701717376709,
          8.430453300476074,
          10.499292373657227,
          3.120927333831787,
          20.12628173828125,
          3.4869747161865234,
          13.60280990600586,
          9.806488037109375,
          21.379653930664062,
          2.558500289916992,
          10.909183502197266,
          3.56125545501709,
          21.033973693847656,
          -14.05478572845459,
          -16.57465171813965,
          -1.3367843627929688,
          -13.307145118713379,
          -16.453580856323242,
          -15.139055252075195,
          -1.8622174263000488,
          0.05556175485253334,
          -10.736652374267578,
          -19.357322692871094,
          -13.682723999023438,
          -10.212357521057129,
          -7.861693382263184,
          -14.547215461730957,
          -12.927875518798828,
          -3.7351186275482178,
          -20.303131103515625,
          -15.890814781188965,
          -9.135748863220215,
          -10.826556205749512,
          6.1342267990112305,
          -0.3738477826118469,
          7.487332344055176,
          30.136960983276367,
          -2.5513947010040283,
          -5.390527725219727,
          5.887366771697998,
          8.052579879760742,
          13.293617248535156,
          4.808326244354248,
          10.043403625488281,
          -3.5196635723114014,
          6.6335601806640625,
          2.9951698780059814,
          0.29168975353240967,
          6.529192924499512,
          4.226467609405518,
          -1.2130746841430664,
          4.188297271728516,
          -4.819647789001465,
          -2.115407943725586,
          -25.086721420288086,
          4.199974060058594,
          -2.869131088256836,
          2.5430192947387695,
          0.42532938718795776,
          -17.16278839111328,
          -11.467446327209473,
          -10.77347469329834,
          -9.709280967712402,
          -9.465895652770996,
          2.909158229827881,
          -3.768646240234375,
          -12.478612899780273,
          -6.63132381439209,
          -3.7310984134674072,
          1.6054363250732422,
          -16.94380760192871,
          -6.166832447052002,
          -9.590152740478516,
          -5.706259727478027,
          -4.294061183929443,
          -14.269194602966309,
          -15.570259094238281,
          -10.372300148010254,
          -8.8964204788208,
          -2.7124671936035156,
          -16.93022918701172,
          -17.421279907226562,
          -13.190397262573242,
          -8.851069450378418,
          -13.061746597290039,
          0.8270415663719177,
          -10.215428352355957,
          -9.530171394348145,
          -13.979098320007324,
          -0.5847605466842651,
          -11.785591125488281,
          5.391536712646484,
          -15.707098960876465,
          -26.508220672607422,
          -10.411067008972168,
          -3.69337797164917,
          -26.843833923339844,
          -22.305767059326172,
          -8.76103401184082,
          -11.703912734985352,
          -17.93974494934082,
          -5.281763076782227,
          -24.14638900756836,
          -19.89970588684082,
          -25.94806480407715,
          -10.971765518188477,
          -8.165480613708496,
          -24.874969482421875,
          -20.45370864868164,
          -21.707887649536133,
          -5.501662254333496,
          -25.095794677734375,
          -12.045530319213867,
          11.584168434143066,
          21.315977096557617,
          20.771621704101562,
          8.416850090026855,
          19.81843376159668,
          15.907463073730469,
          11.395108222961426,
          27.16779899597168,
          16.052215576171875,
          21.82332420349121,
          19.344573974609375,
          16.805221557617188,
          14.092907905578613,
          5.83634614944458,
          26.53861427307129,
          4.98627233505249,
          26.4468994140625,
          2.7311956882476807,
          21.46772003173828,
          25.420705795288086
         ],
         "y": [
          -4.947906494140625,
          21.530719757080078,
          11.641042709350586,
          -8.893977165222168,
          -6.3615031242370605,
          11.74229907989502,
          -13.68713092803955,
          -6.243285179138184,
          -7.480836391448975,
          -10.96567153930664,
          -7.130857944488525,
          -26.05312156677246,
          21.379444122314453,
          -8.241707801818848,
          3.5384418964385986,
          -7.460297107696533,
          -11.53585147857666,
          22.37990951538086,
          -13.386549949645996,
          -6.987489700317383,
          -13.794800758361816,
          -11.084579467773438,
          -4.296988010406494,
          -10.264849662780762,
          -9.333945274353027,
          24.255992889404297,
          -7.300210475921631,
          -18.632814407348633,
          -13.90951919555664,
          -6.872381687164307,
          -17.572864532470703,
          -7.5159831047058105,
          1.1450226306915283,
          -10.423210144042969,
          -6.449275493621826,
          -16.88384437561035,
          -18.565135955810547,
          0.14962859451770782,
          -20.037508010864258,
          -13.37198257446289,
          -0.8824686408042908,
          6.639695644378662,
          5.535533905029297,
          -3.1713361740112305,
          27.799394607543945,
          -4.4159674644470215,
          -13.081083297729492,
          -15.47574234008789,
          -8.76048469543457,
          -6.959766387939453,
          -21.68290138244629,
          -15.649198532104492,
          -13.559431076049805,
          0.640267014503479,
          -17.840627670288086,
          -4.495349884033203,
          -9.208786010742188,
          3.600609302520752,
          -11.008600234985352,
          -16.14583396911621,
          -10.70869255065918,
          -17.472869873046875,
          -1.2469837665557861,
          -12.436103820800781,
          -8.184508323669434,
          -16.159080505371094,
          -7.293708801269531,
          -9.56209659576416,
          -9.453003883361816,
          -3.054520845413208,
          -14.538853645324707,
          -5.555963516235352,
          -20.262531280517578,
          2.7448980808258057,
          -18.623096466064453,
          -2.913465738296509,
          -18.06801414489746,
          -9.783639907836914,
          -17.708904266357422,
          -12.525520324707031,
          12.671036720275879,
          5.2731404304504395,
          10.070235252380371,
          11.276267051696777,
          7.772252082824707,
          5.686242580413818,
          6.506288051605225,
          10.034111976623535,
          -1.7777026891708374,
          7.703449726104736,
          -1.7844003438949585,
          -1.885746717453003,
          4.389527320861816,
          14.96146011352539,
          8.747055053710938,
          9.850485801696777,
          18.522878646850586,
          9.55283260345459,
          3.033768653869629,
          18.607511520385742,
          7.023423671722412,
          6.2332682609558105,
          -0.4593025743961334,
          -2.7598886489868164,
          0.7187276482582092,
          10.81802749633789,
          2.583587169647217,
          1.6916465759277344,
          7.570189476013184,
          0.45168960094451904,
          4.387833595275879,
          0.4821181893348694,
          7.75460958480835,
          -17.01590919494629,
          14.845973014831543,
          -0.6806573271751404,
          13.247868537902832,
          0.3135074973106384,
          -5.470859527587891,
          -1.031191110610962,
          22.72294044494629,
          1.9842605590820312,
          -2.10861873626709,
          15.064866065979004,
          13.649025917053223,
          14.214743614196777,
          4.908204078674316,
          -0.4396931231021881,
          22.347644805908203,
          5.04646110534668,
          12.658980369567871,
          10.21087646484375,
          5.817843437194824,
          4.8763322830200195,
          5.808327674865723,
          23.176259994506836,
          15.678879737854004,
          1.7033122777938843,
          0.8097997307777405,
          8.073330879211426,
          -18.07140350341797,
          -23.67438316345215,
          -9.846227645874023,
          -15.045543670654297,
          -6.997922420501709,
          -20.66811180114746,
          -8.309974670410156,
          -16.792007446289062,
          -18.625282287597656,
          -16.73149299621582,
          -19.827186584472656,
          -21.479284286499023,
          -10.396490097045898,
          2.805621862411499,
          -18.237638473510742,
          -16.659374237060547,
          16.446256637573242,
          -16.578649520874023,
          10.010359764099121,
          -16.664154052734375,
          3.525963306427002,
          -9.402915954589844,
          10.967324256896973,
          -5.0580267906188965,
          8.265405654907227,
          -6.096765041351318,
          -6.860886096954346,
          -6.492287635803223,
          -14.202545166015625,
          7.2191033363342285,
          -4.233539581298828,
          -0.7309901118278503,
          7.093688488006592,
          4.578941822052002,
          -7.951097011566162,
          7.770303249359131,
          1.8222918510437012,
          -1.569687008857727,
          -7.323306083679199,
          2.432588815689087,
          0.9233314394950867,
          2.008662700653076,
          12.50221061706543,
          17.588884353637695,
          9.219015121459961,
          17.099952697753906,
          -1.6538732051849365,
          6.248056411743164,
          8.573320388793945,
          9.005380630493164,
          5.377816677093506,
          3.163182258605957,
          0.18416450917720795,
          5.2689971923828125,
          -1.138353705406189,
          8.783913612365723,
          1.9255558252334595,
          4.130966663360596,
          8.945588111877441,
          6.776242733001709
         ],
         "z": [
          4.459395408630371,
          8.469430923461914,
          0.20084016025066376,
          21.47673797607422,
          28.59128189086914,
          -1.4297348260879517,
          11.38979434967041,
          25.41541290283203,
          18.877641677856445,
          26.485610961914062,
          26.608680725097656,
          -0.6651536822319031,
          2.8161261081695557,
          23.38088607788086,
          19.77296257019043,
          22.696481704711914,
          17.77130699157715,
          7.3568220138549805,
          22.11231803894043,
          26.00506019592285,
          16.624404907226562,
          15.031289100646973,
          7.005248546600342,
          -13.688152313232422,
          17.915189743041992,
          7.879239559173584,
          7.182387828826904,
          12.23013687133789,
          -11.617864608764648,
          2.4723472595214844,
          7.019991874694824,
          4.721248149871826,
          1.2311240434646606,
          9.77799129486084,
          -1.6333314180374146,
          -9.938632011413574,
          -12.707927703857422,
          -1.379171371459961,
          7.490042686462402,
          16.709228515625,
          -7.739842891693115,
          14.14622688293457,
          0.3689606189727783,
          7.726563453674316,
          14.266908645629883,
          4.948354721069336,
          16.197328567504883,
          0.4952230155467987,
          9.46934700012207,
          7.495045185089111,
          -3.7651264667510986,
          -3.2459216117858887,
          -2.1964709758758545,
          6.176910877227783,
          1.1944117546081543,
          0.6415218114852905,
          7.049211025238037,
          -9.643829345703125,
          9.648802757263184,
          9.402849197387695,
          -12.259742736816406,
          -8.943656921386719,
          -13.999763488769531,
          -3.1746928691864014,
          -17.414783477783203,
          -20.275705337524414,
          -8.801103591918945,
          -18.853126525878906,
          -7.524425506591797,
          -20.141817092895508,
          -17.828697204589844,
          -6.145440101623535,
          -12.187325477600098,
          -10.839193344116211,
          1.6999067068099976,
          -15.089362144470215,
          -20.207874298095703,
          -12.031496047973633,
          -19.78229522705078,
          -9.796719551086426,
          -1.888786792755127,
          3.2597222328186035,
          29.900341033935547,
          -2.977961301803589,
          -10.684250831604004,
          -0.7481951117515564,
          -9.375555992126465,
          -7.592347145080566,
          -5.384964466094971,
          -11.123490333557129,
          3.7723422050476074,
          -7.92080545425415,
          -3.816434144973755,
          -3.014146566390991,
          -12.37424087524414,
          -3.4395761489868164,
          0.45067960023880005,
          -16.808639526367188,
          -0.5904219150543213,
          -12.80771255493164,
          -9.528319358825684,
          -16.24529457092285,
          -20.70800018310547,
          1.7105482816696167,
          -15.445150375366211,
          -21.231489181518555,
          -13.05862808227539,
          -21.392173767089844,
          -21.787492752075195,
          -20.058073043823242,
          -22.46089744567871,
          -21.46479034423828,
          -24.761760711669922,
          5.323232173919678,
          -18.77907943725586,
          -23.827512741088867,
          -21.044357299804688,
          -13.219524383544922,
          -19.985458374023438,
          -17.930953979492188,
          10.91976261138916,
          6.3736677169799805,
          28.442569732666016,
          8.104130744934082,
          12.545429229736328,
          5.0339579582214355,
          9.593249320983887,
          16.77005386352539,
          1.0873000621795654,
          21.59762191772461,
          16.7332706451416,
          16.142549514770508,
          16.40657615661621,
          7.82993221282959,
          19.43091583251953,
          5.502141952514648,
          19.01779556274414,
          13.824527740478516,
          13.633443832397461,
          6.748111248016357,
          -2.331172466278076,
          -0.638937771320343,
          11.6112642288208,
          0.7673212885856628,
          -11.544002532958984,
          4.23929500579834,
          -15.741869926452637,
          -4.756656169891357,
          2.927236557006836,
          3.082214117050171,
          -6.556437969207764,
          6.693364143371582,
          14.146547317504883,
          -26.156150817871094,
          1.5376991033554077,
          2.9968338012695312,
          -6.749704360961914,
          -7.750785827636719,
          8.707465171813965,
          -4.874820709228516,
          3.5443265438079834,
          -17.866634368896484,
          -5.809909343719482,
          -4.849856376647949,
          -11.52678108215332,
          -10.9984712600708,
          8.994112968444824,
          4.765151500701904,
          -10.40442180633545,
          -4.0974578857421875,
          2.20236873626709,
          -0.05857357010245323,
          -25.381254196166992,
          -23.87941551208496,
          -5.818136692047119,
          -17.36553955078125,
          1.162163496017456,
          -17.92291259765625,
          -2.7773709297180176,
          -17.787765502929688,
          -6.031368732452393,
          3.963510274887085,
          -7.664010524749756,
          -10.580368041992188,
          6.711097240447998,
          -3.2372989654541016,
          13.913147926330566,
          -6.788234233856201,
          14.916961669921875,
          8.050895690917969,
          8.665105819702148,
          -0.8336024880409241,
          8.889707565307617,
          14.90350341796875,
          -4.366927623748779,
          0.3667706847190857,
          -4.178377151489258,
          6.592097759246826,
          11.740421295166016,
          5.824801445007324
         ]
        }
       ],
       "layout": {
        "template": {
         "data": {
          "bar": [
           {
            "error_x": {
             "color": "#2a3f5f"
            },
            "error_y": {
             "color": "#2a3f5f"
            },
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             }
            },
            "type": "bar"
           }
          ],
          "barpolar": [
           {
            "marker": {
             "line": {
              "color": "#E5ECF6",
              "width": 0.5
             }
            },
            "type": "barpolar"
           }
          ],
          "carpet": [
           {
            "aaxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "baxis": {
             "endlinecolor": "#2a3f5f",
             "gridcolor": "white",
             "linecolor": "white",
             "minorgridcolor": "white",
             "startlinecolor": "#2a3f5f"
            },
            "type": "carpet"
           }
          ],
          "choropleth": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "choropleth"
           }
          ],
          "contour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "contour"
           }
          ],
          "contourcarpet": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "contourcarpet"
           }
          ],
          "heatmap": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmap"
           }
          ],
          "heatmapgl": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "heatmapgl"
           }
          ],
          "histogram": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "histogram"
           }
          ],
          "histogram2d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2d"
           }
          ],
          "histogram2dcontour": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "histogram2dcontour"
           }
          ],
          "mesh3d": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "type": "mesh3d"
           }
          ],
          "parcoords": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "parcoords"
           }
          ],
          "pie": [
           {
            "automargin": true,
            "type": "pie"
           }
          ],
          "scatter": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatter"
           }
          ],
          "scatter3d": [
           {
            "line": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatter3d"
           }
          ],
          "scattercarpet": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattercarpet"
           }
          ],
          "scattergeo": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergeo"
           }
          ],
          "scattergl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattergl"
           }
          ],
          "scattermapbox": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scattermapbox"
           }
          ],
          "scatterpolar": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolar"
           }
          ],
          "scatterpolargl": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterpolargl"
           }
          ],
          "scatterternary": [
           {
            "marker": {
             "colorbar": {
              "outlinewidth": 0,
              "ticks": ""
             }
            },
            "type": "scatterternary"
           }
          ],
          "surface": [
           {
            "colorbar": {
             "outlinewidth": 0,
             "ticks": ""
            },
            "colorscale": [
             [
              0,
              "#0d0887"
             ],
             [
              0.1111111111111111,
              "#46039f"
             ],
             [
              0.2222222222222222,
              "#7201a8"
             ],
             [
              0.3333333333333333,
              "#9c179e"
             ],
             [
              0.4444444444444444,
              "#bd3786"
             ],
             [
              0.5555555555555556,
              "#d8576b"
             ],
             [
              0.6666666666666666,
              "#ed7953"
             ],
             [
              0.7777777777777778,
              "#fb9f3a"
             ],
             [
              0.8888888888888888,
              "#fdca26"
             ],
             [
              1,
              "#f0f921"
             ]
            ],
            "type": "surface"
           }
          ],
          "table": [
           {
            "cells": {
             "fill": {
              "color": "#EBF0F8"
             },
             "line": {
              "color": "white"
             }
            },
            "header": {
             "fill": {
              "color": "#C8D4E3"
             },
             "line": {
              "color": "white"
             }
            },
            "type": "table"
           }
          ]
         },
         "layout": {
          "annotationdefaults": {
           "arrowcolor": "#2a3f5f",
           "arrowhead": 0,
           "arrowwidth": 1
          },
          "coloraxis": {
           "colorbar": {
            "outlinewidth": 0,
            "ticks": ""
           }
          },
          "colorscale": {
           "diverging": [
            [
             0,
             "#8e0152"
            ],
            [
             0.1,
             "#c51b7d"
            ],
            [
             0.2,
             "#de77ae"
            ],
            [
             0.3,
             "#f1b6da"
            ],
            [
             0.4,
             "#fde0ef"
            ],
            [
             0.5,
             "#f7f7f7"
            ],
            [
             0.6,
             "#e6f5d0"
            ],
            [
             0.7,
             "#b8e186"
            ],
            [
             0.8,
             "#7fbc41"
            ],
            [
             0.9,
             "#4d9221"
            ],
            [
             1,
             "#276419"
            ]
           ],
           "sequential": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ],
           "sequentialminus": [
            [
             0,
             "#0d0887"
            ],
            [
             0.1111111111111111,
             "#46039f"
            ],
            [
             0.2222222222222222,
             "#7201a8"
            ],
            [
             0.3333333333333333,
             "#9c179e"
            ],
            [
             0.4444444444444444,
             "#bd3786"
            ],
            [
             0.5555555555555556,
             "#d8576b"
            ],
            [
             0.6666666666666666,
             "#ed7953"
            ],
            [
             0.7777777777777778,
             "#fb9f3a"
            ],
            [
             0.8888888888888888,
             "#fdca26"
            ],
            [
             1,
             "#f0f921"
            ]
           ]
          },
          "colorway": [
           "#636efa",
           "#EF553B",
           "#00cc96",
           "#ab63fa",
           "#FFA15A",
           "#19d3f3",
           "#FF6692",
           "#B6E880",
           "#FF97FF",
           "#FECB52"
          ],
          "font": {
           "color": "#2a3f5f"
          },
          "geo": {
           "bgcolor": "white",
           "lakecolor": "white",
           "landcolor": "#E5ECF6",
           "showlakes": true,
           "showland": true,
           "subunitcolor": "white"
          },
          "hoverlabel": {
           "align": "left"
          },
          "hovermode": "closest",
          "mapbox": {
           "style": "light"
          },
          "paper_bgcolor": "white",
          "plot_bgcolor": "#E5ECF6",
          "polar": {
           "angularaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "radialaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "scene": {
           "xaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "yaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           },
           "zaxis": {
            "backgroundcolor": "#E5ECF6",
            "gridcolor": "white",
            "gridwidth": 2,
            "linecolor": "white",
            "showbackground": true,
            "ticks": "",
            "zerolinecolor": "white"
           }
          },
          "shapedefaults": {
           "line": {
            "color": "#2a3f5f"
           }
          },
          "ternary": {
           "aaxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "baxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           },
           "bgcolor": "#E5ECF6",
           "caxis": {
            "gridcolor": "white",
            "linecolor": "white",
            "ticks": ""
           }
          },
          "title": {
           "x": 0.05
          },
          "xaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          },
          "yaxis": {
           "automargin": true,
           "gridcolor": "white",
           "linecolor": "white",
           "ticks": "",
           "title": {
            "standoff": 15
           },
           "zerolinecolor": "white",
           "zerolinewidth": 2
          }
         }
        },
        "title": {
         "text": "prod cat projection"
        }
       }
      },
      "text/html": [
       "<div>\n",
       "        \n",
       "        \n",
       "            <div id=\"28fc782b-a989-402c-b253-708cca81caed\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>\n",
       "            <script type=\"text/javascript\">\n",
       "                require([\"plotly\"], function(Plotly) {\n",
       "                    window.PLOTLYENV=window.PLOTLYENV || {};\n",
       "                    \n",
       "                if (document.getElementById(\"28fc782b-a989-402c-b253-708cca81caed\")) {\n",
       "                    Plotly.newPlot(\n",
       "                        '28fc782b-a989-402c-b253-708cca81caed',\n",
       "                        [{\"hovertext\": [\"Risk Factors Avian Influenza Infection Chickens Live Bird<br>Retail Stalls Lahore District Pakistan 2009\\u20132010 This study<br>conduct identify risk factor associate AIV infection live<br>bird retail stall LBRS Lahore District Pakistan cross<br>sectional survey LBRS conduct December February use two<br>stage cluster sample base probability proportional size total<br>oropharyngeal swab sample pool collect bird cluster test<br>qRT PCR matrix gene type influenza virus gene<br>subtypes Thirty four sample positive gene also positive<br>sample find positive Data potential risk factor collect<br>questionnaire analyze survey weight logistic regression prevalence odds<br>ratio associate risk factor calculate final multivariable model<br>identify three risk factor infection LRBS namely obtain<br>bird mixed source 4\\u20133 keep bird outside cage<br>4\\u20137 keep chicken breed broiler 7\\u201323 Sourcing bird<br>dealer wholesaler keep bird inside cage avoid mix<br>different breed cage could reduce risk infection LRBS<br>\", \"Pulsed electromagnetic field stimulation prevents steroid induced osteonecrosis<br>rat BACKGROUND Pulsed electromagnetic field PEMF stimulation use<br>successfully treat nonunion fracture femoral head osteonecrosis relatively<br>little known effect prevent steroid induce osteonecrosis The<br>purpose study investigate effect PEMF stimulation prevention steroid<br>induce osteonecrosis rat explore underlying mechanism METHODS Seventy<br>two male adult Wistar rat divide three group<br>treat follow PEMF stimulation group PEMF group intravenously<br>inject lipopolysaccharide LPS day intramuscularly inject methylprednisolone acetate<br>MPSL day subject PEMF stimulation per day week<br>Methylprednisolone treat group MPSL group inject dose LPS<br>MPSL PEMF group without exposure PEMF Control group<br>group inject saline mode time point The incidence<br>osteonecrosis serum lipid level mRNA protein expression transform<br>growth factor TGF proximal femur measure week last<br>MPSL saline injection RESULTS The incidence osteonecrosis PEMF<br>group significantly low observed MPSL group osteonecrosis observe<br>group The serum lipid level significantly low PEMF<br>group MPSL group Compared MPSL group mRNA expression<br>TGF increase reach peak week PEMF treatment remain<br>high week decline week whereas protein expression TGF<br>increase reach peak week PEMF treatment remain high<br>week CONCLUSIONS PEMF stimulation prevent steroid induce osteonecrosis<br>rat underlie mechanism involve decrease serum lipid level<br>increase expression TGF<br>\", \"Different inflammatory response associate Ureaplasma parvum induce UTI<br>urolith formation BACKGROUND Epidemiologic study show strong association<br>Ureaplasmas urogenital tract disease human Since healthy human<br>colonize Ureaplasmas role pathogen remain controversial order begin<br>define role host disease develop rodent model urinary<br>tract infection UTI use Fischer F344 rat Animals<br>inoculate sterile broth log CFU rat adapt strain<br>Ureaplasma parvum RESULTS Infected animal exhibit two distinct<br>profile asymptomatic UTI UTI complicate struvite urolithiasis Inoculum<br>dose parvum affect incidence UTI animal inoculate CFU<br>parvum remain infected However inoculum dose influence immune<br>response parvum Asymptomatic UTI characterize minimal immune response<br>predominantly monocytic lymphocytic limited lesion elevate urinary level<br>IFN MCP UTI complicate struvite formation characterize exaggerated<br>immune response mostly neutrophilic lesion show extensive uroepithelial<br>hyperplasia predominance GRO urine Animals asymptomatic UTI also<br>significantly high rate kidney infection CONCLUSION Complications associate<br>parvum infection primarily dependent upon host specific factor<br>rather Ureaplasma microbial load The immune response F344<br>rat similar occurs human ureaplasmal associate disease Therefore<br>model infection useful tool elucidate parvum host interaction<br>confer UTI disease<br>\", \"longitudinal study bovine coronavirus enteric respiratory infection dairy<br>calf two herd Ohio This prospective longitudinal study<br>examine epidemiology disease syndrome associate bovine coronavirus BCV<br>infection cohort conventional calf day age two dairy<br>herd Ohio The period respiratory shed BCV determine<br>direct immunofluorescent DIF stain nasal epithelial cell ELISA<br>nasal swab supernatant fluids The period fecal shed<br>BCV determine ELISA immunoelectron microscopy IEM The isotype<br>specific antibody titer BCV serum select interval day<br>age post suckle birth total immunoglobulin level examine<br>ELISA zinc sulfate turbidity test respectively calf study<br>evidence BCV respiratory DIF ELISA enteric infection IEM<br>ELISA association diarrhea rhinitis even though calf show<br>increase one serum antibody isotypes BCV calf show<br>BCV respiratory enteric antigen shed ELISA Serological antibody<br>titer increase occur calf day age calf day<br>age two latter calf second rise serum antibody<br>titer BCV initial rise serological antibody titer increase<br>observe one calf This suggest BCV infection may<br>common close herd may occur old calf although<br>many may subclinical may recurrent There statistically significant<br>correlation total serum immunoglobulin level BCV antibody isotype<br>titer serum 24\\u201348 birth clinical disease infection BCV<br>however calve low level IgA BCV antibodies serum<br>24\\u201348 birth significantly great average number day diarrhea<br>calf high level IgA BCV specific antibody serum<br>\", \"Characterization enterotoxigenic coli ETEC Shiga toxin produce coli<br>STEC necrotoxigenic coli NTEC isolate diarrhoeic Mediterranean water<br>buffalo calve Bubalus bubalis Two hundred twenty Escherichia<br>coli isolate Mediterranean water buffalo calve less 4weeks<br>old affect severe diarrhoea lethal outcome characterize presence<br>virulence factor Stx1 Stx2 haemolysins intimin CNF1 CNF2<br>CDT CDT CDT III CDT F17 relate fimbriae<br>F17a F17b F17c F17d The prevalence ETEC STEC<br>NTEC respectively The ETEC isolate positive negative The<br>STEC isolate Stx intimin positive Stx1 frequent Stx2<br>The NTEC isolate CNF Hly positive CNF2 frequent<br>CNF1 Susceptibility assay antimicrobial display high rate resistance<br>antimicrobial test These data show prevalent strain diarrhoeic<br>water buffalo calf NTEC mostly CNF2 HlyA positive<br>strong association CNF2 CDT III CNF2 F17c<br>\", \"Chapter Infectious Diseases Domestic Rabbits<br>\", \"Galactose phosphate uridyltransferase GalT vivo induce antigen Actinobacillus<br>pleuropneumoniae serovar strain L20 provide immunoprotection serovar strain<br>MS71 GALT important antigen Actinobacillus pleuropneumoniae APP show<br>provide partial protection APP infection previous study lab<br>The main purpose present study investigate GALT induce<br>cross protection different APP serotypes elucidate key mechanism<br>immune response GALT antigenic stimulation Bioinformatic analysis demonstrate<br>galT highly conserve gene APP widely distribute across<br>multiple pathogenic strain Homologies two strain range regard<br>galT locus Indirect enzyme link immunosorbent assay ELISA<br>confirm GALT specific antibody could induce inactivated APP<br>L20 MS71 whole cell bacterin preparations recombinant fusion<br>GALT protein derive APP L20 however proven effective<br>cross protective antigen APP sevorar MS71 APP sevorar<br>L20 Histopathological examination confirm recombinant GALT vaccinate animal<br>show less severe pathological sign lung tissue negative<br>control APP challenge Immunohistochemical IHC analysis indicate infiltration<br>neutrophil negative group significantly increase compare normal control<br>surviving animal decrease compare negative group Anti GALT<br>antibodies show mediate phagocytosis neutrophil After interaction anti<br>GALT antibodies survival rate APP challenge vaccinated animal<br>significantly reduce This study demonstrate GALT effective cross<br>protective antigen could use potential vaccine candidate multiple<br>APP serotypes<br>\", \"Multicausal etiology enteric syndrome rabbit Mexico Enteropathies rabbit<br>difficult diagnose etiology involve pathogen act synergistically causing<br>damage intestine The aim present study isolate enteric<br>pathogens rabbit Mexico Using parasitological bacteriological molecular analysis<br>screen sample intestinal content rabbit clinical history enteric<br>disease southeastern part State Mexico Out sample analyze<br>total identification make Eimeria spp find follow Aeromonas<br>spp Escherichia coli identify enteropathogenic coli EPEC presence<br>follow agent also confirm Salmonella spp Klebsiella spp<br>Streptococcus spp Staphylococcus aureus Enterococcus spp Mannheimia spp<br>Rotavirus The concurrent presence Eimeria spp Aeromonas frequent<br>statistical significance presence association clinical profile Eimeria spp<br>Mannheimia spp Salmonella spp Klebsiella spp Streptococcus spp<br>Enterococcus spp<br>\", \"Enteric coronavirus infection adult horse new enteric virus<br>adult horse equine coronavirus ECoV recently recognize associate<br>fever lethargy anorexia less frequently colic diarrhea This<br>enteric virus transmit via feco oral route horse<br>become infected ingest fecally contaminate feed water Various<br>outbreaks report since Japan Europe USA While clinical<br>sign fairly non specific lymphopenia neutropenia often see<br>Specific diagnosis make detection ECoV feces either quantitative<br>real time PCR electron microscopy antigen capture ELISA<br>Supportive treatment usually require infection self limit However<br>rare complication endotoxemia septicemia hyperammonemia associate encephalopathy report<br>related loss barrier function intestinal mucosa This review<br>article focus late information pertain virus epidemiology clinical<br>sign diagnosis pathology treatment prevention ECoV infection adult<br>horse<br>\", \"super spreading ewe infect hundred fever farmer market<br>Germany BACKGROUND May Soest County Health Department inform<br>unusually large number patient hospitalized atypical pneumonia METHODS<br>exploratory interview patient mention visited farmer market sheep<br>lamb Serologic test confirm diagnosis fever ask local<br>health department Germany identiy notify fever patient visit<br>farmer market investigate risk factor infection conduct case<br>control study case fever patient control randomly select<br>Soest citizen cohort study among vendor market The<br>sheep exhibit market herd originate well sheep herd<br>hold vicinity Soest test Coxiella burnetii burnetii RESULTS<br>total report fever case link outbreak The mean<br>incubation period day interquartile range 16\\u201324 day The<br>case control study identify close proximity stop least<br>second sheep pen significant risk factor Vendors within<br>approximately meter sheep pen increase risk disease compare<br>locate far away Wind play significant role The<br>clinical attack rate adult child estimate respectively case<br>hospitalize The ewe lamb well herd test positive<br>burnetii antibody CONCLUSION Due size point source nature<br>outbreak permit assessment fundamental seldom study epidemiological parameter<br>consequence outbreak recommend pregnant sheep display public trimester<br>test animal pet zoos regularly burnetii<br>\", \"Longitudinal study morbidity mortality white veal calf Belgium<br>BACKGROUND Mortality morbidity hardly document white veal industry<br>despite high level antimicrobial drug use resistance The<br>objective present study determine cause epidemiology morbidity mortality<br>dairy beef crossbreed white veal production total calf<br>house production cohort follow one production cycle Causes<br>mortality determine necropsy Morbidity daily record producer RESULTS<br>The total mortality risk significantly high beef veal<br>production compare dairy crossbreds The main cause mortality<br>pneumonia calf risk ruminal disorder idiopathic peritonitis enterotoxaemia<br>enteritis Belgian Blue beef calf likely die pneumonia<br>enterotoxaemia arthritis Detection bovine viral diarrhea virus necropsy<br>associate chronic pneumonia pleuritis calf treat individually morbidity<br>rate case per calf day risk The incidence<br>rate respiratory disease diarrhea arthritis otitis case per<br>calf day risk respectively Morbidity peak first three<br>week arrival gradually decline towards end production cycle<br>CONCLUSIONS The present study provide insight cause epidemiology<br>morbidity mortality white veal calf Belgium house frequent<br>housing system Europe The necropsy finding identify risk<br>period difference production system guide veterinarians producer towards<br>profitable ethical preventive therapeutic protocol<br>\", \"vivo growth porcine reproductive respiratory syndrome virus engineer<br>nsp2 deletion mutant Prior study PRRSV strain non<br>structural protein nsp2 show much amino acid could<br>remove hypervariable region without lose virus viability vitro<br>utilize select nsp2 deletion mutant examine vivo growth<br>Young swine pig group control swine inoculate intramuscularly<br>one nsp2 deletion mutant r\\u0394727\\u2013813 r\\u0394543\\u2013726 r\\u0394324\\u2013523 r\\u0394324\\u2013726<br>full length recombinant virus rVR Serum sample collect<br>various day post inoculation analyze HerdChek ELISA PRRSV<br>real time PCR gamma interferon IFN ELISA nucleotide<br>sequence analysis entire nsp2 cod region Tracheobronchial lymph<br>node weight compare body weight record animal use<br>clinical measurement viral pathogenesis Results show deletion mutant<br>grow less robustly full length recombinant virus yet<br>large deletion virus r\\u0394324\\u2013726 recover parental viral RNA<br>level study end Swine receive r\\u0394727\\u2013813 mutant significant<br>decrease lymph node enlargement compare rVR While swine<br>infection rVR cause rapid rise serum IFN level<br>IFN protein produce infection deletion mutant viruses significantly<br>reduce perhaps due difference viral growth kinetics The<br>r\\u0394543\\u2013726 nsp2 mutant virus although growth impair mimic<br>rVR induce host serum IFN response exhibit week<br>delay Targeted sequence showed deletion stable region cod<br>nsp2 one swine passage The data suggest select<br>nsp2 deletion mutant growth attenuate swine alter induction<br>serum IFN innate cytokine unknown function PRRSV clearance<br>point domain may influence tracheobronchial lymph node size<br>\", \"Differential expression miR collapse steroid induce osteonecrosis femoral<br>head BACKGROUND Femoral head collapse key reference point<br>determine treatment regimen femoral head osteonecrosis However effective<br>preventive measure efficacy hip preserving surgery unsatisfactory due<br>unclear mechanism collapse This study aim identify validate<br>miRNAs differentially express collapse non collapse area osteonecrotic<br>femoral head predict target gene pathway miRNAs RESULTS<br>Nine sample pass quality control test total differentially<br>express miRNAs detect among miRNAs show differential expression<br>group compare group miRNAs show differential expression group<br>compare group miRNAs show differential expression group compare<br>group After combine data three patient miRNAs show<br>differential expression collapse area compare non collapse area<br>Compared normal area miR show significant downregulation Expression<br>result PCR reveal expression hsa miR collapse area<br>significantly low non collapse area normal area gene<br>perdicted target gene hsa miR MATERIALS AND METHODS<br>Femoral head three patient male female treat total<br>hip arthroplasty surgery steroid induce femoral head osteonecrosis<br>select base inclusion exclusion criterion Bone tissue sample<br>obtain collapse area non collapse area normal area<br>accord anatomical structure osteonecrotic femoral head Total RNA<br>extract sample microarray chip scan miRNAs show differential<br>expression fold select validated PCR TargetScan mirBase miRanda<br>bioinformatics software use predict target gene identify possible<br>pathway involve gene CONCLUSIONS miR show significant difference<br>collapse area osteonecrotic femoral head suggest collapse may<br>relate downregulation miR<br>\", \"Cryptosporidium infection veal calf cohort France molecular characterization<br>specie longitudinal study Feces animal collect farm region<br>Brittany France Each sample directly collect rectum animal<br>identify ear tag number Animals sample three time<br>week age After DNA extraction stool sample nest<br>PCR perform amplify partial 18S rDNA kDa glycoprotein<br>gene Cryptosporidium The parasite detect farm One hundred<br>calf find parasitized Cryptosporidium Amplified fragment sequence Cryptosporidium<br>specie identification reveal presence parvum ryanae bovis One<br>animal infect Cryptosporidium ubiquitum The prevalence specie relate<br>age animal parvum cause Cryptosporidium infection week old<br>calve week old animal The analysis result show<br>animal could infect successively parvum ryanae bovis study<br>period parvum gp60 genotyping identifies IIa subtypes represent<br>IIaA15G2R1 This work confirm previous study country show<br>zoonotic parvum dominant specie see young calf<br>\", \"Pyogranulomatous Pleuropneumonia Mediastinitis Ferrets Mustela putorius furo associate<br>Pseudomonas luteola Infection Summary Between three pet ferret<br>different source present acute episode dyspnoea Cytological examination<br>pleural exudate reveal severe purulent inflammation abundant cluster<br>rod shaped microorganism clear surround halo Treatment ineffective<br>ferret die 2\\u20135 day later Two ferret subject<br>necropsy examination reveal pyothorax mediastinal lymphadenopathy multiple white<br>nodule 1\\u20132mm lung Microscopical examination show multifocal necrotizing<br>pyogranulomatous pleuropneumonia lymphadenitis aggregate encapsulate microorganisms positively stain<br>periodic acid\\u2013Schiff alcian blue situ hybridization Pneumocystis spp<br>Ziehl\\u2013Neelsen stain immunohistochemistry distemper coronavirus influenza antigen negative<br>case Electron microscopically bacteria 2\\u20133\\u03bcm long thick electron<br>lucent capsule Microbiology one ferret yield pure culture<br>gram negative bacteria identify phenotypically Pseudomonas luteola This<br>speciation later confirm 16S RNA gene amplification<br>\", \"Respiratory Pathogens Qu\\u00e9bec Dairy Calves Their Relationship Clinical<br>Status Lung Consolidation Average Daily Gain BACKGROUND Bovine<br>respiratory disease BRD important cause morbidity mortality dairy<br>calve Surprisingly field data scant concern prevalence respiratory<br>pathogen involve BRD preweaned dairy calf especially small<br>herd OBJECTIVES identify main respiratory pathogen isolate calf<br>Qu\\u00e9bec dairy herd high incidence BRD determine association<br>presence pathogens clinical sign pneumonia lung consolidation average<br>daily gain ANIMALS Cross\\u2010sectional study use convenience sample<br>preweaned dairy calf dairy herd METHODS enrollment calf<br>weigh clinically examine swab nasal nasopharyngeal lung ultrasonography<br>perform One month later calf reweighed RESULTS Twenty\\u2010two<br>calve clinical BRD ultrasonographic evidence lung consolidation Pasteurella<br>multocida Mannheimia haemolytica Histophilus somni isolate calf respectively<br>Mycoplasma bovis identify PCR test culture calf calf<br>find positive Mycoplasma spp Bovine coronavirus detect calf<br>bovine respiratory syncytial virus Only presence bovis associate<br>high odds clinical sign lung consolidation low average<br>daily gain CONCLUSIONS AND CLINICAL IMPORTANCE Results suggest<br>nasopharyngeal carriage bovis detrimental health growth dairy calf<br>small herd high incidence BRD<br>\", \"Investigation Pathology Pathogens Associated Porcine Respiratory Disease Complex<br>Denmark Summary Respiratory infection among important disease grow<br>pig order elucidate multifactorial aetiology porcine respiratory disease<br>complex PRDC Denmark lungs finishing pig cranioventral bronchopneumonia<br>case group pig without lung lesion control group<br>collect abattoirs The pathogen involve PRDC interaction identify<br>link histopathological diagnosis The lung sample culture bacteria<br>test multiplex polymerase chain reaction presence swine influenza<br>virus type porcine reproductive respiratory syndrome virus European<br>type porcine circovirus type PCV2 porcine respiratory coronavirus<br>porcine cytomegalovirus Mycoplasma hyopneumoniae Mycoplasma hyorhinis All case<br>cranioventral lobular bronchopneumonia consistent PRDC There broad range<br>microscopical lesion case characterize acute subacute chronic bronchopneumonia<br>Five bacterial specie five virus two Mycoplasma spp<br>detect different combination PCV2 hyopneumoniae hyorhinis Pasteurella multocida<br>detect frequently among PRDC affect swine diversity number<br>pathogens high animal compare control clear cut association<br>detect pathogen histological lesion histopathological diagnosis PRDC occur<br>frequently enzootic pneumonia among Danish finishing pig complex<br>vary histopathology<br>\", \"Multidose intramuscular allogeneic adipose stem cell decrease severity<br>canine atopic dermatitis pilot study AIM The aim<br>pilot study evaluate therapeutic safety performance intramuscular treatment<br>protocol multidose allogeneic adipose stem cell ASCs isolate<br>characterize expanded vivo healthy canine donor MATERIALS AND<br>METHODS Twelve dog diagnose canine atopic dermatitis CAD<br>intramuscularly treat 5\\u00d710 cryopreserved ASCs healthy immunize young<br>canine Ehrlichia canis free donor weekly week Treatment<br>efficacy evaluate pruritus index CAD Lesion Index CADLI<br>test Safety adverse effect determine injection site reaction<br>weight blood chemistry liver function whole blood count<br>RESULTS Canine ASCs obtain donor meet minimum quality<br>require type cell show viability thaw The efficacy<br>CADLI score pruritus index dog atopic dermatitis statistically<br>significant efficacy adverse reaction observe intramuscular application site<br>relation animal weight blood cell population liver renal<br>function CONCLUSION These result suggest intramuscular administration cryopreserved<br>ASCs dog atopic dermatitis promise cellular therapeutic product<br>relief symptom disease however duration effect obtain dose<br>dos evaluate well possible immune reaction far know<br>first report use multiple intramuscular dos cryopreserved ASCs<br>treat atopic dermatitis<br>\", \"Strategies control PRRS summary field research experience Various<br>method control PRRS virus publish The technology nursery<br>depopulation appear effectively control spread virus member endemically<br>infected population consist strategic adjustment pigflow base presence<br>specific serologic pattern detect indirect fluorescent antibody test<br>This pattern indicate low seroprevalence antibody detect breed<br>herd recently wean piglet contrast high seroprevalence week<br>old piglet carry swine farm result indicate improvement<br>nursery piglet growth rate mortality level Three example<br>provide follow text Recently modify live virus vaccine<br>RespPRRS NOBL Laboratories Boerhinger Ingleheim become commercially available<br>currently approve use piglets week age however potential<br>use adult animal currently investigation<br>\", \"Bovine respiratory syncytial virus outbreak reduce bull weight<br>gain feed conversion eight month Norwegian beef herd<br>BACKGROUND Cost benefit evaluation measure respiratory disease cattle<br>require account associate production loss Investigations naturally occur<br>respiratory infection herd set opportunity accurate estimate consequence<br>This article present estimate base individual monitoring weight<br>concentrate intake several hundred bull previous respiratory infection<br>outbreak bovine respiratory syncytial virus BRSV main pathogen<br>The aim study analyse association exposure BRSV weight<br>gain feed conversion rate quantify change parameter estimate<br>duration change production RESULTS comparison growth curve bull<br>present outbreak reveal bull severe clinical sign clear<br>consistent trend poor growth rate milder sign The<br>weight age ratio 04\\u20130 lower severely affected bull<br>evident throughout study period month comparison growth rate<br>apparently healthy bull present outbreak comparable group bull<br>exactly year later show reduced growth rate day<br>first group The difference amount extra day need<br>reach reference weight Feed conversion also reduce weight<br>gain kilogram concentrate consume outbreak year CONCLUSION This<br>study indicate significant negative effect performance animal develop<br>severe clinical sign acute stage growth production negatively<br>affect many month apparent recovery addition performance apparently<br>healthy animal expose outbreak severely negatively affect The<br>duration decrease production animal recovery animal show disease<br>previously document These loss easily underestimate contribute significantly<br>cost producer The finding emphasize importance BRSV infection<br>profitability animal welfare cattle husbandry The study also<br>illustrate utilising intra herd comparison health production parameter<br>productive approach estimate consequence outbreak<br>\", \"Complete Genome Sequence Bordetella bronchiseptica Strain KM22 Bordetella<br>bronchiseptica isolate KM22 use experimental infection swine model<br>clinical bronchiseptica infection study host host transmission The<br>draft genome sequence KM22 report Here report complete<br>genome sequence KM22<br>\", \"Expression Dynamics Innate Immunity Influenza Virus Infected Swine<br>The current circulating swine influenza virus subtypes Europe<br>H1N1 H1N2 H3N2 associate clinical outbreak disease However<br>show pig could susceptible strain able cross specie<br>barrier work extend investigation whether different strain able<br>cross specie barrier might give rise different innate<br>immune response could associate pathological lesion For purpose<br>use sample collect previous study healthy pig infect<br>H3N2 Swine four different H3N8 strain circulate different<br>animal specie Pigs clinically inspect four subject group<br>sacrifice day post infection present study group mock<br>exhibit antibody response nucleoprotein protein Pulmonary lesion high<br>titered viral replication observe pig infect swine adapt<br>virus Interestingly pig infect avian seal H3N8 strain<br>also show moderate lesion viral replication whereas equine<br>canine IVs cause overt pathological sign replication barely<br>detectable Swine infection induce interferon IFN alpha interleukin<br>response bronchoalveolar fluid BALF day post infection oppose<br>non swine adapt virus strain However IFN alpha<br>response swine adapt virus associate increase local constitutive<br>expression IFN alpha gene Remarkably Equine strain give<br>rise Serum Amyloid response BALF despite little replication<br>Each virus strain could associate expression cytokine gene<br>proteins infection These response observe well beyond period<br>virus replication suggest prolonged homeostatic imbalance innate immune<br>system<br>\", \"Health zoonotic Infections snow leopard Panthera unica South<br>Gobi desert Mongolia Background Snow leopards Panthera uncia<br>threaten apex predator scatter across mountain Central South<br>Asia Disease threats wild snow leopard investigate Methods<br>Results Between twenty snow leopard South Gobi desert<br>Mongolia capture immobilised health screen radio collar Blood<br>sample external parasite collect pathogen analysis use enzyme<br>link immunosorbent assay ELISA microscopic agglutination test MAT<br>next generation sequence NGS techniques The animal show<br>clinical sign disease however serum antibody significant zoonotic<br>pathogen detect These pathogen include Coxiella burnetii prevalence<br>Leptospira spp Toxoplasma gondii Ticks collect snow leopard<br>contain potentially zoonotic bacteria genus Bacillus Bacteroides Campylobacter<br>Coxiella Rickettsia Staphylococcus Streptococcus Conclusions The zoonotic pathogen<br>identify study short term appear cause illness snow<br>leopard cause illness wild felid Therefore surveillance pathogen<br>implement monitor potential long term disease impact snow<br>leopard population<br>\", \"Identification Novel Hypovirulence Inducing Hypovirus From Alternaria alternata<br>Mycoviruses wide spread throughout almost group fungi small<br>number mycoviruses attenuate growth virulence fungal host Alternaria<br>alternata ascomycete fungus cause leaf spot diseases various<br>crop plant study identify novel ssRNA mycovirus infect<br>alternata mali strain isolate apple orchard China Sequence<br>analyse reveal virus related hypoviruses particular Wuhan insect<br>virus unclassified hypovirus identify insect meta transcriptomics well<br>hypoviruses belong genus Hypovirus therefore virus design Alternaria<br>alternata hypovirus AaHV1 The genome AaHV1 contain single<br>large open reading frame encode putative polyprotein \\u223c479<br>kDa cysteine proteinase like replication associate domain Curing<br>AaHV1 fungal host strain indicate virus responsible slow<br>growth reduce virulence host AaHV1 defective RNA RNA<br>internal deletion emerge fungal subculture presence RNA affect<br>AaHV1 accumulation pathogenicities Moreover AaHV1 could replicate confer<br>hypovirulence Botryosphaeria dothidea fungal pathogen apple white rot<br>disease This finding could facilitate good understand alternata<br>pathogenicity relevant development biocontrol method fungal disease<br>\", \"new generic real time reverse transcription polymerase chain<br>reaction assay vesiviruses vesiviruses detect human sample Different<br>virus belong genus Vesivirus infect broad range animal<br>cause gastroenteritis vesicular skin lesion hemorrhagic disease respiratory<br>disease condition recent report Vesivirus viremia detect PCR<br>sample patient hepatitis unknown etiology USA suggest zoonotic<br>potential vesiviruses These result confirm another laboratory order<br>generic PCR assay RNA polymerase region develop validate<br>RNA different Vesivirus specie Except SMSV serotype specie<br>test detect include one suggest involved zoonotic transmission<br>USA SMSV serotype The generic Vesivirus assay use<br>RNA extract serum sample patient hepatitis stool sample<br>patient gastroenteritis throat swab specimens patient rash illness<br>throat swab nose swabs patient acute respiratory disease<br>cell culture cytopathologic effect enterovirus surveillance pathogen find<br>None find positive study generic Vesivirus assay develop<br>concluded vesiviruses unlikely cause common illness human Netherlands<br>\", \"Association Mannose bind Lectin Polymorphisms Tuberculosis Susceptibility among<br>Chinese Tuberculosis cause infection Mycobacterium tuberculosis Host genetic<br>variability important determinant risk develop human Although association<br>MBL2 polymorphism study various population result controversial study<br>four functional single nucleotide polymorphism SNPs across MBL2<br>gene genotyped direct DNA sequence PCR product case<br>control population Chinese Han origin consist patient pulmonary<br>control find individual carry variant allele namely genotype<br>associate increased susceptibility odds ratio confidence interval 30\\u20131<br>Additionally LYPB haplotype show significant association increase risk<br>27\\u20131 global haplotype association Furthermore individual bear low<br>medium MBL expression haplotype pair increase risk 29\\u20131<br>Thus reduce expression functional MBL secondary MBL2 variant<br>may partially mediate increased susceptibility risk<br>\", \"Isolation characterization novel microsatellite locus West African leaf<br>nose bat Hipposideros aff ruber BACKGROUND Noack leaf<br>nosed bat Hipposideros ruber cryptic specie within Hipposideros<br>caffer specie complex Despite widespread distribution Africa host<br>potentially zoonotic virus genetic structure ecology ruber poorly<br>know Here describe development novel polymorphic microsatellite locus<br>facilitate investigation genetic structure FINDINGS select microsatellite sequence<br>identify high throughput sequence read PCR amplify individual<br>yield consistently amplify scorable locus The number alleles<br>per locus range two observe heterozygosity evidence linkage<br>disequilibrium observe nine marker show departure Hardy Weinberg<br>equilibrium demonstrate successful amplification two closely related specie<br>two divergent lineage caffer specie complex CONCLUSIONS These<br>new marker provide valuable tool investigate genetic structure<br>poorly understood Hipposideros caffer specie complex<br>\", \"Complete Genome Sequence Nephropathogenic Infectious Bronchitis Virus Strain<br>Isolated China Infectious bronchitis virus IBV cause tremendous<br>economic loss poultry industry Here report complete genome<br>analysis result new natural recombination nephropathogenic IBV strain<br>name SAIBK isolate Sichuan province China<br>\", \"binning tool reconstruct viral haplotype assemble contigs MotivationInfections<br>RNA viruses Influenza HIV still pose serious threat<br>human health despite extensive research viral diseases One<br>challenge produce effective prevention treatment strategy high intra<br>specie genetic diversity different strain may different biological<br>property characterize genetic diversity thus important vaccine drug<br>design Next generation sequence technology enables comprehensive characterization<br>know novel strain widely adopt sequence viral population<br>However genome scale reconstruction haplotype still challenge problem<br>particular haplotype assembly program often produce contigs rather<br>full genome mutation one gene mask phenotypic effect<br>mutation another locus cluster contigs genome scale haplotype<br>still need ResultsWe develop contig bin tool VirBin<br>cluster contigs different group group represent haplotype Commonly<br>use feature base sequence composition contig coverage can<br>not effectively distinguish viral haplotype high sequence similarity<br>heterogeneous sequence coverage RNA viruses VirBin apply prototype<br>base cluster cluster region likely contain mutation specific<br>haplotype The tool test multiple simulated sequence data<br>different haplotype abundance distribution contig size also mock<br>quasispecies sequence data The benchmark result contig binning<br>tool demonstrate superior sensitivity precision VirBin contig bin<br>viral haplotype reconstruction Availabilityhttps github com chjiao VirBinContactyannisun<br>cityu edu<br>\", \"Bat origin human coronaviruses Bats recognize natural reservoir<br>large variety virus Special attention pay bat coronaviruses<br>two emerge coronaviruses cause unexpected human disease outbreaks<br>21st century Severe Acute Respiratory Syndrome Coronavirus SARS<br>CoV Middle East Respiratory Syndrome Coronavirus MERS CoV<br>suggest originated bat Various specie horseshoe bat China<br>find harbor genetically diverse SARS like coronaviruses Some<br>strain highly similar SARS CoV even spike protein<br>able use receptor SARS CoV cell entry hand<br>diverse coronaviruses phylogenetically related MERS CoV discover worldwide<br>wide range bat specie classify coronavirus specie MERS<br>CoV Coronaviruses genetically related human coronavirus 229E NL63<br>detect bat well Moreover intermediate host believe play<br>important role transmission emergence coronaviruses bat human Understanding<br>bat origin human coronaviruses helpful prediction prevention another<br>pandemic emergence future<br>\", \"Altered pathogenicity immunogenicity tissue tropism 7kb region sequence<br>avian infectious bronchitis coronavirus strain serial passage embryo<br>study attenuate Chinese LX4 type nephropathogenic infectious bronchitis<br>virus IBV strain LHLJ 04V serial passage embryonated<br>chicken egg Based sequence analysis 7kb region LHLJ<br>04V virus population contain subpopulation mixture genetic mutant<br>The titer virus increase gradually serial passage replication<br>capacity decrease chickens The virus partially attenuate passage<br>P40 P70 fully attenuate P110 lose immunogenicity kidney<br>tropism P110 P70 respectively Amino acid substitution find<br>7kb region primarily spike protein Substitutions subunit occur<br>P40 subpopulation virus passage show substitution Other substitution<br>occur P70 P110 however find subpopulation virus passages<br>deletion UTR observe subpopulation P70 P110 might relate<br>virus replication transcription pathogenicity The change describe 7kb<br>region virus possibly responsible virus attenuation immunogenicity decrease<br>tissue tropism change however can not exclude possibility<br>part genome may also involve change<br>\", \"Alphacoronaviruses New World Bats Prevalence Persistence Phylogeny Potential<br>Interaction Humans Bats reservoirs many different coronaviruses CoVs<br>well many important zoonotic virus sample feces anal<br>swab insectivorous bat family specie different location within<br>Colorado detect alphacoronavirus RNA bat specie big brown<br>bat Eptesicus fuscus prevalence long legged bat Myotis<br>volans prevalence little brown bat Myotis lucifugus prevalence<br>western long eared bat Myotis evotis prevalence Overall<br>juvenile bat twice likely positive CoV RNA adult<br>bat two rural sample site CoV RNAs detect<br>big brown long legged bat three sequential summer<br>study CoV RNA detect big brown bat five<br>urban maternity roost sample throughout period test Individually<br>tag big brown bat positive CoV RNA later<br>sample become CoV RNA negative Nucleotide sequence RdRp<br>gene fell main cluster distinct Old World bat<br>Similar nucleotide sequence find amplicons gene spike gene<br>big brown long legged bat indicate CoV may<br>capable infect bat different genus These data suggest<br>ongoing evolution CoVs bat create possibility continue threat<br>emergence host specie Alphacoronavirus RNA detect high prevalence<br>big brown bat roost close proximity human habitation<br>know direct contact people suggest significant potential opportunity<br>exist cross specie transmission viruses Further CoV surveillance<br>study bat throughout Americas warrant<br>\", \"PEDV PDCoV Pathogenesis The Interplay Between Host Innate<br>Immune Responses Porcine Enteric Coronaviruses Enteropathogenic porcine epidemic<br>diarrhea virus PEDV porcine deltacoronavirus PDCoV member coronavirus<br>family account majority lethal watery diarrhea neonatal pig<br>past decade These two virus pose significant economic<br>public health burden even continue emerge reemerge worldwide<br>The ability evade circumvent subvert host first line<br>defense namely innate immune system key determinant pathogen<br>virulence survival establishment successful infection Unfortunately start unravel<br>underlying viral mechanism use manipulate host innate immune<br>response review gather current knowledge concern interplay viruses<br>component host innate immunity focus type interferon induction<br>signal particular mechanism virus encode gene product antagonize<br>subvert host innate immune response Finally provide perspective<br>advantage gain well understand host pathogen interaction This<br>include implication future development PEDV PDCoV vaccine knowledge<br>molecular mechanism underlie virus pathogenesis virulence host coevolution<br>\", \"Novel Picobirnaviruses Respiratory Alimentary Tracts Cattle Monkeys Large<br>Intra Inter Host Diversity Picobirnaviruses PBVs mostly find<br>animal alimentary sample study among respiratory specimen mammal<br>chicken genogroup PBVs detect three cattle three monkey<br>genogroup PBV positive sample collect one cattle specimen<br>More one PBV sequence type observe two one<br>genogroup PBV positive sample cattle monkey respectively Twenty<br>four complete near complete segment nine respiratory alimentary<br>sample cattle monkey genogroup PBVs one complete segment<br>cattle genogroup PBV sequence Similar study cattle PBVs<br>also show high diversity contrast monkey PBVs observe<br>study cluster three distinct clade Within clade sequence<br>show amino acid identities This unique phenomenon probably<br>due fact monkey locality reside separate troop minimal<br>inter troop contact<br>\", \"Host viral trait predict zoonotic spillover mammal The<br>majority human emerge infectious disease EIDs zoonotic virus<br>originate wild mammal particular concern HIV Ebola SARS<br>1\\u20133 Understanding pattern viral diversity wildlife determinant successful<br>cross specie transmission spillover therefore key goal pandemic<br>surveillance program However analytical tool exist identify host<br>specie likely harbor next human virus virus cross<br>specie boundary 5\\u20137 Here conduct comprehensive analysis yet<br>mammalian host virus relationship show total number virus<br>infect give specie proportion likely zoonotic predictable After<br>control research effort proportion zoonotic virus per specie<br>predict phylogenetic relatedness human host taxonomy human population<br>within specie range may reflect human wildlife contact<br>demonstrate first time bat harbor significantly high proportion<br>zoonotic virus mammalian order identify taxa geographic region<br>large estimate number miss virus miss zoonosis therefore<br>high value future surveillance show phylogenetic host breadth<br>viral trait significant predictor zoonotic potential providing novel<br>framework ass newly discover mammalian virus could infect<br>people<br>\", \"The Role Viral Population Diversity Adaptation Bovine Coronavirus<br>New Host Environments The high mutation rate RNA<br>viruses enables diverse genetic population viral genotype exist<br>within single infected host host genetic diversity could<br>better position virus population respond adapt diverse array<br>selective pressure host switch event Multiple new coronaviruses<br>include SARS identify human sample within last ten<br>year demonstrate potential coronaviruses emergent human pathogen Deep<br>sequencing use characterize genomic change coronavirus quasispecies simulate<br>host switching Three bovine nasal sample infect bovine<br>coronavirus use infect human bovine macrophage lung cell<br>line The virus reproduce relatively well macrophages lung<br>cell line infect efficiently enough allow passage non<br>lab adapt sample Approximately genome amplified passage sequence<br>average coverage nearly 950\\u00d7 sequence 000\\u00d7 Illumina The<br>consensus sequence many passaged sample nucleotide insert consensus<br>sequence spike gene multiple point mutation associate presence<br>insert Deep sequencing reveal insert present rare unpassaged<br>sample could quickly shift dominate population place different<br>environment The insert cod three arginine residue occur<br>region associate fusion entry host cell may allow<br>infection new cell type via heparin sulfate binding<br>Analysis deep sequencing data indicate two distinct genotype<br>circulate different frequency level sample support hypothesis mutation<br>present passaged strain select pre exist pool rather<br>novo mutation subsequent population fixation<br>\", \"Evolution Genome Size Complexity Rhabdoviridae RNA viruses exhibit<br>substantial structural ecological genomic diversity However genome size<br>RNA viruses likely limited high mutation rate result<br>evolution various mechanisms increase complexity minimise genome expansion<br>Here conduct large scale analysis genome sequence animal<br>rhabdovirus include genome determine novo identify pattern genome<br>expansion evolution genome complexity All seven rhabdovirus cluster<br>well support monophyletic group eight correspond establish genus<br>seven assign new genus two taxonomically ambiguous show<br>acquisition loss new gene appear central theme rhabdovirus<br>evolution associate appearance alternative overlap consecutive ORFs within<br>major structural protein gene insertion loss additional ORFs<br>gene junction clade specific manner Changes length gene<br>junction account much variation genome size small large<br>genome frequency new ORFs observe increase direction along<br>genome also identify several new family accessory gene<br>encode region show non canonical expression strategy involve<br>TURBS like termination reinitiation ribosomal frame shift leaky<br>ribosomal scanning appear common conclude rhabdovirus unusual capacity<br>genomic plasticity may link discontinuous transcription strategy negative<br>sense single strand RNA genome propose model account<br>regular occurrence genome expansion contraction throughout evolution Rhabdoviridae<br>\", \"Increasing diversity swine parvoviruses epidemiology African pig Detection<br>infectious viral agent increase globally advent usage sensitive<br>selective novel molecular technique epidemiological study viral disease<br>economic importance swine industry The observation different pig<br>infect member subfamily Parvovirinae family Parvoviridae application novel<br>molecular method like metagenomics bring detection many novel<br>member group Surprisingly list keep increase day day<br>possess zoonotic potential last one decade less ten<br>novel swine infecting virus add subfamily ceaseless effort<br>top gear determine occurrence prevalence old new swine<br>parvovirus herd pig produce country worldwide The story<br>however contrary African continent presently dearth information surveillance<br>initiatives virus among swine herd pig produce country<br>region Timely detection characterization viral pathogen highly imperative<br>implementation effective control prevention spread This review therefore<br>present concise overview epidemiology novel porcine parvovirus globally<br>also provide date highlight report case viral agent<br>African sub region<br>\", \"Development combine canine distemper virus specific PCR protocol<br>differentiation infect vaccinated animal DIVA genetic characterization hemagglutinin<br>gene seven Chinese strain demonstrate dog combined reverse<br>transcription polymerase chain reaction PCR method develop detection<br>differentiation wild type vaccine strain canine distemper virus<br>CDV pair primer use detect CDV wild type<br>strain vaccine Another pair use detect CDV wild<br>type strain 335bp fragment amplify genomic RNA vaccine<br>wild type strain 555bp fragment amplify specifically genomic<br>RNA wild type strain amplification achieve uninfected cell<br>cell infected canine parvovirus canine coronavirus canine adenovirus<br>The combined PCR method detect effectively differentiate CDV<br>wild type vaccine strain two separate PCRs The<br>method use clinical detection epidemiological surveillance The phylogenetic<br>analysis hemagglutinin gene local wild type CDV strain<br>reveal seven local isolates belong Asia lineage cluster<br>closely one another location These result suggest CDV<br>genotype Asia circulate currently domestic dog China<br>\", \"Detection novel circovirus PCV3 pig cardiac multi systemic<br>inflammation BACKGROUND Porcine circovirus cause different clinical syndrome<br>result significant economic loss pork industry Three pig<br>unexplained cardiac multi organ inflammation test negative PCV2<br>know porcine pathogen analyze METHODS Histology use identify<br>microscopic lesion multiple tissue Metagenomics use detect viral<br>sequence tissue homogenates situ hybridization use detect viral<br>RNA expression cardiac tissue RESULTS three case characterize<br>genome new circovirus call PCV3 replicase capsid proteins<br>show identity genetically closest protein bat feces associate<br>circovirus even distant porcine circovirus Common microscopic lesion<br>include non suppurative myocarditis cardiac arteriolitis Viral mRNA<br>detect intralesionally cardiac cell Deep sequence tissue also<br>reveal presence porcine astrovirus three animal well rotavirus<br>porcine cytomegalovirus porcine hemagglutinate encephalomyelitis virus individual case<br>CONCLUSION The pathogenicity molecular epidemiology new circovirus alone<br>context infection warrant investigation<br>\", \"Utilizing poxviral vectored vaccine antibody induction\\u2014Progress prospect Over<br>last decade poxviral vector emerge mainstay approach induction<br>cell mediate immunity vaccination suitability human use lead<br>widespread clinical test candidate vector infectious intracellular pathogen<br>cancer contrast poxvirus widely perceive vaccine field poor<br>choice vector induction humoral immunity However grow body<br>data animal model recent clinical trial suggest vector<br>successfully utilized prime boost cell effective antibody responses<br>Significant progress make context heterologous prime\\u2013boost immunization regimes<br>whereby poxvirus able boost response prim vector lead<br>induction high titre antigen specific antibody response case<br>poxviral vector show stimulate humoral immunity encode transgene<br>particular viral surface protein influenza haemagglutinin veterinary field<br>recombinant poxviral vector make significant impact numerous vector<br>license use variety animal virus study continue explore<br>potential poxviral vector modulate qualitative aspect humoral response<br>well amenability adjuvantation seek improve quantitative antibody immunogenicity<br>Nevertheless underlie mechanism cell induction recombinant poxviruses remain<br>poorly define work necessary help guide rational optimization<br>future poxviral vaccine candidate aim induce antibody<br>\", \"Long term human coronavirus myelin cross reactive cell<br>clone derive multiple sclerosis patient Autoimmune reaction associate<br>involve genetic environmental factor Because murine coronaviruses induce<br>like disease human coronaviruses HCoV attractive candidate environmental<br>factor involve demyelinate pathology previously report isolation HCoV<br>229E myelin basic protein MBP cross reactive cell<br>line TCL patient investigate antigenic cross reactivity molecular<br>level long term cell clone TCC derive patient<br>vitro selection MBP proteolipid protein PLP HCoV strain<br>229E OC43 Overall TCC virus specific specific myelin<br>HCoV myelin cross reactive Twenty eight virus specific<br>TCC myelin specific TCC obtain six healthy donor<br>RACE PCR amplification chain five ten cross reactive<br>TCC confirm clonality sequence identify CDR3 region associate<br>cross reactivity Our finding promise implication investigation role<br>molecular mimicry coronaviruses myelin mechanism related disease initiation<br>relapse<br>\", \"Protection Newborn Animals Maternal Immunization Summary Providing protective<br>immunity neonatal animal early life associate numerous challenge<br>regard vaccine safety efficacy much simpler approach maternal<br>vaccination either pregnancy provide neonate passively transfer immunity<br>human medical societal legal risk immunize pregnant woman<br>important consideration undertake approach contrast maternal vaccination successfully<br>employ animal health industry decades These veterinary vaccine<br>proven safe efficient Although passively transfer antibody extensively<br>study immunological mechanism may equally important provide maternally<br>derive immunity<br>\", \"Overlapping synthetic peptide vaccine Several vaccine strategy aim<br>generate cell mediate immunity CMI microorganisms tumor While<br>epitope base vaccine offer advantage knowledge specific epitope<br>frequency major histocompatibility complex MHC allele require Here<br>show use promiscuous overlap synthetic peptide OSP immunogen<br>generate peptide specific CMI vaccinate outbred mouse different<br>strain inbred mice CMI response also recognize viral<br>protein OSP immunogen also induce CMI vivo dendritic<br>cell cell cocultures involve cell individual different HLA<br>haplotype Thus broad CMI induce OSP different experimental<br>setting use different immunogen without identify either epitope<br>MHC background vaccinee<br>\", \"Situaci\\u00f3n actual desarrollo una vacuna preventiva frente VIH<br>avance epidemia sida convertido obtenci\\u00f3n una vacuna eficaz<br>frente virus inmunodeficiencia humana VIH como objetivo cient\\u00edfico<br>prioritario momento actual disponemos una vacuna preventiva frente<br>infecci\\u00f3n por VIH ning\\u00fan modelo animal conseguido protecci\\u00f3n<br>frente infecci\\u00f3n esta revisi\\u00f3n analizan dificultades existentes desarrollo<br>una vacuna contra sida especial los mecanismos escape<br>viral respuesta inmunitaria describen los prototipos vacunas preventivas<br>terap\\u00e9uticas desarrollo los resultados obtenidos Por otra parte<br>sit\\u00faa esta investigaci\\u00f3n contexto sanitario econ\\u00f3mico social pandemia<br>sida analizan las pol\\u00e9micas actualmente planteadas desarrollo ensayos<br>cl\\u00ednicos con los diferentes tipos vacunas Due uncontrolled<br>progression aid epidemic development HIV vaccine become major<br>objective scientific research effective preventive vaccine human immunodeficiency<br>virus HIV available sterilise immunity yet achieve animal<br>model review major challenge develop aid vaccine particular<br>mechanism involve viral escape immune response analyse result<br>obtain different prototype therapeutic preventive vaccine summarize Finally<br>social economic health aspect relate research HIV vaccines<br>present controversy regard development clinical trial discuss<br>\", \"Kunjin replicon base simian immunodeficiency virus gag vaccine<br>RNA base non cytopathic replicon vector system base<br>flavivirus Kunjin show considerable promise new vaccine delivery<br>system Here describe test mice four different SIVmac239<br>gag vaccine deliver Kunjin replicon virus like particles<br>The four vaccine encode wild type gag gene<br>RNA optimise gag gene codon optimise gag gene<br>modify gag pol gene construct The vaccine behave<br>quite differently induction effector memory central memory response<br>mediation protection respect insert stability SIV gag pol<br>vaccine provide optimal performance These result illustrate RNA<br>base vector RNA sequence antigen profound unforeseen consequence<br>vaccine behaviour<br>\", \"Generation protective efficacy cold adapt attenuate genotype porcine<br>epidemic diarrhea virus The recent emergence emergence porcine<br>epidemic diarrhea virus PEDV underscore urgent need development<br>novel safe effective vaccine prevail strain study generate<br>cold adapt live attenuate vaccine candidate Aram P29<br>short term passage virulent PEDV isolate successively low<br>temperature Vero cell Whole genome sequence identify amino<br>acid change cold adapt strain insertion deletion throughout<br>genome Animal inoculation experiment confirm attenuated phenotype Aram<br>P29 virus natural host Pregnant sow orally administer<br>P29 live vaccine two dos week interval prior<br>parturition newborn piglet challenge parental virus The oral<br>homologous prime boost vaccination P29 significantly improve survival<br>rate piglet notably mitigate severity diarrhea PEDV fecal<br>shed challenge Furthermore strong antibody response PEDV detect<br>serum colostrum immunize sow sera offspring These result<br>demonstrate cold adapt attenuated virus use live vaccine<br>maternal vaccination strategy provide durable lactogenic immunity confer<br>passive protection litter PEDV<br>\", \"Characterization application monoclonal antibody protein SARS coronavirus Severe<br>acute respiratory syndrome coronavirus SARS CoV cause infectious<br>disease respiratory route Diagnosing disease effectively accurately early<br>stage essential prevent disease transmission perform antiviral treatment<br>study raise monoclonal antibody mAbs nucleocapsid protein SARS<br>CoV map epitope use different truncate protein fragment<br>The mapping epitopes valuable construct pair Abs use<br>serological diagnosis The result show six raise mAbs<br>divide two group recognize region amino acids 249\\u2013317<br>group 317\\u2013395 group This region span amino acid<br>249\\u2013395 contains predominant cell epitope locate terminus protein<br>One pair Abs consist protein specific rabbit polyclonal<br>antibody SARS CoV protein specific mAb select construct<br>sandwich ELISA kit The kit able specifically detect<br>SARS CoV proteins serum sample<br>\", \"Efficacy intranasal spray delivery adjuvanted live vaccine infectious<br>bronchitis virus experimentally infected poultry Live vaccine widely<br>use avian industry Such vaccine either inject deliver<br>animal mucosa usually adjuvanted study show live vaccine<br>efficacy improve formulation adjuvant model mucosal delivery live<br>infectious bronchitis vaccine chicken Three adjuvant technology test<br>use intranasal spray delivery method poultry Those technology<br>water oil water emulsion nanoparticles polymer adjuvant Intranasal<br>delivery polymer nanoparticles adjuvanted live vaccine improve significantly<br>antibody titer protection challenge observe compare commercial non<br>adjuvanted reference Moreover spray delivery polymer adjuvanted vaccine<br>show significantly high protection compare non adjuvanted reference<br>Our data demonstrates use MontanideTM adjuvant formulation live<br>poultry vaccine mucosal delivery confer vaccinate animal significantly<br>improve protection pathogen<br>\", \"Evaluation Mucosal Systemic Immune Responses Elicited GPI Adjuvanted<br>Influenza Vaccine Delivered Different Immunization Strategies Vaccines protection<br>respiratory infection optimally induce mucosal immune response respiratory<br>tract addition systemic immune response However current parenteral<br>immunization modality generally fail induce mucosal immunity mucosal<br>vaccine delivery often result poor systemic immunity order<br>find immunization strategy satisfies need induction mucosal systemic<br>immunity compare local systemic immune response elicit two<br>mucosal immunization give either intranasal intrapulmonary IPL route<br>response elicit mucosal prime follow systemic boost immunization<br>The study conduct BALB mouse vaccine formulation influenza<br>subunit vaccine supplement GPI saponin derive adjuvant While<br>optimal mucosal antibody titer obtain two intrapulmonary vaccination<br>optimal systemic antibody response achieve intranasal prime follow<br>intramuscular boost The latter strategy also result best<br>cell response yet ineffective inducing nose lung IgA<br>Successful induction secretory IgA IgG cell response achieve<br>prime boost strategy involve intrapulmonary immunization optimal immunization<br>give via intrapulmonary route Our result underline immunization<br>via lung particularly effective prim well boost local<br>systemic immune response<br>\", \"The attenuation vaccinia Tian Tan strain removal viral<br>M1L K2L gene generate safe vaccinia Tian Tan<br>VTT base vaccine vector necessary develop method attenuate<br>virus modify VTT MVTT2 GFP construct replace viral<br>M1L K2L gene GFP gene comparison parental VTT<br>MVTT2 GFP lose replication capacity rabbit RK13 human<br>HeLa cell line The life cycle viral replication<br>block different stage two cell line determine electron<br>microscope examination MVTT2 GFP less virulent VTT fold<br>measuring mouse body weight loss intranasal viral inoculation<br>fold determine intracranial LD50 value mice The foreign<br>GFP gene stable genetically round passage Vero cell<br>Importantly MVTT2 GFP elicit humoral cell mediate immune<br>response GFP gene mouse With two intramuscular inoculation<br>PFU virus anti GFP antibody reciprocal endpoint titer<br>reach determined ELISA The number IFN secrete cell<br>reach 350SFU per million splenocytes CD8 cell specific<br>epitope GFP Collectively removal M1L K2L gene useful<br>method generate attenuate vaccinia Tian Tan vaccine vector<br>\", \"Comparison efficiency antibody selection semi synthetic scFv non<br>immune Fab phage display libraries protein target rapid<br>development diagnostic immunoassay Rapid development diagnostic immunoassay novel<br>emerge genetically modify pathogen emergency situation dependent timely<br>isolation specific antibody Non immune antibody phage display<br>libraries efficient vitro method select monoclonal antibody hence<br>ideal circumstance Such library construct variety source cell<br>cDNA synthetically generate use variety antibody format typically<br>scFv Fab However antibody source format impact quality<br>antibody generate hence effectiveness methodology timely production antibody<br>carry comparative screen two antibody library semi synthetic<br>scFv library human derive Fab library protective antigen<br>toxin component Bacillus anthracis epsilon toxin Clostridium botulinum<br>show synthetic library produce diverse collection specific scFv<br>phage contain high frequency unnatural amber stop glycosylation<br>site limit conversion IgG also high number lose<br>specificity express IgG contrast limitation overcome use natural<br>human library Antibodies library could use develop sandwich<br>ELISA assay similar sensitivity However ease speed full<br>length IgG could generate human derive Fab library<br>make screen type library preferable method rapid antibody<br>generation diagnostic assay development<br>\", \"Peptide Mimicrying Between SARS Coronavirus Spike Protein Human<br>Proteins Reacts SARS Patient Serum Molecular mimicry define<br>similar structure share molecule dissimilar gene proteins general<br>strategy use pathogen infect host cell Severe acute<br>respiratory syndrome SARS new human respiratory infectious disease<br>cause SARS coronavirus SARS CoV The spike protein<br>SARS CoV play important role virus entry cell<br>study eleven synthetic peptide protein select base sequence<br>homology human proteins Two peptide D07 residues 927\\u2013937<br>D08 residue 942\\u2013951 recognize serum SARS patient Murine<br>hyperimmune serum peptides bound protein human lung epithelial<br>cell A549 Another peptide D10 residues 490\\u2013502 stimulated<br>A549 proliferate secrete The present result suggest select<br>protein region share sequence homology human protein may<br>play important role SARS CoV infection<br>\", \"Human monoclonal antibody prophylaxis SARS coronavirus infection ferret<br>Summary SARS coronavirus continue cause sporadic case severe<br>acute respiratory syndrome SARS China active passive immunoprophylaxis<br>disease induce SARS coronavirus available investigate prophylaxis SARS<br>coronavirus infection neutralise human monoclonal antibody ferret readily<br>infect virus Prophylactic administration monoclonal antibody reduce replication<br>SARS coronavirus lung infect ferret 3\\u00b73 log 2\\u00b76\\u20134\\u00b70<br>log 0\\u00b7001 completely prevent development SARS coronavirus induce<br>macroscopic lung pathology 0\\u00b7013 abolish shed virus pharyngeal<br>secretions The data generate animal model show administration<br>human monoclonal antibody might offer feasible effective prophylaxis<br>control human SARS coronavirus infection<br>\", \"Identification Neutralizing Monoclonal Antibodies Targeting Novel Conformational Epitopes<br>Porcine Epidemic Diarrhoea Virus Spike Protein Since newly<br>identify variant porcine epidemic diarrhoea virus PEDV cause<br>high mortality neonatal piglet devastate swine industry The<br>spike glycoprotein PEDV contain multiple neutralize epitope major<br>target PEDV neutralization vaccine development understand antigenicity new<br>PEDV variant characterize neutralize epitope new genotype PEDV<br>isolate Taiwan PEDV Pintung PEDV generation neutralize monoclonal<br>antibody NmAbs Two NmAbs P4B E10E 1\\u201310 recognize<br>ectodomain full length recombinant PEDV protein exhibit neutralizing<br>ability PEDV virus select Recombinant truncate protein use<br>identify target sequence NmAbs P4B show recognize terminus<br>26K equivalent epitope COE amino acid 575\\u2013639 PEDV<br>Interestingly E10E 1\\u201310 could recognize novel neutralize epitope<br>435\\u2013485 within domain PEDV protein whose importance function<br>yet determine Moreover NmAbs could bind linearized proteins<br>indicate conformational epitope recognize This data could improve<br>understand antigenic structure PEDV protein facilitate future development<br>novel epitope base vaccine<br>\", \"Unique Epitopes Recognized Antibodies Induced Chikungunya Virus Infected<br>Non Human Primates Implications Study Immunopathology Vaccine Development<br>Chikungunya virus CHIKV Alphavirus cause chronic incapacitate arthralgia<br>human Although patient cohort study show production CHIKV<br>specific antibody fine specificity antibody response CHIKV completely<br>define The macaque model CHIKV infection establish due<br>limitation clinical specimen More importantly close relation human<br>allow study chronic infection identify important CHIKV target<br>study serum sample CHIKV infect macaque collect different<br>time point post infection use characterize antibody production<br>pattern kinetics Results reveal anti CHIKV antibody neutralize<br>glycoprotein Capsid nsP1 nsP3 nsP4 protein target anti<br>CHIKV antibody response macaque Furthermore linear cell epitope<br>recognize anti CHIKV antibody identify mapped structural localization<br>This characterize specificity anti CHIKV antibody response macaque<br>demonstrate importance different region CHIKV encode protein adaptive<br>immune response Information study provide critical knowledge aid<br>understand CHIKV infection immunity vaccine design pre clinical<br>efficacy study<br>\", \"Host protective ASP base vaccine parasitic nematode Ostertagia<br>ostertagi trigger cell activation mixed IgG1 IgG2 response<br>The mucus dwell parasite Ostertagia ostertagi one important<br>gastrointestinal nematode cattle Our group previously demonstrate protective<br>capacity vaccine parasite base native activation associate secrete<br>protein ASP1 nASP combination saponin adjuvant QuilA The<br>aim current study analyse effect antigen adjuvant cellular<br>humoral vaccine induce immune response compare native ASP<br>recombinant version express Pichia pastoris pASP replace QuilA<br>Immunization cattle protective nASP QuilA vaccine associate antigen<br>induce proliferation natural killer cell combine IFN secretion<br>induction mixed IgG1 IgG2 antibody response ASP specific<br>activation proliferation cell also observe mouse follow vaccination<br>regime Replacing QuilA nASP pASP significantly decrease capacity<br>vaccine trigger cell activation antibody response fail induce<br>protection challenge infection Reduction structurally anchor disulphide bond<br>nASP completely abolish ability induce cell activation antibody<br>responses highlight importance protein conformation immunostimulatory activity<br>\", \"Dengue Virus Reporter Replicon Valuable Tool Antiviral Drug<br>Discovery Analysis Virus Replication Mechanisms Dengue prevalent arthropod<br>borne viral disease cause dengue virus DENV member<br>Flaviviridae family considerable public health threat country billion<br>people live high risk area However specific antiviral<br>drug license vaccine currently target DENV infection The<br>replicon system factor need viral replication cell Since<br>development replicon system transient stable reporter replicons well<br>reporter virus use study various virological aspect DENV<br>identification DENV inhibitor review summarize DENV reporter replicon<br>system application high throughput screen HTS identification anti<br>DENV inhibitor also describe use system elucidation mechanism<br>virus replication viral dynamic vivo vitro<br>\", \"Protection chickens infectious bronchitis virus multivalent DNA vaccine<br>boost inactivated vaccine The protective efficacy DNA plasmid<br>encode avian infectious bronchitis virus IBV protein investigate<br>chicken Chickens inoculate monovalently plasmid pVAX1 16S1 pVAX1<br>16M pVAX1 16N alone multivalently combination three different<br>plasmid pVAX1 16S1 prime boost immunization protocol IBV<br>develop Chickens immunized multivalent DNA vaccine twice boost<br>inactivated vaccine Antibody titer chicken immunize pVAX1 16S1<br>much high monovalent group protective rate observe pVAX1<br>16S1 group The serum antibody titer prime boost<br>bird significantly high multivalent DNA vaccine group significantly<br>different compare inactivated vaccine group day age Additionally<br>prime boost group also show high level IBV<br>specific cellular proliferation compare monovalent group significant difference<br>find compare multivalent DNA vaccine group prime boost<br>group completely protect followed viral challenge<br>\", \"Biological molecular characterization ArkGA novel Arkansas serotype vaccine<br>highly attenuate efficacious protective homologous challenge Almost commercial<br>poultry vaccinate avian coronavirus infectious bronchitis virus IBV<br>use live attenuate vaccine mass administer spray day<br>hatch Although many different type IBV vaccine use<br>successfully ArkDPI serotype vaccine apply spray infect replicate<br>sufficiently provide protection homologous challenge study examine different<br>Ark vaccine strain Ark99 longer use commercially due<br>reactivity one day old chick determine could attenuate<br>passage embryonated egg still provide adequate protection Further<br>attenuation Ark99 vaccine achieve passage embryonated egg ArkGA<br>P20 P40 designate ArkGA still reactive suitable vaccine<br>candidate However ArkGA P60 give spray little vaccine<br>reaction one day old broiler chick induce protection<br>clinical sign ciliostasis follow homologous challenge addition vaccinate<br>challenged bird significantly less challenge virus important measure<br>protection compare non vaccinate challenged control The full<br>length genome viruses egg passage sequence use Illumina<br>platform data show single nucleotide polymorphisms SNPs accumulate<br>region genome associate viral replication pathogenicity cell tropism<br>ArkGA P60 accumulate SNPs key gene associate pathogenicity<br>polyprotein gene 1ab cell tropism spike gene compare<br>previous passage likely result attenuated phenotype These result<br>indicate ArkGA P60 vaccine safe spray vaccination broiler<br>chick induces suitable protection challenge pathogenic Ark type<br>virus<br>\", \"Biochemical biophysical characterization reovirus cell attachment protein Evidence<br>homotrimer The oligomerization state reovirus cell attachment protein<br>49K monomeric molecular weight determine biochemical biophysical mean<br>Full length protein product designate terminal truncate protein<br>product designate serotype reovirus mRNA transcript synthesizedin vitro<br>cotranslated rabbit reticulocyte lysate product analyze sodium dodecyl<br>sulfate polyacrylamide gel electrophoresis SDS PAGE condition allow<br>identification oligomeric form total four oligomeric protein band<br>correspond A2B1 A1B2 respectively consistently observe suggest protein<br>make three monomeric subunit Biophysical characterization purify use<br>column filtration sucrose gradient sedimentation analysis confirm highly<br>asymmetric shape allow determine molecular weight native protein<br>\\u223c132K trimer Similar biophysical analysis two tryptic fragment<br>terminal fibrous tail 26K monomeric molecular weight terminal<br>globular head 23K monomeric molecular weight yield molecular<br>weight 77K 64K respectively correspond trimer therefore conclude<br>protein homotrimer provide supportive experimental evidence rationale anomalous<br>behavior oligomeric protein SDS polyacrylamide gel couple chemical<br>cross link study part lead previous suggestion might<br>high order oligomer<br>\", \"Functional analysis stem loop structure coronavirus 3\\u2032UTR design<br>series mutation separately destabilize two helical stem designate<br>predict covariation base model coronavirus 3\\u2032UTR Zust Mouse<br>hepatitis virus genome contain three four nucleotide mutation<br>destabilize either viable whereas genome carry mutation viable<br>genome carrying mutation plus compensatory mutation restore base<br>pairing yield virus wild type phenotype Larger mutation<br>completely disrupt generate various phenotype Mutations open lethal<br>Disruptions generate viable lethal mutant Genomes carry original<br>mutation plus compensatory mutation restore base pair viable<br>robust growth phenotypes These result support Zust model<br>coronavirus 3\\u2032UTR suggest stem require virus viability<br>\", \"The Molecular Biology Frog Virus Iridoviruses Infecting Cold<br>Blooded Vertebrates Frog virus FV3 best characterized member<br>family Iridoviridae FV3 study provide insight replication family<br>member serve model viral transcription genome replication virus<br>mediate host shutoff Although broad outline FV3 replication<br>elucidate precise role viral protein remain unknown Current<br>study use knock mediate antisense morpholino oligonucleotides asMO<br>small interfering RNAs siRNA knock follow replacement target<br>gene selectable marker homologous recombination ectopic viral gene<br>expression recombinant viral protein enable researcher systematically ascertain<br>replicative virulence relate gene function addition application molecular<br>tool ecological study provide novel way field biologists<br>identify potential pathogen quantify infection trace evolution ecologically<br>important viral specie review summarize current study use<br>FV3 also iridoviruses infect ectotherm described general principle<br>ascertain use FV3 serve model family study utilize<br>ranaviruses megalocytiviruses confirm extended understanding iridovirus replication Collectively<br>future effort elucidate molecular event viral replication intrinsic<br>extrinsic factor contribute disease outbreaks role host immune<br>system protection disease<br>\", \"The tetraspanin CD9 facilitate MERS coronavirus entry scaffold<br>host cell receptor proteases Infection envelop coronaviruses CoVs<br>initiate viral spike proteins bind cellular receptor follow<br>proteolytic cleavage receptor bound proteins prompt protein mediate<br>virus cell membrane fusion Infection therefore require close<br>proximity receptor proteases consider whether tetraspanins scaffold protein<br>know facilitate CoV infection hold receptor proteases together<br>cell membrane Using knockout cell line find tetraspanin<br>CD9 tetraspanin CD81 form cell surface complex dipeptidyl<br>peptidase DPP4 MERS CoV receptor type transmembrane serine<br>protease TTSP member TMPRSS2 CoV activate protease This<br>CD9 facilitate condensation receptor proteases allowed MERS CoV<br>pseudoviruses enter cell rapidly efficiently Without CD9 MERS<br>CoV virus activate TTSPs traffic endosomes cleave much<br>later less efficiently cathepsins Thus identify DPP4 CD9<br>TTSP protein complex necessary early efficient MERS CoV<br>entry evaluate importance complex vivo CoV infection model<br>use recombinant Adenovirus rAd5 vector express human DPP4<br>mouse lung thereby sensitize animal MERS CoV infection<br>When rAd5 hDPP4 vector express small RNAs silence<br>Cd9 Tmprss2 animal significantly less susceptible indicate CD9<br>TMPRSS2 facilitate robust vivo MERS CoV infection mouse<br>lung Furthermore proteins virulent mouse adapt MERS CoVs<br>acquire CD9 dependent cell entry character suggest CD9<br>selective agent evolution CoV virulence<br>\", \"The Lepidopteran endoribonuclease domain protein P102 display dramatically<br>reduce enzymatic activity form functional amyloid Hemocytes Heliothis<br>virescens Lepidoptera Noctuidae larva produce protein P102 putative<br>endoribonuclease domain previous work show P102 involve Lepidopteran<br>immune response form amyloid fibril catalyze localize melanin<br>deposition around non self intruder encapsulation prevent harmful<br>systemic spreading Here demonstrate P102 belong new class<br>protein least Lepidoptera diminish endoribonuclease activity probably due<br>lack two five catalytically essential residue show P102<br>homolog Trichoplusia Lepidoptera Noctuidae display catalytic site residue<br>identical P102 residual endoribonuclease activity ability form functional<br>amyloid basis result well sequence structural analysis hypothesize<br>Lepidoptera endoribonuclease orthologs catalytic site residue identical P102<br>form subfamily similar function<br>\", \"Evidence RNA pseudoknot loop helix interaction essential efficient<br>ribosomal frameshifting Edited Draper RNA pseudoknots structural element<br>participate variety biological process ribosomal frameshifting site several<br>type pseudoknot identify differ organisation functionality The pseudoknot<br>find infectious bronchitis virus IBV typical posse long<br>stem long loop second group pseudoknots distinguishable contain<br>stem shorter loop The NMR structure one pseudoknot<br>mouse mammary tumor virus MMTV reveal kink stem<br>stem junction kink conformation essential efficient frameshifting recently<br>investigate effect frameshifting modulate stem length stability IBV<br>base pseudoknots find stem least need efficient frameshifting<br>Here describe sequence manipulation necessary bypass requirement stem<br>convert short non functional IBV derive pseudoknot highly<br>efficient kink frameshifter pseudoknot Simple insertion adenine residue<br>stem stem junction essential feature kink pseudoknot sufficient<br>create functional pseudoknot additional change need efficient frameshifting<br>recover last nucleotide loop change The requirement end<br>loop consistent loop helix contact similar describe RNA<br>tertiary structure mutational analysis partner propose interaction loop<br>terminal adenine residue two G\\u00b7C pair near top<br>stem reveal interaction essential efficient frameshifting The specific<br>requirement terminal residue lose loop increase suggest loop<br>helix contact may require pseudoknots short loop<br>\", \"The requirement environmental acidification Ibaraki virus infection host<br>cell The effect environmental acidification Ibaraki virus IBAV<br>infection test use endosomal inhibitory chemical low treatment<br>Treatment target cell endosomal inhibitor significantly decrease progeny<br>virus production IBAV outer capsid proteins VP5 VP2<br>remove virion purify IBAV expose low environment Further<br>experiment show exposure low buffer facilitate IBAV infection<br>cellular endosomal pathway impair bafilomycin Results obtain study<br>suggest acidic environment essential initiate IBAV infection<br>\", \"quadruplexes viruses function potential therapeutic application rich nucleic<br>acid form non canonical quadruplex structure G4s four<br>guanine fold planar arrangement Hoogsteen hydrogen bond Although<br>many biochemical structural study focus DNA sequence contain<br>successive adjacent guanine spontaneously fold G4s evidence vivo<br>relevance recently begin accumulate Complete sequence human genome<br>highlight presence \\u223c300 sequence potentially form G4s Likewise<br>presence putative sequence report various virus genome Human<br>immunodeficiency virus HIV Epstein\\u2013Barr virus EBV papillomavirus HPV<br>Many study focus telomeric G4s dynamic regulate enable<br>telomere synthesis Moreover role G4s propose cellular viral<br>replication recombination gene expression control parallel DNA aptamers<br>form G4s describe inhibitor diagnostic tool detect virus<br>hepatitis virus HAV EBV cauliflower mosaic virus CaMV<br>severe acute respiratory syndrome virus SARS simian virus<br>SV40 Here special emphasis give possible role structure<br>virus life cycle well use form oligonucleotides potential<br>antiviral agent innovative tool<br>\", \"Design Characterization Viral Polypeptide Inhibitors Targeting Newcastle Disease<br>Virus Fusion Paramyxovirus infection detect worldwide emerge zoonotic<br>virus currently specific therapeutic treatment vaccine available many<br>disease Recent study demonstrate peptide derive two heptad<br>repeat region HR1 HR2 paramyxovirus fusion protein could<br>use inhibitor virus fusion The mechanism underlying activity<br>accordance class virus fusion protein human immunodeficiency virus<br>HIV influenza virus fusion proteins member For class<br>virus fusion protein HR1 fragment bind HR2 form<br>six helix bundle three HR1 fragment form central<br>coil bundle surround three coil HR2 fragment post<br>fusion conformational state fusion core hypothesize introduce exogenous<br>HR1 HR2 compete endogenous counterpart result fusion inhibition<br>Using Newcastle disease virus NDV model design several<br>protein inhibitor denote HR212 well asHR121 Helix could<br>bind HR1 HR2 region fusion protein respectively All<br>protein express purified use GST fusion expression system<br>Escherichia coli The HR212 GST HR212 protein bind<br>HR1 peptide vitro display inhibitory activity NDV mediate<br>cell fusion HR121 Helix protein bind HR2 peptide<br>vitro inhibit virus fusion avirulent NDV strain add<br>cleavage fusion protein These result show design HR212<br>HR121 Helix protein could serve specific antiviral agent<br>These data provide additional insight difference virulent avirulent<br>strain NDV<br>\", \"Expression biochemical characterization nsP2 cysteine protease Chikungunya virus<br>Chikungunya virus CHIKV mosquito borne alphavirus cause epidemic<br>fever rash polyarthralgia Africa Asia Although know since<br>1950s new epidemiological clinical feature report recent outbreak<br>Indian Ocean regard emergence new disease Numerous severe<br>form infection describe put emphasis lack efficient antiviral<br>therapy Among virus encode enzymes nsP2 constitutes attractive<br>target development antiviral drug multifunctional protein approximately 90kDa<br>helicase motif terminal portion protein papain like protease<br>activity reside terminal portion The nsP2 proteinase essential<br>enzyme whose proteolytic activity critical virus replication work<br>recombinant CHIKV nsP2pro terminally truncate variant express Escherichia<br>coli purify metal\\u2013chelate chromatography The enzymatic property proteinase<br>determine use specific synthetic fluorogenic substrates This study<br>constitute first characterization recombinant CHIKV nsP2 cysteine protease<br>may useful future drug screening<br>\", \"Structural mechanistic basis preferential deadenylation snRNA Usb1 Post<br>transcriptional modification snRNA central spliceosome function Usb1 exoribonuclease<br>shorten oligo uridine tail snRNA result terminal cyclic<br>phosphate group eukaryotes include human Loss function mutation<br>human Usb1 cause rare disorder poikiloderma neutropenia result<br>snRNAs elongate end aberrantly adenylated Here show human<br>Usb1 remove adenosine fold great efficiency uridines explain<br>presence adenylated snRNAs cell lack Usb1 determine three<br>high resolution crystal structure Usb1 wild type Usb1<br>bound substrate analog adenosine monophosphate inactive mutant bound<br>RNAs terminal adenosine uridine These structure along simulation<br>catalytic mechanism illuminate molecular basis preferential deadenylation snRNA<br>The extent Usb1 processing influence secondary structure snRNA<br>\", \"The dengue virus non structural protein NS1 secrete<br>mosquito cell association intracellular cholesterol transporter chaperone caveolin<br>complex ABSTRACTDengue virus DENV mosquito borne virus family<br>Flaviviridae The RNA viral genome encodes polyprotein translationally<br>process three structural protein seven non structural proteins<br>The non structural protein NS1 multifunctional viral protein<br>actively secrete vertebrate mosquito cell DENV infection mosquito<br>cell NS1 secrete caveolin CAV dependent manner unconventional<br>pathway The caveolin chaperone complex CCC cytoplasmic complex<br>form caveolin chaperone FKBP52 Cy40 CyA responsible cholesterol<br>traffic inside cell work demonstrate infect mosquito cell<br>DENV NS1 secrete early unconventional route bypass Golgi<br>apparatus close association CCC Treatment mosquito cell classic<br>secretion inhibitor brefeldin golgicide Fli show effect NS1<br>secretion significant reduction recombinant luciferase secretion virion release<br>Silencing expression CAV1 FKBP52 siRNAs inhibition CyA cyclosporine<br>result significant decrease NS1 secretion without affect virion<br>release Using localization inmunoprecipitation proximity ligation assay NS1<br>find localize interact protein component CCC mosquito infect<br>cell addition CAV FKBP52 expression find augment DENV<br>infected cell Finally treatment ZIKV infect mosquito cell<br>brefeldin golgicide show effect NS1 secretion affect virion<br>release ZIKV NS1 also find localize CAV infect<br>mosquito cell These result suggest mosquito cell ZIKV<br>NS1 follow secretory pathway observed DENV NS1 The<br>association NS1 cholesterol transporter CCC agree lipoprotein nature<br>secrete hexameric NS1 AUTHOR SUMMARYDengue protein NS1 secrete<br>infected mosquito vertebrate cell human secrete NS1 associated<br>pathogenesis mosquito cell NS1 follow unconventional secretion pathway<br>dependent Caveolin This work show mosquito cell NS1<br>secretion associate chaperone caveolin complex complex form caveolin<br>several chaperone charge cholesterol transport within cell Reduction<br>expression activity chaperone caveolin complex mosquito infect cell<br>diminish secretion NS1 without affect virion release Direct<br>interaction NS1 chaperone caveolin complex protein demonstrate several<br>assay Moreover increase expression caveolin chaperone FKBP52 dengue<br>infection find presumably response high requirement proteins dengue<br>virus infection Results obtain ZIKV infect mosquito cell<br>suggest also ZIKV NS1 release follow unconventional secretory<br>route association chaperone caveolin complex The function secrete<br>NS1 within mosquito unclear However give importance soluble<br>NS1 vertebrate host manipulation NS1 secretory route may<br>prove valuable strategy dengue mosquito control patient treatment<br>\", \"High Resolution Analysis Coronavirus Gene Expression RNA Sequencing<br>Ribosome Profiling Members family Coronaviridae large genome RNA<br>viruses typically region kilobases Several coronaviruses Severe acute<br>respiratory syndrome relate coronavirus SARS CoV Middle East<br>respiratory syndrome relate coronavirus MERS CoV medical importance<br>high mortality rate case SARS CoV significant pandemic<br>potential Other coronaviruses Porcine epidemic diarrhea virus Avian<br>coronavirus important livestock pathogen Ribosome profiling technique exploit<br>capacity translate ribosome protect around nucleotide mRNA ribonuclease<br>digestion Ribosome protect mRNA fragment purify subjected deep<br>sequencing map back transcriptome give global snap shot<br>translation Parallel RNA sequence allows normalization transcript abundance<br>Here apply ribosome profile cell infect Murine coronavirus<br>mouse hepatitis virus strain A59 MHV A59 model<br>coronavirus genus SARS CoV MERS CoV The data<br>obtain allow study kinetics virus transcription translation exquisite<br>precision study timecourse positive negative sense genomic subgenomic<br>viral RNA production relative translation efficiency different virus<br>ORFs Virus mRNAs find translate efficiently host mRNAs<br>rather virus translation dominate host translation later time<br>point due high level virus transcript Triplet phase<br>profile data allow precise determination translate read frames<br>reveal several translate short open read frame upstream<br>embed within know virus protein cod region Ribosome<br>pause site identify virus replicase polyprotein pp1a ORF<br>investigate experimentally Contrary expectation ribosome find pause ribosomal<br>frameshift site knowledge first application ribosome profile RNA<br>virus<br>\", \"Venezuelan equine encephalitis virus non structural protein nsP3<br>interacts RNA helicases DDX1 DDX3 infect cell The<br>mosquito bear New World alphavirus Venezuelan equine encephalitis<br>virus VEEV Category select agent approve vaccine therapy<br>treat infected human Therefore imperative identify novel target<br>target effective therapeutic intervention aim identify validate interaction<br>VEEV nonstructural protein nsP3 host protein determine consequence<br>interaction viral multiplication use tag nsP3 infectious clone<br>rTC nsP3 identify validate two RNA helicases DDX1<br>DDX3 interact VEEV nsP3 addition DDX1 DDX3 knockdown<br>result decrease infectious viral titer Furthermore propose functional<br>model nsP3 DDX3 complex interacts host translational machinery<br>essential viral life cycle This study lead future<br>investigation understand importance VEEV nsP3 viral multiplication apply<br>information discovery novel host target therapeutic option<br>\", \"Contribution porcine aminopeptidase porcine deltacoronavirus infection Porcine deltacoronavirus<br>PDCoV member genus Deltacoronavirus emerge swine enteropathogenic coronavirus<br>CoV Although outstanding effort lead identification Alphacoronavirus Betacoronavirus<br>receptor receptor Deltacoronavirus unclear Here compare amino acid<br>sequence several representative CoVs Phylogenetic analysis show PDCoV<br>spike protein close cluster contain transmissible gastroenteritis virus<br>TGEV utilize porcine aminopeptidase pAPN functional receptor Ectopic<br>expression pAPN non susceptible BHK cell render susceptible<br>PDCoV These result indicate pAPN may functional receptor<br>PDCoV infection However treatment APN specific antibody inhibitor<br>completely block PDCoV infection IPI porcine intestinal epithelial<br>cell pAPN knockout IPI cell completely block TGEV<br>infection slightly decrease PDCoV infection Homologous model pAPN<br>terminal domain CTD PDCoV TGEV show TGEV CTD<br>adopt turn \\u03b21\\u2013\\u03b22 \\u03b23\\u2013\\u03b24 form tip barrel recognize<br>pAPN However top residue \\u03b21\\u2013\\u03b22 turn PDCoV CTD<br>possibility support interaction pAPN \\u03b23\\u2013\\u03b24 turn fail contact<br>pAPN also discuss evolution variation PDCoV CTD base<br>structure information provide clue explain usage pAPN PDCoV<br>Taken together result present herein reveal pAPN likely<br>critical functional receptor PDCoV although involve PDCoV infection<br>\", \"BUHO MATLAB Script Study Stress Granules Processing Bodies<br>High Throughput Image Analysis The spontaneous reversible formation<br>focus filament contain protein involve different metabolic process<br>common nucleus cytoplasm Stress granules SGs process body<br>PBs belong novel family cellular structure collectively know<br>mRNA silence foci harbour repress mRNAs associate protein<br>SGs PBs highly dynamic form upon stress dissolve<br>thus release repress mRNAs accord change cell physiology<br>addition aggregate contain abnormal protein frequent neurodegenerative disorder<br>spite grow relevance supramolecular aggregate diverse cellular function<br>reliable automated tool systematic analysis lack Here report<br>MATLAB Script term BUHO high throughput image analysis<br>cellular focus use BUHO assess number size distribution<br>distinct object minimal deviation manually obtain parameter BUHO<br>successfully address induction SGs PBs mammalian insect cell<br>expose different stress stimulus also use BUHO ass<br>dynamic specific mRNA silence focus term Smaug focus<br>focus primary neuron upon synaptic stimulation Finally use<br>BUHO analyze role candidate gene formation RNAi base<br>experiment find FAK56D GCN2 PP1 govern formation The<br>role PP1 conserve mammalian cell judge effect PP1<br>inhibitor salubrinal involve dephosphorylation translation factor eIF2\\u03b1 All<br>experiment analyze manually BUHO result differ less average<br>value The automated analysis user friendly method allow<br>high throughput image processing short time provide robust<br>flexible reliable alternative laborious sometimes unfeasible visual scrutiny<br>\", \"Factors affect translation program ribosomal frameshifting site Cocksfoot<br>mottle virus RNA vivo The ratio proteins P27<br>replicase Cocksfoot mottle virus CfMV regulate via program<br>ribosomal frameshift PRF minimal frameshift signal slippery UUA<br>AAC heptamer downstream stem\\u2013loop structure insert dual reporter<br>vector direct PRF efficiency yeast bacteria P27 encode<br>CfMV sequence flank minimal frameshift signal cause fold<br>increase PRF efficiencies yeast bacteria addition expect fusion<br>proteins termination product end putatively frameshift site find<br>yeast cell propose amount premature translation termination control<br>mRNAs play role determine calculate \\u22121PRF efficiency expression<br>CfMV P27 dual reporter vector contain minimal frameshift<br>signal reduce production downstream reporter whereas replicase expression<br>pronounce effect This finding allow propose CfMV protein<br>P27 may influence translation frameshift site mechanism need<br>elucidate<br>\", \"Comprehensive structural analysis design incomplete polypeptide chain replicase<br>nonstructural protein severe acute respiratory syndrome coronavirus The<br>cotranslational folding recognize cooperative process occur nearly completion<br>polypeptide sequence domain Here investigate challenge face polypeptide<br>segment non vectorial barrel fold Besides biological interest<br>behind SARS coronavirus non structural protein nsp1 amino<br>acid study model two structural feature motivate use<br>work recombinant production dependent temperature great solubility express<br>low temperature This indication cotranslational guidance native protein<br>conformation Conversely nsp1 six strand mixed parallel antiparallel<br>barrel intricate long range interaction indicate need full<br>length protein fold properly use non denature purification<br>condition allow characterization polypeptide chain different length mimic<br>landscape cotranslational fold barrel avoid major technical hindrance<br>work nascent polypeptide bound ribosome Our result show<br>partially fold state form soon amino acid second<br>strand present amino acid These partially fold state<br>different base length polypeptide chain The native helix<br>amino acid 24\\u201337 identified transient structure 20\\u201330 propensity<br>identify presence regular secondary structure fourth native strand<br>present amino acid parallel collapse non native structure<br>Interestingly polypeptide sequence native strand characteristic helix Our<br>comprehensive analysis support idea incomplete polypeptide chain one<br>nascent protein much earlier end translation adopt abundance<br>specific transient fold instead disorder conformation<br>\", \"RNA pseudoknots downstream frameshift site retroviruses RNA pseudoknot<br>structural motif could implications wide range biological process<br>RNAs study potential RNA pseudoknots downstream know suspected<br>retroviral frameshift site predict Rous sarcoma virus primate<br>immunodeficiency viruses HIV HIV SIV equine infectious anemia<br>virus visna virus bovine leukemia virus human cell<br>leukemia virus type mouse mammary tumor virus Mason<br>Pfizer monkey virus simian SRV type retrovirus Also<br>putative RNA pseudoknots detect gag pol overlap two<br>retrotransposons Drosophila gypsy mouse intracisternal particle For sequence<br>thermodynamic stability statistical significance secondary structure involve predicted<br>tertiary structure assess compare Our result show stem<br>loop structure pseudoknots thermodynamically highly stable statistically significant<br>relative configuration potentially occur gag pool gag pro<br>pro pol junction domains viruses nucleotide upstream downstream<br>possible frameshift site include Moreover structural feature predict<br>pseudoknots follow frameshift site propol overlap HTLV HTLV<br>retrovirus structurally well conserve The occurence eight compensatory<br>base change tertiary interaction two related sequence allow<br>conservation tertiary structure spite sequence divergence The result<br>support possible control mechanism frameshifting propose Brierley Jacks<br>\", \"The rubella virus glycoprotein arrest novel post pre<br>Golgi compartment Evidence accumulate distinct compartment exist secretory<br>pathway interpose rough RER Golgi stack study define<br>novel post RER pre Golgi compartment unassembled subunit<br>rubella virus glycoprotein accumulate When express CHO cell<br>absence glycoprotein transport Golgi complex arrest The compartment<br>accumulate consist tubular network smooth membrane continuity RER<br>distinctive property either RER Golgi previously characterize intermediate<br>compartment lack RER Golgi membrane protein disrupt agent<br>disrupt either RER thapsigargin ionomycin Golgi nocodazole brefeldin<br>However luminal proteins bear KDEL signal access compartment<br>Kinetically site arrest lie distal site palmitylation occur<br>proximal low temperature degree block Taken together find<br>suggest site arrest corresponds locate close exit site<br>This compartment could identify morphologically highly amplified cell<br>overexpressing unassembled subunit may counterpart among transitional element<br>non transfected cell conclude site arrest may represent<br>new compartment differentiate proximal moiety intermediate compartment<br>\", \"Exploring challenge health system preparedness communicable disease Arbaeen<br>mass gather qualitative study Background Infectious diseases common<br>problem mass gathering especially lack health system preparedness<br>Since Iran one important country walk path Arbaeen<br>vital role provide health service pilgrims experience health<br>challenge participant key importance The aim study explore<br>stakeholder experience health system preparedness challenge provide suggestion<br>prevent infectious disease Arbaeen mass gathering Methods qualitative<br>research method use conventional content analysis approach The<br>number participant include executive manager health policymakers enter<br>study among participant Semi structure interview use generate<br>data Interviews analyze mean content analysis face face<br>interview Results Data analysis result extraction four main<br>theme sub theme Health infrastructure defect Iraq three<br>sub theme health abandonment Iraq weaknesses health culture<br>problem relate health system poor control causative factor<br>infectious disease three sub theme underlying factor prevalence<br>contagious disease health system response communicable disease ignore<br>risk Arbaeen ceremony low perception risk pilgrim three<br>sub theme lack awareness pilgrim fatalism pilgrims unhygienic<br>belief pilgrim ineffectiveness health education two sub theme<br>train shortage target group educational content problem show<br>participant experience health system challenge cop infectious disease<br>Arbaeen ceremony Conclusion Pilgrim base training plan control<br>challenge may change threat opportunity improve health participant<br>mass gather Arbaeen region<br>\", \"Multicenter EuroTravNet GeoSentinel Study Travel relate Infectious Diseases<br>Europe analyze prospective data European patient seek treatment<br>GeoSentinel sit Gastrointestinal illness particularly tourists fever visit<br>friend relatives VFRs skin disorder tourist common reason<br>seek medical care Diagnoses vary country origin region<br>visit category traveler VFRs return sub Saharan Africa<br>Indian Ocean islands likely experience falciparum malaria group<br>Multiple correspondence analysis identify Italian French Swiss VFRs<br>expatriate traveler sub Saharan Africa Indian Ocean Islands<br>likely exhibit febrile illnesses German tourist Southeast south<br>central Asia likely seek treatment acute diarrhea Non<br>European travelers patient industrialized country less likely acquire<br>certain travel associate infectious disease These result consider<br>practice travel medicine development health recommendation European traveler<br>\", \"Infectiologie itin\\u00e9rante R\\u00e9sum\\u00e9 Objectifs Une prise conscience impose<br>\\u00e9gard d\\u00e9veloppement actuel des \\u00e9pid\\u00e9mies infection s\\u00e9v\\u00e8res d\\u00e9veloppement<br>consid\\u00e9rable transport internationaux beaucoup amplifi\\u00e9 circulation agent infectieux<br>v\\u00e9hicul\\u00e9s par voyageurs migrant mais aussi par les<br>animaux marchandises Observations distribution infection potentiel \\u00e9pid\\u00e9mique dont<br>certaines malady \\u00e9mergentes tendance devenir plan\\u00e9taire Cette \\u00e9volution<br>impose renforcement surveillance internationale capacit\\u00e9s r\\u00e9ponse aux alertes<br>Solutions Les pay industrialis\\u00e9s mieux \\u00e9quip\\u00e9s moyens structure<br>op\\u00e9rationnelles interviennent fa\\u00e7on pr\\u00e9pond\\u00e9rante dans cette lutte Leur<br>contribution est n\\u00e9cessaire dans les pay d\\u00e9veloppement particuli\\u00e8rement<br>affect\\u00e9s par les end\\u00e9mo \\u00e9pid\\u00e9mies infectieuses cette contribution<br>progresse doit \\u00eatre soutenue Cette revue rapporte travers<br>des observation personnelles pathology les plus mena\\u00e7antes travers<br>monde Objectives become aware current development international travel<br>contribute amplify circulation sort infectious agent describe current<br>situation Observations Infectious agent carry mean traveller immigrant<br>well animal good merchandise The distribution infection potentially<br>responsible epidemic tend become worldwide emergent disease Solutions<br>Such situation impose enhancement international surveillance development resource<br>control response alert Developed country posse advance equipment<br>interventional structure may intervene predominantly fight epidemic The<br>necessary contribution industrialize country increase develop country infectious<br>epidemic spread intervention must sustain The content article<br>review major threaten infectious epidemic disease contribution personal<br>experience<br>\", \"Infection Prevention Control Tropics Tropical setting present unique<br>challenge practice infection prevention control These multi faceted<br>due difference climate culture social political milieu low<br>middle income country situate tropic well lack resource<br>The emergence communicable disease low vaccination coverage also<br>lead nosocomial augmentation community outbreaks increase economic burden<br>hospital management Addressing challenge require innovative low cost<br>tailor solution suit tropical environment<br>\", \"The Politics SARS Rational Responses Ambiguity Symbols Chaos<br>The main focus article SARS event political process<br>involve political leader administrator health professional How understand<br>reaction SARS main actor institution What aspects preoccupy<br>definition SARS change process selection jurisdiction choose China<br>Hong Kong SAR Canada World Health Organization explore<br>question The start point view reaction can not<br>primarily see instrumental base rational standard operating procedure<br>SOPs technical expertise may better understood garbage perspective<br>From review event publicly report find suggested garbage<br>theory politician administrator response SARS outbreak combination compete<br>rationality overlap agendas Critical decision trigger extraneous factor<br>administrative action shape dramatic switch one set standard<br>operating procedures another event unfold The public health<br>issue constantly vie agendas compelling alignment among occurred<br>professional technical rationale solve problem become dominant<br>\", \"Low medium low risk perception dengue chikungunya Zika<br>outbreaks infectious disease physician France Western Europe BACKGROUND<br>Many tropical country currently experience dengue DEN chikungunya<br>CHIK also recently Zika ZIKA epidemic particularly Latin<br>America Although risk transmission spread infection temperate region<br>remain controversial issue vector borne diseases widely report<br>medium focus preventive strategy national international policy maker<br>public health authority context want determine extent risk<br>perception infectious disease physician current future risk arboviral<br>disease introduction autochthonous case development epidemic scenario France<br>Western Europe METHODS aim developed original standardize questionnaire<br>survey disseminate French Infectious Diseases Society physician member<br>RESULTS find physician perceive risk introduction outbreak development<br>DEN CHIK ZIKA France low medium low Generalized<br>Linear Model identify medical school training extent professional<br>experience awareness French national plan regard arboviral infection<br>significant predictor low risk perception among respondent CONCLUSION<br>Despite fact arboviral disease increasingly import France sometimes<br>result sporadic autochtonous transmission French physicians perceive risk<br>high Better communication education target health professional citizen<br>need enhance effectiveness French national plan prepare arboviral<br>disease ELECTRONIC SUPPLEMENTARY MATERIAL The online version article<br>s12889 contains supplementary material available authorized user<br>\", \"insightful recollection since distort key theory bear year<br>ago During last three decade many effort make<br>study protein cleavage sit disease cause enzyme HIV<br>human immunodeficiency virus protease SARS severe acute respiratory<br>syndrome coronavirus main proteinase become increasingly clear via<br>minireview motivation drive aforementioned study quite wise result<br>acquire study reward particularly develop peptide drug<br>\", \"Dark Side Therapies Immunoglobulin Concentrates The Adverse Events<br>Therapy human immunoglobulin IgG concentrate success story ongoing<br>decade ever increase demand plasma product The success<br>IgG concentrate clinical level document slowly increase number<br>register indication rapid increase label use topic dealt<br>another contribution special issue Frontiers Immunology part success<br>adverse event profile IgG concentrate even life long<br>need therapy excellent Transmission pathogen last decade could<br>entirely control antecedent introduction authority regulatory network instal<br>quality standard plasma fractionation industry The cornerstone regulatory<br>network current good manufacturing practice Non infectious AEs<br>occur rarely mainly mild moderate However recent time<br>increase frequency hemolytic thrombotic AEs raise worry question<br>possible background AEs Below review element non infectious<br>AEs particularly focus hemolysis thrombosis discuss introduction plasma<br>fractionation ion exchange chromatography polish immunoaffinity chromatographic step<br>might alter repertoire specificity influence profile efficacy IgG<br>concentrate<br>\", \"One health ecological evolutionary framework tackle Neglected Zoonotic<br>Diseases Understanding complex population biology transmission ecology multihost<br>parasite declare one major challenge biomedical science 21st<br>century Neglected Zoonotic Diseases NZDs perhaps neglect Neglected<br>Tropical Diseases NTDs Here consider multihost parasite transmission<br>evolutionary dynamic may affect success human animal disease<br>control program particularly neglect disease develop world review<br>different type zoonotic interaction occur ecological evolutionary potential<br>relevance current human control activity make suggestion development<br>empirical evidence base theoretical framework well understand predict<br>outcome interaction particular consider whether preventive chemotherapy current<br>mainstay NTD control successful without One Health approach<br>Transmission within animal reservoir human important ecological evolutionary<br>consequence drive evolution establishment drug resistance well provide<br>selective pressure spill\\u2010over host switching hybridization introgressions animal<br>human parasites Our aim highlight importance elucidate disease<br>ecology include identify key host tailor control effort<br>accordingly understanding parasite evolution precisely infectious agent may<br>respond adapt anthropogenic change Both elements essential alleviate<br>disease risk NZDs human domestic animal wildlife<br>\", \"The forecasting International Expo tourism use quantitative qualitative<br>technique This study predict number visitor international tourism<br>Expo hold Korea unprecedented event host city Forecasting<br>demand mega event receive limited attention literature Unlike<br>study forecast international tourism demand forecast Expo demand<br>involve use quantitative forecasting model qualitative technique data<br>limitation Combining quantitative technique willingness visit WTV survey<br>predict Expo demand million visitor comparison use Delphi<br>method expert predict Expo demand million visitor For<br>study Delphi method provide conservative estimate estimate combine<br>quantitative technique WTV Policy implication present directed toward<br>Expo planner practitioner term demand supply side application<br>result decision make process future challenge surround demand<br>forecasting<br>\", \"farm biosecurity perceive professional visit Swedish farm BACKGROUND<br>farm biosecurity important part disease prevention control apply<br>live animal contact well indirect contact via professional<br>visit farm work The objective study investigate professional<br>visit animal farm Sweden daily work perceive farm<br>condition biosecurity factor influence biosecurity routine describe obstacle<br>biosecurity Suggestions improvement also ask Questionnaires distribute professional<br>visit farm daily work veterinarian livestock haulier artificial<br>insemination technician animal welfare inspector cattle hoof trimmer<br>The sample convenience sample base accessibility register collaboration<br>organisation distribute questionnaire Respondents ask availability certain biosecurity<br>condition related farm visit facility hand wash available<br>important different factor routine open end question describe<br>obstacle suggestion improvement RESULTS After data clean response<br>person There difference proportion visit farm report certain<br>biosecurity measure place relate animal specie present farm<br>general visit pig farm high proportion biosecurity measure<br>place whereas condition poorer sheep goat farm horse<br>farm There also difference visitor category perceive condition<br>biosecurity vary group livestock haulier access hand wash<br>facility often veterinarians group majority respondent perceive obstacle<br>farm biosecurity among veterinarian perceive obstacle Many report<br>obstacle related basic biosecurity access soap water Responsibility<br>identify key issue farmer expect visitor take responsibility<br>keep biosecurity provide adequate farm condition CONCLUSIONS Many<br>respondent report obstacle keep good biosecurity relate farm<br>condition There gap come responsibility need clarify Visitors<br>need take responsibility avoid spread disease farmer need<br>assume responsibility provide adequate condition farm biosecurity<br>\", \"Translational Rodent Models Research Parasitic Protozoa\\u2014A Review Confounders<br>Possibilities Rodents particular Mus musculus long invaluable history<br>model human diseases biomedical research although translational value<br>challenge number case provide example rodent suboptimal model<br>human biology discus confounders influence experiment may explain<br>misleading result Infections rodent protozoan parasite exception require<br>close consideration upon model choice focus significant difference<br>inbred outbred wild animal importance factor microbiota gain<br>attention crucial variable infection experiment Frequently mouse rat<br>model choose convenience availability institution rather unbiased evaluation<br>whether provide answer give question Apart general discussion<br>translational success failure provide example infection single celled<br>parasite choose lab rodent give contradictory misleading result<br>possible discuss reason present emerge alternative traditional rodent<br>model humanize mice organoid primary cell culture call<br>recombinant inbred strain Collaborative Cross collection also potential<br>solution certain challenge addition emphasize advantage use wild<br>rodent certain immunological ecological behavioral question The experimental<br>challenge availability specie specific reagent come use non<br>model system also discuss Our intention foster critical<br>judgment traditional newly available translational rodent model research<br>parasitic protozoa complement exist mouse rat model<br>\", \"Nonconventional opponent review malaria leishmaniasis among United States<br>Armed Forces United States military engage different country<br>culture throughout world personnel become exposed new biosphere<br>well There many infectious pathogen endemic two particular<br>importance Plasmodium Leishmania respectively cause malaria leishmaniasis These<br>parasite know cause significant disease burden endemic locale<br>thus pose threat military traveler This review introduce<br>reader basic life cycle disease mechanism Local military<br>epidemiology describe specific action take military prevention treatment<br>purpose Complications measure regard human health also discuss<br>include possible chemical toxicity Additionally poor recognition diseases<br>upon individual return lead complication treatment delay United<br>States examine Information canine leishmaniasis poorly study relative<br>human manifestation importance due utilization dog military endeavor<br>present Future implication American healthcare system regard malaria<br>leishmaniasis also present<br>\", \"SARS hospital priority set qualitative case study evaluation<br>BACKGROUND Priority set one difficult issue face hospital<br>fund restriction change patient need deadly communicable disease<br>outbreak Severe Acute Respiratory Syndrome SARS Toronto amplifies<br>difficulties hospital priority set The purpose study describe<br>evaluate priority set hospital response SARS use ethical<br>framework accountability reasonableness METHODS This study conduct large<br>tertiary hospital Toronto Canada There two data source<br>key document email bulletin interview key informant Analysis<br>use modify thematic technique three phase open cod<br>axial cod evaluation RESULTS Participants describe type priority<br>setting decision decision making process reason use Although<br>hospital leadership make effort meet condition accountability reasonableness<br>acknowledge decision make ideal describe good practice opportunity<br>improvement CONCLUSIONS Accountability reasonableness framework use guide fair<br>priority set health care organization hospital midst crisis<br>SARS guidance incomplete consequence uncertain information constantly change<br>hour hour decision involve life death fairness important<br>rather less<br>\", \"Tackling challenge health equity China Summary term economic<br>development China widely acclaim miracle economy Over period<br>rapid economic growth however China reputation health slip<br>1970s China shin example health development longer Government<br>public concern health equity grow China health equity<br>challenge truly daunt vicious cycle three synergistic factor<br>social determinant health become inequitable imbalance roles market<br>government develop concern among public grow fairness health<br>With economic boom grow government revenue China unlike<br>country challenge health inequity afford necessary reform economic<br>development hand hand improve health equity Reforms improve<br>health equity receive immense popular support governmental commitment<br>interest public health community worldwide<br>\", \"Emerging Reemerging Infectious Diseases Global Overview<br>\", \"Coping flexibility college student depressive symptom BACKGROUND The<br>current study explore prevalence depress mood among Chinese<br>undergraduate student examine cop pattern degree flexibility flexibility<br>pattern associate mood METHODS set questionnaire assess cop<br>pattern cop flexibility depressive symptom administer student men<br>woman RESULTS total participant complete entire set questionnaire<br>report frequency two stressful life event criterion need<br>calculate variance perceive controllability Findings show high level<br>depressive symptom significantly associate high level event frequency<br>event impact low level perceive controllability cop effectiveness<br>ratio strategy situation fit Depressive symptom significantly associate<br>cognitive flexibility variance perceive controllability Gender significant moderator<br>report association CONCLUSIONS Findings indicate Chinese university student<br>depressive symptom report experience great number negative event<br>non depress university student addition undergraduate depressive symptom<br>likely undergraduates utilize maladaptive cop method Such finding<br>highlight potential importance intervention aim help undergraduate student<br>low cop flexibility develop skill cope stressful life<br>event<br>\", \"Ruling trouble Unacceptable behaviour control Hong Kong public<br>housing Resident satisfaction contingent housing design construction also<br>neighbourhood quality However many neighbourhood around world distress<br>neighbourhood nuisance unacceptable behaviour resident While interventionist approach<br>incentive adopt many country curb problem punitive measure<br>use Hong Kong instead The Housing Department launch<br>Marking Scheme Tenancy Enforcement Public Housing Estates immediately<br>SARS epidemic The scheme operate penalty point system<br>sit tenant expel public housing receive sixteen point<br>misdeed commit Yet mark scheme put onto stage<br>without prior public consultation Besides criticised unfair tenure<br>bias enforcement Also whether scheme widely accept highly<br>doubtful Against background study aim explore tenant view<br>regard resident unacceptable behaviour mark scheme Hong Kong<br>public housing The finding research offer valuable insight<br>perceive extent cause neighbourhood problem Moreover research let<br>public administrator know acceptability mark scheme among public<br>housing tenant<br>\", \"Les virus bougent p\\u00e9rils plan\\u00e9taires SUMMARY Emerging viral<br>disease nothing new Smallpox probably reach Europe Asia<br>5th century yellow fever emerge Americas 16th century<br>consequence African slave trade Dengue fever arise simultaneously<br>South East Asia Africa North America 18th century<br>1918\\u20131919 call Spanish flu spread like wildfire five<br>continent kill million people The second half 20th<br>century saw emergence HIV AIDS among viral disease<br>Even worry fact emerge emerge viral disease tendency<br>spread quickly widely last decade invade whole country<br>continent witness recent outbreak Nipah virus West Nile<br>Rift Valley fever SARS monkeypox avianflu H5N1 Chikungunya<br>The complex factor underlie new trend briefly discuss<br>\", \"Bibliometric Analysis Publications Pluripotent Stem Cell Research OBJECTIVE<br>Human pluripotent stem cell self renew cell ability<br>differentiate variety cell view great potential field regenerative<br>medicine Research pluripotent stem cell hold great promise<br>patient specific therapy various disease study pluripotent stem<br>cell article publish screen retrieve Science Citation Index<br>Expanded SCI EXPANDED MATERIALS AND METHODS retrospective study<br>publication trend citation trend top article distribution journal<br>Web Science category analyze Five bibliometric indicator include<br>total article independent article collaborative article first author<br>article correspond author article apply compare publication country<br>institution RESULTS The impact top article change year<br>year Top cite article previous publication year recent<br>year Induced pluripotent stem cell embryonic stem cell<br>use author keywords pluripotent stem cell research addition<br>winner Nobel Prize physiology medicine Prof Shinya Yamanaka<br>publish four top ten frequently cite article CONCLUSION<br>The comprehensive analysis highly cite article stem cell<br>field could identify milestone important contributor give historic<br>perspective scientific progress<br>\", \"Repositioning salicylanilide anthelmintic drug treat adenovirus infection The<br>reposition drug already approve regulatory agency indication emerge<br>alternative development new antimicrobial therapy The reposition process<br>involve low risk cost novo development novel antimicrobial<br>drug Currently infection adenovirus show steady increment high<br>clinical impact immunosuppressed immunocompetent patient The lack safe<br>efficacious drug treat infection support search new antiviral<br>drug Here evaluate anti adenovirus activity niclosanide oxyclozanide<br>rafoxanide three salicylanilide anthelmintic drug Also carry cytotoxicity<br>evaluation partial characterization mechanism action drug The salicylanilide<br>anthelmintic drug show significant anti adenovirus activity low<br>micromolar concentration little cytotoxicity Moreover mechanistic assay suggest<br>difference way drug exert anti adenovirus activity Niclosamide<br>rafoxanide target transport HAdV particle endosome nuclear envelope<br>whilst oxyclozanide specifically target adenovirus immediately early gene<br>E1A transcription Data suggest study salicylanilide anthelmintic drug<br>could suitable clinical evaluation development new antiviral drug<br>treat infection adenovirus immunosuppressed patient immunocompetent individual community<br>acquire pneumonia<br>\", \"Toxic proteins plant Plants evolve synthesize variety noxious<br>compound cope unfavorable circumstance among large group toxic<br>proteins play critical role plant defense predator microbes<br>wide range harmful protein discover different plant include<br>lectin ribosome inactivating protein protease inhibitor ureases arcelins<br>antimicrobial peptide pore form toxin fulfill role plant<br>defense proteins exhibit various degree toxicity towards animal<br>insect bacteria fungi Numerous study carry investigate toxic<br>effect mode action plant proteins order explore possible<br>application Indeed biological activity toxic plant protein also<br>consider potentially useful tool crop protection biomedical application<br>cancer treatment Genes encode toxic plant protein introduce<br>crop genome use genetic engineering technology order increase<br>plant resistance pathogen diseases Despite availability ample information<br>toxic plant proteins publication attempt summarize research progress<br>make last decade This review focus diversity toxic<br>plant proteins view toxicity well mode action Furthermore<br>outlook towards biological role proteins potential application discuss<br>\", \"Inhibition SARS CoV 3CL protease flavonoid There severe<br>panic cause Severe Acute Respiratory Syndrome SARS Middle<br>East Respiratory Syndrome Coronavirus Therefore research target virus<br>require Coronaviruses CoVs rise target flavonoids The antiviral<br>activity flavonoid CoVs presume directly cause inhibit like<br>protease 3CLpro Here apply flavonoid library systematically probe<br>inhibitory compound SARS CoV 3CLpro Herbacetin rhoifolin pectolinarin<br>find efficiently block enzymatic activity SARS CoV 3CLpro<br>The interaction three flavonoid confirm use tryptophan base<br>fluorescence method induced fit dock analysis indicate S3\\u2032<br>sit involve bind flavonoid The comparison previous study<br>show Triton play critical role object false positive<br>overestimated inhibitory activity flavonoid With systematic analysis three<br>flavonoid suggest template design functionally improve inhibitor<br>\", \"Proteome analysis vaccinia virus IHD infect HEK cell<br>dimensional gel electrophoresis MALDI PSD TOF solid phase<br>support terminally sulfonated peptide BACKGROUND Despite successful eradication<br>smallpox WHO lead vaccination programme pox virus infection<br>remain considerable health threat The possible use smallpox<br>bioterrorism agent well continuous occurrence zoonotic pox virus<br>infection document relevance deepen understand virus host interaction<br>Since permissiveness pox infection independent host surface receptor<br>correlate ability virus infiltrate antiviral host response directly<br>depend host proteome set report proteome HEK293 cell<br>infect Vaccinia Virus strain IHD analyze dimensional gel<br>electrophoresis MALDI PSD TOF bottom approach RESULTS The<br>cellular viral proteome VACV IHD infect HEK293 cell<br>inactivate VACV IHD treat well non infect cell<br>compare Derivatization peptides sulfophenyl isothiocyanate SPITC carry ZipTip\\u03bc<br>C18 column enable protein identification via peptide primary<br>sequence provide improved ratio well signal intensity PSD<br>spectra The expression human protein modulate viral infection<br>Effects inactivate infectious virus host proteome concern energy<br>metabolism protein associate gene expression protein biosynthesis quite<br>similar These effect might therefore attribute virus entry<br>virion proteins However modulation protein involve apoptosis clearly<br>correlate infectious virus CONCLUSIONS The proteome analysis infected<br>cell provide insight apoptosis modulation regulation cellular gene<br>expression regulation energy metabolism The confidence protein identification<br>clearly improve peptide derivatization SPITC solid phase support<br>Some identified protein describe context poxvirus infection need<br>characterise identify mean apoptosis modulation pathogenesis<br>\", \"Recent Reports Solid Phase Cyclohexapeptide Synthesis Applications Macrocyclic<br>peptides privileged scaffold drug development constitute significant portion<br>macrocyclic drug market today field span infectious disease<br>oncology Developing orally bioavailable peptide base drug remain<br>challenge task however macrocyclization linear peptide effective strategy<br>improve membrane permeability proteolytic stability oral bioavailability overall<br>drug like characteristic class Significant advance solid phase<br>peptide synthesis SPPS enable efficient construction macrocyclic peptide<br>peptidomimetic library macrolactamization perform resin solution phase The<br>primary goal review summarize solid phase cyclohexapeptide synthesis<br>use resin solution phase macrocyclization methodology publish since<br>also highlight broad application range natural product total<br>synthesis synthetic methodology development medicinal chemistry drug development<br>analyse conformational physiochemical property<br>\", \"Influence supercritical water treatment heavy metal medical waste<br>incinerator fly ash work medical waste incinerator fly<br>ash different type incinerator subject supercritical water SCW<br>SCW H2O2 SCWH treatments Sequential extraction experiment show<br>SCW treatment heavy metal exchangeable carbonate form ash<br>could transfer relatively stable form residual fraction organic<br>matter fraction SCWH treatment could stabilize heavy metal<br>Fe\\u2013Mn oxides residual fraction However behavior quite different<br>heavy metal could leach residue fraction SCW SWCH<br>treatments The leached tended absorb onto Fe\\u2013Mn oxides<br>organic matter near neutral environment could react Ca2<br>low increase mobility element Therefore necessary neutralize acidic<br>ash near neutral condition subject SCW SCWH treatment<br>effectively stabilize hazardous element ash Consequently believe SCWH<br>treatment effective alternative hazardous element detoxification fly ash<br>\", \"High Throughput Screening Small Molecule Enhancers Interferon Signaling<br>Pathway Drive Next Generation Antiviral Drug Discovery Most<br>current strategy antiviral therapeutic target virus specifically directly<br>alternative approach drug discovery might enhance immune response<br>broad range virus Based clinical observation human successful<br>genetic strategy experimental model reason improved interferon IFN<br>signal system might better protect viral infection Here<br>aim identify small molecular weight compound might mimic<br>beneficial effect improve antiviral defense Accordingly develop cell<br>base high throughput screen HTS assay identify small<br>molecule enhance IFN signal pathway component The assay<br>base phenotypic screen increase IFN stimulate response element<br>ISRE activity fully automate robust format Application assay<br>system library compound include already approve approvable drug<br>lead identification compound significant ISRE activity From choose<br>anthracycline antibiotic idarubicin validation mechanism base activity sub<br>range find idarubicin action increase ISRE activity manifest<br>member drug class independent cytotoxic topoisomerase inhibitory effect<br>well endogenous IFN signal production also observe compound<br>confer consequent increase IFN stimulate gene ISG expression<br>significant antiviral effect use similar dose range cell<br>culture system inoculate encephalomyocarditis virus EMCV The antiviral<br>effect also find compound concentration one observe cytotoxicity<br>Taken together result provide proof concept use activator<br>component IFN signal pathway improve IFN efficacy antiviral<br>immune defense well validated HTS approach identify small<br>molecule might achieve therapeutic benefit<br>\", \"Evaluation polyphenols Broussonetia papyrifera coronavirus protease inhibitors The<br>current study design assess inhibitory activity Broussonetia papyrifera<br>derive polyphenols chymotrypsin like papain like coronavirus cysteine<br>proteases The isolated compound broussochalcone broussochalcone hydroxyisolonchocarpin papyriflavonol<br>methylbut enyl trihydroxyflavane kazinol kazinol broussoflavan kazinol kazinol<br>All polyphenols potent papain like protease pro chymotripsin<br>like protease 3CL pro therefore investigated structural feature<br>responsible selectivity Compound potent inhibitor pro value The<br>active compound display kinetic behavior bind constant interaction<br>pro determine surface plasmon resonance analysis Our result<br>suggest papyrifera constituent promise candidate development potential anti<br>coronaviral agent<br>\", \"Broad spectrum antiviral activity carbodine carbocyclic analogue cytidine<br>Carbocyclic cytidine Cyd broad spectrum antiviral agent active<br>DNA virus pox vaccinia RNA virus toga Sindbis<br>Semliki forest corona RNA viruses orthomyxo influenza paramyxo<br>parainfluenza measles rhabdo vesicular stomatitis RNA viruses reo<br>The target enzyme Cyd suppose CTP synthetase convert<br>UTP CTP keep assumption observation Cyd effect dose<br>dependent inhibition RNA synthesis virus infect uninfected cell<br>exogenous addition either Urd Cyd reverse antiviral cytocidal<br>activity Cyd whereas addition dThd dCyd fail The<br>selectivity Cyd Sindbis vesicular stomatitis reo virus markedly<br>increase Cyd combine Cyd This combination may therefore<br>worth pursue chemotherapeutic modality treatment virus infection<br>\", \"Structure base virtual screening SARS 3CLpro identify novel<br>non peptidic hit Severe acute respiratory syndrome highly<br>infectious upper respiratory tract disease cause SARS CoV<br>previously unidentified human coronavirus SARS 3CLpro viral cysteine<br>protease critical pathogen life cycle hence therapeutic target<br>importance The recently elucidate crystal structure enzyme provide<br>opportunity discovery inhibitor rational drug design current study<br>Gold dock program utilized conduct extensive docking study<br>target crystal structure develop robust predictive docking protocol<br>The validated docking protocol use conduct structure base<br>virtual screening Asinex Platinum collection Biological evaluation screen<br>selection compound carry identify novel inhibitor viral protease<br>\", \"Computational Docking Study Ion Channel HCV Genotype Genotype<br>Its Interaction Natural Compounds BACKGROUND The current standard<br>care therapy hepatitis virus HCV infection consist two<br>regime namely interferon base interferon free treatment The<br>treatment combination ribavirin pegylated interferon expensive mildly effective<br>associate severe side effect two direct act antiviral<br>DAA drug boceprevir telaprevir license show enhanced sustain<br>virologic response SVR phase III clinical trial however<br>interferon free treatment sensitive HCV genotype infection The<br>variable nature HCV limit number inhibitor develop thus<br>aim expand repertoire available drug target result target<br>virus assembly therapeutically AIM conduct study predict structure<br>protein HCV genotypes Approximately amino acid residue encode<br>HCV render channel sensitive inhibitor make promising target<br>novel therapy HCV protein form small membrane know<br>viroporin essential effective self assembly large channel conduct<br>cation assembly discharge infectious virion particle METHOD study<br>screen drug flavonoid know disrupt translation production HCV<br>proteins target active site residue HCV genotype GT3<br>isolatek3a HCV genotype GT4 isolateED43 Furthermore conduct quantitative<br>structure\\u2013activity relationship dock interaction study RESULTS The drug<br>DNJ form high number hydrogen bond interaction model<br>protein high interaction energy follow BIT225 flavonoid screen<br>demonstrate Epigallocatechin gallate EGCG nobiletin quercetin bind mode<br>GT3 GT4 Thus predict protein molecule HCV GT3<br>GT4 provide general avenue target structure base antiviral<br>compound CONCLUSIONS hypothesize inhibitor viral identify screen may<br>new class potent agent confirmation vitro vivo essential<br>This structure guide drug design GT3 GT4 lead<br>identification drug like natural compound confirm new target<br>rapidly increase era HCV<br>\", \"efficient method recovery target ssDNA base amino modify<br>silica coat magnetic nanoparticles paper improve recovery method<br>target ssDNA use amino modify silica coat magnetic<br>nanoparticles ASMNPs report This method take advantage amino<br>modify silica coat magnetic nanoparticles prepare use water<br>oil microemulsion technique employ amino modify silica shell<br>iron oxide core magnetic nanoparticles The nanoparticles silica<br>surface amino group conjugate desired bio molecule many<br>exist amino group chemistry research linear DNA probe<br>immobilize onto nanoparticles streptavidin conjugation use covalent bond<br>target ssDNA TMR CGCATAGGGCCTCGTGATAC successfully recovered crude sample<br>magnet field special recognition hybridization design ssDNA fragment<br>severe acute respiratory syndrome SARS virus much low<br>concentration target ssDNA also recover high efficiency good<br>selectivity<br>\", \"Phosphonylated Acyclic Guanosine Analogues Triazole Linker novel series<br>amino chloro purin methyl triazol alkylphosphonates amino oxo<br>dihydro purin methyl triazol alkylphosphonates acyclic analogue guanosine<br>synthesize assessed antiviral activity broad range DNA RNA<br>viruses cytostatic activity toward three cancerous cell line<br>HeLa L1210 CEM They devoid antiviral activity however<br>several phosphonates find slightly cytostatic HeLa cell 80\\u2013210<br>range Compounds 17k 17k show high inhibitory effect<br>15\\u201330 proliferation murine leukemia L1210 human lymphocyte CEM<br>cell line<br>\", \"Production anti Trichophyton rubrum egg yolk immunoglobulin therapeutic<br>potential treating dermatophytosis The aim study estimate therapeutic<br>potential specific egg yolk immunoglobulin IgY dermatophytosis cause<br>Trichophyton rubrum The IgY produce immunize hen cell<br>wall protein rubrum extract egg PEG precipitation purify<br>ammonium sulfate precipitation The cross reactivity fungi growth<br>inhibition rubrum vitro therapeutic effect rubrum infection BALB<br>mouse specific IgY evaluate Anti rubrum cell wall<br>proteins IgY anti trCWP IgY present certain degree<br>cross reactivity different fungi vitro vivo activity research<br>Anti trCWP IgY show significant dose dependent growth<br>inhibitory effect rubrum vitro significant dose dependent therapeutic<br>effect rubrum infection BALB mouse<br>\", \"The phenylalanine ammonia lyase gene family Isatis indigotica<br>Fort molecular clone characterization expression analysis Phenolic compound<br>metabolite phenylpropanoid pathway play important role growth environmental<br>adaptation many plant Phenylalanine ammonia lyase PAL first<br>key enzyme phenylpropanoid pathway The present study design<br>investigate whether multi gene family Indigotic characterize property<br>conduct comprehensive survey transcription profile database use tBLASTn<br>analysis Several bioinformatics method employ perform prediction composition<br>physicochemical character The expression level IiPAL gene various<br>tissue indigotica stress treatment examine quantitative real time<br>PCR Protoplast transient transformation use observe location IiPALs<br>IiPALs functionally characterize expression pET 32a vector Escherichia<br>colis strain BL21 DE3 Integration transcripts metabolite accumulation<br>use reveal relation IiPALs target compound new gene<br>IiPAL2 identify IiPALs conserve enzymatic active site Ala<br>Ser Gly classify member dicotyledon IiPAL1 IiPAL2 express<br>root stem leave flower high expression level IiPAL1<br>IiPAL2 observe stem root respectively The two gene<br>respond exogenous elicitor different manner Subcellular localization experiment<br>show IiPALs localize cytosol The recombinant protein show<br>catalyze conversion Phe trans cinnamic acid Correlation analysis<br>indicate IiPAL1 close biosynthesis secondary metabolite IiPAL2 conclusion<br>present study provide basis elucidation role IiPALs gene<br>biosynthesis phenolic compound help metabolic engineering improve accumulation<br>bioactive component indigotica<br>\", \"Biguanide modifiable pharmacophore recruitment endogenous inhibit cysteinyl cathepsins<br>review implication ABSTRACT Excessive activity cysteinyl cathepsins CysCts<br>contribute progress many disease however therapeutic inhibition problematic<br>natural inhibitor proteases CysHis dyads CysHis Xaa triad<br>Biguanide form bidentate metal complex two imino nitrogen<br>Here discuss phenformin phenylethyl biguanide model recruitment endogenous<br>inhibit CysHis CysHis peptidolysis Phenformin interactive anti proteolytic<br>agent bioassay live tissue Benzoyl arginine amide BAA<br>classical substrate papain like protease amide bond scissile<br>review structure BAA phenformin complex compare silico Their<br>chemistry dimension discuss light active site papain like<br>protease The phenyl moiety structure bind substrate bind<br>site typical many protease When phenyl moiety BAA<br>bind scissile amide bond directed position thiolate imidazolium<br>ion pair hydrolyze However phenyl moiety phenformin bind<br>coordinate direct identical position catalysis inhibit Phenformin stabilize<br>sandwich drug protease active site Hundreds biguanide derivative<br>synthesize nitrogen position many conceivable Various substituent moiety<br>register various array substrate bind site align coordinate<br>catalytic partner diverse protease Biguanide identify modifiable pharmacophore<br>synthesis therapeutic CysCt inhibitor wide range potency specificities<br>GRAPHICAL ABSTRACT Phenformin Complex Image see text<br>\", \"Structural characterization immunomodulatory effect polysaccharide HCP Houttuynia cordata<br>Immunomodulation natural polysaccharide hot topic research recent year<br>order explore immunomodulatory effect Houttuynia cordata Thunb water<br>extract study polysaccharide HCP molecular weight 000Da isolated<br>chromatography use DEAE Sepharose Sephacryl columns The structure<br>characterization HCP perform Fourier transform infrared spectroscopy FTIR<br>acidic hydrolysis PMP derivation HPLC analysis nuclear magnetic<br>resonance spectrum NMR HCP elucidate pectic polysaccharide linear<br>chain link galacturonic acid residue part carboxyl group<br>methyl esterify part hydroxyl group acetylate The bioactivity<br>assay show HCP could increase secretion interleukin tumor<br>necrosis factor TNF macrophage inhibitory protein MIP macrophage<br>inhibitory protein MIP RANTES regulate activation normal cell<br>express secreted human peripheral blood mononuclear cell PBMCs<br>play critical role innate immune system shape adaptive<br>immunity Our result imply HCP could immune enhancer<br>\", \"Simple Platform Rapid Development Antimicrobials Recent infectious outbreak<br>highlight need platform technology quickly deploy develop therapeutic<br>need contain outbreak present simple concept rapid development<br>new antimicrobial The goal produce little one week<br>thousand dose intervention new pathogen test feasibility system<br>base antimicrobial synbodies The system involve create array<br>peptide select broad capability bind kill viruses bacteria<br>The peptide pre screen low cell toxicity prior<br>large scale synthesis Any pathogen assay chip find<br>peptides bind kill Peptides combine pairs synbodies screen<br>activity toxicity The lead synbody quickly produce large<br>scale completion entire process one week<br>\", \"Effects Non Conservative Sequence Properties glucuronidase Aspergillus terreus<br>clone glucuronidase gene AtGUS Aspergillus terreus encode amino<br>acid transform glycyrrhizin glycyrrhetinic acid monoglucuronide GAMG glycyrrhetinic<br>acid Based sequence alignment terminal non conservative sequence<br>show low identity specie thus partial sequence AtGUS<br>1\\u2013592 amplify determine effect non conservative sequence enzymatic<br>property AtGUS AtGUS express coli BL21 produce AtGUS<br>AtGUS respectively similar optimum temperature 55\\u00b0C AtGUS AtGUS<br>condition thermal stability AtGUS enhance 65\\u00b0C metal ion<br>show opposite effect AtGUS AtGUS respectively Furthermore AtGUS<br>nearly one seventh AtGUS whereas catalytic efficiency AtGUS<br>fold high AtGUS reveal truncation non conservative sequence<br>significantly improve catalytic efficiency AtGUS Conformational analysis illustrate<br>significant difference secondary structure AtGUS AtGUS circular dichroism<br>The result show truncation non conservative sequence could<br>preferably alter influence stability catalytic efficiency enzyme<br>\", \"Graphene graphene oxide biofunctionalization application biotechnology Graphene basic<br>building block fullerene carbon nanotube graphite Graphene unique<br>planar structure well novel electronic property attract great<br>interest scientist This review selectively analyzes current advance<br>field graphene bioapplications particular biofunctionalization graphene biological application<br>fluorescence resonance energy transfer base biosensor development use<br>graphene graphene base nanomaterials investigation graphene graphene base<br>nanomaterials live cell study summarize detail Future perspectives<br>possible challenge rapidly develop area also discuss<br>\", \"Lobeglitazone Novel Thiazolidinedione Improves Non Alcoholic Fatty Liver<br>Disease Type Diabetes Its Efficacy Predictive Factors Related<br>Responsiveness Despite rapidly increase prevalence non alcoholic fatty<br>liver disease NAFLD type diabetes T2D treatment modality<br>currently available investigate hepatic effect novel thiazolidinedione TZDs<br>lobeglitazone Duvie T2D patient NAFLD recruit drug na\\u00efve<br>metformin treat T2D patient NAFLD conduct multicenter prospective<br>open label exploratory clinical trial Transient liver elastography<br>Fibroscan Echosens Paris France control attenuation parameter CAP<br>use non invasively quantify hepatic fat content Fifty<br>patient CAP value treat daily lobeglitazone week The<br>primary endpoint decline CAP value secondary endpoint include<br>change component glycemic lipid liver profile Lobeglitazone treat<br>patient show significantly decrease CAP value baseline week<br>regardless glycemic control Lobeglitazone improve HbA value well<br>lipid liver profile treat patient Moreover multivariable linear<br>regression analysis show hepatic fat reduction lobeglitazone independently<br>associate baseline value CAP liver stiffness liver enzyme<br>metformin use Lobeglitazone treatment reduce intrahepatic fat content<br>assess transient liver elastography improve glycemic liver lipid<br>profile T2D patient NAFLD Further randomize control trial<br>use liver histology end point necessary evaluate efficacy<br>lobeglitazone NAFLD treatment Clinical trial NCT02285205<br>\", \"Severe Acute Respiratory Syndrome Temporal Stability Geographic Variation<br>Death Rates Doubling Times analyze temporal stability geographic<br>trend cumulative case fatality rate average double time<br>severe acute respiratory syndrome SARS part account correlation<br>case fatality rate double time difference control measure<br>Factors may alter future estimate case fatality rate<br>reason heterogeneity double time among country implication control<br>SARS discuss<br>\", \"Prebiotic probiotic supplementation prevents rhinovirus infection preterm infant<br>randomize placebo control trial Background Simple safe strategy<br>prevention viral respiratory tract infection RTIs need Objective<br>hypothesize early prebiotic probiotic supplementation would reduce risk<br>virus associate RTIs first year life cohort preterm<br>infant Methods randomized double blind placebo control trial<br>ClinicalTrials gov NCT00167700 preterm infant gestational age \\u226532<br>\\u226436 week birth weight treat Turku University Hospital<br>Turku Finland allocate receive oral prebiotics galacto oligosaccharide<br>polydextrose mixture probiotic Lactobacillus rhamnosus ATCC placebo microcrystalline<br>cellulose day life The primary outcome incidence clinically<br>define virus associate RTI episode confirm nasal swab<br>use nucleic acid test Secondary outcome severity duration<br>RTIs Results significantly low incidence RTIs detect infant<br>receive prebiotics rate ratio probiotic compare receiving placebo<br>Also incidence rhinovirus induce episode comprise RTI episode<br>find significantly low prebiotic probiotic group compare placebo<br>group difference emerge among study group rhinovirus RNA<br>load infection duration rhinovirus RNA shed duration severity<br>rhinovirus infection occurrence rhinovirus RNA asymptomatic infant Conclusions<br>Gut microbiota modification specific prebiotics probiotic might offer<br>novel cost effective mean reduce risk rhinovirus infection<br>\", \"Non invasive ventilation acute respiratory failure Summary Non<br>invasive mechanical ventilation increasingly use avoid serve alternative<br>intubation Compared medical therapy instance invasive mechanical ventilation<br>improve survival reduces complication select patient acute respiratory<br>failure The main indication exacerbation chronic obstructive pulmonary<br>disease cardiogenic pulmonary oedema pulmonary infiltrates immunocompromised patient<br>wean previously intubate stable patient chronic obstructive pulmonary<br>disease Furthermore technique use postoperative patient neurological disease<br>palliate symptom terminally ill patient help bronchoscopy however<br>study need situation regard first line treatment Non<br>invasive ventilation implement alternative intubation provide intensive care<br>high dependency unit When use prevent intubation otherwise<br>stable patient safely administer adequately staffed monitor ward<br>\", \"Nontraumatic Osteonecrosis Distal Tibia Case Presentation Review Literature<br>Osteonecrosis although commonly occur hip also affect leg<br>foot foot commonly occur talus The incidence osteonecrosis<br>occur tibia relatively rare report case woman present<br>clinic ankle pain idiopathic nature Subsequent magnetic resonance<br>imaging show finding consistent osteonecrosis bilateral distal tibias<br>several lesion locate shoulder hip calcaneus The present<br>report also serve review etiology treatment osteonecrosis relate<br>low extremity<br>\", \"Pharmacotherapy Lower Respiratory Tract Infections Elderly\\u2014Focused Antibiotics Lower<br>respiratory tract infection LRTIs refer inflammation trachea bronchus<br>bronchiole lung tissue Old people increase risk develop<br>LRTIs compare young adult The prevalence LRTIs elderly<br>population relate underlying disease age also variety clinical<br>issue history hospitalization previous antibacterial therapy mechanical ventilation<br>antibiotic resistance These factor mention led increase prevalence<br>mortality LRTIs elderly new medical strategy target LRTIs<br>population urgently need After systematic review current randomize<br>controlled trial relate study recommend novel pharmacotherapies demonstrate<br>advantage management LRTIs people age also briefly review<br>current medication respiratory communicable disease elderly Various source<br>information use ensure relevant study include search Pubmed<br>MEDLINE OvidSP EMBASE OvidSP ClinicalTrials gov Strengths limitation<br>drug evaluate base whether novelty mechanism favorable pharmacokinetic<br>pharmacodynamic profile avoidance interaction intolerance simplicity dose ability<br>cope challenge mainly evaluated primary secondary endpoint The<br>purpose review recommend promise antibiotic treatment LRTIs elderly<br>hospital outpatient set base exist result clinical study<br>novel antibiotic briefly review current medication respiratory communicable<br>disease elderly aim well management LRTIs clinical practice<br>\", \"Influenza vaccination respiratory virus interference among Department Defense<br>personnel 2017\\u20132018 influenza season Purpose Receiving influenza vaccination<br>may increase risk respiratory virus phenomenon know virus<br>interference Test negative study design often utilized calculate<br>influenza vaccine effectiveness The virus interference phenomenon basic<br>assumption test negative vaccine effectiveness study vaccination change<br>risk infection respiratory illness thus potentially bias vaccine<br>effectiveness result positive direction This study aim investigate<br>virus interference compare respiratory virus status among Department<br>Defense personnel base influenza vaccination status Furthermore individual<br>respiratory virus association influenza vaccination examine Results compare<br>vaccination status people non influenza respiratory virus people<br>pan negative result Comparing vaccinated non vaccinated patient<br>adjust odds ratio non flu viruses confidence interval<br>Additionally vaccination status case influenza compare three different<br>control group control non influenza positive control pan<br>negative control The adjust ORs comparison among three<br>control group vary much range 46\\u20130 Conclusions Receipt<br>influenza vaccination associate virus interference among population Examining<br>virus interference specific respiratory virus show mixed result<br>Vaccine derive virus interference significantly associate coronavirus human<br>metapneumovirus however significant protection vaccination associate influenza virus<br>also parainfluenza RSV non influenza virus coinfections<br>\", \"Single detection human bocavirus high viral load severe<br>respiratory tract infection previously healthy child BACKGROUND Human<br>bocavirus newly discover parvovirus Multiple study confirm presence<br>human bocavirus1 HBoV1 respiratory tract sample child The<br>viral load presentation single detection role causative agent<br>severe respiratory tract infection thoroughly elucidate METHODS investigate<br>presence HBoV1 quantitative polymerase chain reaction PCR nasopharyngeal<br>aspirate specimen child hospitalize respiratory tract infections The<br>sample analyze respiratory virus PCR respiratory virus viral<br>culture RESULTS least one virus detect child two<br>virus detect child HBoV1 detect child case HBoV1<br>virus detect Seasonal variation observe high HBoV1 infection<br>rate summer cutoff value copy use distinguish high<br>low HBoV1 viral load nasopharyngeal aspirate High viral<br>load HBoV1 note predominantly absence viral agent whereas<br>primarily detection case low HBoV1 viral load There<br>difference clinical symptom severity HBoV1 single detection detection<br>case HBoV1 single detection high viral load group<br>prevalent among child dyspnea wheeze low viral load<br>group clinical severity significant difference record high viral<br>load low viral load group HBoV1 positive patient<br>associate severe respiratory tract infection patient belong HBoV1<br>high viral load group patient case HBoV1 single<br>detection CONCLUSIONS HBoV1 high viral load frequently find<br>detection respiratory virus single detection high viral load<br>could etiological agent severe respiratory tract infection ELECTRONIC<br>SUPPLEMENTARY MATERIAL The online version article doi contains<br>supplementary material available authorized user<br>\", \"cluster randomize clinical trial comparing fit\\u2010tested non\\u2010fit\\u2010tested N95<br>respirator medical mask prevent respiratory virus infection health<br>care worker Please cite paper MacIntyre cluster randomize<br>clinical trial comparing fit\\u2010tested non\\u2010fit\\u2010tested N95 respirator medical<br>mask prevent respiratory virus infection health care worker<br>Influenza Other Respiratory Viruses DOI 1750\\u20102659 Background compare<br>efficacy medical mask N95 respirator fit test non<br>fit test health care worker HCWs Methods cluster<br>randomize clinical trial RCT HCWs Beijing hospital perform<br>winter Participants wear masks respirator entire work shift<br>week Outcomes include clinical respiratory illness CRI influenza\\u2010like<br>illness ILI laboratory\\u2010confirmed respiratory virus infection influenza convenience<br>no\\u2010mask respirator group health worker nine hospital compare<br>Findings The rate CRI 3\\u00b79 versus 6\\u00b77 ILI<br>0\\u00b73 versus 0\\u00b76 laboratory\\u2010confirmed respiratory virus 1\\u00b74 versus<br>2\\u00b76 influenza 0\\u00b73 versus infection consistently low N95<br>group compare medical mask intention\\u2010to\\u2010treat analysis value adjust<br>cluster non\\u2010fit\\u2010tested N95 respirator significantly protective medical mask<br>CRI outcomes significant The rate outcomes high convenience<br>no\\u2010mask group compare intervention arm There significant difference<br>outcome N95 arm without fit test Rates fit<br>test failure low post hoc analysis adjust potential<br>confounders N95 mask hospital level significant medical mask<br>vaccination handwash high\\u2010risk procedure Interpretation Rates infection medical<br>mask group double N95 group benefit respirator suggest<br>would need confirm large trial study may underpowered<br>The finding fit test specific type respirator use<br>study can not generalize respirator Trial registration Australian<br>New Zealand Clinical Trials Registry ANZCTR ACTRN ACTRN12609000257268<br>http www anzctr org<br>\", \"Multiplex PCR reveal virus frequent bacteria child cystic<br>fibrosis Background Cystic fibrosis degenerative disease characterize progressive<br>epithelial secretory gland dysfunction associate repeat respiratory infection<br>Bacterial infection frequent child cystic fibrosis rapid Methods<br>screen wide variety potentially involved virus unavailable recently<br>frequency viral presence unknown Multiplex PCR enable screen<br>many virus involve respiratory infection Objectives This study<br>aim evaluate frequency virus bacteria respiratory specimen child<br>cystic fibrosis clarify incidence characteristic seasonality age patient<br>different virus detect child cystic fibrosis Study design<br>year prospective study obtain paired nasopharyngeal swab sputum<br>specimens child cystic fibrosis clinical respiratory examination separate<br>least 14days analyze virus nasopharyngeal swab specimen multiplex<br>PCR bacteria sputum standard method Results analyze paired<br>specimen child detect virus bacteria Bacteria commoner spring<br>summer virus commoner autumn winter every season Staphylococcus<br>aureus common bacteria rhinovirus common virus Nearly infection<br>Haemophilus influenzae occur autumn winter Viruses prevalent child<br>year old bacteria prevalent child \\u226512 year old<br>Conclusions Multiplex PCR screen respiratory virus feasible child<br>cystic fibrosis clinical implication screen warrant study<br>\", \"High Rate Viral Identification Coinfections Infants Acute Bronchiolitis<br>OBJECTIVES determine virus risk factor associate hospital intensive<br>care unit ICU admission infant acute bronchiolitis INTRODUCTION<br>Bronchiolitis major cause morbidity infant Widespread use molecular<br>base method yield new insight etiology impact viral<br>etiology early outcome still unclear METHODS Seventy seven<br>infant bronchiolitis two year age visit emergency unit<br>include Using molecular base method sample test different<br>respiratory virus Logistic regression model use identify clinical<br>virological variable associate main endpoint hospital admission ICU<br>admission RESULTS identify least one virus patient coinfections<br>find nearly patient RSV common pathogen follow rhinovirus<br>Identification RSV associate increase risk hospital admission univariate<br>model Younger age enterovirus infection associate increase risk<br>hospital admission atopy first degree relative show protective<br>effect Prematurity associate increase risk admission ICU Coinfections<br>associate bad outcome CONCLUSIONS Molecular base method result<br>high rate viral identification change significant role RSV<br>acute bronchiolitis Younger age enterovirus infection risk factor<br>hospital admission prematurity appear significant risk factor admission<br>ICU acute viral bronchiolitis<br>\", \"Aspergillus meningitis rare clinical manifestation central nervous system<br>aspergillosis Case report review case Summary Objectives describe<br>pathogenesis clinical presentation cerebrospinal fluid finding outcome Aspergillus<br>meningitis meningoencephalitis arachnoiditis Methods case Aspergillus meningitis describe<br>comprehensive review English language literature conduct identify report<br>case Aspergillus meningitis describe January December Results Ninety<br>three case include one described herein Aspergillus meningitis<br>identify Fifty two individual without predispose factor know<br>cause immunosuppression Acute chronic meningitis diagnose patient meningoencephalitis<br>remain present spinal arachnoiditis ventriculitis Cerebrospinal fluid culture<br>Aspergillus spp positive case galactomannan antigen test Diagnosis<br>life achieve patient case fatality rate The overall<br>case fatality rate Conclusions Aspergillus meningitis may occur<br>immunocompetent immunocompromised patient run acute chronic course The<br>finding systematic review extend information life threaten infection<br>could assist physician achieve improved outcome<br>\", \"Co\\u2010infections Respiratory Viruses Dogs Bacterial Pneumonia BACKGROUND Bacterial<br>pneumonia inflammation lower airway lung parenchyma secondary bacterial<br>infection The pathogenesis dog complex role canine respiratory<br>virus fully evaluate OBJECTIVES The aim study investigate<br>occurrence viral co\\u2010infections dog ass demographic clinical variable<br>well disease severity associate viral co\\u2010infections ANIMALS Twenty<br>household dog cause opportunistic bacteria dog chronic day<br>tracheobronchitis cause Bordetella bronchiseptica BBTB METHODS Prospective cross\\u2010sectional<br>observational study Diagnosis confirm clinical laboratory finding diagnostic<br>image cytologic microbiologic analysis bronchoalveolar lavage transtracheal wash<br>fluid Canine parainfluenza virus CPIV canine adenovirus canine<br>herpes virus canine influenzavirus canine distemper virus canine<br>respiratory coronavirus CRCoV canine pneumovirus well bronchiseptica Mycoplasma<br>spp analyze respiratory sample use PCR assay RESULTS<br>CPIV detect CRCoV dog Respiratory virus detect dog<br>BBTB There significant difference clinical variable dog without<br>viral co\\u2010infection CONCLUSION AND CLINICAL IMPORTANCE Respiratory viruses<br>find frequently dog may therefore play important role<br>etiology pathogenesis Clinical variable disease severity differ dog<br>without viral co\\u2010infection<br>\", \"Changes pulmonary tuberculosis prevalence evidence population survey populous<br>province China BACKGROUND This paper report finding prevalence<br>survey conduct Shandong China province population million This<br>study aim estimate prevalence province comparison survey compare<br>yield case different case find approach METHODS population<br>base cross sectional survey conduct use multi stage<br>random cluster sample adult participate survey response rate<br>Doctors interview classified participant suspect case present persistent<br>cough abnormal chest ray CXRAY Three sputum specimen<br>suspect case collect sent smear microscopy culture RESULTS<br>Adjusted prevalence rate bacteriologically confirm case per adult<br>Shandong Compared survey prevalence decline bacteriologically confirm case<br>present persistent cough The yield bacteriologically confirm case<br>symptom screen CXRAY Over case among CONCLUSIONS The<br>prevalence rate bacteriologically confirm case significantly reduce compare<br>The survey raise challenge identify case without clear<br>symptom<br>\", \"Diagnostic performance multiplex PCR pulmonary sample versus nasopharyngeal<br>aspirate community acquire severe low respiratory tract infection<br>Background PCR base technique diagnosis community acquire severe<br>low respiratory tract infection become standard care However<br>relative ability identify either atypical bacteria virus cause<br>LRTI clinical sample various source yet determine Objectives<br>study design The aim study compare diagnostic yield<br>nasopharyngeal aspirate pulmonary sample etiological diagnosis severe acute<br>lower respiratory tract infection multiplex PCR Patients adult<br>community acquire pneumonia acute exacerbation chronic obstructive pulmonary<br>disease Results obtain concordant result pair sample discordant<br>result pathogen evidence low respiratory tract sample Conclusions<br>Pulmonary sample similar diagnostic sensitivity virus detection multiplex<br>PCR nasopharyngeal aspirate contrast study diagnostic efficacy pulmonary<br>sample Legionella pneumophila simple aspirate clearly superior<br>\", \"Safety Efficacy Profile Echinacea purpurea Prevent Common Cold<br>Episodes Randomized Double Blind Placebo Controlled Trial Objective<br>investigate safety risk efficacy benefit Echinacea purpurea extract<br>prevention common cold episode large population month period<br>Methods healthy subject allocate receive either alcohol extract<br>freshly harvest purpurea herba root placebo Participants require<br>record adverse event rate cold relate issue diary<br>throughout investigation period Nasal secretion sample acute cold<br>screen virus Results total adverse event occur Echinacea<br>placebo treatment Nine participant experienced adverse event least<br>possibly related study drug adverse drug reaction Thus<br>safety Echinacea noninferior placebo Echinacea reduce total number<br>cold episode cumulate episode day within group pain<br>killer medicate episode Echinacea inhibit virally confirm cold<br>especially prevent enveloped virus infection Echinacea show maximal<br>effect recurrent infection preventive effect increase therapy compliance<br>adherence protocol Conclusions Compliant prophylactic intake purpurea month<br>period appear provide positive risk benefit ratio<br>\", \"Pathological ultrastructural analysis surgical lung biopsy patient swine\\u2010origin<br>influenza type H1N1 acute respiratory failure BACKGROUND Cases<br>H1N1 pulmonary infection evolve acute respiratory failure death<br>co\\u2010infections lung injury predominate immune response thus require<br>early diagnosis improve treatment OBJECTIVE perform detailed histopathological<br>analysis open lung biopsy specimens five patient ARDS<br>confirm H1N1 METHODS Lung specimens underwent microbiologic analysis<br>examination optical electron microscopy Immunophenotyping use characterize macrophage<br>natural killer cell expression cytokine iNOS RESULTS The<br>pathological feature observe necrotizing bronchiolitis diffuse alveolar damage<br>alveolar hemorrhage abnormal immune response Ultrastructural analysis show<br>viral\\u2010like particle case CONCLUSIONS Viral\\u2010like particles successfully demonstrate<br>lung tissue ultrastructural examination without confirmation virus RT\\u2010PCR<br>nasopharyngeal aspirate Bronchioles epithelium rather endothelium probably primary<br>target infection diffuse alveolar damage consequence effect airways<br>obliteration dysfunction innate immunity suggest treatment focus epithelial<br>repair<br>\", \"Prevalence non influenza respiratory virus acute respiratory infection<br>case Mexico BACKGROUND Acute respiratory infection lead cause<br>morbidity mortality worldwide Although viral aetiological agent estimate<br>involved case majority agent never specifically identify Since<br>diagnostic surveillance effort influenza virus apply worldwide However<br>insufficient epidemiological information available many respiratory virus cause<br>Acute respiratory infection METHODS This study evaluate presence<br>non influenza respiratory virus pharyngeal exudate sample use<br>qPCR All sample meet operational definition probable case<br>influenza like illness severe acute respiratory infection previous<br>negative result influenza qPCR RESULTS The presence least<br>one non influenza virus observe sample The frequent<br>virus rhinovirus human respiratory syncytial virus HRSV human<br>metapneumovirus HMPV total case infection cause virus identify<br>Approximately positive case child year age CONCLUSION study<br>identify non influenza respiratory virus could occur season<br>year This study provide evidence prevalence seasonality wide<br>range respiratory virus circulate Mexico constitute risk population<br>Additionally data suggest include test widely diagnostic algorithm<br>influenza may reduce use unnecessary antibiotic reduce hospitalisation<br>time enrich national epidemiological data respect infection cause<br>virus<br>\", \"Respiratory Viruses Respiratory viruses frequent causative agent disease<br>human significant impact morbidity mortality worldwide Common respiratory<br>agent several virus family well adapt efficient person<br>person transmission circulate global scale community base study<br>conduct past five decade confirm virus predominant etiological<br>agent acute respiratory infection ARIs The respiratory viruses<br>commonly circulate continent endemic epidemic agent influenza virus<br>respiratory syncytial virus parainfluenza virus metapneumovirus rhinovirus coronaviruses<br>adenoviruses bocaviruses Although vaccine effective antiviral drug yet<br>available virus much progress make understanding biology fundamental<br>issue host\\u2013parasite relationship This article summary current knowledge<br>virus<br>\", \"Asymptomatic Middle East Respiratory Syndrome coronavirus infection use<br>serologic survey Korea OBJECTIVES The rate asymptomatic infection<br>Middle East Respiratory Syndrome MERS coronavirus vary serologic<br>study conduct determine asymptomatic MERS infection rate healthcare<br>worker non healthcare worker exposure status METHODS Study<br>participant select contact MERS patient base priority system<br>region strongly affect MERS outbreak sero epidemiological survey<br>perform contact average duration exposure test month collect<br>serum test use enzyme link immunespecific assay ELISA<br>immunofluorescence assay IFA plaque reduction neutralization antibody test<br>PRNT Among contact ELISA positive case exhibit positive<br>IFA PRNT result RESULTS The asymptomatic infection rate<br>confidence interval The asymptomatic MERS case patient hospitalize<br>patient zero floor hospital time The case quarantine<br>home week discharge underlying disease include hypertension angina<br>degenerative arthritis CONCLUSIONS The asymptomatic infection acquire via<br>healthcare associate transmission Thus necessary extend serologic study<br>include inpatient contact symptom<br>\", \"simple compact smartphone base device quantitative readout colloidal<br>gold lateral flow immunoassay strip Colloidal gold lateral<br>flow immunoassay strip AuNPs LFIS widely apply qualitative<br>diagnostic tool point care test POCT strip reader<br>incorporate use could extend quantitative analysis However cost<br>non portability render commercial strip reader unavailable use<br>either home testing community rural hospital diagnosis This<br>particularly true site test Here smartphone base reader<br>design print quantitatively assess AuNPs LFIS The basic<br>principle devise rely smartphone ambient light sensor SPALS<br>This sensor harness measure transmit light intensity originate<br>line strip transmit light intensity vary concentration AuNP<br>line validate approach newly develop smartphone ambient light<br>sensor base reader SPALS reader use readout AuNPs<br>LFIS three analytical target cadmium ion Cd2 limit<br>detection LOD clenbuterol LOD porcine epidemic diarrhea virus<br>PEDV LOD The result show good consistency result<br>conventional image analysis approach indicate smartphone base device<br>appropriate use AuNPs LFIS readout Compared traditional analysis<br>method develop AuNPs LFIS reader easier operate low<br>cost portable provide site quantitative analysis tool AuNPs<br>LFIS enhance apply range AuNPs LFIS<br>\", \"Detection feline infectious peritonitis virus infection cell culture<br>peripheral blood mononuclear leukocytes experimentally infected cat use<br>biotinylated cDNA probe dot blot hybridization assay use<br>biotinylated cDNA probe able detect feline infectious peritonitis<br>virus FIPV RNA Felis catus whole fetus fcwf<br>cell infect FIPV isolate DF2 UCDI UCD2 The<br>probe cross hybridize dot blot assay nucleic acid<br>closely related feline coronavirus feline enteric coronavirus FECV<br>construct probe kilobase cDNA prepare FIPV DF2 genomic<br>RNA molecularly clone The recombinant cDNA clone digest<br>restriction endonuclease Rsa basepair Rsa fragment isolated vector<br>DNA agarose electrophoresis glassmilk purification This fragment complementary<br>three fourth nucleocapsid gene The hybridization probe prepared<br>random prim label presence biotin dUTP Using avidin<br>alkaline phosphatase conjugate chemiluminescent substrate detection system virus<br>could detect infected cell vivo study probe use<br>detect FIPV RNA peripheral blood mononuclear leukocytes PBML<br>isolate various post infection day PID cat experimentally<br>infect FIP produce coronavirus isolate FIPV FIPV DF2<br>Viral RNA could detect PBML isolate cat PID<br>PBML PID There consistent pattern however hybridization result<br>prognosis severity disease time sample Despite cross hybridization<br>FECV RNA probe useful diagnosis FIP cat infect<br>FECV likely become viremic<br>\", \"Human adenovirus nasopharyngeal blood sample child without respiratory<br>tract infection Background Human adenovirus HAdV double strand<br>DNA virus associate respiratory tract infection RTI child<br>Using polymerase chain reaction PCR test HAdV often<br>detect together virus specie even healthy control Objectives<br>The aim study compare molecular detection HAdV culture<br>examine association various method RTI Study design Nasopharyngeal<br>aspirate NPA collect child admit RTI control The<br>NPAs examine viral bacterial pathogen use inhouse real<br>time PCR assay base TaqMan probes addition bacterial<br>viral culture HAdV concentration evaluate semi quantitatively value<br>quantitatively use ADENOVIRUS gene\\u00ae Results HAdV DNA detect<br>patient sample control Compared control patient 4\\u201310 mono<br>detection HAdV DNA 0\\u201313 HAdV positive sample grow<br>adenovirus culture HAdV DNA load child RTI consist<br>two cluster one cluster high viral load copy<br>one cluster low viral load whereas among control<br>nearly low viral load 2\\u201327 available plasma sample<br>positive HAdV DNA patient Conclusion The detection HAdV<br>DNA per qualitative PCR useful diagnostic test Detection<br>HAdV use viral culture high viral HAdV DNA<br>load two method strongly associate RTI child<br>\", \"Application Broad Spectrum Sequence Based Pathogen Identification Urban<br>Population broad spectrum detection platform provide sequence level<br>resolution target region would significant impact public health<br>case management mean expand understand etiology diseases previously<br>develop respiratory pathogen microarray RPM demonstrated capability platform<br>purpose This newly develop RPM use analyze well<br>characterize nasal wash specimen patient present febrile respiratory<br>illness Washington metropolitan region For specimen RPM result<br>compare composite reference assay viral bacterial culture appropriate<br>PCR PCR result Across panel RPM assay show<br>\\u226598 overall agreement organism detect compare reference method<br>Additionally RPM result provide sequence information allow phylogenetic<br>classification circulate influenza virus \\u223c250 clinical specimen allow<br>monitoring genetic variation well antigenic variability prediction Multiple<br>pathogens 2\\u20134 detect specimen notably increase abundance respiratory<br>colonizer esp pneumoniae viral infection This first ever<br>comparison broad spectrum viral bacterial identification technology type<br>large battery conventional gold standard assay confirms utility<br>approach medical surveillance investigation complex etiology illness cause<br>respiratory infection<br>\", \"Experimental Design Analysis Microarray Data The advent microarray<br>technology significantly change way quantitatively measure observe gene<br>expression mRNA level within give biological sample interest<br>allow monitor ten hundred thousand gene within single<br>experiment The two main array platform spot two<br>colour array one colour situ synthesize array Microarrays<br>use wide range application include gene annotation investigation<br>gene gene interaction elucidation gene regulatory network gene<br>expression profile Saccharomyces cerevisiae fungal organism Academic researcher<br>pharmaceutical agricultural industry enormous interest develop microarrays diagnostic<br>tool use basic research pathogens fungi interact host<br>Microarray experiment generate vast quantity raw gene expression<br>data therefore good experimental design statistical analysis require<br>extraction accurate useful information regard expression gene review<br>firstly provide overview arrival development microarray technology focus<br>issue surround experimental design processing microarray image follow<br>discussion method clean normalize raw gene expression data<br>final discussion importance statistical analysis play identify differentially<br>express gene<br>\", \"Development recombinase polymerase amplification assay lateral flow dipstick<br>rapid detection feline parvovirus Feline panleukopenia cause feline<br>parvovirus FPV single strand DNA virus typically highly<br>contagious often present lethal syndrome The broad spectrum<br>possible host suggest potential transmission animal person close<br>contact pet FPV thus serve example importance new<br>rapid point care field diagnostic tool control prevention<br>transmission especially among rare wild animal pet cat<br>Recombinase polymerase amplification RPA real time isothermal method<br>could affordable alternative PCR combine lateral flow dipstick<br>LFD indicator study report novel FPV lateral flow<br>dipstick RPA LFD RPA instant detection method capable<br>detect range different FPV strain The LFD RPA<br>assay consist specific primer probe nucleic acid strip<br>capable detect copy target nucleic acid per reaction<br>one order magnitude high sensitivity traditional PCR The<br>suitable reaction condition assay min This paper develop<br>efficient visual detection system eliminate need professional staff<br>expensive sophisticated equipment field detection<br>\", \"Visualizing Viral Infection Vivo Multi Photon Intravital Microscopy<br>Viral pathogen adapt host organism exploit cellular machinery<br>virus replication modulate host cell efficient systemic dissemination<br>immune evasion Much knowledge effect virus infection cell<br>originate vitro image study use experimental culture system<br>consist cell line primary cell Recently intravital microscopy<br>use multi photon excitation fluorophores apply observe virus<br>dissemination pathogenesis real time physiological condition live organisms<br>Critical step viral infection pathogenesis could study direct<br>visualization fluorescent virus particle virus infect cell immune<br>response viral infection review summarize late research vivo<br>study viral infection use multi photon intravital microscopy<br>IVM Initially underlie principle multi photon microscopy introduce<br>experimental challenge microsurgical animal preparation fluorescent label strategy<br>intravital image discuss highlight recent study combine IVM<br>optogenetic tool transcriptional analysis powerful approach extend significance<br>vivo image study viral pathogen<br>\", \"Deep Sequencing Identify Causes Viral Encephalitis Deep sequencing<br>allow rapid accurate characterization microbial DNA RNA sequence<br>many type sample Deep sequencing also call next<br>generation sequence NGS develop assist diagnosis wide variety<br>infectious disease study seven frozen brain sample decease<br>subject recent encephalitis investigate RNA sample extract randomly<br>reverse transcribed sequence The sequence analysis perform blinded<br>fashion confirm pathogen specific PCR This analysis successfully<br>identify measles virus sequence two brain sample herpes<br>simplex virus type sequence three brain sample pathogen<br>identify two brain specimen These result concordant pathogen<br>specific PCR partially concordant prior neuropathological examination demonstrate<br>deep sequencing accurately identify viral infection freeze brain<br>tissue<br>\", \"FilmArray\\u00ae respiratory panel performance respiratory sample neonatal care<br>unit FilmArray Respiratory Panel Idaho Technology Inc Salt<br>Lake City USA performance retrospectively evaluate respiratory sample<br>collect neonate reference neonatology unit Using FilmArray assay<br>sample positive least respiratory virus negative FilmArray result<br>concordant respiratory sample test laboratory develop real time<br>PCR assay sample FilmArray assay detect additional virus<br>human rhinovirus enterovirus bocavirus addition FilmArray result respiratory<br>sample test concordant Seegene Seeplex RV15\\u00ae detection assay<br>Seegene Inc Seoul South Korea discordant The advantage<br>FilmArray rapid detection respiratory viruses hour wide number<br>pathogen detectable single assay reduce hand time<br>\", \"Comparison two commercial molecular assay simultaneous detection respiratory<br>viruses clinical sample use two automatic electrophoresis detection<br>system Two molecular assay compare real time PCR<br>viral culture simultaneous detection common viruses respiratory sample<br>multiplex ligation dependant probe amplification MLPA dual priming<br>oligonucleotide system DPO addition positive detection MLPA DPO<br>identify use two different automatic electrophoresis system panel<br>culture positive negative sample test molecular assay presence<br>influenza virus respiratory syncytial virus human metapneumovirus rhinovirus<br>coronaviruses parainfluenza virus adenovirus One hundred twenty nine<br>sample positive detect least one method Sixty nine<br>sample positive cell culture exclude human metapneumovirus coronaviruses<br>PCR MLPA DPO The MLPA yield result one<br>attempt sample include sample repeat DPO assay due<br>inconclusive result The MLPA assay perform well combination<br>either electrophoresis system performance DPO assay influence electrophoresis<br>system Both molecular assay comparable real time PCR<br>sensitive viral culture detect dual infection easily Results<br>obtain within day<br>\", \"Comparisons substitution insertion deletion probe resequencing mutational analysis<br>use oligonucleotide microarrays Although oligonucleotide probe complementary single<br>nucleotide substitution commonly use microarray base screen genetic<br>variation little known hybridization property probe complementary small<br>insertion deletion necessary define hybridization property latter probe<br>order improve specificity sensitivity oligonucleotide microarray base mutational<br>analysis disease relate gene Here compare contrast hybridization<br>property oligonucleotide microarrays consist 25mer probe complementary possible<br>single nucleotide substitution insertion one two base deletion<br>cod region ATM ataxia telangiectasia mutate gene Over<br>different dye label single strand nucleic acid target<br>represent ATM cod exon apply microarrays assess hybridization<br>specificity compare relative hybridization signal probe perfectly match<br>ATM sequence contain mismatch Probes complementary two base<br>substitution display high average specificity follow complementary single<br>base substitution single base deletion single base insertion<br>case hybridization specificity strongly influence sequence context possible<br>intra intermolecular probe target structure Furthermore single nucleotide<br>substitution probe display consistent hybridization specificity data follow<br>single base deletion two base deletion single nucleotide<br>insertion Overall study provide valuable empirical data use<br>accurately model hybridization property insertion deletion probe improve<br>design interpretation oligonucleotide microarray base resequencing mutational analysis<br>\", \"Specific detection H5N1 avian influenza virus field specimens<br>one step PCR assay BACKGROUND Continuous outbreaks highly<br>pathogenic H5N1 avian influenza Asia result urgent effort<br>improve current diagnostics aid containment virus lower threat<br>influenza pandemic report development PCR base assay highly<br>specific H5N1 avian influenza virus METHODS one step<br>reverse transcription PCR assay develop detect H5N1 avian<br>influenza virus The specificity assay show test sub<br>type influenza virus viral bacterial pathogen field sample<br>RESULTS Validation field specimen Vietnam Malaysia show assay<br>specific without cross reactivity number infuenza strain well<br>human respiratory relate pathogen Detection allantoic fluid H5N1<br>positive sample suggest reliable sample source accurate detection<br>CONCLUSION The assay develop study indicate primer specific<br>H5N1 influenza virus show field test result assay<br>would highly useful diagnostic tool help identify control<br>influenza epidemic<br>\", \"Development nest PCR assay detection canine coronavirus diagnostic<br>test canine coronavirus CCV infection base nested polymerase<br>chain reaction PCR assay develop test use follow<br>coronavirus strain CCV USDA strain CCV field strain<br>feline infectious peritonitis virus FIPV field strain trasmissible<br>gastroenteritis virus TGEV Purdue strain bovine coronavirus BCV<br>9WBL strain infectious bronchitis virus IBV strain fecal<br>sample dog CCV enteritis segment gene encode transmembrane<br>protein CCV target sequence primer The test describe<br>present study able amplify CCV TGEV strain also<br>give positive result fecal sample CCV infect dog<br>PCR sensitivity high isolation cell culture therefore use<br>diagnosis CCV infection dog<br>\", \"Controlled Microwave Heating Accelerates Rolling Circle Amplification Rolling<br>circle amplification RCA generate single strand DNAs RNA<br>diverse application isothermal technique range sensitive detection nucleic<br>acid analysis single nucleotide polymorphism Microwave chemistry widely<br>apply increase reaction rate well product yield purity<br>The objective present research apply microwave heat RCA<br>indicate factor contribute microwave selective heating effect The<br>microwave reaction temperature strictly control use microwave applicator<br>optimize enzymatic scale reaction Here show microwave assist<br>RCA reaction catalyze either four thermostable DNA polymerase<br>accelerate fold compare conventional RCA Furthermore temperatures individual<br>buffer component specifically influence microwave heating conclude microwave<br>heat accelerate isothermal RCA DNA differential heating mechanism<br>microwave temperature reaction component although overall reaction temperature<br>\", \"Simple rapid detection human enterovirus reverse transcription loop<br>mediate isothermal amplification cryopreservation affect detection ability Human<br>enterovirus EV71 primary pathogen hand foot mouth disease<br>HFMD EV71 infection may lead neurologic damage high<br>incidence fatality compare HFMD pathogens effective drug vaccine<br>EV71 infection currently unavailable desirable determine pathogen HFMD<br>accurately quickly early treatment current study reverse transcription<br>loop mediate isothermal amplification LAMP technology develop detect<br>EV71 The efficacy detect EV71 compare regular nested<br>reverse transcription polymerase chain reaction PCR After detect<br>clinical specimen result show LAMP specifically detect EV71<br>Coxsackie virus A16 exhibit specificity sensitivity high regular<br>PCR The finding indicate LAMP practical method EV71<br>diagnostic application particularly small county institute medical service<br>The detection ability LAMP significantly affect cryopreservation clinical<br>specimen repeatedly subject freeze thawing treatment<br>\", \"multiplex PCR assay detection five influenza virus use<br>dual priming oligonucleotide system BACKGROUND cost effective accurate<br>rapid simultaneous multiplex assay require testing diagnose conventional<br>emerge virus clinical virology laboratories develop optimize dual<br>priming oligonucleotide DPO multiplex PCR assay detect influenza<br>virus include seasonal H1N1 pandemic H1N1 H3N2 influenza<br>H5N1 METHODS The optimized multiplex DPO PCR use<br>detect clinical human sample The result compare obtained<br>qPCR conventional PCR immunochromatographic assay RESULTS Specificity analysis<br>reveal DPO PCR assay amplify target virus without<br>cross amplification Statistical analysis demonstrate multiplex DPO PCR<br>sensitivity high immunochromatographic assay low qPCR significant difference<br>observe compare conventional PCR detect influenza Additional experiment<br>use sample panel indicate significant difference number positive<br>sample detect multiplex DPO PCR qPCR apply value<br>low CONCLUSIONS The five target simultaneous multiplex DPO<br>PCR assay could easily adopt routine practice This<br>approach cost effective short run time low technical<br>requirement detection influenza virus early diagnosis clinical laboratory<br>\", \"Microbial molecular marker epidemiological surveillance era high throughput<br>sequence update IMMEM conference<br>\", \"Comparison performance ITS1 ITS2 barcodes amplicon base sequencing<br>bioaerosols This paper present performance two eukaryotic genomic<br>ribosomal region ITS1 ITS2 describe fungal diversity aerosol<br>sample use amplicon base High Throughput Sequencing HTS<br>Composting sit biomethanization facility dairy farm affect presence<br>fungi visit collect air sample The amplicon base<br>HTS approach target enrichment method relies amplification specific<br>target use particular primer sequence Thus result highly<br>dependent quality amplification For reason author paper use<br>shotgun metagenomic approach compare outcome amplicon base method<br>Indeed shotgun metagenomic rely amplification prior sequence gene<br>sequence without specific target addition culture method add<br>analysis biomethanization dairy farm sample validate contribution fungal<br>diversity aerosols The result obtain unequivocal towards ITS1<br>outperformance ITS2 term richness taxonomic coverage The differential<br>abundance analysis demonstrate taxon exclusively detect ITS2 vice<br>versa ITS1 However shotgun metagenomic approach show taxonomic<br>profile resemble ITS1 ITS2 Based result neither barcodes<br>evaluate perfect term distinguish specie Using barcodes offer<br>broad view fungal aerosol population However actual knowledge<br>author strongly recommend use ITS1 universal fungal barcode<br>quick general analysis diversity limit financial resource available<br>primarily due ability capture taxonomic profile similar obtain<br>use shotgun metagenomic The culture comparison amplicon base<br>sequencing show complementarity approach describe abundant taxon<br>\", \"Nasopharyngeal Protein Biomarkers Acute Respiratory Virus Infection Infection<br>respiratory mucosa viral pathogens trigger complex immunologic event<br>affect host seek characterize response proteomic analysis nasopharyngeal<br>lavage human subject experimentally challenge influenza H3N2 human<br>rhinovirus develop target assay measure peptide involve host<br>response allow classification acute respiratory virus infection Unbiased<br>proteomic discovery analysis identify peptide correspond unique protein<br>reveal infection H3N2 induce significant alteration protein expression<br>These include protein involve acute inflammatory response innate<br>immune response complement cascade These data provide insight<br>nature biological response viral infection upper respiratory tract<br>protein dysregulated viral infection form basis signature accurately<br>classifies infect state Verification signature use targeted mass<br>spectrometry independent cohort subject challenge influenza rhinovirus demonstrate<br>performs high accuracy AUROC TPR TNR With development<br>clinical diagnostic signature may utility rapid screen emerge<br>infection avoidance inappropriate antibacterial therapy rapid implementation appropriate<br>therapeutic public health strategy<br>\", \"Identification characterization unrecognized virus stool sample non polio<br>acute flaccid paralysis child simplify VIDISCA BACKGROUND The<br>use sequence independent method combine next generation sequence<br>identification purpose clinical sample appear promise excite result<br>achieve understand unexplained infection One sequence independent method<br>Virus Discovery base cDNA Amplified Fragment Length Polymorphism<br>VIDISCA capable identifying virus would remain unidentified standard<br>diagnostics cell culture METHODS VIDISCA normally combine next<br>generation sequence however set simplify VIDISCA use case<br>next generation sequence possible Stool sample patient unexplained<br>acute flaccid paralysis show cytopathic effect rhabdomyosarcoma cell<br>mouse cell use test efficiency method characterize virus<br>VIDISCA positive sample amplify sequenced gene specific primer<br>RESULTS Simplified VIDISCA detect seven virus proportion eukaryotic<br>viral sequence sample range Human enterovirus B97 B100<br>echovirus echovirus human parechovirus type human astrovirus probably<br>type recombinant tetnovirus identify Phylogenetic analysis base VP1<br>region demonstrate human enterovirus divergent isolate circulate community<br>CONCLUSION Our data support simplify VIDISCA protocol efficiently<br>identify unrecognized virus grow cell culture low cost<br>limited time without need advance technical expertise Also<br>complex data interpretation avoid thus method use powerful<br>diagnostic tool limit resource Redesigning routine diagnostics might<br>lead additional detection previously undiagnosed virus clinical sample<br>patient ELECTRONIC SUPPLEMENTARY MATERIAL The online version article<br>doi 422X contains supplementary material available authorized user<br>\", \"Space Time Clustering Characteristics Tuberculosis China OBJECTIVES China<br>one tuberculosis high burden country world major public<br>health problem China spatial analysis could apply detect<br>geographic distribution cluster target intervention epidemic METHODS Spatial<br>analysis apply detect cluster county base notification data<br>national notifiable infectious disease case report surveillance system<br>Two indicator epidemic use include new sputum smear<br>positive notification rate total notification rate Global Moran<br>ArcGIS used ass whether cluster trend significant SaTScan<br>software use retrospective space time analysis Possion probability<br>model utilized identify geographic area time period potential<br>cluster notification rate county level RESULTS Two indicator<br>notification present significant spatial autocorrelation globally year Global<br>Moran total notification rate positive trend time The<br>likely cluster two indicator similar spatial distribution size<br>south central region China secondary cluster two region<br>northeastern China western China Besides secondary cluster total<br>notification rate two large cluster center Inner Mongolia<br>Gansu Qinghai provinces several small cluster Shanxi Henan<br>Hebei Jiangsu provinces CONCLUSION The total notification case<br>cluster significantly special area year cluster trend aggregate<br>time The likely secondary cluster overlap among two<br>indicator higher burden risk transmission These focus geographic<br>area control effort prioritize<br>\", \"iGPCR Drug Web Server Predicting Interaction GPCRs Drugs<br>Cellular Networking Involved many disease cancer diabetes neurodegenerative<br>inflammatory respiratory disorder protein couple receptor GPCRs among<br>frequent target therapeutic drug time consume expensive determine<br>whether drug GPCR interact cellular network purely mean<br>experimental technique Although computational method develop regard base<br>knowledge dimensional structure protein unfortunately usage quite limited<br>structure GPCRs still unknown overcome situation sequence base<br>classifier call iGPCR drug develop predict interaction GPCRs<br>drug cellular networking predictor drug compound formulate dimensional<br>fingerprint via 256D vector GPCR PseAAC pseudo amino<br>acid composition generate grey model theory prediction engine<br>operate fuzzy near neighbour algorithm Moreover user friendly<br>web server iGPCR drug establish http www jci<br>bioinfo iGPCR Drug For convenience experimental scientist step<br>step guide provide use web server get desire<br>result without need follow complicate math equation present<br>paper integrity The overall success rate achieve iGPCR<br>drug via jackknife test remarkably high rate exist<br>peer method develop although web server ever establish<br>anticipate iGPCR Drug may become useful high throughput<br>tool basic research drug development approach present also<br>extend study drug target interaction network<br>\", \"Yang Zhong explorer road forever<br>\", \"Cybernetics base Dynamic Infection Model Analyzing SARS COV<br>Infection Stability Predicting Uncontrollable Risks Since December COVID<br>rag Wuhan subsequently China world propose Cybernetics base<br>Dynamic Infection Model CDIM dynamic infection process probability<br>distribute incubation delay feedback principle Reproductive trend stability<br>SARS COV infection city analyze uncontrollable risk forecast<br>really happen The infection mechanism city depict use<br>philosophy cybernetics approach control engineering Distinguished epidemiological model<br>SIR SEIR etc compute theoretical number infect people<br>close population CDIM consider immigration emigration population system<br>input administrative medical resource dynamic control variables The<br>epidemic regulation simulate model support decision make contain<br>outbreak City case study demonstrate verification validation<br>\", \"Determinants Taiwanese tourist hotel industry cycle This paper<br>contribute tourism literature examine determinant Taiwanese tourist hotel<br>industry THI cycle This study use Markov switch<br>model MSM propose Hamilton analyze Taiwanese tourist hotel<br>industry cycle The MSM decomposes tourist hotel industry<br>cycle two distinct state high growth low growth<br>HGS LGS The mean growth rate HGS average<br>growth rate LGS period December February The corresponding<br>standard deviation two regime imply HGS stable LGS<br>Moreover probability stay HGS probability remain LGS The<br>expected duration HGS LGS month respectively Further paper<br>investigate factor keep THI HGS Empirical test result<br>show growth international tourism market industrial production growth<br>rate two key factor keep THI HGS SARS<br>outbreak adverse effect<br>\", \"Pathway Network Approaches Identification Cancer Signature Markers Omics<br>Data The advancement high throughput omic technology past<br>year make possible perform many complex assay much<br>short time traditional approach The rapid accumulation wide<br>availability omic data generate technology offer great opportunity<br>unravel disease mechanism also present significant challenge extract<br>knowledge massive data evaluate finding address challenge number<br>pathway network base approaches introduce This review article<br>evaluate method discuss application cancer biomarker discovery use<br>hepatocellular carcinoma HCC example<br>\", \"Dynamics seasonal airborne propagation Staphylococcus aureus academic dental<br>clinic OBJECTIVE Staphylococcus aureus strain disseminate dental treatment<br>occasionally lead contamination infection patient dentists important public<br>health problem The dynamic airborne propagation genetic diversity<br>aureus isolate academic dental clinic environment investigate use<br>isoenzyme type MATERIAL AND METHODS The isoenzymes previously<br>report isolates obtain fresh culture extract use glass<br>bead Nine isoenzymes investigate use multilocus enzyme electrophoresis<br>MLEE The genetic diversity relationship among strain electrophoretic<br>type determine use statistic previously describe Nei SAHN<br>group method UPGMA algorithm RESULTS Clonal pattern analysis<br>indicate high level genetic polymorphism occur among ETs<br>group five taxon Each taxon present one cluster<br>moderately relate contain two identical highly related isolates<br>reveal seasonal airborne propagation dental clinic environment CONCLUSIONS<br>These data suggest occurrence active microevolutionary process aureus<br>well possibility environmental propagation month time span Such<br>finding important show multiuser academic dental clinic retain<br>certain strain spreadable different niche<br>\", \"The Transmissibility Highly Pathogenic Avian Influenza Commercial Poultry<br>Industrialised Countries BACKGROUND With increased occurrence outbreak H5N1<br>worldwide concern virus could enter commercial poultry farm<br>severe economic consequence METHODOLOGY PRINCIPAL FINDINGS analyse data<br>four recent outbreak highly pathogenic avian influenza HPAI<br>commercial poultry estimate farm farm reproductive number HPAI<br>The reproductive number key measure transmissibility HPAI farm<br>level use evaluate effectiveness control measure outbreak mean<br>farm farm reproductive number prior control range maximum<br>farm base reproductive number range Enhanced bio security<br>movement restriction prompt isolation infect farm four outbreak<br>substantially reduce reproductive number remain close threshold value<br>necessary ensure disease eradicated CONCLUSIONS SIGNIFICANCE Our result<br>show depend particular situation outbreak avian influenza occur<br>current control might enough eradicate disease therefore close<br>monitoring outbreak require The method use estimate reproductive<br>number straightforward implement use real time therefore useful<br>tool inform policy decision<br>\", \"Topology evolve mutagenized viral population quasispecies expansion compression<br>operation negative selection BACKGROUND The molecular event evolutionary<br>force underlie lethal mutagenesis virus virus extinction excess<br>mutation well understood Here apply first time phylogenetic<br>method Partition Analysis Quasispecies PAQ monitor genetic distance<br>intra population structure mutant spectra foot mouth disease<br>virus FMDV quasispecies subject mutagenesis base nucleoside analogues<br>RESULTS Phylogenetic PAQ analyse reveal highly dynamic variation<br>intrapopulation diversity FMDV quasispecies The population diversity first<br>suffers strike expansion presence mutagens compression either presence<br>mutagenic analogue discontinue mutation decrease sensitivity mutagen select<br>The pattern mutation find population agreement behavior correspond<br>nucleotide analogue FMDV vitro Mutations accumulate preferred genomic<br>site ratio indicate operation negative purifying selection population<br>subject mutagenesis evidence unusually elevate genetic distance obtain<br>FMDV population approach extinction CONCLUSION Phylogenetic PAQ analysis<br>provide adequate procedure describe evolution viral sequence subject<br>lethal mutagenesis These method define change intra population<br>structure precisely mutation frequency Shannon entropy PAQ sensitive<br>variation intrapopulation genetic distance Strong negative purifying selection<br>operate FMDV population subject enhanced mutagenesis The quantification<br>provide evidence extinction imply unusual increase intrapopulation complexity<br>support lethal defection model virus extinction<br>\", \"Epidemics panic bioterrorist attack mathematical model Summary bioterrorist<br>attack usually cause epidemic panic targeted population present<br>epidemiologic aspect phenomenon three component model host information<br>attack social network propose mathematical model panic counter<br>measure function time population expose bioterrorist attack The<br>model comprise ordinary differential equation graphically present combination<br>equation parameter Clinically present model sequence psychic condition<br>disorder initiate act bioterrorism This model might helpful<br>attacked community timely properly apply counter measure minimize<br>human mental suffer bioterrorist attack<br>\", \"Distribution Risk Factors Pandemic Influenza H1N1 Mainland China<br>Data report case pandemic influenza H1N1 obtain China<br>Information System Disease Control Prevention The spatiotemporal distribution<br>pattern case characterize spatial analysis The impact travel<br>relate risk factor invasion disease analyze use survival<br>analysis climatic factor related local transmission identify use<br>multilevel Poisson regression county level The result show<br>epidemic span large geographic area affect area western<br>China Significant difference incidence find among age group<br>incidence peak school age child Overall epidemic spread<br>southeast northwest Proximity airport intersect national highway freeways<br>railway variable associate presence disease county Lower temperature<br>lower relative humidity climatic factor facilitate local transmission<br>correction effect school summer vacation public holiday well<br>population density density medical facility These finding indicate<br>intervention focus domestic travel population density climatic factor<br>could play role mitigate public health impact future<br>influenza pandemic<br>\", \"Preliminary estimation basic reproduction number novel coronavirus nCoV<br>China data driven analysis early phase outbreak BackgroundsAn<br>ongoing outbreak novel coronavirus nCoV pneumonia hit major<br>city China Wuhan December subsequently reach province region<br>China country present estimate basic reproduction number nCoV<br>early phase outbreak MethodsAccounting impact variation disease report<br>rate model epidemic curve nCoV case time series<br>mainland China January January exponential growth With estimate<br>intrinsic growth rate estimate use serial interval two<br>well know coronavirus diseases MERS SARS approximation true<br>unknown FindingsThe early outbreak data largely follow exponential<br>growth estimate mean range associate fold fold increase<br>report rate demonstrate change report rate substantially affect<br>estimate ConclusionThe mean estimate nCoV range significantly large<br>Our finding indicate potential nCoV cause outbreak<br>\", \"Documentary Research Human Respiratory Droplet Characteristics Respiratory droplet<br>characteristic key determine droplet borne pathogen transmission provide<br>scientific basis formulate disease prevention droplet transmission control<br>measure Through study data information exist document paper<br>give respiratory droplet characteristic like size concentration velocity<br>etc Meanwhile droplet evaporation droplet borne pathogen activity<br>transmission discuss The droplet size significant difference human<br>health level gender age The size droplet produce<br>health people 10\\u03bcm produce patient 10\\u03bcm patient droplet<br>concentration high The coughed droplet concentration change size<br>peak rule The velocity cough droplet big range<br>25m transmission distance<br>\", \"Application corona discharge generate air ion filtration aerosolize<br>virus inactivation filter virus The effect corona discharge<br>generate air ion filtration aerosolize bacteriophage MS2 study<br>carbon fiber ionizer instal upstream medium efficiency air<br>filter generate air ion use charge virus aerosols<br>increase filtration efficiency After virus aerosol capture filter<br>certain time interval expose newly incoming air ion<br>flow Captured virus particle detach filter sonication antiviral<br>efficiency due air ion calculate count plaque form<br>unit The antiviral efficiency increase ion exposure time<br>ion concentration When concentration positive air ion ion<br>cm3 antiviral efficiency exposure time 45min respectively When<br>ionizer operate bipolar mode number concentration positive negative<br>ion 6\\u00d7106 4\\u00d7106 ion cm3 respectively antiviral efficiency<br>exposure time 45min respectively quantitative parameter performance evaluation<br>air ion susceptibility constant bacteriophage MS2 positive negative<br>bipolar air ion calculate 5\\u00d710\\u22123 4\\u00d710\\u22123 5\\u00d710\\u22123 respectively<br>These susceptibility constant show bipolar ion treatment effective<br>time unipolar ion treatment<br>\", \"Theoretical condition coexistence viral strain difference phenotypic trait<br>bifurcation analysis investigate dynamic wild type viral strain<br>generates mutant strain differ phenotypic property infectivity virulence<br>mutation rate study mean mathematical model bifurcation analysis<br>condition wild type mutant virus compete host cell<br>coexist The coexistence condition formulate term basic reproductive<br>number strain maximum value mutation rate virulence pathogens<br>The analysis reveals parameter space divide five region<br>distinct dynamic organize around degenerate Bogdanov\\u2013Takens zero Hopf<br>bifurcation latter give rise curve transcritical bifurcation periodic<br>orbit These result provide new insight condition viral<br>population may contain multiple coexist strain stable manner<br>\", \"Association energy use poor visibility Hong Kong SAR<br>China city reliance energy increase develop Moreover combustion<br>fossil fuel inevitably generate air pollutant include carbon<br>dioxide nitrogen oxide sulfur dioxide particulate matter others<br>Combining anthropogenic air pollutant visibility many Asian city<br>include Hong Kong deteriorate rapidly past decade This<br>paper explore relationship energy use meteorological factor change<br>visibility Hong Kong use long term time series<br>data The total use primary energy increase number<br>hour reduced visibility increase period time Bivariate correlation<br>show poor visibility significantly associate energy use annual<br>mean air temperature Multiple regression analysis indicate burn<br>aviation gasoline significantly adversely affect visibility Results illustrate<br>number clear day Hong Kong decrease particular due<br>increase air traffic<br>\", \"Pandemic H1N1 Influenza Community Transmission Was Established One<br>Australian State When Virus Was First Identified North<br>America BACKGROUND mid June State Victoria Australia appear<br>high notification rate pandemic H1N1 influenza world hypothesise<br>community transmission pandemic influenza already well establish Victoria<br>time test novel virus commence contrast true pandemic<br>part Australia include Western Australia METHODS use data<br>detailed case follow patient confirm infection Victoria demonstrate<br>difference pandemic curve two Australian state opposite side<br>continent model pandemic state use susceptible infect remove<br>model Bayesian inference accounting import case RESULTS Epidemic<br>transmission occur earlier Victoria later Only first Victorian<br>case locally acquire three brother one family contrast<br>first case associate importation Victoria Using plausible model<br>input data estimation effective reproductive number Victorian epidemic<br>require invoke earlier date commencement transmission explain observe<br>data This required modelling epidemic CONCLUSION Strong circumstantial<br>evidence support modelling suggests community transmission pandemic influenza<br>well establish Victoria time test novel virus commence<br>Australia The virus likely enter Victoria already become<br>establish around time first identify Mexico<br>\", \"Strategies enhance analytical performance nanomaterial base sensor provide<br>state art review main strategy enhancement analytical performance<br>sensor use nanomaterials particularly nanowires carbon base material<br>emphasize way overcome problem device device variation discuss<br>study influence nanomaterial characteristic sensor dimension operational condition<br>sense performance application appropriate calibration model<br>\", \"Realized generation time contraction impact infectious period reproduction<br>number population size One key characteristic transmission dynamic<br>infectious disease generation time refers time interval infection<br>secondary case infection infector The generation time distribution<br>together reproduction number determines rate infection spread population<br>When define generation time distribution calendar time two<br>definition plausible accord whether regard infection time infector<br>infection time infectee The result measurement respectively call<br>forward generation time backward generation time observe mean<br>forward generation time contract around peak epidemic This<br>contraction effect previously attribute either competition among potential<br>infectors depletion susceptibles population The first explanation require<br>many infectives contraction occur whereas latter explanation suggest<br>contraction occur even infectives With simulation study show<br>competition depletion cause mean forward generation time contract<br>Our result also reveal distribution infectious period reproduction<br>number strong effect size timing contraction well mean<br>value generation time forward backward scheme Author summaryInfectious<br>disease remain one great threat human health commerce<br>analysis epidemic data one important application statistic public<br>health Thus reliable estimate fundamental infectious disease parameter<br>critical public health decision maker order take appropriate<br>action global prevention management outbreaks health emergency key<br>example give prediction model reproduction number rely generation<br>time distribution usually estimate contact trace data collect<br>precise calendar time The forward scheme use prediction<br>model knowledge evolution time crucial correctly estimate parameter<br>interest therefore important characterize cause lead contraction mean<br>forward generation time course outbreak paper firstly identify<br>impact epidemiological quantity reproduction number infectious period population<br>size mean forward backward generation time Moreover analyze<br>phenomena competition among infectives depletion susceptible individual highlight<br>effect contraction mean forward generation time The upshot<br>investigation variance infectious period distribution reproduction number strong<br>impact generation time affect mean value evolution time<br>Furthermore competition depletion cause contraction even small value<br>reproduction number suggest epidemic model generation time consider<br>time inhomogeneous estimator account depletion compete risk prefer<br>inference generation interval distribution<br>\", \"Report Federal Panel Formaldehyde The Federal Panel Formaldehyde<br>conclude definitive experiment exist demonstrate mutagenicity carcinogenicity formaldehyde<br>laboratory condition Formaldehyde induces gene mutation chromosomal aberration<br>variety test system Inhalation formaldehyde cause cancer nose<br>rat The concentration formaldehyde inhaled air cause nasal<br>cancer Fisher rat within order magnitude human may<br>expose The data presently available permit direct assessment<br>carcinogenicity formaldehyde man Epidemiologic study expose human population<br>progress may clarify situation Other experimental human study<br>toxic effect teratogenicity reproductive disorder yet inadequate health<br>risk assessment The CIIT month study animal carcinogenicity<br>yet completely evaluated Additional data expect effect prolonged<br>exposure lower dos formaldehyde possible carcinogenicity formaldehyde mouse<br>The panel recommend comprehensive health risk assessment experiment<br>conduct effect mode exposure ingestion skin penetration effect<br>human pharmacokinetics formaldehyde man animal possible role formaldehyde<br>reproductive chronic respiratory disorder conclusion panel formaldehyde presume<br>pose carcinogenic risk human<br>\", \"Identification characterization novel type lectin MjLTL2 kuruma shrimp<br>Marsupenaeus japonicus type lectins LTLs belong lectin family<br>characterize conserved structural motif carbohydrate recognition domain LTLs<br>homologous leguminous lectin study identify functionally characterize LTL<br>kuruma shrimp Marsupenaeus japonicus designate LTL MjLTL2 MjLTL2<br>contain signal peptide Lectin leg domain coil coil<br>transmembrane domain MjLTL2 distribute hemocytes heart hepatopancreas gill<br>stomach intestine high expression level see hemocytes hepatopancreas<br>tissue MjLTL2 upregulated follow challenge shrimp Vibrio anguillarum<br>white spot syndrome virus WSSV MjLTL2 agglutinate several<br>bacteria without Ca2 addition MjLTL2 could bind several<br>Gram positive negative bacteria bind lipopolysaccharide peptidoglycan However<br>MjLTL2 could enhance clearance anguillarum vivo presence WSSV<br>infection MjLTL2 knockdown RNA interference result day lower<br>cumulative mortality japonicus Moreover less VP19 VP24 VP26<br>VP28 mRNAs extract hemocytes MjLTL2 knockdown shrimp control<br>These result suggest MjLTL2 involve immune response shrimp<br>\", \"Impact genotype porcine reproductive respiratory syndrome virus interferon<br>response plasmacytoid dendritic cell Porcine reproductive respiratory syndrome<br>PRRS virus PRRSV infection characterize prolonged viremia viral<br>shed consistent incomplete immunity Type interferons IFN essential<br>mount efficient antiviral innate adaptive immune response recent<br>study North American PRRSV genotype isolates induce even<br>strongly inhibit IFN plasmacytoid dendritic cell pDC represent<br>professional IFN produce cell Since inhibition IFN expression<br>might initiate PRRSV pathogenesis characterize PRRSV effect host<br>modify factor IFN response pDC Surprisingly variety type<br>type PRRSV directly stimulate IFN secretion pDC The<br>effect require live virus mediate TLR7 pathway Furthermore<br>IFN significantly enhance pDC production IFN response PRRSV<br>exposure PRRSV inhibition IFN responses enrich pDC stimulate<br>CpG oligodeoxynucleotides weak absent prototype genotype PRRSV suppress<br>response high level suppression induced Chinese highly pathogenic<br>PRRSV isolate Taken together find demonstrate pDC respond<br>PRRSV suggest suppressive activity pDC moderate strain dependent<br>Thus pDC may source systemic IFN response report<br>PRRSV infected animal contribute puzzle immunopathogenesis PRRS<br>\", \"Novel PDE4 Inhibitors Derived Chinese Medicine Forsythia Cyclic<br>adenosine monophosphate cAMP crucial intracellular second messenger molecule<br>convert extracellular molecule intracellular signal transduction pathway generate<br>cell stimulus specific effect Importantly specific phosphodiesterase PDE<br>subtypes control amplitude duration cAMP induce physiological process<br>therefore prominent pharmacological target currently use variety field<br>Here test extract traditional Chinese medicine Forsythia suspense<br>seed use year relieve respiratory symptom Using structural<br>functional analysis find major lignin Forsynthin act immunosuppressant<br>inhibiting PDE4 inflammatory immune cell Moreover several novel<br>selective small molecule derivative Forsythin test vitro murine<br>model viral bacterial pneumonia sepsis cytokine drive systemic<br>inflammation Thus pharmacological target PDE4 may promise strategy<br>immune related disorder characterize amplified host inflammatory response<br>\", \"Evaluation Anti Toxoplasma gondii Effect Ursolic Acid Novel<br>Toxoplasmosis Inhibitor This study carry evaluate anti parasitic<br>effect ursolic acid Toxoplasma gondii gondii induces toxoplasmosis<br>particularly humans The anti parasitic effect ursolic acid<br>gondii infect cell gondii evaluate different specific assay<br>include immunofluorescence stain animal test Ursolic acid effectively<br>inhibit proliferation gondii compare sulfadiazine consistently induce anti<br>gondii activity effect particular formation parasitophorous vacuole membrane<br>PVM host cell markedly decrease treat ursolic acid<br>effectively suppress Moreover survival rate gondii strongly inhibit<br>gondii group treat ursolic acid inhibitory concentration gondii<br>measure The gondii infect mice treat ursolic acid<br>indicate survival rate activity normal group These result<br>demonstrate ursolic acid cause anti gondii action effect<br>strongly block proliferation gondii direct selective gondii inhibitory<br>ability well increase survival gondii infect mouse This<br>study show ursolic acid potential use promising anti<br>gondii candidate substance develop effective anti parasitic drug<br>\", \"Neuronal CCL2 expression drive inflammatory monocyte infiltration brain<br>acute virus infection BackgroundViral encephalitis dangerous compromise need<br>robustly clear pathogen brain need protect neuron bystander<br>injury Theiler murine encephalomyelitis virus TMEV infection C57Bl<br>mouse model viral encephalitis compromise result hippocampal damage<br>permanent neurological sequela previously identify brain infiltrate inflammatory<br>monocyte primary driver hippocampal pathology mechanism involve recruit<br>cell brain unclear MethodsChemokine expression level hippocampus assess<br>microarray ELISA PCR immunofluorescence Monocyte infiltration acute TMEV<br>infection measure flow cytometry CCL2 level manipulate immunodepletion<br>specific removal neuron mice generate cross line express<br>Cre recombinase behind synapsin promoter animal floxed CCL2<br>ResultsInoculation brain TMEV induce hippocampal production proinflammatory chemokine<br>CCL2 peak hour postinfection whereas inoculation inactivate TMEV<br>elicit response Immunofluorescence reveal hippocampal neuron express high<br>level CCL2 timepoint Genetic deletion CCR2 systemic immunodepletion<br>CCL2 abrogate blunted infiltration inflammatory monocyte brain acute<br>infection Specific genetic deletion CCL2 neuron reduce serum<br>hippocampal CCL2 level inhibit inflammatory monocyte infiltration brain<br>ConclusionsWe conclude intracranial inoculation infectious TMEV rapidly induce<br>expression CCL2 neuron cellular source necessary CCR2 dependent<br>infiltration inflammatory monocyte brain acute stage encephalitis These<br>finding highlight unique role neuronal production chemokines initiation<br>leukocytic infiltration infected central nervous system<br>\", \"Combination treatment quercetin resveratrol attenuate high fat diet<br>induce obesity associate inflammation rat via AMPK\\u03b11 SIRT1<br>signal pathway Diet induce obesity associate systemic inflammation<br>consider originate predominantly adipose tissue Quercetin resveratrol two<br>dietary polyphenols exhibit anti inflammatory property anti insulin<br>resistance administer isolation combination CQR remain unknown whether<br>CQR reduces high fat diet HFD induce obesity<br>inflammation rat current study male Wistar rat divide<br>two group one feed normal diet fat one<br>fed HFD fat week Following removal obesity resistant<br>rat HFD group remain rat divide two sub<br>group HFD group HFD CQR group administer day<br>resveratrol day quercetin The result reveal HFD CQR<br>group significantly low body weight week compare HFD<br>group significantly reduce visceral adipose tissue weight adipocyte<br>size Serum lipid profile also significantly ameliorate HFD<br>CQR group CQR attenuate expression systemic proinflammatory adipokines<br>include leptin tumor necrosis factor monocyte chemoattractant protein<br>interleukin also reduce recruitment mast cell epididyotic adipose<br>tissue EAT Furthermore CQR reverse HFD induce suppression<br>adenosine monophosphate activate protein kinase AMPK\\u03b11 phosphorylation sirtuin<br>SIRT1 expression EAT conclusion CQR may suppress obesity<br>associate inflammation via AMPK\\u03b11 SIRT1 signal pathway rat<br>feed HFD<br>\", \"Cytokine nitric oxide pattern dog immunized LBSap vaccine<br>experimental challenge Leishmania chagasi plus saliva Lutzomyia longipalpis<br>study present dog vaccinate Leishmania Leishmania chagasi challenge<br>infection use preparation Leishmania braziliensis promastigote proteins saponin<br>adjuvant LBSap Vaccination LBSap induce prominent type immune<br>response characterize increased level interleukin interferon gamma IFN<br>production peripheral blood mononuclear cell PBMC upon stimulation<br>soluble vaccine antigen Importantly result show type responsiveness<br>sustain challenge infection day chagasi challenge infection PBMCs<br>LBSap vaccinate dog produce IFN concomitant nitric oxide<br>stimulate Leishmania antigen compare PBMCs respective control group<br>saponin treat non treat control dog Moreover transform<br>growth factor TGF decrease supernatant SLcA stimulate PBMCs<br>LBSap group day Bone marrow parasitological analysis reveal<br>decreased frequency parasitism presence vaccine antigen conclude vaccination<br>dog LBSap vaccine induce long lasting type immune<br>response chagasi challenge infection<br>\", \"Methamphetamine induces autophagy pro survival response apoptotic endothelial<br>cell death Kappa opioid receptor Methamphetamine METH psychostimulant<br>high abuse potential severe neurotoxicity Recent study animal<br>model indicate METH impair blood\\u2013brain barrier BBB suggest<br>neurotoxic effect result METH abuse could due barrier<br>disruption report chronic exposure METH disrupt barrier function<br>primary human brain microvascular endothelial cell HBMECs human<br>umbilical vein endothelial cell HUVECs early pro survival<br>response observe follow acute exposure induction autophagic mechanism<br>Acute METH exposure induces early increase Beclin1 LC3<br>recruitment This mediate inactivation protein kinase Akt mammalian<br>target rapamycin mTOR p70S6K pathway upregulation ERK1 Blockade<br>Kappa opioid receptor KOR treatment autophagic inhibitor accelerate<br>METH induce apoptosis suggest early autophagic response survival<br>mechanism endothelial cell mediate kappa opioid receptor Our<br>study indicate kappa opioid receptor therapeutically exploit attenuate<br>METH induce BBB dysfunction<br>\", \"Studies intrathecal humoral immune response canine distemper encephalitis<br>Albumin IgG quantitated paired cerebrospinal fluid CSF serum<br>sample dog demyelinate canine distemper virus CDV infection<br>mean rocket immunoelectrophoresis The IgG index indicator intrathecal<br>immunoglobulin synthesis normal animal non inflammatory demyelinate lesion<br>elevate dog inflammatory myelin lesion Specific antibody CDV<br>myelin quantitated CSF serum dog elevate IgG index<br>Eight dog significant amount antimyelin antibody dog neutralize<br>anti CDV antibody CSF Whereas pathogenetic significance antimyelin<br>antibody remain uncertain intrathecal antiviral immune response provide<br>plausible explanation immunopathologic destruction myelin distemper<br>\", \"Autoimmune Cholangitis SJL Mouse Antigen Non specific Primary<br>biliary cirrhosis PBC autoimmune disease characterize intrahepatic bile<br>duct destruction production anti mitochondrial antibody AMA The<br>absence animal model strike impedance define molecular basis<br>disease Previous work suggest SJL mouse immunize pyruvate<br>dehydrogenase complex PDC major mitochondrial autoantigen PBC lead<br>development lymphoid cell infiltration portal tract model system<br>coin autoimmune cholangitis hypothesize pathology would augment immunization<br>occur presence IFN injection Accordingly SJL mouse immunize<br>PDC purpose control casein Subgroups mouse also treat<br>exogenous IFN expect mouse immunize PDC without IFN<br>develop high titer AMAs contrast mouse immunize casein<br>develop antinuclear antibody More importantly livers mouse immunize<br>PDC immunize casein display lymphoid cell infiltration portal<br>tract irrespective bile duct size Indeed significant difference<br>experimental control group histologic analysis Thus autoimmune cholangitis<br>mouse antigen non specific<br>\", \"Bystander CD4 cell mediate demyelination mouse infect neurotropic<br>coronavirus Demyelination follow infection mouse neurotropic coronavirus mouse<br>hepatitis virus strain JHM MHV immune mediate demonstrate<br>MHV specific CD4 CD8 cell capable cause demyelination<br>independent cell subset Recent work also demonstrate activated<br>bystander CD8 cell mediate significant demyelination The ability<br>bystander CD4 cell mediate demyelination investigate use CD4<br>cell transgenic mouse The result indicate bystander CD4<br>cell unable cause demyelination MHV infect mice despite<br>recruited central nervous system CNS irrespective activation status<br>These result suggest CD4 cell must recognize antigen<br>CNS order cause demyelination<br>\", \"Transcriptional Regulation Antiviral Interferon Stimulated Genes Interferon stimulate<br>gene ISGs group gene product coordinately combat pathogen<br>invasion particular viral infection Transcription ISGs occur rapidly<br>upon pathogen invasion classically provoke via activation Janus<br>kinase signal transducer activator transcription JAK\\u2013STAT pathway mainly<br>interferon IFNs However plethora recent study report variety<br>non canonical mechanism regulate ISG transcription These new<br>study extremely important understand quantitative temporal difference ISG<br>transcription specific circumstance Because canonical non canonical regulatory<br>mechanism essential define nature host defense associate detrimental<br>proinflammatory effect comprehensively review state rapidly evolve field<br>clinical implication recently acquire knowledge respect<br>\", \"Mechanisms HIV non progression robust sustain CD4 cell<br>proliferative response p24 antigen correlate control viraemia lack<br>disease progression long term transfusion acquire HIV infection<br>BACKGROUND Elite non progressors plasma viral load copy<br>antiretroviral naive constitute tiny fraction HIV infect individual<br>After year follow cohort long term non progressors<br>LTNP identify individual transfusion acquire HIV infection remain<br>LTNP year infection retain elite LTNP status examine<br>mechanism differentiate delayed progressors LTNP cohort RESULTS survival<br>advantage confer subject least one host genetic factor<br>HLA chemokine receptor TLR polymorphisms viral attenuate factor<br>defective nef associate slow progression However antiviral immune<br>response differentiate course disease beyond second decade infection<br>stable p24 specific proliferative response associate control viraemia<br>retention non progressor status p24 response absent decline<br>viraemic subject Strong Gag dominant cytotoxic lymphocyte CTL<br>responses identify LTNP Pol dominant CTL nef defective<br>HIV infection CTL associate control viraemia combine p24<br>proliferative response However CTL prevent late disease progression<br>Individuals sustain viral suppression CTL recognise numerous Gag<br>epitopes strong restricted response one two immunodominant epitope<br>effective time fail contain viraemia course study Viral<br>escape mutant HLA B27 restrict Gag p24 epitope<br>detect individual whereas decline negative p24 proliferative response<br>occur concurrent increase viraemia CONCLUSION Detectable viraemia study<br>entry predictive loss LTNP status disease progression differentiate<br>slow progressors elite LTNP retain potent virological control<br>Sustained immunological suppression viraemia independently associate preserve p24<br>proliferative response regardless strength breadth CTL response decline<br>protective p24 response precede correlated loss non progressor<br>status sign disease progression<br>\", \"Visceral target organ systemic Louis encephalitis virus infection<br>hamster Ultrastructural aspect Louis encephalitis virus infection major<br>extraneutral organ tissue suckle hamster examine pancreas exocrine<br>endocrine portion equally affect virus feature apparently unique<br>flavivirus accumulation virus particle type secretory granule organ<br>Virus particle see within myocardial fiber within smooth<br>muscle cell endothelial cell small blood vessel heart<br>intestine lamina propria severely infect virus particle accumulate<br>cell type<br>\", \"Multifunctional viral protein \\u03b334 manipulates nucleolar protein NOP53<br>optimal viral replication HSV ensure efficient virus replication<br>herpes simplex virus type HSV encodes several viral<br>protein counter host defense response upon infection Among<br>protein multifunctional viral protein \\u03b334 crucially interfere disrupts<br>several antiviral pathway multiple level The current study<br>show \\u03b334 utilizes nucleolar protein NOP53 facilitate dephosphorylation<br>eukaryotic initiation factor eIF2\\u03b1 efficient viral translation Our<br>study show ectopic expression NOP53 greatly increase intracellular<br>extracellular viral yield HSV wild strain type interferon<br>deficient Vero cell subtly promote viral replication \\u03b334<br>deletion mutant virus HSV \\u0394\\u03b334 NOP53 migrate nuclei<br>HSV infect cell redistribute incompletely infection either HSV<br>\\u0394\\u03b334 ICP4 deletion mutant virus HSV d120 replication<br>inadequate Ectopic expression \\u03b334 consequently induces cytoplasmic translocation<br>NOP53 response HSV \\u0394\\u03b334 infection Increase NOP53 two<br>form transient transfection vitro expression attenuate phosphorylation level<br>eIF2\\u03b1 HSV infect cell fails affect eIF2\\u03b1 phosphorylation<br>induce HSV \\u0394\\u03b334 infection Knockdown NOP53 impair specific<br>interaction \\u03b334 protein phosphatase PP1\\u03b1 disrupt ability \\u03b334<br>maintain HSV virulence NOP53 knockdown also significantly reduce<br>tissue damage decrease viral yield liver HSV infected<br>mice Our finding expand understand underlie mechanism viral<br>protein \\u03b334 induces NOP53 redistribution cytoplasmic NOP53 facilitate<br>\\u03b334 recruitment PP1\\u03b1 dephosphorylate eIF2\\u03b1 optimal viral replication<br>This paper also demonstrate block specific interaction \\u03b334<br>PP1\\u03b1 would useful approach development antiviral agent<br>\", \"Intestinal microbiota chronic constipation Chronic constipation prevalent burdensome<br>gastrointestinal disorder whose aetiology pathophysiology remain poorly understood<br>likely multifactorial Differences composition intestinal microbiota demonstrate constipated<br>patient healthy control compare Growing evidence indicate alteration<br>intestinal microbiota may contribute constipation constipation relate symptoms<br>The intestinal microbiota collection microorganism live within gastrointestinal<br>tract perform many important health promote function The<br>intestinal microbiota aid breakdown food product absorbable nutrient<br>stimulate host immune system prevent growth pathogenic bacteria<br>produce great variety biologically important compound review summarize<br>current evidence support role intestinal microbiota pathogenesis management<br>chronic constipation The discussion shed light novel mechanism<br>intestinal microbiota gut function interaction invaluable ultimately develop<br>new therapeutic tool treatment chronic constipation<br>\", \"Porcine arterivirus activate pathway I\\u03baB degradation Nuclear factor<br>kappaB critical regulator innate adaptive immune function well<br>cell proliferation survival The present study demonstrate first<br>time virus belong Arteriviridae family activate MARC cell<br>alveolar macrophage porcine reproductive respiratory syndrome virus PRRSV<br>infect cell activation characterize translocation cytoplasm nucleus increase<br>DNA bind activity regulate gene expression activation increase<br>PRRSV infection progress viral dose dependent manner inactivation<br>PRRSV significantly reduce level activation Degradation I\\u03baB protein<br>detect late PRRSV infection overexpression dominant negative form<br>I\\u03baB\\u03b1 I\\u03baB\\u03b1DN significantly suppress activation induce PRRSV However<br>I\\u03baB\\u03b1DN affect viral replication viral cytopathic effect PRRSV<br>infection induce oxidative stress cell generate reactive oxygen<br>specie ROS antioxidant inhibit DNA bind activity PRRSV<br>infect cell suggest ROS mechanism activate PRRSV infection<br>Moreover dependent expression matrix metalloproteinase MMP MMP observe<br>PRRSV infect cell observation implies activation biologically significant<br>aspect PRRSV pathogenesis The result present provide basis<br>understand molecular pathway pathology immune evasion associate disease<br>cause PRRSV<br>\", \"Recapitulation SARS CoV Infection Cholangiocyte Damage Human Liver<br>Organoids The newly emerge pandemic coronavirus SARS CoV<br>pose significant public health threat worldwide However mode<br>virus transmission tissue tropism well establish yet Recent<br>finding substantial liver damage patient ACE2 cholangiocytes healthy<br>liver tissue prompt hypothesize human liver ductal organoids<br>could serve model determine susceptibility mechanism underline liver<br>damage upon SARS CoV infection single cell RNA<br>sequence find long term liver ductal organoid culture<br>preserve human specific ACE2 population cholangiocytes Moreover human<br>liver ductal organoids permissive SARS CoV infection support<br>robust replication Notably virus infection impair barrier bile<br>acid transport function cholangiocytes dysregulation gene involve tight<br>junction formation bile acid transportation could explain bile<br>acid accumulation consequent liver damage patient These result<br>indicate control liver damage cause directly viral infection<br>value treat COVID patient Our finding also provide<br>application human organoids investigate tropism pathogenesis SARS CoV<br>would facilitate novel drug discovery<br>\", \"Endorphin protect mice neurological disease induce murine coronavirus<br>MHV JHM The neurotropic murine coronavirus MHV JHM<br>JHMV cause encephalitis paralytic demyelinate disease susceptible strain<br>mice rat serve model human demyelinate disease multiple<br>sclerosis communication report single intracerebral administration naturally occur<br>neuropeptide endorphin reduce incidence JHMV induce paralytic demyelinating<br>disease 40\\u201350 C57Bl mouse Protection disease accompany significantly<br>reduce virus replication brain early day post infection<br>occur irradiate immunoincompetent mouse The data suggest endorphin<br>engage immune mechanism host resistance JHMV infection protect<br>mouse disease<br>\", \"Atg5 Supports Rickettsia australis Infection Macrophages Vitro Vivo<br>Rickettsiae cause life threaten infection human Macrophages one<br>initial target rickettsiae inoculation tick However remain poorly<br>understood rickettsia remain free macrophage prior establish infection<br>microvascular endothelial cell Here demonstrate concentration Rickettsia australis<br>significantly great infected tissue Atg5 flox flox mouse<br>counterpart Atg5 flox flox Lyz Cre mouse association<br>reduce level interleukin serum The great concentration australis<br>Atg5 flox flox bone marrow derive macrophage BMMs<br>Atg5 flox flox Lyz Cre BMMs vitro abolish<br>exogenous treatment recombinant Rickettsia australis induce significantly increase<br>level light chain LC3 form LC3 LC3 puncta<br>Atg5 competent BMMs Atg5 deficient BMMs p62 turnover<br>observe Further analysis find colocalization LC3 small portion<br>australis Rickettsia contain double membrane bound vacuole BMMs<br>mouse Moreover treatment rapamycin significantly increase concentration australis<br>BMMs compare untreated BMM control Taken together result<br>demonstrate Atg5 favor australis infection mouse macrophages association<br>suppress level production active autophagy flux These data<br>highlight contribution Atg5 macrophages pathogenesis rickettsial disease<br>\"], \"marker\": {\"color\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 7, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9], \"opacity\": 0.8, \"size\": 5}, \"mode\": \"markers+text\", \"text\": [\"d14089\", \"d8648\", \"d7744\", \"d562\", \"d1339\", \"d867\", \"d14250\", \"d4470\", \"d6063\", \"d7320\", \"d9110\", \"d3203\", \"d13305\", \"d8791\", \"d17820\", \"d12105\", \"d18115\", \"d16261\", \"d4781\", \"d11823\", \"d16369\", \"d13060\", \"d15572\", \"d15498\", \"d2584\", \"d12575\", \"d10682\", \"d9981\", \"d17141\", \"d11731\", \"d4022\", \"d8430\", \"d15169\", \"d15696\", \"d13397\", \"d9490\", \"d10968\", \"d18185\", \"d2645\", \"d12604\", \"d4074\", \"d17761\", \"d18445\", \"d3960\", \"d3773\", \"d3995\", \"d15766\", \"d18130\", \"d6907\", \"d9846\", \"d2562\", \"d17309\", \"d7535\", \"d2021\", \"d15150\", \"d10389\", \"d12279\", \"d12164\", \"d9640\", \"d4132\", \"d162\", \"d17486\", \"d8680\", \"d13367\", \"d2149\", \"d1182\", \"d11869\", \"d10799\", \"d17748\", \"d3167\", \"d14843\", \"d17015\", \"d11906\", \"d2897\", \"d14105\", \"d9469\", \"d7176\", \"d13324\", \"d680\", \"d7566\", \"d14773\", \"d8063\", \"d6337\", \"d838\", \"d6508\", \"d16190\", \"d5222\", \"d10947\", \"d11812\", \"d3889\", \"d10463\", \"d13184\", \"d15061\", \"d7147\", \"d2036\", \"d5279\", \"d8163\", \"d3741\", \"d927\", \"d11095\", \"d15013\", \"d1253\", \"d16173\", \"d8578\", \"d14484\", \"d4327\", \"d9059\", \"d14290\", \"d5\", \"d5678\", \"d11209\", \"d1397\", \"d15023\", \"d5179\", \"d6811\", \"d15739\", \"d2132\", \"d13740\", \"d8820\", \"d3065\", \"d12675\", \"d8301\", \"d1939\", \"d2040\", \"d5924\", \"d16108\", \"d4145\", \"d10620\", \"d12277\", \"d17612\", \"d8264\", \"d2290\", \"d12185\", \"d10166\", \"d6481\", \"d9294\", \"d8300\", \"d13095\", \"d720\", \"d14225\", \"d5387\", \"d589\", \"d18098\", \"d7377\", \"d6796\", \"d2750\", \"d14320\", \"d10344\", \"d4988\", \"d2626\", \"d7160\", \"d7253\", \"d2765\", \"d11435\", \"d4977\", \"d10993\", \"d5339\", \"d16468\", \"d12961\", \"d10850\", \"d10124\", \"d9909\", \"d13938\", \"d16922\", \"d3901\", \"d10886\", \"d14136\", \"d7361\", \"d7623\", \"d4447\", \"d9045\", \"d16591\", \"d6916\", \"d17345\", \"d15120\", \"d4562\", \"d8131\", \"d2484\", \"d17087\", \"d7310\", \"d5892\", \"d9752\", \"d10889\", \"d14331\", \"d16944\", \"d13760\", \"d4373\", \"d10336\", \"d307\", \"d7552\", \"d2454\", \"d5662\", \"d7724\", \"d20\", \"d13974\", \"d12299\", \"d1527\", \"d16940\", \"d351\", \"d14941\"], \"textposition\": \"bottom center\", \"type\": \"scatter3d\", \"x\": [-14.758460998535156, -1.5593979358673096, 6.747421741485596, 6.883765697479248, -2.411346673965454, -0.8346594572067261, 16.403703689575195, -2.6025631427764893, 0.37998080253601074, 3.4557249546051025, 4.183548927307129, 10.026732444763184, 3.781789779663086, -1.4950026273727417, 7.55897331237793, 4.557369232177734, 5.658880710601807, 0.3425261378288269, 4.998603820800781, 5.413995265960693, -7.62047004699707, 6.476214408874512, -10.560863494873047, 1.0201822519302368, -6.028838634490967, 7.65266752243042, -7.2347092628479, 5.497418403625488, -13.37186336517334, -4.304635524749756, 10.058675765991211, -6.817416191101074, 12.596207618713379, -5.139933109283447, -7.448812007904053, -14.152554512023926, 1.1511213779449463, -4.533917427062988, -6.416491508483887, 2.294384241104126, 0.6566562056541443, 16.404918670654297, 1.7772338390350342, 16.748964309692383, -5.955498695373535, 10.919309616088867, 10.806684494018555, 4.968904495239258, 10.049087524414062, 10.026217460632324, 9.19050121307373, 2.537949562072754, 8.316756248474121, 5.790224075317383, 7.640994548797607, 5.345418453216553, 15.553848266601562, 8.579740524291992, 8.986109733581543, 9.633508682250977, 11.697203636169434, 0.23885835707187653, 16.09632110595703, 12.810994148254395, 7.243946075439453, 3.932512044906616, 24.179935455322266, 2.746701717376709, 8.430453300476074, 10.499292373657227, 3.120927333831787, 20.12628173828125, 3.4869747161865234, 13.60280990600586, 9.806488037109375, 21.379653930664062, 2.558500289916992, 10.909183502197266, 3.56125545501709, 21.033973693847656, -14.05478572845459, -16.57465171813965, -1.3367843627929688, -13.307145118713379, -16.453580856323242, -15.139055252075195, -1.8622174263000488, 0.05556175485253334, -10.736652374267578, -19.357322692871094, -13.682723999023438, -10.212357521057129, -7.861693382263184, -14.547215461730957, -12.927875518798828, -3.7351186275482178, -20.303131103515625, -15.890814781188965, -9.135748863220215, -10.826556205749512, 6.1342267990112305, -0.3738477826118469, 7.487332344055176, 30.136960983276367, -2.5513947010040283, -5.390527725219727, 5.887366771697998, 8.052579879760742, 13.293617248535156, 4.808326244354248, 10.043403625488281, -3.5196635723114014, 6.6335601806640625, 2.9951698780059814, 0.29168975353240967, 6.529192924499512, 4.226467609405518, -1.2130746841430664, 4.188297271728516, -4.819647789001465, -2.115407943725586, -25.086721420288086, 4.199974060058594, -2.869131088256836, 2.5430192947387695, 0.42532938718795776, -17.16278839111328, -11.467446327209473, -10.77347469329834, -9.709280967712402, -9.465895652770996, 2.909158229827881, -3.768646240234375, -12.478612899780273, -6.63132381439209, -3.7310984134674072, 1.6054363250732422, -16.94380760192871, -6.166832447052002, -9.590152740478516, -5.706259727478027, -4.294061183929443, -14.269194602966309, -15.570259094238281, -10.372300148010254, -8.8964204788208, -2.7124671936035156, -16.93022918701172, -17.421279907226562, -13.190397262573242, -8.851069450378418, -13.061746597290039, 0.8270415663719177, -10.215428352355957, -9.530171394348145, -13.979098320007324, -0.5847605466842651, -11.785591125488281, 5.391536712646484, -15.707098960876465, -26.508220672607422, -10.411067008972168, -3.69337797164917, -26.843833923339844, -22.305767059326172, -8.76103401184082, -11.703912734985352, -17.93974494934082, -5.281763076782227, -24.14638900756836, -19.89970588684082, -25.94806480407715, -10.971765518188477, -8.165480613708496, -24.874969482421875, -20.45370864868164, -21.707887649536133, -5.501662254333496, -25.095794677734375, -12.045530319213867, 11.584168434143066, 21.315977096557617, 20.771621704101562, 8.416850090026855, 19.81843376159668, 15.907463073730469, 11.395108222961426, 27.16779899597168, 16.052215576171875, 21.82332420349121, 19.344573974609375, 16.805221557617188, 14.092907905578613, 5.83634614944458, 26.53861427307129, 4.98627233505249, 26.4468994140625, 2.7311956882476807, 21.46772003173828, 25.420705795288086], \"y\": [-4.947906494140625, 21.530719757080078, 11.641042709350586, -8.893977165222168, -6.3615031242370605, 11.74229907989502, -13.68713092803955, -6.243285179138184, -7.480836391448975, -10.96567153930664, -7.130857944488525, -26.05312156677246, 21.379444122314453, -8.241707801818848, 3.5384418964385986, -7.460297107696533, -11.53585147857666, 22.37990951538086, -13.386549949645996, -6.987489700317383, -13.794800758361816, -11.084579467773438, -4.296988010406494, -10.264849662780762, -9.333945274353027, 24.255992889404297, -7.300210475921631, -18.632814407348633, -13.90951919555664, -6.872381687164307, -17.572864532470703, -7.5159831047058105, 1.1450226306915283, -10.423210144042969, -6.449275493621826, -16.88384437561035, -18.565135955810547, 0.14962859451770782, -20.037508010864258, -13.37198257446289, -0.8824686408042908, 6.639695644378662, 5.535533905029297, -3.1713361740112305, 27.799394607543945, -4.4159674644470215, -13.081083297729492, -15.47574234008789, -8.76048469543457, -6.959766387939453, -21.68290138244629, -15.649198532104492, -13.559431076049805, 0.640267014503479, -17.840627670288086, -4.495349884033203, -9.208786010742188, 3.600609302520752, -11.008600234985352, -16.14583396911621, -10.70869255065918, -17.472869873046875, -1.2469837665557861, -12.436103820800781, -8.184508323669434, -16.159080505371094, -7.293708801269531, -9.56209659576416, -9.453003883361816, -3.054520845413208, -14.538853645324707, -5.555963516235352, -20.262531280517578, 2.7448980808258057, -18.623096466064453, -2.913465738296509, -18.06801414489746, -9.783639907836914, -17.708904266357422, -12.525520324707031, 12.671036720275879, 5.2731404304504395, 10.070235252380371, 11.276267051696777, 7.772252082824707, 5.686242580413818, 6.506288051605225, 10.034111976623535, -1.7777026891708374, 7.703449726104736, -1.7844003438949585, -1.885746717453003, 4.389527320861816, 14.96146011352539, 8.747055053710938, 9.850485801696777, 18.522878646850586, 9.55283260345459, 3.033768653869629, 18.607511520385742, 7.023423671722412, 6.2332682609558105, -0.4593025743961334, -2.7598886489868164, 0.7187276482582092, 10.81802749633789, 2.583587169647217, 1.6916465759277344, 7.570189476013184, 0.45168960094451904, 4.387833595275879, 0.4821181893348694, 7.75460958480835, -17.01590919494629, 14.845973014831543, -0.6806573271751404, 13.247868537902832, 0.3135074973106384, -5.470859527587891, -1.031191110610962, 22.72294044494629, 1.9842605590820312, -2.10861873626709, 15.064866065979004, 13.649025917053223, 14.214743614196777, 4.908204078674316, -0.4396931231021881, 22.347644805908203, 5.04646110534668, 12.658980369567871, 10.21087646484375, 5.817843437194824, 4.8763322830200195, 5.808327674865723, 23.176259994506836, 15.678879737854004, 1.7033122777938843, 0.8097997307777405, 8.073330879211426, -18.07140350341797, -23.67438316345215, -9.846227645874023, -15.045543670654297, -6.997922420501709, -20.66811180114746, -8.309974670410156, -16.792007446289062, -18.625282287597656, -16.73149299621582, -19.827186584472656, -21.479284286499023, -10.396490097045898, 2.805621862411499, -18.237638473510742, -16.659374237060547, 16.446256637573242, -16.578649520874023, 10.010359764099121, -16.664154052734375, 3.525963306427002, -9.402915954589844, 10.967324256896973, -5.0580267906188965, 8.265405654907227, -6.096765041351318, -6.860886096954346, -6.492287635803223, -14.202545166015625, 7.2191033363342285, -4.233539581298828, -0.7309901118278503, 7.093688488006592, 4.578941822052002, -7.951097011566162, 7.770303249359131, 1.8222918510437012, -1.569687008857727, -7.323306083679199, 2.432588815689087, 0.9233314394950867, 2.008662700653076, 12.50221061706543, 17.588884353637695, 9.219015121459961, 17.099952697753906, -1.6538732051849365, 6.248056411743164, 8.573320388793945, 9.005380630493164, 5.377816677093506, 3.163182258605957, 0.18416450917720795, 5.2689971923828125, -1.138353705406189, 8.783913612365723, 1.9255558252334595, 4.130966663360596, 8.945588111877441, 6.776242733001709], \"z\": [4.459395408630371, 8.469430923461914, 0.20084016025066376, 21.47673797607422, 28.59128189086914, -1.4297348260879517, 11.38979434967041, 25.41541290283203, 18.877641677856445, 26.485610961914062, 26.608680725097656, -0.6651536822319031, 2.8161261081695557, 23.38088607788086, 19.77296257019043, 22.696481704711914, 17.77130699157715, 7.3568220138549805, 22.11231803894043, 26.00506019592285, 16.624404907226562, 15.031289100646973, 7.005248546600342, -13.688152313232422, 17.915189743041992, 7.879239559173584, 7.182387828826904, 12.23013687133789, -11.617864608764648, 2.4723472595214844, 7.019991874694824, 4.721248149871826, 1.2311240434646606, 9.77799129486084, -1.6333314180374146, -9.938632011413574, -12.707927703857422, -1.379171371459961, 7.490042686462402, 16.709228515625, -7.739842891693115, 14.14622688293457, 0.3689606189727783, 7.726563453674316, 14.266908645629883, 4.948354721069336, 16.197328567504883, 0.4952230155467987, 9.46934700012207, 7.495045185089111, -3.7651264667510986, -3.2459216117858887, -2.1964709758758545, 6.176910877227783, 1.1944117546081543, 0.6415218114852905, 7.049211025238037, -9.643829345703125, 9.648802757263184, 9.402849197387695, -12.259742736816406, -8.943656921386719, -13.999763488769531, -3.1746928691864014, -17.414783477783203, -20.275705337524414, -8.801103591918945, -18.853126525878906, -7.524425506591797, -20.141817092895508, -17.828697204589844, -6.145440101623535, -12.187325477600098, -10.839193344116211, 1.6999067068099976, -15.089362144470215, -20.207874298095703, -12.031496047973633, -19.78229522705078, -9.796719551086426, -1.888786792755127, 3.2597222328186035, 29.900341033935547, -2.977961301803589, -10.684250831604004, -0.7481951117515564, -9.375555992126465, -7.592347145080566, -5.384964466094971, -11.123490333557129, 3.7723422050476074, -7.92080545425415, -3.816434144973755, -3.014146566390991, -12.37424087524414, -3.4395761489868164, 0.45067960023880005, -16.808639526367188, -0.5904219150543213, -12.80771255493164, -9.528319358825684, -16.24529457092285, -20.70800018310547, 1.7105482816696167, -15.445150375366211, -21.231489181518555, -13.05862808227539, -21.392173767089844, -21.787492752075195, -20.058073043823242, -22.46089744567871, -21.46479034423828, -24.761760711669922, 5.323232173919678, -18.77907943725586, -23.827512741088867, -21.044357299804688, -13.219524383544922, -19.985458374023438, -17.930953979492188, 10.91976261138916, 6.3736677169799805, 28.442569732666016, 8.104130744934082, 12.545429229736328, 5.0339579582214355, 9.593249320983887, 16.77005386352539, 1.0873000621795654, 21.59762191772461, 16.7332706451416, 16.142549514770508, 16.40657615661621, 7.82993221282959, 19.43091583251953, 5.502141952514648, 19.01779556274414, 13.824527740478516, 13.633443832397461, 6.748111248016357, -2.331172466278076, -0.638937771320343, 11.6112642288208, 0.7673212885856628, -11.544002532958984, 4.23929500579834, -15.741869926452637, -4.756656169891357, 2.927236557006836, 3.082214117050171, -6.556437969207764, 6.693364143371582, 14.146547317504883, -26.156150817871094, 1.5376991033554077, 2.9968338012695312, -6.749704360961914, -7.750785827636719, 8.707465171813965, -4.874820709228516, 3.5443265438079834, -17.866634368896484, -5.809909343719482, -4.849856376647949, -11.52678108215332, -10.9984712600708, 8.994112968444824, 4.765151500701904, -10.40442180633545, -4.0974578857421875, 2.20236873626709, -0.05857357010245323, -25.381254196166992, -23.87941551208496, -5.818136692047119, -17.36553955078125, 1.162163496017456, -17.92291259765625, -2.7773709297180176, -17.787765502929688, -6.031368732452393, 3.963510274887085, -7.664010524749756, -10.580368041992188, 6.711097240447998, -3.2372989654541016, 13.913147926330566, -6.788234233856201, 14.916961669921875, 8.050895690917969, 8.665105819702148, -0.8336024880409241, 8.889707565307617, 14.90350341796875, -4.366927623748779, 0.3667706847190857, -4.178377151489258, 6.592097759246826, 11.740421295166016, 5.824801445007324]}],\n",
       "                        {\"template\": {\"data\": {\"bar\": [{\"error_x\": {\"color\": \"#2a3f5f\"}, \"error_y\": {\"color\": \"#2a3f5f\"}, \"marker\": {\"line\": {\"color\": \"#E5ECF6\", \"width\": 0.5}}, \"type\": \"bar\"}], \"barpolar\": [{\"marker\": {\"line\": {\"color\": \"#E5ECF6\", \"width\": 0.5}}, \"type\": \"barpolar\"}], \"carpet\": [{\"aaxis\": {\"endlinecolor\": \"#2a3f5f\", \"gridcolor\": \"white\", \"linecolor\": \"white\", \"minorgridcolor\": \"white\", \"startlinecolor\": \"#2a3f5f\"}, \"baxis\": {\"endlinecolor\": \"#2a3f5f\", \"gridcolor\": \"white\", \"linecolor\": \"white\", \"minorgridcolor\": \"white\", \"startlinecolor\": \"#2a3f5f\"}, \"type\": \"carpet\"}], \"choropleth\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"type\": \"choropleth\"}], \"contour\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"colorscale\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]], \"type\": \"contour\"}], \"contourcarpet\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"type\": \"contourcarpet\"}], \"heatmap\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"colorscale\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]], \"type\": \"heatmap\"}], \"heatmapgl\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"colorscale\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]], \"type\": \"heatmapgl\"}], \"histogram\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"histogram\"}], \"histogram2d\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"colorscale\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]], \"type\": \"histogram2d\"}], \"histogram2dcontour\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"colorscale\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]], \"type\": \"histogram2dcontour\"}], \"mesh3d\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"type\": \"mesh3d\"}], \"parcoords\": [{\"line\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"parcoords\"}], \"pie\": [{\"automargin\": true, \"type\": \"pie\"}], \"scatter\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scatter\"}], \"scatter3d\": [{\"line\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scatter3d\"}], \"scattercarpet\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scattercarpet\"}], \"scattergeo\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scattergeo\"}], \"scattergl\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scattergl\"}], \"scattermapbox\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scattermapbox\"}], \"scatterpolar\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scatterpolar\"}], \"scatterpolargl\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scatterpolargl\"}], \"scatterternary\": [{\"marker\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"type\": \"scatterternary\"}], \"surface\": [{\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}, \"colorscale\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]], \"type\": \"surface\"}], \"table\": [{\"cells\": {\"fill\": {\"color\": \"#EBF0F8\"}, \"line\": {\"color\": \"white\"}}, \"header\": {\"fill\": {\"color\": \"#C8D4E3\"}, \"line\": {\"color\": \"white\"}}, \"type\": \"table\"}]}, \"layout\": {\"annotationdefaults\": {\"arrowcolor\": \"#2a3f5f\", \"arrowhead\": 0, \"arrowwidth\": 1}, \"coloraxis\": {\"colorbar\": {\"outlinewidth\": 0, \"ticks\": \"\"}}, \"colorscale\": {\"diverging\": [[0, \"#8e0152\"], [0.1, \"#c51b7d\"], [0.2, \"#de77ae\"], [0.3, \"#f1b6da\"], [0.4, \"#fde0ef\"], [0.5, \"#f7f7f7\"], [0.6, \"#e6f5d0\"], [0.7, \"#b8e186\"], [0.8, \"#7fbc41\"], [0.9, \"#4d9221\"], [1, \"#276419\"]], \"sequential\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]], \"sequentialminus\": [[0.0, \"#0d0887\"], [0.1111111111111111, \"#46039f\"], [0.2222222222222222, \"#7201a8\"], [0.3333333333333333, \"#9c179e\"], [0.4444444444444444, \"#bd3786\"], [0.5555555555555556, \"#d8576b\"], [0.6666666666666666, \"#ed7953\"], [0.7777777777777778, \"#fb9f3a\"], [0.8888888888888888, \"#fdca26\"], [1.0, \"#f0f921\"]]}, \"colorway\": [\"#636efa\", \"#EF553B\", \"#00cc96\", \"#ab63fa\", \"#FFA15A\", \"#19d3f3\", \"#FF6692\", \"#B6E880\", \"#FF97FF\", \"#FECB52\"], \"font\": {\"color\": \"#2a3f5f\"}, \"geo\": {\"bgcolor\": \"white\", \"lakecolor\": \"white\", \"landcolor\": \"#E5ECF6\", \"showlakes\": true, \"showland\": true, \"subunitcolor\": \"white\"}, \"hoverlabel\": {\"align\": \"left\"}, \"hovermode\": \"closest\", \"mapbox\": {\"style\": \"light\"}, \"paper_bgcolor\": \"white\", \"plot_bgcolor\": \"#E5ECF6\", \"polar\": {\"angularaxis\": {\"gridcolor\": \"white\", \"linecolor\": \"white\", \"ticks\": \"\"}, \"bgcolor\": \"#E5ECF6\", \"radialaxis\": {\"gridcolor\": \"white\", \"linecolor\": \"white\", \"ticks\": \"\"}}, \"scene\": {\"xaxis\": {\"backgroundcolor\": \"#E5ECF6\", \"gridcolor\": \"white\", \"gridwidth\": 2, \"linecolor\": \"white\", \"showbackground\": true, \"ticks\": \"\", \"zerolinecolor\": \"white\"}, \"yaxis\": {\"backgroundcolor\": \"#E5ECF6\", \"gridcolor\": \"white\", \"gridwidth\": 2, \"linecolor\": \"white\", \"showbackground\": true, \"ticks\": \"\", \"zerolinecolor\": \"white\"}, \"zaxis\": {\"backgroundcolor\": \"#E5ECF6\", \"gridcolor\": \"white\", \"gridwidth\": 2, \"linecolor\": \"white\", \"showbackground\": true, \"ticks\": \"\", \"zerolinecolor\": \"white\"}}, \"shapedefaults\": {\"line\": {\"color\": \"#2a3f5f\"}}, \"ternary\": {\"aaxis\": {\"gridcolor\": \"white\", \"linecolor\": \"white\", \"ticks\": \"\"}, \"baxis\": {\"gridcolor\": \"white\", \"linecolor\": \"white\", \"ticks\": \"\"}, \"bgcolor\": \"#E5ECF6\", \"caxis\": {\"gridcolor\": \"white\", \"linecolor\": \"white\", \"ticks\": \"\"}}, \"title\": {\"x\": 0.05}, \"xaxis\": {\"automargin\": true, \"gridcolor\": \"white\", \"linecolor\": \"white\", \"ticks\": \"\", \"title\": {\"standoff\": 15}, \"zerolinecolor\": \"white\", \"zerolinewidth\": 2}, \"yaxis\": {\"automargin\": true, \"gridcolor\": \"white\", \"linecolor\": \"white\", \"ticks\": \"\", \"title\": {\"standoff\": 15}, \"zerolinecolor\": \"white\", \"zerolinewidth\": 2}}}, \"title\": {\"text\": \"prod cat projection\"}},\n",
       "                        {\"responsive\": true}\n",
       "                    ).then(function(){\n",
       "                            \n",
       "var gd = document.getElementById('28fc782b-a989-402c-b253-708cca81caed');\n",
       "var x = new MutationObserver(function (mutations, observer) {{\n",
       "        var display = window.getComputedStyle(gd).display;\n",
       "        if (!display || display === 'none') {{\n",
       "            console.log([gd, 'removed!']);\n",
       "            Plotly.purge(gd);\n",
       "            observer.disconnect();\n",
       "        }}\n",
       "}});\n",
       "\n",
       "// Listen for the removal of the full notebook cells\n",
       "var notebookContainer = gd.closest('#notebook-container');\n",
       "if (notebookContainer) {{\n",
       "    x.observe(notebookContainer, {childList: true});\n",
       "}}\n",
       "\n",
       "// Listen for the clearing of the current output cell\n",
       "var outputEl = gd.closest('.output');\n",
       "if (outputEl) {{\n",
       "    x.observe(outputEl, {childList: true});\n",
       "}}\n",
       "\n",
       "                        })\n",
       "                };\n",
       "                });\n",
       "            </script>\n",
       "        </div>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fig = go.Figure(data=plot, layout=layout)\n",
    "plt.iplot(fig)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
